{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.6.5"
    },
    "colab": {
      "name": "scraper.ipynb",
      "provenance": [],
      "include_colab_link": true
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/ashwinamrutphale/aml-homeworks/blob/master/scraper.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "GAv8JFIaRIp-",
        "colab_type": "text"
      },
      "source": [
        "# Scraping UIUC CS faculty homepages"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jVXCTmoWRIqA",
        "colab_type": "text"
      },
      "source": [
        "## System setup \n",
        "\n",
        "Before we start, make sure to install the required libraries\n",
        "    \n",
        "    pip install bs4\n",
        "    pip install selenium\n",
        "\n",
        "Since UIUC's website has some javascript rendered HTML content, we'll be using Selenium for scraping the content loaded dynamically by javascript. For this,you would also need to download a selenium supported browser webdriver.\n",
        "\n",
        "e.g. For Chrome, download the appropriate webdriver from here: http://chromedriver.chromium.org/downloads, unzip it and save in current directory."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "x3dwT4f0RPGI",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 408
        },
        "outputId": "48d030a3-d5ef-43d9-8fb0-d97763f46d1a"
      },
      "source": [
        "!pip install bs4\n",
        "!pip install selenium\n",
        "!apt-get update # to update ubuntu to correctly run apt install\n",
        "!apt install chromium-chromedriver\n",
        "!cp /usr/lib/chromium-browser/chromedriver /usr/bin"
      ],
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: bs4 in /usr/local/lib/python3.6/dist-packages (0.0.1)\n",
            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.6/dist-packages (from bs4) (4.6.3)\n",
            "Requirement already satisfied: selenium in /usr/local/lib/python3.6/dist-packages (3.141.0)\n",
            "Requirement already satisfied: urllib3 in /usr/local/lib/python3.6/dist-packages (from selenium) (1.24.3)\n",
            "Ign:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease\n",
            "Ign:2 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease\n",
            "Hit:3 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release\n",
            "Hit:4 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release\n",
            "Get:5 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]\n",
            "Hit:6 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease\n",
            "Hit:8 http://archive.ubuntu.com/ubuntu bionic InRelease\n",
            "Hit:9 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease\n",
            "Hit:11 http://ppa.launchpad.net/marutter/c2d4u3.5/ubuntu bionic InRelease\n",
            "Get:12 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]\n",
            "Hit:13 http://archive.ubuntu.com/ubuntu bionic-backports InRelease\n",
            "Fetched 177 kB in 2s (108 kB/s)\n",
            "Reading package lists... Done\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "chromium-chromedriver is already the newest version (77.0.3865.90-0ubuntu0.18.04.1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 134 not upgraded.\n",
            "cp: '/usr/lib/chromium-browser/chromedriver' and '/usr/bin/chromedriver' are the same file\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZX3XpFqQRIqB",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "from bs4 import BeautifulSoup\n",
        "from selenium import webdriver \n",
        "from selenium.webdriver.chrome.options import Options\n",
        "import re \n",
        "import urllib"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "I03pA0HMRIqF",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 71
        },
        "outputId": "a07d4f1b-bc64-403e-c5ad-426587964bfe"
      },
      "source": [
        "#create a webdriver object and set options for headless browsing\n",
        "import sys\n",
        "sys.path.insert(0,'/usr/lib/chromium-browser/chromedriver')\n",
        "from selenium import webdriver\n",
        "chrome_options = webdriver.ChromeOptions()\n",
        "chrome_options.add_argument('--headless')\n",
        "chrome_options.add_argument('--no-sandbox')\n",
        "chrome_options.add_argument('--disable-dev-shm-usage')\n",
        "browser = webdriver.Chrome('chromedriver',chrome_options=chrome_options)\n"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.6/dist-packages/ipykernel_launcher.py:8: DeprecationWarning: use options instead of chrome_options\n",
            "  \n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FDuWNcnGRIqJ",
        "colab_type": "text"
      },
      "source": [
        "If you visit UIUC's CS Faculty Directory Listing: https://cs.illinois.edu/people/faculty/department-faculty , you'll notice that it has all the faculty listed there. \n",
        "Clicking on a faculty's image takes you to the Faculty Profile page. \n",
        "The profile page has more info about the faculty and sometimes contains the faculty hompage url under the Overview Tab > For More Information which is ultimately what we want to scrape.\n",
        "\n",
        "Before we start scraping, we'll define some helper functions"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Bc5RW7r5RIqK",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#uses webdriver object to execute javascript code and get dynamically loaded webcontent\n",
        "def get_js_soup(url,browser):\n",
        "    browser.get(url)\n",
        "    res_html = browser.execute_script('return document.body.innerHTML')\n",
        "    soup = BeautifulSoup(res_html,'html.parser') #beautiful soup object to be used for parsing html content\n",
        "    return soup\n",
        "\n",
        "#tidies extracted text \n",
        "def process_bio(bio):\n",
        "    bio = bio.encode('ascii',errors='ignore').decode('utf-8')       #removes non-ascii characters\n",
        "    bio = re.sub('\\s+',' ',bio)       #repalces repeated whitespace characters with single space\n",
        "    return bio\n",
        "\n",
        "''' More tidying\n",
        "Sometimes the text extracted HTML webpage may contain javascript code and some style elements. \n",
        "This function removes script and style tags from HTML so that extracted text does not contain them.\n",
        "'''\n",
        "def remove_script(soup):\n",
        "    for script in soup([\"script\", \"style\"]):\n",
        "        script.decompose()\n",
        "    return soup\n",
        "\n",
        "\n",
        "#Checks if bio_url is a valid faculty homepage\n",
        "def is_valid_homepage(bio_url,dir_url):\n",
        "    try:\n",
        "        #sometimes the homepage url points to the faculty profile page\n",
        "        #which should be treated differently from an actual homepage\n",
        "        ret_url = urllib.request.urlopen(bio_url).geturl() \n",
        "    except:\n",
        "        return False       #unable to access bio_url\n",
        "    urls = [re.sub('((https?://)|(www.))','',url) for url in [ret_url,dir_url]] #removes url scheme (https,http) or www \n",
        "    return not(urls[0]== urls[1])"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "d2dat_ShRIqP",
        "colab_type": "text"
      },
      "source": [
        "We will now start scraping.\n",
        "\n",
        "First, let's get links to all Faculty Profile pages by scraping the Directory Listing. You can use your browser's developer tools to find the required links within the HTML content. In Chrome, this can be done by right cliking on the webpage and choosing Inspect. Some basic knowledge of HTML and CSS would be required. After a bit of digging, you should notice that the link can be found under the <a\\> tag under <div\\> with class \"photo nocaption\" as shown:\n",
        "\n",
        "\n",
        "![Screen%20Shot%202019-02-11%20at%2012.39.36%20PM.png](attachment:Screen%20Shot%202019-02-11%20at%2012.39.36%20PM.png)\n",
        "\n",
        "Now we can specify exactly what needs to be extracted from the directory listing page"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6yi8Ql1JRIqQ",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#extracts all Faculty Profile page urls from the Directory Listing Page\n",
        "def scrape_dir_page(dir_url,browser):\n",
        "    print ('-'*20,'Scraping directory page','-'*20)\n",
        "    faculty_links = []\n",
        "    faculty_base_url = 'https://biology.ucsd.edu'\n",
        "    #execute js on webpage to load faculty listings on webpage and get ready to parse the loaded HTML \n",
        "    soup = get_js_soup(dir_url,browser)     \n",
        "    for link_holder in soup.find_all('td',class_='name'): #get list of all <div> of class 'photo nocaption'\n",
        "        #print(link_holder)\n",
        "        try:\n",
        "          rel_link = link_holder.find('a')['href'] #get url\n",
        "          print(rel_link)\n",
        "          #url returned is relative, so we need to add base url\n",
        "          faculty_links.append(faculty_base_url+rel_link) \n",
        "        except Exception:\n",
        "          pass  # o\n",
        "        \n",
        "    print ('-'*20,'Found {} faculty profile urls'.format(len(faculty_links)),'-'*20)\n",
        "    return faculty_links"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sYOOc14ERIqU",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "f8ed9ddf-0970-4c25-f612-278931414f7a"
      },
      "source": [
        "dir_url = 'https://biology.ucsd.edu/publicinfo/dwc?action=faculty_research_list' #url of directory listings of CS faculty\n",
        "faculty_links = scrape_dir_page(dir_url,browser)\n",
        "print(faculty_links)"
      ],
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "-------------------- Scraping directory page --------------------\n",
            "/research/faculty/sackerman\n",
            "/research/faculty/oakbari\n",
            "/research/faculty/eallen\n",
            "/research/faculty/njallen\n",
            "/research/faculty/carmour\n",
            "/research/faculty/kasahina\n",
            "/research/faculty/jsayres\n",
            "/research/faculty/eazim\n",
            "/research/faculty/tsb\n",
            "/research/faculty/mbanghart\n",
            "/research/faculty/adbarton\n",
            "/research/faculty/e1bennett\n",
            "/research/faculty/dberg\n",
            "/research/faculty/ebier\n",
            "/research/faculty/blbloodgood\n",
            "/research/faculty/sbriggs\n",
            "/research/faculty/sbrody\n",
            "/research/faculty/wbusch\n",
            "/research/faculty/mabutler\n",
            "/research/faculty/hcang\n",
            "/research/faculty/schalasani\n",
            "/research/faculty/lchao\n",
            "/research/faculty/lchilukuri\n",
            "/research/faculty/adchisholm\n",
            "/research/faculty/schoe\n",
            "/research/faculty/jchory\n",
            "/research/faculty/mchrispeels\n",
            "/research/faculty/ecleland\n",
            "/research/faculty/abcoleman\n",
            "/research/faculty/hcookandersen\n",
            "/research/faculty/j2cooke\n",
            "/research/faculty/kcooper\n",
            "/research/faculty/ncrawford\n",
            "/research/faculty/mddaugherty\n",
            "/research/faculty/midavid\n",
            "/research/faculty/abdesai\n",
            "/research/faculty/rdoolittle\n",
            "/research/faculty/rjdutton\n",
            "/research/faculty/jecker\n",
            "/research/faculty/dengle\n",
            "/research/faculty/mestelle\n",
            "/research/faculty/rmevans\n",
            "/research/faculty/efarley\n",
            "/research/faculty/gfeng\n",
            "/research/faculty/rafirtel\n",
            "/research/faculty/dforbes\n",
            "/research/faculty/pfortes\n",
            "/research/faculty/kfrench\n",
            "/research/faculty/fgage\n",
            "/research/faculty/pgeiduschek\n",
            "/research/faculty/tgentner\n",
            "/research/faculty/sglasgow\n",
            "/research/faculty/jwgolden\n",
            "/research/faculty/sgolden\n",
            "/research/faculty/agoldrath\n",
            "/research/faculty/mgoulding\n",
            "/research/faculty/bjgrant\n",
            "/research/faculty/mgreen\n",
            "/research/faculty/rgreenspan\n",
            "/research/faculty/shalpain\n",
            "/research/faculty/rhampton\n",
            "/research/faculty/shan\n",
            "/research/faculty/nhao\n",
            "/research/faculty/dhargreaves\n",
            "/research/faculty/jhasty\n",
            "/research/faculty/shedrick\n",
            "/research/faculty/dhelinski\n",
            "/research/faculty/mhetzer\n",
            "/research/faculty/dholway\n",
            "/research/faculty/ahuffaker\n",
            "/research/faculty/enfuhui\n",
            "/research/faculty/arhunter\n",
            "/research/faculty/thwa\n",
            "/research/faculty/jbelmonte\n",
            "/research/faculty/xij014\n",
            "/research/faculty/yijin\n",
            "/research/faculty/k5jones\n",
            "/research/faculty/ajuavinett\n",
            "/research/faculty/s2jun\n",
            "/research/faculty/jkadonaga\n",
            "/research/faculty/smkaech\n",
            "/research/faculty/jkarlseder\n",
            "/research/faculty/akiger\n",
            "/research/faculty/ckintner\n",
            "/research/faculty/dkleinfeld\n",
            "/research/faculty/jkohn\n",
            "/research/faculty/tkomiyama\n",
            "/research/faculty/wkristan\n",
            "/research/faculty/mkronenberg\n",
            "/research/faculty/skryazhimskiy\n",
            "/research/faculty/ckurle\n",
            "/research/faculty/rlande\n",
            "/research/faculty/slauberth\n",
            "/research/faculty/j2law\n",
            "/research/faculty/kflee\n",
            "/research/faculty/aleschziner\n",
            "/research/faculty/jleutgeb\n",
            "/research/faculty/sleutgeb\n",
            "/research/faculty/blillemeier\n",
            "/research/faculty/byl015\n",
            "/research/faculty/smlo\n",
            "/research/faculty/lil034\n",
            "/research/faculty/vlundblad\n",
            "/research/faculty/jlykkeandersen\n",
            "/research/faculty/dmlyumkis\n",
            "/research/faculty/emacagno\n",
            "/research/faculty/cmadigan\n",
            "/research/faculty/rmalinow\n",
            "/research/faculty/tmarkow\n",
            "/research/faculty/smayfield\n",
            "/research/faculty/lmcdonnell\n",
            "/research/faculty/wmcginnis\n",
            "/research/faculty/smel\n",
            "/research/faculty/jrmeyer\n",
            "/research/faculty/mmontal\n",
            "/research/faculty/montminy\n",
            "/research/faculty/cmurre\n",
            "/research/faculty/seneal\n",
            "/research/faculty/jnieh\n",
            "/research/faculty/animmerjahn\n",
            "/research/faculty/mniwarosen\n",
            "/research/faculty/coshea\n",
            "/research/faculty/koegema\n",
            "/research/faculty/mtowens\n",
            "/research/faculty/spanda\n",
            "/research/faculty/apasquinelli\n",
            "/research/faculty/gpatrick\n",
            "/research/faculty/gpekkurnaz\n",
            "/research/faculty/mperry\n",
            "/research/faculty/kpetrie\n",
            "/research/faculty/sapfaff\n",
            "/research/faculty/lpillus\n",
            "/research/faculty/jpogliano\n",
            "/research/faculty/kpogliano\n",
            "/research/faculty/jposakony\n",
            "/research/faculty/pprice\n",
            "/research/faculty/jprunedapaz\n",
            "/research/faculty/sreckpeterson\n",
            "/research/faculty/preinagel\n",
            "/research/faculty/sarifkin\n",
            "/research/faculty/cmroot\n",
            "/research/faculty/mrosenfeld\n",
            "/research/faculty/kroy\n",
            "/research/faculty/arussell\n",
            "/research/faculty/oryder\n",
            "/research/faculty/asaghatelian\n",
            "/research/faculty/msaier\n",
            "/research/faculty/mschaechter\n",
            "/research/faculty/ischeffler\n",
            "/research/faculty/eschmelz\n",
            "/research/faculty/jischroeder\n",
            "/research/faculty/tsejnowski\n",
            "/research/faculty/gsshadel\n",
            "/research/faculty/rshaw\n",
            "/research/faculty/jshurin\n",
            "/research/faculty/dsmith\n",
            "/research/faculty/lgsmith\n",
            "/research/faculty/nspitzer\n",
            "/research/faculty/sstockwell\n",
            "/research/faculty/c8su\n",
            "/research/faculty/ssubramani\n",
            "/research/faculty/gsuel\n",
            "/research/faculty/xis101\n",
            "/research/faculty/etour\n",
            "/research/faculty/dtraver\n",
            "/research/faculty/etroemel\n",
            "/research/faculty/kmtye\n",
            "/research/faculty/evilla\n",
            "/research/faculty/gwahl\n",
            "/research/faculty/jw800\n",
            "/research/faculty/cware\n",
            "/research/faculty/sawasserman\n",
            "/research/faculty/gwienhausen\n",
            "/research/faculty/jwilhelm\n",
            "/research/faculty/cwills\n",
            "/research/faculty/yangxu\n",
            "/research/faculty/myanofsky\n",
            "/research/faculty/dyelon\n",
            "/research/faculty/d7zhang\n",
            "/research/faculty/y3zhao\n",
            "/research/faculty/yezheng\n",
            "/research/faculty/yzou\n",
            "/research/faculty/eizuniga\n",
            "-------------------- Found 183 faculty profile urls --------------------\n",
            "['https://biology.ucsd.edu/research/faculty/sackerman', 'https://biology.ucsd.edu/research/faculty/oakbari', 'https://biology.ucsd.edu/research/faculty/eallen', 'https://biology.ucsd.edu/research/faculty/njallen', 'https://biology.ucsd.edu/research/faculty/carmour', 'https://biology.ucsd.edu/research/faculty/kasahina', 'https://biology.ucsd.edu/research/faculty/jsayres', 'https://biology.ucsd.edu/research/faculty/eazim', 'https://biology.ucsd.edu/research/faculty/tsb', 'https://biology.ucsd.edu/research/faculty/mbanghart', 'https://biology.ucsd.edu/research/faculty/adbarton', 'https://biology.ucsd.edu/research/faculty/e1bennett', 'https://biology.ucsd.edu/research/faculty/dberg', 'https://biology.ucsd.edu/research/faculty/ebier', 'https://biology.ucsd.edu/research/faculty/blbloodgood', 'https://biology.ucsd.edu/research/faculty/sbriggs', 'https://biology.ucsd.edu/research/faculty/sbrody', 'https://biology.ucsd.edu/research/faculty/wbusch', 'https://biology.ucsd.edu/research/faculty/mabutler', 'https://biology.ucsd.edu/research/faculty/hcang', 'https://biology.ucsd.edu/research/faculty/schalasani', 'https://biology.ucsd.edu/research/faculty/lchao', 'https://biology.ucsd.edu/research/faculty/lchilukuri', 'https://biology.ucsd.edu/research/faculty/adchisholm', 'https://biology.ucsd.edu/research/faculty/schoe', 'https://biology.ucsd.edu/research/faculty/jchory', 'https://biology.ucsd.edu/research/faculty/mchrispeels', 'https://biology.ucsd.edu/research/faculty/ecleland', 'https://biology.ucsd.edu/research/faculty/abcoleman', 'https://biology.ucsd.edu/research/faculty/hcookandersen', 'https://biology.ucsd.edu/research/faculty/j2cooke', 'https://biology.ucsd.edu/research/faculty/kcooper', 'https://biology.ucsd.edu/research/faculty/ncrawford', 'https://biology.ucsd.edu/research/faculty/mddaugherty', 'https://biology.ucsd.edu/research/faculty/midavid', 'https://biology.ucsd.edu/research/faculty/abdesai', 'https://biology.ucsd.edu/research/faculty/rdoolittle', 'https://biology.ucsd.edu/research/faculty/rjdutton', 'https://biology.ucsd.edu/research/faculty/jecker', 'https://biology.ucsd.edu/research/faculty/dengle', 'https://biology.ucsd.edu/research/faculty/mestelle', 'https://biology.ucsd.edu/research/faculty/rmevans', 'https://biology.ucsd.edu/research/faculty/efarley', 'https://biology.ucsd.edu/research/faculty/gfeng', 'https://biology.ucsd.edu/research/faculty/rafirtel', 'https://biology.ucsd.edu/research/faculty/dforbes', 'https://biology.ucsd.edu/research/faculty/pfortes', 'https://biology.ucsd.edu/research/faculty/kfrench', 'https://biology.ucsd.edu/research/faculty/fgage', 'https://biology.ucsd.edu/research/faculty/pgeiduschek', 'https://biology.ucsd.edu/research/faculty/tgentner', 'https://biology.ucsd.edu/research/faculty/sglasgow', 'https://biology.ucsd.edu/research/faculty/jwgolden', 'https://biology.ucsd.edu/research/faculty/sgolden', 'https://biology.ucsd.edu/research/faculty/agoldrath', 'https://biology.ucsd.edu/research/faculty/mgoulding', 'https://biology.ucsd.edu/research/faculty/bjgrant', 'https://biology.ucsd.edu/research/faculty/mgreen', 'https://biology.ucsd.edu/research/faculty/rgreenspan', 'https://biology.ucsd.edu/research/faculty/shalpain', 'https://biology.ucsd.edu/research/faculty/rhampton', 'https://biology.ucsd.edu/research/faculty/shan', 'https://biology.ucsd.edu/research/faculty/nhao', 'https://biology.ucsd.edu/research/faculty/dhargreaves', 'https://biology.ucsd.edu/research/faculty/jhasty', 'https://biology.ucsd.edu/research/faculty/shedrick', 'https://biology.ucsd.edu/research/faculty/dhelinski', 'https://biology.ucsd.edu/research/faculty/mhetzer', 'https://biology.ucsd.edu/research/faculty/dholway', 'https://biology.ucsd.edu/research/faculty/ahuffaker', 'https://biology.ucsd.edu/research/faculty/enfuhui', 'https://biology.ucsd.edu/research/faculty/arhunter', 'https://biology.ucsd.edu/research/faculty/thwa', 'https://biology.ucsd.edu/research/faculty/jbelmonte', 'https://biology.ucsd.edu/research/faculty/xij014', 'https://biology.ucsd.edu/research/faculty/yijin', 'https://biology.ucsd.edu/research/faculty/k5jones', 'https://biology.ucsd.edu/research/faculty/ajuavinett', 'https://biology.ucsd.edu/research/faculty/s2jun', 'https://biology.ucsd.edu/research/faculty/jkadonaga', 'https://biology.ucsd.edu/research/faculty/smkaech', 'https://biology.ucsd.edu/research/faculty/jkarlseder', 'https://biology.ucsd.edu/research/faculty/akiger', 'https://biology.ucsd.edu/research/faculty/ckintner', 'https://biology.ucsd.edu/research/faculty/dkleinfeld', 'https://biology.ucsd.edu/research/faculty/jkohn', 'https://biology.ucsd.edu/research/faculty/tkomiyama', 'https://biology.ucsd.edu/research/faculty/wkristan', 'https://biology.ucsd.edu/research/faculty/mkronenberg', 'https://biology.ucsd.edu/research/faculty/skryazhimskiy', 'https://biology.ucsd.edu/research/faculty/ckurle', 'https://biology.ucsd.edu/research/faculty/rlande', 'https://biology.ucsd.edu/research/faculty/slauberth', 'https://biology.ucsd.edu/research/faculty/j2law', 'https://biology.ucsd.edu/research/faculty/kflee', 'https://biology.ucsd.edu/research/faculty/aleschziner', 'https://biology.ucsd.edu/research/faculty/jleutgeb', 'https://biology.ucsd.edu/research/faculty/sleutgeb', 'https://biology.ucsd.edu/research/faculty/blillemeier', 'https://biology.ucsd.edu/research/faculty/byl015', 'https://biology.ucsd.edu/research/faculty/smlo', 'https://biology.ucsd.edu/research/faculty/lil034', 'https://biology.ucsd.edu/research/faculty/vlundblad', 'https://biology.ucsd.edu/research/faculty/jlykkeandersen', 'https://biology.ucsd.edu/research/faculty/dmlyumkis', 'https://biology.ucsd.edu/research/faculty/emacagno', 'https://biology.ucsd.edu/research/faculty/cmadigan', 'https://biology.ucsd.edu/research/faculty/rmalinow', 'https://biology.ucsd.edu/research/faculty/tmarkow', 'https://biology.ucsd.edu/research/faculty/smayfield', 'https://biology.ucsd.edu/research/faculty/lmcdonnell', 'https://biology.ucsd.edu/research/faculty/wmcginnis', 'https://biology.ucsd.edu/research/faculty/smel', 'https://biology.ucsd.edu/research/faculty/jrmeyer', 'https://biology.ucsd.edu/research/faculty/mmontal', 'https://biology.ucsd.edu/research/faculty/montminy', 'https://biology.ucsd.edu/research/faculty/cmurre', 'https://biology.ucsd.edu/research/faculty/seneal', 'https://biology.ucsd.edu/research/faculty/jnieh', 'https://biology.ucsd.edu/research/faculty/animmerjahn', 'https://biology.ucsd.edu/research/faculty/mniwarosen', 'https://biology.ucsd.edu/research/faculty/coshea', 'https://biology.ucsd.edu/research/faculty/koegema', 'https://biology.ucsd.edu/research/faculty/mtowens', 'https://biology.ucsd.edu/research/faculty/spanda', 'https://biology.ucsd.edu/research/faculty/apasquinelli', 'https://biology.ucsd.edu/research/faculty/gpatrick', 'https://biology.ucsd.edu/research/faculty/gpekkurnaz', 'https://biology.ucsd.edu/research/faculty/mperry', 'https://biology.ucsd.edu/research/faculty/kpetrie', 'https://biology.ucsd.edu/research/faculty/sapfaff', 'https://biology.ucsd.edu/research/faculty/lpillus', 'https://biology.ucsd.edu/research/faculty/jpogliano', 'https://biology.ucsd.edu/research/faculty/kpogliano', 'https://biology.ucsd.edu/research/faculty/jposakony', 'https://biology.ucsd.edu/research/faculty/pprice', 'https://biology.ucsd.edu/research/faculty/jprunedapaz', 'https://biology.ucsd.edu/research/faculty/sreckpeterson', 'https://biology.ucsd.edu/research/faculty/preinagel', 'https://biology.ucsd.edu/research/faculty/sarifkin', 'https://biology.ucsd.edu/research/faculty/cmroot', 'https://biology.ucsd.edu/research/faculty/mrosenfeld', 'https://biology.ucsd.edu/research/faculty/kroy', 'https://biology.ucsd.edu/research/faculty/arussell', 'https://biology.ucsd.edu/research/faculty/oryder', 'https://biology.ucsd.edu/research/faculty/asaghatelian', 'https://biology.ucsd.edu/research/faculty/msaier', 'https://biology.ucsd.edu/research/faculty/mschaechter', 'https://biology.ucsd.edu/research/faculty/ischeffler', 'https://biology.ucsd.edu/research/faculty/eschmelz', 'https://biology.ucsd.edu/research/faculty/jischroeder', 'https://biology.ucsd.edu/research/faculty/tsejnowski', 'https://biology.ucsd.edu/research/faculty/gsshadel', 'https://biology.ucsd.edu/research/faculty/rshaw', 'https://biology.ucsd.edu/research/faculty/jshurin', 'https://biology.ucsd.edu/research/faculty/dsmith', 'https://biology.ucsd.edu/research/faculty/lgsmith', 'https://biology.ucsd.edu/research/faculty/nspitzer', 'https://biology.ucsd.edu/research/faculty/sstockwell', 'https://biology.ucsd.edu/research/faculty/c8su', 'https://biology.ucsd.edu/research/faculty/ssubramani', 'https://biology.ucsd.edu/research/faculty/gsuel', 'https://biology.ucsd.edu/research/faculty/xis101', 'https://biology.ucsd.edu/research/faculty/etour', 'https://biology.ucsd.edu/research/faculty/dtraver', 'https://biology.ucsd.edu/research/faculty/etroemel', 'https://biology.ucsd.edu/research/faculty/kmtye', 'https://biology.ucsd.edu/research/faculty/evilla', 'https://biology.ucsd.edu/research/faculty/gwahl', 'https://biology.ucsd.edu/research/faculty/jw800', 'https://biology.ucsd.edu/research/faculty/cware', 'https://biology.ucsd.edu/research/faculty/sawasserman', 'https://biology.ucsd.edu/research/faculty/gwienhausen', 'https://biology.ucsd.edu/research/faculty/jwilhelm', 'https://biology.ucsd.edu/research/faculty/cwills', 'https://biology.ucsd.edu/research/faculty/yangxu', 'https://biology.ucsd.edu/research/faculty/myanofsky', 'https://biology.ucsd.edu/research/faculty/dyelon', 'https://biology.ucsd.edu/research/faculty/d7zhang', 'https://biology.ucsd.edu/research/faculty/y3zhao', 'https://biology.ucsd.edu/research/faculty/yezheng', 'https://biology.ucsd.edu/research/faculty/yzou', 'https://biology.ucsd.edu/research/faculty/eizuniga']\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ydrh0Tl5RIqZ",
        "colab_type": "text"
      },
      "source": [
        "Now let's scrape the Faculty Profile pages to find faculty homepage urls. \n",
        "\n",
        "The faculty homepage urls are listed under the Overview tab > For more information on the Faculty profile page. So, again let's try to find where these urls are placed in the HTML content. \n",
        "\n",
        "You'll notice the Overview tab is the <div\\> with id <overview\\>. \n",
        "\n",
        "There could multiple headers (<h\\>) under this tab e.g. Research etc., we're looking for the one with text \"For more information\".\n",
        "\n",
        "There could be multiple listings (<li\\>) under the header e.g. Reasearch Group Page, etc. There is no specific way to identify the one corresponding to the homepage. But, if you look at a few profile pages, you'll notice a few patterns which are generally found in the anchor text accompanying the homepage url. e.g. presence of substrings like 'page' or 'site' are good indicators as long as other words like 'group' or 'course' are not present. \n",
        "\n",
        "Ofcourse, there might still be a few cases where the patterns do not work. For such cases and for cases where the homepage url is actually not listed on the profile page, we will treat the faculty profile page as the homepage.\n",
        "\n",
        "\n",
        "![Screen%20Shot%202019-02-11%20at%201.36.02%20PM.png](attachment:Screen%20Shot%202019-02-11%20at%201.36.02%20PM.png)\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "HuiseIfXRIqa",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def scrape_faculty_page(fac_url,browser):\n",
        "\n",
        "    soup = get_js_soup(fac_url,browser)\n",
        "    homepage_found = False\n",
        "    bio_url = ''\n",
        "    bio = ''\n",
        "    #find overview tab\n",
        "    try:\n",
        "        overview_tab = soup.find('div',id='bio')\n",
        "        research_tab = soup.find('div',id='research')\n",
        "        publication_tab = soup.find('div',id='pub')\n",
        "    except:\n",
        "        print (soup)\n",
        "    #define patterns for the anchor text of homepage url\n",
        "    faculty_name = soup.find('h1',id='facName').get_text().lower() #find faculty name\n",
        "    bio = bio + faculty_name + ' '\n",
        "    try:\n",
        "      bio = bio + process_bio(overview_tab.get_text(separator=' ') + ' ')\n",
        "    except:\n",
        "      pass\n",
        "    try:\n",
        "      bio = bio + process_bio(research_tab.get_text(separator=' ') + ' ')\n",
        "    except:\n",
        "      pass\n",
        "    try:\n",
        "      bio = bio + process_bio(publication_tab.get_text(separator=' ') + ' ')\n",
        "    except:\n",
        "      pass\n",
        "    print(bio)\n",
        "    \n",
        "#     homepage_txts = ['site','page',' '+faculty_last_name]\n",
        "#     exceptions = ['course ','research','group','cs','mirror','google scholar']\n",
        "#     #find the homepage url and extract all text from it\n",
        "#     for hdr in overview_tab.find_all('h2'):  #first find the required header\n",
        "#         if hdr.text.lower() == 'for more information':\n",
        "#             next_tag = hdr.find_next('li')\n",
        "#             #find <li> which has homepage url\n",
        "#             while next_tag is not None: \n",
        "#                 cand = next_tag.find('a')\n",
        "#                 next_tag = next_tag.next_sibling    #sibling means element present at the same level\n",
        "#                 cand_text = cand.get_text().lower()\n",
        "#                 if (any(hp_txt in cand_text for hp_txt in homepage_txts) and \n",
        "#                     not any(e in cand_text for e in exceptions)): #compare text to predefined patterns\n",
        "#                     bio_url = cand['href'] \n",
        "#                     homepage_found = True\n",
        "#                     #check if homepage url is valid\n",
        "#                     if not(is_valid_homepage(bio_url,fac_url)): \n",
        "#                         homepage_found = False\n",
        "#                     else:\n",
        "#                         try:`\n",
        "#                             bio_soup = remove_script(get_js_soup(bio_url,browser)) \n",
        "#                         except:\n",
        "#                             print ('Could not access {}'.format(bio_url))\n",
        "#                             homepage_found = False\n",
        "#                     break \n",
        "#             if homepage_found:\n",
        "#                 #get all the text from homepage(bio) since there's no easy to filter noise like navigation bar etc\n",
        "#                 bio = process_bio(bio_soup.get_text(separator=' ')) \n",
        "\n",
        "#     if not homepage_found:\n",
        "#         bio_url = fac_url #treat faculty profile page as homepage\n",
        "#         #we're only interested in some parts of the profile page namely the address\n",
        "#         #and information listed under the Overview, Research, Publication and Awards tab\n",
        "#         bio = soup.find('div',class_='dirprofile fullpage').get_text(separator=' ')+': '\n",
        "#         for tab in soup.find_all('div',class_='tab-pane'):\n",
        "#             bio += tab.get_text(separator=' ')+'. '\n",
        "#         bio = process_bio(bio)\n",
        "\n",
        "    return fac_url,bio"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WbR5rlBbRIqe",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "fde69e5a-c956-447d-c5a1-d2880e1365ea"
      },
      "source": [
        "#Scrape all faculty homepages using profile page urls\n",
        "bio_urls, bios = [],[]\n",
        "tot_urls = len(faculty_links)\n",
        "for i,link in enumerate(faculty_links):\n",
        "    print ('-'*20,'Scraping faculty url {}/{}'.format(i+1,tot_urls),'-'*20)\n",
        "    bio_url,bio = scrape_faculty_page(link,browser)\n",
        "    bio_urls.append(bio_url)\n",
        "    bios.append(bio)"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "-------------------- Scraping faculty url 1/183 --------------------\n",
            "susan ackerman  Biography Susan Ackerman received her Ph.D. from UCLA and was a postdoctoral fellow at University of Illinois Medical School and the Wistar Institute. Prior to her move to UCSD in 2016, Dr. Ackerman was a Professor at The Jackson Laboratory in Bar Harbor, Maine where she was a faculty member for nineteen years. She has been an Investigator of the Howard Hughes Medical Institute since 2005.  Research The goal of our laboratory is to define the molecular pathways necessary to maintain homeostasis in both developing and aging mammalian neurons. To do this we utilize forward genetics to identify mutations that are associated with loss of neurons in the aging mouse brain. To further dissect pathways underlying homeostatic disruption and disease, we also use forward genetics to identify genetic variants that enhance or suppress neural phenotypes. Our approach allows the identification, without a priori assumptions, of molecules critical for neuron homeostasis and survival, and indeed we have discovered disruptions in several novel pathways that were not previously associated with loss of neuronal function or survival. We are particularly interested in the role of alterations in translation elongation, translational fidelity, proteostasis, and RNA metabolism in neuronal function.  Select Publications Ishimura R, Nagy G, Dotu I, Chuang JH, Ackerman SL. Activation of GCN2 by ribosome stalling links translation elongation with translation initiation. eLife. 2016. 10.7554 e14295. Liu Y, Lee JW, Ackerman SL. Mutations in the microtubule-associated protein 1A (Map1a) gene cause Purkinje cell degeneration. J Neurosci. 2015. 35:4587-98. Jia Y, Jucius TJ, Cook SA, Ackerman SL. Loss of Clcc1 results in ER stress, misfolded protein accumulation, and neurodegeneration. J Neurosci. 2015. 35:3001-9. Liu Y, Satz JS, Vo MN, Nangle LA, Schimmel P, Ackerman SL. Deficiencies in tRNA synthetase editing activity cause cardioproteinopathy. Proc Natl Acad Sci U S A. 2014. 111:17570-5. Ishimura R, Nagy G, Dotu I, Zhou H, Yang XL, Schimmel P, Senju S, Nishimura Y, Chuang JH, Ackerman SL. RNA function. Ribosome stalling induced by mutation of a CNS-specific tRNA causes neurodegeneration. Science. 2014. 345:455-9. Liu Y, Zaun HC, Orlowski J, Ackerman SL. CHP1-mediated NHE1 biosynthetic maturation is required for Purkinje cell axon homeostasis. J Neurosci. 2013. 33:12656-69. Jia Y, Mu JC, Ackerman SL. Mutation of a U2 snRNA gene causes global disruption of alternative splicing and neurodegeneration. Cell. 2012 148:296-308. Zhao L, Spassieva SD, Jucius TJ, Shultz LD, Shick HE, Macklin WB, Hannun YA, Obeid LM, Ackerman SL. A deficiency of ceramide biosynthesis causes cerebellar Purkinje cell neurodegeneration and lipofuscin accumulation. PLoS Genet. 2011 (5):e1002063. Kim D, Ackerman SL. The UNC5C netrin receptor regulates dorsal guidance of mouse hindbrain axons. J Neurosci. 2011 31:2167-79. Zhao L, Rosales C, Seburn K, Ron D, Ackerman SL. Alteration of the unfolded protein response modifies neurodegeneration in a mouse model of Marinesco-Sjgren syndrome. Hum Mol Genet. 2010 19:25-35. \n",
            "-------------------- Scraping faculty url 2/183 --------------------\n",
            "omar akbari  Biography In May of 2005, Omar S. Akbari received a B.S./M.S. in Biotechnology from the University of Nevada, Reno . In December of 2008, he received a Ph.D. in Cell and Molecular Biology from the University of Nevada, Reno where he studied the role cis-regulatory modules play in cellular identity along the anterio-posterior axis in developing Drosophila melanogaster embryos. In May of 2009, he joined the laboratory of professor Bruce A. Hay at the California Institute of Technology as a Senior Postdoctoral Scholar to innovate synthetic biology of disease vectors. In summer of 2015, he became an Assistant Professor of Entomology in the Center for Infectious Disease Vector Research (CIDVR) at the University of California, Riverside . In fall of 2017, he joined the faculty as an Assistant Professor in the Cell and Developmental Biology Section , within the Division of Biological Sciences , at the University of California, San Diego .  Research Mosquitoes are perhaps the most dangerous animals in the world. They are the primary vectors for major human diseases such as yellow fever, zika, dengue fever, and malaria, which together infect hundreds of millions of humans worldwide, killing millions each year, with over 50% of the worlds population presently at risk (WHO). There are currently no effective vaccines for either dengue fever, ZIKA or malaria, and mosquitoes, including the disease-causing agents they transmit, are rapidly evolving resistance to commonly used pesticides and anti-malarial drugs, respectively. Therefore, given the number of infections and deaths, current approaches for prevention of mosquito-borne diseases are immeasurably inefficient. What remains critical for vector control is the development of catalytic approaches requiring only small efforts that can generate long lasting solutions. With rapid advances in insect genetic engineering, mathematical modeling of wild populations, synthetic biology, and the comprehensive understanding of dengue and plasmodium lifecycles in mosquitoes, unique opportunities have arisen to eliminate infectious diseases through genetic manipulation of wild insect vector populations. Therefore, our research focuses on studying the basic genetics and physiology of mosquitoes with the overall goal of developing innovative, novel, creative, synthetic biology inspired genetic control technologies for reducing the burden of mosquito-borne diseases on humans. The underlying hypothesis inspiring this work is that the introduction and spread of genes that prevent mosquitoes from transmitting pathogens should, in theory, lead to reduced transmission of these pathogens resulting in reductions of human infections and/or death. To test this hypothesis, we first need a broad understanding of the biology of the mosquito that can be used to develop gene-based strategies for engineering mosquitoes that are resistant to pathogens; secondly, we need to engineer mosquitoes that are resistant to all types of infections; and lastly, we need to develop tools to rapidly drive these laboratory developed genes into wild mosquito populations. Together, these aims can conceivably provide a foundation that has the potential to revolutionize vector control of mosquitoes.  Select Publications Li, M., Bui, M., Ferree, P.M*, Akbari, O.S.*, Embryo microinjection and transplantation technique for genome manipulation of Nasonia vitripennis. (2017). J. Vis. Exp., e56990, doi:10.3791/56990 (2017) *Corresponding Author Li M., Bui M., Yang T., White B.J., Akbari O.S.* Germline Cas9 Expression Yields Highly Efficient Genome Engineering in a Major Worldwide Disease Vector, Aedes aegypti. (2017). PNAS ; published ahead of print November 14, 2017, doi:10.1073/pnas.1711538114 *Corresponding Author Li1, Y., Jing, X.A., Aldrich, J.C., Chen, J., Akbari, O.S., Ferree, P.M.* Unique sequence organization and small RNA expression of a selfish B chromosome. (2017) Aug 5. doi: 10.1007/s00412-017-0641-x. Adelman, Z., Akbari, O.S., Bauer, J., Bier, E., Bloss, C., Carter, S., Callender, C., Costero-Saint Denis, A., Cowhey P., Dass, B., Delborne, J., Devereaux, M., Ellsworth, P., Gantz V., Gibson C., Hay B.A., Hoddle M., James A.A., James, S., Jorgenson, L., Kalichman M., Marshall J., McGinnis W., Newman J., Pearson A., Quemada H., Rudenko L., Shelton A., Vinetz J., Weisman, J., Wong B., Wozniak C., Rules of the road for insect gene drive research and testing. Nat Biotechnol. 2017 Aug 8;35(8):716-718. doi: 10.1038/nbt.3926. Marshall J.M.*, Buchman, A.B., Sanchez C. H.M., Akbari O.S.*, Overcoming evolved resistance to population-suppressing homing-based gene drives. Scientific Reports. 2017 June 19. doi: DOI: 10.1038/s41598-017-02744-7 *Co-Corresponding Authors Li M., Au. L.U, Douglah, B. J. White, D., Ferree, P.M.*, Akbari. O.S.*, Generation of heritable germline mutations in Nasonia vitripennis using CRISPR/Cas9. *Co-Corresponding Authors, Scientific Reports. 2017 Apr 19;7(1):901. doi: 10.1038/s41598-017-00990-3. Choi H.M., Calvert C.R., Husain N., Huss D., Barsi J.C., Deverman B.E., Hunter R.C., Kato M., Lee S.M., Abelin A.C., Rosenthal A.Z., Akbari O.S., Li Y., Hay B.A., Sternberg P.W., Patterson P.H., Davidson E.H., Mazmanian S.K., Prober D.A., van de Rijn M., Leadbetter J.R., Newman D.K., Readhead C., Bronner M.E., Wold B., Lansford R., Sauka-Spengler T., Fraser S.E., Pierce N.A. Mapping a multiplexed zoo of mRNA expression. Development. 2016 Oct 1;143(19):3632-3637. Cheng, B., Kuppanda, N., Aldrich, J., Akbari, O.S.*, Ferree, P.M.* Male-killing Spiroplasma Alters Behavior of the Dosage Compensation Complex during Drosophila melanogaster embryogenesis. Curr Biol. 2016 May 23;26(10):1339-45. doi: 10.1016/j.cub.2016.03.050. *Co-Corresponding Authors Hall, A.B., Papathanos, P.A., Sharma, A., Cheng, C., Akbari, O.S., Assour, L., Bergman, N.H., Cagnetti, A., Crisanti, A., Dottorini, T., Fiorentini, E., Galizi, R., Hnath, J., Jiang, X., Koren, S., Nolan, T., Radune, D., Sharakhova, M.V., Steele, A., Timoshevskiy, V.A., Windbichler, N., Zhang, S., Hahn, M.W., Phillippy A.M., Emrich, S.J., Sharakhov, I.V., Tu, Z., Besansky, N.J. Radical remodeling of the Y chromosome in a recent radiation of malaria mosquitoes. Proc Natl Acad Sci U S A. 2016 Apr 12;113(15):E2114-23. doi: 10.1073/pnas.1525164113. Champer J, Buchman A, Akbari OS.* Cheating evolution: engineering gene drives to manipulate the fate of wild populations. Nat Rev Genet. 2016 Mar;17(3):146-59. doi: 10.1038/nrg.2015.34. Review. *Corresponding Author Marshall, J.M., Akbari, O.S. Chapter 9 - Gene Drive Strategies for Population Replacement, Genetic Control of Malaria and Dengue, 1st Edition, Zach Adelman. 2015. ISBN: 978-0-12-800246 Ferree PM*, Fang C, Mastrodimos M, Hay BA, Amrhein H, Akbari OS.* Identification of Genes Uniquely Expressed in the Germ-Line Tissues of the Jewel Wasp Nasonia vitripennis. G3 (Bethesda). 2015 Oct 13;5(12):2647-53. doi: 10.1534/g3.115.021386. *Co-Corresponding Authors Li J., Olvera A.I., Akbari O.S., Moradian A., Sweredoski M.J., Hess S., Hay B.A. Vectored antibody gene delivery mediates long-term contraception. Curr Biol. 2015 Oct 5;25(19):R820-2. doi: 10.1016/j.cub.2015.08.002. Akbari O.S., Bellen H.J., Bier E., Bullock S.L., Burt A., Church G.M., Cook K.R., Duchek P., Edwards O.R., Esvelt K.M., Gantz V.M., Golic K.G., Gratz S.J., Harrison M.M., Hayes K.R., James A.A., Kaufman T.C., Knoblich J., Malik H.S., Matthews K.A., O'Connor-Giles K.M., Parks A.L., Perrimon N., Port F., Russell S., Ueda R., Wildonger J. BIOSAFETY. Safeguarding gene drive experiments in the laboratory. Science. 2015 Aug 28;349(6251):927-9. doi: 10.1126/science.aac7932. Akbari, O.S., Papathanos, P., Kennedy, K., Sandler, J., and Hay, B.A. Identification of germline transcriptional regulatory elements in Aedes aegypti. Scientific Reports, 2014. DOI: 10.1038/srep03954. Papathanos P.P, Windbichler N., Akbari, O.S., Sex Ratio Manipulation for Insect Population Control. Chapter 6. 2014. Transgenic Insects: Techniques and Applications edited by Benedict, M.Q. DOI. 10.1079/9781780644516.0083 Akbari, O.S.*, Antosheckin, I., Hay, B.A, Ferree, P.M*. Transcriptome profiling of the Nasonia Vitripennis testis reveals novel transcripts expressed from the selfish B. Chromosome, Paternal Sex Ratio. G3, 2013. DOI:pii: g3.113.007583v1. *Co-Corresponding Authors. Akbari, O.S., Marshall, J., Chen, CH., Huang, H., Antosheckin, I., and Hay, B.A. Novel Synthetic Medea selfish genetic elements drive population replacement in Drosophila, and a theoretical exploration of Medea-dependent population suppression. ACS Synthetic Biology, 2012. DOI: 10.1021/sb300079h. Akbari, O.S., Antoshechkin, I., Amrhein, H., Williams, B., Sandler, J.,Diloreto, R., Buchman. A., and Hay, B.A. The Complete Developmental Transcriptome of the Mosquito Aedes aegypti, an invasive species and disease vector. G3, 2013. G3 DOI:pii: g3.113.006742v1. Akbari, O.S., Matzen, K., Marshall, J.M, Huang,H., and Hay, B.A. A Synthetic gene-drive system for local, reversible modification and suppression of insect populations. Current Biology, 2013. DOI: 10.1016/j.cub.2013.02.059. Ho, M.C., Schiller, B.J., Akbari, O.S., Bae, E., and Drewell, R.A. Disruption of the abdominal-B promoter tethering element results in a loss of long-range enhancer-directed Hox gene expression in Drosophila. PLoS One, 2011. 6, e16283. PMC3025016. Oliver, D., Sheehan, B., South, H., Akbari, O.S., and Pai, C.Y. The chromosomal association/dissociation of the chromatin insulator protein Cp190 of Drosophila melanogaster is mediated by the BTB/POZ domain and two acidic regions. BMC Cell Biol 11, 2010. 101. PMC3022720. Akbari, O.S., Oliver, D., Eyer, K., and Pai, C.Y. An Entry/Gateway cloning system for general expression of genes with molecular tags in Drosophila melanogaster. BMC Cell Biol, 2009. 10, 8. PMC2654426. Akbari, O.S., Bae, E., Johnsen, H., Villaluz, A., Wong, D., and Drewell, R.A. A novel promoter-tethering element regulates enhancer-driven gene expression at the bithorax complex in the Drosophila embryo. Development, 2008. 135, 123-131. PMC2205987, NIHMS37020. Akbari, O.S., Schiller, B.J., Goetz, S.E., Ho, M.C., Bae, E., and Drewell, R.A. The abdominal-B promoter tethering element mediates promoter-enhancer specificity at the Drosophila bithorax complex. Fly (Austin), 2007. 1, 337-339. PMC2394718, NIHMS41586. Akbari, O.S., Bousum, A., Bae, E., and Drewell, R.A. Unraveling cis-regulatory mechanisms at the abdominal-A and Abdominal-B genes in the Drosophila bithorax complex. Developmental Biology, 2006. 293, 294-304. PMC16545794. \n",
            "-------------------- Scraping faculty url 3/183 --------------------\n",
            "eric allen  Biography Eric Allen received his B.S. from the University of Oregon and his Ph.D. from the Scripps Institution of Oceanography at UC San Diego. He performed postdoctoral research at the University of California, Berkeley as an NSF Postdoctoral Fellow in Microbial Biology and joined the faculty of Biological Sciences and the Scripps Institution of Oceangraphy at UCSD in 2006.  Research Research in our laboratory centers upon the use of environmentally-derived genome sequence information (metagenomics) to explore the genetic potential, ecology, and evolution of environmental microbial populations. The nature of this work relies equally upon field-based collections, bioinformatics (tool development, genome assembly, annotation, and comparative analyses) and the tools of molecular ecology and genetics. Together, these approaches enable us to utilize environmental genome sequence data to understand natural microbial phenomena including metabolic activities, biosynthetic processes, environmental adaptation, lateral gene transfer events, biogeographical patterning, biogeochemical cycling, and microbe-microbe/microbe-host interactions. Current projects include: Marine Metazoan Microbiomes Marine microorganisms are major contributors to biogeochemical cycling and the maintenance of global homeostasis, yet relatively little is known about the diversity and function of microbes found in intimate association with marine animals. Our current understanding of these complex microbial communities has largely been built upon cultivation-based studies and limited contemporary studies using cultivation-independent methods that survey 16S rRNA gene marker diversity. We use both vertebrate (marine fish) and invertebrate (sponge, mollusks) to investigate the host-associated microbial biology of our oceans, employing metabolic functional gene markers and deep metagenomic sequencing and assembly to complement additional phylogenetic techniques such as 16S rRNA gene amplicon sequencing. We are also developing laboratory-based bioreactor cultivation systems to propagate host-associated microbial communities under controlled conditions to allow experimental manipulation of host microbiota in vitro . These studies are revealing novel microbial diversity previously unidentified in the marine environment and provide new insight into symbiotic relationships between marine animals and their microbial partners. Genomics of Marine Microbial HOC Synthesis Halogenated organic compounds (HOCs) are widely distributed among organisms occupying higher trophic levels of the marine food web. Chemical analysis of environmental and biological samples has shown that these HOCs originate from both natural and anthropogenic sources. In this work we seek a molecular-level understanding of the microbial genes and enzymatic processes responsible for the biosynthesis of HOCs by marine microorganisms, with the ultimate goal of enabling genetic source tracking through DNA sequence-based biosynthetic signatures, across all trophic levels of the marine food web. In addition to the biosynthesis of these compounds, we are also investigating microbial activities that degrade or transform these pollutants within marine animals, e.g. the fish gut microbiome. Omega-3 Polyunsaturated Fatty Acids Long-chain omega-3 polyunsaturated fatty acids (PUFAs) such as eicosapentaenoic acid (EPA; 20:5n-3) and docosahexaenoic acid (DHA; 22:6n-3) are well known for their beneficial roles in human health and development, and their ameliorative activities in diverse disease conditions, including cardiovascular, inflammatory, autoimmune, and neurodegenerative disorders. Select lineages of cosmopolitan marine prokaryotic and eukaryotic microorganisms synthesize these compounds via a unique fatty acid synthase/polyketide synthase mechanism that is distinct from the canonical desaturase/elongase-mediated pathway employed by the majority of eukaryotic single-cell microorganisms and metazoans. We are investigating the genetics and regulation of bacterial omega-3 PUFA synthesis in several lineages of marine bacteria via bacterial genetic methods to uncover genetic determinants involved in PUFA synthesis and explore the physiological role of these unique marine lipids. In this process we have engineered native and recombinant bacterial strains that synthesize high quantities of these molecules, providing a novel route for the biotechnological and/or nutraceutical exploitation of these marine bacterial resources. Bioinformatics We develop and apply bioinformatics tools and pipelines for the annotation and analysis of microbial genomes and metagenomes, collaborating with colleagues in many diverse areas of experimental biology and microbial ecology, as well as marine natural product chemistry and marine geochemistry. Our special interests include metagenomic sequence assembly of environmental and host-associated DNA samples, determining the role of specific environmental habitats in cross-species transmission of microbial genomes via horizontal gene transfer, and developing in silico tools for the discovery and novelty assignment of secondary metabolite biosynthesis genes within microbial genomes and metagenomes.  Select Publications Podell, S, Blanton JM, Neu A, Agarwal V, Biggs JS, Moore BS, Allen EE. 2019. Pangenomic comparison of globally distributed Poribacteria associated with sponge hosts and marine particles. ISME Journal 13:468-481. DOI: 10.1038/s41396-018-0292-9 Allemann, MN, Shulse CN, Allen EE. 2019. Linkage of marine bacterial polyunsaturated fatty acid and long-chain hydrocarbon biosynthesis. Front Microbiol 10:702. DOI: 10.3389/fmicb.2019.00702 Schorn, MA, Jordan PA, Podell S, Blanton JM, Agarwal V, Biggs JS, Allen EE, Moore BS. 2019. Comparative genomics of cyanobacterial symbionts reveals distinct, specialized metabolism in tropical Dysideidae sponges. mBio 10:e00821-19. DOI: 10.1128/mBio.00821-19 Neu, AT, Allen EE, Roy K. 2019. Diversity and composition of intertidal gastropod microbiomes across a major marine biogeographic boundary. Environ Microbiol Rep 11:434-447. DOI: 0.1111/1758-2229.12743 Kolody, B, McCrow J, Zeigler Allen L, Aylward F, Fontanez K, Moustafa A, Moniruzzaman M, Chavez F, Scholin C, Allen EE, Worden A, DeLong EF, Allen AE. 2019. Diel transcriptional response of a California Current plankton microbiome to light, low iron, and enduring viral infection. ISME Journal (in press) Allemann, MN, Allen EE. 2018. Characterization and application of marine microbial omega-3 polyunsaturated fatty acid synthesis. Methods in Enzymology 605:3-32. DOI: 10.1016/bs.mie.2018.02.018 Minich, JJ, Humphrey G, Benitez RAS, Sanders J, Swafford A, Allen EE, Knight R. 2018. High-throughput miniaturized 16S rRNA amplicon library preparation reduces costs while preserving microbiome integrity. mSystems 3:e00166-18. DOI: 10.1128/mSystems.00166-18 Agarwal, V, Blanton JM, Podell S, Taton A, Schorn MA, Busch J, Lin Z, Schmidt EW, Jensen PR, Biggs JS, Golden JW, Allen EE, Moore BS. 2017. Metagenomic discovery of polybrominated diphenyl ether biosynthesis by marine sponges. Nature Chemical Biology 13:537-543. DOI: 10.1038/nchembio.2330 Agarwal, V, Gamal AEA, Yamanaka K, Poth D, Kersten RD, Schorn M, Allen EE, Moore BS. 2014. Biosynthesis of polybrominated aromatic organic compounds by marine bacteria. Nature Chemical Biology 10:640-647. DOI: 10.1038/nchembio.1564 Podell, S, Emerson JB, Jones CM, Ugalde JA, Welch S, Heidelberg KB, Banfield JF, Allen EE. 2014. Seasonal fluctuations in ionic concentrations drive microbial succession in a hypersaline lake community. ISME Journal 8:979-990. DOI: 10.1038/ismej.2013.221 \n",
            "-------------------- Scraping faculty url 4/183 --------------------\n",
            "nicola allen \n",
            "-------------------- Scraping faculty url 5/183 --------------------\n",
            "jon armour \n",
            "-------------------- Scraping faculty url 6/183 --------------------\n",
            "kenta asahina  Research The goal of my laboratory is to understand the genetic and neural mechanisms underlying social interactions. We are particularly interested in the neural circuits controlling aggressive interactions of the vinegar fly Drosophila melanogaster , and how genes functioning in these circuits mediate intracellular and inter-neuronal communications. To this end, we combine multiple genetic, physiological and behavioral methods to manipulate functions of genes and neurons, and elucidate the behavioral consequence of such manipulations. Our overarching aim is to elucidate how genes and neurons shape animals choice of behavior during social interactions, and how behavioral experience influence the operation of neural circuits controlling social behaviors. \n",
            "-------------------- Scraping faculty url 7/183 --------------------\n",
            "janelle ayres \n",
            "-------------------- Scraping faculty url 8/183 --------------------\n",
            "eiman azim \n",
            "-------------------- Scraping faculty url 9/183 --------------------\n",
            "tim baker  Biography Tim Baker received his Ph.D. from UCLA and was a Jane Coffin Childs postdoctoral fellow at Cambridge University, a NIH Postdoctoral Fellow and the Charles A. King Trust Postdoctoral Fellow at Brandeis University. From 1983-2004, he was on the Faculty of Purdue University where he earned the Herbert Newby McCoy Award for Scientific Achievement and a NIH NIGMS MERIT award. He received a joint faculty appointment in Biological Sciences and Chemistry & Biochemistry at UCSD in 2004.  Research Viruses are one of natures smallest (nano), most efficient, machines, whose primary function is to infect and replicate in host cells. The predominant thrust of our research over the past 25 years has been to image and determine the three-dimensional (3D) structures of viruses to gain insights about the mechanisms they use to interact with their hosts so they can replicate and assemble a new set of mature, infectious particles. To accomplish this, we use cryo-electron microscopy (CryoEM) and 3D image-reconstruction methods to study viruses and various kinds of virus-complexes (e.g. virus-antibody or virus-receptor) and virus-like particles at sub-nanometer (< 10-) resolutions. At this level of resolution, we can discern and distinguish viral components (protein, nucleic acid, lipid, carbohydrate) and secondary structural details such as a-helices and b-sheets in proteins. CryoEM is a powerful tool because it permits the structures of biological samples to be preserved in a near native state. Samples are flash-frozen (vitrified) in liquid ethane in a sub-millisecond time frame, which prevents ice crystal formation that would damage the specimen. This technique uses no chemical fixatives or stains and thereby helps preserve the hydrated structure of the virus (or any macromolecular complex). Specimens are then loaded into one of two, computer-controlled, transmission electron microscopes housed in Bonner Hall, both of which are able to maintain specimens at liquid nitrogen or lower temperature. These microscopes are used to collect upwards of tens of thousands or more virus images that are needed to obtain high-resolution 3D reconstructions. Such large numbers are required because each individual virus image is extremely noisy owing to radiation-induced damage to the sample caused by the high voltage electron beam and because the contrast in unstained samples is extremely low. In house, computer reconstruction techniques are continuously being developed and tested to more reliably and efficiently extract usable information from the noisy image data. We have developed a processing pipeline, called Auto3DEM, which automates nearly all of the tedious steps in the 3D reconstruction process. In favorable cases, we are able to obtain reconstructions at sub-nanometer resolution in a day or less after the images are recorded. In addition, we are developing tools designed to provide real time, 3D reconstruction feedback (i.e. in a few minutes or less) to the microscopist to further improve success of data collection. Viral pathogens from many different families are currently under investigation. These infect a wide range of hosts and include: human parvoviruses (bocavirus and several serotypes of adeno-associated viruses both as native virions and complexed with neutralizing antibodies and receptor molecules); human papilloma viruses that cause cancer (HPV-16) or genital warts (HPV -6 and HPV-11) or are listed as high risk (HPV-52); four distinct totiviruses (the invertebrate IMNV, the fungal HvV190S, and the protozoan TVV1 and GLV); the alphavirus group of small, enveloped, ssRNA viruses (e.g. Sindbis virus) that are human and insect pathogens, some of which are listed as possible agents of bio-terrorism; two large, dsDNA viruses (the insect CIV and frog FV3); three fungal partitiviruses (PsV-S, PsV-F, FpV-01); four bacteriophages (P22, Sf6, CUS-3, and 29); and an insect tetravirus (NV).  Select Publications Brodin, J. D., X. I. Ambroggio, C. Tang, K. N. Parent, T. S. Baker and F. A. Tezcan (2012) Metal-directed, chemically-tunable assembly of a monomeric protein into one-, two- and three-dimensional crystalline protein arrays. Nat. Chem., 4:375-382. Highlighted in Chem. & Eng. News (2012) 90(11):9. DiMattia, M. A., H-J. Nam, K. V. Vliet, M. Mitchell, A. Bennett, B. Gurda, R. McKenna, N. H. Olson, R. S. Sinkovits, M. Potter, Y. Sakai, B. J. Byrne, G. Aslanidi, S. Zolotukhin, N. Muzyczka, T. S. Baker, and M. Agbandje-McKenna (2012) Structural insight into the unique properties of adeno-associated virus serotype 9. J. Virol. 86:6947-6958. Gurda, B. L., C. Raupp, M. Naumer, R. Popa-Wagner, N. H. Olson, R. McKenna, T. S. Baker, J. A. Kleinschmidt, and M. Agbandje-McKenna (2012) Mapping a neutralizing epitope onto the capsid of Adeno-associated virus serotype 8. J. Virol. 86:7739-7751.Parent, K. N., C. T. Deedas, E. H. Egelman, S. R. Casjens, T. S. Baker, and C. M. Teschke (2012) The use of stepwise molecular display utilizing icosahedral and helical complexes of phage coat and decoration proteins in the development of robust nanoscale display vehicles. Biomaterials, 33:5628-5637. Parent, K. N., E. B. Gilcrease, S. R. Casjens, and T. S. Baker (2012) Structural evolution of the P22-like phages: Comparison of Sf6 and P22 procapsid and virion architectures. Virology, 427:177-188. Ku, T.-H., M.-P. Chien, M. P. Thompson, R. S. Sinkovits, N. H. Olson, T. S. Baker, and N. C. Gianneschi (2011) Controlling and switching the morphology of micellar nanoparticles with enzymes. J. Amer. Chem. Soc., 133:8392-8395. Shen, P. S., D. Enderlein, C. D. S. Nelson, W. S. Carter, M. Kawano, L. Xing, R. D. Swenson, N. H. Olson, T. S. Baker, R. H. Cheng, W. J. Atwood, R. Johne, and D. M. Belnap (2011) The structure of avian polyomavirus reveals variably sized capsids, non-conserved inter-capsomere interactions, and a possible location of the minor capsid protein VP4. Virology, 411:142-152. Tang, J., J. Jose, P. Chipman, W. Zhang, R. J. Kuhn, and T. S. Baker (2011) Molecular links between the envelope glycoprotein and nucleocapsid core in Sindbis virus. J. Mol. Biol. 414:442-459. Tang, J., G. C. Lander, A. Olia, R. Li, S. Casjens, P. Prevelige Jr., G. Cingolani, T. S. Baker, and J. E. Johnson (2011) Peering down the barrel of a bacteriophage portal: the genome packaging and release valve in P22. Structure, 19:496-502. Yan, X. D., K. N. Parent, R. P. Goodman, J. Tang, J. Shou, M. L. Nibert, R. Duncan, and T. S. Baker (2011) Virion structure of Baboon reovirus, a fusogenic orthoreovirus that lacks an adhesion fiber. J. Virol., 85:7483-7495. \n",
            "-------------------- Scraping faculty url 10/183 --------------------\n",
            "matthew banghart  Biography Matt Banghart obtained his Ph.D. in Chemistry from UC Berkeley and pursued postdoctoral training in the Department of Neurobiology at Harvard Medical School with Bernardo Sabatini. Matt was a Helen Hay Whitney Postdoctoral Fellow and the recipient of a K99/R00 Pathway to Independence Award from NIDA. He is currently establishing his independent research laboratory at UCSD.  Research Research in the Banghart lab involves two intertwined themes: neuromodulatory mechanisms and molecular tool development. Several questions of interest include: How do neuromodulators transform neural circuit function? What principles determine the spatial and temporal domains of neuromodulatory signaling? When and where are neuromodulators released in the brain? How are neuromodulatory pathways compromised in disease and how can they be restored? We address these questions in the mammalian basal ganglia, a collection of neural circuits that rely heavily on neuromodulators to mediate decision-making, goal-oriented movements and behavioral reinforcement. Our experimental approach utilizes contemporary genetically-targeted neuroanatomical methods, whole-cell electrophysiology, two-photon imaging and rodent behavioral analysis, often in conjunction with optogenetic and chemical-genetic perturbations. To facilitate these efforts, we develop novel molecular tools for observing and manipulating neuromodulatory activity with an emphasis on photoactivatable (or caged) neurotransmitters, optical sensors and genetically-targeted pharmacological probes. Fluorescently labeled neurons demarcate neuromodulatory hot-spots known as patches (or striosomes) in the dorsal striatum.  Select Publications Banghart MR, He XJ, Sabatini BL. A Caged Enkephalin Optimized for Simultaneously Probing Mu and Delta Opioid Receptors. ACS Chem Neurosci. 2017 Dec 27. PMID: 29266926 Otopalik AG, Sutton AC, Banghart M, Marder E. When complex neuronal structures may not matter. Elife. 2017 Feb 6;6. pii: e23508. PMID: 28165322 Banghart MR, Neufeld SQ, Mulder N, Sabatini BL. Enkephalin disinhibits mu-receptor rich striatal patches via delta opioid receptors. Neuron (2015), 88(6), 1227-39. PMID 22671460 Olson J, Banghart MR, Sabatini BL, Ellis-Davies GCR. Spectral evolution of a photochemical protecting group for orthogonal two-color uncaging with visible light. Journal of the American Chemical Society (2013), 135(42), 15948-54. PMID 24117060 Banghart MR, Williams JT, Shah RC, Lavis LD, Sabatini BL. Caged naloxone reveals opioid signaling deactivation kinetics. Molecular Pharmacology (2013), 84(5), 687-95. PMID 23960100 Banghart MR, Sabatini BL. Photoactivatable neuropeptides for spatiotemporally precise delivery of opioids in neural tissue. Neuron (2012), 73, 249-59. PMID 22284180 Tochitsky I, Banghart MR, Mourot A, Yao JZ, Gaub B, Kramer RH, Trauner D. Optical control of genetically engineered neuronal nicotinic acetylcholine receptors. Nature Chemistry (2012), 4(2), 105-11. PMID 22270644 Dvir T, Banghart MR, Timko BP, Langer R, Kohane DS. Photo-targeted nanoparticles. Nano Letters (2010), 10 (1), 250-254. PMID 19904979 Banghart MR, Mourot A, Fortin DL, Yao JZ, Kramer RH, Trauner D. Photochromic blockers of voltage-gated potassium channels. Angewandte Chemie International Edition (2009), 48 (48), 9097-9101. PMID 19882609 Fortin DL, Banghart MR, Dunn TW, Borges K, Wagenaar DA, Gaudry Q, Karakossian MH, Otis TS, Kristan WB, Trauner D, Kramer RH. Photochemical control of endogenous ion channels and cellular excitability. Nature Methods (2008), 5(4), 331-338. PMID 18311146 Banghart MR, Volgraf M, Trauner D. Engineering light gated ion channels. Biochemistry (2006), 45(51), 15129-15141. PMID 17176035 Banghart M, Borges K, Isacoff E, Trauner D, Kramer RH. Light-activated ion channels for remote control of neuronal firing. Nature Neuroscience (2004), 7(12), 1381-1386. PMID 15558062 Miller AK, Banghart MR, Beaudry CM, Suh JM, Trauner D. Development of novel Lewis acid catalyzed cycloisomerizations: synthesis of bicyclo[3.1.0]hexenes and cyclopentenones. Tetrahedron (2003), 59(45), 8919-8930. \n",
            "-------------------- Scraping faculty url 11/183 --------------------\n",
            "andrew barton  Biography Andrew Barton received his Ph.D in Climate Physics and Chemistry from Massachusetts Institute of Technology in 2011, and was an NSF International Research Postdoctoral Fellow hosted jointly between Duke University and the University of Liverpool in the United Kingdom. Dr. Barton joined the faculty of Biological Sciences and the Scripps Institution of Oceanography in 2016.  Research Marine phytoplankton are microscopic photosynthetic organisms living in the ocean surface. They form the base of the marine food chain, and can thus impact larger marine organisms, from zooplankton to commercially important fish and even whales. Phytoplankton also play instrumental roles in the global cycles of carbon, oxygen, and other elements. Yet despite their tremendous ecological, societal, and global importance, we are only beginning to explore the great biodiversity of phytoplankton, and understand their complex ecology and interactions with the environment. Our research seeks to map the distribution of phytoplankton species in the ocean, and illuminate the fundamental biological and ecological processes that underpin these patterns. We investigate how changes in Earths climate, including natural variability and long-term changes driven by human activities, have the potential to alter phytoplankton species distributions and community composition. To illustrate and study these complex natural processes, we develop cutting-edge computer models that simulate the marine environment and the many types of plankton living in the ocean. We integrate these models with real observations from the sea, using large compilations of environmental, ecological, and biological data to glean novel inferences about marine plankton life. Our research is building understanding of a critical, if individually tiny, component of life on Earth.  Select Publications Barton, A.D., Irwin, A.J., Finkel, Z.V., and C. Stock (2016). Anthropogenic climate change drives shift and shuffle in North Atlantic phytoplankton communities. Proceedings of the National Academy of Sciences USA 113: 2964-2969. Dave, A.C., Barton, A.D., Lozier, M.S., and G.A. McKinley (2015). What drives seasonal change in oligotrophic area in the subtropical North Atlantic? Journal of Geophysical Research Oceans 120, doi:10.1002/2015JC01078. Barton, A.D., Ward, B.A., R.G. Williams, and M.J. Follows (2014). The impact of small-scale fluid turbulence on phytoplankton nutrient uptake rates and community structure. Limnology and Oceanography Fluids and Environments 4, 34-49. Barton, A.D., Williams, R.G., and M.S. Lozier (2014). Physical controls of variability in North Atlantic phytoplankton communities. Limnology and Oceanography 60, 181-197. Barton, A.D., Pershing, A.J., Litchman, E., Record, N.R., Edwards, K.F., Finkel, Z.V., Kirboe, T., and B.A. Ward (2013). The biogeography of marine plankton traits. Ecology Letters 16, 522-534. Barton, A.D., Finkel, Z.V., Ward, B.A., Johns, D.G., and M.J. Follows (2013). On the roles of cell size and trophic strategy in North Atlantic diatom and dinoflagellate communities. Limnology and Oceanography 58 (1), 254-266. Mariani, P., Andersen, K.H., Visser, A.W., Barton, A.D., and T. Kirboe (2013). Control of plankton seasonal succession by adaptive grazing. Limnology and Oceanography 58 (1), 173-184. Ward, B.A., Dutkiewicz, S., Barton, A.D., and M.J. Follows (2011). Biophysical aspects of resource acquisition and competition in algal mixotrophs. The American Naturalist 178 (1), 98-112. Barton, A.D., Dutkiewicz, S., Flierl, G., Bragg, J., and M.J. Follows (2010). Response to Comment on Patterns of diversity in marine phytoplankton. Science 329, 512-d. Barton, A.D., Dutkiewicz, S., Flierl, G., Bragg, J., and M.J. Follows (2010). Patterns of diversity in marine phytoplankton. Science 327, 1509-1511. Barton, A.D., and K.S. Casey (2005). Climatological context for large-scale coral bleaching observed since 1979. Coral Reefs 24 (4), 536-554. Barton, A.D., Greene, C.H., Monger, B.C., and A.J. Pershing (2003). The Continuous Plankton Recorder Survey and North Atlantic Oscillation: Interannual- to multidecadal-scale patterns of phytoplankton variability in Northwest Shelf, Northeast Shelf, and Central North Atlantic Ecosystems. Progress in Oceanography 58, 337-358. \n",
            "-------------------- Scraping faculty url 12/183 --------------------\n",
            "eric bennett  Biography Dr. Bennett received his PhD from Stanford University. He then carried out postdoctoral work at the Harvard Medical School.  Research My lab is interested in understanding the fundamental principles of how proteins are destroyed within cells. The cellular environment must constantly adapt to changing conditions and one mechanistic strategy commonly employed is the targeted modification of proteins by ubiquitin chains and subsequent degradation by the proteasome. The ubiquitin-proteasome system (UPS) can be viewed as the cellular waste management system. The UPS is utilized as the cellular quality control system as it is tasked with the degradation of potentially toxic misfolded and misassembled proteins. The inability of the UPS to properly destroy these proteins has been linked to various human pathologies including amyotrophic lateral sclerosis (ALS, commonly referred to as Lou Gehrig's disease), Parkinson's disease, and many human cancers. Thus, understanding the mechanisms of how the UPS recognizes proteins to be destroyed and how the pathway becomes dysfunctional will be instrumental in understanding the etiology of these diseases. The lab uses integrated approaches to understand UPS activity coupling biochemistry, cell biology, and systems-level approaches. Deubiquitylating enzyme regulation The ubiquitylation of substrates occurs via a hierarchy of enzyme activity where ubiquitin activating enzymes (E1s) activate free ubiquitin and transfers the activated ubiquitin to ubiquitin conjugating enzymes, (E2s), who partner with ubiquitin ligases (E3s) to finally conjugate ubiquitin to substrates. Deubiquitylating enzymes (Dubs) remove ubiquitin chains from substrates thereby antagonizing ubiquitin ligase activity. While the basic enzymatic function of Dubs is understood, how their activity is regulated, what cellular pathways are regulated by Dub activity, and what cellular substrates are the targets for Dub regulationare not fully understood. We previously characterized 75 of the 100 human Dubs based on their cellular interaction partners using proteomic technologies. During this study we established an integrative proteomic platform, termed CompPASS, to rapidly analyze interaction proteomics data. We found that Dubs not only frequently associate with ubiquitin ligases but are themselves often modified by ubiquitin. This suggests that Dub activity may be regulated by E3 ligases and vice versa. The lab is interested in understanding the mechanisms of Dub activity regulation via association with ubiquitin ligases using both biochemical and cellular techniques. Cullin-RING ligase regulation One family of ubiquitin ligases are the multi-subunit ligases consisting of Cullin-RING ligases (CRLs) and the anaphase promoting complex (APC). Cullin-RING ligases are modular assemblies that can target many cellular proteins for regulated degradation by dynamically remodeling its subunit architecture. CRLs are known to degrade many critical oncoproteins and tumor suppressors, and as such there has been recent interest in the development of therapeutics targeting this pathway as an anti-cancer strategy. CRLs are also regulated through a switch like mechanism in which the ubiquitin like protein Nedd8 modifies and subsequently activates the cullin subunit. Thus, by simply adding and removing the Nedd8 modification, CRL activity can be switched on and off. We are utilizing quantitative proteomics to study the cellular mechanisms used to control CRL activity. Additionally, this allows us to probe the dynamic nature of CRL assemblies and to understand how their architecture is remodeled to adapt to changes in the cellular environment. Global quantitative proteomic analysis of UPS function The cellular proteome is continually monitored by the protein folding and degradation machinery to prevent the accumulation of potentially toxic misfolded or misassembled proteins. Proteins that contain errors in their primary sequence that arise during the translation process need to be recognized and selectively degraded by the UPS. When this quality control function breaks down it can have dire consequences on cellular and organismal fitness. Thus, the ability to globally monitor the activity of the UPS would allow for the identification of conditions that lead to UPS dysfunction. We have developed technologies that allow for the interrogation of the UPS activity on a systematic level. From these studies it is clear that mistranslated proteins are constantly ubiquitylated and cleared from cells. We are interested in trying to answer several questions related to quality control mechanisms. For instance, what are the ubiquitin ligases responsible for ubiquitylation mistranslated substrates? How do changes in free amino acid levels within the cell effect protein degradation and UPS function? Can we monitor alterations in UPS function in human pathologies associated with a decrease in quality control function? How does changing the redox potential within cells effect the quality control machinery? By understanding how proteins are recognized as quality control substrates and how the cell responds to challenges in protein homeostasis we hope to develop therapeutics that modulate quality control function.  Select Publications Harper JW, Bennett EJ. Proteome complexity and the forces that drive proteome imbalance. Nature. 2016 Sep 14;537(7620):328-38. PMID: 27629639 Arribas-Layton M, Dennis J, Bennett EJ, Damgaard CK, Lykke-Andersen J. The C-terminal RGG domain of human Lsm4 promotes processing body formation stimulated by arginine dimethylation. Mol Cell Biol. 2016 May 31 PMID: 27247266 Gendron JM, Webb K, Yang B, Rising L, Zuzow N, Bennett EJ. Using the ubiquitin-modified proteome to monitor distinct and spatially restricted protein homeostasis dysfunction. Mol Cell Proteomics. 2016 May 16. PMID: 27185884 Fu R, Olsen MT, Webb K, Bennett EJ, Lykke-Andersen J. Recruitment of the 4EHP-GYF2 cap-binding complex to tetraproline motifs of tristetraprolin promotes repression and degradation of mRNAs with AU-rich elements. RNA. 2016 Mar;22(3):373-82. PMID: 26763119 Higgins R, Gendron JM, Rising L, Mak R, Webb K, Kaiser SE, Zuzow N, Riviere P, Yang B, Fenech E, Tang X, Lindsay S, Christianson JC, Hampton RY, Wasserman SA, Bennett EJ. The unfolded protein response triggers site-specific regulatory ubiquitylation of 40S ribosomal proteins. Mol Cell. 2015 Jul 2;59(1):35-49. PMID: 26051182 Erickson SL, Corpuz EO, Maloy JP, Fillman C, Webb K, Bennett EJ, Lykke-Andersen J. A competition between decapping complex formation and ubiquitin-mediated proteasomal degradation controls human Dcp2 decapping activity. Mol Cell Biol. 2015 Apr 13.PMID: 25870104 Facette MR, Park Y, Sutimantanapi D, Luo A, Cartwright HN, Yang B, Bennett EJ, Sylvester AW, Smith LG. The SCAR/WAVE complex polarizes PAN receptors and promotes division asymmetry in maize. Nat Plants. 2015 Jan 26;1:14024. PMID: 27246760 Lyumkis D, Oliveira dos Passos D, Tahara EB, Webb K, Bennett EJ, Vinterbo S, Potter CS, Carragher B, Joazeiro CA. Structural basis for translational surveillance by the large ribosomal subunit-associated protein quality control complex. Proc Natl Acad Sci U S A. 2014 Nov 11;111(45):15981-6. PMID: 25349383 Ikeuchi Y, Dadakhujaev S, Chandhoke AS, Huynh MA, Oldenborg A, Ikeuchi M, Deng L, Bennett EJ, Harper JW, Bonni A, Bonni S. TIF1gamma Regulates Epithelial Mesenchymal Transition by Operating as a SUMO E3 Ligase for theTranscriptional Regulator SnoN1. J Biol Chem. 2014 Sep 5;289(36):25067-78. PMID: 25059663 Lykke-Andersen J, Bennett EJ. Protecting the proteome: Eukaryotic cotranslational quality control pathways.J Cell Biol. 2014 Feb 17;204(4):467-76. PMID: 24535822 Mejia LA, Litterman N, Ikeuchi Y, de la Torre-Ubieta L, Bennett EJ, Zhang C, Harper JW, Bonni A. A novel Hap1-Tsc1 interaction regulates neuronal mTORC1 signaling and morphogenesis in the brain. J Neurosci. 2013 Nov 13;33(46):18015-21. PMID: 24227713 Ziemba A, Hill S, Sandoval D, Webb K, Bennett EJ, Kleiger G. Multimodal mechanism of action for the Cdc34 acidic loop: a case study for why ubiquitin-conjugating enzymes have loops and tails. J Biol Chem. 2013 Nov 29;288(48):34882-96. PMID: 24129577 Tan MK, Lim HJ, Bennett EJ, Shi Y, Harper JW. Parallel SCF Adaptor Capture Proteomics Reveals a Role for SCFFBXL17 in NRF2 Activation via BACH1 Repressor Turnover. Molecular Cell 2013 Sep 10. PMID: 24035498 Zhang C, Mejia LA, Huang J, Valnegri P, Bennett EJ, Anckar J, Jahani-Asl A, Gallardo G, Ikeuchi Y, Yamada T, Rudnicki M, Harper JW, Bonni A. The X-linked intellectual disability protein PHF6 associates with the PAF1 complex and regulates neuronal migration in the mammalian brain. Neuron 2013 Jun 19;78(6):986-93. PMID: 23791194 Carrano AC, Bennett EJ. Using the ubiquitin modified proteome to monitor protein homeostasis function. Mol Cell Proteomics 2013 May 23. PMID: 23704779 Armour SM, Bennett EJ, Braun CR, Zhang XY, McMahon SB, Gygi SP, Harper JW, Sinclair DA. A high-confidence interaction map identifies SIRT1 as a mediator of acetylation of USP22 and the SAGA coactivator complex. Mol Cell Biol. 2013 Apr;33(8):1487-502. PMID: 23382074 Monda JK, Scott DC, Miller DJ, Lydeard J, King D, Harper JW, Bennett EJ, Schulman BA. Structural Conservation of Distinctive N-terminal Acetylation-Dependent Interactions across a Family of Mammalian NEDD8 Ligation Enzymes. Structure 2013 Jan 8;21(1):42-53. PMID: 23201271 Christianson JC*, Olzmann JA*, Shaler TA, Sowa ME, Bennett EJ, Richter CM, Tyler RE, Greenblatt EJ, Harper JW, Kopito RR. Defining Human ERAD Networks through an Integrative Mapping Strategy. Nature Cell Biology 2011 Nov 27;14(1):93-105. PMID: 22119785 *Authors contributed equally Scott DC, Monda JK, Bennett EJ, Harper JW, Schulman BA. N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex. Science 2011 Nov 4;334(6056): 674-8. PMID: 21940857 Kim W*, Bennett EJ*, Huttlin EL, Guo A, Li J, Possemato A, Sowa ME, Rad R, Rush J, Comb MJ, Harper JW, and Gygi SP. Systematic and quantitative assessment of the ubiquitin modified proteome Molecular Cell 2011 Oct 1;44(2):325-40. PMID: 21906983 *Authors contributed equally Bennett EJ, Rush J, Gygi SP, Harper JW. Dynamics of cullin-RING ubiquitin ligase assembly revealed by systematic quantitative proteomics. Cell 2010 Dec 10;143(6):951-965. Bennett EJ, Harper JW. Ubiquitin gets CARDed. Cell. 2010 Apr 16;141(2):220-222. Bennett EJ, Sowa ME, Harper JW. Cellular regulation by deubiquitinating enzymes. Nat. Rev. Mol Cell Bio. 2010 Feb Poster insert. Sowa ME*, Bennett EJ*, Gygi SP, Harper JW. Defining the Human Deubiquitinating Enzyme Interaction Landscape. Cell. 2009 Jul 23;138(2):389-403. *Authors contributed equally Bennett EJ, Harper JW. DNA damage: ubiquitin marks the spot. Nat Struct Mol Biol. 2008 Jan;15(1):20-22. Bennett EJ, Shaler TA, Woodman B, Ryu KY, Zaitseva TS, Becker CH, Bates GP, Schulman H, Kopito RR. Global changes to the ubiquitin system in Huntingtons disease. Nature 2007 Aug 9;448(7154):704-708 Bennett EJ, Bence NF, Jayakumar R, Kopito RR. Global impairment of the ubiquitinproteasome system by nuclear or cytoplasmic protein aggregates precedes inclusion body formation. Molecular Cell 2005 Feb 4;17(3):351-365. Bence NF, Bennett EJ, Kopito RR. Application and Analysis of the GFPu Family of Ubiquitin-Proteasome System Reporters. Methods in Enzymology 2005;399:481-490. Bennett EJ, Bjerregaard J, Knapp JE, Chavous DA, Friedman AM, Royer WE, and OConnor CM. Catalytic Implications from the Drosophila Protein L-Isoaspartyl Methyltransferase Structure and Site-Directed Mutagenesis. Biochemistry 2003 Nov 11;42(44):12844-12853. \n",
            "-------------------- Scraping faculty url 13/183 --------------------\n",
            "darwin berg  Research We are interested in the role of nicotinic cholinergic signaling both in guiding the formation of complex neural networks and in regulating their function. We find that endogenous nicotinic activity promotes early glutamatergic (excitatory) innervation of neurons during development. Further, it helps drive the conversion of GABAergic transmission from excitation to inhibition at this time. Direct exposure to nicotine during early development produces long-lasting changes in synaptic wiring that persist into the adult and remain long after nicotine cessation. These kinds of changes suggest serious behavioral consequences and increased vulnerability to subsequent nicotine challenges. We are also pursuing other mechanisms guiding network formation, including the role of micro-RNAs in orchestrating major transitions in brain development. Our laboratory takes a multidisciplinary approach. We want to identify the molecular players mediating nicotinic signaling and understand how they work. We want to understand how these components interact in complex ways to regulate circuit construction and system output. To do this, we combine molecular, physiological, imaging, and biochemical techniques, applying them both in acute slices and in vivo. Recent collaborations have also incorporated computational and ultrastructural analyses. We use viral constructs and mutant mice to manipulate signaling pathways in vivo both during development and in the adult to gain insight into mechanism. It is an exciting time to be investigating nicotinic signaling in the central nervous system. At the behavioral level nicotinic cholinergic activity has been implicated in a broad array of phenomena including cognition, memory formation, and arousal. It is also associated with numerous pathologies including Alzheimer's disease, schizophrenia, and addiction. These physiological consequences of nicotinic activity offer intriguing windows into higher brain function while at the same time suggesting biomedical applications. How the cellular and molecular features of nicotinic signaling collaborate to achieve these effects in the brain remains a mystery and presents ongoing challenges. Molecular tools are now in place to make major advances in these areas.  Select Publications Halff AW, Gomez-Varela D, John D, Berg DK (2014) A novel mechanism for nicotinic potentiation of glutamatergic synapses. J Neurosci 34:2051-2064. (PMID: 24501347) Gomez-Varela D, Berg DK (2013) Lateral mobility of presynaptic 7-containing nicotinic receptors and its relevance for glutamate release. J Neurosci 33:17062-17071. (PMID: 20592206 Wang, X, Lippi G, Carlson D, Berg DK (2013) Activation of7-containing nicotinic receptors on astrocytes triggers AMPA receptor recruitment to glutamatergic synapses. J Neurochem 127:632-643. (PMID: 24032433) Lozada AF, Wang X, Gounko NV, Massey KA, Duan J, Liu Z, Berg DK (2012a) Glutamatergic synapse formation is promoted by 7-containing nicotinic acetylcholine receptors. J Neurosci 32:7651-7661. (PMID: 22649244 Lozada AF, Wang X, Gounko NV, Massey KA, Duan J, Liu Z, Berg DK (2012b) Induction of dendritic spines by 2-containing nicotinic receptors. J Neurosci 32:8391-8400. (PMID: 22699919) Gomez-Varela D, Schmidt M, Schoellerman J, Peters E, Berg DK (2012) PMCA2 via PSD-95 controls signaling by 7-containing nicotinic acetylcholine receptors on aspiny neurons. J Neurosci 32:6894-6905. (PMID: 22593058) Fernandes CC, Berg DK, Gomez-Varela D (2010)Lateral mobility of nicotinic acetylcholine receptors on neurons is determined by receptor composition, local domain, and cell type.J Neurosci 30:8841-8851 (and Journal cover). (PMID: 20592206) Campbell NR, Fernandes CC, Halff AW, Berg DK (2010)Endogenous signaling through7-containing nicotinic receptors promotes maturation and integration of adult-born neurons in the hippocampu. J Neurosci 30:8734-8744. (PMID: 20592195) Neff RA, Conroy WG, Schoellerman JD, Berg DK (2009) Synchronous and asynchronous transmitter release at nicotinic synapses are differentially regulated by postsynaptic PSD-95 proteins. J Neurosci 29:15770-15779. (PMID: 200116093 Liu Z, Neff RA, Berg DK (2006) Sequential interplay of nicotinic and GABAergic signaling guides neuronal development. Science 314:1610-1613. PMID: 1715833) Coggan JS, Bartol TM, Esquenazi E, Stiles JR, Lamont S, Martone ME, Berg DK, Ellisman MH, Sejnowski TJ (2005) Ectopic neurotransmitter release at a neuronal synapse. Science 309:446-451. PMID: 1602073) Conroy WG, Liu Z, Nai Q, Coggan JS, Berg DK (2003) PDZ-containing proteins provide a functional postsynaptic scaffold for nicotinic receptors in neurons. Neuron 38:759-771. PMID: 1279796) Chang K, Berg DK (2001) Voltage-gated channels block nicotinic regulation of CREB phosphorylation and gene expression in neurons. Neuron 32:855-865. PMID: 1173803) Liu, Q.-s., Kawai, H., and Berg, D.K. (2001). -Amyloid peptide blocks the response of a7-containing nicotinic receptors on hippocampal neurons. Proc. Natl. Acad. Sci. (USA) 98: 4734-4739. (PMID: 11274373 Zhang Z-W, Coggan JS, Berg DK (1996) Synaptic currents generated by neuronal acetylcholine receptors sensitive to-bungarotoxin. Neuron 17:1231-1240. PMID: 898216) Zhang Z-W, Vijayaraghavan S, Berg DK (1994) Neuronal acetylcholine receptors that bind-bungarotoxin with high affinity function as ligand-gated ion channels. Neuron 12:167-177. PMID: 750733) Margiotta, J.F. and Berg, D.K. (1982). Functional synapses are established between ciliary ganglion neurones in dissociated cell culture. Nature 296: 152-154. (PMID: 7063016) Nishi, R. and Berg, D.K. (1979). Survival and development of ciliary ganglion neurones grown alone in cell culture. Nature 277: 232-234. (PMID: 551252) Berg, D.K. and Hall, Z.W. (1974). Fate of -bungarotoxin bound to acetylcholine receptors in normal and denervated muscle. Science 184: 473-475. (PMID: 4819679 \n",
            "-------------------- Scraping faculty url 14/183 --------------------\n",
            "ethan bier  Biography Ethan Bier received his Ph.D. from Harvard University. He has received the Sloan Foundation Award, the McKnight Award for Neuroscience, the Basil O'Connor Award, and the American Cancer Society Junior Faculty Award.  Research There are two primary areas of research in my lab: 1) developmental patterning, and 2) using Drosophila as model to understand human disease processes. Developmental Studies With respect to developmental patterning, we have had a long standing interest in understanding how graded BMP signaling establishes cell fates along the dorsal-ventral (DV) axis of Drosophila embryos and how wing vein form along borders between discrete domains in the wing imaginal disc. Our studies on BMPs in DV patterning began with our cloning of the short gastrulation ( sog ) gene [1], which encodes a secreted BMP antagonist homologous to vertebrate Chordin [2]. In collaboration with David Kimelmans group (Univ. Washington), we showed that sog activity was conserved in Xenopus embryos where, like chordin , it could induce a secondary neural axis. Our studies also revealed that Sog acts to prevent BMP signaling from spreading ventrally into the Drosophila neuroectoderm via an autoactivation loop, a process that is also conserved in vertebrate embryos [3], thus helping to reverse the prior paradigm for neural induction as a positive-acting process as originally formulated by Hans Spemann in the early 20th century. We also have studied formation of the Sog protein gradient in the dorsal epidermal region of the embryo which we visualized [4] and analyzed quantitatively in collaboration with Arthur Landers group (UCI) [5]. This BMP activity gradient arises as a consequence of an adjacent source/sink configuration wherein ventrally produced Sog diffuses dorsally, where it is degraded by the metalloprotease Tolloid and removed by endocytosis [4]. Most recently, in collaboration with Shannon Fishers group (U Penn) and Giorgos Pyrowolakis (Univ. Freiburg) we have been examining the role that BMPs play in patterning the dorsal and lateral regions of the CNS in both flies and zebrafish. In flies, BMPs function as they do during neural induction by repressing expression of the neural identity genes msh and ind (whichdefine dorsal and lateral CNS cell fates, respectively). A twist , however, is that BMPs act in a dose-dependent fashion to repress ind more efficiently than msh , thereby providing a patterning cue [6]. These experiments should shed light on the evolutionary origins of the condensed CNS, which we believe arose once in the ancestors of bilateria [7, 8]. Our studies on wing vein patterning are focused on how primary positional information provided by BMP and Hedgehog morphogen gradients is refined into sharp borders abutting broad gene expression domains to define the positions along which wing vein primordia will develop [9, 10]. Wing veins are rigid hollow fluid conducting channels that function as structural supports for flight and as conduits bringing nutrients to sensory organs in the wing. Although formation of each vein is initiated separately, these linear structures are all induced by the same general mechanism (termed for-export-only signaling) in which cells in one domain produce a diffusible factor to which they are unable to respond [10]. The vein inducing signal diffuses into the adjacent cell territory where cells nearest the source of the signal can respond by activating expression of vein specific identity genes such as knirps (for the L2 vein) [11, 12] or abrupt (for the L5 vein) [13]. Most recently we have become interested in the evolution of the gene regulatory networks underlying wing development in dipteral species with different vein patterns than in D. melanogaster . Analyzing Mechanisms of Human Disease in Flies With regard to using Drosophila as tool to study human disease [14], we have used this system to identify the Rho-GEF pebble as a candidate gene involved in Angelman syndrome [15], to examine an interaction between g-secretase and peroxiredoxins that may be relevant to Alzheimers disease [16], to identify a cooperative interaction between the cell adhesion molecule DSCAM and a collagen subunit (COL6A2) that when over-expressed leads to congenital heart defects in flies and mice and may contribute to heart defects in Down syndrome patients (in collaboration with Geoff Rosenfeld, UCSD, Medicine) [17], and, in collaboration with Michael Karin (UCSD, Pharmacology) to examine mechanisms of metabolic homeostasis relevant to aging [18]. Our current focus is on identifying genetic pathways that are disrupted by pathogenic bacterial and viral factors. For example, in collaboration with Michael Karin and Victor Nizet (UCSD, Pediatrics) we have examined the cell biological effects of anthrax toxins [19-21]. These studies identified the small GTPase Rab11 and its exocyst binding partner Sec15 as targets of the barrier disruptive anthrax toxins Edema Factor (EF) and Lethal Factor (LF) [19]. EF functions by down-regulating expression and activity of Rab11 on late recycling endosomal membranes, and thereby indirectly leads to down-regulation of Sec15, which tethers the endosome to docking sites in the plasma membrane. LF does not affect Rab11, but does inhibit Sec15 expression, and functions cooperatively with EF to disrupt exocyst mediated trafficking and endothelial barrier integrity. We are also conducting a similar analysis on the effect of cholera toxin (Ctx, an ADP ribosylase that activates Gsa proteins thereby constitutively activating endogenous adenylate cyclase to produce elevated levels of cAMP) in intestinal epithelial cells [22]. These studies have revealed that in addition to its classic well characterized role in inducing transcellular Cl- ion secretion via the CFTR channel, Ctx also weakens cell junctions, which may facilitate the accompanying efflux of Na+ counter ions and water via the paracellular route, thus resulting in the profuse salty diarrhea characteristic of cholera. Importantly, the junction disrupting effects of both EF and Ctx can be reversed by over-expressing Rab11, suggesting a new avenue for therapeutic intervention in these diseases. In collaboration with David Broide (UCSD, Medicine) we have also begun to examine whether endocytic recycling plays a role maintaining barrier integrity in airway epithelial cells, and whether defects in this trafficking contribute to asthma pathogenesis. Insights derived from these synergistic studies will have relevance to a broad range of diseases that increase epithelial or endothelial permeability and may lead to therapeutic strategies that selectively counteract such effects by restoring endocytic trafficking and improving barrier function. Quantitative Imaging and Genome Analysis In collaboration with the McGinnis group (UCSD, CDB) and Terry Hwa's lab (UCSD, Physics) we have also developed new methods and quantitative tools to detect multiple gene expression patterns in single cells using multiplex in situ hybridization [23] as well as computational techniques to quantitatively measure gene expression levels [24]. We have applied these to generate testable models of pattern formation in both the developing embryo and wing. For example, most recently, we have proposed that differences in gene length alone can create a transient nested pattern of gene expression to help seed morphogen-dependent patterning across a developing field of cells [24]. We have also developed bioinformatic tools for mining and comparing genome data [25, 26].  Select Publications Francois, V., M. Solloway, J.W. O'Neill, J. Emery, and E. Bier. Dorsal-ventral patterning of the Drosophila embryo depends on a putative negative growth factor encoded by the short gastrulation gene . 1994. Genes Dev 8: 2602-16. Francois, V. and E. Bier. Xenopus chordin and Drosophila short gastrulation genes encode homologous proteins functioning in dorsal-ventral axis formation . 1995. Cell 80: 19-20. Biehs, B., V. Francois, and E. Bier. The Drosophila short gastrulation gene prevents Dpp from autoactivating and suppressing neurogenesis in the neuroectoderm . 1996. Genes Dev 10: 2922-34. Srinivasan, S., K.E. Rashka, and E. Bier. Creation of a Sog morphogen gradient in the Drosophila embryo . 2 002. Dev Cell 2: 91-101. Mizutani, C.M., Q. Nie, F.Y. Wan, Y.T. Zhang, P. Vilmos, R. Sousa-Neves, E. Bier, J.L. Marsh, and A.D. Lander. Formation of the BMP activity gradient in the Drosophila embryo . 2005. Dev Cell 8: 915-24. Mizutani, C.M., N. Meyer, H. Roelink, and E. Bier. Threshold-Dependent BMP-Mediated Repression: A Model for a Conserved Mechanism That Patterns the Neuroectoderm . 2006. PLoS Biology 4: e313. Mizutani, C.M. and E. Bier. EvoD/Vo: the origins of BMP signalling in the neuroectoderm . 2008. Nat Rev Genet . Bier, E. Evolution of development: diversified dorsoventral patterning . 2011. Curr Biol 21: R591-4. Biehs, B., M.A. Sturtevant, and E. Bier. Boundaries in the Drosophila wing imaginal disc organize vein-specific genetic programs . 1998. Development 125: 4245-57. Bier, E. Drawing lines in the Drosophila wing: initiation of wing vein development . 2000. Curr Opin Genet Dev 10: 393-8. Lunde, K., B. Biehs, U. Nauber, and E. Bier. The knirps and knirps-related genes organize development of the second wing vein in Drosophila . 1998. Development 125: 4145-54. Lunde, K., J.L. Trimble, A. Guichard, K.A. Guss, U. Nauber, and E. Bier. Activation of the knirps locus links patterning to morphogenesis of the second wing vein in Drosophila . 2003. Development 130: 235-48. Cook, O., B. Biehs, and E. Bier. brinker and optomotor-blind act coordinately to initiate development of the L5 wing vein primordium in Drosophila . 2004. Development 131: 2113-24. Bier, E. Drosophila , the golden bug, emerges as a tool for human genetics . 2005. Nat Rev Genet 6: 9-23. Reiter, L.T., T.N. Seagroves, M. Bowers, and E. Bier. Expression of the Rho-GEF Pbl/ECT2 is regulated by the UBE3A E3 ubiquitin ligase . 2006. Hum Mol Genet 15: 2825-35. Wangler, M.F., L.T. Reiter, G. Zimm, J. Trimble-Morgan, J. Wu, and E. Bier. Antioxidant proteins TSA and PAG interact synergistically with Presenilin to modulate Notch signaling in Drosophila . 2011. Protein Cell 2: 554-63. Grossman, T.R., A. Gamliel, R.J. Wessells, O. Taghli-Lamallem., K. Jepsen, K. Ocorr, J.R. Korenberg, K.L. Peterson, M.G. Rosenfeld, R. Bodmer, and E. Bier. Over-expression of DSCAM and COL6A2 cooperatively generates congenital heart defects . 2011. PLoS Genet 7: e1002344. Lee, J.H., A.V. Budanov, E.J. Park, R. Birse, T.E. Kim, G.A. Perkins, K. Ocorr, M.H. Ellisman, R. Bodmer, E. Bier, and M. Karin. Sestrin as a feedback inhibitor of TOR that prevents age-related pathologies . 2010. Science 327: 1223-8. Guichard, A., S.M. McGillivray, B. Cruz-Moreno, N.M. van Sorge, V. Nizet, and E. Bier. Anthrax toxins cooperatively inhibit endocytic recycling by the Rab11/Sec15 exocyst . 2010. Nature 467: 854-8. Guichard, A., V. Nizet, and E. Bier. New insights into the biological effects of anthrax toxins: linking cellular to organismal responses . 2012. Microbes Infect 14: 97-118. Guichard, A., J.M. Park, B. Cruz-Moreno, M. Karin, and E. Bier. Anthrax lethal factor and edema factor act on conserved targets in Drosophila . 2006. Proc Natl Acad Sci U S A 103: 3244-9. Guichard, A., B. Cruz-Moreno, B. Aguilar, J. Kuang, A.A. Kurkciyan, Z. Wang, S. Hang, G.P. Pineton de Chambrun, D.F. McCole, P. Watnick, N.M. van Sorge, V. Nizet, and E. Bier. (2013). Cholera toxin disrupts barrier function by inhibiting exocyst-mediated trafficking of host proteins to intestinal cell junctions. Cell Host & Microbe 14, 294-305. Kosman, D., C.M. Mizutani, D. Lemons, W.G. Cox, W. McGinnis, and E. Bier. Multiplex detection of RNA expression in Drosophila embryos . 2004. Science 305: 846. McHale, P., C.M. Mizutani, D. Kosman, D.L. MacKay, M. Belu, A. Hermann, W. McGinnis, E. Bier, and T. Hwa. Gene length may contribute to graded transcriptional responses in the Drosophila embryo . 2011. Dev Biol 360: 230-40. Reiter, L.T., L. Potocki, S. Chien, M. Gribskov, and E. Bier. A systematic analysis of human disease-associated gene sequences in Drosophila melanogaster . 2001. Genome Res 11: 1114 Do, L.H., F.F. Esteves, H.J. Karten, and E. Bier. Booly: a new data integration platform . 2010. BMC Bioinformatics 11: 513. \n",
            "-------------------- Scraping faculty url 15/183 --------------------\n",
            "brenda bloodgood  Research The goal of our lab is to understand how neuronal computations change in response to an animals interactions with the environment. From the moment an animal is born, its brain is working to extract information from its surroundings and initiate appropriate behavioral responses. This is done through the activity of excitatory and inhibitory neurons that are organized into synaptically connected circuits. Our lab is interested in understanding how experience, via the execution of activity-dependent gene expression, regulates the connectivity of inhibitory and excitatory neurons and how these processes relate to animal behavior and disease states. We focus on addressing three main questions: Which components of the inhibitory microcircuitry change their strength in response to activity-dependent changes in gene expression? What are the cellular and molecular mechanisms that underlie these changes in connectivity? How do changes in inhibition affect excitatory synaptic signaling, synaptic integration, and the propagation of information to downstream neurons? Our studies are performed in the mouse hippocampus to take advantage of its role in spatial learning and memory and the thoroughly characterized circuit architecture. Our experimental approaches are multidisciplinary we manipulate gene expression and molecular function in specific cell types while probing synaptic connectivity with optogenetics, viral circuit tracing, two-photon uncaging of neurotransmitters, and electrophysiology.  Select Publications Bloodgood, B.L.*, Sharma, N.*, Browne, H.A., Trepman, A.Z., Greenberg, M.E., Domain-specific regulation of inhibitory synapses by the activity-dependent transcription factor Npas4, Nature, 2013 Nov 7. 503(7474):121-5. Greer, P. L., Hanayama, R., Bloodgood, B. L., Flavell, S.W., Mardinly, A.R., Lipton, D.M., Kim, TK., Griffith, E.C., Waldon, Z., Maehr, R., Ploegh, H. L., Chowdhury, S., Worley, P. F., Steen, J., Greenberg, M. E. (2010). The AngelmanSyndrome-associated ubiquitin ligase Ube3a regulates synapse development and function through the ubiquination of Arc. Cell 140:704-716. Bloodgood B. L., Giessel A. J., Sabatini B. L. (2009). Biphasic synaptic Ca influx arising from compartmentalized electrical signals in dendritic spines. PLoS Biol. 7:e1000190. Lin Y., Bloodgood B.L., Hauser J.L., Lapan A.D., Koon A.C., Kim T.K., Hu L.S., Malik A.N., Greenberg M.E. (2008). Activity-dependent regulation of inhibitory synapse development by Npas4. Nature 455:1198-1204. Bloodgood, B.L., and Sabatini B.L. (2008). Regulation of synaptic signaling by postsynaptic, non-glutamate receptor ion channels. J. Physiol. 586:1475-1480. PMC2375695 Shankar G. M, Bloodgood B. L., Townsend M., Walsh D. M., Dennis J. Selkoe D. J., Sabatini B. L. (2007). Natural oligomers of the Alzheimer amyloid-beta protein induce reversible synapse loss by modulation of an NMDAR dependent signaling pathway. J. Neurosci. 27:2866-2875. Bloodgood, B.L., Sabatini B.L. (2007). Ca2+ signaling in dendritic spines. Curr.t Opin. Neurobiol. 17: 345-351. PMID: 17451936 Bloodgood B.L., Sabatini B.L. (2007). Nonlinear regulation of unitary synaptic signals by CaV2.3 voltage-sensitive calcium channels located in dendritic spines. Neuron 53: 249-260. Bloodgood B.L., Sabatini B.L. (2005). Neuronal activity regulates diffusion across the neck of dendritic spines. Science 310:866-869. Ngo-Anh T.J., Bloodgood B.L., Lin M., Sabatini B.L., Maylie J., and Adelman J.P. (2005). SK channels and NMDA receptors form a Ca2+-mediated feedback loop in dendritic spines. Nat. Neurosci. 8:642-649. \n",
            "-------------------- Scraping faculty url 16/183 --------------------\n",
            "steve briggs  Biography Briggs earned his Ph.D. from Michigan State University and spent the first half of his career in industry as Director at Pioneer/DuPont; President of Torrey Mesa Research Institute; and Senior Vice President for Corporate Research at Diversa. He served as Senior Staff Investigator at Cold Spring Harbor Laboratory. Briggs joined the faculty at UCSD as Professor where he served as Chair of the Section of Cell & Developmental Biology, Associate Dean of the Division of Biology, and Interim Chair of the Department of Chemistry and Biochemistry. He is a member of the National Academy of Sciences, a Fellow of the American Association for the Advancement of Science, and a Fellow of the National Academy of Inventors.  Research We study molecular signaling networks with a focus on the innate immune system of plants. To support our studies, we developed a proteomics capability that is sensitive enough to discover new genes and gene models (1); discover genes based solely on comparison between the proteomes of mutant and wild type plants (2), and discover mechanisms of signaling mediated by protein phosphorylation (3). We used mass spectrometry to generate a proteomics atlas, elements of which have been published (4, 5, 6). When combined with RNA-seq data, our integrated gene expression atlas was used to derive co-expression networks and gene regulatory networks from the proteome and transcriptome alone and from integration of the data. We found that the transcriptome does not predict the proteome either quantitatively or qualitatively (4) and networks derived from protein levels have little overlap with networks of the same genes measured according to their mRNA levels (7). Recent reports on host-microbe interactions include studies of host peptide signaling (8), pathogen effector secretion (9, 10), self-acetylation of host proteins caused by a pathogen effector (11), and biosynthesis of defense metabolites in response to infection (12). We collaborate with other labs to understand problems in signaling that are less central to our interests but that can be uniquely addressed using our expertise. These include understanding endoplasmic reticulum stress (13, 14) and circadian signaling in plants (15); chemotactic signaling in Dictyostelium (16, 17); and the molecular determinants of chronic lymphocytic leukemia (18, 19). Castellana NE, Shen Z, He Y, Walley JW, Cassidy CJ, Briggs SP, Bafna V. An Automated Proteogenomic Method Uses Mass Spectrometry to Reveal Novel Genes in Zea mays. Mol Cell Proteomics. 2014 Jan;13(1):157-67. doi: 10.1074/mcp.M113.031260. Zhang X, Facette M, Humphries JA, Shen Z, Park Y, Sutimantanapi D, Sylvester AW, Briggs SP, Smith LG. Identification of PAN2 by quantitative proteomics as a leucine-rich repeat-receptor-like kinase acting upstream of PAN1 to polarize cell division in maize. Plant Cell. 2012 Nov;24(11):4577-89. doi: 10.1105/tpc.112.104125. Epub 2012 Nov 21. PubMed PMID: 23175742; PubMed Central PMCID: PMC3531853. Qiao H, Shen Z, Huang SS, Schmitz RJ, Urich MA, Briggs SP, Ecker JR. Processing and subcellular trafficking of ER-tethered EIN2 control response to ethylene gas. Science. 2012 Oct 19;338(6105):390-3. doi: 10.1126/science.1225974. Epub 2012 Aug 30. PubMed PMID: 22936567; PubMed Central PMCID: PMC3523706. Walley JW, Shen Z, Sartor R, Wu KJ, Osborn J, Smith LG, Briggs SP. Reconstruction of protein networks from an atlas of maize seed proteotypes. Proc Natl Acad Sci U S A. 2013 Dec 3;110(49):E4808-17. doi: 10.1073/pnas.1319113110. Epub 2013 Nov 18. PubMed PMID: 24248366; PubMed Central PMCID: PMC3856832. Facette MR, Shen Z, Bjrnsdttir FR, Briggs SP, Smith LG. Parallel proteomic and phosphoproteomic analyses of successive stages of maize leaf development. Plant Cell. 2013 Aug;25(8):2798-812. doi: 10.1105/tpc.113.112227. Epub 2013 Aug 9. PubMed PMID: 23933881; PubMed Central PMCID: PMC3784581. Marcon C, Malik WA, Walley JW, Shen Z, Paschold A, Smith LG, Piepho HP, Briggs SP, Hochholdinger F. A high resolution tissue-specific proteome and phosphoproteome atlas of maize primary roots reveals functional gradients along the root axis. Plant Physiol. 2015 Mar 16. pii: pp.00138.2015 Walley JW, Sartor RC, Shen Z, Schmitz RJ, Wu KJ, Urich MA, Nery JR, Smith LG, Schnable JC, Ecker JR, Briggs SP. Integration of omic networks in a developmental atlas of maize. 2016 Science. Aug 19;353(6301):814-8. doi: 10.1126/science.aag1125. PMID: 27540173 Huffaker A, Pearce G, Veyrat N, Erb M, Turlings TC, Sartor R, Shen Z, Briggs SP, Vaughan MM, Alborn HT, Teal PE, Schmelz EA. Plant elicitor peptides are conserved signals regulating direct and indirect antiherbivore defense. Proc Natl Acad Sci U S A. Apr 2;110(14):5707-12. doi: 10.1073/pnas.1214668110. 2013 Chaudhary R, Atamian HS, Shen Z, Briggs SP, Kaloshian I. Potato aphid salivary proteome: enhanced salivation using resorcinol and identification of aphid phosphoproteins. J Proteome Res. 2015 Apr 3;14(4):1762-78. doi: 10.1021/pr501128k. Chaudhary R, Atamian HS, Shen Z, Briggs SP, Kaloshian I. GroEL from the endosymbiont Buchnera aphidicola betrays the aphid by triggering plant defense. Proc Natl Acad Sci U S A. 2014 Jun 17;111(24):8919-24. doi: 10.1073/pnas.1407687111. Walley JW, Shen Z, McReynolds MR, Schmelz EA, Briggs SP. Fungal-induced protein hyperacetylation in maize identified by acetylome profiling. 2018 Proc Natl Acad Sci U S A. Jan 2;115(1):210-215. doi:10.1073/pnas.1717519115. Epub 2017 Dec 19. PMID: 29259121 Mafu S, Ding Y, Murphy KM, Yaacoobi O, Addison JB, Wang Q, Shen Z, Briggs SP, Bohlmann J, Castro-Falcon G, Hughes CC, Betsiashvili M, Huffaker A, Schmelz EA, Zerbe P. 2018 Discovery, Biosynthesis and Stress-Related Accumulation of Dolabradiene-Derived Defenses in Maize. Plant Physiol. Apr;176(4):2677-2690. doi: 10.1104/pp.17.01351. Epub 2018 Feb 23. PMID: 29475898 Wang JZ, Li B, Xiao Y, Ni Y, Ke H, Yang P, de Souza A, Bjornson M, He X, Shen Z, Balcke GU, Briggs SP, Tissier A, Kliebenstein DJ, Dehesh K. 2017 Initiation of ER Body Formation and Indole Glucosinolate Metabolism by the Plastidial Retrograde Signaling Metabolite, MEcPP. Mol Plant. Nov 6;10(11):1400-1416. doi:10.1016/j.molp.2017.09.012. Epub 2017 Sep 28. PMID: 28965830 Walley J, Xiao Y, Wang JZ, Baidoo EE, Keasling JD, Shen Z, Briggs SP, Dehesh K. Plastid-produced interorgannellar stress signal MEcPP potentiates induction of the unfolded protein response in endoplasmic reticulum. Proc Natl Acad Sci U S A. 2015 May 12;112(19):6212-7. doi: 10.1073/pnas.1504828112. Epub 2015 Apr 28. PubMed PMID: 25922532; PubMed Central PMCID: PMC4434691. Huang H, Alvarez S, Bindbeutel RK, Shen Z, Naldrett MJ, Evans BS, Briggs SP, Hicks LM, Kay SA, Nusinow DA. Identification of evening complex associated proteins in Arabidopsis by affinity purification and mass spectrometry. Mol Cell Proteomics. 2015 Nov 6. pii: mcp.M115.054064 Lacal Romero J, Shen Z, Baumgardner K, Wei J, Briggs SP, Firtel RA. The Dictyostelium GSK3 kinase GlkA coordinates signal relay and chemotaxis in response to growth conditions. 2018 Dev Biol. Mar 1;435(1):56-72.doi: 10.1016/j.ydbio.2018.01.007. Epub 2018 Jan 20. PMID: 29355521 Khanna A, Lotfi P, Chavan AJ, Montao NM, Bolourani P, Weeks G, Shen Z, Briggs SP, Pots H, Van Haastert PJ, Kortholt A, Charest PG. 2016 The small GTPases Ras and Rap1 bind to and control TORC2 activity. Sci Rep. May 13;6:25823. doi: 10.1038/srep25823. PMID: 27172998 Hasan MK, Yu J, Widhopf GF 2nd, Rassenti LZ, Chen L, Shen Z, Briggs SP, Neuberg DS, Kipps TJ. Wnt5a induces ROR1 to recruit DOCK2 to activate Rac1/2 in chronic lymphocytic leukemia. 2018 Blood. Jul 12;132(2):170-178. doi: 10.1182/blood-2017-12-819383. Epub 2018 Apr 20. PMID: 29678828May 3. PMID: 284655282016 Yu J, Chen L, Cui B, Widhopf GF 2nd, Shen Z, Wu R, Zhang L, Zhang S, Briggs SP, Kipps TJ. Wnt5a induces ROR1/ROR2 hetero-oligomerization to enhance leukemia chemotaxis and proliferation. J Clin Invest. 2015 Dec 21. pii: 83535. doi: 10.1172/JCI83535.  Select Publications Walley JW, Shen Z, McReynolds MR, Schmelz EA, Briggs SP. Fungal-induced protein hyperacetylation in maize identified by acetylome profiling. Proc Natl Acad Sci U S A. Jan 2;115(1):210-215. doi:10.1073/pnas.1717519115. Epub 2017 Dec 19. PMID: 29259121 Mafu S, Ding Y, Murphy KM, Yaacoobi O, Addison JB, Wang Q, Shen Z, Briggs SP, Bohlmann J, Castro-Falcon G, Hughes CC, Betsiashvili M, Huffaker A, Schmelz EA, Zerbe P. Discovery, Biosynthesis and Stress-Related Accumulation of Dolabradiene-Derived Defenses in Maize. Plant Physiol. Apr;176(4):2677-2690. doi: 10.1104/pp.17.01351. Epub 2018 Feb 23. PMID: 29475898 Lacal Romero J, Shen Z, Baumgardner K, Wei J, Briggs SP, Firtel RA. The Dictyostelium GSK3 kinase GlkA coordinates signal relay and chemotaxis in response to growth conditions. Dev Biol. Mar 1;435(1):56-72.doi: 10.1016/j.ydbio.2018.01.007. Epub 2018 Jan 20. PMID: 29355521 Hasan MK, Yu J, Widhopf GF 2nd, Rassenti LZ, Chen L, Shen Z, Briggs SP, Neuberg DS, Kipps TJ. Wnt5a induces ROR1 to recruit DOCK2 to activate Rac1/2 in chronic lymphocytic leukemia. Blood. Jul 12;132(2):170-178. doi: 10.1182/blood-2017-12-819383. Epub 2018 Apr 20. PMID: 29678828May 3. PMID: 28465528 Hasan MK, Yu J, Chen L, Cui B, Widhopf Ii GF, Rassenti L, Shen Z, Briggs SP, Kipps TJ. Wnt5a induces ROR1 to complex with HS1 to enhance migration of chronic lymphocytic leukemia cells. Leukemia. Dec;31(12):2615-2622. doi: 10.1038/leu.2017.133. Epub 2017 May 3. PMID: 28465529 Marcon C, Malik WA, Walley JW, Shen Z, Paschold A, Smith LG, Piepho HP, Briggs SP, Hochholdinger F. A high resolution tissue-specific proteome and phosphoproteome atlas of maize primary roots reveals functional gradients along the root axis. Plant Physiol. 2015 Mar 16. pii: pp.00138.2015 Yu J, Chen L, Chen Y, Hasan MK, Ghia EM, Zhang L, Wu R, Rassenti LZ, Widhopf GF, Shen Z, Briggs SP, Kipps TJ. Wnt5a induces ROR1 to associate with 14-3-3 for enhanced chemotaxis and proliferation of chronic lymphocytic leukemia cells. Leukemia. Dec;31(12):2608-2614. doi: 10.1038/leu.2017.132. Epub 2017 May 3. PMID: 28465528 Wang JZ, Li B, Xiao Y, Ni Y, Ke H, Yang P, de Souza A, Bjornson M, He X, Shen Z, Balcke GU, Briggs SP, Tissier A, Kliebenstein DJ, Dehesh K. Initiation of ER Body Formation and Indole Glucosinolate Metabolism by the Plastidial Retrograde Signaling Metabolite, MEcPP. Mol Plant. Nov 6;10(11):1400-1416. doi:10.1016/j.molp.2017.09.012. Epub 2017 Sep 28. PMID: 28965830 Walley JW, Sartor RC, Shen Z, Schmitz RJ, Wu KJ, Urich MA, Nery JR, Smith LG, Schnable JC, Ecker JR, Briggs SP. Integration of omic networks in a developmental atlas of maize. Science. Aug 19;353(6301):814-8. doi: 10.1126/science.aag1125. PMID: 27540173 Khanna A, Lotfi P, Chavan AJ, Montao NM, Bolourani P, Weeks G, Shen Z, Briggs SP, Pots H, Van Haastert PJ, Kortholt A, Charest PG. The small GTPases Ras and Rap1 bind to and control TORC2 activity. Sci Rep. May 13;6:25823. doi: 10.1038/srep25823. PMID: 27172998 Yu J, Chen L, Cui B, Widhopf GF 2nd, Shen Z, Wu R, Zhang L, Zhang S, Briggs SP, Kipps TJ. Wnt5a induces ROR1/ROR2 heterooligomerization to enhance leukemia chemotaxis and proliferation. J Clin Invest. Feb;126(2):585-98. PMID: 26690702 Huang H, Alvarez S, Bindbeutel RK, Shen Z, Naldrett MJ, Evans BS, Briggs SP, Hicks LM, Kay SA, Nusinow DA. Identification of evening complex associated proteins in Arabidopsis by affinity purification and mass spectrometry. Mol Cell Proteomics. Jan;15(1):201-17. doi: 10.1074/mcp.M115.054064. Epub 2015 Nov 6. PMID: 26545401 Marcon C, Malik WA, Walley JW, Shen Z, Paschold A, Smith LG, Piepho HP, Briggs SP, Hochholdinger F. A high-resolution tissue-specific proteome and phosphoproteome atlas of maize primary roots reveals functional gradients along the root axes. Plant Physiol. May;168(1):233-46. doi: 10.1104/pp.15.00138. Epub 2015 Mar 16. PubMed PMID: 25780097; PubMed Central PMCID: PMC4424028. Yu J, Chen L, Cui B, Widhopf GF 2nd, Shen Z, Wu R, Zhang L, Zhang S, Briggs SP, Kipps TJ. Wnt5a induces ROR1/ROR2 heterooligomerization to enhance leukemia chemotaxis and proliferation. J Clin Invest. Dec 21. pii: 83535. doi: 10.1172/JCI83535. Nakasaki M, Hwang Y, Xie Y, Kataria S, Gund R, Hajam EY, Samuel R, George R, Danda D, M J P, Nakamura T, Shen Z, Briggs S, Varghese S, Jamora C. The matrix protein Fibulin-5 is at the interface of tissue stiffness and inflammation in fibrosis. Nat Commun. Oct 15;6:8574. doi: 10.1038/ncomms9574. Pineda G, Shen Z, Ponte de Albuquerque C, Reynoso E, Chen J, Tu C-C, Tang W, Briggs S, Zhou H, Wang JY. Proteomics Studies of the Interactome of RNA Polymerase II C-Terminal Repeated Domain. BMC Research Notes Oct 29;8:616. doi: 10.1186/s13104-015-1569-y Walley J, Xiao Y, Wang J-Z, Baidoo EE, Keasling JD, Shen Z, Briggs SP, Dehesh K. Plastid-produced interorgannellar stress signal MEcPP potentiates induction of the unfolded protein response in endoplasmic reticulum. Proc Natl Acad Sci USA April 28, doi: 10.1073/pnas.1504828112 Xiong J, Todorova D, Su NY, Kim J, Lee PJ, Shen Z, Briggs SP, Xu Y. Stemness factor Sall4 is required for DNA damage response in embryonic stem cells. J Cell Biol. Mar 2;208(5):513-20. doi: 10.1083/jcb.201408106. Chaudhary R, Atamian HS, Shen Z, Briggs SP, Kaloshian I. Potato aphid salivary proteome: enhanced salivation using resorcinol and identification of aphid phosphoproteins. J Proteome Res. Apr 3;14(4):1762-78. doi: 10.1021/pr501128k. \n",
            "-------------------- Scraping faculty url 17/183 --------------------\n",
            "stuart brody  Biography Stuart Brody received his Ph.D. from Stanford University and was a postdoctoral fellow and assistant professor at Rockefeller University. Professor Brody was a recipient of an NIH Career Development Award from 1974 to 1979.  Research Studies on the biochemical genetics of Neurospora focuses on its circadian rhythms This rhythm is visibly expressed in Neurospora as a spore-forming rhythm in a petri dish (see photo below). Areas of the culture showing the developmental phenomena of spore formation can be seen as \"bands\" on the surface of the agar. Mutations affecting the rhythm lead to different numbers of bands on a plate as seen in the cla-1 strain below. We are investigating the effects of several classes of mutations on the clock: analyzing the effects of mutants such as vivid (vvd) that affect the light response of the clock and lead to very short ( as low as 6 hrs.) periods in constant light; describing and analyzing the components of a second oscillator, the frq-less oscillator (FLO) in Neurospora , employing mutants such as csp-1 and rco-1 which affect downstream processes; using fluorescent tags to describe the synthesis and localization of key clock proteins , such as FRQ in vivo; understanding how the clock-controlled genes ( ccg) are regulated by the core oscillator.  Select Publications Lakin-Thomas, P.L. and S. Brody (2000). Circadian rhythms in Neurospora crassa : Lipid deficiencies restore robust rhymicity to frq null and white-collar mutants. Proc. Natl. Acad. Sci. USA 97:256-261. Granshaw, T., Tsukamoto, M., and S. Brody. ( 2003). Circadian Rhythms in Neurospora crassa : Farnesol or geraniol allow expression of rhythmicity in the otherwise arrhythmic strains frq10, wc-1 and wc-2. J.Biol. Rhythms 18:287-296. Lakin-Thomas, P.L. and S. Brody. ( 2004). Circadian Rhythms in Microorganisms: New Complexities. Ann. Review of Microbiol. 58:489-519. Brody, S. ( 2005). Genetics and Kinetics. Fungal Genetics and Biology. 42:81-96. Lombardi, L.M. and S. Brody (2005). Circadian Rhythms in Neurospora crassa : Clock gene homologues in fungi. Fungal Genetics and Biology. 42:887-892. Lombardi, L.M., Schneider,K. and S. Brody (2007) Circadian Rhythms in Neurospora crassa : Clock mutant effects in the absence of a frq-based oscillator. Genetics 175:1175-1183. Castro-Longoria, E., Brody,S. and S. Bartnicki-Garcia ( 2007) Kinetics of circadian band development in Neurospora crassa . Fungal Genetics and Biology 44:672-681. \n",
            "-------------------- Scraping faculty url 18/183 --------------------\n",
            "wolfgang busch \n",
            "-------------------- Scraping faculty url 19/183 --------------------\n",
            "madeline butler \n",
            "-------------------- Scraping faculty url 20/183 --------------------\n",
            "hu cang \n",
            "-------------------- Scraping faculty url 21/183 --------------------\n",
            "sreekanth chalasani \n",
            "-------------------- Scraping faculty url 22/183 --------------------\n",
            "lin chao  Biography Lin Chao received his Ph.D. from the University of Massachusetts, Amherst. He was an NIH postdoctoral fellow at Princeton Universit/y./ Lin joined UCSD in 1999 and he served as the Chair of the Section of Ecology, Behavior and Evolution from 2004 to 2008. In 2014, he received the UCSD Chancellor's Associates Award for Excellence in undergraduate teaching.  Research We use a combination of computer modeling and laboratory experimentation to study ecological and evolutionary processes. Ecological and evolutionary studies are often difficult to investigate because they occur slowly over long periods of time. Computer models can speed up time, but we also take advantage of the fact that microbes, bacteria and viruses provide that same advantage because of their short generation times. Furthermore, they can be easily cultured in the laboratory under controlled conditions. However, we also realize that too much control can be dangerous because an experiment that is too controlled becomes artificial and not much more than a simulation. And a simulation is always more easily obtained from a computer model. Thus, we are careful to design experiments that teach us more than a computer model. We make sure that the experiments allow us to measure an unknown that could not be determined a priori from a computer model. We have used microbial systems to study topics ranging from the ecology of host-parasite co-existence and the evolution of sex, mutation rates, transposable elements, group and individual adaptations, game theory strategies, Fishers Geometric Model of Adaptive Evolution, deleterious and compensatory mutations. As new information or interests develop, we often revisit older topics or explore new ones. Time lapse photograph E. coli cells descending from a single cell. By recording division of all cells via time lapse, the doubling time or life history fitness of every individual in a population can be determined. Our current research focuses on evolution of microbial aging or senescence, which grew out of our interest in genetic damage or mutations. Because most mutations are deleterious, they lead to fitness loss unless removed by natural selection. The transmission of mutations from mother to daughter is governed by well the well known rules of genetic inheritance and assortment. However, non-genetic or phenotypic damage can also decrease fitness and, unlike mutations, the transmission rules for phenotypic damage are not well understood. If a mother cell has ten units of phenotypic damage, for example ten proteins damaged by oxidation, should she distribute the damage evenly between her two daughters or give all ten units to one daughter and none to the other? Our theoretical models have shown that a lineage maximizes its fitness by allocating all the damage to a single daughter. Thus, asymmetry should evolve and the daughter receiving all or more damage should senesce over time. Time lapse photography of dividing cells has already shown that bacteria divide asymmetrically. We are now making our own time lapse series to test more directly the prediction of our models.  Select Publications Chao, L., Rang, C. U., Proenca, A. M. and Chao, J.U. 2016. Asymmetrical Damage Partitioning in Bacteria: A Model for the Evolution of Stochasticity, Determinism, and Genetic Assimilation. PLOS Computational Biology DOI: 10.1371/journal.pcbi.1004700 Campbell EM, Chao L. 2014. A Population Model Evaluating the Consequences of the Evolution of Double-Resistance and Tradeoffs on the Benefits of Two-Drug Antibiotic Treatments. PLoS ONE doi:10.1371/journal.pone.0086971 Samuel L Daz-Muoz, Olivier Tenaillon, Daniel Goldhill, Kristen Brao, Paul E Turner and Lin Chao. 2013. Electrophoretic mobility confirms reassortment bias among geographic isolates of segmented RNA phages. BMC Evolutionary Biology 13:206. Rang, C. U., Peng, A. Y. and Chao, L. 2012. Bacterial aging requires an extrinsic damage agent. Microbiology 158:15531559 Rang, C. U., A. Y. Peng, and L Chao. 2011. Temporal Dynamics of Bacterial Aging and Rejuvenation. Current Biology. In Press. Chao L. 2010. A Model for Damage Load and Its Implications for the Evolution of Bacterial Aging. PLoS Genet 6(8): e1001076. doi:10.1371/journal.pgen.1001076 OKeefe, K. J., O. K. Silander, H. McCreery, D. M. Weinreich, K. M. Wright, L. Chao, S. V. Edwards, S. K. Remold, and P. E. Turner. 2010. Biogeography of sexual reassortment in RNA phage. Evolution. doi:10.1111/j.1558-5646.2010.01040.x. Tenaillon, O., Silander, O., Uzan, J-P., and L. Chao. 2007. Quantifying Organismal Complexity using a Population Genetic Approach. PLoS ONE 2:1-8. Olin K. Silander, Olivier Tenaillon and Lin Chao. 2007. Understanding the Evolutionary Fate of Finite Populations:The Dynamics of Mutational Effects. PLoS Biol 5:922-931. Ackermann, M., Chao, L, Bergstrom, C. T., and Doebeli, M. 2007. On the Evolutionary Origin of Aging. Aging Cell 6:235-244. Poon, A. and L. Chao. 2006. Functional origins of fitness-effect sizes of compensatory mutations in the DNA bacteriophage PhiX174. Evolution 60: 2032-2043. Chao, L., C. U. Rang, and L. E. Wong. 2002. The distribution of spontaneous mutants and inferences about the replication mode of the RNA bacteriophage 6. J. of Virology 76:32763281. Burch, L. C. and L. Chao. 2000. Evolvability of an RNA virus determined by its mutational neighborhood. Nature 406:625-628. Turner, P. E. and L. Chao. 1999. Prisoners Dilemma in an RNA Virus. Nature 398:441-443. \n",
            "-------------------- Scraping faculty url 23/183 --------------------\n",
            "lakshmi chilukuri \n",
            "-------------------- Scraping faculty url 24/183 --------------------\n",
            "andrew chisholm  Research Morphogenesis and regeneration of skin and neurons in C. elegans My labs interests are in the early development of epidermal and neuronal tissues, and in the responses of epidermal and neuronal cells to damage. We are studying these processes in the nematode worm Caenorhabditis elegans . C. elegans is an excellent organism for analyzing fundamental aspects of development. Worm genetics is simple and cheap; gene function can also be probed using genome wide RNA interference screens. Embryogenesis takes 12 hours and its dynamics can be studied using timelapse microscopy and fluorescent markers. We are interested in epidermal (skin) development as a model for epithelial morphogenesis. The worm epidermis is a simple epithelium that encases the animal. In embryogenesis the epidermis spreads out over substrate cells to enclose the embryo (enclosure) and subsequently undergoes elongation (Chisholm and Hardin 2005). Pathways involved in these processes are evolutionarily conserved, and several are implicated in cancer or other genetic diseases. Eph signaling and the role of the neuronal substrate in epidermal enclosure We showed that signaling via the C. elegans Eph receptor tyrosine kinase and its ephrin ligands are required for enclosure (Chin-Sang et al., 1999). Ephrin signaling acts to promote earlier movements of neuroblasts that form a substrate for epidermal enclosure. We are currently using genetics, laser microsurgery and quantitative timelapse microscopy to define the cellular and molecular basis of these neuroblast migrations. Using advanced imaging and new image analysis algorithms we have been able to track the migrations of all nuclei in the embryo during epidermal morphogenesis. We plan to use mathematical modeling to understand how forces are generated in the substrate and in the epidermis to promote spreading. Movements of epidermal cells (green) and neurons (red) during epidermal enclosure (Claudiu Giurumescu and Alan Kang) The extracellular matrix and epidermal cell shape change In large-scale genetic screens for epidermal morphogenesis mutants we identified numerous genes affecting epidermal cell-matrix attachments and the extracellular matrix, in many cases providing the first genetic models for a conserved gene family. Among these we identified peroxidasin, a poorly understood enzyme affecting the extracellular matrix (Gotenstein et al., 2010), as essential for elongation and later tissue adhesion. We are currently focusing on peroxidasin and its interactors, with the goal of identifying substrates of peroxidasin activity. Epidermal wound healing responses The epidermis forms the first line of defense against injury and microbial pathogens. In collaboration with Jonathan Ewbank (Marseille) we have developed C. elegans as a model for epidermal wound responses and repair pathways (Pujol et al., 2008). We have identified mutants in which the epidermal damage response is constitutively activated (Tong et al., 2009). We are focusing on the role of calcium signals in damage sensing and how injury signals are interpreted by the epidermis. A ring of actin bundles forms after wounding (Suhong Xu) C. elegans is a genetically accessible model for axon regeneration The ability of axons to regrow after injury has been known for decades, yet axon regrowth mechanisms remain poorly understood owing in part to the lack of genetically tractable models. In collaboration with Yishi Jins lab (UCSD) we have developed C. elegans as a genetically accessible model for axon regeneration (Wu et al., 2007). C. elegans neurons display robust regenerative responses that critically depend on the DLK-1 MAP kinase pathway and second messenger cascades (Ghosh-Roy et al., 2010). We are performing large-scale screens to identify new regeneration genes; results from a pilot screen of 650 genes have identified a large number of new genes whose role in axon regrowth was not previously suspected (Chen et al., 2011). We are continuing to expand this screen and to dissect the roles of these novel regeneration factors. Axon regeneration after laser injury (Lizhen Chen)  Select Publications Chisholm Lab selected publications, arranged by topic: Morphogenesis Gotenstein JR, Swale RE, Fukuda T, Wu Z, Giurumescu CA, Goncharov A, Jin Y, Chisholm A.D. 2010. The C. elegans peroxidasin PXN-2 is essential for embryonic morphogenesis and inhibits adult axon regeneration. Development 137: 3603-3613 Dejima, K., Kang, S.-R., Mitani, S., Cosman, P., and Chisholm, A.D. 2014. Syndecan defines precise spindle orientation by modulating Wnt signaling in C. elegans . Development 141: 4354-65. Reviews: Chisholm, A.D., and Hardin, J.D. 2005. Chapter on Epidermal Morphogenesis for Wormbook. Chisholm, A.D., and Hsiao, T.I. 2012. The C. elegans epidermis as a model skin I: development, patterning, and growth. WIREs Developmental Biology. doi:10.1002/wdev79 Epidermal Wound Healing Pujol, N., Cypowyj, S., Ziegler, K., Millet, K., Astrain, A., Goncharov, A., Jin, Y., Chisholm, A.D., and Ewbank, J.J. 2008. Distinct innate immune responses to infection and wounding in the C. elegans epidermis. Curr. Biol. 18: 481-489. Tong, A., Lynn, G., Ngo, V., Wong, D., Moseley, S.L., Ewbank., J.J., Goncharov, A., Wu, Y., Pujol, N., and Chisholm, A.D. 2009. Negative regulation of C. elegans epidermal damage responses by Death Associated Protein Kinase. Proc. Natl. Acad. Sci. U.S.A 106: 1457-61. Xu, S., and Chisholm, A.D. 2011. A Gq Ca2+ signaling pathway promotes actin-mediated epidermal wound closure in C. elegans . Curr. Biol. 21: 1960-7. Xu, S., and Chisholm, A.D. 2014. Skin wounding triggers a mitochondrial ROS burst that promotes wound repair. Dev. Cell 31: 48-60. Chuang, M., Hsiao, T.I., Tong, A., Xu, S., Chisholm, A.D. 2016. DAPK interacts with Patronin and the microtubule cytoskeleton in C. elegans epidermal development and wound repair. eLife , Sep 23;5. pii: e15833. Reviews: Chisholm, A.D., and Xu, S. 2012. The C. elegans epidermis as a model skin II: differentiation and physiological roles. WIREs Developmental Biology. doi:10.1002/wdev77 Chisholm, A.D. 2015. Epidermal wound healing in the nematode C. elegans . Advances in Wound Care 4: 264-71. Axon Regeneration Wu, Z., Ghosh-Roy, A., Yanik, M.F., Zhang, J.Z., Jin, Y., and Chisholm, A.D. 2007. C. elegans neuronal regeneration is influenced by life stage, synaptic branching, and ephrin signaling. Proc. Natl. Acad. Sci. U.S.A. 104: 15132-15137. Chen, L., Wang, Z., Ghosh-Roy, A., Hubert, T., Yan, D., ORourke, S., Bowerman, B., Wu, Z., Jin, Y., Chisholm, A.D. 2011. Axon regeneration pathways identified by systematic genetic screening in C. elegans . Neuron 71: 1043-1057. Chuang, M., Goncharov, A., Wang, S., Oegema, K., Jin, Y., Chisholm, A.D. 2014. The microtubule minus end binding protein Patronin/PTRN-1 is required for axon regeneration in C. elegans . Cell Reports, Nov 6;9(3): 874-83 Chen, L., Chuang, M., Koorman, T., Boxem, M., Jin, Y., and Chisholm, A.D. 2015. Axon injury triggers EFA-6 mediated destabilization of axonal microtubules via TACC and doublecortin like kinase. eLife , Epub 9/4/15. eLife.08695. Chen, L., Liu, Z., Zhou, Y., Wei, C., Zhou, Y., Rosenfeld, M.G., Fu, X.-D., Chisholm, A.D., and Jin, Y. 2016. CELF RNA binding proteins promote axon regeneration in C. elegans and mammals through alternative splicing of Syntaxins. eLife, 6/2/16. pii: e16072 Reviews: Chisholm, A.D. 2013. Cytoskeletal dynamics in C. elegans axon regeneration. Annual Review of Cell and Developmental Biology 29: 271-97. PMCID: PMC3183269 Chisholm, A.D., Hutter, H., Jin, Y., and Wadsworth, W.G. 2016. The genetics of axon guidance and axon regeneration in Caenorhabditis elegans . Genetics (WormBook). Methods and Tools Giurumescu, C.A, Kang, S., Planchon, T.A., Betzig, E., Bloomekatz, J., Yelon, D., Cosman, P., and Chisholm, A.D. 2012. Quantitative semi-automated analysis of morphogenesis with single cell resolution in complex embryos. Development 139: 4271-9. Xu, S., and Chisholm, A.D. 2016. Highly efficient optogenetic cell ablation in C. elegans using miniSOG. Sci. Rep. 6: 21271 Wang S, Tang NH, Lara-Gonzalez P, Zhao Z, Cheerambathur DK, Prevo B, Chisholm AD, Desai A, Oegema K. 2017. A toolkit for GFP-mediated tissue-specific protein degradation in C. elegans . Development . 144(14): 2694-2701 For a complete list search Pubmed for Chisholm AD[Author], or search Google Scholar for Andrew Chisholm. \n",
            "-------------------- Scraping faculty url 25/183 --------------------\n",
            "senyon choe \n",
            "-------------------- Scraping faculty url 26/183 --------------------\n",
            "joanne chory  Biography Joanne Chory received an A.B. degree in biology with honors from Oberlin College, OH, a Ph.D. in microbiology from the University of Illinois at Urbana-Champaign, and conducted postdoctoral research at Harvard Medical School. In 1988, she joined the faculty of the Salk Institute, where she has remained. Dr. Chory has served on numerous advisory committees and editorial boards, and has received multiple awards. In 2003, Dr. Chory was named Scientific American's Research Leader in Agriculture. She is a member of the U.S. National Academy of Sciences, the German National Academy of Sciences (Leopoldina), the American Academy of Arts and Sciences, is a fellow of the American Association for the Advancement of Science. Joanne Chory is a foreign affiliate of the French Academy of Science, a foreign member of the Royal Society of London, and is an associate member of EMBO.  Research Our lab wants to understand the mechanisms by which plants respond to changes in their light environment. We use the reference plant, Arabidopsis thaliana , to identify components of the phototransduction pathways that link changes in the light environment with differential growth and global alteration of the transcriptome. Our genetic analyses indicate that light responses are not simply endpoints of linear signal transduction pathways, but are the result of the integration of information from a variety of photoreceptors acting through a complex network of interacting signaling components. Our studies have also revealed that plant steroid hormones and auxin are involved in light-regulated development of plants. Current studies are focused on defining the mechanistic links between the photoreceptor signaling pathways and endogenous developmental programs that involve the action of plant hormones. Our studies are relevant to plant adaptation to global climate change.  Select Publications Kinoshita, T., A. Cano-Delgado, H. Seto, S. Hirayama, S. Fujioka, S. Yoshida, and J. Chory. (2005). Direct binding of brassinosteroids to the extracellular domain of plant receptor kinase BRI1. Nature . 433: 167-171. Jaillais, Y., M. Hothorn, Y. Belkhadir, T. Dabi, Z.L. Nimchuk, E.M. Meyerowitz, and J. Chory. (2011). Tyrosine phosphorylation controls brassinosteroid receptor activation by triggering membrance release of its kinase inhibitor. Genes Dev. 25: 232-237. Tao, Y., K. Ljung, J-L. Ferrer, F. Pojer, F. Hong, J. Long, L. Li, J. Moreno, M. Bowman, L. Ivans, Y. Chen, J. Lim, Y. Zhao, C. Ballare, G. Sandberg, J. Noel, and J. Chory. (2008). Rapid synthesis of auxin via a new tryptophan-dependent pathway is required for shade avoidance in plants. Cell . 133: 164-176. Michael, T.P., T.C. Mockler, G. Breton, C. McEntee, A. Byer, J.D. Trout, S.P. Hazen, H.D. Priest, C.M. Sullivan, R. Shen, S.A. Givan,, M. Yanovsky, F. Hong, S.A. Kay, and J. Chory. (2008). Network discovery pipeline elucidates conserved time of day specific cis-regulatory modules. PLoS Gen. 4:e14. Epub 2007 Dec 18. Koussevitzky, S., A. Nott, T.C. Mockler, F. Hong, G. Sacheto-Marts, M. Surpin, J. Lim, R. Mittler, and J. Chory. (2007). Signals from chloroplasts converge to regulate nuclear gene expression. (Article). Science . 316: 715-719. Epub: 2007 Mar 29. (Accompanying Perspectives) (*co-first authors). Michael, T. Priest, C.M. Sullivan, R. Shen, S.A. Givan,, M. Yanovsky, F. Hong, S.A. Kay, and J. Chory. (2008). Network discovery pipeline elucidates conserved time of day specific cis-regulatory modules. PLoS Gen. 4:e14. Epub 2007 Dec 18. Woodson, J.D., J.M. Perez-Ruiz, and J. Chory. (2011). Heme synthesis by plastid ferrochelatase I regulates nuclear gene expression in plants. Curr. Biol . 24: 897-903. Epub 2011 May 24. Hothorn, M., Y. Belkhadir, M. Dreux, T. Dabi, J.P. Noel, I.A. Wilson, and J. Chory. (2011). Structural basis of steroid hormone perception by a membrane receptor kinase. Nature, 474: 467-471. Epub2011 Jun12. \n",
            "-------------------- Scraping faculty url 27/183 --------------------\n",
            "maarten chrispeels  Biography Maarten Chrispeels received his Ph.D. from the University of Illinois and was a postdoctoral fellow at the DOE Plant Research Laboratory at Michigan State University. Professor Chrispeels was awarded a Guggenheim Fellowship and is a member of the National Academy of Sciences. He is the editor in chief of Plant Physiology.  Research Maarten Chrispeels came to the US in 1960 for graduate study at University of Illinois. After receiving his PhD in Agronomy, he became a postdoctoral fellow at the DOE Plant Research Laboratory at Michigan State University and studied plant biochemistry. His active research career at UCSD spanned 42 years. Professor Chrispeels was elected to membership in the US National Academy of Sciences in 1996. For 10 years he served as the Director of the San Diego Center for Molecular Agriculture (SDCMA) on the UCSD campus. Research Research in the Chrispeels laboratory concerned the biosynthesis and transport of proteins and glycoproteins in developing legume seeds and focused on storage proteins and lectins. They elucidated the role of the Golgi apparatus in the modification of asparagine-linked glycans and the transport of glycoproteins to protein storage vacuoles. They used transgenic technologies to identify the domains of proteins that allow them to be targeted to vacuoles. This work led to the study of a unique protein in the common bean ( Phaseolus vulgaris ) called amylase inhibitor. They showed that this plant defense protein is evolutionarily related to the lectin phytohemagglutinin and used the gene that encodes amylase inhibitor to produce the first insect resistant seeds using genetic engineering. This project was carried out in collaboration with scientists at Purdue University and at the CSIRO in Canberra, Australia. A year after aquaporins were described in animal cells (by Nobel Laureate Peter Agres lab) the Chrispeels lab found homologous proteins in the tonoplasts of plants and aquaporins became the main focus of the lab until Chrispeels retirement from research in 2007. The discovery of aquaporins in plants can be said to have revolutionized the study of plant-water relations. Together with his collaborators Chrispeels published some 180 research papers and 50 reviews and book chapters. Teaching For many years Professor Chrispeels taught courses in plant biology at different levels (introductory courses for non-science majors or for science majors, or advanced courses in plant development and plant biotechnology. To aid in teaching he and his former student David Sadava co-wrote or edited several books, including Plants, Food and People (1977), Plants, Genes and Agriculture (1994) and Plants, Genes and Crop Biotechnology (2003). After his retirement from research he continued to teach at the university and became involved in outreach to San Diego area high schools. Public service and honors For more than 25 years Chrispeels played an active role in the journal Plant Physiology, first as a member of the editorial board, then as an associate member, and from 1990 to 1997 as its Editor in Chief. For 15 years he was involved with the Millennium Science Initiative in Chile and became the convener of the Program Committee that evaluates research proposals from Chilean Scientists. As director of the SDCMA he organized ten major symposia and other events for San Diego area plant biologists. He also produced a colorful 16-page brochure entitled Foods from Genetically Engineered Crop s that was widely distributed throughout the world. His work was recognized by his peers through his election to the US National Academy of Sciences (1996), and receipt of the Stephen Hales Prize of the American Society of Plant Biologists (ASPB) in 1996. In 2000 he received a Doctor Honoris Causa from the University of Guelph (Canada) and was elected to the rank of Fellow of the ASPB in 2006. In 2007 he became a Fellow of the American Association for the Advancement of Science, and in 2008 he was elected to Foreign Corresponding membership in the Academia Chilena de Ciencias. Industrial activities Professor Chrispeels holds several patents, has consulted for a number of biotechnology companies and is the co-founder of two companies: Phylogics (with Dr. Jeffrey Moore) and Arterra Biosciences in Naples, Italy (with Dr. Gabriella Colucci).  Select Publications Martinez, I.M., M.J. Chrispeels, (2003). Genomic analysis of the unfolded protein response in Arabidopsis shows its connection to important cellular processes. Plant Cell, Vol. 15 , 1-16. Apone, F., N. Alyeshmerni, K. Wiens, D. Chalmers, M.J. Chrispeels and G. Colucci. (2003). The G-protein coupled receptor GCR1 and the +/- -subunit of the heterotrimeric G-protein GPA1 regulate cell division through activation of phosphatidylinositol-specific phospholipase C (PI-PLC). Plant Physiol. 133 : 571-579. Bibyut, K., S.A. Moore, W. Tate, L. Molvig, R.L. Morton, D. P. Rees, P. Chialese, M. J. Chrispeels, L. M. Tabe and T.J.V. Higgins (2004). Transgenic chickpea seeds expressing high levels of a bean alpha-amylase inhibitor. Molecular Breeding 14 : 73-82. Aroca, R., G. Amodeo, S. Fernandez-Illescas, E.M. Herman, F. Chaumont and M. J. Chrispeels (2005). The roles of aquaporins and membrane damage in chilling and hydrogen peroxide induced changes in the hydraulic conductance of maize roots. Plant Physiology 137 : 341-353. Chrispeels, M.J. (2005). L'agricoltura del nuovo millenio mette radici nella genomica. Darwin March/April pp. 64-73. \n",
            "-------------------- Scraping faculty url 28/183 --------------------\n",
            "elsa cleland  Research Research in the Cleland lab focuses on the intersection between plant community and ecosystem ecology, utilizing a combination of experimental, observational and data-synthesis techniques. We have a particular focus on global changes such as climate change, altered precipitation, nitrogen deposition, and invasive species. Current research projects include: 1) Quantifying ecophysiological responses of California native shrubs to shifting precipitation and nitrogen availability in the greenhouse, in comparison to measurements made across natural gradients. 2) Identifying environmental manipulations that maximize the success of ecological restoration efforts in coastal sage scrub ecosystems, and 3) Evaluating the role of phenology (or timing) in structuring native and invasive plant communities. Research into the impacts of global change offers the opportunity to ask fundamental questions about how resources and climate influence the structure and functioning of natural ecosystems, while contributing to a greater understanding of these pressing environmental problems.  Select Publications Waterton, J & EE Cleland (2016). Tradeoff between early emergence and herbivore susceptibility mediates exotic success in an experimental California plant community. Ecology and Evolution 6: 8942-8953. Kopp, CW & EE Cleland (2015). A range-expanding shrub species alters plant phenological response to experimental warming. PloS one 10 (9), e0139029. Cleland, EE, E Esch & J McKinney (2015). Priority effects vary with species identity and origin in an experiment varying the timing of seed arrival. Oikos 124: 33-40. Kopp, CW & EE Cleland (2014). Shifts in plant species elevational range limits and abundances observed over nearly five decades in a western North America mountain range. Journal of Vegetation Science 25: 135-146. Wainwright, CE & EE Cleland (2013). Exotic species display greater germination plasticity and higher germination rates than native species across multiple cues. Biological Invasions 15: 2253-2264. Wolkovich, EM, BI Cook, JM Allen, TM Crimmins, JL Betancourt, S Travers, S Pau, J Regetz, TJ Davies, NJB Kraft, TR Ault, K Bolmgren, SJ Mazer, GJ McCabe, BJ McGill, C Parmesan, N Salamin, MD Schwartz & EE Cleland (2012). Warming experiments underpredict plant phenological responses to climate change. Nature 485: 494-497. Cleland, EE, JM Allen, TM Crimmins, JA Dunne, S Pau, S Travers, ES Zavaleta & EM Wolkovich (2012). Phenological tracking enables positive species responses to climate change. Ecology 93: 17651771. \n",
            "-------------------- Scraping faculty url 29/183 --------------------\n",
            "aaron coleman  Biography Aaron Coleman obtained his Ph.D. in Biomedical Sciences from the University of California, Riverside. He conducted post-doctoral research at the Beckman Research Institute at City of Hope National Medical Center. He joined the Division of Biological Sciences in 2007.  Research Biology Education Research All fields of biology are based on methodical experimentation and careful interpretation of data. Laboratory classes where students perform well-designed experiments using relevant techniques are an essential part of an undergraduate education in the biological sciences. An effective laboratory curriculum needs to immerse students in the experimental process. This means not only teaching the concepts underlying the laboratory work but also fostering the development of scientific reasoning. Our research investigates approaches to improve laboratory education on two fronts. First, we are trying to understand how students learn the scientific reasoning skills that are necessary for them to understand research and become scientifically-literate citizens. Second, we seek to design biochemistry laboratory modules that allow students to have a research-like experience in terms of the experimental systems and techniques used, but that are feasible to implement in high-enrollment courses. Teaching Scientific Reasoning The Vision and Change in Biology Undergraduate Education report (2011) from AAAS lists six core competencies that should be expected from graduates with a degree in the biological sciences. Competencies such as the ability to apply the process of science are best, if not exclusively, learned in the context of a laboratory class. Despite its importance, we are only beginning to understand how students learn the reasoning skills required to apply the process of science. In particular, what are the common misconceptions that interfere with this learning process, and what are the most effective strategies for teaching these skills in the laboratory? These questions are the focus of our research. One area of investigation has been to determine how students use the concepts of correlation, necessity, and sufficiency in interpreting experimental data. These concepts are derived from the basic idea of correlation vs. causation, and using them correctly is essential for reading and evaluating the scientific literature. We found that students frequently make two specific types of reasoning errors that interfere with their ability to correctly interpret data (Coleman et al., 2015). When presented with an experiment that only establishes a correlation between a potential intermediate and a known effect, students frequently interpret the intermediate as being sufficient (causative) for the effect. Also, when presented with an experiment that only determines the necessity of an intermediate for an effect, students frequently interpret it as also providing information on the sufficiency of the intermediate, and vice versa. By identifying these common errors, lab course curricula can be modified to correct for them and more successfully teach these scientific reasoning skills. Another ongoing area of investigation is to understand how the degree of inquiry in a laboratory course affects the learning of scientific reasoning skills. Inquiry is defined as the students determining one or more elements of the scientific process carried out in a lab experiment or project, rather than these being provided by the instructor. We have created a rubric of inquiry elements, such as interpreting preliminary data to form a hypothesis or designing an experiment by selecting different experimental conditions to test. Allowing the students to determine these elements is thought to improve learning outcomes but increases the logistical difficulty of implementing the experiment, particularly in a high-enrollment course. By understanding which inquiry-elements are most important for the learning of specific scientific reasoning skills, these competing factors can be balanced to maximize learning outcomes within the constraints of a given lab course. We are currently studying how specific inquiry elements affect students learning of hypothesis formation, data interpretation, and experimental design. Research on Fibroblast Growth Factor Signal Transduction There are 18 members of the fibroblast growth factor (FGF) family that act in a paracrine or endocrine fashion to accomplish a diverse array of biological effects, from pattern formation during development to wound healing. All of this is mediated through only four transmembrane FGF receptors. I am interested in deciphering the signal transduction by which individual FGFs produce their unique biological responses.  Select Publications Kram, K.E., Yim, K.M., Coleman A.B. and Sato, B.K. (2016) Battle of the bacteria: Characterizing the evolutionary advantage of stationary phase growth. J. Microbiology and Biology Education 17:261-268. Coleman, A.B., Lam, D.P., Soowal, L.N. (2015) Correlation, Necessity, and Sufficiency: Common Errors in the Scientific Reasoning of Undergraduate Students for Interpreting Experiments. Biochemistry and Molecular Biology Education 43:305 315. Coleman, A.B. (2010) New Ideas for an Old Enzyme: A short, question-based laboratory project for the purification and identification of an unknown LDH isozyme. Biochemistry and Molecular Biology Education 38:253-260. Coleman, A.B. (2003). Positive and negative regulation of cellular sensitivity to anti-cancer drugs by FGF-2. Drug Resistance Updates6:85-94. Coleman, A.B., Metz, M., Donohue, C., Schwarz, R., and Kane, S.E. (2002) Chemosensitization by fibroblast growth factor-2 is not dependent on proliferation, S-phase accumulation, or p53 status. Biochemical Pharmacology64:1111-1123. Coleman, A.B., Momand, J., and Kane, S.E. (2000) Basic fibroblast growth factor sensitizes NIH 3T3 cells to apoptosis induced by cisplatin. Molecular Pharmacology 57:324-333. \n",
            "-------------------- Scraping faculty url 30/183 --------------------\n",
            "heidi cook-andersen \n",
            "-------------------- Scraping faculty url 31/183 --------------------\n",
            "james cooke  Biography James Cooke received his Ph.D. from the University of British Columbia and was a postdoctoral fellow at the Hospital for Sick Children in Toronto. Dr. Cooke joined the faculty in 2015. He is currently the Head Faculty Advisor for the Saltman Quarterly.  Research As a teaching faculty member, my research is centred on undergraduate student learning. Our group is broadly interested in retention of course concepts by undergraduate students. One ongoing project uses multi-stage collaborative exams to try and improve long-term retention of course content. Our early results suggested that benefits conferred by writing answers in groups varied by individual. As a follow-up, we are measuring student behaviours during collaborative exams to identify whether certain behaviours are predictive of improved retention at a later time point. If we are able to identify specific actions that are beneficial to student learning, we can then guide future students to engage in these behaviours. Another ongoing project is evaluating whether we can improve content retention and exam performance by reducing the cognitive load due to jargon. We are curious to know if mastering key jargon terms prior to class removes barriers imposed by jargon, freeing up students ability to focus on the deeper, conceptual nature of the material. This project is done in collaboration with Dr. Timothy Rickard in the Department of Psychology at UCSD, and with Dr. Mark McDaniel at the University of Washington, St. Louis.  Select Publications Goodman BE, Barker MK, Cooke JE. (2018) Best practices in active and student-centered learning in physiology classes. Advances in Physiology Education. 42: 417423, 2018 Cooke, J.E., Weir, L., Clarkston, B. (2017) Two-stage collaborative exams differentially influence retention in low-, mid- and high-performing students. Accepted. Han, L, Campanucci, V.A., Cooke, J, Salter M.W. (2013) Identification of a single amino acid in GluN1 that is critical for glycine primed internalization of NMDA receptors. Molecular Brain. 6(1): 36-50. Whitehead, R.A., Puil, E., Ries, C.R., Schwarz, S.K.W., Wall, R.A., Cooke, J.E., Putrenko, I, Sallam, N.A., MacLeod, B.A. (2012) GABAB receptor-mediated selective peripheral analgesia by the non-proteinogenic amino acid, isovaline. Neuroscience. 213: 154-160. Cooke, J.E., Mathers, D.A., Puil, E. (2012) R-isovaline: a subtype-specific agonist at GABAB receptors? Neuroscience. 201: 85-95. Cooke, J.E., Mathers, D.A., Puil, E. (2009) Isovaline causes inhibition by increasing potassium conductance in thalamic neurons. Neuroscience. 164(3): 1235-1243. Mathers, D.A., McCarthy, S.M., Cooke J.E., Ghavanini, A.A., Puil E. (2009) Effects of the -amino acid antagonist TAG on thalamocortical inhibition. Neuropharmacology. 56(8): 1097-1105. Cooke, J.E., Zhang, H., Kelly, J.B. (2007) Detection of sinusoidal amplitude modulated sounds: deficits after bilateral lesions of auditory cortex in the rat. Hearing Research. 231(1-2): 90-99. Kelly, J.B., Cooke, J.E., Gilbride, P.C., Mitchell, C., Zhang, H. (2006) Behavioral limits of auditory temporal resolution in the rat: amplitude modulation and duration discrimination. The Journal of Comparative Psychology. 120(2): 98-105. \n",
            "-------------------- Scraping faculty url 32/183 --------------------\n",
            "kimberly cooper  Biography Kimberly Cooper completed her PhD at the Fred Hutchinson Cancer Research Center in Seattle, WA. She then carried out her postdoctoral studies in the Department of Genetics at Harvard Medical School. She joined the Division of Biological Sciences faculty at U.C. San Diego in November 2013.  Research I can get lost in the exhibits and drawers of a natural history museum, and to me the skeletons are the best part. Its fascinating that different species are so easily recognized by the isolated bones that are left behind when the natural processes of decomposition and fossilization are complete. One cant help wonder how the dolphin turned an arm into a flipper, how the aye-aye got its creepy long finger, and how the horse evolved to run gracefully on a single toe. Decades of research in the mouse and chick model systems as well as analyses of human birth defects have unlocked some of the mysteries of how the limb skeleton is patterned along three coordinate axes from the shoulder to finger tips, thumb to pinky finger, and back of the hand to palm. Yet neither human syndromes, nor the sledgehammer approach to knocking out genes in the mouse replicate the extraordinary range of adaptive morphologies observed in diverse species. This suggests a vast and untapped source of information about limb development and plasticity exists in the 400 million year genetic experiment thats been continuing since the first fishapod flopped onto land. To gain a window into what evolution tells us about the developmental potential of the limb, we primarily focus on the three-toed jerboa, a small bipedal rodent that is closely related to the laboratory mouse and amenable to rearing in captivity. Compared to its quadrupedal relative, the jerboa has greatly elongated hindlimbs with three toes and metatarsals that fuse into a single bone. The unusual morphology of the jerboa skeleton allows us to ask what keeps one bone from growing into its adjacent neighbor? What mechanisms determine skeletal size and the relative proportions of individual elements? How are the correct numbers and positions of the digits established, and are the same mechanisms of digit loss redeployed in species that converge on similar morphologies (i.e. jerboas and horses, pigs and camels)? How can the forelimb and hindlimb evolve independently when both utilize the same developmental pathways? We apply classical embryology, cell biology, quantitative microscopy, high throughput sequence analysis, and mouse genetics in an integrative approach to understand these developmental mechanisms at the tissue and genome level.  Select Publications Tstutsumi, R., Tran, M.P., and Cooper, K.L. (2017) Changing While Staying the Same: Preservation of Structural Continuity During Limb Evolution by Developmental Integration. Review. Integrative and Comparative Biology. Moore, T.Y., Cooper, K.L., Biewener, A.A., Vasudevan, R. (2017) Unpredictability of escape trajectory explains predator evasion ability and microhabitat preference of desert rodents. Nature Communications 8: 440. Saxena, A. and Cooper, K.L. (2017) The origins, scaling, and loss of tetrapod digits. Review. Proc. Biol. Sci. 372(1713):20150482. Saxena A and Cooper KL (2016) Fin to limb within our grasp. New and Views. Nature. 537: 176177. Moore TY, Organ CL, Edwards SV, Biewener AA, Tabin CJ, Jenkins FA Jr, Cooper KL. (2015) Multiple Phylogenetically Distinct Events Shaped the Evolution of Limb Skeletal Morphologies Associated with Bipedalism in the Jerboas. Curr Biol. 25(21): 2785-2794. Cooper KL and Shapiro MD (2015) Preface to the special issue on evolution and morphological diversity. Dev Dyn. 244(10): 1181-1183. Cooper KL (2015) Self-organization in the limb: a Turing mechanism for digit development. Review. Curr Opin Genet Dev. 32: 92-97. Cook JA, Edwards SV, Lacey E, Guralnick RP, Soltis PS, Soltis DE, Welch CK, Bell KC, Galbreath KE, Himes C, Allen JM, Heath TA, Carnaval AC, Cooper KL, Liu M, Hanken J, Ickert-Bond S (2014) Aiming Up: Natural History Collections as Emerging Resources for Innovative Undergraduate Education in Biology. Bioscience. 64(8): 725-734. Cooper KL, Sears KE, Uygur A, Maier J, Baczkowski KS, Brosnahan M, Antczak D, Skidmore JA, Tabin CJ (2014) Patterning and post-patterning modes of evolutionary digit loss in mammals. Nature. 511(7507): 41-45. Cooper KL, Oh S, Sung Y, Dasari R, Kirschner MK, Tabin CJ (2013) Multiple phases of chondrocyte enlargement underlie differences in skeletal proportions. Nature 495: 375-378. Cooper KL (2011) The Lesser Egyptian Jerboa, Jaculus jaculus : A unique rodent model for evolution and development. Cold Spring Harbor Protocols (12): 1451-1456. Jordan B, Vercammen P, Cooper KL (2011) Husbandry and breeding of the Lesser Egyptian Jerboa, Jaculus jaculus. Cold Spring Harbor Protocols (12): 1457-1461. Cooper KL, Hu J, ten Berge D, Fernandez-Teran M, Ros M, Tabin CJ (2011) Initiation of proximal-distal patterning of the vertebrate limb by signals and growth. Science. 332(6033): 1083-1086. \n",
            "-------------------- Scraping faculty url 33/183 --------------------\n",
            "nigel crawford  Biography Nigel Crawford received his Ph.D. from the Massachusetts Institute of Technology. He was an NSF Postdoctoral Fellow in the Department of Biochemistry at Stanford University.  Research Plants have evolved elaborate regulatory networks to respond rapidly to changes in the environment. Environmental and internal signals such as nitrogen and carbon metabolites, phytohormones, light, and circadian rhythms dramatically affect metabolic and developmental programs in plants. We are studying the metabolism of nitrate and are determining how it regulates plant metabolism and growth using microarray and mutant analyses in Arabidopsis. We have identified genes involved in nitrate uptake, reduction and regulation and have generated comprehensive datasets of nitrate-responsive genes. We have also studied nitric oxide metabolism and signaling and identified several genes that regulate nitric oxide accumulation and affect plant development and stress responses. Our work on nitrate uptake has focused on the functions of the AtNRT1.1 (CHL1) gene. AtNRT1.1 encodes a nitrate transporter whose expression is induced by nitrate, acid pH and carbon. For over twenty years, AtNRT1.1 was thought to function only in low affinity nitrate uptake in roots. Our work has shown that AtNRT1.1s function is much broader. First, AtNRT1.1 contributes to high affinity nitrate uptake in young plants and thus spans both high and low affinity systems. AtNRT1.1's contribution to each system depends on the environmental conditions in which the plants are grown and the age of the plants. Second, AtNRT1.1 is induced by auxin and is preferentially expressed in actively growing regions of the plant (root tips, young leaves and flowers) and contributes to their growth. AtNRT1.1 is also expressed and functions in guard cells. Compared to wildtype plants, Atnrt1.1 mutants have reduced stomatal opening and reduced transpiration rates in the light or when deprived of CO2 in the dark. At the cellular level, Atnrt1.1 mutants show reduced nitrate accumulation in guard cells during stomatal opening and fail to show a nitrate-induced depolarization of guard cells. In wildtype guard cells, nitrate induces depolarization, and nitrate concentrations increase three-fold during stomatal opening. These results identify the first anion transporter that functions in stomatal opening and suggest that nitrate plays an important role in stomatal movements and plant transpiration. In addition to serving as a nutrient and as an osmolyte, nitrate also acts as a signal. Nitrate reprograms nitrogen and carbon metabolism and influences root and shoot growth. To understand how plants perceive and respond to nitrate, we performed transcriptome analyses in Arabidopsis. Using ATH1 microarrays containing over 22,000 probe sets, we have uncovered over a thousand genes that are rapidly induced or repressed by nitrate treatment. In addition to known nitrate-responsive genes (e.g. those encoding nitrate transporters, nitrate reductase, nitrite reductase, ferredoxin reductase and enzymes in the pentose phosphate pathway), genes involved in glycolysis, trehalose-6-P biosynthesis, iron transport/metabolism, and sulfate uptake/reduction are also induced by nitrate. Because nitrate can be metabolized to nitrite, ammonium and amino acids, it is possible that downstream metabolites of nitrate could be the signals that induce some of these responses. To identify genes that respond specifically to nitrate, we constructed a nitrate reductase-null mutant in Arabidopsis. Microarray analysis revealed that almost 600 genes respond to nitrate (5 mM nitrate for 2 h) in both WT and mutant plants. This group of genes is over-represented most significantly in the functional categories of energy, metabolism, and glycolysis and gluconeogenesis. Because the nitrate response of these genes was nitrate reductase-independent, nitrate and not a downstream metabolite served as the signal for these genes. A different suite of genes was identified that require nitrate reduction for their response, implicating a downstream metabolite as the signal. We have shown that nitrite, the product of nitrate reduction, can act as a signal that is more potent than nitrate itself. To elucidate the genes and mechanisms that mediate these responses, we have developed several tools to facilitate our studies of nitrate signaling. We have developed a transient expression system to assay nitrate induction of gene expression and used it to identify a powerful cis-regulatory module that contains three nitrate enhancer elements (Plant Phys 154: 423-432). We used this regulatory module to drive GFP/YFP and GUS expression in transgenic plants to identify mutants defective in nitrate signaling. The first two genes identified encoded a transcription factor (AtNLP7) and the nitrate transporter AtNRT1.1 (Plant Phys. 151: 472-478). Atnrt1.1 mutants are defective in nitrate signaling under conditions where nitrate uptake is replete; thus, AtNRT1.1 serves as a nitrate regulator as well as transporter. Independently, the lab of Dr. Yi-Fang Tsay has shown that AtNRT1.1 transport function can be separated from its signaling function indicating that AtNRT1.1 acts as a nitrate transceptor (Cell 138: 1184-1194). In collaboration with Dr. Yong Wangs lab at Shandong Agricultural University, we identified a third gene encoding a novel, nuclear-localized protein (NRG2) that functions upstream of NRT1.1 to regulate nitrate-responsive genes (Plant Cell 28: 485-504). Lastly, we used the nitrate enhancer DNA in a one-hybrid screen of transcription factors. We identified a member of a large, plant-specific gene family (TCP20) that plays a role in nitrate foraging in plant roots, specifically in the systemic response system. These tools are making it possible to identify and study numerous genes that function in nitrogen signaling.  Select Publications Wang, R., X. Xing, Y. Wang, A. Tran, and N.M. Crawford (2009) A genetic screen for nitrate-regulatory mutants captures the nitrate transporter gene NRT1.1. Plant Physiol., 151: 472-478. PMID: 19633234 Krouk, G., N.M. Crawford, G.M. Coruzzi, Y.-F. Tsay (2010) Nitrate signaling: adaptation to fluctuating environments. Current Opinion Plant Biology, 13: 265-272. PMID: 20093067 Wang, Y., A. Ries, K. Wu, A. Yang, N.M. Crawford (2010) The Arabidopsis prohibitin gene PHB3 functions in nitric oxide-mediated responses and in hydrogen peroxide-induced nitric oxide accumulation. Plant Cell, 22: 249-259. PMID: 20068191 Wang, R., P. Guan, M. Chen, X. Xing, Y. Zhang, N.M. Crawford (2010) Multiple regulatory elements in the Arabidopsis NIA1 promoter act synergistically to form a nitrate enhancer. Plant Phys. 154: 423-432. PMID: 20668061 Krouk, G., S. Ruffel, R.A. Gutirrez, A. Gojon, N.M. Crawford, G.M. Coruzzi, B. Lacombe (2011) A framework integrating plant growth with hormones and nutrients. Trends Plant Sci., 16: 178-182. PMID: 21393048 Guan, P., R. Wang, P. Nacry, G. Breton, S.A. Kay, J.L. Pruneda-Paz, A. Davani, N.M. Crawford (2014) Nitrate foraging by Arabidopsis roots is mediated by the transcription factor TCP20 through the systemic signaling pathway. Proc. Natl. Acad. Sci., 111: 15267- 15272. PMID: 25288754 Medici, A., A. Marshall-Colon, E. Ronzier, W. Szponarski, R. Wang, A. Gojon, N.M. Crawford, S. Ruffel, G.M. Coruzzi, G. Krouk (2015) AtNIGT1/HRS1 integrates nitrate and phosphate signals at the Arabidopsis root tip. Nature Communications, 6: 6274, doi:10.1038/ncomms7274. PMID: 25723764 Xu, N., R. Wang, L. Zhao, C. Zhang, Z. Li, Z. Lei, F. Liu, P. Guan, Z. Chu, N.M. Crawford, Y. Wang (2016) The Arabidopsis NRG2 protein mediates nitrate signaling and interacts with and regulates key nitrate regulators. The Plant Cell, 28: 485-504. PMID: 26744214 \n",
            "-------------------- Scraping faculty url 34/183 --------------------\n",
            "matthew daugherty  Biography Matt Daugherty received his undergraduate degree from Wesleyan University and then spent three years working for a start-up biotech company in Chicago. He received his Ph.D. from UC San Francisco and did his postdoctoral training at the Fred Hutchinson Cancer Research Center in Seattle WA. He joined the Division of Biological Sciences at UCSD in 2016.  Research \"It takes all the running you can do, to keep in the same place.\" - The Red Queen to Alice, in Lewis Carrolls Through the Looking Glass Pathogens and their hosts are locked in a constant struggle for evolutionary dominance. Any host adaptation that leads to more effective defense against a pathogen will almost inevitably drive adaptation in the pathogen to escape such defenses. Thus, the only way either side can survive is to continually adapt, as proposed in Leigh Van Valens Red Queen hypothesis. These escalating arms races play out at the level of rapid evolution of host-pathogen interactions, constantly selecting for molecular innovations that either protect the host or evade host defenses. However, the molecular details of these rapidly evolving host-pathogen interactions, and the consequences of these conflicts on human susceptibility to current and emerging infectious diseases, are mostly unexplored. Our research aims to use these evolutionary principles, combined with virology and mechanistic biochemistry, to understand how our immune system has evolved to defend against pathogens and how pathogens counter-evolve to defeat host immunity. One of the approaches we take is to use comparative genomics to identify characteristic signatures of pathogen-driven innovation (e.g. rapid sequence evolution, horizontal gene transfer, gene birth and death, and gene recombination). Interestingly, we expect these rapidly evolving genes to be at the center of host-pathogen conflicts, yet many of them are almost entirely uncharacterized. This provides an opportunity to use pathogens to discover novel biological functions that are involved in not only pathogen immunity but also more global cellular regulation. Current projects in the lab focus on functional characterization of two such poorly understood systems: post-transcriptional modification of RNA to molecularly distinguish between different RNAs (e.g. host RNA versus viral RNA), and post-translational modifications of proteins to regulate their function. The ultimate goal of our research is to understand the rapidly evolving molecular means by which hosts and pathogens battle each other, while at the same time providing insight into how genetic variation resulting from these conflicts contributes to differences in human susceptibility to infectious diseases.  Select Publications Moens, U., Calvignac-Spencer, S., Lauber, C., Ramqvist, T., Feltkamp, M.C.W., Daugherty, M.D., Verschoor, E.J. and B. Ehlers (2017). ICTV Virus Taxonomy Profile: Polyomaviridae. J General Virology. 98, 1159-60. Chavarria-Smith, J., Mitchell, P.S., Ho, A.M., Daugherty, M.D.* and R.E. Vance*. (* co-corresponding authors) (2016). Functional and Evolutionary Analyses Identify Proteolysis as a General Mechanism for NLRP1 Inflammasome Activation. PLoS Pathogens. 12, e1006052. Daugherty, M.D., Schaller, A.M., Geballe, A.P, and H.S. Malik. (2016). Evolution-guided functional analyses reveal diverse antiviral specificities encoded by IFIT1 genes in mammals. eLife. 5, e14228. Chou, S.*, Daugherty, M.D.*, Peterson, S.B., Biboy, J., Yang, Y., Jutras, B.L., Fritz-Laylin, L.K., Ferrin, M.A., Harding, B.N., Jacobs-Wagner, C., Yang, X.F., Vollmer, W., Malik, H.S. and J.D. Mougous. (*these authors contributed equally) 2015. Transferred interbacterial antagonism genes augment eukaryotic innate immune function. Nature. 518, 98-101. Daugherty, M.D., Young, J.M., Kerns, J.A., and H.S. Malik. 2014. Rapid evolution of PARP genes suggests a broad role for ADP-ribosylation in host-virus conflicts. PLoS Genetics. 10, e1004403. Carter, J.J.*, Daugherty, M.D.*, Qi, X., Bheda-Malge, A., Wipf, G.C., Robinson, K., Roman, A., Malik, H.S. and D.A. Galloway. (*these authors contributed equally) 2013. Identification of an overprinting gene in Merkel cell polyomavirus provides evolutionary insight into the birth of viral genes. Proceedings of the National Academy of Sciences. 110, 12744-9. Daugherty, M.D. and H.S. Malik. 2012. Rules of engagement: molecular insights from host-virus arms races. Annual Review of Genetics. 46, 677-700. Daugherty, M.D., Liu, B, and A.D. Frankel. 2010. Structural basis for cooperative RNA binding and export complex assembly by HIV Rev. Nature Structural and Molecular Biology. 17, 1337-42. Daugherty, M.D., DOrso, I., and A.D. Frankel. 2008. A solution to limited genomic capacity: Using adaptable binding surfaces to assemble the functional HIV Rev oligomer on RNA. Molecular Cell. 31, 824-834. \n",
            "-------------------- Scraping faculty url 35/183 --------------------\n",
            "michael david  Biography Michael David received his Ph.D in Pharmacology from the University of Vienna, Austria and did his postdoctoral research at the Center of Biologics Evaluation and Research in Bethesda, Maryland. He was an Erwin-Schroedinger and a Fogarty Fellow, and received Scholar Awards from the Sidney Kimmel Cancer Foundation and the National Multiple Sclerosis Society.  Research Interferons (IFNs) represent a group of cytokines that are best characterized for their ability to interfere with viral infection, but are also appreciated for their antiproliferative (antitumor) effects and their immunemodulatory properties. Consequently, interferons are used in clinical settings for the treatment of viral infections (e.g. Hepatitis C), malignancies (e.g. melanoma) and autoimmune disorders (e.g. Multiple Sclerosis). Our goal is to understand the mechanisms by which interferonsmediate these pleiotropic effects on a cellular and molecular level by investigating the following areas: 1) Characterization of the signal transduction mechanisms that permit IFNs to activate the transcription of defined cellular genes: Formation of DNA-binding complexes comprised of tyrosine phosphorylated STAT transcription factors involves the activation of members of the JAK family of tyrosine kinases. However, additional post-translational events ( e.g. modification through ubiquitin-like molecules, protein methylation, etc.) exert control over activation of STAT proteins, in addition, not all biological effects of IFNs are exclusively mediated by the JAK/STAT pathway. 2) Define the cellular and molecular events that govern the immunemodulatory properties of interferons: We have identified an important novel function of type I interferons and STAT1 in the positive and negative selection of T cells during thymic development, as well as T cell activation. Our findings are likely to be of great significance as a link between viral infections and the development of autoimmune diseases, and furthermore have direct implications on the efficacy of interferons in the treatment of immune disorders. 3) Elucidate the pathways by which the immune system recognizes bacterial or viral infection and initiates an innate immune response: The induction of type I interferons is one of the earliest innate immune responses to viral as well as bacterial pathogens. We had previously discovered that Toll-like receptors (TLRs) activate the transcription factor IRF3, which is essential for the production of interferons as well as other pro-inflammatory cytokines after pathogen recognition. We are interested in the signaling mechanisms that lead to IRF3 activation, and how invading pathogens (e.g. Hepatitis C virus, B. anthracis) can interfere with these pathways and consequently paralyze the hosts innate immune response?  Select Publications Tanabe Y, Nishibori T, Su L, Arduini RM, Baker DP, David M. (2005). Cutting edge: role of STAT1, STAT3, and STAT5 in IFN-alpha beta responses in T lymphocytes. J. Immunol. 174(2) :609-613. Nishibori T, Tanabe Y, Su L, David M. (2004). Impaired development of CD4+ CD25+ regulatory T cells in the absence of STAT1: increased susceptibility to autoimmune disease. J. Exp. Med. 199(1) :25-34. Mowen KA, Tang J, Zhu W, Schurter BT, Shuai K, Herschman HR, David M. (2001). Arginine methylation of STAT1 modulates IFNalpha/beta-induced transcription. Cell 104(5) :731-734. Navarro L, David M. (1999). p38-dependent activation of interferon regulatory factor 3 by lipopolysaccharide. J. Biol. Chem. 274(50) :35535-35538. Dang O, Navarro L, Anderson K, David M. (2004). Cutting edge: anthrax lethal toxin inhibits activation of IFN-regulatory factor 3 by lipopolysaccharide. J. Immunol. 172(2) :747-751. Chiang E, Dang O, Anderson K, Matsuzawa A, Ichijo H, David M. (2006). Cutting edge: apoptosis-regulating signal kinase 1 is required for reactive oxygen species-mediated activation of IFN regulatory factor 3 by lipopolysaccharide. J. Immunol. 176(10) :5720-5724. \n",
            "-------------------- Scraping faculty url 36/183 --------------------\n",
            "arshad desai  Biography Arshad Desai received his Ph.D. in cell biology from the University of California, San Francisco and conducted postdoctoral work at the European Molecular Biology Laboratory in Heidelberg and the Max Planck Institute for Cell Biology & Genetics in Dresden. He joined the UC San Diego faculty in late 2002 and has been the recipient of a Damon Runyon Scholar Award, the American Society for Cell Biology Early Career Award, and a Keith Porter Fellowship. He is a Senior Editor of the Journal of Cell Biology, has chaired review panels for the National Institutes of Health and the European Research Council, and is a lifetime fellow of the American Society for Cell Biology.  Research We are focused on understanding the pathways and molecular mechanisms that ensure accurate genome inheritance during cell division. Errors in genome inheritance are the cause of birth defects and are implicated in both cancer initiation and progression. In addition, the machinery involved in genome inheritance is a prominent target in cancer chemotherapy. Our interest in cell division stems from a long-standing fascination with the microtubule cytoskeleton, comprised of highly dynamic protein polymers that employ energy from assembly-coupled nucleotide hydrolysis to perform useful work in the cell. This interest led us to focus on the kinetochore, the macromolecular machine assembled on the centromere regions of chromosomes during cell division in order to harness the intrinsic dynamics of microtubule polymers for chromosome segregation. To study the pathways and mechanisms operating during chromosome segregation, we have developed the early C. elegans embryo as a powerful model for discovery and characterization of the underlying molecular machinery. Our early efforts led to the definition of the core microtubule binding protein network at the kinetochore and the identification of a conserved protein family involved in epigenetic propagation of centromeric chromatin. In more recent work, we have elucidated multiple mechanisms acting in concert with the core microtubule-binding machinery to ensure accurate chromosome segregation and have begun to untangle the complex coupling between mechanical and signaling events at the kinetochore that is critical for accurate segregation. In addition to our focus on the dynamic kinetochore-microtubule interface and the regulatory mechanisms operating to ensure its correct formation, we are also pursuing projects on the epigenetic basis of centromere identity, and on the checkpoint pathways coordinating chromosome segregation with cell cycle progression. Our findings are relevant to the genesis of cancer genome instability and are leading us to collaboratively explore new mechanism-targeted approaches focused on cell division for cancer therapy. Our use of C. elegans is also leading us to discover new conserved cell division-independent functions for ancient chromosome segregation machinery, e.g. in neurodevelopment.  Select Publications Lara-Gonzalez P, Moyle M, Budrewicz J, Mendoza J, Oegema K, Desai, A. The G2-to-M transition is ensured by a dual mechanism that protects cyclin B from degradation by Cdc20-activated APC/C. Developmental Cell ( in press ) Cheerambathur DK*, Prevo B*, Chow TL, Hattersley N, Wang S, Zhao Z, Kim T, Gerson-Gurwitz A, Oegema K, Green R, Desai A. 2019. The kinetochore-microtubule coupling machinery is repurposed to direct sensory nervous system morphogenesis. Developmental Cell 48(6):864-872.e7. doi: 10.1016/j.devcel.2019.02.002 Kim T*, Lara-Gonzalez P*, Prevo B, Meitinger F, Cheerambathur D, Oegema K, Desai A. Kinetochores accelerate or delay APC/C activation by directing Cdc20 to opposing fates. Genes & Development 2017 31(11):1089-1094. Santaguida S, Richardson A, Iyer DR, M'Saad O, Zasadil L, Knouse KA, Wong Y, Rhind N, Desai A, Amon A. Chromosome mis-segregation generates cell cycle-arrested cells with complex karyotypes that are eliminated by the immune system. 2017. Developmental Cell 41(6):638-651. Cheerambathur DK, Prevo B, Oegema K, Desai A. Dephosphorylation of the Ndc80 tail stabilizes kinetochore-microtubule attachments via the Ska complex. Developmental Cell 2017 41(4):424-437. ( # corresponding authors) Hattersley N, Cheerambathur D, Moyle M, Stefanutti M, Richardson A, Lee KY, Dumont J, Oegema K, Desai A. 2016. A nucleoporin docks protein phosphatase 1 to direct meiotic chromosome segregation and nuclear assembly. Developmental Cell 38(5):463-77. Gerson-Gurwitz A, Wang S, Sathe S, Green R, Yeo GW, Oegema K, Desai A. 2016. A small RNA-catalytic Argonaute pathway tunes germline transcript levels to ensure embryonic divisions. Cell 165(2):396-409. Folco HD, Campbell CS, May K, Espinoza C, Oegema K, Ren B, Hardwick K, Grewal S, Desai A. 2015. The N-terminal tail of CENP-A confers epigenetic stability to centromeres via the CENP-T branch of the CCAN in fission yeast. Current Biology 25(3):348-56. Cheerambathur D, Gassmann R, Cook B, Oegema K, Desai A. 2013. Crosstalk between microtubule attachment complexes ensures accurate chromosome segregation. Science 342(6163):1239-1242. Campbell C, Desai A. 2013. Tension sensing by Aurora B kinase is independent of survivin-based centromere localization. Nature 497(7447):118-21. Gassmann R*, Rechtsteiner A*, Yuen K*, Egelhofer T, Muroyama A, Barron F, Maddox P, Monen J, Ercan S, Oegema K, Lieb J, Strome S, Desai A. 2012. An inverse relationship to germline transcription defines centromeric chromatin in C. elegans . Nature 484(7395):534-7. Maddox P, Hyndman F, Monen J, Oegema K, Desai A. 2007. Functional genomics identifies a conserved Myb domain-containing protein family required for assembly of CENP-A chromatin. Journal of Cell Biology 176:757-63. Cheeseman I, Chappie J, Wilson-Kubalek E, Desai A. 2006. The conserved KMN network constitutes the core microtubule-binding site of the kinetochore. Cell 127:983-97. Desai A, Rybina S, Mller-Reichert T, Shevchenko A, Shevchenko A, Hyman A, Oegema K. 2003. KNL-1 directs assembly of the microtubule-binding interface of the kinetochore in C. elegans . Genes & Development 17:2421-35. \n",
            "-------------------- Scraping faculty url 37/183 --------------------\n",
            "russell doolittle  Biography Russell Doolittle received his Ph.D. from Harvard University. He was named a Guggenheim Fellow and was elected to the National Academy of Sciences and the American Academy of Arts and Sciences. Professor Doolittle is a co-recipient of the Paul Ehrlich Prize.  Research Our group is mainly concerned with the structure and evolution of proteins. In this regard, we have two quite distinct projects under way. One is very general and has to do with reconstructing the evolutionary histories of a wide variety of proteins. This is a computer-based study that draws upon published sequence data for its raw material. We employ a number of sequence searching and alignment programs, many of which we have written ourselves. Among the questions we are trying to answer are: When did prokaryotic organisms diverge from eukaryotes? How many rudimentary families of proteins are there? How are \"new proteins\" invented? All of these questions can be answered, given enough sequences to compare Our second major research interest is laboratory-based and deals with the invention and evolution of vertebrate blood plasma proteins, and especially the clotting proteins. In the past, we have cloned and sequenced a number of these proteins from the most primitive of vertebrates, the lamprey. Comparison with the corresponding mammalian proteins has afforded us some important clues as to how these proteins function. We also succeeded in identifying equivalent gene products in even more distantly related creatures, including protochordates and invertebrates. Of all of these, we have focused most on the fibrinogen molecule. Our goal here is to understand how fibrin formation (clotting) occurs and was invented. In this regard, after many years of trying we managed to crystallize a native fibrinogen molecule (chicken fibrinogen = 320 KDa) and solved its X-ray structure. We have also solved the structure of various fibrinogen fragments with bound ligands involved in fibrin formation, and also factor XIII-generated crosslinked versions. The crystallography project should shed light not only on how clotting works but also on where some of the components came from. Fibrinogen is a multidomained mosaic protein, a key part of which is found in numerous other animal proteins, including various cytotactins (e.g., tenascin and T-cell factors). Crystal structure of fragment D from human fibrinogen  Select Publications Doolittle RF Coagulation in vertebrates with a focus on evolution and inflammation. J. Innate Immunity 3:9-16 (2011). Doolittle RF The protochordate Ciona intestinalis has a protein like full-length vertebrate fibrinogen. J. Innate Immunity 3:9-16 (2011). Watson JW and Doolittle RF Peptide-derivatized albumins that inhibit fibrin polymerization. Biochemistry 50:9923-9927 (2011). Doolittle RF, McNamara K and Lin, K. Correlating structure and function during the evolution of fibrinogen-related domains. Protein Science 21:1808-1823 (2012). Doettke SHC, Doolittle, RF, Wang Y-L and Kadonaga JT. TRFII and the evolution of the bilateria. Genes and Development 28:2071-2076 (2014). Doolittle RF Clotting of mammalian fibrinogens by papain: a re-examination. Biochemistry 53:6687-6694 (2014). Doolittle RF Bioinformatic characterization of Genes and proteins involved in blood clotting in lampreys. J. Mol. Evol. 81:121-130 (2015). Doolittle RF Some important milestones in the field of blood clotting. J. Innate Immunity 58:23-29 (2016). Doolittle RF The conversion of fibrinogen to fibrin: a brief history of some key events. Matrix Biology epub ahead of print (2016). \n",
            "-------------------- Scraping faculty url 38/183 --------------------\n",
            "rachel dutton  Research Microbial communities play transformative roles in everything from Earth's geochemical cycles to the human body. However, the complexity and difficulty in manipulating microbial communities makes understanding the mechanisms involved in community formation incredibly challenging. One approach to understanding the basic principles of community formation is to use simplified model communities, in much the same way as model organisms such as E. coli have been used to elucidate basic principles of cellular metabolism, genetics, and biochemistry. Our lab has focused on the use of microbial communities from cheese as models due to their simplicity, culturability, and experimental tractability. These communities show reproducible and dynamic patterns of community formation which depend on widespread interactions between species. We are now developing genetic, cell biological, and chemical approaches to studying species interactions in this model microbial community. As with any model system, our goal is to gain insight into the workings of more complex systems.  Select Publications Wolfe BE and Dutton RJ. Fermented Foods as Experimentally Tractable Microbial Ecosystems. Cell. Mar 26;161(1):49-55. (2015) Wolfe BE, Button JE, Santarelli M, Dutton RJ. Cheese rind communities provide tractable systems for in situ and in vitro studies of microbial diversity. Cell . July 17;158,422-33. (2014) David LA, Maurice CF, Gootenberg DB, Button JE, Ling A, Biddinger SB, Dutton RJ, Turnbaugh PJ. Diet rapidly and reproducibly alters the human gut microbiome. Nature . Jan 23;505(7484):559-63 (2014). Button JE and Dutton RJ. Quick Guide to Cheese Microbes. Current Biology . Aug 7; 22(15):587-9. (2012) Wolfe BE, Dutton RJ. Towards an ecosystems approach to cheese microbiology. Cheese and Microbes. ASM Press and Microbiology Spectrum (2014) \n",
            "-------------------- Scraping faculty url 39/183 --------------------\n",
            "joseph ecker \n",
            "-------------------- Scraping faculty url 40/183 --------------------\n",
            "dannielle engle  Biography Dannielle Engle finished her Ph.D. in the laboratory of Dr. Geoffrey M. Wahl at the Salk Institute for Biological Studies in the Biological Sciences program of the University of California, San Diego. Engle completed her postdoctoral fellowship in the laboratory of Dr. David. A. Tuveson beginning at the Cambridge Research Institute and finishing at Cold Spring Harbor Laboratory as a Senior Fellow. Engle also organized scientific outreach programs as the Lustgarten Foundation Pancreatic Cancer Research Laboratory Ambassador and is a recipient of the Theodore T. Puck and NCI K99/R00 Pathway to Independence Awards. Engle returned to the Salk Institute in 2019 as an Assistant Professor.  Research Having lost close family members to pancreatic cancer, Dannielle Engle has a personal connection to the disease. Engle uses her personal and scientific passion as well as her expertise in modeling to facilitate progress in pancreatic cancer research by creating better representations of what actually happens in patients. The identification of biomarkers and therapeutic targets has been hindered by the limited access to pure sample populations and accurate models. When looking at a patient's blood samples, there can be hidden signals from the incipient tumor, but these are masked and diluted by the multitude of other signals from all over the body. In addition, when looking at the pancreatic tumor tissue itself, most of our time is spent looking at signals from non-cancer cell types in the scar-like tissue that takes up the majority of the tumor in human patients. Traditional approaches have used cell lines that do not accurately represent the early stages of pancreatic cancer. Engle tackles these challenges by creating organoid and genetically engineered mouse models to better recapitulate pancreatic disease. Using these systems, the Engle lab identifies new therapeutic targets in organoids and tests their efficacy in autochthonous, immune competent mouse models. In addition, the need for more effective treatment options goes hand-in-hand with the ability to diagnose pancreatic cancer before it spreads. Accordingly, the Engle lab also focuses on the identification of biomarkers that unambiguously differentiate between pancreatic cancer and other inflammatory conditions.  Select Publications Wang, Y. V., Leblanc, M., Fox, N., Mao, J. H., Tinkum, K. L., Krummel, K., . . . Wahl, G. M. (2011). Fine-tuning p53 activity through C-terminal modification significantly contributes to HSC homeostasis and mouse radiosensitivity. Genes & development, 25(13), 1426-1438. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/21724834 . doi:10.1101/gad.2024411 Spike, B. T., Engle, D. D., Lin, J. C., Cheung, S. K., La, J., & Wahl, G. M. (2012). A mammary stem cell population identified and characterized in late embryogenesis reveals similarities to human breast cancer. Cell stem cell, 10(2), 183-197. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/22305568 . doi:10.1016/j.stem.2011.12.018 Sherman, M. H., Yu, R. T., Engle, D. D., Ding, N., Atkins, A. R., Tiriac, H., . . . Evans, R. M. (2014). Vitamin D receptor-mediated stromal reprogramming suppresses pancreatitis and enhances pancreatic cancer therapy. Cell, 159(1), 80-93. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/25259922 . doi:10.1016/j.cell.2014.08.007 Boj, S. F., Hwang, C. I., Baker, L. A., Chio, II, Engle, D. D., Corbo, V., . . . Tuveson, D. A. (2015). Organoid models of human and mouse ductal pancreatic cancer. Cell, 160(1-2), 324-338. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/25557080 . doi:10.1016/j.cell.2014.12.021 Naguib, A., Bencze, G., Engle, Dannielle D., Chio, Iok I. C., Herzka, T., Watrud, K., . . . Trotman, Lloyd C. (2015). P53 Mutations Change Phosphatidylinositol Acyl Chain Composition. Cell reports, 10(1), 8-19. Retrieved from http://www.sciencedirect.com/science/article/pii/S2211124714010213 . doi:http://dx.doi.org/10.1016/j.celrep.2014.12.010 Roy, I., McAllister, D. M., Gorse, E., Dixon, K., Piper, C. T., Zimmerman, N. P., . . . Dwinell, M. B. (2015). Pancreatic Cancer Cell Migration and Metastasis Is Regulated by Chemokine-Biased Agonism and Bioenergetic Signaling. Cancer Res, 75(17), 3529-3542. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/26330165 . doi:10.1158/0008-5472.CAN-14-2645 Zambirinis, C. P., Levie, E., Nguy, S., Avanzi, A., Barilla, R., Xu, Y., . . . Miller, G. (2015). TLR9 ligation in pancreatic stellate cells promotes tumorigenesis. The Journal of experimental medicine, 212(12), 2077-2094. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/26481685 . doi:10.1084/jem.20142162 Ireland, L., Santos, A., Ahmed, M. S., Rainer, C., Nielsen, S. R., Quaranta, V., . . . Mielgo, A. (2016). Chemoresistance in Pancreatic Cancer Is Driven by Stroma-Derived Insulin-Like Growth Factors. Cancer Res, 76(23), 6851-6863. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/27742686 . doi:10.1158/0008-5472.CAN-16-1201 Nielsen, S. R., Quaranta, V., Linford, A., Emeagi, P., Rainer, C., Santos, A., . . . Schmid, M. C. (2016). Macrophage-secreted granulin supports pancreatic cancer metastasis by inducing liver fibrosis. Nature cell biology, 18(5), 549-560. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/27088855 . doi:10.1038/ncb3340 Seifert, L., Werba, G., Tiwari, S., Giao Ly, N. N., Alothman, S., Alqunaibit, D., . . . Miller, G. (2016). The necrosome promotes pancreatic oncogenesis via CXCL1 and Mincle-induced immune suppression. Nature, 532(7598), 245-249. Retrieved from http://dx.doi.org/10.1038/nature17403 . doi:10.1038/nature17403 hlund, D., Handly-Santana, A., Biffi, G., Elyada, E., Almeida, A. S., Ponz-Sarvise, M., . . . Tuveson, D. A. (2017). Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer. The Journal of experimental medicine. Retrieved from http://jem.rupress.org/content/jem/early/2017/02/23/jem.20162024.full.pdf . doi:10.1084/jem.20162024 Quaranta, V., Rainer, C., Nielsen, S. R., Raymant, M. L., Ahmed, M. S., Engle, D. D., . . . Schmid, M. C. (2018). Macrophage-Derived Granulin Drives Resistance to Immune Checkpoint Inhibition in Metastatic Pancreatic Cancer. Cancer Res, 78(15), 4253-4269. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/29789416 . doi:10.1158/0008-5472.CAN-17-3876 Tiriac, H., Belleau, P., Engle, D. D., Plenker, D., Deschnes, A., Somerville, T., . . . Tuveson, D. A. (2018). Organoid profiling identifies common responders to chemotherapy in pancreatic cancer. Cancer discovery. Retrieved from http://cancerdiscovery.aacrjournals.org/content/candisc/early/2018/05/30/2159-8290.CD-18-0349.full.pdf . doi:10.1158/2159-8290.Cd-18-0349 Wolff, R. A., Wang-Gillam, A., Alvarez, H., Tiriac, H., Engle, D., Hou, S., . . . Levine, A. J. (2018). Dynamic changes during the treatment of pancreatic cancer. Oncotarget, 9(19), 14764-14790. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/29599906 . doi:10.18632/oncotarget.24483 Engle, D. D., Tiriac, H., Rivera, K. D., Pommier, A., Whalen, S., Oni, T. E., . . . Tuveson, D. A. (2019). The glycan CA19-9 promotes pancreatitis and pancreatic cancer in mice. Science, 364(6446), 1156-1162. Retrieved from https://www.ncbi.nlm.nih.gov/pubmed/31221853 . doi:10.1126/science.aaw3145 \n",
            "-------------------- Scraping faculty url 41/183 --------------------\n",
            "mark estelle  Research The plant hormone auxin has been implicated in virtually every stage of plant growth and development from embryogenesis to senescence. My laboratory is using the genetically tractable plant Arabidopsis thaliana to identify and characterize auxin response pathways. Our work has shown that auxin promotes the rapid degradation of a family of transcriptional repressors called the Aux/IAA proteins via the ubiquitin proteasome pathway. Auxin interacts directly with a ubiquitin protein ligase called SCFTIR1, and promotes an interaction between the E3 and the Aux/IAAs. We are now studying the mechanism SCFTIR1 action and the role of the complex in various aspects of plant growth. In addition, we are turning our attention to the complex transcriptional networks that mediate auxin growth responses. Our ultimate goal is to understand the systems that mediate auxin-dependent development at the level of the cell and organism.  Select Publications Shani, E., Salehin, M., Zhang, Y., Sanchez, S.E., Doherty, C., Wang, R., Castillejo-Mangado, C., Song, L., Tal, I., Pisanty, O., Ecker, J.R., Kay, S.E., Pruneda-Paz, J., and Estelle, M. 2016. Plant stress tolerance requires auxin-sensitive Aux/IAA transcriptional repressors. Jan 18. pii: S0960-9822(16)31463-4. doi: 10.1016/j.cub.2016.12.016. Lavy, M., Prigge, M.J., Tao, S., Shain, S. Kuo, A., and Estelle M. 2016. Constitutive auxin response reveals complex interactions between Aux/IAA and ARF proteins in Physcomitrella. eLIFE, Jun 1;5. pii: e13325. doi: 10.7554/eLife.13325. Prigge,M.J., Greenham, K., Zhang, Y., Santner, A., Castillejo, C., Mutka, A.M., OMalley, R.C., Ecker, J.R., Kunkel, B.N., Estelle, M. 2016. The Arabidopsis Auxin Receptor F-box proteins AFB4 and AFB5 are Required for Response to the Synthetic Auxin, Picloram. G3 In Press. PMID: 26976444 Tal, I., Zhang, Y., Jrgensen, M.E., Pisanty, O., Barbosa, I.C.R., Zourelidou, M., Regnault, T., Crocoll, C., Olsen, C.E., Weinstain, R., Schwechheimer, C., Halkier, B.A., Nour-Eldin, H.H., Estelle, M2. and Shani, E2. 2016 The Arabidopsis NPF3 protein is a GA transporter. Nature Communications, DOI: 10.1038/ncomms11486. Wang, R., Zhang, Y., Kieffer, M., Yu, H., Kepinski, S., and Estelle, M. 2016 HSP90 regulates temperature-dependent seedling growth in Arabidopsis by stabilizing the auxin co-receptor F-box protein TIR1. Nature Communications, DOI: 10.1038/ncomms10269. Gao, Y1., Zhang, Y., Zhang, D., Dai, X., Estelle, M2., and Zhao, Y2. 2015. Auxin Binding Protein 1 (ABP1) is not required for either auxin signaling or Arabidopsis development. Proc. Nat. Acad. Sci. 112(7):2275-80. doi: 10.1073/pnas.1500365112. Yu, H., Zhang, Y., Moss, B.L., Bargmann, B.O.R., Wang, R., Prigge, M., Nemhauser, J.L., Estelle, M. 2015. Untethering the TIR1 auxin receptor from the SCF complex increases its stability and inhibits auxin response. Nature Plants. doi:10.1038/nplants.2014.30 Yu, H., Moss, B., Jang, Michael Prigge, S.S., Klavins, E., Nemhauser, J., and Mark Estelle. 2013. Mutations in the TIR1 auxin receptor that increase affinity for Aux/IAA proteins result in auxin hypersensitivity, Plant Phys. 162(1):295-303 Yu H, Karampelias M, Robert S, Peer WA, Swarup R, Ye S, Ge L, Cohen J, Murphy A, Friml J, and Mark Estelle. 2013. ROOT ULTRAVIOLET B-SENSITIVE1/weak auxin response3 is essential for polar auxin transport in Arabidopsis. Plant Physiol.162:965-76 Shani, E., Weinstain, R., Zhang, Y., Castillejo Mangado, C., Kaiserli, E., Chory, J., Tsien, R.Y., and M. Estelle. 2013. Gibberellins accumulate in the elongating endodermal cells of Arabidopsis roots. Proc., Natl. Acad. Sci. USA 110:4834-9 Chapman, E.C., Greenham, K., Castillejo, C., Sartor, R., Bialy, A., Sun, T-p & Mark Estelle, 2012. Hypocotyl Transcriptome Reveals Auxin Regulation of Growth-Promoting Genes Through GA-Dependent and -Independent Pathways, PLoS ONE, 7(5):e36210. CaldernVillalobos, L.I.A., Lee, S., Brandt, W., Armitage, L., Sheard, L. B., Tan, X., Parry, G., Mao, H., Zheng, N., Napier, R., Kepinski, S., and M. Estelle. 2012. TIR1/AFBs and Aux/IAAs constitute a combinatorial co-receptor system to perceive auxin with differential sensitivities. Nat. Chem. Biol. 8:477-485 \n",
            "-------------------- Scraping faculty url 42/183 --------------------\n",
            "ronald evans \n",
            "-------------------- Scraping faculty url 43/183 --------------------\n",
            "emma farley \n",
            "-------------------- Scraping faculty url 44/183 --------------------\n",
            "gen-sheng feng  Biography Dr. Feng received his PhD degree at Indiana University Bloomington and did a postdoctoral fellowship at the University of Toronto.  Research Molecular Signaling in Cancer, Metabolism and Stem Cells At the heart of cancer research is the elucidation of oncogenes and tumor suppressor genes (or anti-oncogenes), which has been a driving force in the cancer field for several decades. Most recent studies in this and other labs have revealed dual roles of genes in promoting and suppressing hepatocellular carcinoma (HCC). Deciphering the paradoxical effects of these genes may provide a fresh view on the fundamental mechanisms of carcinogenesis, which can lead to development of mechanism-based therapeutics for HCC. Our current focuses are on delineating genetic and epigenetic interactions and cell-cell communications that drive hepatocarcinogenesis. We are very interested in isolating and characterizing cancer stem cells, CSCs (or tumor-initiating cells, TICs) in the liver, and in dissecting the dynamic interplay between tumors and the hepatic microenvironment. Efforts are also being devoted to elucidation of metabolic changes in tumor cells and to search of biomarkers for early diagnosis of liver cancers. We are also interested in deciphering the common and distinct biochemical pathways in self-renewal of hematopoietic stem cells (HSCs) and leukemia stem cells (LSCs), with the objective of understanding how an HSC pool is maintained throughout life, and with the ultimate goal of identifying novel targets for elimination of LSCs. Another focus is on dissecting the signaling events involved in metabolic disorders such as obesity and diabetes. Ongoing experiments are aimed at understanding the intertwining of leptin and estrogen signaling cascades, to illustrate why estrogen has a leptin-like effect in control of metabolism, why postmenopausal women have the tendency to develop obesity, and why obesity promotes breast cancer. Whereas obesity has become a serious health problem in developed countries worldwide, one ongoing project in the lab is to investigate an indispensable physiological role of adipogenesis and adipose tissue in mammals, as revealed by the most recent data obtained in this lab.  Select Publications Zhu HH, X Luo, K Zhang, J Cui, H Zhao, Z Ji, Z Zhou, J Yao, L Zeng, K Ji, WQ Gao, ZY Zhang, and GS Feng (2015). Shp2 and Pten have antagonistic roles in myeloproliferation but cooperate to promote erythropoiesis in mammals. Proc Natl Acad Sci USA 112, 13342-7. Deng Z, S Ma, H Zhou, Aiping Zang, Y Fang, T Li, H Shi, M Liu, M Du, PR Taylor, HH Zhu, J Chen, G Meng, F Li, C Chen, Y Zhang, X Zhang, E Pearlman, X Li, GS Feng and H Xiao (2015). Tyrosine phosphatase SHP2 mediates Dectin-induced SYK activation and anti-fungal TH17 response. Nature Immunol. 16, 642-52. Li S, DDF Hsu, B Li, X Luo, N Alderson, L Qiao, L Ma, HH Zhu, Z He, K Suino-Powell, K Ji, J Li, J Shao, HE Xu, T Li, and GS Feng (2014). Cytoplasmic tyrosine phosphatase Shp2 coordinates hepatic regulation of bile acid and FGF15/19 signaling to repress bile acid synthesis. Cell Metabolism 20, 320-332 (recommended by Faculty 1000, Previewed, Cell Metabolism 20, 203-204. He Z, HH Zhu, TJ Bauler, J Wang, T Ciaraldi, N Alderson, S Li, MA Raquil, K Ji, S Wang, J Shao, R Henry, PD King, and GS Feng (2013). Non-receptor tyrosine phosphatase Shp2 promotes adipogenesis through inhibition of p38 MAP kinase. Proc Natl Acad Sci USA. 110, E79-88. He Z, SS Zhang, Q Meng, S Li, HH Zhu, MA Raquil, N Alderson, H Zhang, J Wu, L Rui, D Cai, and GS Feng (2012). Shp2 controls female body weight and energy balance by integrating leptin and estrogen signals. Mol Cell Biol 32, 1867-1878, 2012. PMCID 3347413 Feng GS (2012).Conflicting roles of molecules in hepatocarcinogenesis: paradigm or paradox (Invited Review). Cancer Cell 21:150-154. He Z, SS Zhang, Q Meng, S Li, HH Zhu, MA Raquil, N Alderson, H Zhang, J Wu, L Rui, D Cai, and GS Feng (2012). Shp2 controls female body weight and energy balance by integrating leptin and estrogen signals. Mol Cell Biol 32:1867-1878. Qian, YW,Y Chen, W Yang, J Fu, J Cao, JJ Zhu, B Su, T Luo, YB Ren, RY Dai, JJ Li, XF Zhao, MC Wu, GS Feng, and HY Wang (2012). p28(GANK) prevents degradation of Oct4 and promotes expansion of tumor-initiating cells in hepatocarcinogenesis. Gastroenterology 142:1547-1558. Huang J, Yan J, Zhang J, Zhu S, Wang Y, Shi T, Zhu C, Chen C, Liu X, Cheng J, Mustelin T, Feng GS, Chen G, and Yu J (2012). SUMO1 modification of PTEN regulates tumorigenesis by controlling its association with the plasma membrane. Nature Commun 3:911. Huang H, Woo AJ, Waldon Z, Schindler Y, Moran TB, Zhu HH, Feng GS, Steen H, and Cantor AB (2012). A Src family kinase-Shp2 axis controls RUNX1 activity in megakaryocyte and T-lymphocyte differentiation. Genes Dev 26:1587-1601. Zhu, HH, K Ji, N Alderson, Z He, S Li, W Liu, DE Zhang, L Li, and GS Feng (2011). A Kit-Shp2-Kit signaling axis regulates a functional hematopoietic stem and progenitor cell pool. Blood 117:5350-5361. Bard-Chapeau EA, S Li, J Ding, SS Zhang, HH Zhu, F Princen, DD Fang, T Han, B Bailly-Maitre, V Poli, H Wang and GS Feng (2011). PTPN11/Shp2 acts as a tumor suppressor in hepatocellular carcinogenesis. Cancer Cell 19:629-639. Zhang SS, E Hao, J Yu, W Liu, J Wang, F Levine, and GS Feng (2009). Coordinated regulation by Shp2 tyrosine phosphatase of signaling events controlling insulin biosynthesis in pancreatic beta-cells. Proc Natl Acad Sci USA 106:7531-7536. PMCID 2678606 Ke K, EE Zhang, K Hagihara, D Wu, YH Pang, R Klein, T Curran, B Ranscht, and GS Feng (2007). Deletion of Shp2 in the brain leads to defective proliferation and differentiation in neural stem cells and early postnatal lethality. Mol Cell Biol 27:6706-6717. Ke Y, D Wu, F Princen, T Nguyen, Y Pang, J Lesperance, WJ Muller, RG Oshima, and GS Feng (2007). Role of Gab2 in mammary tumorigenesis and metastasis. Oncogene 26:4951-4960. Bard-Chapeau, EA, J Yuan, N Droin, S Long, EE Zhang, TV Nguyen, and GS Feng (2006). Concerted functions of Gab1 and Shp2 in liver regeneration and hepatoprotection. Mol Cell Biol 26:4664-4674. Chapeau E, AL Hevener, S Long, EE Zhang, JM Olefsky and GS Feng (2005). Deletion of Gab1 in the liver leads to enhanced glucose tolerance and improved hepatic insulin action. Nature Medicine 11:567-571. Zhang EE, E Chapeau, K Hagihara and GS Feng (2004). Neuronal Shp2 tyrosine phosphatase controls energy balance and metabolism. Proc Natl Acad Sci USA 101:16064-16069. You M, LM Flick, DH Yu, and GS Feng (2001). Modulation of the nuclear factor-kappaB pathway by Shp-2 tyrosine phosphatase in interleukin (IL)-6 induction by IL-1 or tumor necrosis factor. J Exp Med 193:101-109. Feng GS, and T Pawson (1994). Phosphotyrosine phosphatases with SH2 domains: regulators of signal transduction (review article). Trends Genet 10:54-58. Feng GS, CC Hui, and T Pawson (1993). SH2-containing phosphotyrosine phosphatase as a target of protein-tyrosine kinases. Science 259:1607-1611. \n",
            "-------------------- Scraping faculty url 45/183 --------------------\n",
            "rick firtel  Biography Rick Firtel was an undergraduate in Biology and Chemistry and Rufus Choate Scholar at Dartmouth College (1966) and received his PhD at Caltech in James Bonners laboratory as an NSF fellow (1971). Rick did his postdoctoral work with Harvey Lodish at MIT as a Helen Hay Whitney Foundation Fellow and joined the faculty at UCSD in 1973. He has been the recipient of an American Cancer Society Faculty Research Award, an International Union Against Cancer Fellowship, a Japanese Society for the Promotion of Sciences Fellowship, and a John Simon Guggenheim Fellowship. He has served as Director of the Center for Molecular Genetics (1995-2004), Chair of the Section of Cell and Developmental Biology (1999-2006), and Associate Dean of Biological Sciences (2008-2017).  Research Chemotaxis of wild-type cells to cAMP emitted from a pipette. Our laboratory focuses on the signal transduction pathways and the mechanics that control directional cell movement, or chemotaxis, and morphogenesis of eukaryotic cells. Chemotaxis is a basic property of many eukaryotic cells and plays critical roles in diverse biological pathways including wound healing, migration of leukocytes to sites of inflammation and bacterial infection, morphogenesis, and metastasis of a variety of cancer cell types. Chemotaxis is mediated by small molecule ligands (chemoattractants and chemokines) that interact with cell surface receptors to control downstream effector pathways. Cells are able to sense and respond to even weak chemoattractant gradients and to amplify a weak chemical gradient into a steep intracellular gradient of signaling molecules. Cells then translate this into changes in the cytoskeleton that control the mechanical forces that drive cell movement. Co-activation of Ras and PI3K at the leading edge of chemotaxing cells using real-time reporters Over the past few years, great progress has been made in elucidating the molecular mechanisms controlling the ability of cells to sense and respond to chemical gradients. My laboratory and others have discovered that Ras lies at the top of the signal transduction cascade leading to chemotaxis. We demonstrated that Ras controls chemotaxis through the activation of phosphatidylinositol 3-kinase (PI3K) and Tor Complex 2, which are also essential for cell growth and survival in metazoan cells. Our goal is to reveal the molecular mechanisms regulating these and other signaling pathways that enable cells to chemotax. Because most of these components are highly conserved evolutionarily in cells ranging from Dictyostelium amoebae to human leukocytes, we use Dictyostelium as our primary experimental system, which allows us to employ a wide range of genetic as well as biochemical approaches to dissect these signaling pathways. These methods are combined with in vivo, real-time analysis of GFP fusions of signaling proteins in living cells to understand the spatio-temporal changes of these proteins in response to directional signals. These diverse approaches have allowed us elucidate the basic principles by which signal transduction pathways control directional sensing and chemotaxis that apply to human leukocytes as well as Dictyostelium cells. A chemotaxing Dictyostelium cell (gray) forms stationary traction adhesions with the substratum along its anterior-posterior axis that transmit force (indicated by heat map) and allow the cell to move. The colored cylinder shows the path of the migrating cell over time. A major unanswered question in cell motility is how the signaling pathways and the reorganizations of the cytoskeleton function together to control cell movement. In collaboration with the groups of Juan Lasheras and Juan Carlos del Alamo in the Department of Mechanical and Aerospace Engineering, we have developed quantitative approaches to examine the time evolution of the traction forces that mediate each stage of the cell motility cycle. We have made the novel finding that amoeboid cell movement is controlled by a complex interplay between lateral and normal (vertical) forces mediated by cortical tension controlled through the lateral cell cortex as well as the more classic F-actin/myosin-mediated axial forces. We have achieved this by integrating the biochemical and mechanical measurements of cell motility and obtained the quantitative information necessary to connect specific biochemical and signaling processes to the physical events regulating the mechanical forces controlling cell motility and, in do so, understand how cells move. In collaboration with Wouter-Jan Rappel, Alex Groisman, and Bill Loomis, we are attempting to mathematically model the signaling events controlling cell polarization.  Select Publications Kumagai, A., M. Pupillo, R. Gunderson, R. Miake-Lye, P.N. Devreotes and R.A. Firtel (1989). Regulation and function of G protein subunits in Dictyostelium . Cell 57:265-275. (PMID: 2539262) Firtel, R.A. and A.L. Chapman (1990). A role for cAMP-dependent protein kinase in early Dictyostelium development. Genes Devel. 4:18-28. Hjorth, A.L., C. Pears, J.G. Williams and R.A. Firtel (1990). A developmentally regulated trans-acting factor(s) recognizes dissimilar G/C-rich elements controlling a class of cAMP-inducible Dictyostelium genes. Genes Devel. 4:419-432. Haberstroh, L. and R.A. Firtel (1990). A spatial gradient of expression of a cAMP-regulated prespore cell-type-specific gene in Dictyostelium . Genes Devel. 4:596-612. Esch, R.K. and R.A. Firtel (1991). Cyclic AMP and cell sorting control the spatial expression of a developmentally essential cell-type-specific ras gene in Dictyostelium . Genes Devel. 5:9-21. Hadwiger, J.A. and R.A. Firtel (1992). Analysis of G4, a G protein subunit required for multicellular development in Dictyostelium . Genes Devel. 6:38-49. Howard, P., B. Sefton and R.A. Firtel (1992). Analysis of a spatially regulated phosphotyrosine phosphatase identifies tyrosine phosphorylation as a key regulatory pathway in Dictyostelium . Cell 71:637-647. Howard, P.K., B.M. Sefton and R.A. Firtel (1993). Tyrosine phosphorylation of actin in Dictyostelium associated with cell shape changes. Science 259:241-244. Schnitzler, G.R., W.H. Fischer and R.A. Firtel (1994). Cloning and characterization of the G-box binding factor, an essential component of the developmental switch between early and late development in Dictyostelium . Genes Devel. 8:502-514. Dynes, J.L., A.M. Clark, G. Shaulsky, A. Kuspa, W.F. Loomis and R.A. Firtel (1994). LagC is required for cell-cell interactions that are essential for cell-type differentiation in Dictyostelium . Genes Devel. 8:948-958. Schnitzler, G.R., C. Briscoe, J.M. Brown and R.A. Firtel (1995). Serpentine cAMP receptors may act though a G protein-independent pathway to induce postaggregative development in Dictyostelium . Cell 81:737-745. Gaskins, C., A.M. Clark, L. Aubry, J.E. Segall and R.A. Firtel (1996). The Dictyostelium MAP kinase ERK2 regulates multiple, independent developmental pathways. Genes Devel. 10:118-128. Ma, H., M. Gamper, C. Parent and R.A. Firtel (1997). The Dictyostelium MAP kinase kinase DdMEK1 regulates chemotaxis and is essential for chemoattractant-mediated activation of guanylyl cyclase. EMBO J. 16:4317-4332. Aubry, L and R.A. Firtel (1998). Spalten, a protein containing G-protein-like and PP2C domains, is essential for cell-type differentiation in Dictyostelium . Genes Devel. 12:1525-1538. Lee, S., C.A. Parent, R. Insall and R.A. Firtel (1999). A novel Ras interacting protein required for chemotaxis and cyclic adenosine monophosphate signal relay in Dictyostelium . Mol. Biol. Cell 10:2829-2845. (PMID: 10473630) Chung, C.Y. and R.A. Firtel (1999). PAKa, a putative PAK family member, is required for cytokinesis and the regulation of the cytoskeleton in Dictyostelium discoideum cells during chemotaxis. J. Cell Biol. 147:559-575. (PMID: 10545500) Meili, R., C. Ellsworth and R.A. Firtel (2000). A novel Akt/PKB-related kinase is essential for morphogenesis in Dictyostelium . Curr. Biol. 10:708-717. (PMID: 10873800) Moniakis, J., S. Funamoto, M. Fukuzawa, J. Meisenhelder, T. Araki, T. Abe, R. Meili, T. Hunter, J. Williams and R.A. Firtel (2001). An SH2-domain-containing kinase negatively regulates the phosphatidylinositol-3 kinase pathway. Genes Devel. 15:687-698. (PMID: 11274054) Chung, C.Y., G. Potikyan and R.A. Firtel (2001). Control of cell polarity and chemotaxis by Akt/PKB and PI3 kinase through the regulation of PAKa. Mol. Cell 7:937947. (PMID: 11389841) Funamoto, S., K. Milan, R. Meili and R.A. Firtel (2001). Role of phosphatidylinositol 3 kinase and a downstream pleckstrin homology domain-containing protein in controlling chemotaxis in Dictyostelium . J. Cell Biol. 153:795-809. (PMID: 11352940) Mohanty, S., S. Lee, N. Yadava, M.J. Dealy, R.S. Johnson and R.A. Firtel (2001). Regulated protein degradation controls PKA function and cell-type differentiation in Dictyostelium . Genes Devel. 15:1435-1448. (PMID: 11390363) Sobko, A., H. Ma, and R A. Firtel (2002). Regulated SUMOylation and ubiquitination of DdMEK1 is required for proper chemotaxis. Devel. Cell 2:745-756. (PMID: 12062087) Funamoto, S., R. Meili, S. Lee, L. Parry and R.A. Firtel (2002). Spatial and temporal regulation of 3-phosphoinositides by PI 3-kinase and PTEN mediates chemotaxis. Cell 109:611-623. (PMID: 12062104) Sasaki, A.T., C. Chun, K. Takeda and R.A. Firtel (2004). Localized Ras signaling at the leading edge regulates PI3K, cell polarity, and directional cell movement. J. Cell Biol. 167:505-518. (PMID: 15534002) Park, K.C., F. Rivero, R. Meili, S. Lee, F. Apone and R.A. Firtel (2004). Rac regulation of chemotaxis and morphogenesis in Dictyostelium . EMBO J. 23:4177-4189. (PMID: 15470506) Lee, S., F.I. Comer, A. Sasaki, I.X. McLeod, Y. Duong, K. Okamura, J.R. Yates III, C.A. Parent and R.A. Firtel (2005). TOR Complex 2 integrates cell movement during chemotaxis and signal relay in Dictyostelium . Mol. Biol. Cell 16:4572-4583. (PMID: 16079174) Jeon, T.J., D. Lee, S. Merlot, G. Weeks and R.A. Firtel (2007). Rap1 controls cell adhesion and cell motility through the regulation of myosin II. J. Cell Biol. 176:1021-1033. (PMID: 17371831) Sasaki, A., C. Janetopoulos, S. Lee, P.G. Charest, K. Takeda, L.W. Sundheimer, R. Meili, P.N. Devreotes and R.A. Firtel (2007). G protein-independent Ras/PI3K/F-actin circuit regulates basic cell motility. J. Cell Biol. 178:185-191. (PMID: 17635933) Del Alamo, J.C., R. Meili, B. Alonso-Latorre, J. Rodriguez-Rodriguez, A. Aliseda, R.A. Firtel and J.C. Lasheras (2007). Spatio-temporal analysis of eucaryotic cell motility by improved force cytometry. Proc. Natl. Acad. Sci. USA 104:13343-13348. (PMID: 17684097) Jeon, T.J., D. Lee, S. Lee, G. Weeks and R.A. Firtel (2007). Regulation of Rap1 activity by RapGAP1 controls cell adhesion at the front of chemotaxing cells. J. Cell Biol. 179:833-843. (PMID: 18039932) Zhang, S., P.G. Charest and R.A. Firtel (2008). Spatiotemporal regulation of Ras activity provides directional sensing. Curr. Biol. 18:1587-1593. (PMID: 18948008) Meili, R., B. Alonso-Latorre, J.C. del Alamo, R.A. Firtel and J.C. Lasheras (2010). Myosin II is essential for the spatiotemporal organization of traction forces during cell motility. Mol. Biol. Cell 21:405-417. (PMID: 19955212) Charest, P.G., Z. Shen, A. Lakoduc, A.T. Sasaki, S. Briggs and R.A. Firtel (2010). A Ras signaling complex controls the RasC-TORC2 pathway and directed cell migration. Devel. Cell 18:737-749. (PMID: 2049388) Lee, S., Z. Shen, D.N. Robinson, S. Briggs and R.A. Firtel (2010). Involvement of the cytoskeleton in controlling leading-edge function during chemotaxis. Mol. Biol. Cell 21:1810-1824. (PMID: 20375144) Bastounis, E., R. Meili, B. Alonso-Latorre, J.C. del Alamo, J.C. Lasheras and R.A. Firtel (2011). The Scar/WAVE complex is necessary for proper regulation of traction stresses during amoeboid motility. Mol. Biol. Cell 22:3995-4003. (PMID: 21900496) Takeda, K., D. Shao, M. Adler, P.G. Charest, W.F. Loomis, H. Levine, A. Groisman, W. Rappel and R.A. Firtel (2012). Incoherent feedforward control governs adaptation of activated Ras in a eukaryotic chemotaxis pathway. Science Signaling 5:ra2. (PMID: 22215733) Bastounis, E., R. Meili, B. Alvarez-Gonzalez, J. Francois, J.C. del Alamo, R.A. Firtel and J.C. Lasheras (2014). Both contractile axial and lateral traction force dynamics drive amoeboid cell motility. J. Cell Biol. 204:1045-1061. (PMID: 24637328) lvarez-Gonzlez B, R. Meili, E. Bastounis, R.A. Firtel, L.C. Lasheras, J.C. Del lamo. (2015). Three-dimensional balance of cortical tension and axial contractility enables fast amoeboid migration. Biophys J. 108:821-832. (PMID: 25692587) \n",
            "-------------------- Scraping faculty url 46/183 --------------------\n",
            "douglass forbes  Biography Douglass Forbes received her Ph.D. from the University of Oregon and was an American Cancer Society postdoctoral fellow at UCSF in the Dept. of Biochemistry and Biophysics. She was a PEW Fellow in the Biomedical Sciences, and an elected member of the Governing Council of the American Society of Cell Biology. Dr. Forbes served as the Vice Chair of Cell and Developmental Biology from 2000-2005 and 2007-2009. She has served on multiple editorial boards, both past and present.  Research Our lab has two primary areas of interest: (1) the structure, function and assembly of the vertebrate nucleus, and (2) the mechanism by which the cell regulates where and how to assemble major mitotic structures, such as the mitotic spindle, the nuclear membranes and nuclear pore complexes. The Nucleus: Structure, Function and Assembly A focus of the research in my laboratory is the eukaryotic nucleus: its structure and function. We work both in vivo and in vitro. For in vivo studies, we use mammalian cultured cell lines to examine nuclear transport or assembly using RNAi technology to knock down expression of genes of interest to determine their effect on function. For in vitro studies, we use a nuclear reconstitution system which efficiently assembles nuclei in the test tube from soluble and membrane components. This system makes use of an extract of Xenopus eggs, which naturally store abundant amounts of disassembled nuclear components in preparation for the rapid cleavages of early division. Upon addition of DNA or chromatin to the in vitro interphase extract, nuclei containing nuclear membranes, a nuclear lamina, and nuclear pores quickly assemble. The in vitro reconstituted nuclei are capable of nuclear import, DNA replication, pol III transcription, and even mitotic disassembly. In parallel, we use a mitotic egg extract to study spindle and kinetochore assembly. A long-standing interest of the laboratory has been the nuclear pore, a large macromolecular complex of 120 million daltons that spans the nuclear membranes. All communication between the nucleus and cytoplasm occurs through the nuclear pore. In vivo, the pore complex imports nuclear proteins and exports mRNA, tRNA, snRNAs, and ribosomal precursors. We study: (1) assembly of the nucleus itself, (2) assembly of nuclear pores, (3) the regulation of these processes, and (4) the mechanism of nuclear transport, including mRNA export, protein export, and protein import. Using antibodies we can immunodeplete a nuclear assembly extract of individual nuclear pore proteins, then reconstitute nuclei which lack that protein. Such \"designer\" nuclei can be tested for alterations in nuclear pore structure, pore function, or pore assembly. In this way we discovered the protein responsible for initiating pore assembly at the nucleus (Rasala et al, 2006, 2008). This protein, ELYS, acts by binding to chromatin at AT-rich sequences and recruiting key structural subunits of the nuclear pore to these sites to initiate nuclear pore assembly. With experiments of this type, we hope to molecularly dissect and elucidate the gates that \"guard the fortress\" of the genome. Regulation of the Major Mitotic Assembly Events: Cellular GPSs The second major interest of the laboratory involves understanding the regulation of assembly of the major mitotic structures. Specifically, we are focused on how assembly is regulated such that the mitotic spindle, and later the nuclear membranes and nuclear pores, form only around chromatin, i.e., in a spatially precise manner. We have shown that an abundant cellular protein, Importin beta, acts in mitosis to negatively regulate both nuclear membrane fusion and nuclear pore assembly everywhere except in the vicinity of the chromosomes (Harel et al, M.B.C, 2003). The small GTPase Ran, produced in its GTP-bound form only near chromatin, acts as the counteracting positive regulator, a GPS to tell where assembly should occur. We have extended this work to Transportin, a second karyopherin. We believe that Importin beta and Transportin act as global regulators of cellular events involving the genome, including those controlling nuclear import, spindle formation, correct nuclear membrane assembly, and nuclear pore assembly.  Select Publications Forbes, D.J., Travesa, A., Nord, M., and C. Bernis. (2015). Nuclear transport factors: Global regulation of mitosis. Current Opinion in Cell Biology 35:78-90. Schwartz, M., Travesa, A., Martell, S., and D.J. Forbes. (2015). Analysis of the initiation of nuclear pore assembly by ectopically targeting nucleoporins to chromatin. Nucleus 6:40-54 Bernis, C., Swift-Taylor, B., Nord, M., Carmona, S., Chook, Y.M., and Forbes, D.J. (2014). Transportin acts to regulate mitotic assembly events by target binding rather than Ran sequestration. Molecular Biology of the Cell 25: Jan 29. [Epub ahead of print]. Bernis, C. and Forbes, D.J. (2014). Analysis of nuclear reconstitution, nuclear envelope assembly, and nuclear pore assembly using Xenopus in vitro assays. In: Nuclear Pore Complex and Nucleocytoplasmic Transport, Methods in Cell Biology, Volume 122, Chapter 8, pp. 165-191. Ed. Valrie Doye, Elsevier Press (in press) Powers, M. and Forbes, D.J. (2013). Nucleocytoplasmic transport: Beginning to gel? Current Biology 22: R1006-1009. Fichtman, B., Ramos, C., Rasala, B., Harel, A., and Forbes, D.J. (2010). Inner/outer nuclear membrane fusion in nuclear pore assembly: Biochemical demonstration and molecular analysis. Molecular Biology of the Cell 21:4197-4211. Lachish-Zalait, A., Lau, C., Fichtman, B., Zimmerman, E., Harel, A., Gaylord, M., Forbes, D.J, and Elbaum, M. (2009). Transportin mediates nuclear entry of DNA in vertebrate systems. Traffic 10:1414-1428 Lau, C.K., Delmar, V.A., Chan, R.C., Phung, Q., Bernis, C., Fichtman, B., Rasala, B.A., Forbes. D.J. (2009). Transportin regulates major mitotic assembly events: From spindle to nuclear pore assembly. Molecular Biology of the Cell 20:4043-4058 Brown, C., Kennedy, C., Delmar, V., Forbes, D.J. and P. Silver. (2008). Global histone acetylation induces functional genomic reorganization at mammalian nuclear pores. Genes and Development 22:627-639. Chakraborty, P., Wang, Y., Wei, J-H. , van Deursen, J., Yu, H., Malureanu, L., Dasso, M., Forbes, D.J., Levy, D., Seemann, J. , and B. Fontoura. (2008). Nucleoporin levels regulate cell cycle progression and phase-specific gene expression. Developmental Cell 15:657-667. Rasala, B., Ramos, C., Harel, A., and Forbes, D.J. (2008). Capture of AT-rich chromatin by ELYS recruits POM121 and NDC1 to initiate nuclear pore assembly. Molecular Biology of the Cell 19:3982-3996 Orjalo, A., Arnaoutov, A. , Shen, Z., Boyarchuk, Y., Zeitlin, S., Fontoura, B., Briggs, S., Dasso, M., and Forbes, D.J. (2006). The Nup107-160 nucleoporin complex is required for correct bipolar spindle assembly. Molecular Biology of the Cell 17: 3806-3818. Rasala, B., Orjalo, A., Shen, Z., Briggs, S., and Forbes, D.J. (2006). ELYS is a dual nucleoporin/kinetochore protein required for nuclear pore assembly and proper cell division. Proc. Natl. Acad. Sci., U.S.A. 103:17801-17806. Harel, A. and Forbes, D.J. (2004). Importin beta: Conducting a much larger cellular symphony. Molecular Cell 16:319-330. Harel, A., Chan, R., Lachish-Zalait, A., Zimmerman, A., Elbaum, M., and Forbes, D.J. (2003). Importin beta negatively regulates nuclear membrane fusion and NPC assembly. Molecular Biology of the Cell 14:4387-4396. Harel, A., Orjalo, A., Vincent, T., Lachish-Zalait, A., Vasu, S., Shah, S., Zimmerman, A., Elbaum, M. and Forbes, D.J. (2003). Removal of a single pore subcomplex results in vertebrate nuclei devoid of nuclear pores. Molecular Cell 11:853-864. \n",
            "-------------------- Scraping faculty url 47/183 --------------------\n",
            "p. a. george fortes \n",
            "-------------------- Scraping faculty url 48/183 --------------------\n",
            "kathleen french \n",
            "-------------------- Scraping faculty url 49/183 --------------------\n",
            "fred gage \n",
            "-------------------- Scraping faculty url 50/183 --------------------\n",
            "e. peter geiduschek  Biography E. Peter Geiduschek received his Ph.D. from Harvard University. He was awarded a Guggenheim Fellowship and has served on the editorial boards of several scientific journals. Dr. Geiduschek is a member of the National Academy of Sciences and the American Academy of Arts and Sciences and is also a fellow of the American Association for the Advancement of Science and the American Academy of Microbiology.  Research The central theme of my research has been the enzymology of gene regulation, specifically of transcriptional regulation and its principal focus has been on fundamental mechanisms, approached from a molecular genetic and biochemical perspective. It is now widely appreciated that the transcriptional apparatus of all cells (archaeal, bacterial and eukaryotic) has common evolutionary roots. It follows that transcription-regulatory mechanisms should also share certain broadly similar mechanistic features. In that spirit, my laboratory worked in parallel with bacteria and their viruses, with eukaryotes, and with archaea.  Select Publications Kassavetis, G.A., Soragni, E., Driscoll, R. and Geiduschek, E.P. (2005). Reconfiguring the connectivity of a multiprotein complex: Fusions of yeast TBP with Brf1, and the function of TFIIIB. Proc. Natl. Acad. Sci. U.S.A. 102:15406-15411. Nechaev, S. and Geiduschek, E.P. (2008) Dissection of the bacteriophage T4 late promoter complex. J. Mol. Biol. 379:402-413. Geiduschek, E.P. (2009) Without a license, or Accidents waiting to happen. Annu. Rev. Biochem. 78:1-28. Geiduschek, E.P. (2010) An introduction to transcription and gene regulation. J. Biol. Chem. 285:25885-25892. Wilkinson, S.P., Ouhammouch, M. and Geiduschek, E.P. (2010) Transcriptional activation in the context of repression mediated by archaeal histones. Proc. Natl. Acad. Sci. U.S.A. 107:6777-6781. Geiduschek, E.P. and Kassavetis, G.A. (2010) Transcription of the T4 late genes. Virology J. 7:288. Twist K.-A.F., Campbell, E.A., Deighan, P., Nechaev, S., Jain, V.,Geiduschek, E.P., Hochschild, A., and Darst, S.A. (2011) Crystal structure of the bacteriophage T4 late-transcription co-activator in complex with the beta-subunit flap domain of E.coli RNA polymerase. Proc. Natl. Acad. Sci. U.S.A. 108: in press. \n",
            "-------------------- Scraping faculty url 51/183 --------------------\n",
            "tim gentner \n",
            "-------------------- Scraping faculty url 52/183 --------------------\n",
            "stacey glasgow  Biography Stacey Glasgow received her Ph.D. in Integrative Biology from UT Southwestern Medical Center with Dr. Jane E. Johnson. She carried out her postdoctoral training with Dr. Benjamin Deneen in the Center for Cell and Gene Therapy at Baylor College of Medicine. Stacey is the recipient of a K01 Mentored Research Scientist Development Award issued by the NIH/NCI.  Research Glia are the most abundant cell type in the adult central nervous system (CNS) and support a diverse array of physiological functions including myelination, synaptogenesis, neurotransmission, and blood-brain barrier maintenance. Consistent with this functional heterogeneity, disruptions in normal gliogenesis and glial maintenance have been reported as defining events in the onset and progression of several neurological disorders and malignancies. Despite their essential and emerging roles in CNS functions and pathologies, the molecular mechanisms that govern glial cell specification from neural progenitors and how these mechanisms are correlated with associated diseases are poorly defined. Specification and lineage commitment involves coordinated gene expression regulated by chromatin remodeling, transcriptional cascades, and binding site recognition. Understanding how these regulatory layers converge to influence gene expression remains a key question with implications in neural development, function, and associated disease. In my laboratory, we use interdisciplinary approaches to study both glial development and ultimately, neurodegenerative diseases and glioma. Specifically, we are interested in dissecting the mechanisms underlying transcriptional regulation and chromatin architecture of glial cell development and applying these developmental concepts to glioma tumorigenesis. Methodologically, we utilize manipulation of both chick and mouse model systems in combination with biochemical and transcriptional assays to help elucidate these molecular underpinnings.  Select Publications Laug D, Glasgow SM, Deneen B. (2018) A glial blueprint for gliomagenesis. Nature Reviews Neuroscience. Jul;19(7):393-403. Glasgow SM, Carlson J, Zhu W, Kang P, Chaboub LS, Lee HK, Clovis Y, Lozzi B, McEvilly RJ, Rosenfeld MG, Creighton C, Lee SK, Mohila C, Deneen B. (2017) Glial specific enhancers and chromatin structure regulate NFIA expression and glioma tumorigenesis. Nature Neuroscience 11: 1520-1528. Chaboub LS, Manalo JM, Lee HK, Glasgow SM, Chen F, Kawazaki Y, Akiyama T, Creighton CJ, Mohila CA and Deneen B. (2016) Temporal profiling of astrocyte precursors reveals parallel roles for asef during development and after injury. Journal of Neuroscience 36(47): 11904-11917. Thiebes K, Nam H, Cambronne X, Shen R, Glasgow SM, Cho H, Kwon J, Goodman R, Lee JW, Lee SH, Lee SK (2015) miR-218 is essential to establish motor neuron fate as a downstream effector of Isl1-Lhx3. Nature Communications 6. Glasgow SM, Zhu W, Stolt CC, Huang TW, Chen F, LoTurco JJ, Neul JL, Wegner M, Mohila C, Deneen B. (2014) Mutual antagonism between Sox10 and NFIA regulates diversification of glial lineages and glioma subtypes. Nature Neuroscience 17: 1322-1329. Glasgow SM, Laug D, Brawley V, Zhang Z, Corder A, Yin Z , Wong STC, Li XN, Foster AE, Ahmed A, and Deneen B. (2013) The miR223-NFIA Axis Regulates Glial Precursor Proliferation and Tumorigenesis in the CNS. Journal of Neuroscience 33(33): 13560-13568. **Featured in Science Editors Choice http://www.sciencemag.org/content/341/6150/1044.4.short Molofsky A*, Glasgow SM*, Chaboub L, Tsai H-H, Murnen AT, Fancy SF, Yuen T, Mahireddy L, Baranzini S, Deneen B, Rowitch DH, Oldham M. (2013) Expression profiling of putative astrocyte precursors in the developing spinal cord identifies Sox9-Nfe2l1 interactions. Glia 61(9):1518-32. Fancy SF*, Glasgow SM*, Finley M, Rowitch DH, and Deneen B (2012) Evidence that NFIA inhibits repair after white matter injury. Annals of Neurology 72(2):224-233. Kang P*, Lee HK*, Glasgow SM, Finley M, Donti T, Gaber ZB, Graham BH, Foster AE, Novitch BG, Gronostajski RM, and Deneen B (2012) Sox9 and NFIA coordinate a transcriptional regulatory cascade during the initiation of gliogenesis. Neuron 74(1):79-94. Henke RM, Savage TK, Meredith DM, Glasgow S.M., Hori K, Dumas J, MacDonald RJ, and Johnson JE. (2009) Neurog2 is a Direct Downstream Target of the Ptf1a-Rbpj Transcription Complex in Dorsal Neural Tube. Development 136, 2945-2954 Hori K, Cholewa-Waclaw J, Nakada Y, Glasgow SM, Masui T, Henke RM, Wildner H, Martarelli B, Beres TM, Epstein JA, Magnuson MA, Macdonald RJ, Birchmeier C, Johnson JE. A nonclassical bHLH Rbpj transcription factor complex is required for specification of GABAergic neurons independent of Notch signaling. Genes Dev. 2008 Jan 15; 22(2):166-78. Glasgow, SM, Henke, RM, MacDonald, RJ, Wright, C, Johnson, JE (2005) PTF1a Specifies Dorsal Horn GABAergic Over Glutamatergic Cell Fate. Development 132, 5461-5469 \n",
            "-------------------- Scraping faculty url 53/183 --------------------\n",
            "james w. golden  Biography Dr. James W. Golden received a B.S. (1977) in Microbiology from the University of Maryland-College Park and a Ph.D. (1983) in Biology from the University of Missouri-Columbia. After postdoctoral work as an NIH Fellow at The University of Chicago, he joined the Department of Biology at Texas A&M University in 1986. He was promoted to Associate Professor in 1990 and then to Professor in 1996. Dr. Golden moved to the University of California, San Diego in 2008.  Research Genetics, Molecular Biology, and Biotechnology of Cyanobacteria Current areas of research include the development of improved genetic tools for cyanobacterial genetic engineering and the use of these tools to answer basic-science questions and for biotechnology applications. Our biotechnology-related research involves genetic engineering of cyanobacteria to synthesize bioactive natural products and biofuels. Other projects include identifying cyanobacterial genes related to resistance to grazing by amoebal and ciliate predators, the study of genes required for biosynthesis of cyanobacterial toxins, and the identification of cyanobacterial genes that affect fitness for growth during spaceflight or on Mars. These research projects include work with different strains of cyanobacteria, but focus on the laboratory model strain Synechococcus elongatus, strain PCC 7942, which has excellent genetics and is widely used for synthetic biology studies. Past research has focused on the developmental biology of cyanobacterial heterocyst formation, with an emphasis on the genetic regulation of cellular differentiation and the cell-to-cell signaling mechanisms that control multicellular pattern formation. This research uses methods of genetics and molecular biology to understand basic principles of regulation and signaling pathways that control development in a simple prokaryotic multicellular organism, the filamentous cyanobacterium Anabaena (Nostoc), strain PCC 7120. Like all cyanobacteria, Anabaena uses light energy for photosynthesis. Anabaena is also capable of nitrogen fixation, a process that is incompatible with photosynthesis because the nitrogenase enzyme is destroyed by oxygen, a byproduct of photosynthesis. Anabaena solves this problem by spatially separating the two processes into different cell types: photosynthetic vegetative cells and nitrogen-fixing heterocysts. Anabaena grows as a very simple multicellular organism composed of filaments of vegetative cells containing about 10 percent heterocysts. Heterocysts differentiate from vegetative cells at semiregular intervals along the filament and supply fixed nitrogen to neighboring vegetative cells to support their growth. We identified a gene, patS, which encodes a small peptide that functions as a diffusible cell-to-cell signal that acts to control heterocyst pattern formation.  Select Publications Taton A, Ma AT, Ota M, Golden SS, Golden JW. 2017. NOT Gate Genetic Circuits to Control Gene Expression in Cyanobacteria. ACS Synth Biol doi:10.1021/acssynbio.7b00203. pmcid: https://www.ncbi.nlm.nih.gov/pubmed/28803467 Kim YS, Kim IS, Boyd JS, Taton A, Golden JW, Yoon HS. 2017. Enhanced biomass and oxidative stress tolerance of Synechococcus elongatus PCC 7942 overexpressing the DHAR gene from Brassica juncea. Biotechnol Lett 39(10), 1499-1507, doi:10.1007/s10529-017-2382-6. pmcid: https://www.ncbi.nlm.nih.gov/pubmed/28667417 Agarwal V, Blanton JM, Podell S, Taton A, Schorn MA, Busch J, Lin Z, Schmidt EW, Jensen PR, Paul VJ, Biggs JS, Golden JW, Allen EE, Moore BS. 2017. Metagenomic discovery of polybrominated diphenyl ether biosynthesis by marine sponges. Nat Chem Biol 13:537-543. pmcid: 5391271 https://www.ncbi.nlm.nih.gov/pubmed/28319100 Chen, Y., A. Taton, M. Go, R. E. London, L. M. Pieper, S. S. Golden, and J. W. Golden. 2016. Self-replicating shuttle vectors based on pANS, a small endogenous plasmid of the unicellular cyanobacterium Synechococcus elongatus PCC 7942. Microbiology 162(12):2029-2041. PMID: 27902432. https://www.ncbi.nlm.nih.gov/pubmed/27902432 Taton, A., F. Unglaub, N. E. Wright, W. Y. Zeng, J. Paz-Yepes, B. Brahamsha, B. Palenik, T. C. Peterson, F. Haerizadeh, S. S. Golden, and J. W. Golden. 2014. Broad-host-range vector system for synthetic biology and biotechnology in cyanobacteria. Nucleic Acids Res. 42:e136. PMID: 25074377, PMCID: PMC4176158. http://www.ncbi.nlm.nih.gov/pubmed/25074377 Ma, A. T., C. M. Schmidt, and J. W. Golden. 2014. Regulation of gene expression in diverse cyanobacterial species by using theophyllin -responsive riboswitches. Appl. Environ. Microbiol. 80:6704-6713. PMID: 25149516, PMCID: PMC4249034. http://www.ncbi.nlm.nih.gov/pubmed/25149516 Flaherty, B. L., D. Johnson, and J. W. Golden. 2014. Deep sequencing of HetR-bound DNA reveals novel HetR targets in Anabaena sp. strain PCC7120. BMC Microbiol 14:255. PMCID: 4192349. http://www.ncbi.nlm.nih.gov/pubmed/25278209 Taton, A., E. Lis, D. M. Adin, G. Dong, S. Cookson, S. A. Kay, S. S. Golden, and J. W. Golden. 2012. Gene transfer in Leptolyngbya sp. strain BL0902, a cyanobacterium suitable for production of biomass and bioproducts. PLoS ONE 7:e30901. PMCID: PMC3265524. http://www.ncbi.nlm.nih.gov/pubmed/22292073 Flaherty, B. L., F. Van Nieuwerburgh, S. R. Head, and J. W. Golden. 2011. Directional RNA deep sequencing sheds new light on the transcriptional response of Anabaena sp. strain PCC 7120 to combined-nitrogen deprivation. BMC Genomics, 12:332. Saha, S. K., and J. W. Golden. 2011. Overexpression of pknE blocks heterocyst development in Anabaena sp. strain PCC 7120. J. Bacteriol. http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=21421755 Mella-Herrera, R. A., M. R. Neunuebel, K. Kumar, S. K. Saha, and J. W. Golden. 2011. The sigE gene is required for normal expression of heterocyst-specific genes in Anabaena sp. strain PCC 7120. J. Bacteriol. 193:1823-1832. http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=21317330 Mella-Herrera, R. A., M. R. Neunuebel, and J. W. Golden. 2011. Anabaena sp. strain PCC 7120 conR contains a LytR-CpsA-Psr domain, is developmentally regulated, and is essential for diazotrophic growth and heterocyst morphogenesis. Microbiology 157:617-626. http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=21088107 Kumar, K., R. A. Mella, and J. W. Golden. 2010. Cyanobacterial heterocysts. Cold Spring Harb. Perspect. Biol. 2009. 2:a000315. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2845205/pdf/cshperspect-PRK-a000315.pdf Neunuebel, M. R. and J. W. Golden. 2008. The Anabaena sp. strain PCC 7120 gene all2874 encodes a diguanylate cyclase and is required for normal heterocyst development under high-light growth conditions. J. Bacteriol. 190:6829-6836. http://jb.asm.org/cgi/content/abstract/190/20/6829 Aldea, M. R., K. Kumar, and J. W. Golden, 2008. Heterocyst development and pattern formation, p. 75-90, In S. C. Winans and B. L. Bassler (eds.), Chemical Communication Among Microbes. ASM Press, Washington, D.C. Aldea, M. R., R. A. Mella, and J. W. Golden, 2007. Sigma factor genes sigC, sigE, and sigG are upregulated in heterocysts of the cyanobacterium Anabaena sp. strain PCC 7120. J. Bacteriol. 189:8392-8396. http://jb.asm.org/cgi/content/abstract/189/22/8392 Wu, X., D. W. Lee, R. A. Mella, and J. W. Golden. 2007. The Anabaena sp. strain PCC 7120 asr1734 gene encodes a negative regulator of heterocyst development. Mol. Microbiol. 64:782-794. http://www.blackwell-synergy.com/doi/abs/10.1111/j.1365-2958.2007.05698.x Carrasco, C. D., S. D. Holliday, A. Hansel, P. Lindblad, and J. W. Golden. 2005. Heterocyst-specific excision of the Anabaena sp. strain PCC 7120 hupL element requires xisC. J. Bacteriol. 187: 6031-6038. http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=16109944 Liu, T., J. W. Golden, and D. P. Giedroc. 2005. A Zinc(II)/Lead(II)/Cadmium(II)-Inducible Operon from the Cyanobacterium Anabaena Is Regulated by AztR, an alpha3N ArsR/SmtB Metalloregulator. Biochemistry 44:8673-8683. http://pubs3.acs.org/acs/journals/doilookup?in_doi=10.1021/bi050450%2b Khudyakov, I. Y., and J. W. Golden. 2004. Different functions of HetR, a master regulator of heterocyst differentiation in Anabaena sp. PCC 7120, can be separated by mutation. Proc. Natl. Acad. Sci. USA 101:16040-16045. http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=15520378 \n",
            "-------------------- Scraping faculty url 54/183 --------------------\n",
            "susan golden  Biography Susan Golden received a B.A. (1978) in Biology from Mississippi University for Women and a Ph.D. (1983) in Genetics from the University of Missouri. After postdoctoral research at the University of Chicago, she joined the faculty of Biology at Texas A&M University (1986), where she was promoted to Distinguished Professor in 2003. She joined the Division of Biological Sciences at UCSD in 2008. During her graduate work she developed genetic tools for the cyanobacterium Synechococcus elongatus (PCC 7942), the first cyanobacterium shown to be subject to genetic transformation. This led to work on regulation of light-responsive photosynthesis gene expression in this organism during her postdoctoral research and at Texas A&M. In the early 1990s she began a collaborative project with C.H. Johnson (Vanderbilt University) and T. Kondo (Nagoya University) that demonstrated circadian rhythms of gene expression in S. elongatus, which is currently the only model organism for a prokaryotic circadian clock. The molecular basis of timekeeping in S. elongatus is now a major focus of her lab. Susan is a member of the Faculty of 1000 Biology, a Fellow of the American Academy of Microbiology, and a Member of the National Academy of Sciences.  Research Circadian Rhythms of Gene Expression in Cyanobacteria Diverse eukaryotes, and at least one group of prokaryotescyanobacteriause a circadian (24 h) clock to control physiological events and gene expression. S. elongatus shows a circadian rhythm of bioluminescence when it is transformed with a reporter gene that encodes the luciferase of a bioluminescent marine bacterium or firefly. We are using this reporter system to identify components of circadian system. A group of interacting proteins, KaiA, KaiB, and KaiC are at the heart of the cyanobacterial clock, and kinases help to set the clock and transmit temporal information from it to the genes it regulates. Our goal is to understand the basic mechanism of timekeeping and how the clock becomes synchronized with the environment and controls cellular processes. Functional Genomics in S. elongatus We are identifying all of the genes in the S. elongatus genome that contribute to circadian rhythms by creating a mutation in each gene and assaying each mutant for circadian phenotypes. This project uses transposon insertions into cosmids and plasmids that contain pieces of S. elongatus DNA; each clone that carries a transposon insertion can be used for recombination into the S. elongatus genome to produce a gene \"knockout.\" Metabolic Engineering of Cyanobacteria for the production of Biofuels and other Molecules of Interest Because cyanobacteria grow photosynthetically using water and CO 2 and are easy to manipulate genetically, they are attractive organisms for the production of molecules that have industrial applications. One such application is the production of biofuels as a supplementation of or eventual replacement of petroleum fuels. We are using the powerful genetic tools that have been developed for S. elongatus to explore the production of biofuels in cyanobacteria.  Select Publications Yang Y, Lam V, Adomako M, Simkovsky R, Jakob A, Rockwell NC, Cohen SE, Taton A, Wang J, Lagarias JC, Wilde A, Nobles DR, Brand JJ, Golden SS. Phototaxis in a wild isolate of the cyanobacterium Synechococcus elongatus. Proc Natl Acad Sci U S A. 2018 Dec 14. pii: 201812871. doi: 10.1073/pnas.1812871115. [Epub ahead of print] PMID: 30552139 Broddrick JT, Welkie DG, Jallet D, Golden SS, Peers G, Palsson BO. Predicting the metabolic capabilities of Synechococcus elongatus PCC 7942 adapted to different light regimes. Metab Eng. 2018 Nov 13;52:42-56. doi: 10.1016/j.ymben.2018.11.001. [Epub ahead of print] PMID: 30439494 Cohen SE, McKnight BM, Golden SS. Roles for ClpXP in regulating the circadian clock in Synechococcus elongatus. Proc Natl Acad Sci U S A. 2018 Aug 14;115(33):E7805-E7813. doi: 10.1073/pnas.1800828115. Epub 2018 Jul 30. PMID: 30061418 Welkie DG, Rubin BE, Chang YG, Diamond S, Rifkin SA, LiWang A, Golden SS. Genome-wide fitness assessment during diurnal growth reveals an expanded role of the cyanobacterial circadian clock protein KaiA. Proc Natl Acad Sci U S A. 2018 Jul 24;115(30):E7174-E7183. doi: 10.1073/pnas.1802940115. Epub 2018 Jul 10. PMID: 29991601 Rubin BE, Huynh TN, Welkie DG, Diamond S, Simkovsky R, Pierce EC, Taton A, Lowe LC, Lee JJ, Rifkin SA, Woodward JJ, Golden SS. High-throughput interaction screens illuminate the role of c-di-AMP in cyanobacterial nighttime survival. PLoS Genet. 2018 Apr 2;14(4):e1007301. doi: 10.1371/journal.pgen.1007301. eCollection 2018 Apr. PMID: 29608558 Taton, A., A.T. Ma, M. Ota M, S.S. Golden, and Golden JW. 2017. NOT gate genetic circuits to control gene expression in cyanobacteria. ACS Synth Biol. 2017 Aug 21. doi:10.1021/acssynbio.7b00203. [Epub ahead of print Nagar E., S. Zilberman, E. Sendersky, R. Simkovsky, E. Shimoni, D. Gershtein, M. Herzberg, S.S. Golden, and R. Schwarz R. 2017. Type 4 pili are dispensable for biofilm development in the cyanobacterium Synechococcus elongatus. Environ Microbiol. 2017 Jun 5. doi: 10.1111/1462-2920.13814. [Epub ahead of print] Tseng R., N.F. Goularte, A. Chavan, J. Luu, S.E. Cohen, Y.-G. Chang, J. Heisler, S. Li, A.K. Michael, S. Tripathi, S.S. Golden, A. LiWang., and C.L. Partch. 2017. Structural basis of the day-night transition in a bacterial circadian clock. Science. Mar 17;355(6330):1174-1180. doi: 10.1126/science.aag2516. Epub 2017 Mar 16. Diamond S, Rubin BE, Shultzaberger RK, Chen Y, Barber CD, Golden SS. 2017. Redox crisis underlies conditional light-dark lethality in cyanobacterial mutants that lack the circadian regulator, RpaA. Proc Natl Acad Sci U S A. 2017 Jan 10. pii: 201613078. doi: 10.1073/pnas.1613078114. [Epub ahead of print] PMID: 28074036 Broddrick JT, Rubin BE, Welkie DG, Du N, Mih N, Diamond S, Lee JJ, Golden SS, Palsson BO. 2016. Unique attributes of cyanobacterial metabolism revealed by improved genome-scale metabolic modeling and essential gene analysis. Proc Natl Acad Sci U S A. 2016 Dec 20;113(51):E8344-E8353. doi: 10.1073/pnas.1613446113. PMID: 27911809 Parnasa R, Nagar E, Sendersky E, Reich Z, Simkovsky R, Golden S, Schwarz R. 2016. Small secreted proteins enable biofilm development in the cyanobacterium Synechococcus elongatus. Sci Rep. 2016 Aug 25;6:32209. doi: 10.1038/srep32209. PMID: 27558743 Boyd JS, Cheng RR, Paddock ML, Sancar C, Morcos F, Golden SS. 2016. A Combined Computational and Genetic Approach Uncovers Network Interactions of the Cyanobacterial Circadian Clock. J Bacteriol. 2016 Aug 25;198(18):2439-47. doi: 10.1128/JB.00235-16. PMID: 27381914 Simkovsky R, Effner EE, Iglesias-Snchez MJ, Golden SS. Appl Environ Microbiol. 2016. Mutations in Novel Lipopolysaccharide Biogenesis Genes Confer Resistance to Amoebal Grazing in Synechococcus elongatus. 2016 Apr 18;82(9):2738-50. doi: 10.1128/AEM.00135-16. PMID: 26921432 Free PMC Article Rubin, B.E., K.M. Wetmore, M.N. Price, S. Diamond, R.K. Shultzaberger, L.C. Lowe, G. Curtin G, A.P. Arkin, A. Deutschbauer, and S.S. Golden. 2015. The essential gene set of a photosynthetic organism. Proc Natl Acad Sci U S A. 2015 Oct 27. pii: 201519220. [Epub ahead of print] PMID: 26508635 Luke, C.S., J. Selimkhanov, L. Baumgart, S.E. Cohen, S.S. Golden, N.A. Cookson, and J. Hasty. 2015. A microfluidic platform for long term monitoring of algae in a dynamic environment. ACS Synth Biol. 2015 Sep 15. PMID: 26332284 Chang, Y.-G., S.E. Cohen, C. Phong, W.K. Myers, Y.-I. Kim, R. Tseng, J. Lin, L. Zhang, J.S. Boyd, Y. Lee, S. Kang, D. Lee, S. Li, R.D. Britt, M.J. Rust, S.S. Golden, and A. LiWang. 2015. Protein Fold Switch Joins the Circadian Oscillator to Clock Output in Cyanobacteria. Science 349(6245):324-8. doi: 10.1126/science.1260031. Epub 2015 Jun 25. PMID: 26113641 Diamond S, D. Jun, B.E. Rubin, and S.S. Golden. 2015. The circadian oscillator in Synechococcus elongatus controls metabolite partitioning during diurnal growth. Proc Natl Acad Sci U S A. 2015 2015 Apr 14;112(15):E1916-25. doi: 10.1073/pnas.1504576112. Epub 2015 Mar 30. PMID: 25825710 Espinosa, J., J.S. Boyd, R. Cantos, P. Salinas, S.S. Golden, and A. Contreras. 2015. Cross-talk and regulatory interactions between the essential response regulator RpaB and cyanobacterial circadian clock output. Proc. Natl. Acad. Sci. U S A. pii: 201424632. [Epub ahead of print] PMID: 25653337 Shultzaberger, R.K., J.S. Boyd, T. Katsuk, S.S. Golden, and R.J. Greenspan. 2014. Single mutations in sasA enable a simpler cikA gene network architecture with equivalent circadian properties. 2014. Proc. Natl. Acad. Sci. U S A, 111:E5069-75. doi: 10.1073/pnas.1419902111 PMID: 25385627 Cohen, SE., M.L. Erb, J. Selimkhanov, G. Dong, J. Hasty, J. Pogliano, and S.S. Golden. 2014. Dynamic Localization of the Cyanobacterial Circadian Clock Proteins. 2014. Current Biology, 24:1836-44. PMID: 25385627 \n",
            "-------------------- Scraping faculty url 55/183 --------------------\n",
            "ananda goldrath  Research Immunological memory ensures that, once infected by a particular virus or bacteria, individuals are generally protected from a second encounter with that same pathogen. This ability of lymphocytes to remember is the basis for protection following vaccination. Little is known about the signaling pathways and molecular mechanisms that regulate the formation and maintenance of memory T cell numbers and functional capacity. CD8+ T cells are crucial to the control of infection by pathogens that reside in the cytoplasm such as viruses, intracellular bacteria and protozoan parasites. CD8+ T cells recognize pathogen-derived peptides presented by MHC class I molecules, acting as sentinels in the detection of foreign cytoplasmic proteins. Upon encounter with antigen, naive CD8+ T cells proliferate and differentiate to effector T cells, which are armed to eliminate infected cells. Following antigen clearance, the majority of CD8+ effector cells die; however, a subset survive and differentiate further to long-lived memory T cells. The goal of our research is to understand how CD8+ T cell immunological memory is generated and maintained by identifying the transcriptional and signaling events that regulate the survival and differentiation of T cells as they navigate the immune response and become long-lived memory T cells.  Select Publications Milner JJ, Toma C, Yu B, Zhang K, Omilusik K, Phan AT, Wang D, Getzler AJ, Nguyen T, Crotty S, Wang W, Pipkin ME*, Goldrath AW*. 2017. Nature. Runx3 programs CD8+ T cell residency in non-lymphoid tissues and tumors. 552::253-257. Yu B, Zhang K, Milner JJ, Toma C, Chen R, Scott-Browne JP, Pereira RM, Crotty S, Chang JT, Pipkin ME, Wang W*, Goldrath AW*. 2017. Epigenetic landscapes reveal transcription factors that regulate CD8+ T cell differentiation. Nature Immunology. 5:573-582. Shaw LA, Blanger S, Omilusik KD, Cho S, Scott-Browne JP, Nance JP, Goulding J, Lasorella A, Lu LF, Crotty S*, Goldrath AW*. 2016. Id2 reinforces TH1 differentiation and inhibits E2A to repress TFH differentiation. Nat Immunol. 17(7): 834-43. PMCID: PMC4915968. Omilusik KD, Nadjsombati MS, Shaw LA, Yu B, Milner JJ, Goldrath AW. 2018. Journal of Experimental Medicine. Sustained Id2 regulation of E proteins is required for terminal differentiation of effector CD8+ T cells. 215:773-783. Omilusik KD, Best JA, Yu B, Goossens S, Weidemann A, Nguyen JV, Seuntjens E, Stryjewska A, Zweier C,Roychoudhuri R, Gattinoni L, Bird LM, Higashi Y, Kondoh H, Huylebroeck D, Haigh J, AW Goldrath. Transcriptional repressor ZEB2 promotes terminal differentiation of CD8+ effector and memory T cell populations during infection. Journal of Experimental Medicine. 2015. 212(12): 2027-39. Chang JT, Wherry EJ, Goldrath AW. Molecular regulation of effector and memory T cell differentiation. Nature Immunology. 2014. 15(12):1104-1115. Doedens AL, Phan AT, Stradner MH, Fujimoto JK, Nguyen JV, Yang E, Johnson RS, Goldrath AW. Hypoxia-Inducible Factors enhance CD8+ T cell effector responses to persistent antigen. Nature Immunology. 2013. Nature Immunology. 14(11):1172-82. Best JA, Blair DA, Knell J, Yang E, Mayya V, Doedens A, Dustin ML, Goldrath AW; The Immunological Genome Project Consortium. Transcriptional insights into the CD8+ T cell response to infection and memory T cell formation. Nature Immunology. 2013 14(4):404-12. Yang CY, Best JA, Knell J, Yang E, Sheridan AD, Jesionek AK, Li HS, Rivera RR, Lind KC, DCruz LM, Watowich SS, Murre C, Goldrath AW. The transcriptional regulators Id2 and Id3 control formation of distinct CD8+ memory T cell subsets. Nature Immunology. 2011 12(12):1221-29. D'Cruz LM, Knell J, Fujimoto JK, Goldrath AW. An essential role for the transcription factor HEB in thymocyte survival, Tcra rearrangement and the development of natural killer T cells. Nat Immunol. 2010 Mar;11(3):240-9. Cheung KP, Yang E, Goldrath AW. Memory-like CD8+ T cells generated during homeostatic proliferation defer to antigen-experienced memory cells. J Immunol. 2009 Sep 1;183(5):3364-72. Cannarile MA, Lind NA, Rivera R, Sheridan AD, Camfield KA, Wu BB, Cheung KP, Ding Z, Goldrath AW. Transcriptional regulator Id2 mediates CD8+ T cell immunity. Nat Immunol. 2006 Dec;7(12):1317-25. *Co-corresponding authors \n",
            "-------------------- Scraping faculty url 56/183 --------------------\n",
            "martyn goulding \n",
            "-------------------- Scraping faculty url 57/183 --------------------\n",
            "barry grant  Biography Barry Grant received his undergraduate degree from Queens University Belfast and masters in Bioinformatics from the University of York (UK). He then worked at DeCode Genetics, Iceland before obtaining a Ph.D. from the University of York and Marie Curie Research Institute. Barry a was postdoctoral fellow at UC San Diego and the Howard Hughes Medical Institute. Before joining our faculty in 2017 Barry was an Assistant Professor of Computational Medicine and Bioinformatics at the University of Michigan, Ann Arbor.  Research My research focuses on the development and application of state-of-the-art computational and theoretical techniques to investigate the structure, dynamics and interactions of biomedically important proteins. At the interface of molecular biology, biophysics and bioinformatics the unique insight gained from our research is tightly coupled to a wide range of biochemical and biophysical experiments. Using this approach, we have recently developed a new class of allosteric Ras & Rho molecular switch inhibitors, reported the first rationally designed faster velocity kinesin motors, demonstrated for the first time that kinesin processivity can be modulated by rational mutation of the motor domain, discovered key atomistic detriments of allosteric activation in G proteins, and developed the Bio3D software package used by thousands of researchers around the world. Our specific areas of expertise include: Bioinformatics Development and application of bioinformatics methods for the analysis of protein sequence-structure-function relationships. Biomolecular Simulation Application of advanced molecular simulation techniques to probe the structure, dynamics and interactions of biomolecules. Drug Discovery & Design Virtual screening, structure-based drug design, cheminformatics and molecular modeling supporting drug discovery and design.  Select Publications Complete bibliography and full-text options available from: http://thegrantlab.org/publications/ Li H, Yao XQ, Grant BJ. Comparative structural dynamic analysis of GTPases. PLoS Comput Biol . 14(11): e1006364. PMCID: In progress Muretta JM, Reddy BJN, Scarabelli G, Thompson AF, Jariwala S, Major J, Venere M, Rich JN, Willard B, Thomas DD, Stumpff J, Grant BJ, Gross SP, Rosenfeld SS. A posttranslational modification of the mitotic kinesin Eg5 that enhances its mechanochemical coupling and alters its mitotic function. PNAS . 2018; 115 (8) E1779-E1788109. PMCID: PMC5828613 Yue Y, Blasius TL, Zhang S, Jariwala S, Walker B, Grant BJ, Cochran JC and Verhey KJ. Altered chemomechanical coupling causes impaired motility of the kinesin-4 motors KIF27 and KIF7. J Cell Bio . 2018;217(4):1319-1334. PMCID: PMC5881503 Atherton J, Jiang K, Stangier MM, Luo Y, Hua S, Houben K, van Hooff JJE, Joseph AP, Scarabelli G, Grant BJ, Roberts AJ, Topf M, Steinmetz MO, Baldus M, Moores CA, Akhmanova A. A structural model for microtubule minus-end recognition and protection by CAMSAP proteins. Nat Struct Mol Biol . 2017; 24, 931943. PMCID: PMC6134180 Yao XQ, Cato MC, Labudde E, Beyett TS, Tesmer JJG, Grant BJ. Navigating the conformational landscape of G protein-coupled receptor kinases during allosteric activation. J Biol Chem . 2017;292(39), 16032-16043. PMCID: PMC5625036 Stender, Nwachukwu, Kastrata, Kim, Strid, Yakir, Srinivasan, Nowak, Izard, Erumbi, Carlson, Katzenellenbogen, Yao, Grant, Leong, Lin, Frasor, Nettles, Glass. (2017) Structural and molecular mechanisms of cytokine-mediated endocrine resistance in human breast cancer cells. Molecular Cell . 65(6), 1122-1135. Jariwala, Skjrven, Yao, Grant. (2017) Investigating protein sequence-structure-dynamics relationships with Bio3D-web. Journal of Visualized Experiments (JoVE) . Jul 16;(125). doi: 10.3791/55640. Prakash, Sayyed-Ahmad, Cho, Dolino, Chen, Li, Grant, Hancock, Gorfe. (2017) Computational and biochemical characterization of two partially overlapping interfaces and multiple weak-affinity K-Ras dimers. Scientific Reports . 7, 40109 doi:10.1038/srep40109. Nwachukwu, Srinivasan, Bruno, Nowak, Wright, Minutolo, Erumbi, Izard, Yao, Grant, Kojetin, Elemento, Katzenellenbogen, Nettles. (2017) Systems structural biology analysis of ligand effects on ER predicts cellular response to environmental estrogens and anti-hormone therapies. Cell Chemical Biology . 24(1), 35-45. Yao XQ, Skjaerven L, Grant BJ. (2016) Rapid characterization of allosteric networks with ensemble normal mode analysis. J Phys Chem B . 120(33), 8276-88. DOI: 10.1021/acs.jpcb.6b019912016. Skjaerven L, Jariwala S, Yao XQ, Grant BJ. (2016) Online interactive analysis of protein structure ensembles with Bio3D-web. Bioinformatics . 32(22), 3510-3512. Yao XQ, Malik RU, Griggs NW, Skjaerven L, Traynor JR, Sivaramakrishnan S, Grant BJ. (2016) Dynamic coupling and allosteric networks in the alpha subunit of heterotrimeric G proteins. J Biol Chem . 291(9):4742-53. PMCID: 4813496. Scarabelli G, Soppina V, Yao XQ, Atherton J, Moores CA, Verhey KJ, Grant BJ. (2015) Mapping the processivity determinants of the kinesin-3 motor domain. Biophys J . 109(8):1537-40. PMCID: 4624112. Skjaerven L, Yao XQ, Scarabelli G, Grant BJ. (2014) Integrating protein structural dynamics and evolutionary analysis with Bio3D. BMC Bioinformatics . 15:399. PMCID: 4279791. Scarabelli G, Grant BJ. (2014) Kinesin-5 allosteric inhibitors uncouple the dynamics of nucleotide, microtubule, and neck-linker binding sites. Biophys J . 107(9):2204-13. PMCID: 4223232. Yao XQ, Grant BJ. (2013) Domain-opening and dynamic coupling in the alpha-subunit of heterotrimeric G proteins. Biophys J . 105(2):L08-10. PMCID: 3714883. Scarabelli G, Grant BJ. (2013) Mapping the structural and dynamical features of kinesin motor domains. PLoS Comput Biol . 9(11):e1003329. PMCID: 3820509. Grant BJ, Gheorghe DM, Zheng W, Alonso M, Huber G, Dlugosz M, McCammon JA, Cross RA. (2011) Electrostatically biased binding of kinesin to microtubules. PLoS Biol . 9(11):e1001207. PMCID: 3226556. Grant BJ, Rodrigues AP, ElSawy KM, McCammon JA, Caves LS. (2006) Bio3D: An R package for the comparative analysis of protein structures. Bioinformatics . 2006;22(21):2695-6. \n",
            "-------------------- Scraping faculty url 58/183 --------------------\n",
            "melvin green \n",
            "-------------------- Scraping faculty url 59/183 --------------------\n",
            "ralph greenspan  Biography Dr. Ralph J. Greenspan has worked on the genetic basis of behavior and brain function in fruit flies (Drosophila melanogaster) almost since the inception of the field, studying with one of its founders, Jeffery Hall, at Brandeis University in Massachusetts, where he received his Ph.D. in biology in 1979. He is currently Director of the Center for Brain Activity Mapping of the Kavli Institute for Brain and Mind at the University of California San Diego, and Professor in the Division of BIologys Neurobiology Section and the Dept. of Cognitive Science.  Research Dr. Greenspans research includes studies of the consequences of mutations and localized genetic alterations in the nervous system, molecular identification of genes underlying naturally occurring variation in behavior, and the genetic analysis of sleep and attention in the fruit fly. His current research addresses large-scale network interactions pertaining to the action of genes and neurons. In 2011, he was one of the small team of scientists that produced the white paper for the White House Office of Science and Technology Policy that eventuated in the BRAIN Initiative, and in 2014 spearheaded the effort to establish the state-funded Cal-BRAIN program and serves as its co-Director. In addition to research papers, he has authored: Fly Pushing: The Theory and Practice of Drosophila Genetics, An Introduction to Nervous Systems, and How Genes Influence Behaviour (with Jonathan Flint and Ken Kendler).  Select Publications Shaw, P.J., Cirelli, C., Greenspan, R.J. and Tononi, G. (2000) Correlates of sleep and waking in Drosophila melanogaster. Science 287: 1834-1837. Toma, D.P., White, K.P., Hirsch, J. and Greenspan, R.J. (2002) Identification of genes involved in Drosophila melanogaster geotaxis, a complex behavioral trait. Nature Genetics 31: 349-353. Nitz, D.A., van Swinderen, B., Tononi, G. and Greenspan, R.J. (2002) Electrophysiological correlates of rest and activity in Drosophila melanogaster. Current Biology 12: 1934-1940. van Swinderen, B. and Greenspan, R.J. (2003) Salience Modulates 20-30 Hz Brain Activity in Drosophila. Nature Neuroscience 6: 579-586. van Swinderen, B., Nitz, D.A. and Greenspan, R.J. (2004) Uncoupling of brain activity from movement defines arousal states in Drosophila. Current Biology 14: 81-87. Andretic, R., van Swinderen, B. and Greenspan, R.J. (2005) Dopaminergic modulation of arousal in Drosophila. Current Biology 15: 1165-1175. Dierick, H.A. and Greenspan, R.J. (2006) Molecular analysis of flies selected for aggressive behavior. Nature Genetics 38: 1023-1031. Foltenyi, K., Greenspan, R.J., and Newport, J.W. (2007) Activation of Egf-r and ERK by Rhomboid signaling regulates the consolidation and maintenance of sleep in Drosophila. Nature Neuroscience 10: 1160 1167. Alivisatos, A.P., Chun, M., Church, G.M., Greenspan, R.J., Roukes, M.L. and Yuste, R. (2012) The brain activity map project and the challenge of functional connectomics. Neuron 74: 970 974. Lin, C-Y., Chuang, C-C., Hua, T-E., Chen, C-C., Dickson, B.J., Greenspan, R.J. and Chiang, A-S. (2013) A comprehensive wiring diagram of the protocerebral bridge for visual information processing in the Drosophila brain. CELL Reports 3: 1739-1753. Shih, C-T, Sporns, O, Yuan, S-L, Lin, Y-J, Su, T-S, Chuang, C-C., Lo, C-C, Greenspan, RJ and Chiang, A-S. (2014) Toward the Drosophila connectome: Structural analysis of the brain network. Curr. Biol., 25: 1249-1258. Grover D, Katsuki T, Greenspan RJ. (2016) Flyception: imaging brain activity in freely walking fruit flies. Nat Methods. 13: 569-572. \n",
            "-------------------- Scraping faculty url 60/183 --------------------\n",
            "shelley halpain  Biography Shelley Halpain received her B.S. in Biological Sciences at The University of California Irvine, and her Ph.D. in Neuroscience from the Rockefeller University. Her postdoctoral training was under Professor Paul Greengard, a 2001 Nobel laureate in Physiology or Medicine. Dr. Halpain was on the faculty at the University of Virginia Health Sciences Center 1992-1996, then at The Scripps Research Institute 1996-2007. She became Professor in the Division of Biological Sciences at UCSD in 2007. Dr. Halpain served as Council Delegate for the AAAS Section on Neuroscience from 1998-2000, is a member of national and international scientific review panels, and is a frequent contributor of reviews and commentary for international science journals. Professor Halpain's research has been published in top scientific journals and has been featured in National Geographic magazine.  Research Dr. Halpains research concerns the molecular basis for neural development. Her laboratory uses advanced light microscopy and quantitative cellular imaging methods to investigate neurite outgrowth and synapse formation. The lab also develops automated, high-content screening assays of neuronal morphology for use in drug discovery and identification of novel signaling pathways. Proteomics analyses are used to investigate key players in neuritogenesis. A particular focus is the role that cytoskeletal proteins, especially microtubules and actin filaments, play in establishing and maintaining neuronal networks. The laboraotorys work is fundamentally relevant to many brain disorders, ranging from Alzheimers disease to mood disorders and autism.  Select Publications Dehmelt, L., Smart, F., Ozer, R., and Halpain, S. (2003) The role of MAP2c in the reorganization of microtubules and lamellipodia during neurite initiation, J. Neuroscienc, 23:9479-9490. Roger, B., Al-Bassam, J., Dehmelt, L., Milligan, R., and Halpain, S. (2004) MAP2c, but not tau, binds and bundles F-actin via its microtubule binding domain. Current Biology 14:363-371. Calabrese, B. and Halpain, S. (2005) Essential role for the PKC target MARCKS in maintaining dendritic spine morphology. Neuron 48:77-90. Calabrese, B., Wilson, M., and Halpain, S. (2006) Development and regulation of dendritic spine synapses. Physiology 21:38-47. Dehmelt, L., Nalbant, P., Steffan, W., and Halpain, S. (2006) A microtubule-based, dynein-dependent force induces local cell protrusions: implications for neurite initiation. Brain Cell Biology 35:43-60. Al-Bassam, J., Roger, B., Halpain, S., and Milligan, R.A. (2007) Analysis of the weak interactions of ADP-Unc104 and ADP-Kinesin with microtubules and their inhibition by MAP2c. Cell Motil. Cytoskeleton 64(5):377-389. Calabrese, B., Shaked, G.M., Tabarean IV, Braga, J., Koo, E.H., and Halpain, S. (2007) Rapid, concurrent alterations in pre- and postsynaptic structure induced by naturally-secreted amyloid-beta protein. Molecular Cell Neuroscience 35:183-193. \n",
            "-------------------- Scraping faculty url 61/183 --------------------\n",
            "randy hampton  Biography Randy Hampton received his Ph. D. from the University of Wisconsin-Madison and did his postdoctoral work at UC Berkeley.  Research Our group studies a variety of questions that surround how the cell selectively destroys proteins. This work started with understanding how these sorts of mechanisms are employed to allow regulation of cellular cholesterol synthesis. These studies are ongoing and have led us in a number of interesting directions including cellular management of misfolded proteins, lipid signaling, and the dynamics of the ER and other organelles. For a complete picture of what weve done, what we do, and what we want to do, see the Hampton lab web site listed above. Thanks for stopping by!  Select Publications Higgins R, Gendron JM, Rising L, Mak R, Webb K, Kaiser SE, Zuzow N, Riviere P, Yang B, Fenech E, Tang X, Lindsay SA, Christianson JC, Hampton RY, Wasserman SA, Bennett EJ. The Unfolded Protein Response Triggers Site-Specific Regulatory Ubiquitylation of 40S Ribosomal Proteins. Mol Cell. (2015) 59:35-49. Theesfeld CL, Hampton RY. Insulin-induced gene protein (INSIG)-dependent sterol regulation of Hmg2 endoplasmic reticulum-associated degradation (ERAD) in yeast.J Biol Chem. (2013) 288:8519-30. Hampton RY, Sommer T. Finding the will and the way of ERAD substrate retrotranslocation. Curr Opin Cell Biol. (2012) 24:460-6. Hampton RY, Lord JM, Roberts LM. Cytosolic entry of Shiga-like toxin a chain from the yeast endoplasmic reticulum requires catalytically active Hrd1p. Li S, Spooner RA, PLoS One. (2012) 7:e41119. Theesfeld CL, Pourmand D, Davis T, Garza RM, Hampton RY. The sterol-sensing domain (SSD) directly mediates signal-regulated endoplasmic reticulum-associated degradation (ERAD) of 3-hydroxy-3-methylglutaryl (HMG)-CoA reductase isozyme Hmg2.J Biol Chem. (2011) 286:26298-307. Hampton RY. San1-mediated quality control: substrate recognition \"sans\" chaperones. Molecular Cell. (2011) 41:2-3. Heck JW, Cheung SK, Hampton RY. Cytoplasmic protein quality control degradation mediated by parallel actions of the E3 ubiquitin ligases Ubr1 and San1. Proc Natl Acad Sci U S A. (2010) 107:1106-11. Carroll SM, Hampton RY. Usa1p is required for optimal function and regulation of the Hrd1p endoplasmic reticulum-associated degradation ubiquitin ligase. J Biol Chem. (2010) 285:5146-56. Garza RM, Tran PN, Hampton RY. Geranylgeranyl pyrophosphate is a potent regulator of HRD-dependent 3-Hydroxy-3-methylglutaryl-CoA reductase degradation in yeast. J Biol Chem. (2009) 284:35368-80. Sato BK, Schulz D, Do PH, Hampton RY. Misfolded membrane proteins are specifically recognized by the transmembrane domain of the Hrd1p ubiquitin ligase. Molecular Cell. (2009) 34:212-22 \n",
            "-------------------- Scraping faculty url 62/183 --------------------\n",
            "sung han \n",
            "-------------------- Scraping faculty url 63/183 --------------------\n",
            "nan hao  Biography Nan Hao received his Ph.D in Biochemistry and Biophysics from the University of North Carolina at Chapel Hill. He was a postdoctoral fellow with Timothy Elston at the University of North Carolina and a postdoctoral fellow with Erin OShea at Harvard University/Howard Hughes Medical Institute.  Research In recent years, genomic sequencing and systematic analysis have revealed many molecular components that control gene expression at multiple levels, and detailed the myriad of interactions among and between these regulatory modules. However, it remains unclear how these molecular networks operate in time and space to carry out diverse cellular functions. The goal of our lab is to understand how network architecture governs the dynamics and function of regulatory responses in the context of stresses , aging , or diseases . Our lab uses multidisciplinary approaches, integrating biology, engineering and computer science, in our studies. We develop and use novel microfluidics and imaging technologies to quantitatively track the dynamics of molecular processes in single living cells and construct computational models to describe and predict cellular behaviors. In particular, our laboratory focuses on the following three major research themes: Decoding the dynamics of stress responses Cells could transmit environmental information and initiate cellular responses through regulating the temporal dynamics of signaling activities. Our current work focuses on understanding: (1) the network structures that encode the environmental information into signaling dynamics; (2) the gene circuits that decode signaling dynamics into functions; (3) the mechanisms that underlie cellular memory of environmental changes. Probing the causes of cellular aging Cellular aging plays an important role in many diseases, such as cancers and neurodegenerative disorders, but the mechanisms that drive the aging process remain largely unclear. We aim to combine transformative microfluidics and single-cell imaging technologies with computational modeling to obtain a quantitative and predictive understanding of the aging process and to develop new strategies to promote longevity. Quantifying the heterogeneity in cancer cells We have expanded our quantitative biology research into mammalian cell systems. We have developed new microfluidic and imaging technologies to track individual mammalian cells over a very long period of time. Our current work focuses on studying the heterogeneous signaling dynamics in cancer cells and how these dynamics influence cancer cell growth.  Select Publications Li Y, Jin M, OLaughlin R, Bittihn P, Tsimring LS, Pillus L, Hasty J, Hao N. (2017) Multi-generational silencing dynamics control cell aging. Proc Natl Acad Sci U S A. Accepted. Shao B, Yuan H, Zhang R, Wang X, Zhang S, Ouyang Q, Hao N*, Luo C*. (2017) Reconstructing the regulatory circuit of cell fate determination in yeast mating response. PLoS Comput Biol. 13(7):e1005671. (* co-correspondence) Jiang Y, AkhavanAghdam Z,Tsimring LS, Hao N. (2017) Coupled feedback loops control the stimulus-dependent dynamics of the yeast transcription factor Msn2. J Biol Chem. 292(30):12366-12372. AkhavanAghdam Z, Sinha J, Tabbaa OP, Hao N. (2016) Dynamic control of gene regulatory logic by seemingly redundant transcription factors. Elife. 5. pii: e18458. Hao N, Budnik BA, Gunawardena J, OShea EK. (2013) Tunable signal processing through modular control of transcription factor translocation. Science 339(6118):460-4. Hao N, O'Shea EK. (2011) Signal-dependent dynamics of transcription factor translocation controls gene expression. Nat Struct Mol Biol. 19(1):31-9. Hao N, Nayak S, Behar M, Shanks RH, Nagiec MJ, Errede B, Hasty J, Elston TC, Dohlman HG. (2008) Regulation of cell signaling dynamics by the protein kinase-scaffold Ste5. Mol Cell. 30(5):649-56. \n",
            "-------------------- Scraping faculty url 64/183 --------------------\n",
            "diana hargreaves \n",
            "-------------------- Scraping faculty url 65/183 --------------------\n",
            "jeff hasty  Biography Jeff Hasty received his Ph.D. in Physics from the Georgia Institute of Technology in 1997, where he worked with Kurt Wiesenfeld. He was a postdoctoral fellow with Jorge Vinals at the Supercomputing Research Institute ('97-'98), and apostdoctoral fellow with Jim Collins in the Applied BioDynamics Lab at Boston University ('98-'01). Somewhereduring his postdoctoral stay at Boston University, he mutated into a hybrid computational/molecular biologist. He is currently at the University of California, San Diego, where he is a Professor in the Departmentsof Molecular Biology and Bioengineering, and the Director of the BioCircuits Institute. His main interest is thedesign and construction of synthetic gene-regulatory and signaling networks.  Research Dr. Hasty's research focuses on the construction and utilization of synthetic gene circuits for dissecting, analyzing, and controlling the dynamical interactions involved in gene regulation.  Select Publications Tal Danino, Octavio Mondragn-Palomino, Lev Tsimring, Jeff Hasty (2010). A synchronized quorum of genetic clocks. Nature 426:326-330. Natalie Cookson, Scott Cookson, Lev Tsimring, Jeff Hasty (2009). Cell cycle-dependent variations in protein concentration. Nucleic Acids Research 1-6. Natalie Cookson, Lev Tsimring, Jeff Hasty (2009). The pedestrian watchmaker: Genetic clocks from engineered oscillators. FEBS Letters 583:39313937. Matthew Bennett, Jeff Hasty (2009). Microfluidic devices for measuring gene network dynamics in single cells. Nature Reviews Genetics, 10:628-638. Jesse Stricker, Scott Cookson, Matthew Bennett, William Mather, Lev Tsimring, Jeff Hasty (2008). A fast, robust and tunable synthetic gene oscillator. Nature 456:516-519. Matthew Bennett, Wyming Lee Pang, Natalie Ostroff, Bridget Baumgartner, Sujata Nayak, Lev Tsimring, Jeff Hasty (2008). Metabolic gene regulation in a dynamically changing environment. Nature 454:1119-1122. \n",
            "-------------------- Scraping faculty url 66/183 --------------------\n",
            "stephen hedrick  Biography Dr. Hedrick received his Ph.D. from the University of California, Irvine and conducted postdoctoral studies at the National Institutes of Health.  Research Immune homeostasis: the sum of lymphocyte dynamics Although homeostatic regulation is universal in biological systems, it is prominently in evidence within lymphocyte populations, and like almost every aspect of the immune system, the dynamics of lymphoid populations are wildly exaggerated. An infectious agent can provoke quiescent, antigen-specific T cells to suddenly grow in size, proliferate more than 10,000-fold, and then at a prescribed point in the process collectively undergo an equally dramatic and selective contraction in numbers. Although such events are extravagant compared with the constant turnover exhibited by other organismal systems, they are not without control. The height of the expansion and the plateau of survival are both regulated and in some manner self-limited. Moreover, the body appears to have a set-point for total lymphocytes diffuse throughout the body, about which the populations oscillate. In addition to interesting biology, this regulation of lymphocytes is important for almost all of the diseases affecting an aging population: immune hypersensitivity, including asthma; autoimmunity, such as rheumatoid arthritis and psoriasis; cardiovascular disease; cancer; and metabolic diseases such as Type II diabetes. We are interested in understanding the conceptual basis for these immunopathologies. Lymphocyte survival and quiescence. We have found that a family of Forkhead transcription factors (named Foxo) are essential for the maintenance of many aspects of T cell homeostasis. Two of the family members, Foxo1 and Foxo3, have unique functions within the immune system, where Foxo1 regulates survival and homing properties of T cells (Kerdiles et al., 2009) and Foxo3 regulates the ability of dendritic cells to promote T cell survival (Dejean et al., 2009). In addition, the absence of Foxo1 in T cells results in chronic immune inflammation and autoreactive antibodies. The factors also have overlapping functions in that the T cell-specific deletion of Foxo1,3 results in florid autoimmunity. We are interested in understanding the program of gene expression controlled by Foxo transcription factors, and how the disregulation of this control results in immunopathology. Lymphocyte expansion and survival. A central signaling pathway in T cell activation is centered around the mitogen-activated protein kinases, Erk1 and Erk2. These factors are collectively essential for progression through the cell cycle, and individually they regulate proliferation, survival and the fate of antigen-activated T cells. We have found that antigen-mediated T cell activation is governed in part by the strength of signal transmitted by Erk2, and this regulates the magnitude of T cell expansion (DSouza et al., 2008). We have also found that the absence of Erk2 promotes an alternative differentiation program in CD4 T cells resulting in an abnormally high number of Foxp3+ T regulatory cells. We are exploring a connection between Erk2, TGFb, and Foxo1 in this process. Alternate forms of lymphocyte programmed death. The contraction of a lymphocyte population following the clearance of an infectious agent occurs via apoptotic death defined as the induction of DNA fragmentation, nuclear blebbing, and cell shrinkage. In many cases, infectious agents have evolved mechanisms to prevent apoptotic death, and in particular, many viruses prevent apoptosis, and this appears to promote virulence. A second pathway of cell occurs when apoptosis is inhibited, and we are studying the basis of this mode of programmed cell death (Chen et al., 2008).  Select Publications Delpoux, A., Lai, C.Y., Hedrick, S.M., and Doedens, A.L. (2017). FOXO1 opposition of CD8+ T cell effector programming confers early memory properties and phenotypic diversity. Proc Natl Acad Sci U S A 114, E8865E8874. Hedrick, S.M. (2017). Understanding Immunity through the Lens of Disease Ecology. Trends Immunol 38, 888903. Hedrick, S.M. (2018). The Imperative to Vaccinate. J Pediatr 201, 259263. Tsau, J.S., Huang, X., Lai, C.Y., and Hedrick, S.M. (2018). The Effects of Dendritic Cell Hypersensitivity on Persistent Viral Infection. J Immunol 200, 13351346. Delpoux, A., Michelini, R.H., Verma, S., Lai, C.Y., Omilusik, K.D., Utzschneider, D.T., Redwood, A.J., Goldrath, A.W., Benedict, C.A., and Hedrick, S.M. (2018). Continuous activity of Foxo1 is required to prevent anergy and maintain the memory state of CD8+ T cells. J Exp Med 215, 575594. Utzschneider, D.T., Delpoux, A., Wieland, D., Huang, X., Lai, C.Y., Hofmann, M., Thimme, R., and Hedrick, S.M. (2018). Active Maintenance of T Cell Memory in Acute and Chronic Viral Infection Depends on Continuous Expression of FOXO1. Cell Rep 22, 34543467. Stienne, C., Michieletto, M.F., Benamar, M., Carri, N., Bernard, I., Nguyen, X.H., Lippi, Y., Duguet, F., Liblau, R.S., Hedrick, S.M., Saoudi, A., and Dejean, A.S. (2016). Foxo3 Transcription Factor Drives Pathogenic T Helper 1 Differentiation by Inducing the Expression of Eomes. Immunity 45, 774787. Allison, K.A., Sajti, E., Collier, J.G., Gosselin, D., Troutman, T.D., Stone, E.L., Hedrick, S.M., and Glass, C.K. (2016). Affinity and dose of TCR engagement yield proportional enhancer and gene activity in CD4+ T cells. Elife 5. Stone, E.L., Pepper, M., Katayama, C.D., Kerdiles, Y.M., Lai, C.Y., Emslie, E., Lin, Y.C., Yang, E., Goldrath, A.W., Li, M.O., Cantrell, D.A., and Hedrick, S.M. (2015). ICOS Coreceptor Signaling Inactivates the Transcription Factor FOXO1 to Promote Tfh Cell Differentiation. Immunity 42, 239251. \n",
            "-------------------- Scraping faculty url 67/183 --------------------\n",
            "donald helinski  Biography Don Helinski earned his Ph.D at Case Western University and held a postdoctoral fellowship at Stanford University. Before joining the faculty at UCSD, he was on the faculty at Princeton University. He is a member of the National Academy of Sciences and the American Academy of Arts and Sciences. He is also a Fellow of the American Association for the Advancement of Science and an elected member of the European Molecular Biology Organization and the American Academy of Microbiology. \n",
            "-------------------- Scraping faculty url 68/183 --------------------\n",
            "martin hetzer \n",
            "-------------------- Scraping faculty url 69/183 --------------------\n",
            "david holway  Biography David Holway received a B.A. in zoology from UC Berkeley and a Ph.D in biology from the University of Utah. He conducted postdoctoral research at UC San Diego.  Research Research in our lab is currently divided into two areas: Biological invasions: The introduction of species into new environments has increasingly become an economically costly and environmentally disruptive phenomenon. Our research on invasions focuses on social insects (ants, bees and wasps) and primarily encompasses the following questions. (i) What factors control susceptibility to invasion? (ii) What accounts for how invasion impacts change in magnitude over time? (iii) Why do native species assemblages differ in their ability to recover following experimental invader removal? Pollination services: Declining pollinator populations threaten to compromise the integrity of pollination services in both agricultural and non-managed systems. Our research addresses this general issue from the following perspectives. (i) To what degree does land-use intensification affect pollinator assemblages? (ii) To what extent do pollinator declines compromise pollination services? (iii) How do non-native species interact with native pollinators, and when do these interactions negatively impact pollination services?  Select Publications Hung J., J.M. Kingston, A. Lee, D.A. Holway & J.R. Kohn. Non-native honey bees disproportionately dominate abundant floral resources in a hotspot of pollinator diversity. Proceedings of the Royal Society B, In press. Hung, J., J.S. Ascher, J.A. Davids & D.A. Holway. Ecological filtering in scrub fragments restructures the taxonomic and functional composition of native bee assemblages. Ecology, In press Menke, S.B., P.S. Ward & D.A. Holway. 2018. Long-term record of Argentine ant invasions reveals enduring ecological impacts. Ecology, 99:11941202. Hung, J., J.M. Kingston, M. Albrecht, D.A. Holway & J.R. Kohn. 2018. The worldwide importance of honey bees as pollinators in natural habitats. Proceedings of the Royal Society B 285: 20172140. Nabors, A.J., H.J. Cen, K-L.J. Hung, J.R. Kohn & D.A. Holway. 2018. The effect of removing numerically dominant, non-native honey bees on seed set of a native plant. Oecologia 186:281-289. Hung, J., J.S. Ascher & D.A. Holway. 2017. Urbanization-induced habitat fragmentation erodes multiple components of temporal diversity in a Southern California native bee assemblage. PLoS One 12:e0184136. Schochet, A.B., K-L J. Hung, & D.A. Holway. 2016. Bumble bee species exhibit divergent responses to urbanization in a Southern California landscape. Ecological Entomology 41:685-692. Hanna, C., I. Naughton, C. Boser & D.A. Holway. 2015. Testing the effects of ant invasions on non-ant arthropods with high-resolution taxonomic data. Ecological Applications 25:1841-1850. LeVan, K.E. & D.A. Holway. 2015. Ant-aphid interactions increase ant floral visitation and reduce plant reproduction via decreased pollinator visitation. Ecology 96:1620-1630. Hanna, C., I. Naughton, C. Boser, R. Alarcn, K-L J. Hung, & D.A. Holway. 2015. Floral visitation by the Argentine ant reduces bee visitation and plant seed set. Ecology 96:222-230. \n",
            "-------------------- Scraping faculty url 70/183 --------------------\n",
            "alisa huffaker  Biography Dr. Huffaker received her Ph.D. from the Institute of Biological Chemistry at Washington State University. She continued at the Institute of Biological Chemistry for her postdoctoral training in plant defense signaling. Dr. Huffaker joined the section of Cell and Developmental Biology in 2014.  Research Like animals, plants continually interact with a plethora of other organisms in their environment, including potentially pathogenic microbes and herbivorous insects that can cause disease or damage. To minimize the detrimental impact of invading organisms, plants must mount a rapid and robust immune response. My research interests center on plant peptide signals, Peps, which regulate broad spectrum plant defense responses against pathogens and herbivores, and their application to manipulate plant resistance to biotic attack. Multigene families encode Plant Elicitor Peptides (Peps) and their receptors (PEPRs), which act together to coordinate plant immunity. Plant recognition of invading organisms occurs through perception of foreign molecules associated with attacking microbes and herbivores, known as elicitors, MAMPs (microbe-associated molecular patterns) or HAMPs (herbivore-associated molecular patterns). Perception of these molecules by pattern recognition receptors (PRRs) leads to subsequent launching of complex and multilayered defenses. These defenses, also known as basal or innate immune responses, have damaging effects on a wide array of pathogens and pests and involve accumulation of antimicrobial proteins and toxic metabolites. In addition to perceiving molecules from foreign organisms to detect biotic attack, plants also produce and recognize endogenous elicitors to assist in recognition and amplification of the immune response. Arabidopsis Plant Elicitor Peptide1 (AtPep1) was the first endogenous peptide signal found to regulate plant anti-microbial defenses. AtPep1 is derived from a larger precursor encoded by the Arabidopsis gene designated AtPROPEP1 , which is expressed in response to treatment with microbe-derived elicitors. AtPROPEP1 is a member of an eight gene family in Arabidopsis thaliana , each of which encode peptides that interact with the Plant Elicitor Peptide Receptors (PEPRs) to regulate plant immune responses. Treatment of Arabidopsis plants with AtPep1 induces production of defensive phytohormones and expression of pathogen defense genes. Transgenic over-expression of AtPROPEP1 causes constitutive expression of defense genes and enhances resistance to root degradation by the oomycete Pythium irregulare . Similarly, pretreatment of Arabidopsis plants with AtPeps prior to infection with Pseudomonas syringae Pv. tomato DC3000 restricts bacterial growth and decreases symptoms of disease. Peps coordinate and augment pathogen defense responses both spatially and temporally through amplification of MAMP-induced signaling. AtPeps are active only in Arabidopsis and closely related Brassicaceous species, but gene orthologs have been identified in more than 50 plant species. These Pep orthologs also regulate innate immunity. Treatment of maize plants with the ortholog ZmPep1 prior to infection with fungal pathogens caused accumulation of defense transcripts and metabolites and reduced symptoms and cell death caused by both Cochliobolus heterostrophus (Southern leaf blight) and Colletotrichum heterostrophus (Anthracnose stalk rot). Peps also mediate anti-herbivore defenses. Upon attack by herbivores, plants emit a complex blend of chemicals composed predominately of terpenes and six carbon (C6) green leafy volatiles (GLVs). These volatiles deployed by the plant recruit other organisms from the community to defend it from damage. Beneficial insects such as wasp species are attracted to the volatile blend and either eat or lay eggs in the attacking herbivorous pests. A maize Plant Elicitor Peptide, ZmPep3, is a potent signal promoting emission of herbivore associated volatiles and treatment of maize leaves with ZmPep3 results in accumulation of transcripts encoding biosynthetic enzymes for volatiles and other defense metabolites. These ZmPep3-induced responses are effective as direct and indirect defenses against herbivores, limiting growth of Spodoptera exigua larvae and attracting nave Cotesia marginiventris , a parasitoid of Lepidopteran larvae. Similar regulation of anti-herbivore defense by Peps occurs in numerous and diverse plant species. Some Peps promote plant defense against herbivores. Because PROPEP genes and signaling by Peps are ubiquitous throughout angiosperm plants, knowledge gleaned from Pep research in maize and Arabidopsis can be extrapolated and applied to other plants. Peps act as regulatory nodes for defenses against pathogens and herbivores, and are valuable candidate tools for genetic improvement of crop resistance. Study of Peps have generated a number of compelling questions for future research. How is signal specificity achieved? What signaling components are involved? How can this endogenous regulation of the plant immune system be improved to protect plants? As individual Peps appear to have differing functions, were studying the overlap of signaling and defense strategies mediated by individual Peps and by exogenous elicitors from other organisms. We hope to decode Pep-induced response specificity and to identify molecular components that can be successfully used to enhance resistance against a variety of attackers, including biotrophic and nectrotrophic pathogens as well as specialist and generalist herbivores. Current lab efforts are focused on 1. Study of Pep-PEPR ligand-receptor interactions and determination of structural requirements for signaling activity. 2. Characterization of downstream Pep signaling components identified through genetic screens and profiling of rapid changes in transcriptome and phosphoproteome of maize and Arabidopsis. 3. Development of Pep-based strategies for enhancement of plant immunity against pathogens and pests.  Select Publications Schmelz EA, Huffaker A, Sims JW, Christensen SA, Lu X, Okada K, Peters RJ (2014) Biosynthesis, elicitation and roles of monocot terpenoid phytoalexins. Plant J. 79(4):659-78. PMID: 24450747 Huffaker A, Pearce G, Veyrat N, Erb M, Turlings TC, Sartor R, Shen Z, Briggs SP, Vaughan MM, Alborn HT, Teal PE, Schmelz EA (2013) Plant elicitor peptides are conserved signals regulating direct and indirect antiherbivore defense. Proc Natl Acad Sci U S A. 110(14):5707-12. PMID: 23509266 Schmelz EA, Huffaker A, Carroll MJ, Alborn HT, Ali JG, Teal PE (2012) An amino acid substitution inhibits specialist herbivore production of an antagonist effector and recovers insect-induced plant defenses. Plant Physiol. 160(3):1468-78. PMID: 23008466 Anderson RG, Casady MS, Fee RA, Vaughan MM, Deb D, Fedkenheuer K, Huffaker A, Schmelz EA, Tyler BM, McDowell JM (2012) Homologous RXLR effectors from Hyaloperonospora arabidopsidis and Phytophthora sojae suppress immunity in distantly related plants. Plant J. doi: 10.1111/j.1365-313X.2012.05079.x. PMID: 22709376 Dafoe NJ, Huffaker A, Vaughan MM, Duehl AJ, Teal PE, Schmelz EA (2011) Rapidly induced chemical defenses in maize stems and their effects on short-term growth of Ostrinia nubilalis. J Chem Ecol. 37(9):984-91. PMID: 21833765 Huffaker A, Kaplan F, Vaughan MM, Dafoe NJ, Ni X, Rocca JR, Alborn HT, Teal PE, Schmelz EA (2011) Novel acidic sesquiterpenoids constitute a dominant class of pathogen-induced phytoalexins in maize. Plant Physiol. 156(4):2082-97. PMID: 21690302 Yamaguchi Y, Huffaker A (2011) Endogenous peptide elicitors in higher plants. Curr Opin Plant Biol. 14(4):351-7. PMID: 21636314 Schmelz EA, Kaplan F, Huffaker A, Dafoe NJ, Vaughan MM, Ni X, Rocca JR, Alborn HT, Teal PE (2011) Identity, regulation, and activity of inducible diterpenoid phytoalexins in maize. Proc Natl Acad Sci U S A. 108(13):5455-60. PMID: 21402917 Huffaker A, Dafoe NJ, Schmelz EA (2011) ZmPep1, an ortholog of Arabidopsis elicitor peptide 1, regulates maize innate immunity and enhances disease resistance. Plant Physiol. 155(3):1325-38. PMID: 21205619 Yamaguchi Y, Huffaker A, Bryan AC, Tax FE, Ryan CA (2010) PEPR2 is a second receptor for the Pep1 and Pep2 peptides and contributes to defense responses in Arabidopsis. Plant Cell. 22(2):508-22. PMID: 20179141 Ryan CA, Huffaker A, Yamaguchi Y (2007) New insights into innate immunity in Arabidopsis. Cell Microbiol. 9(8):1902-8. PMID: 17593247 Huffaker A, Ryan CA (2007) Endogenous peptide defense signals in Arabidopsis differentially amplify signaling for the innate immune response. Proc Natl Acad Sci U S A. 104(25):10732-6. PMID: 17566109 \n",
            "-------------------- Scraping faculty url 71/183 --------------------\n",
            "enfu hui  Biography Enfu Hui obtained his Ph.D. in 2009 from UW Madison. He then completed his postdoctoral training at UCSF as a fellow of the Leukemia & Lymphoma Society. He joined the Division of Biological Sciences at UCSD in July 2016 and was named as a Searle Scholar and Pew Scholar.  Research Ideally, the immune system identifies tumors as threatening elements and deploys immune cells (T cells) to find and kill them. However, many tumor cells have evolved to employ a protein called PD-L1 to blind T cells from carrying out their functions and evade immune defenses. PD-L1 protects tumor cells by activating a \"molecular brake\" or checkpoints known as PD-1 to stop T cells. Remarkably, drugs that block PD-1 or its ligand PD-L1 have proven to release the PD-1 brake from T cells, and demonstrated unprecedented clinical activities in a variety of human cancers. Still, durable response is limited to a small subset of cancer patients. Identification of reliable predictive biomarkers, and targeting other immune checkpoints, either alone or in combination with PD-1 inhibitors, hold the promise of extending the therapy to more tumor types and a larger populations of patients. However, these efforts are slowed down by the incomplete molecular understanding of immune checkpoints. We are a group of biochemists and cell biologists aiming to dissect the mechanisms of immune checkpoints, using cell-free membrane reconstitution, time-resolved live cell microscopy and cutting-edge cell biology approaches. We recently uncovered two novel aspects of PD-L1/PD-1 signaling. Intracellularly, we showed that the T cell costimulatory receptor CD28 is a primary target of PD-1 associated phosphatases. Extracellularly, we discovered that PD-L1, a key weapon of tumor cells, can be neutralized in cis by PD-1 expressed on the same cells. Now, we will continue to elucidate the PD-1 signaling pathway by identification of novel regulators and targets of PD-1. In addition, we are investigating whether and how additional immune brakes operate in conjunction with PD-1 to turn off T cell activity. We will identify the triggering molecules that turn these brakes on, and signal transducers these brakes recruit to suppress the immune response. Our findings could lead to the development of novel biomarkers and drug targets of cancer immunotherapy.  Select Publications Zhao, Y., Harrison, D. L., Song, Y., Ji, J., Huang, J., and Hui, E. Antigen-Presenting Cell-Intrinsic PD-1 Neutralizes PD-L1 in cis to Attenuate PD-1 Signaling in T Cells. Cell Rep 2018;24:379-90 e376. Hui, E., Cheung, J., Zhu, J., Su, X., Taylor, M. J., Wallweber, H. A., Sasmal, D. K., Huang, J., Kim, J. M., Mellman, I., and Vale, R. D. T cell costimulatory receptor CD28 is a primary target for PD-1-mediated inhibition. Science 2017;355, 1428-1433. Carbone, C. B., Kern, N., Fernandes, R. A., Hui, E., Su, X., Garcia, K. C., and Vale, R. D. In vitro reconstitution of T cell receptor-mediated segregation of the CD45 phosphatase. Proceedings of the National Academy of Sciences of the United States of America 2017;114, E9338-E9345. Su X, Ditlev JA, Hui E, Xing W, Banjade S, Okrut J, King DS, Taunton J, Rosen MK, Vale RD. Phase separation of signaling molecules promotes T cell receptor signal transduction. Science 2016;352(6285):595-9. doi: 10.1126/science.aad9964. Chang, C. W., Hui, E., Bai, J., Bruns, D., Chapman, E. R., and Jackson, M. B. A structural role for the synaptobrevin 2 transmembrane domain in dense-core vesicle fusion pores. The Journal of Neuroscience 2015;35 (14) 5772-80. Liu, H., Bai, H., Hui, E., Yang, L., Evans, C. S., Wang, Z., Kwon, S. E., and Chapman, E. R. Synaptotagmin 7 functions as a Ca2+-sensor for synaptic vesicle replenishment. Elife 2014;e01524. Hui E, Vale RD. In vitro membrane reconstitution of the T-cell receptor proximal signaling network. Nature structural & molecular biology . 2014;21(2):133-42. doi: 10.1038/nsmb.2762 Hui E, Gaffaney JD, Wang Z, Johnson CP, Evans CS, Chapman ER. Mechanism and function of synaptotagmin-mediated membrane apposition. Nature structural & molecular biology . 2011;18(7):813-21. doi: 10.1038/nsmb.2075. Zhang Z, Hui E, Chapman ER, Jackson MB. Regulation of exocytosis and fusion pores by synaptotagmin-effector interactions. Molecular biology of the cell . 2010;21(16):2821-31. doi: 10.1091/mbc.E10-04-0285. Zhang Z, Hui E, Chapman ER, Jackson MB. Phosphatidylserine regulation of Ca2+-triggered exocytosis and fusion pores in PC12 cells. Molecular biology of the cell . 2009;20(24):5086-95. doi: 10.1091/mbc.E09-08-0691. Hui E, Johnson CP, Yao J, Dunning FM, Chapman ER. Synaptotagmin-mediated bending of the target membrane is a critical step in Ca(2+)-regulated fusion. Cell . 2009;138(4):709-21. doi: 10.1016/j.cell.2009.05.049. Shahin V, Datta D, Hui E, Henderson RM, Chapman ER, Edwardson JM. Synaptotagmin perturbs the structure of phospholipid bilayers. Biochemistry . 2008;47(7):2143-52. doi: 10.1021/bi701879g. PubMed PMID: 18205405; PubMed Central PMCID: PMC3095487. Paddock BE, Striegel AR, Hui E, Chapman ER, Reist NE. Ca2+-dependent, phospholipid-binding residues of synaptotagmin are critical for excitation-secretion coupling in vivo. The Journal of Neuroscience 2008;28(30):7458-66. doi: 10.1523/JNEUROSCI.0197-08.2008. Gaffaney JD, Dunning FM, Wang Z, Hui E, Chapman ER. Synaptotagmin C2B domain regulates Ca2+-triggered fusion in vitro: critical residues revealed by scanning alanine mutagenesis. The Journal of Biological Chemistry . 2008;283(46):31763-75. doi: 10.1074/jbc.M803355200. Chicka MC, Hui E, Liu H, Chapman ER. Synaptotagmin arrests the SNARE complex before triggering fast, efficient membrane fusion in response to Ca2+. Nature structural & molecular biology . 2008;15(8):827-35. doi: 10.1038/nsmb.1463. Hui E, Bai J, Chapman ER. Ca2+-triggered simultaneous membrane penetration of the tandem C2-domains of synaptotagmin I. Biophysical Journal 2006;91(5):1767-77. doi: 10.1529/biophysj.105.080325. Czibener, C., Sherer, N. M., Becker, S. M., Pypaert, M., Hui, E., Chapman, E. R., Mothes, W., and Andrews, N. W. Ca2+ and synaptotagmin VII-dependent delivery of lysosomal membrane to nascent phagosomes. J Cell Biol 2006;174, 997-1007. Hui E, Bai J, Wang P, Sugimori M, Llinas RR, Chapman ER. Three distinct kinetic groupings of the synaptotagmin family: candidate sensors for rapid and delayed exocytosis. Proceedings of the National Academy of Sciences of the United States of America. 2005;102(14):5210-4. doi: 10.1073/pnas.0500941102. \n",
            "-------------------- Scraping faculty url 72/183 --------------------\n",
            "tony hunter \n",
            "-------------------- Scraping faculty url 73/183 --------------------\n",
            "terry hwa  Biography Dr. Terence Hwa is the Presidential Chair Professor with a joint appointment in the Department of Physics at U.C. San Diego. He is the inaugural Director of the Ph.D. Specialization in Quantitative Biology at U.C. San Diego.  Research Current activities in our lab are in the area of quantitative and systemic biology. This is an emerging area of research at the interface of biology, biochemistry, engineering, and physics. In this post-genome era, it is clear that the complexity of a biological organism resides not merely in the intricacies of its components (e.g., proteins), but more importantly in the array of interactions these components can have with each other. This shift of paradigm is initiating a revolution in biology: Instead of focusing on genes, proteins, and pathways, bioloigsts are beginning to think in terms of modules and networks. However, a significant challenge lies in the fact that much of the interactions of the subsystems are yet unknown. Our research is focused on microbiology, particularly on the bacterium E. coli which is the best characterized living organism both molecularly and physiologically. In the long run, we wish to develop a comprehensive understanding of the organism, i.e., understanding its physiological response under various growth conditions in molecular terms. However even for E. coli, a global approach seems premature in our opinion due to the many missing molecular players and interactions. Thus, we work on smaller sub-systems, with the insistence on developing quantitative links from molecules all the way to cell physiology. Our emphasis can be regarded as the \"vertical approach\", which complements the \"horizontal approach\" favored by the 'omics technology. An introduction to our vertical, quantitative approach is described in a recent course on quantitative molecular biology developed by our laboratory. Vertical integration requires a command of the vast knowledge accumulated over many decades of research in molecular and microbiology, as well as a combination of experimental and theoretical efforts. Some of the experiments are being carried out in our own lab; others are done together with biology collaborators. Our experiments are often theory-motivated, yet discoveries made in the lab has inspired fresh theoretical view points. In similar ways, our theoretical studies play the dual role of analyzing experimetal results and guiding new generation of experiments. The close interaction between theory and experiment allows the theorists to be in contact with biological reality, and allows the experimentalists to have a good sense of the power (and limitations) of quantitative analysis. In this way, we hope to train a new generation of scientists who can freely exploit opportunities at the interface between biology and physics. Below are some topics our lab has been working in recent years; all projects involve experimental and theoretical components. gene regulation: quantitative characterization of combinatorial transcriptional control; de novo synthesis of novel transcriptional control; quantitative characterization of post-transriptional control by regulatory small RNA; protein-protein interaction: specificity and cross-talk in two-component signaling; assembly of macromolecular complexes; molecular evolution: directed evolution of promoters, cis-regulatory sequences, and signaling molecules; analysis of evolved seqeunces to deduce rules of molecular interaction; genetic circuits: quantitative characterization of various endogenous genetic circuits; synthesis and characterization of artificial circuits; metabolic control: interaction and coordination of metabolic pathways; systems being studied include carbon/nitrogen metabolism, aerobic/anarobic control, Fe/oxidative stress; physiology & cell growth:coordination of growth and metabolic activities; control of ribosome synthesis and cell growth; control of cell division.  Select Publications David Erickson, Severin Schink, Vadim Patsalo, James R. Williamson, Ulrich Gerland, Terence Hwa. A global resource allocation strategy governs growth transition kinetics of E. coli, Nature 551: 119123 (2017). Jonas Cremer, Markus Arnoldini, Terence Hwa. Effect of water flow and chemical environment on microbiota growth and composition in the human colon, PNAS 114: 64538-43 (2017). Markus Basan, Sheng Hui, Hiroyuki Okano, Zhongge Zhang, Yang Shen, James R. Williamson, Terence Hwa. Overflow metabolism in E. coli results from efficient proteome allocation, Nature 528: 99-104 (2015). J. Barrett Deris, Minsu Kim, Zhongge Zhang, Hiroyuki Okano, Rutger Hermsen, Alexander Groisman, Terence Hwa The innate growth bistability of antibiotic resistant bacteria. Science 342: 1237435 (2013). M. Scott, C. W. Gunderson, E. Mateescu, Z. Zhang and T. Hwa Interdependence of cell growth and gene expression: origins and consequences, Science 330: 1099-1102 (2010). \n",
            "-------------------- Scraping faculty url 74/183 --------------------\n",
            "juan carlos izpisua belmonte \n",
            "-------------------- Scraping faculty url 75/183 --------------------\n",
            "xin jin \n",
            "-------------------- Scraping faculty url 76/183 --------------------\n",
            "yishi jin  Biography Dr. Jin received her B.S. degree from Peking University, China, and her Ph.D. from the University of California, Berkeley. She completed her postdoctoral training at MIT.  Research The Jin lab research focuses on the molecular genetic mechanisms underlying the development and function of the nervous system using the nematode Caenorhabditis elegans . The transparency, defined anatomy, and rapid life cycle of this organism greatly facilitate our studies at the subcellular resolution. Moreover, the entre cell lineage and connectome are known, enabling functional understanding at deep levels. Through forward genetic screening in combination with multi-layered molecular and cellular manipulations, we are discovering key molecules that play conserved roles in synapse formation, maintenance, and function, as well as those underlying adult axon regeneration. Our ultimate goal is to connect the studies of basic mechanisms to the understanding of human neurological disorders and neuronal repair. How do neurons form synapses with their partners? Synapses are unique structures that neurons use to communicate with others. At the presynaptic terminal, neurons develop elaborate subcellular organizations to facilitate the accumulation of synaptic vesicles and their release (Kittelmann et al., 2013). We label different components of presynaptic terminals in vivo using fluorescent markers. Through forward genetic screening, we have identified multiple signaling pathways that involve conserved scaffolding proteins, MAP kinases, GTPases, and ubiquitin-mediated protein degradation to specify spatial domains of the presynaptic terminals (Nakata et al., Cell 2005; Grill et al., Neuron, 2007; Yan et al., Cell 2009; Kittelmann et al ., 2013). Synapses also undergo remodeling as animals grow. Our recent studies have elucidated roles of dynamic microtubule cytoskeleton in synaptic remodeling (Kurup et al., Current Biology, 2015). Furthermore, combining genomics with genetics, we have discovered unexpected pathways linking synapse development to nuclear pre-mRNA regulation (Van Epps et al., Development, 2010; Chen et al., Genes and Development, 2015). How do neural circuits operate to control movement? C. elegans movement, namely locomotion, depends on balanced excitation and inhibition to body muscles. The locomotion circuit consists excitatory cholinergic and inhibitory GABAergic motor neurons. We have examined the roles of neuronal ACh receptors in modulating the excitation state of these two types of neurons (Jospin et al., PLoSBiol, 2009). Our analysis of genetic mutations also led us to establish a novel model for epilepsy (Stawicki et al., Current Biology, 2011; Stawicki et al., PLoSGenetics, 2013; Zhou et al., eLife, 2013). In our recent studies, we have discovered novel inter-tissue interactions in the regulation of neuronal circuits, and identified an immunoglobulin protein that plays instructive roles to control synapse numbers through phagocytosis (Cherra et al., Neuron, 2016). How do adult axons regenerate? Understanding how neural circuit forms during normal development has direct implications to the process of how injured neurons repair and regain function in adult animals. Using an ultrafast laser-based microsurgery procedure to perform axotomy in C. elegans neurons (Yanik et al., Nature 2004), we have identified numerous pathways regulating the rate and accuracy of adult axon regeneration (Chen et al., Neuron, 2011). Our current studies focus on the roles of the conserved MAPKKK DLK-1 and the microtubule regulator EFA-6 in axon regeneration (Yan et al., Cell 2009; Yan and Jin, Neuron, 2012; Chen et al., Neuron, 2011; Chen et al.,, eLife, 2015). By advancing our knowledge of intrinsic signaling pathways, we hope to develop new strategies to enhance regrowth ability of injured axons. Novel applications of optogenetic methodology: Optogenetic manipulation of cells and molecules using genetically encoded tags is revolutionizing neurobiology. Mini Singlet Oxygen Generator (miniSOG) is a genetically encoded photosensitizer, created by R. Tsien's lab (Shu et al., PLoS Biology, 2011). We have employed the ability of inducible ROS generation by miniSOG to develop innovative optogenetic tools to perturb neural circuit (Qi et al., PNAS 2012) and to perform genome wide random mutagenesis (Noma and Jin, Nat. Comm. 2015).  Select Publications Kim, K., Tang, N-H., Andrusiak, M., Wu, Z., Chisholm, A.D, and Jin, Y. (2018). A Neuronal PiRNA pathway inhibits axon regeneration in C. elegans. Neuron. Published online: January 30, 2018 Noma, K., Goncharov, A., Ellisman, M., and Jin, Y. (2017) Microtubule-dependent ribosome localization in C. elegans neurons. eLife. 6. pii: e26376. doi: 10.7554/eLife.26376. Cherra, S. J., and Jin, Y. (2016) A Two-Immunoglobulin-Domain Transmembrane Protein Mediates an Epidermal-Neuronal Interaction to Maintain Synapse Density. Neuron, pii: S0896-6273(15)01126-5. Noma, K., and Jin, Y. (2015) Optogenetic mutagenesis in Caenorhabditis elegans. Nature Communications, Dec 3;6:8868. Chen, F., Zhou, Y., Qi, Y. B. Khivansara, V., Li, H., Chun, S. Y., Kim, J. K., Fu, X., and Jin, Y. (2015) Context-dependent modulation of Pol II CTD phosphatase SSUP-72 regulates alternative polyadenylation in neuronal development. Genes & Development, 29:2377-90. Chen, L., Chuang, M., Koorman, T., Boxem, M., Jin, Y.*, and Chisholm, A.D*. (2015) Axon injury triggers EFA-6 mediated destabilization of axonal microtubules via TACC and Doublecortin-like kinase. eLife Sep 4;4. Kurup, N., Yan, D., Goncharov, A., and Jin, Y. (2015). Dynamic microtubules drive circuit rewiring in the absence of neurite remodeling. Curr Biol. 25(12):1594-605. Qi, B. Y., Garren, E., Shu, X., Tsien, R. Y., and Jin, Y. (2012). Photo-inducible cell ablation in C. elegansusing the genetically encoded singlet oxygen generating protein miniSOG. Proc Natl Acad Sci U S A. 109:7499-504 Stawicki, T.M., Zhou, K., Yochem J., Chen, L., and Jin, Y. (2011). TRPM channels modulate epileptic-like convulsions via systemic ion homeostasis. Current Biology 21(10):883-888. Yan, D., Wu, Z., Chisholm, A. D., and Jin, Y. (2009). The DLK-1 kinase promotes mRNA stability and local translation in synapses and axon regeneration. Cell 138:1005-1018. Nakata, K., Abrams, B., Grill, B., Goncharov, A., Huang, X., Chisholm, A. D., and Jin, Y. (2005). Regulation of a DLK-1 and p38 MAP kinase pathway by the ubiquitin ligase RPM-1 is required for presynaptic development. Cell 120:407-420. Yanik, M. F., Cinar, H., Cinar, H. N., Chisholm, A. D., Jin, Y., and Ben-Yakar, A. (2004). Neurosurgery: functional regeneration after laser axotomy. Nature 432:822. Zhen, M., Huang, X., Bamber, B., and Jin, Y. 2000. Regulation of presynaptic terminal organization by C. elegans RPM-1, a putative guanine nucleotide exchanger with a Ring-H2 finger domain. Neuron 26:331-343. Zhen, M., and Jin, Y. 1999. The liprin protein SYD-2 regulates the differentiation of presynaptic termini in C. elegans. Nature 401:371-375. Hallam, S.J., and Jin, Y. 1998 lin-14 regulates the timing of synaptic remodelling in Caenorhabditis elegans. Nature 395:78-82. Relevant Reviews Jin, Y. and Qi, Y. B. (2017). Building stereotypic connectivity: mechanistic insights into neuronal structural plasticity in C. elegans. Curr Opin Neurobiol. 48:97-105. He, Z., and Jin, Y. (2016) Intrinsic Control of Axon Regeneration: Mechanisms and Applications. Neuron, 90:437-51. PMID 27151637. Andrusiak A., and Jin, Y. (2016) Context specificity of stress MAP Kinase signaling: the story as told by C. elegans. J. Biol. Chem., Feb 23. pii: jbc.R115.711101. Kim, K. W., and Jin, Y. (2015). Neuronal responses to stress and injury in C. elegans. FEBS Lett. 589(14):1644-1652. Sharifnia, P., and Jin, Y. (2015). Regulatory roles of RNA binding proteins in the nervous system of C. elegans. Front Mol Neurosci. 7:100. Cherra, S., and Jin, Y. (2015). Adances in synapse formation: Forging connections in the worm. WIRES Developmental Biology. 4(2):85-97. \n",
            "-------------------- Scraping faculty url 77/183 --------------------\n",
            "katherine jones \n",
            "-------------------- Scraping faculty url 78/183 --------------------\n",
            "ashley juavinett  Biography Ashley Juavinett received her Ph.D. from UC San Diego and was a Postdoctoral Fellow at Cold Spring Harbor Laboratory. She returned to UC San Diego as an Assistant Teaching Professor in 2018.  Research My research and writing focuses broadly on education and careers in the field of neuroscience. Im interested in capitalizing on open access datasets as a course-based undergraduate research experience. We are developing scaffolds for such datasets that will enable students to ask their own questions of cutting edge data. I am interested in whether these approaches will create realistic, engaging laboratory experiences and ultimately invite diverse students into the field of neuroscience.  Select Publications Juavinett, A.L., Erlich, J., Churchland, A.C. (2018). Decision-making behaviors: weighing ethology, complexity, and sensorimotor compatibility. Current Opinion Neurobiology 49:42-50. PMID: 29179005. Juavinett, A.L., Nauhaus, I., Garrett, M.G., Callaway, E.M. (2017). Automated identification of mouse visual areas with intrinsic signal imaging. Nature Protocols 12(1):32-43. PMID: 27906169 Kim. E.J., Juavinett, A.L., Kyubwa, E.M., Jacobs, M.W., Callaway, E.M. (2015). Three Types of Cortical Layer 5 Neurons That Differ in Brain-wide Connectivity and Function. Neuron 88(6): 1253-67. doi: 10.1016/j.neuron.2015.11.002. Juavinett, A.L., Callaway, E.M. (2015). Pattern and Component Motion Responses in Mouse Visual Cortex. Current Biology 25(13): 1759-64. doi: 10.1016/j.cub.2015.05.028. Stewart, J.L., Juavinett, A.L., May, A.C., Davenport, P.W., Paulus, M.P. (2015). Do You Feel Alright? Attenuated Neural Processing of Aversive Interoceptive Stimuli in Current Stimulant Users. Psychophysiology 52(2): 249-62. doi: 10.1111/psyp.12303 Pineda, J.A., Juavinett, A.L., Datko, M. (2012). Self-regulation of brain oscillations as a treatment for aberrant brain connections in children with autism. Medical Hypotheses 79(6): 790-8. \n",
            "-------------------- Scraping faculty url 79/183 --------------------\n",
            "suckjoon jun  Biography Suckjoon Jun received his Ph.D. in theoretical physics from Simon Fraser University in Vancouver, Canada. He continued to work as a theorist for his post-doctoral training at the FOM-Institute AMOLF in Amsterdam and briefly at Necker Hospital in Paris with NSERC (Canada) and Marie Curie (Europe) post-doctoral fellowships. He was a Bauer Fellow at Harvard University between 2007-2012 before moving to UCSD, during which he completely switched to experimental physical and systems biology.  Research The driving force of our lab is the curiosity and desire to understand why the biological systems are the way they are. We are often attracted to long-standing questions in biology, for which quantitative, inter- and multi-disciplinary approaches are essential. Some of these questions include: What is the driving force underlying chromosome segregation in bacteria? What is the relationship between growth and cell cycle? What is the origin of the plasticity of cell shape? While our research is naturally question-driven, we also develop tools when necessary. In doing so, we try to meet the highest standard in the relevant discipline (be it physics, biology, or engineering). We value both logic and intuition, but more of \"night science\" than \"day science.\"  Select Publications Fangwei Si, Guillaume Le Treut, John T Sauls, Stephen Vadia, Petra Anne Levin, Suckjoon Jun. Mechanistic origin of cell-size control and homeostasis in bacteria ( Current Biology , under review) Fabai Wu, Pinaki Swain, Louis Kuijpers, Xuan Zheng, Kevin Felter, Margot Guurink, Jacopo Solari, Suckjoon Jun, Thomas S. Shimizu, Debasish Chaudhuri, Bela Mulder & Cees Dekker. Cell boundary confinement sets the size and position of the E. coli chromosome. ( Cell , under review) K. Sekar, R. Ruscon, JT Sauls, T. Fuhrer, E. Noor, J. Nguyen, VI Fernandez, MF Buffing, M. Berney, S. Jun, R. Stocker, U. Sauer. \"Synthesis and degradation of FtsZ quantitatively predicts the first cell division in starved bacteria\". Mol. Sys. Biol. , (2018) 14, e8623 Arshad Desai and Suckjoon Jun. Promoting an \"Auteur Theory\" for Young Scientists: Preserving Excitement and Creativity\". BioEssays , 1800147 (2018) Suckjoon Jun, Fangwei Si, Rami Pugatch, Mathew Scott. Fundamental Principles in Bacterial Physiology - History, Recent progress, and the Future with Focus on Cell Size Control: A Review. Reports on Progress in Physics , 81 056601 (2018) Suckjoon Jun and Michael Rust. A fundamental unit of cell size in bacteria. TRENDS in Genetics 33(7), 433-435 (2017) Fangwei Si*, Dongyang Li*, Sarah E. Cox, John T. Sauls, Omid Azizi, Cindy Sou, Amy B. Schwartz, Michael J. Erickstad, Yonggun Jun, Xintian Li, Suckjoon Jun. Invariance of the initiation mass and predictability of cell size in Escherichia coli. Current Biology 27, 12781287 (2017) Xin-tian Li, Yonggun Jun, Michael J. Erickstad, Steven D. Brown, Adam Parks, Donald L. Court & Suckjoon Jun. tCRISPRi: tunable and reversible, one-step control of gene expression. Scientific Reports 6, 39076 (2016) John T Sauls, Dongyang Li, Suckjoon Jun Adder and quantitative, phenomenological approach to cell size homeostasis in bacteria. Current Opinion in Cell Biology , 38:38-44 (2016) Steven B. Brown and Suckjoon Jun. Complete Genome Sequence of Escherichia coli NCM3722. Genome Announcements , 3(4):e00879-15 (2015) Sattar Taheri-Araghi, Steven B. Brown, John T. Sauls, Dustin McIntosh, & Suckjoon Jun. Single-cell physiology. Annual Review of Biophysics 44:3.1-3.20 (2015) Suckjoon Jun. Chromosome, cell cycle, and entropy. Biophysical Journal 108, 785-786 (2015) Sattar Taheri-Araghi*, Serena Bradde*, Norbert Hill, Petra A. Levin, Johan Paulsson, Massimo Vergassola*, Suckjoon Jun*. Cell size control and homeostasis in bacteria. Current Biology 25, 385-391 (2015) Suckjoon Jun and Sattar Taheri-Araghi. Cell-size maintenance: universal strategy revealed. TRENDS in Microbiology 23, 4-6 (2015) Ariel Amir, Farinaz Babaeipour, Dustin McIntosh, David R. Nelson & Suckjoon Jun. Bending forces plastically deform growing bacterial cell walls. Proc. Nat. Acad. Sci. USA (2014) Brenda Youngren, Henrik J. Nielsen, Suckjoon Jun & Stuart Austin. The multi-fork Escherichia coli chromosome is a self-duplicating and self-segregating thermodynamic ring polymer. Genes & Development . 28, 71-84 (2014) James Pelletier and Suckjoon Jun. Isolation and characterization of bacterial nucleoids in microfluidic devices. The Bacterial Nucleoid. Methods in Molecular Biology , vol 1624. Humana Press, New York, NY Sattar Taheri & Suckjoon Jun. Single-cell cultivation in microfluidic devices. The Handbook of Hydrocarbon and Lipid Microbiology (Springer, 2016) \n",
            "-------------------- Scraping faculty url 80/183 --------------------\n",
            "james kadonaga  Biography Jim Kadonaga was an undergraduate in Chemistry at MIT, where he received the Alpha Chi Sigma Prize as well as the American Institute of Chemists Certificate. He carried out his graduate studies with Jeremy R. Knowles in the Department of Chemistry at Harvard University, where he was a DuPont Fellow. Jim was a postdoctoral associate with Robert Tjian at UC Berkeley as a Fellow of the Miller Institute for Basic Research in Science, American Cancer Society California Division, and Lucille P. Markey Charitable Trust. He joined the UCSD faculty in 1988 and was one of 15 scientists to be named as a Presidential Faculty Fellow by President George H.W. Bush in 1992. In 1994, he was elected to Fellow of the American Association for the Advancement of Science, and in 1995, he was elected to Fellow of the American Academy of Microbiology. He served as the Chair of the Section of Molecular Biology from 2003 to 2007. In 2012, he received the UCSD Chancellor's Associates Award for Excellence in Research in Science and Engineering. In 2017, he was elected to the American Academy of Arts & Sciences. Jim is presently the Amylin Endowed Chair and a Distinguished Professor in the Section of Molecular Biology. Profile in J. Cell Biol.  Research Transcriptional Regulation and Chromatin Dynamics A diverse range of biological phenomena, including many human diseases, can be traced to the proper or improper expression of a gene. New insights into the mechanisms of gene expression will have a broad impact upon science and society. The objective of our work is to understand the regulation of RNA polymerase II transcription. We study the transcription process as well as the chromatin template to decipher the mechanisms by which genes are regulated in animals. Development of Sequence-specific DNA Affinity Chromatography & the Purification and Cloning of Transcription Factor Sp1 Prior to my arrival at UCSD, I was involved in early studies of eukaryotic sequence-specific DNA-binding transcriptional factors as a postdoctoral researcher in the laboratory of Robert Tjian at UC Berkeley. In this work, I developed sequence-specific DNA affinity chromatography for the purification of transcription factor Sp1 (Fig. 1). The purification of Sp1 enabled me to isolate its corresponding cDNA and to characterize its binding to DNA via three zinc finger motifs (Kadonaga et al., 1987). Sequence-specific DNA affinity chromatography has since been used by many investigators to purify numerous factors that function in a wide range of biological processes. The remainder of the work below was carried out in my lab at UCSD. The RNA Polymerase II Core Promoter In the area of transcription, we have focused on the RNA polymerase II core promoter. The core promoter is the stretch of DNA that directs transcription initiation, and it typically comprises sequences from about -40 to +40 nt relative to the +1 transcription start site. The core promoter is the gateway to transcription, as it is the site at which the signals that activate transcription ultimately converge. Core promoters are diverse in terms of their structure and function. The transcriptional activity of core promoters is driven by DNA sequence motifs such as the TATA box, initiator (Inr), TCT motif, downstream core promoter element (DPE), motif ten element (MTE), and others (Fig. 2). Any particular core promoter will have one or more core promoter motifs. There are, however, no universal core promoter elements. In addition, there are core promoter motifs that have yet to be discovered. We study core promoters at three levels: DNA sequence motifs, transcription factors, and biological networks. (1). DNA sequence motifs. We have identified and characterized new core promoter elements, such as the DPE and MTE, and articulated the nature and properties of the TCT motif. We are continuing to search for new core promoter elements. We also seek to gain a better understanding of the known elements. (2). Transcription factors. In Drosophila , we have found that TATA-less transcription with TCT-dependent core promoters does not require TBP (TATA box-binding protein), but is instead dependent upon TRF2 (TBP-related factor 2). We also discovered that Caudal protein (a DNA-binding transcription factor that is a key regulator of the Hox genes) is a DPE-specific activator. This finding is a specific example of the concept that some transcriptional enhancers are specific for core promoters with DPE or TATA motifs (Fig. 3). More generally, we seek to understand the different sets of transcription factors that function at different types of core promoters. (3). Biological networks. We found that some core promoter motifs have specific functions in biological networks the DPE with the Hox gene network and the TCT motif with the ribosomal protein gene network (Fig. 4). Notably, in Drosophila, both the DPE and TCT motifs function with TRF2 rather than TBP. Thus, these findings have also led us to investigate the evolution of TRF2 (Fig. 5). It is important to understand the function of the core promoter in a broader biological context. Our current studies include the identification and analysis of new core promoter motifs and structures, particularly in humans. We are also investigating the distinct mechanisms of transcription from different types of core promoters as well as the basis of enhancer-core promoter specificity. The core promoter continues to be a rich source of new discoveries. Early Studies of the Role of Chromatin in the Regulation of Transcription We carried out early experiments on the role of chromatin structure in the regulation of transcription by RNA polymerase II. These studies were initiated in the late 1980s this was a time when many people in the transcription field felt that chromatin was unimportant. We were primarily motivated to perform this work because chromatin is the natural state of DNA in the eukaryotic nucleus. We found that sequence-specific DNA-binding transcription factors act primarily to counteract chromatin-mediated repression we termed this phenomenon \"antirepression\" (Laybourn and Kadonaga, 1991, 1992; Paranjape et al., 1994). This paradigm was in contrast to the prevailing belief that sequence-specific factors increase the rate of the transcription reaction in a manner that is independent of chromatin structure. These studies of chromatin and transcription led to the model for transcriptional activation that is shown in Fig. 6 (Pazin et al., 1994). In this project, we were also able to achieve ligand-regulated transcription in vitro by the estrogen receptor by using chromatin templates (Kraus and Kadonaga, 1998). Today, it is generally accepted that sequence-specific activators counteract chromatin-mediated repression of basal (unactivated) transcription, and chromatin is extensively studied as Chromatin Assembly and Dynamics Chromatin assembly is important for processes, such as DNA replication, transcription, and repair, that involve the disruption and reformation of nucleosomes. Our studies of chromatin assembly began with the development of a crude extract from Drosophila embryos, termed the S-190, that is competent for ATP-dependent chromatin assembly. We then undertook the biochemical fractionation and purification of the essential components in the S-190 extract. This work included the discovery, purification, and cloning of ACF (ATP-utilizing chromatin assembly and remodeling factor; Ito et al., 1997, 1999) and the discovery that Asf1 is a chromatin assembly factor that serves as a chaperone for newly synthesized histones H3 and H4 (Tyler et al., 1999). The analysis of the mechanism of the ATP-dependent assembly of periodic nucleosome arrays led to the model shown in Fig. 7. With purified and defined factors, we have been studying the mechanism of chromatin assembly. This work led to the discovery of the prenucleosome, which is a stable conformational isomer of the nucleosome that associates with approximately 80 bp DNA (Fig. 8). Prenucleosomes are formed rapidly upon the deposition of histones onto DNA, and can be converted into canonical nucleosomes by an ATP-driven chromatin assembly factor such as ACF. Moreover, p300 acetylates histone H3K56 in prenucleosomes but not in nucleosomes. Multiple independent lines of evidence suggest that prenucleosomes or prenucleosome-like particles are at active chromatin such as promoters (Fig. 9). First, there is a strong correlation between psoralen-crosslinked species observed with prenucleosomes assembled in vitro and the active yeast PHO5 promoter in vivo. Second, chromatin particles with prenucleosome-sized DNA and core histones are observed at the upstream region of active promoters in mouse cells. Third, prenucleosomes, but not nucleosomes, can be acetylated at H3K56 by p300, and H3K56 acetylation occurs at active promoters and enhancers in yeast, Drosophila , and humans. It is possible that the transcriptional machinery is optimized to function in conjunction with a prenucleosome. More generally, prenucleosomes or prenucleosome-like particles are observed at active chromatin. In other words, prenucleosomes are generated in dynamic chromatin at which there is active disassembly and reassembly of nucleosomes. Annealing Helicases Our studies of chromatin dynamics took an unexpected turn in the course our studies of the HARP protein (also known as SMARCAL1). We were interested in HARP because it is in the same family of ATPases (the SNF2-like family) as the chromatin assembly proteins ISWI (the ATPase subunit of ACF) and Chd1. In addition, mutations in HARP were found to be responsible for a rare human disorder known as Schimke immuno-osseous dysplasia (SIOD). We thus embarked on the analysis of HARP with the notion that it might alter chromatin structure; however, HARP did not exhibit chromatin remodeling activity. Instead, we discovered that HARP possesses a novel annealing (reverse) helicase activity. In other words, HARP is a motorized molecular zipper that rewinds DNA, such as single DNA strands that are bound by RPA, the major single-stranded DNA-binding protein in the nucleus (Fig. 10). In addition, we found a second annealing helicase termed annealing helicase 2 (AH2, also known as ZRanB3). Unlike HARP, which forms a stable complex with RPA, AH2 does not bind to RPA. More generally, in these studies, we have identified a DNA structure-modifying activity that acts in opposition to the DNA-unwinding activities of helicases and polymerases. NDF, a Nucleosome Destabilizing Factor that Facilitates Transcription through Nucleosomes Our studies of prenucleosomes led us to examine factors that mediate the destabilization or disassembly of conventional nucleosomes. To this end, we devised a nucleosome destabilization assay that was based on the ability of p300 to acetylate histone H3K56 in a partially unwrapped nucleosome. By using this assay, we identified a nucleosome destabilizing activity in Drosophila embryo extracts, and then purified this activity by conventional chromatography. This work led to the identification of a factor that we term NDF, for nucleosome destabilizing factor. NDF is present in most animals, and perhaps in plants, but it does not appear to have a homolog in Saccharomyces cerevisiae . NDF has a conserved PWWP motif, interacts with nucleosomes near the dyad, destabilizes nucleosomes in an ATP-independent manner, and facilitates transcription by RNA polymerase II through nucleosomes in a purified and defined transcription system as well as in cell nuclei (Fig. 11). Upon transcriptional induction, NDF is recruited to the transcribed regions of thousands of genes and co-localizes with a subset of H3K36me3-enriched regions (Fig. 12). Notably, the recruitment of NDF to gene bodies is accompanied by an increase in the transcript levels of many of the NDF-enriched genes. In addition, the global loss of NDF results in a decrease in the RNA levels of many genes. In humans, NDF is present at high levels in all tested tissue types, is essential in stem cells, and is frequently overexpressed in breast cancer. These findings indicate that NDF is a nucleosome destabilizing factor that is recruited to gene bodies during transcriptional activation and facilitates RNA polymerase II transcription through nucleosomes. The properties of NDF indicate that it has a key role in the transcription of chromatin by RNA polymerase II in animals. The further analysis of NDF should reveal additional factors and mechanisms that orchestrate the procession of RNA polymerase II through active genes.  Select Publications Fei, J., Ishii, H., Hoeksema, M. A., Meitinger, F., Kassavetis, G. A., Glass, C. K., Ren, B., and Kadonaga, J. T. (2018). NDF, a nucleosome destabilizing factor that facilitates transcription through nucleosomes. Genes Dev. 32, 682-694. PMCID: PMC6004073 Khuong, M.T., Fei, J., Cruz-Becerra, G., and Kadonaga, J.T. (2017). A simple and versatile system for the ATP-dependent assembly of chromatin. J. Biol. Chem. 292, 19478-19490. PMCID: PMC5702684 Vo ngoc, L., Wang, Y.-L., Kassavetis, G.A., and Kadonaga, J.T. (2017). The punctilious RNA polymerase II core promoter. Genes Dev. 31, 1289-1301. PMCID: PMC5580651 Vo ngoc, L., Cassidy, C.J., Huang, C.Y., Duttke, S.H.C., and Kadonaga, J.T. (2017). The human initiator is a distinct and abundant element that is precisely positioned in focused core promoters. Genes Dev. 31, 6-11. PMCID: PMC5287114 Fei, J., Torigoe, S.E., Brown, C.R., Khuong, M.T., Kassavetis, G.A., Boeger, H., and Kadonaga, J.T. (2015). The prenucleosome, a stable conformational isomer of the nucleosome. Genes Dev. 29 , 2563-2575. PMCID: PMC4699385 Khuong, M.T., Fei, J., Ishii, H., and Kadonaga, J.T. (2015). Prenucleosomes and active chromatin. Cold Spring Harbor Symp. Quant. Biol. 80 , in press. doi: 10.1101/sqb.2015.80.027300 Duttke, S.H.C., Lacadie, S.A., Ibrahim, M.M., Glass, C.K., Corcoran, D.L., Benner, C., Heinz, S., Kadonaga, J.T., and Ohler, U. (2015). Human promoters are intrinsically directional. Mol. Cell 57 , 674-684. Co-corresponding authors. PMCID: PMC4336624 Ishii, H., Kadonaga, J. T. , and Ren, B. (2015). MPE-seq, a new method for the genome-wide analysis of chromatin structure. Proc. Natl. Acad. Sci. USA 112 , E3457-E3465. Co-corresponding authors. PMCID: PMC4500276 Duttke, S. H. C., Doolittle, R. F., Wang, Y.-L., and Kadonaga, J. T. (2014). TRF2 and the evolution of the bilateria. Genes Dev. 28 , 2071-2076. PMCID: PMC4180970 Wang, Y.-L., Duttke, S. H. C., Chen, K., Johnston, J., Kassavetis, G. A., Zeitlinger, J., and Kadonaga, J. T. (2014). TRF2, but not TBP, mediates the transcription of ribosomal protein genes. Genes Dev. 28 , 1550-1555. PMCID: PMC4102762 Torigoe, S. E., Patel, A., Khuong, M. T., Bowman, G. D., and Kadonaga, J. T. (2013). ATP-dependent chromatin assembly is functionally distinct from chromatin remodeling. eLife 2 , e00863. PMCID: PMC3748710 Kadonaga, J. T. (2012). Perspectives on the RNA polymerase II core promoter. WIREs Dev. Biol. 1 , 40-51. PMCID: PMC3695423 Torigoe, S. E, Urwin, D. L., Ishii, H., Smith, D. E., and Kadonaga, J. T. (2011). Identification of a rapidly formed non-nucleosomal histone-DNA intermediate that is converted into chromatin by ACF. Mol. Cell 43 , 638-648. PMCID: PMC3160715 Yusufzai, T., and Kadonaga, J. T. (2010). Annealing helicase 2 (AH2), a DNA-rewinding motor with an HNH motif. Proc. Natl. Acad. Sci. USA 107 , 20970-20973. PMCID: PMC3000258 Parry, T. J., Theisen, J. W. M., Hsu, J.-Y., Wang, Y.-L., Corcoran, D. L., Eustice, M., Ohler, U., and Kadonaga, J. T. (2010). The TCT motif, a key component of an RNA polymerase II transcription system for the translational machinery. Genes Dev. 24 , 2013-2018. PMCID: PMC2939363 Juven-Gershon, T., and Kadonaga, J. T. (2010). Regulation of gene expression via the core promoter and the basal transcriptional machinery. Dev. Biol. 339, 225-229. PMCID: PMC2830304 Yusufzai, T., Kong, X., Yokomori, K., and Kadonaga, J. T. (2009). The annealing helicase HARP is recruited to DNA repair sites via an interaction with RPA. Genes Dev. 23 , 2400-2404. PMCID: PMC2764493 Rattner, B. P., Yusufzai, T., and Kadonaga, J. T. (2009). HMGN proteins act in opposition to ATP-dependent chromatin remodeling factors to restrict nucleosome mobility. Mol. Cell 34 , 620-626. PMCID: PMC2709789 Yusufzai, T., and Kadonaga, J. T. (2008). HARP is an ATP-driven annealing helicase. Science 322 , 748-750. PMCID: PMC2587503 Juven-Gershon, T., Hsu, J.-Y., and Kadonaga, J. T. (2008). Caudal, a key developmental regulator, is a DPE-specific transcription factor. Genes Dev. 22 , 2823-2830. PMCID: PMC2569877 Hsu, J.-Y., Juven-Gershon, T., Marr, M. T., 2nd, Wright, K. J., Tjian, R., and Kadonaga, J. T. (2008). TBP, Mot1, and NC2 establish a regulatory circuit that controls DPE-dependent versus TATA-dependent transcription. Genes Dev. 22, 2353-2358. PMCID: PMC2532932 Santoso, B., and Kadonaga, J. T. (2006). Reconstitution of chromatin transcription with purified components reveals a chromatin-specific repressive activity of p300. Nature Struct. Mol. Biol. 13, 131-139. Lusser, A., Urwin, D. L., and Kadonaga, J. T. (2005). Distinct activities of CHD1 and ACF in ATP-dependent chromatin assembly. Nature Struct. Mol. Biol. 12, 160-166. Lim, C. Y., Santoso, B., Boulay, T., Dong, E., Ohler, U., and Kadonaga, J. T. (2004). The MTE, a new core promoter element for transcription by RNA polymerase II. Genes Dev. 18, 1606-1617. Kadonaga, J. T. (2004). Regulation of RNA polymerase II transcription by sequence-specific DNA-binding factors. Cell 116, 247-257. Fyodorov, D. V., Blower, M. D., Karpen, G. H., and Kadonaga, J. T. (2004). Acf1 confers unique activities to ACF/CHRAC and promotes the formation rather than disruption of chromatin in vivo. Genes Dev. 18, 170-183. Alexiadis, V., and Kadonaga, J. T. (2002). Strand pairing by Rad54 and Rad51 is enhanced by chromatin. Genes Dev. 16, 2767-2771. Butler, J. E. F., and Kadonaga, J. T. (2002). The RNA polymerase II core promoter: a key component in the regulation of gene expression. Genes Dev. 16, 2583-2592. Fyodorov, D. V., and Kadonaga, J. T. (2002). Dynamics of ATP-dependent chromatin assembly by ACF. Nature 418, 897-900. Butler, J. E. F., and Kadonaga, J. T. (2001). Enhancer-promoter specificity mediated by DPE or TATA core promoter motifs. Genes Dev. 15, 2515-2519. Willy, P. J., Kobayashi, R., and Kadonaga, J. T. (2000). A basal transcription factor that activates or represses transcription. Science 290, 982-984. Tyler, J. K., Adams, C. R., Chen, S.-R., Kobayashi, R., Kamakaka, R. T., and Kadonaga, J. T. (1999). The RCAF complex mediates chromatin assembly during DNA replication and repair. Nature 402, 555-560. Ito, T., Levenstein, M. E., Fyodorov, D. V., Kutach, A. K., Kobayashi, R., and Kadonaga, J. T. (1999). ACF consists of two subunits, Acf1 and ISWI, that function cooperatively in the ATP-dependent catalysis of chromatin assembly. Genes Dev. 13, 1529-1539. Blackwood, E. M., and Kadonaga, J. T. (1998). Going the distance: a current view of enhancer action. Science 281, 60-63. Kraus, W. L., and Kadonaga, J. T. (1998). p300 and estrogen receptor cooperatively activate transcription via differential enhancement of initiation and reinitiation. Genes Dev. 12, 331-342. Burke, T. W., and Kadonaga, J. T. (1997). The downstream core promoter element, DPE, is conserved from Drosophila to humans and is recognized by TAFII60 of Drosophila. Genes Dev. 11, 3020-3031. Ito, T., Bulger, M., Pazin, M. J., Kobayashi, R., and Kadonaga, J. T. (1997). ACF, an ISWI-containing and ATP-utilizing chromatin assembly and remodeling factor. Cell 90, 145-155. Pazin, M. J., and Kadonaga, J. T. (1997). What's up and down with histone deacetylation and transcription? Cell 89, 325-328. Pazin, M. J., Bhargava, P., Geiduschek, E. P., and Kadonaga, J. T. (1997). Nucleosome mobility and the maintenance of nucleosome positioning. Science 276, 809-812. Pazin, M. J., and Kadonaga, J. T. (1997). SWI2/SNF2 and related proteins: ATP-driven motors that disrupt protein-DNA interactions? Cell 88, 737-740. Burke, T. W., and Kadonaga, J. T. (1996). Drosophila TFIID binds to a conserved downstream basal promoter element that is present in many TATA-box-deficient promoters. Genes Dev. 10, 711-724. Pazin, M. J., Kamakaka, R. T., and Kadonaga, J. T. (1994). ATP-dependent nucleosome reconfiguration and transcriptional activation from preassembled chromatin templates. Science 266, 2007-2011. Paranjape, S. M., Kamakaka, R. T., and Kadonaga, J. T. (1994). Role of chromatin structure in the regulation of transcription by RNA polymerase II. Annu. Rev. Biochem. 63, 265-297. Kamakaka, R. T., Bulger, M., and Kadonaga, J. T. (1993). Potentiation of RNA polymerase II transcription by Gal4-VP16 during but not after DNA replication and chromatin assembly. Genes Dev. 7, 1779-1795. Tyree, C. M., George, C. P., Lira-DeVito, L. M., Wampler, S. L., Dahmus, M. E., Zawel, L., and Kadonaga, J. T. (1993). Identification of minimal set of proteins that is sufficient for accurate initiation of transcription by RNA polymerase II. Genes Dev. 7, 1254-1265. Laybourn, P. J., and Kadonaga, J. T. (1992). Threshold phenomena and long-distance activation of transcription by RNA polymerase II. Science 257, 1682-1685. Laybourn, P. J., and Kadonaga, J. T. (1991). Role of nucleosomal cores and histone H1 in regulation of transcription by RNA polymerase II. Science 254, 238-245. Croston, G. E., Kerrigan, L. A., Lira, L., Marshak, D. R., and Kadonaga, J. T. (1991). Sequence-specific antirepression of histone H1-mediated inhibition of basal RNA polymerase II transcription. Science 251, 643-649. Kamakaka, R. T., Tyree, C. M., and Kadonaga, J. T. (1991). Accurate and efficient RNA polymerase II transcription with a soluble nuclear fraction derived from Drosophila embryos. Proc. Natl. Acad. Sci. USA 88, 1024-1028. Kadonaga, J. T., Courey, A. J., Ladika, J., and Tjian, R. (1988). Distinct regions of Sp1 modulate DNA binding and transcriptional activation. Science 242, 1566-1570. Kadonaga, J. T., Carner, K. R., Masiarz, F. R., and Tjian, R. (1987). Isolation of cDNA encoding transcription factor Sp1 and functional analysis of the DNA binding domain. Cell 51, 1079-1090. Kadonaga, J. T., and Tjian, R. (1986). Affinity purification of sequence-specific DNA binding proteins. Proc. Natl. Acad. Sci. USA 83, 5889-5893. \n",
            "-------------------- Scraping faculty url 81/183 --------------------\n",
            "susan kaech \n",
            "-------------------- Scraping faculty url 82/183 --------------------\n",
            "jan karlseder \n",
            "-------------------- Scraping faculty url 83/183 --------------------\n",
            "amy kiger  Biography Amy Kiger received her Ph.D. as a Howard Hughes Medical Institute Predoctoral Fellow from the Department of Developmental Biology at Stanford School of Medicine. She then completed postdoctoral studies in the Department of Genetics at Harvard Medical School as a Fellow of the Jane Coffin Childs Memorial Fund for Medical Research.As Principal Investigator in her own lab, Amy Kiger has been awarded fellowships from The David & Lucille Packard Foundation, The Sidney Kimmel Foundation for Cancer Research, The March of Dimes and The Hellman Family Foundation, as well as grants from the agencies Human Frontier Science Program and National Institutes of Health.  Research Cellular remodeling Investigating membrane regulation of dynamic cell organization & shape Cells must continuously maintain integrity and compartmentalization with changes in cell size and shape that occur in development, immunity, aging and disease. How cellular spatiotemporal organization is achieved with remodeling is therefore a fundamental problem of broad significance. Our laboratory is studying central roles for novel membrane trafficking processes, under control of spatiotemporal lipid regulation, that coordinate dynamic cell structure. Using a foundation in genetic and cell biology experiments in the fruit fly, Drosophila, we are linking specific lipid kinase and phosphatase activities to cell remodeling. Our work aims to identify conserved mechanisms of membrane regulation in immune and muscle cell functions, and provide insights into related human diseases. Our research is currently focused in three major areas: roles for phosphoinositide lipid regulation in cellular remodeling roles for autophagy in cellular remodeling, and systems-biology approaches to understanding cellular remodeling To address these problems, we are investigating questions specific to the dynamic organization of two distinct cell types, immune cells and muscle cells. The different elaborated morphologies and functions of immune cells and muscle cells are helping us to illuminate both broadly conserved and tissue-specific requirements in cellular remodeling. Regulation of cellular remodeling by phosphoinositide-directed endocytic recycling Phosphoinositide lipids, the seven different forms of phosphorylated phosphatidylinositol, provide a membrane code that controls a wide array of cellular functions. However, gaps remain in understanding the complex functional relationships between the relevant kinases, phosphatases and the specific roles mediated through phosphoinositide regulation. The large family of Myotubularin (MTM) phosphoinositide phosphatases (15 human, 7 fly, 1 yeast) are selective for PI(3)P and PI(3,5)P 2 turnover, pointing to possible roles in complex regulation of endocytosis and/or autophagy. Further confounding the understanding of MTM family functions are that, curiously, half of the members encode for catalytically inactive, or pseudo-phosphatases. Importantly, mutations in MTM catalytic and pseudophosphatase members are both associated with human disease, including centronuclear myopathy (XLMTM) and Charcot-Marie-Tooth neuropathy (CMT4B). How potential MTM roles in endocytosis or autophagy relate to disease are not understood. We identified mtm , a Drosophila MTM1/MTMR2 phosphoinositide phosphatase, as necessary and sufficient for formation of dynamic immune cell protrusions and immune cell recruitment to wounds. We determined that Mtm and Class II PI3-kinase, called Pi3K68D in flies, co-regulates a PI(3)P subpool on endosomes for endocytic recycling important for immune cell dynamic cortical functions. We have also demonstrated a central role for a Pi3K68D/Mtm pathway for phosphoinositide regulation of integrin endocytic trafficking crucial for maintenance of muscle attachments during muscle cell remodeling. We showed that mtm mutant myofibers exhibit hallmarks of the human XLMTM myopathy, and that the fly defects predicted disruption of integrin localization in human XLMTM. The identification of functionally distinct PI(3)P subpools under control of Class II PI3-kinase has broad implications, including therapeutic strategies for MTM-related human disease. Additional implications include possible roles for an Mtm pathway to regulate integrin trafficking in cell migration and metastasis. In ongoing studies , we are addressing how phosphoinositide-directed endocytic recycling provides key input into the dynamic balance between distinct, cortical F-actin structures in immune cells. In muscle, we are exploring the regulation of integrin trafficking during cellular remodeling, as well as the relationships between mtm and orthologs of other genes also mutated in different forms of human centronucelar myopathy. We are pursuing how PI3KC2 and Mtm are spatiotemporally regulated. Important insights are coming from investigation of the MTM pseudo-phosphatase, Sbf, as a key coordinator of both PI3KC2/Mtm phosphoinositide regulation and membrane trafficking. Regulation of cellular remodeling by autophagy Autophagy is the membrane-mediated process of cellular self-eating through encapsulation of proteins or organelles for lysosomal delivery and degradation, with well-known roles in stress responses. Autophagy has been the focus of growing interest due to its involvement in many processes, including immunity, neurodegeneration and cancer. We showed that autophagy is continuously required for macrophage cell spreading, with functions in proper cytoskeletal reorganization, and has a conserved role in the cortical remodeling of both Drosophila and mouse macrophages. Consequently, macrophages disrupted for autophagy are impaired in their recruitment to epidermal wounds. We showed a requirement for the p62 multiadaptor involved in selective targeting of ubiquitinated proteins for autophagic degradation, pointing to a role for selective autophagy (versus bulk autophagy) as a regulatory mechanism for cortical remodeling. This is the first demonstration of a true cell remodeling function at the level of cell shape for autophagy, which has been previously proposed but never shown. In ongoing studies , we are continuing to address how autophagy specifically regulates immune cell remodeling, in part through the use of systems biology approaches to identify targets of selective autophagy. We are also investigating phosphoinositide regulation of autophagy. Functional genomics of cellular remodeling The regulation of cell morphology can be viewed as a complex system that must integrate the temporal-spatial control of multiple cellular processes, including cytoskeletal and membrane dynamics, cell adhesion and growth. To understand morphogenetic programs, we are applying functional genomics to comprehensively probe the cellular-genetic networks that control cellular remodeling. We pioneered RNA-interference (RNAi) screens in Drosophila cells that permit unprecedented genome-wide loss-of-function analysis of metazoan cell morphology. Our RNAi microscopy screens have identified gene functions required to maintain distinct round or flat immune cell morphologies. Now, we are using RNAi screens to study how the addition of a hormonal cue culminates in the change of cells from a round to elongated cell shape, relevant to numerous cellular behaviors in vivo. Furthermore, co-RNAi screens targeting combinations of multiple genes are an ideal method for identifying new components of a pathway through suppressor or enhancer effects. In this way, we have isolated and continue to seek modifiers of specific phosphoinositide and autophagy pathways important in cellular remodeling.  Kiger Lab Select Publications Fujita N., W. Huang, T. Lin, J. Groulx, S. Jean, J. Nguyen, Y. Kuchitsu, I. Koyama-Honda, N. Mizushima, M. Fukuda, A. A. Kiger. 2017. Genetic screen in Drosophila muscle identifies autophagy-mediated T-tubule remodeling and a Rab2 role in autophagy. eLIFE , 6:e23367. PMID: 28063257. Jean S, Cox S, Nassari S, Kiger AA. Starvation-induced MTMR13 and RAB21 activity regulates VAMP8 to promote autophagosome-lysosome fusion. EMBO Rep . (2015) 16:297-311. Jean S, Kiger AA. Classes of phosphoinositide 3-kinases at a glance. J Cell Sci . (2014) 127:923-8. Jean, S. and A. A. Kiger. 2012. Coordination between Rab GTPase and phosphoinositide regulation and function. Nat Rev Mol Cell Biol 13: 463-470. Jean, S., S. Cox, E.J. Schmidt, F.L. Robinson and A. Kiger. 2012. Sbf/MTMR13 coordinates PI(3)P and Rab21 regulation in endocytic control of cellular remodeling. Mol Biol Cell 23(14):2723-40. Ribeiro, I., L. Yuan, G. Tanentzapf, J. Dowling, A. A. Kiger. 2011. Phosphoinositide regulation of integrin trafficking required for muscle attachment and maintenance. PLoS Genet , 7(2): e1001295. Velichkova, M., J. Juan, P. Kadandale, S. Jean, I. Ribeiro, V. Raman, C. Stefan and A. A. Kiger. 2010. Drosophila Mtm and class II PI3K coregulate a PI(3)P pool with cortical and endolysosomal functions. J Cell Biol. 190(3): 407-25. Kadandale, P. and A. A. Kiger. 2010. Role of selective autophagy in cellular remodeling: self-eating into shape. Autophagy 6:8, 1-2. Kadandale, P., J.D. Stender, C.K. Glass and A. A. Kiger. 2010. Conserved role for autophagy in Rho1-mediated cortical remodeling and blood cell recruitment. Proc Natl Acad Sci 107(23): 10502-7. Selected Publications Asano Y, Jimnez-Dalmaroni A, Liverpool TB, Marchetti MC, Giomi L, Kiger A, Duke T, Baum B. 2009. Pak3 inhibits local actin filament formation to regulate global cell polarity. HFSP J. 3(3): 194-203. Eggert, U. S., A. A. Kiger, C. Richter, Z. E. Perlman, N. Perrimon, T. J. Mitchison and C. M. Field. 2004. Parallel chemical genetic and genome-wide RNAi screens identify cytokinesis inhibitors and targets. PLoS , 2(12): e379 . Boutros*, M., A. A. Kiger*, S. Armknecht, K. Kerr, M. Hild, B. Koch, S. Haas, R. Paro, N. Perrimon . 2004. Genome-wide RNAi analysis of growth and viability in Drosophila cells. Science , 303: 832-835. Kiger, A. A., B. Baum, S. Jones, M. Jones, A. Coulson, C. Echeverri, N. Perrimon. 2003. A functional genomic analysis of cell morphology using RNA-interference. J. Biol. , 2: 27. Kiger*, A. A., D. L. Jones*, C. Schulz, M. Rogers and M.T. Fuller. 2001. Stem Cell Self-Renewal Specified by JAK-STAT Activation in Response to a Support Cell Cue. Science 294: 2542-2545 . Kiger, A. A., H. White-Cooper and M.T. Fuller. 2000. Somatic support cells restrict germline stem cell self-renewal and promote differentiation. Nature 407: 750-754. *equal authorship. \n",
            "-------------------- Scraping faculty url 84/183 --------------------\n",
            "christopher kintner \n",
            "-------------------- Scraping faculty url 85/183 --------------------\n",
            "david kleinfeld  Biography David Kleinfeld received his Ph.D. from UCSD and was a Member of Technical Staff at the former AT&T Bell Laboratories prior to joining the faculty at UCSD. He is a recipient of a David and Lucile Packard Foundation Interdisciplinary Science Award, a NIH Directors Pioneer Award, a NINDS Research Program Award, and currently holds the Experimental Biophysics Endowed Chair through Physics. David is a fellow of the American Association for the Advancement of Science and a member of the American Academy of Arts and Sciences. He currently leads the Specialization on Computational Neuroscience through the Neurosciences Graduate Program.  Research Our laboratory is focused on two problems in neurobiology: (1) Active sensing, where we are delineating the brainstem circuitry that coordinates orofacial motor actions, e.g., sniffing, licking, head bobbing, and whisking, into behaviors. This work involves anatomy, behavior, and electro- and opto-physiology with rodents ad strives to make connections with control theory. (2) Blood flow in the brain, for which we connect measurements of the topology of the vasculature with neuronal control of flow dynamics from the level of large-scale vascular networks down to single microvessels. This work involves anatomy, physiology, and deep-brain optical imaging with rodents and strives to make connections with graph theory and fluid dynamics. Our efforts in both areas involve a broad range of approaches together with the opportunity to develop new tools. Please see our web site for further information concerning our work and recent publications. \n",
            "-------------------- Scraping faculty url 86/183 --------------------\n",
            "joshua kohn  Biography Joshua R. Kohn received his Ph.D. from the University of Pennsylvania and was a postdoctoral fellow at both the University of Toronto and Cornell University.  Research We use both molecular and experimental tools to study plant mating systems and floral evolution. The mating system can vary from obligate outcrossing to complete selfing. The mating system is a prime factor in explaining differences in levels of genetic variation among populations and species. Limited variation, as is often found in selfing populations, may strongly reduce the evolutionary potential of selfers. Current work focuses on three issues: 1) The molecular evolution of the self-incompatibility locus, 2) The evolution of mating system diversity in higher plants. 3) Floral variation, its molecular causes and selective consequences. Self-Incompatibility Many plants reject their own pollen to avoid the detrimental effects of inbreeding. Self-pollen recognition and rejection is often controlled by a single locus (S) where a match between genotype of pollen and style results in arrest of pollen tube germination or growth. Rare alleles at this locus have a selective advantage, being compatible with more potential mates. This frequency-dependent selection leads to some of the highest levels of polymorphism known for any locus with 30-50 alleles often segregating within single populations. Alleles at the S-locus are also very old because, if they drift towards low frequency, they become selectively favored to increase. At the molecular level, allelic lineages are often tens of millions of years old, older than the species and genera in which they currently reside. This is reflected in the fact that an allele found in one species is often more closely related to an allele found in another species than it is to other alleles from the same species. Because of this property, the S-locus provides a tool for historical inference that extends much deeper in time than neutral variation. Using RT-PCR to amplify S-alleles from stylar tissue, we can survey S-allele diversity within and between natural populations and simultaneously gain sequence information that can be used to study evolutionary processes above the species level. We have worked primarily on systems in which RNases are the stylar product of the S-locus, as is true in the families Solanaceae, Scrophulariaceae, and Rosaceae. Current work also includes assaying S-locus variation in the Papaveraceae, a family with an entirely different molecular mechanism of self-incompatibility. Fig. 1. Floral variation in the Papaveraceae, a family with gametophytic self-incompatibility. The evolution of plant mating systems Flowering plants exhibit far more breeding system diversity than do most animals. We use both experimental and phylogenetic approaches to test evolutionary hypotheses concerning mating system diversity. We are particularly interested in the evolution of separate sexes vs. hermaphroditism and self-fertilization vs. outcrossing. Dove-tailing with our work on incompatibility, we have used inferences from the 40 million year history of shared S-locus polymorphism to trace the history of mating system changes across entire plant families. These data have provided some of the strongest evidence yet that selfing is often an evolutionary dead end. Ongoing work is aimed at assessing whether incompatibility, a trait that is frequently lost but very rarely gained, provides a macroevolutionary advantage in terms of either increased speciation or reduced extinction. Floral variation The California flora is the richest in the continental US providing abundant systems for ecological and evolutionary studies. In San Diego County, students in the lab have done two Ph.D. theses on Mimulus aurantiacus, bush monkeyflower, a species with a sensitive stigma that closes rapidly after being touched by its hummingbird pollinator. Stigma closure was found to increase the hermaphroditic flowers male reproductive success as flowers with closed stigmas exported 2-3 times more pollen to the stigmas of subsequently-visited flowers than did flowers with open stigmas. This plant also has two color morphs locally. The red morph occurs close to the coast while the yellow morph occurs inland with a narrow hybrid zone in between. Experimental studies have shown that hummingbirds, though they visit yellow populations, strongly prefer the red-flowered form when given a choice and that hawkmoths only visit the yellow form. Flower color variation is caused by the presence of red, anthocyanin pigments in red flowers and their absence in yellow flowers, though both forms make red pigments in their stems. Progress is being made in understanding the molecular basis of color variation so that selection from the molecular to the phenotypic level can be better understood. The Kohn lab accepts students interested in any aspect of plant reproductive ecology. Fig. 2. Color morphs of Mimulus aurantiacus .  Select Publications Kono, Y; Kohn, JR. 2015. Range and Frequency of Africanized Honey Bees in California (USA). PLoS ONE v. 10(9): p. e0137407-0. Rightmyer, MG; Kono, Y; Kohn, JR; Hung, KL. 2014. A new species of Triepeolus (Hymenoptera: Apidae), with comments on T. utahensis (Cockerell) and T. melanarius Rightmyer. Zootaxa v. 3872(1): p. 48-56. Handelman, C, JR Kohn. 2014. Hummingbird color preference within a natural hybrid population of Mimulus aurantiacus (Phrymaceae). Plant Species Biology doi: 10.1111/j.1442-1984.2012.00393.x Biliniski, P, JR Kohn. 2012. Sites of self-pollen tube inhibition in Papaveraceae (sensu lato). Plant Systematics and Evolution 98: 1239-1247. Goldberg E, JR Kohn, R Lande, KA Robertson, SA Smith and B Igic. Species selection maintains self-incompatibility. Science 330: 493-495. Paape T and JR Kohn. 2011. Differential strengths of selection on S-RNases from Physalis and Solanum (Solanaceae). BMC Evolutionary Biology 11: 243 doi:10.1186/1471-2148-11-243 Paape T, T Miyake T, N Takebayashi N, D Wolf D and JR Kohn JR. 2011. Evolutionary genetics of an S-Like polymorphism in Papaveraceae with putative function in self-incompatibility. PLoS ONE 6(8): e23635. doi:10.1371/journal.pone.0023635 Newbigin. E, T Paape and JR Kohn. 2008. RNase-based self-incompatibility: puzzled by pollen-S. Plant Cell 20:2286-2292. Igic, B, R Lande and JR Kohn 2008. Loss of self-incompatibility and its evolutionary consequences. International Journal of Plant Sciences 169:93-104. Paape T., B Igic, SD Smith, R Olmstead, L Bohs and JR Kohn. 2008. A 15-My-old genetic bottleneck. Molecular Biology and Evolution 25:655-663. Streisfeld, MA, and JR Kohn. 2007. Environment and pollinator-mediated selection on parapatric floral races of Mimulus aurantiacus . Journal of Evolutionary Biology 20:122-132. Igic B, and JR Kohn. 2006. The distribution of plant mating systems: study bias against obligately outcrossing species. Evolution 60:1098-1103. Igic, B, L Bohs, and JR Kohn. 2006. Molecular fossils reveal unidirectional breeding system shifts. Proceedings of the National Academy of Sciences USA 103:1359-1363. Streisfeld, MA, and JR Kohn. 2005. Contrasting patterns of floral and molecular variation across a cline in Mimulus aurantiacus . Evolution 59:2548-2559. Yeung K., JS Miller, AE Savage, BC Husband, B Igic and JR Kohn. 2005 Association of ploidy and sexual system in Lycium californicum (Solanaceae). Evolution 59:2048-55. Biliniski, P, JR Kohn. 2012. Sites of self-pollen tube inhibition in Papaveraceae (sensu lato). Plant Systematics and Evolution 98: 1239-1247. Igic, B. and J. R. Kohn. 2001. Evolutionary relationships among self-incompatibility RNases. Proceedings of the National Academy of Sciences USA 98:13167-13171. \n",
            "-------------------- Scraping faculty url 87/183 --------------------\n",
            "takaki komiyama  Biography Dr. Komiyama was a Postdoc at Janelia Farm, Howard Hughes Medical Institute, received his Phd in Neurosciences at Stanford University (2006) and his BA in Biochemistry at University of Tokyo (2001). His Honors include: Helen Hay Whitney Foundation Postdoctoral Fellowship, Harold M. Weintraub Graduate Student Award, Japan-Stanford Association Graduate Fellowship, Stanford Graduate Fellowship, Pew Scholars Program, Packard Fellowship and Sloan Research Fellowship. Dr. Komiyama is a NYSCF-Robertson Investigator.  Research Animals constantly modify their behavior through experience. Flexible behavior is key to our ability to adapt to the ever-changing environment. My laboratory is interested in studying the activity of neuronal ensembles in behaving animals, and how it changes with learning. We have recently set up a paradigm where mice learn to associate sensory information (two different odors) to motor outputs (lick vs no-lick) under head-fixation. We combined this with two-photon calcium imaging, which can monitor the activity of a microcircuit of many tens of neurons simultaneously from a small area of the brain. Imaging the motor cortex during the learning of this task revealed neurons with diverse task-related response types. Intriguingly, different response types were spatially intermingled; even immediately adjacent neurons often had very different response types. As the mouse learned the task under the microscope, the activity coupling of neurons with similar response types specifically increased, even though they are intermingled with neurons with dissimilar response types. This suggests that intermingled subnetworks of functionally-related neurons form in a learning-related way, an observation that became possible with our cutting-edge technique combining imaging and behavior. We are working to extend this study. How plastic are neuronal microcircuits during other forms of learning? How plastic are they in other parts of the brain? What are the cellular and molecular mechanisms of the microcircuit plasticity? Are the observed activity and plasticity required for learning? How does the activity of identified individual neurons change over days to weeks? We are asking these questions, combining a variety of techniques including in vivo two-photon imaging, optogenetics, electrophysiology, genetics and behavior. Figure 1: In vivo two-photon calcium imaging from head-fixed mice during learning Figure 2: A snapshot of calcium imaging  Select Publications Lu, Y., Liu, X., Hattori, R., Ren, C., Komiyama, T., and Kuzum, D. (2018) \"Ultra-low Impedance Graphene Microelectrodes with High Optical Transparency for Simultaneous Deep 2-photon Imaging in Transgenic Mice.\" Advanced Functional Materials, 1800002. DOI: 10.1002/adfm.201800002. Li, W.L., Chu, M., Wu, A., Suzuki, Y., Imayoshi, I., Komiyama, T. (2018) \"Adult-born neurons facilitate olfactory bulb pattern separation during task engagement.\" eLife, 2018;7:e33006 Mitani, A., Dong, M. and Komiyama, T. (2018) \"Brain-computer interface with inhibitory neurons reveals subtype-specific strategies.\" Current Biology, 28(1), 77-83. Chu, M.W., Li, W.L. and Komiyama, T. (2017) Lack of pattern separation in sensory inputs to the olfactory bulb during perceptual learning. eNeuro*, *0287-17.201. Hwang, E.J.*, Dahlen, J.E.*, Mukundan, M. and Komiyama, T. (2017) History-dependent action selection bias in the posterior parietal cortex. Nature Communications*,* 8, 1242. Hattori, R.*, Kuchibhotla, K.*, Froemke, R.#, and Komiyama, T.#(2017) Functions and dysfunctions of neocortical inhibitory neuron subtypes. (Review) Nature Neuroscience , 20, 1199-1208. Peters, A.J., Lee, J., Hedrick, N.G., O'Neil, K., and *Komiyama, T*. (2017) Reorganization of corticospinal output during motor learning. *Nature Neuroscience*, *20, 1133-41 - *Rated by Faculty of 1000** Makino, H.*#, Ren, C.*, Liu, H.*, Kim, A.N., Kondapaneni, N., Liu, X., Kuzum, D. and *Komiyama, T.#*(2017) Transformation of cortex-wide emergent properties during motor learning. *Neuron*, 94(4), 880-90. - *Preview* by Banerjee and Long, *Neuron* 94(4), 698-700. Peters, A.J., Liu, H. and *Komiyama, T.* (2017) Learning in the rodent motor cortex. (Review) *Annual Review of Neuroscience*, *40, 77-97.* McIntyre, J.C., Thiebaud, N., McGann, J.P., *Komiyama, T.* and Rothermel, M. (2017) Neuromodulation in chemosensory systems. (Review) *Chemical Senses*, 42(5), 375-9. Makino, H.*, Hwang, E.J.*, Hedrick, N.G.* and Komiyama, T. (2016) Circuit mechanisms of sensorimotor learning. (Review) Neuron, 92(4), 705-21. Chu, M.W., Li, W.L. and Komiyama, T. (2016) Balancing the robustness and efficiency of odor representations during learning. Neuron, 92(1), 174-86. Makino, H. and Komiyama, T. (2015) Learning enhances the relative impact of top-down processing in the visual cortex. Nature Neuroscience, 18(8), 1116-22. Chen, S.X., Kim, A.N., Peters, A.J. and Komiyama, T. (2015) Subtype-specific plasticity of inhibitory circuits during motor learning. Nature Neuroscience, 18(8), 1109-15. Boyd, A., Kato, H.K., Komiyama, T. and Isaacson, J.S. (2015) \"Broadcasting of cortical activity to the olfactory bulb.\" Cell Reports, 10(7), 1032-9. Peters, A.J., Chen, S.X. and Komiyama, T. (2014) Emergence of Reproducible Spatiotemporal Activity During Motor Learning. Nature, online. Guo ZV, Hires SA, Li N, O'Connor DH, Komiyama T, Ophir E, Huber D, Bonardi C, Morandell K, Gutnisky D, Peron S, Xu NL, Cox J, Svoboda K. (2014) Procedures for behavioral experiments in head-fixed mice. PLoS One, 9(2):e88678. Kato, H.K., Gillet, S.N, Peters, A.J., Isaacson, J.S. and Komiyama, T. (2013) Parvalbumin-Expressing Interneurons Linearly Control Olfactory Bulb Output. Neuron, 80(5), 1218-31. Kato, H.K.*, Chu, M.W.*, Isaacson, J.S. and Komiyama, T. (2012) Dynamic Sensory Representations in the Olfactory Bulb: Modulation by Wakefulness and Experience. Neuron, 76(5), 962-975. Sweeney LB, Chou YH, Wu Z, Joo W, Komiyama T, Potter CJ, Kolodkin AL, Garcia KC, Luo L. (2011) Secreted semaphorins from degenerating larval ORN axons direct adult projection neuron dendrite targeting. Neuron, 72(5), 734-47. Komiyama, T.*, Sato, T.R., O'Connor, D.H., Zhang, Y.X., Huber, D., Hooks, B.M., Gabbito, M. and Svoboda, K. (2010) Learning-related fine-scale specificity imaged in motor cortex circuits of behaving mice. Nature, 464(7292), 1182-1186. (*: correspondence) Faculty of 1000 Article Factor: 15 O'Connor, D.H., Clack, N.G., Huber, D., Komiyama, T., Myers, E.W. and Svoboda, K. (2010) Vibrissa-based object localization in head-fixed mice. J. Neuroscience, 30, 1947-1967. Komiyama, T. and Luo, L. (2007) Intrinsic control of precise dendritic targeting by an ensemble of transcription factors. Current Biology 17, 278-285. Komiyama, T., Sweeney, L.B., Schuldiner, O., Garcia, K.C. and Luo, L. (2007) Graded expression of Semaphorin-1a cell-autonomously directs dendritic targeting of olfactory projection neurons. Cell 128, 399-410. Sweeney, L.B.*, Couto, A., Chou, Y-H., Berdnik, D., Dickson, B.J., Luo, L. and Komiyama, T.* (2007) Temporal target restriction of olfactory receptor neurons through Semaphorin-1a/PlexinA mediated axon-axon interactions. Neuron 53, 185-200. (*: equal contribution) Komiyama, T. and Liqun Luo (2006) Development of wiring specificity of the olfactory system. Current Opinion in Neurobiology 16(1), 67-73. Komiyama, T., Carlson, J.R. and Luo, L. (2004) Olfactory receptor neuron axon targeting: Intrinsic transcriptional control and hierarchical interactions. Nature Neuroscience 7, 819-825. Komiyama, T.*, Johnson, W.A., Luo, L. and Jefferis, G.S.X.E.* (2003) From lineage to wiring specificity: POU domain transcription factors control precise connections of Drosophila olfactory projection neurons. Cell 112, 157-167. (*: equal contribution) \n",
            "-------------------- Scraping faculty url 88/183 --------------------\n",
            "william kristan  Biography Bill Kristan received his Ph.D. from the University of Pennsylvania. He was a postdoctoral fellow at Stanford University and an assistant research biologist at UC Berkeley. He held a Guggenheim Fellowship and was a Fellow of the University of Bielefeld, Germany.  Research My lab studies how networks of nerve cells produce different behaviors and how these neuronal networks are established during embryogenesis. We use electrophysiological, imaging, anatomical, computational, and embryological techniques to characterize these circuits in the relatively tractable nervous system of the medicinal leech, and to construct computerized models of them.  Select Publications Briggman K.L., Abarbanel H.D.I., Kristan W.B. Jr. (2005) Optical imaging of neuronal populations during decision-making. Science 307:896-901. Kristan W.B. Jr., Calabrese R.L., Friesen W.O. (2005) Neuronal basis of leech behaviors. Prog. Neurobiol. 76:279-327. Thomson E.E., Kristan W.B. (2006) Encoding and decoding touch location in the leech CNS. J. Neurosci. 26:8009-8016. Briggman K.L., Kristan W.B. Jr. (2006) Imaging dedicated and multifunctional neural circuits generating distinct behaviors. J. Neurosci. 26:10925-10933. Marin-Burgin A., Kristan W.B. Jr., French K.A. (2008) From synapses to behavior: Development of a sensory-motor circuit in the leech. Special issue of Dev. Neurobiology 68:779-787. Baca S.M., Marin-Burgin A, Wagenaar D.A., Kristan W.B. Jr. (2008) Widespread inhibition proportional to excitation controls the gain of a leech behavioral circuit. Neuron 57:276-289. Briggman K.L., Kristan W.B. Jr. (2008) Multifunctional pattern generating circuits. Annu. Rev. Neurosci. 31:271-294. \n",
            "-------------------- Scraping faculty url 89/183 --------------------\n",
            "mitchell kronenberg \n",
            "-------------------- Scraping faculty url 90/183 --------------------\n",
            "sergey kryazhimskiy  Biography Sergey Kryazhimskiy received his Ph.D. from Princeton University in 2008. He did his postdoctoral work at the University of Pennsylvania and Harvard University where he was awarded the Burroughs Wellcome Fund Career Award at Scientific Interface. He will join the EBE faculty in January 2016.  Research We are interested in uncovering general principles of how mutations affect phenotypes at multiple levels of biological organization (gene expression, metabolic fluxes, etc.), and how these effects combine to determine organisms fitness across environments. We approach this problem using experiments in model microorganisms S. cerevisiae and E.coli and mathematical modeling. Our lab has three complementary themes. First, we empirically characterize how the effects of hundreds of mutations on fitness depend on the external environment and on the genetic background of the organism. In technical terms, we characterize the statistics of epistasis and pleiotropy. Second, we attempt to predict from first principles how mutations are expected to perturb cellular physiology and fitness. Third, we measure some of these mutational perturbations of cell physiology (e.g., changes in gene expression profiles) and test our theoretical predictions.  Select Publications Neverov AD, Kryazhimskiy S, Plotkin JB, Bazykin GA (2015). Coordinated evolution of influenza A surface proteins. PLoS Genetics 11: e1005404 (PMID: 26247472) Baym M, Kryazhimskiy S, Lieberman T, Chung H, Desai MM, Kishony R (2015). Inexpensive multiplexed library preparation for megabase-sized genomes. PLoS One 10: e0128036 (PMID: 26000737) Kryazhimskiy S, Rice DP, Jerison ER, Desai MM (2014). Global epistasis makes adaptation predictable despite sequence-level stochasticity. Science 344: 15191522 (PMID: 24970088) Kryazhimskiy S, Rice DP, Desai MM (2012). Population subdivision and adaptation in asexual populations of Saccharomyces cerevisiae. Evolution 66: 19311941 (PMID: 22671557) Kryazhimskiy S, Tkaik G, Plotkin JB (2009). The dynamics of adaptation on correlated fitness landscapes. Proc Natl Acad Sci USA 106: 1863818643 (PMID: 19858497) \n",
            "-------------------- Scraping faculty url 91/183 --------------------\n",
            "carolyn kurle  Research Research in the Kurle lab focuses on understanding how animal foraging and trophic patterns drive community ecology. We use stable isotope biogeochemistry to answer questions regarding trophic interactions, niche partitioning, and habitat use, and our research informs conservation and management of species and ecosystems. We also seek to understand the best use of biogeochemical tools to address ecological problems. Our work takes place mostly in marine, coastal, and island systems.  Select Publications Kurle, CM and McWhorter, J (2017) Spatial and temporal variability within marine isoscapes: implications for interpreting stable isotope data from marine systems. Marine Ecology Progress Series. doi: 10.3354/meps12045 Turner Tomaszewicz, CN, Seminoff, JA, Peckham, SH, Avens, L, and Kurle, CM (2017) Intrapopulation variability in the timing of ontogenetic habitat shifts in sea turtles revealed using 15N values from bone growth rings. Journal of Animal Ecology. doi: 10.1111/1365-2656.12618 Kurle, CM, Bakker, V, Copeland, H, Burnett, J, Jones, J, Brandt, J, and Finkelstein, M (2016) Terrestrial scavenging of marine mammals: cross-ecosystem contaminant transfer and potential risks to endangered California condors (Gymnogyps californianus). Environmental Science and Technology. doi: 10.1021/acsest.6b01990 Turner Tomaszewicz, CN, Seminoff, JA, Ramirez, MD, and Kurle, CM (2016) Techniques for extracting sequential annual bone growth layers for stable isotope analysis. Methods in Ecology and Evolution. doi: 10.1111/2041-210X.12522 Steinitz, R, Lemm, JM, Pasachnik, SA, and Kurle, CM (2016) Diet-tissue stable isotope (13C and 15N) discrimination factors for multiple tissues from terrestrial reptiles. Rapid Communications in Mass Spectrometry. 30:9-21 Hopkins III, JB and Kurle, CM (2016) Measuring the realized niches of animals using stable isotopes: from rats to bears. Methods in Ecology and Evolution. 7:210-221 Turner Tomaszewicz, CN, Seminoff, JA, and Kurle, CM (2015). Effects of demineralization on the stable isotope analysis of bone samples Rapid Communications in Mass Spectrometry. 29:1879-1888 Bowden, JJ, Eskildsen, A, Hansen, RR, Olsen, K, Kurle, CM, and Hye, TT (2015) High-arctic butterflies become smaller with rising temperatures. Biology Letters. 11: 20150574 Turner Tomaszewicz, CN, Seminoff, JA, Avens, L, Goshe, LR, Peckham, SH, Rodriguez-Baron, JM, Bickerman, K, and Kurle CM (2015) Determining age and residency duration of loggerhead turtles (Caretta caretta) at a North Pacific bycatch hotspot using skeletochronology. Biological Conservation. 185:134-142 Kurle, CM, Koch, PL, Tershy, BR, and Croll, DA (2014) The effects of sex, tissue type, and dietary components on stable isotope discrimination factors (13C and 15N) in mammalian omnivores. Isotopes in Environmental and Health Studies, Special Issue. 50:307-321. Kurle CM and Cardinale, BJ (2011) Ecological factors associated with the strength of trophic cascades in streams. Oikos 120:1897-1908 Kurle CM, Sinclair ES, Edwards AE, and Gudmundson CJ (2011) Temporal and spatial variation in the stable nitrogen and carbon isotope ratios of common marine prey species from Alaskan waters. Marine Biology 158:2389-2404 Kurle CM (2009) Interpreting temporal variation in omnivore foraging ecology via stable isotope modeling. Functional Ecology 23:733-744 Kurle CM, Croll DA, and Tershy BT (2008) Introduced rats indirectly change marine intertidal communities from algae to invertebrate dominated. Proceedings of the National Academy of Sciences 105:3800-3804 \n",
            "-------------------- Scraping faculty url 92/183 --------------------\n",
            "russell lande  Research Russell Lande pioneered the field of quantitative genetics and phenotypic evolution in natural populations with a series of theoretical papers on mutation and maintenance of heritable variation, sexual dimorphism and sexual selection, life history evolution, phenotypic plasticity, plant mating systems, and the measurement of natural selection on correlated characters. His concepts of the G matrix and the selection gradient are now standard tools in evolutionary biology. He is an expert in stochastic demography, population viability, and extinction risk assessment, with notable contributions to conceptual and applied conservation biology.  Select Publications Chevin, L.-M. and R. Lande. 2015. Evolution of environmental cues for phenotypic plasticity. Evolution 69: 2767-2775. Lande, R. and E. Porcher. 2015. Maintenance of quantitative genetic variance under partial self-fertilization, with implications for evolution of selfing. Genetics 200: 891906. Lande, R. 2015. Evolution of phenotypic plasticity in colonizing species. Molecular Ecology 24: 2038-2045. Penz, C., P. DeVries, J. Tufto and R. Lande. 2015. Butterfly dispersal across Amazonia and its implications for biogeography. Ecography 38: 410-418. Devaux, C., R. Lande and E. Porcher. 2014. Pollination ecology and inbreeding depression control individual flowering phenologies and mixed mating. Evolution 68: 3051-3065. S. Engen, R. Lande, and B.-E. Saether. 2014. Evolutionary consequences of non-selective harvesting in density-dependent populations. American Naturalist 184: 714-726. Grotan, V., R. Lande, I.A. Chacon and P.J. DeVries. 2014. Seasonal cycles of diversity and similarity in a Central American rainforest butterfly community. Ecography 37: 509-516. Lande, R. 2014. Evolution of phenotypic plasticity and environmental tolerance of a labile quantitative character in a fluctuating environment. Journal of Evolutionary Biology 27: 866-875. Engen, S., R. Lande, and B.-E. Saether. 2013. A quantitative genetic model of r- and K-selection in a fluctuating population. American Naturalist 181: 725-736. Chevin, L.-M. and R. Lande. 2013. Evolution of discrete phenotypes from continuous norms of reaction. American Naturalist 182: 13-27. Tufto, J., R. Lande, T.-H. Ringsby, S. Engen, B.-E. Saether, T.R. Walla and P.J. DeVries. 2012. Estimating Brownian motion dispersal rate, longevity and population density from spatially explicit mark-recapture data on tropical butterflies. Journal of Animal Ecology 81: 756-769 Lande, R. 2009. Adaptation to an extraordinary environment by evolution of phenotypic plasticity and gen eticassimilation. Journal of Evolutionary Biology 22: 1435-1446. \n",
            "-------------------- Scraping faculty url 93/183 --------------------\n",
            "shannon lauberth  Biography Dr. Shannon Lauberth received her Ph.D. in Biochemistry and Molecular Biology from Saint Louis University. She performed her postdoctoral training at the Rockefeller University where she was a recipient of a Ruth L. Kirschstein National Research Service Award (NRSA) and an active member of the Anderson Cancer Center.  Research Mutations resulting in the inactivation of the tumor suppressor p53 are the most common lesions present in human malignancies, and thereby contribute significantly to the pathogenesis of human cancer. Importantly, the reactivation of wild-type p53 leads to efficient tumor regression indicating the clinical implications for understanding p53 function. In normal cells, the primary function of p53 is to respond to stress by activating multiple target genes that then execute specific pathways that include cell cycle arrest and programmed cell death. The activation kinetics of the various p53 target genes varies significantly, ranging from a rapid induction of cell cycle control genes to a delayed induction of pro-apoptotic genes. Establishing the proper activation kinetics and balance between the p53-response pathways is essential for preventing the propagation of cells that harbor DNA mutations and can lead to tumor development. A fundamental question that remains is how p53 determines which of the diverse transcription programs to activate in order to facilitate the appropriate stress response. Changes in gene expression that arise in the absence of alterations in the DNA sequence are referred to as epigenetic. A primary epigenetic mechanism that contributes to gene regulation includes the posttranslational modifications of histone proteins. Histone modifications function by facilitating direct effects on chromatin structure and through the recruitment of chromatin-associated factors, or effector proteins that translate histone modifications into specific biological outcomes. A wide array of histone modifications and effector proteins have been identified, and as a result, we are faced with the challenge of understanding the mechanisms by which histone modifications, both independently and in conjunction with each other, function through effector interactions to regulate gene expression. A role for histone modifications in the regulation of p53 target gene expression is demonstrated by several studies showing an accumulation of histone modifications at p53 target gene promoters in response to stress. Yet, the precise functions of histone modifications in regulating the p53 gene network remain to be elucidated. The primary goal of our research is to understand the molecular basis for the epigenetic regulation of the gene-selective functions of p53. Toward this goal, we are employing a powerful combination of biochemical, molecular, and genomic approaches. By implementing this multi-faceted strategy, we recently revealed a mechanism by which the prominent histone modification, histone H3 trimethylated at lysine 4 (H3K4me3), through direct interactions with the general initiation factor TAF3/TFIID, facilitates selective p53-dependent activation of cell cycle control genes (Figure 1). By exploiting cell free transcription systems reconstituted with purified factors and recombinant chromatin templates, we identified direct (causal) effects of this histone modification and a functional interplay between this histone mark and core promoter DNA sequences. Thus, our in vitro assays provide a powerful and unique means to identify regulatory mechanisms that function both independently and cooperatively at both the genetic and epigenetic level to mediate coordinated transcriptional control. We are also developing single cell methods to quantitatively measure the temporal dynamics of the recruitment/stabilization of epigenetic readers during a p53 dependent response and to measure whether changes in effector binding correlate with the kinetics and distinct patterns of p53 target gene expression. Ongoing projects in the lab are focused on: The biochemical identification and characterization of histone modifications and effector proteins that regulate differential p53 target gene expression during stress responses. By employing in vitro transcription systems with recombinant chromatin templates, the concerted function of epigenetic marks and effector proteins will be determined at the chromatin level. Single cell methods will be employed to quantitatively measure temporal changes in the accumulation of histone modifications and effector protein recruitment/stabilization and to determine whether changes in effector interactions correlate with the kinetics of p53 target gene expression. The biological significance of histone effector proteins in regulating the tumor suppressor functions of p53 will be examined using cell-based assays and mouse knockout models. Our long-term research goal is to develop target-based screening methods using our defined in vitro systems to examine the effects of small molecule compounds on the functions of histone effector proteins. Together our ongoing studies will aid in the understanding of how histone modifications precisely contribute to the dynamics of p53 target gene regulation, and will ultimately advance the potential for target based screening approaches that will allow for the development of therapeutic targeting strategies, an issue of great significance in light of the recent successes in targeting histone modification effector proteins/readers for developing cancer therapies.  Select Publications Rahnamoun H, Lee J, Sun Z, Lu H, Ramsey KM, Komives EA, and Lauberth SM. RNAs Interact with BRD4 to Promote Enhanced Chromatin Engagement and Transcription Activation. Nature Structural and Molecular Biology 2018 August 3, 25(8):687-697. PubMed PMID: 30076409. Rahnamoun H, Hong J, Sun Z, Lee J, Lu H, and Lauberth SM. Mutant p53 regulates enhancer-associated H3K4 monomethylation through interactions with the methyltransferase MLL4. JBC 2018 Aug 24;293(34):13234-13246. PubMed PMID: 29954944. Rahnamoun H, Lu, H, Duttke S, Glass CK, Benner C, and Lauberth SM. Mutant p53 shapes the enhancer landscape of cancer cells in response to chronic immune signaling. Nature Communications 2017 September 29, 8(1):754. PubMed PMID: 28963538. Stijf-Bultsma Y, Sommer L, Tauber M, Baalbaki M, Giardoglou P, Jones DR, Gelato KA, van Pelt J, Shah Z, Rahnamoun H, Toma C, Anderson KE, Hawkins P, Lauberth SM, Haramis AP, Hart D, Fischle W, Divecha N. The basal transcription complex component TAF3 transduces changes in nuclear phosphoinositides into transcriptional output. Mol. Cell 2015 May 7, 58(3):453-67. PubMed PMID: 25866244. Lauberth SM., Nakayama T., Wu X., Ferris A., Tang, Z., Hughes S.H., and Roeder, R.G. (2013). H3K4me3 Interactions with TAF3 Regulate Preinitiation Complex Assembly and Selective Gene Activation. Cell, 1021-1036. Featured in: PICking H3K4me3, Beth Moorefield, Nature Struc. Mol. Biol. 20, 411 (2013) doi:10.1038/nsmb.2561, Published online 03 April 2013. Lauberth SM., Bilyeu A.C., Firulli, B.A., Kroll K.L., and Rauchman M. (2007). A phosphomimetic mutation in the Sall1 repression motif disrupts recruitment of the nucleosome remodeling and deacetylase complex and repression of Gbx2. J. Biol. Chem. 282, 34858-34868. Lauberth SM., and Rauchman M. (2006). A conserved 12-amino acid motif in Sall1 recruits the nucleosome remodeling and deacetylase corepressor complex. J. Biol. Chem. 281, 23922-312013. \n",
            "-------------------- Scraping faculty url 94/183 --------------------\n",
            "julie law \n",
            "-------------------- Scraping faculty url 95/183 --------------------\n",
            "kuo lee \n",
            "-------------------- Scraping faculty url 96/183 --------------------\n",
            "andres leschziner \n",
            "-------------------- Scraping faculty url 97/183 --------------------\n",
            "jill leutgeb  Biography Jill Leutgeb received her Ph.D. in Neurophysiology from the Leibniz Institute for Neurobiology, Germany, and was a postdoctoral fellow at the Kavli Institute for Systems Neuroscience, Norway, in the lab of Nobel laureates May-Britt and Edvard Moser. She received the prestigious Young Researcher Prize from the Royal Norwegian Society of Sciences and Letters in recognition of her postdoctoral research. Dr. Leutgeb joined UC San Diego in 2008 and is now Associate Professor in the Neurobiology Section of the Division of Biological Sciences. She is also a member of the Center for Neural Circuits and Behavior, and the Traumatic Brain Injury Team of the Kavli Institute of Brain and Mind, UC San Diego. Dr. Leutgeb has received the Hellman Fellowship, the Ray Thomas Edwards Foundation Early Career Award, and the Walter F. Heiligenberg Professorship in Neuroethology.  Research How do neural networks in the brain support the formation of distinct and long-lasting memories? Dr. Jill Leutgebs research is focused on understanding these questions by identifying neural circuit computations which underlie learning and memory. In addition, she also investigates how the function of memory circuits is altered in the development and expression of acquired epilepsies. Her laboratory combines high-density electrophysiology with behavioral testing, theoretical modeling, and pharmacological and molecular manipulations as a multidisciplinary approach to understanding the neural basis of cognition. By understanding the biological basis of memory in health and its dysfunction in disease, the long-term research goals of Dr. Leutgebs laboratory are to develop new therapeutic approaches for restoring memory loss and for seizure prediction, intervention, or prevention.  Select Publications Chenani A, Sabariego M, Schlesiger MI, Leutgeb JK, Leutgeb S, Leibold C (2019) Hippocampal CA1 replay becomes less prominent but more rigid without inputs from medial entorhinal cortex. Nat Commun, 10:1341. Diehl GW, Hon OJ, Leutgeb S, Leutgeb JK (2019) Stability of medial entorhinal cortex representations over time. Hippocampus, 29:284. Ewell LA, Fischer KB, Leibold C, Leutgeb S, Leutgeb JK (2019) The impact of pathological high frequency oscillations on hippocampal network activity in rats with chronic epilepsy. eLife, 8:e42148. doi:10.7554/eLife.42148. Sabariego M, Schnwald A, Boublil BL, Zimmerman DT, Ahmadi S, Gonzalez N, Leibold C, Clark RE, Leutgeb JK, Leutgeb S. (2019) Time cells in the hippocampus are neither dependent on mEC inputs nor necessary for spatial working memory. Neuron, doi: 10.1038/s41467-019-09280-0. Sanders H, Ji D, Sasaki T, Leutgeb JK, Wilson MA, Lisman JE (2019) Temporal coding and rate remapping: Representation of non-spatial information in the hippocampus. Hippocampus, 29:111-127. Zutshi I, Fu M, Lilascharoen V, Leutgeb JK, Lim BK, Leutgeb S (2018) Recurrent circuits within medial entorhinal cortex superficial layers support grid cell firing. Nat Commun, 9:3701-3716. Sasaki T, Piatti VC, Hwaun E, Ahmadi S, Lisman JE, Leutgeb S, Leutgeb JK (2018) Dentate network activity is necessary for spatial working memory by supporting CA3 sharp-wave ripple generation and prospective firing of CA3 neurons. Nat Neurosci, 21:258-269. Diehl GW, Hon OJ, Leutgeb S, Leutgeb JK (2017) Grid and non-grid cells in medial entorhinal cortex represent spatial location and environmental features with complementary coding schemes. Neuron, 94: 83-92. Lippi G, Fernandes CC, Ewell LA, John D, Romoli B, Curia G, Taylor SR, Frady EP, Jensen AB, Liu JC, Chaabane MM, Belal C, Nathanson JL, Zoli M, Leutgeb JK, Biagini G, Yeo GW, Berg DK (2016) Micro RNA-101 regulates multiple developmental programs to constrain excitation in adult neural networks. Neuron, 92:1337-1351. Ewell LA, Liang L, Armstrong C, Soltesz I, Leutgeb S, Leutgeb JK (2015) Brain state is a major factor in preseizure hippocampal network activity and influences success of seizure intervention. J Neuroscience, 35:15635-48. Schlesiger MI, Cannova CC, Boublil B, Hales JB, Mankin EA, Brandon MP, Leutgeb JK, Leibold C, Leutgeb S (2015) The medial entorhinal cortex is necessary for the temporal organization of hippocampal neuronal activity. Nat Neuroscience 18:1123-32. Mankin EA, Diehl GW, Sparks FT, Leutgeb S, Leutgeb JK (2015) Hippocampal CA2 activity patterns change over time to a larger extent than between spatial contexts. Neuron, 85:190-201. Sasaki T, Leutgeb S, Leutgeb JK (2014) Spatial and memory circuits in the medial entorhinal cortex. Curr Opin Neurobiol, http://dx.doi.org/10.1016/j.conb.2014.10.008. Hales JB, Schlesiger MI, Leutgeb JK, Squire LR, Leutgeb S, Clark RE (2014) Medial entorhinal cortex lesions only partially disrupt hippocampal place cells and hippocampus-dependent place memory. Cell Rep, 9:893-901. Brandon MP, Koenig J, Leutgeb JK, Leutgeb S (2014) New and distinct hippocampal place codes are generated in a new environment during septal inactivation. Neuron, 82:789-796. Piatti VC, Ewell LA, Leutgeb JK (2013) Neurogenesis in the dentate gyrus: carrying the message or dictating the tone. Front Neurosci 7:50. DOI:10.3389/fnins.2013.00050. Schlesiger MI, Cressey JC, Boublil B, Koenig J, Melvin NR, Leutgeb JK, Leutgeb S (2013) Hippocampal activation during the recall of remote spatial memories in radial maze tasks. Neurobiol Learn Mem, 106:324-333. Lu L*, Leutgeb JK*, Tsao A, Henriksen EJ, Leutgeb S, Barnes CA, Witter MP, Moser M-B, Moser EI (2013) Impaired hippocampal rate coding after lesions of the lateral entorhinal cortex. Nature Neurosci, 16:1085-1093. *Equal co-first authors Mankin EA, Sparks FT, Slayyeh B, Sutherland RJ, Leutgeb S, Leutgeb JK (2012) A neuronal code for extended time in the hippocampus. PNAS, 109:19462-19467. Koenig J, Linder AN, Leutgeb JK, Leutgeb S (2011) The spatial periodicity of grid cells in not sustained during reduced theta oscillations. Science 332:592-595. Leutgeb JK, Leutgeb S (2010) The dentate gyrus: a comprehensive guide to structure, function, and clinical implications. Hippocampus online ahead of press, DOI: 10.1002/hipo.20662. Alme CB, Buzzetti RA, Marrone DF, Leutgeb JK, Chawla MK, Schaner MJ, Bohanick JD, Khoboko T, Leutgeb S, Moser EI, Moser MB, McNaughton BL, Barnes CA (2010) Hippocampal granule cells opt for early retirement. Hippocampus 20:1109-1123. Colgin LL, Leutgeb S, Jezek K, Leutgeb JK, Moser EI, McNaughton BL, Moser MB (2010) Attractor-map versus autoassociation based attractor dynamics in the hippocampal network. J. Neurophysiol,. 104:35-50. Leutgeb JK, Leutgeb S, Moser M-B, Moser EI (2007) Pattern separation in dentate gyrus and CA3 of the hippocampus. Science 315:961-966. Leutgeb JK, Moser EI (2007) Enigmas of the dentate gyrus. Neuron 55:176-178. Leutgeb S, Leutgeb JK (2007) Pattern separation, pattern completion and new neuronal codes within a continuous CA3 map. Learn. Mem. 14:745-757. Leutgeb S, Leutgeb JK, Moser EI, Moser M-B (2006) Fast rate coding in hippocampal CA3 cell assemblies. Hippocampus 16:765-774. Leutgeb JK, Leutgeb S, Treves A, Meyer R, Barnes CA, McNaughton BL, Moser M-B, Moser EI (2005) Progressive transformation of hippocampal neuronal representations in \"morphed\" environments. Neuron 48:345-358. Leutgeb S, Leutgeb JK, Moser M-B, Moser EI (2005) Place cells, spatial maps and the population code for memory. Curr. Opin. Neurobiol. 15:738-746. Leutgeb S, Leutgeb JK, Barnes CA, Moser EI, McNaughton BL, Moser M-B (2005) Independent codes for spatial and episodic memory in hippocampal neuronal ensembles. Science 309:619-623. Leutgeb S, Leutgeb JK, Treves A, Moser M-B, Moser EI (2004) Distinct ensemble codes in hippocampal areas CA3 and CA1. Science 305:1295-1298. \n",
            "-------------------- Scraping faculty url 98/183 --------------------\n",
            "stefan leutgeb  Research Life-long memories are formed at an instant, but considerably outlast the neuronal activity that instigated them or even the synaptic modifications that initially retained them. In addition to early processes at the cellular level, the long-term retention of memories therefore requires that entire populations of neurons in widely distributed neural systems are reorganized. Such reorganization at the systems level is not only needed to ensure that each distinct memory is retained for long time periods, but also to provide a framework that allows for the integration of individual learning events into an accumulating knowledge base. The main objective of research in our laboratory is to describe neuronal mechanisms of long-term memory storage at the systems level and to investigate how coordinated neuronal activity and synaptic plasticity in distributed cell assemblies can result in the formation of new cell assemblies. In addition, we are interested in the translational implications of this basic research and in understanding whether the neurodegenerative processes underlying dementia can result from a failure to appropriately organize neuronal activity and synaptic plasticity during our adult lives. This is addressed by recording from many single neurons (up to 100) in the brain simultaneously and by testing how their activity is coordinated before, during, and long after learning. The recording methods are complemented by computational and analytical approaches, and also by molecular techniques that allow us to manipulate the activity of neuronal networks and to test whether the identified mechanisms are necessary for memory formation. Using these methods, we previously discovered neuronal network mechanisms that combine spatial and nonspatial information in the mammalian hippocampus, and showed that orthogonal encoding of the two types of information is used to generate very different neuronal firing patterns for very similar sensory input. Such pattern separation is thought to be a prerequisite for storing a large number of separate memories. To test this hypothesis, we currently investigate how multiple memories are encoded in the hippocampus as well as in a more widely distributed cortical network. In complementary research program, we aim to investigate how memory processing is altered in the aged brain and when the brain is affected by neurodegenerative disorders. Since memory processing first appears relatively intact in degenerating neuronal networks, but then catastrophically fails, we aim to determine in which way and to what extent a cell assembly can be degraded before failing to support memory retrieval.  Select Publications Sasaki T, Piatti VC, Hwaun E, Ahmadi S, Lisman JE, Leutgeb S, Leutgeb JK (2018). Dentate network activity is necessary for spatial working memory by supporting CA3 sharp-wave ripple generation and prospective firing of CA3 neurons. Nat Neurosci. doi: 10.1038/s41593-017-0061-5. Zutshi I, Leutgeb JK, Leutgeb S (2017). Theta sequences of grid cell populations can provide a movement-direction signal. Curr Opin Behav Sci. 17:147-154. Diehl GW, Hon OJ, Leutgeb S, Leutgeb JK (2017) Grid and non-grid cells in medial entorhinal cortex represent spatial location and environmental features with complementary coding schemes. Neuron 94, 83-92. Ewell LA, Liang L, Armstrong C, Soltsz I, Leutgeb S, Leutgeb JK (2015) Brain state is a major factor in preseizure hippocampal network activity and influences success of seizure intervention. J Neurosci 35, 15635-48. Schiller D, Eichenbaum H, Buffalo EA, Davachi L, Foster DJ, Leutgeb S, Ranganath C (2015) Memory and Space: Towards an Understanding of the Cognitive Map. J Neurosci 35:13904-13911. Schlesiger MI*, Cannova CC*, Boublil BL, Hales JB, Mankin EA, Brandon MP, Leutgeb JK, Leibold C, Leutgeb S (2015). The medial entorhinal cortex is necessary for temporal organization of hippocampal neuronal activity. Nat Neurosci 18, 1123-1132. *These authors contributed equally Mankin EA, Diehl GW, Sparks FT, Leutgeb S, Leutgeb JK (2015) Hippocampal CA2 activity patterns change over time to a larger extent than between spatial contexts. Neuron 85 190-201. Ewell LA, Leutgeb S. Replay to remember: a boost from dopamine (2014) Nat Neurosci 17, 1629-31. News and Views. Sasaki T, Leutgeb S, Leutgeb, JK (2014) Spatial and memory circuits in the medial entorhinal cortex. Curr Opin Neurobiol 32C, 16-23. Hales JB*, Schlesiger MI*, Leutgeb JK, Squire LR, Leutgeb S**, Clark RE** (2014) Medial entorhinal cortex lesions only partially disrupt hippocampal place cells and hippocampus-dependent place memory. Cell Reports 9, 893-901. *These authors contributed equally. **These authors contributed equally. Brandon, MP, Koenig J, Leutgeb JK, and Leutgeb S (2014) New and distinct hippocampal place codes are generated in a new environment during septal inactivation. Neuron 82, 789-96. Brandon MP, Koenig J, Leutgeb S (2014) Parallel and convergent processing in grid cell, head-direction cell, boundary cell, and place cell networks. Wiley Interdiscip Rev Cogn Sci 5, 207-219. Lu L*, Leutgeb JK*, Tsao A, Henriksen EJ, Leutgeb S, Barnes CA, Witter MP, Moser MB, Moser EI (2013) Impaired hippocampal rate coding after lesions of the lateral entorhinal cortex. Nature Neuroscience 16, 1085-1093. *These authors contributed equally. Dulcis D, Jamshidi P, Leutgeb S, Spitzer NC (2013) Neurotransmitter switching in the adult brain regulates behavior. Science 340, 449-453. Schlesiger MI, Cressey JC, Boublil B, Julie Koenig J, Neal R. Melvin NR, Leutgeb JK, Leutgeb S (2013) Hippocampal activation during the recall of remote spatial memories in radial maze tasks. Neurobiol Learn Mem 106, 324-333. Schlesiger MI, Cressey JC, Boublil B, Julie Koenig J, Neal R. Melvin NR, Leutgeb JK, Leutgeb S (2013) Hippocampal activation during the recall of remote spatial memories in radial maze tasks. Neurobiol Learn Mem 106, 324-333. Mankin EA, Sparks FT, Slayyeh B, Sutherland RJ, Leutgeb S, Leutgeb JK (2012) A neuronal code for extended time in the hippocampus. Proc Natl Acad Sci U S A 109, 19462-19467. Koenig J, Linder AN, Leutgeb JK, Leutgeb S (2011). The spatial periodicity of grid cells is not sustained during reduced theta oscillations. Science 332, 592-595. Download reprint . Colgin LL*, Leutgeb S*, Jezek K, Leutgeb JK, Moser EI, McNaughton BL, Moser MB (2010). Attractor-map versus autoassociation based attractor dynamics in the hippocampal network. J. Neurophysiol. 104, 35-50. *These authors contributed equally. Alme CB, Buzzetti RA, Marrone DF, Leutgeb JK, Chawla MK, Schaner MJ, Bohanick JD, Khoboko T, Leutgeb S, Moser EI, Moser MB, McNaughton BL (2010) Barnes CA. Hippocampal granule cells opt for early retirement. Hippocampus 20, 1109-1123. Kjelstrup KB, Solstad T, Brun VH, Hafting T, Leutgeb S, Witter MP, Moser EI, Moser MB (2008). Finite scale of spatial representation in the hippocampus. Science 321, 140-143. Leutgeb S (2008). Neuroscience. Detailed differences. Science 319, 1623-1624. Brun VH, Leutgeb S, Wu H-Q, Schwarcz R, Witter MP, Moser EI and Moser M-B (2008). Impaired spatial representation in CA1 after lesion of direct input from entorhinal cortex. Neuron 57, 290-302. Leutgeb S, Leutgeb JK (2007). Pattern separation, pattern completion, and new neuronal codes in a continuous CA3 map. Learn. Mem. 14, 745-757. Leutgeb JK, Leutgeb S, Moser M-B, Moser EI (2007). Distinct mechanisms for pattern separation in dentate gyrus and CA3 of the hippocampal formation. Science 315, 961-966. Leutgeb S, Leutgeb JK, Moser EI, Moser M-B (2006). Fast rate coding in hippocampal CA3 cell ensembles. Hippocampus 16, 765-774. Leutgeb S, Leutgeb JK, Moser M-B, Moser EI (2005). Place cells, spatial maps and the population code for memory. Curr. Opin. Neurobiol. 15, 738-746. Leutgeb S, Leutgeb JK, Barnes CA, Moser EI, McNaughton BL, Moser M-B (2005). Independent codes for spatial and episodic memory in hippocampal neuronal ensembles. Science 309, 619-623. \n",
            "-------------------- Scraping faculty url 99/183 --------------------\n",
            "bjorn lillemeier  Research The Lillemeier laboratory studies the spatio-temporal organization of the plasma membrane and its impact on signal transduction. We are specifically interested in T cell antigen recognition, activation and signaling. We are also investigating other aspects of membrane biology, e.g. virus-host interactions, anthrax toxin uptake and membrane aging. To achieve this, our research team uses cutting edge fluorescence microscopy techniques, e.g. super-resolution microscopy and cross-correlation spectroscopy. Another focus in the laboratory is to study the qualitative and quantitative differences of signal transduction that determine T cell faith and function. To this end, we use innovative biochemical and molecular biological approaches.  Select Publications Klammt C., Lillemeier B.F. (2012). How membrane structures control T cell signaling. Frontiers Immunol. 3:291. Kasuboski J.M., Sigal Y.J., Joens M.S., Lillemeier B.F., and Fitzpatrick J.A.J. (2012). Superresolution microscopy: A comparative treatment. Curr. Protocols Cytometry 2.17.12.17.24. Fitzpatrick J.A., Lillemeier B.F. (2011). Fluorescence correlation spectroscopy: linking molecular dynamics to biological function in vitro and in situ . Curr. Opin. Struct. Biol. 21:650-660. Lillemeier B.F. and Davis M.M. (2011). Probing the plasma membrane structure of immune cells through the analysis of membrane sheets by electron microscopy. Meth. Mol. Biol. 748:169-182. Kuhns M.S., Girvin A.T., Klein L.O., Chen R., Jensen K.D., Newell E.W., Huppa J.B., Lillemeier B.F., Huse M., Chien Y.H., Garcia K.C., Davis M.M. (2010). Evidence for a functional sidedness to the alpha-beta-TCR. PNAS USA 107:5094-5099. Huppa J.B., Axmann M., Mrtelmaier M.A., Lillemeier B.F., Newell E.W., Brameshuber M., Klein L.O., Schtz G.J., Davis M.M. (2010). Measuring the affinity and kinetics of T cell receptor binding in situ . Nature 463:963-967. Lillemeier B.F., Mrtelmaier M.A., Forstner M.B., Huppa J.B., Groves J.T., Davis M.M. (2010). TCR and LAT are expressed on separate protein islands onT cell membranes and concatenate during activation. Nature Immunol. 11:90-96. Davis M.M., Krogsgaard M., Huse M., Huppa J., Lillemeier B.F., Li Q.J. (2007). T cells as a self-referential, sensory organ. Ann. Rev. Immunol. 25:681-695. Lillemeier B.F., Pfeiffer J.R., Surviladze Z., Wilson B.S., Davis M.M. (2006). Plasma membrane-associated proteins are clustered into islands attached to the cytoskeleton. PNAS USA 103:18992-18997. Huse M., Lillemeier B.F., Kuhns M.S., Chen D.S., Davis M.M. (2006). T cells use two directionally distinct pathways for cytokine secretion. Nature Immunol. 7:247-55. Kerr I.M., Costa-Pereira A.P., Lillemeier B.F., Strobl B. (2003). Of JAKs, STATs, blind watchmakers, jeeps and trains. FEBS Lett. 546:1-5. Hilkens CM, Is'harc H, Lillemeier BF, Strobl B, Bates PA, Behrmann I, Kerr IM. (2001). A region encompassing the FERM domain of Jak1 is necessary for binding to the cytokine receptor gp130. FEBS Lett. 7:87-91. Lillemeier B.F., Kster M., Kerr I.M. (2001). STAT1 from the cell membrane to the DNA. EMBO J. 20:2508-2517. van Ham M., van Lith M., Lillemeier B.F., Tjin E., Gruneberg U., Rahman D., Pastoors L., van Meijgaarden K., Roucard C., Trowsdale J., Ottenhoff T., Pappin D., Neefjes J. (2000). Modulation of the major histocompatibility complex class II-associated peptide repertoire by human histocompatibility leukocyte antigen (HLA)-DO. J. Exper. Med. 191(7):1127-1136. \n",
            "-------------------- Scraping faculty url 100/183 --------------------\n",
            "byungkook lim  Biography Byungkook Lim received his Ph.D. in Molecular & Cell Biology from UC Berkeley with Dr. Mu-ming Poo and carried out his postdoctoral training with Dr. Robert Malenka in the Department of Psychiatry and Behavioral Science at the Stanford University School of Medicine. Byungkook is a recipient of Kingenstein-Simmons Fellowship, Searle Scholar Fellowship, NIMH Biobehavioral Research Awards for Innovative New Scientists (BRAINS) and BRAIN Initiative Cell Census Network (BICCN) awards.  Research Lab of Functional Anatomy of the Neural Circuitry The basal ganglia and interconnected cortical, amygdalar and thalamic circuits subserve several crucial brain functions including reward-based learning, motor learning, motivated behavior and stress-related behaviors. Dysfunction in these circuits due to alterations of synaptic properties has been proposed to play a critical role in a variety of neuropsychiatric disorders including anxiety, addiction, autism spectrum disorder and major depressive disorder. However, the brain circuits relaying information from or to basal ganglia and their detailed synaptic organization have not been clearly delineated. The main objective of research in our laboratory is to understand the role of the circuit-specific synaptic organization in basal ganglia in animal models of neuropsychiatric disease including drug addiction, feeding disorder, depressive disorder, and movement disorder. Given the circuit and synaptic complexity of the brain, characterizing specific neural circuitry responsible for behavioral changes has been challenging. Dendtiric spines of amygdala neurons projecting to dopamine receptor 1-expressing medium spiny neurons (D1-MSNs) in nucleus accumbens To address the anatomical and functional properties of basal ganglia circuitry, our laboratory applies and develops a variety of molecular strategies (ex. virus-mediated tracing and gene expression, transgenic mice expressing marker proteins in specific population of neurons) together with electrophysiological approaches. Furthermore, we applies cell-type specific or input-specific optogenetic manipulation in vitro and in vivo in pharmacological animal models of human disease as well as a battery of behavioral testing. Future research in my own lab will continue with this strategy and set out to anatomically and functionally define neuronal identity and connectivity of brain circuits beyond basal ganglia and their interplay in mediating reward- and motivation-related behavior in the pathogenesis of neuropsychiatric and neuropathological disease. Double rabies virus-mediated retrograde tracing showing cortical neurons projecting to dorsal striatum (red) and neurons projecting to nucleus accumbens (green).  Select Publications Shin, S., Pribiag, H., Lilascharoen, V., Knowland, D., Wang, X., Lim, B. K. (2018) Drd3 signaling in the lateral septum mediates early life stress-induced social dysfunction. Neuron, 2018 Jan 3. Liu, J., Conde, K., Zhang, P., Lilascharoen, V., Xu, Z., Lim, B. K., Seeley, R. J., Zhu, J. J., Scott, M. M., Pang, Z.P. (2017) Enhanced AMPA Receptor Trafficking Mediates the Anorexigenic Effect of ndogenous Glucagon-like Peptide-1 in the Paraventricular Hypothalamus. Neuron. 2017 Oct 13. Knowland, D., Lilascharoen, V., Pacia, C. C., Shin, D., Wang, E., Lim, B. K. (2017) Distinct ventral pallidal neural populations mediate separate symptoms of depression. Cell Jul 13;170(2):284-297 Yoo, J. H., Zell, V., Wu, J., Punta, C., Ramajayam, N., Shen, X., Faget, L., Lilascharoen, V., Lim, B. K., Hnasko, T. S. (2017) Activation of Pedunculopontine Glutamate Neurons Is Reinforcing. J Neurosci. 37(1):38-46. Rothwell, P. E., Hayton, S. J., Sun, G. L., Fuccillo, M. V., Lim, B. K., Malenka, R. C. (2015) Input- and Output-Specific Regulation of Serial Order Performance by Corticostriatal Circuits. Neuron 88(2):345-356. Schwartz, N., Temkin, P., Jurado, S., Lim, B. K., Polepalli, J. S., Malenka, R. C. (2014) Decreased motivation during chronic pain requires plasticity in the nucleus accumbens core. Science 345(6196) 535-542. Rothwell, P. E, Fuccillo, M. V., Maxeiner, S., Hayton, S. J., Lim, B. K., Malenka, R. C., Sdhof, T. C. (2014) Autism-associated Neuroligin-3 mutations impair striatal circuits underlying repetitive motor routins. Cell. 158(1):198-212. Lammel, S.*, Lim, B. K.*, Malenka, R. C. (2014) Reward and aversion in a heterogeneous midbrain dopamine system. Neuropharmacology. 76 Pt. B: 351-9 (* equal contribution) Kim, S. Y., Adhikari, A., Lee, S. Y., Marshel, J. H., Kim, C. K., Mallory, C. S., Lo, M., Pak, S., Mattis, J., Lim, B. K., Malenka, R. C., Warden, M. R., Neve, R., Tye, K. M., Deisseroth, K. (2013) Diverging neural pathways assemble a behavioural state from separable features in anxiety. Nature. 496(7444): 219-223. Lammel, S.*, Lim, B. K.*, Ran, C., Huang, K. W., Tye, K., Deisseroth, K., Malenka, R. C. (2012) Input-specific control of reward and aversion in the ventral tegmental area. Nature (Article) 491(7423): 212-217 (* equal contribution) Lim, B. K., Huang, K. W., Grueter, B. A., Rothwell, P. E., Malenka, R. C. (2012) Anhedonia requires MC4R-mediated synaptic adaptation in nucleus accumbens. Nature (Article) 487(7406): 183-189. Shelly, M., Cancedda, L., Lim, B. K., Popescu, A. T., Cheng, P. L., Gao, H., Poo, M. M. (2011) Semaphorin3A regulates neuronal polarization by suppressing axon formation and promoting dendrite growth. Neuron. 71(3):433-446. Lu, H., Cheng, P. L., Lim, B. K., Khoshnevisrad, N., Poo, M. M. (2010) Elevated BDNF expression after cocaine withdrawal facilitates LTP in medial prefrontal cortex by suppressing GABA inhibition. Neuron. 67(5):821-833. Lim, B. K., Cho, S. J., Sumbre, G., Poo, M. M. (2010) Region-specific contribution of ephrin-B and Wnt signaling to receptive field plasticity in developing optic tectum. Neuron. 65(6):899-911. Shelly, M.*, Lim, B. K.*, Cancedda, L., Heilshorn, S. C., Gao, H., Poo, M. M. (2010) Local and long-range reciprocal regulation of cAMP and cGMP in axon/dendrite formation. Science. (Article) 327(5965):547-552. (* equal contribution) Lu, H., Lim, B., Poo, M. M. (2009) Cocaine exposure in utero alters synaptic plasticity in the medial prefrontal cortex of postnatal rats. J Neurosci. 29(40):12664-12674. Lim, B. K., Matsuda, N., Poo, M. M. (2008) Ephrin-B reverse signaling promotes structural and functional synaptic maturation in vivo. Nat Neurosci. 11(2):160-169. \n",
            "-------------------- Scraping faculty url 101/183 --------------------\n",
            "stanley lo  Biography Stanley Lo received his Ph.D. in biochemistry from Harvard University and was a senior research associate at the Searle Center for Advancing Learning and Teaching at Northwestern University. He joined the faculty of Biological Sciences in 2014. Stanley is the biochemistry editor for CourseSource, an open-access journal of peer-reviewed teaching resources for college biological science courses. He was a National Academies Education Fellow in the Life Sciences in 2011-2012 and has been a National Academies Education Mentor in the Life Sciences since 2013.  Research Our research centers upon two key aspects of undergraduate biology education: faculty teaching and student learning. The broad mission is to understand how faculty teaching promotes student learning, what factors support sustainable classroom innovations, and how students learn core concepts and develop core competencies in biological sciences. Specifically, we investigate how faculty conceptions (beliefs and understanding) and approaches (intentions and strategies) to teaching inform instructional practices in the classroom. Current projects include identifying specific faculty actions in different instructional practices (e.g. the use of personal response devices or clickers) that correlate with positive student outcomes, examining the underlying conceptions and approaches to teaching that are aligned with effective instructional practices, and exploring factors that promote sustainable faculty implementation of these practices. We use a combination of quantitative and qualitative methods (e.g. surveys, observations, interviews, and statistics) to examine how faculty teach in a longitudinal timeframe (i.e. over multiple years and many courses). For student learning, we are interested in defining learning progressions for core concepts and competencies in biological sciences. Learning progressions are descriptions of increasingly sophisticated ways of thinking about a concept or performing a task. Essentially, learning progressions are conceptual pathways that students navigate as they develop understanding of a concept or skills in a competency. Defining learning progressions will provide the theoretical foundation to design instructional material that specifically targets transitions along these conceptual pathways. We are interested in understanding how students and scientists formulate arguments based on evidence and to develop instructional material that promote learning in scientific argumentation.  Select Publications Lo SM, McElroy KA, Francis NJ. (2013) Chromatin modification by PSC occurs at one PSC per nucleosome and does not require the acidic patch of histone H2A. PLoS ONE 7(10): e47162. PMID: 23071745. DOI: 10.1371/journal.pone.0047162 Lo SM*, Follmer NE*, Lengsfeld BM, Madamba EV, Seong S, Grau DJ, Francis NJ. (2012) A bridging model for persistence of a Polycomb group protein complex through DNA replication in vitro. Molecular Cell 46(6): 784-96. PMID: 22749399. DOI:10.1016/j.molcel.2012.05.038 *equal contribution Lo SM, Francis NJ. (2010) Inhibition of chromatin remodeling by Polycomb group protein posterior sex combs is mechanistically distinct from nucleosome binding. Biochemistry 49(44): 9438-48. PMID: 20873869. DOI: 10.1021/bi100532a Lo SM, Ahuja NK, Francis NJ. (2008) Polycomb group protein suppressor 2 of zeste is a functional homolog of posterior sex combs. Molecular and Cellular Biology 29(2): 515-525. PMID: 18981224. 10.1128/MCB.01044-08 \n",
            "-------------------- Scraping faculty url 102/183 --------------------\n",
            "li-fan lu  Biography Li-Fan Lu received his Ph.D. in Immunology from Dartmouth College. He did his postdoctoral training first at the University of Washington and then at the Memorial Sloan-Kettering Cancer Center where he was the Leukemia and Lymphoma Society Fellow. In 2011, he received the K99/R00 Career Development Award through NIAID.  Research During the last decade a unique mechanism of negative regulation of immune responses and inflammation by a dedicated population of so-called regulatory T (Treg) cells has become the focus of intensive investigation. Expression of the transcription factor Foxp3, a hallmark of Treg cells, is necessary for the establishment and maintenance of a distinct transcriptional program that determines metabolic, signaling, and effector features (e.g., suppressive function), distinguishing these cells from other T cell lineages. The long-term goals of our research are to identify the molecular determinants of differentiation and function of Treg cells and to ultimately gain insight into the complex biology of these cells, including the maintenance of immunological tolerance to self and the regulation of immune responses to pathogens, commensals, and tumors. 1. Identification of miRNAs crucial for mediating Treg cell biology in healthy and diseased mice Several thousands genes are differentially expressed in Foxp3+ Treg cells, including a subset of non-coding small regulatory microRNA (miRNA), some of which are directly regulated by Foxp3 suggesting a role for miRNA-mediated regulation of gene expression in Treg cell differentiation, maintenance or function. Catastrophic consequences and the development of complex disease phenotypes in mice harboring Treg cells devoid of miRNA suggested miRNAs control multiple facets of Treg cell biology (Fig.1). Furthermore, our subsequent studies in miR-155 and miR-146a have demonstrated that individual miRNA could affect distinct aspect of Treg cell biology. By employing genetic, biochemical, immunological approaches and whole animal experimentation, we aim to identify miRNA expression profiles in Treg cells isolated from different tissues in unmanipulated mice as well as in mice suffering different pathologies. Moreover, we are interested in examining how miRNAs themselves are regulated in response to different environmental cues. Ultimately, we hope to elucidate how such regulation of miRNA expression impacts their respective targets and how the dynamic miRNA-dependent regulation enforces Treg cell functionality in diverse disease settings. 2. Elucidating molecular and cellular mechanisms by which Treg cells control type I inflammation Recent studies revealed that the expression of Th lineage-specific transcription factors in Treg cells is crucial for Treg cell-mediated control of corresponding immune responses in particular tissue and inflammatory settings. Our analysis of the role of miR-146a and SOCS1 in Treg cell-dependent Th1 regulation have further extended this concept and demonstrated that both lack of Stat1 and unrestrained Stat1 activation in Treg cells would result in a similar breakdown of immunologic tolerance (Fig.2). These results suggest that Treg cell-mediated suppression of a given Th-specific immunopathology is depending upon an optimal range of activation in corresponding Th-lineage specific transcription factors. One fundamental question that remained is how Treg cells control Th1 responses. At the molecular level, we are interested in identifying important effector molecules and signalling pathways underlying Treg cell-mediated Th1 regulation. At the cellular level, we hope to understand the complex interplay between Treg cells, effector T cells and other immune cells in the surrounding environment specifically in a type I inflammatory setting. Moreover, IFN-mediated Th1 responses and Treg cells have been associated with inflammation in many clinical settings such as infection (e.g., T. gondii), autoimmune diseases (e.g., multiple sclerosis and rheumatoid arthritis), tumors, and organ transplantation. Through our study, we hope to establish the foundation for the future development of means to manipulating Treg and T effector cell function for better treatment in human patients.  Select Publications Cho, S.*, Lee, H.M.*, Yu, I.S., Choi, Y.S., Huang, H.Y., Hashemifar, S.S., Lin, L.L., Chen, M.C., Afanasiev, N.D., Khan, A.A., Lin, S.W., Rudensky, A.Y., Crotty, S. and Lu, L.F. (*these authors contributed equally) 2018. Differential cell-intrinsic regulations of germinal center B and T cells by miR-146a and miR-146b. Nat Commun. 9(1):2757. PMCID: PMC6048122 Becher, B., Waisman, A. and Lu, L.F. 2018. Conditional Gene-Targeting in Mice: Problems and Solutions. Immunity 48(5):835-836. Wu, C.J. and Lu, L.F. 2017. MicroRNA in Immune Regulation. In: . Curr Top Microbiol Immunol. Springer, Berlin, Heidelberg Cho, S., Wu, C.J., Nguyen, D.T., Lin, L.L., Chen, M.C., Khan, A.A., Yang, B.H., Fu, W., and Lu, L.F. 2017. A novel miR-24-TCF1 axis in modulating effector T cell responses. J Immunol. 198(10):3919-26. PMCID: PMC5437976 Cruz, L.O., Hashemifar, S.S., Wu, C.J., Cho, S., Nguyen, D.T., Lin, L.L., Khan, A.A. and Lu, L.F. 2017. Excessive expression of miR-27 impairs regulatory T cell-mediated immunological tolerance. J. Clin. Invest. 127(2):530-42. PMCID: PMC5272185 Cho, S.*, Wu, C.J.*, Yasuda, T.*, Cruz, L.O., Khan, A.A., Lin, L.L., Nguyen, D.T., Miller, M., Lee, H.M., Kuo, M.L., Broide, D.H., Rajewsky, K., Rudensky, A.Y. and Lu, L.F. (*these authors contributed equally) 2016. MiR-23~27~24 clusters control effector T cell differentiation and function. J. Exp. Med. 213(2):235-49. PMCID: PMC4749926 Lu, L.F.*, Gasteiger, G.*, Yu, I.S., Chaudhry, A., Hsin, J.P., Lu, Y., Bos, P.D., Lin, L.L., Zawislak. C.L., Cho, S., Sun, J.C., Leslie, C.S., Lin, S.W. and Rudensky, A.Y. (*these authors contributed equally) 2015. A single miRNA-mRNA interaction affects the immune response in a context- and cell-type-specific manner. Immunity 43(1):52-64. PMCID: PMC4529747 Lee, H.M., Fleige, A., Forman, R., Cho, S., Khan, A.A., Lin, L.L., Nguyen, D.T., OHara-Hall, A., Yin, Z., Hunter, C.A., Muller, W. and Lu, L.F. 2015. IFN signaling endows DCs with the capacity to control type I inflammation during parasitic infection through promoting T-bet+ regulatory T cells. PLOS Pathog. 11(2):e1004635. PMCID: PMC4450074 Josefowicz, S.Z.*, Lu, L.F.* and Rudensky, A.Y. (*these authors contributed equally) 2012. Regulatory T cells: mechanisms of differentiation and function. Annu. Rev. Immunol. 30:531-64 Lu, L.F., Boldin, M.P., Chaudhry, A., Lin, L.L., Taganov, K.D., Hanada, T., Yoshimura, A., Baltimore, D. and Rudensky, A.Y. 2010. Function of miR-146a in controlling Treg cell-mediated regulation of Th1 responses. Cell 142(6): 914-29. PMCID: PMC3049116 Lu, L.F. and Rudensky A. 2009. Molecular orchestration of differentiation and function of regulatory T cells. Genes Dev. 23(11): 1270-1282. PMCID: PMC2763538 Lu, L.F.*, Thai, T.H.*, Calado, D.P.*, Chaudhry, A., Kubo, M., Tanaka, K., Loeb, G.B., Lee, H., Yoshimura, A., Rajewsy, K. and Rudensky, A.Y. (*these authors contributed equally) 2009. Foxp3-dependent microRNA155 confers competitive fitness to regulatory T cells through targeting SOCS1 protein. Immunity 30(1):80-91. PMCID:PMC2654249 Liston, A.*, Lu, L.F.*, Carroll D.O., Tarakhovsky, A. and Rudensky, A.Y. (*these authors contributed equally) 2008. Dicer-dependent microRNA pathway safeguards regulatory T cell function. J. Exp. Med. 205(9):1993-2004. PMCID: PMC2526195 Lu, L.F.*, Ahonen, C.L.*, Lind, E.F., Raman V.S., Cook, W.J., Lin, L.L. and Noelle R.J. (*these authors contributed equally) 2007. The in vivo function of the alternative TRAF2 binding site on CD40-driven B cell growth and differentiation. Blood 110(1): 193-200. PMCID: PMC1896113 Lu, L.F., Lind, E.F., Gondek, D.C., Bennett, K.A., Gleeson, M.W., Pino-Lagos, K., Scott, Z.A., Coyle, A.J., Reed, J.L., Van Snick, J, Strom, T.B., Zheng, X.X. and Noelle R.J. 2006. Mast cells as essential intermediaries in regulatory T cell tolerance. Nature 442 (7106): 997-1002. \n",
            "-------------------- Scraping faculty url 103/183 --------------------\n",
            "victoria lundblad \n",
            "-------------------- Scraping faculty url 104/183 --------------------\n",
            "jens lykke-andersen  Biography Jens Lykke-Andersen received his Ph.D. from University of Copenhagen, Denmark in 1997. He was a postdoctoral fellow at Yale University Medical School before joining the faculty of MCD Biology at University of Colorado Boulder in 2001. He was named a Pew Scholar in 2003. He joined the Division of Biological Sciences at UCSD in 2009.  Research Our laboratory studies the mechanisms of regulation of translation and mRNA turnover in human gene expression. Research in recent years has revealed the importance of regulated mRNA translation and stability in the correct control of gene expression, and how its deregulation can lead to disease. Many of the general factors that direct mRNA translation and the enzymes that degrade mRNAs have been described in recent years. Our laboratory is interested in how these factors are differentially regulated on individual mRNAs to control their rates of translation and mRNA turnover, and how this is regulated by cell signaling. Current research questions in the lab include: How are specific mRNAs targeted for rapid decay? The mechanisms by which specific mRNAs are targeted for rapid degradation remains poorly understood. We study the basic mechanisms that underlie the recognition and degradation of mRNAs targeted for various mRNA decay pathways, including the nonsense-mediated mRNA decay (NMD) pathway, an mRNA quality control pathway targeting aberrant mRNAs that fail to produce full-length protein, and the mRNA decay pathway that targets mRNAs containing AU-rich elements (AREs) in their 3UTRs for rapid degradation. How is translation and mRNA decay regulated by signaling? The response of cells to changes in their environment often requires co-regulation of gene networks, but little is known about how this occurs at the post-transcriptional level. In two recent studies from the lab, we identified the stress granule-associated proteins TIA-1 and TIAR as key translational regulators in response to cell growth conditions of the network of 5TOP mRNAs encoding protein biosynthesis factors (Damgaard and Lykke-Andersen, Genes&Dev, 2011), and we identified a mechanism by which an mRNA decay factor, TTP, which controls the degradation of mRNAs encoding cytokines, fails to recruit deadenylases to target mRNAs when TTP becomes phosphorylated in response to infection, thereby allowing cytokine production and an immune response (Clement et al., MCB, 2011). A current major focus of the lab is to understand the mechanisms by which signaling events modulate the activities of factors that control translation and degradation of regulated mRNA networks. How do mRNA turnover pathways deal with the mRNP? Just like our genomic DNA is coated in chromatin, mRNAs are coated with proteins in mRNPs. We recently observed that in the nonsense-mediated mRNA decay (NMD) pathway, failure of a central RNA helicase, called Upf1, to hydrolyze ATP causes the accumulation of partially degraded mRNP intermediates resistant to exonucleolytic decay (Franks et al., Cell, 2010). These observations suggest that active disassembly of the mRNP constitutes a critical step in mRNA turnover. Consistent with this idea, earlier observations primarily in yeast have shown that stalled ribosomes and strong RNA structure can act as obstacles to mRNA degradation, and that cap binding factors and poly-A binding protein can inhibit decapping and deadenylation, respectively. A current major focus of the laboratory is to understand the importance of ATPases/RNA helicases and of post-translational mRNP modifications in mRNA turnover, to learn whether mRNP remodeling is as critical to mRNA turnover as chromatin remodeling is to transcription.  Select Publications Dumdie JN, Cho K, Ramaiah M, Skarbrevik D, Mora-Castilla S, Stumpo DJ, Lykke-Andersen J, Laurent LC, Blackshear PJ, Wilkinson MF, Cook-Andersen H. (2018) Chromatin Modification and Global Transcriptional Silencing in the Oocyte Mediated by the mRNA Decay Activator ZFP36L2. Dev Cell. 44(3):392-402. Lardelli RM*, Schaffer AE*, Eggens VR*, Zaki MS, Grainger S, Sathe S, Van Nostrand EL, Schlachetzki Z, Rosti B, Akizu N, Scott E, Silhavy JL, Heckman LD, Rosti RO, Dikoglu E, Gregor A, Guemez-Gamboa A, Musaev D, Mande R, Widjaja A, Shaw TL, Markmiller S, Marin-Valencia I, Davies JH, de Meirleir L, Kayserili H, Altunoglu U, Freckmann ML, Warwick L, Chitayat D, Blaser S, alayan AO, Bilguvar K, Per H, Fagerberg C, Christesen HT, Kibaek M, Aldinger KA, Manchester D, Matsumoto N, Muramatsu K, Saitsu H, Shiina M, Ogata K, Foulds N, Dobyns WB, Chi NC, Traver D, Spaccini L, Bova SM, Gabriel SB, Gunel M, Valente EM, Nassogne MC, Bennett EJ, Yeo GW, Baas F*, Lykke-Andersen J*, Gleeson JG*. (2017) Biallelic mutations in the 3' exonuclease TOE1 cause pontocerebellar hypoplasia and uncover a role in snRNA processing. Nat Genet. 49(3):457-464. D'Lima NG, Ma J, Winkler L, Chu Q, Loh KH, Corpuz EO, Budnik BA, Lykke-Andersen J, Saghatelian A, Slavoff SA. (2017) A human microprotein that interacts with the mRNA decapping complex. Nat Chem Biol. 13(2):174-180. Martinez FJ, Pratt GA, Van Nostrand EL, Batra R, Huelga SC, Kapeli K, Freese P, Chun SJ, Ling K, Gelboin-Burkhart C, Fijany L, Wang HC, Nussbacher JK, Broski SM, Kim HJ, Lardelli R, Sundararaman B, Donohue JP, Javaherian A, Lykke-Andersen J, Finkbeiner S, Bennett CF, Ares M Jr, Burge CB, Taylor JP, Rigo F, Yeo GW. (2016) Protein-RNA Networks Regulated by Normal and ALS-Associated Mutant HNRNPA2B1 in the Nervous System. Neuron. 92(4):780-795. Durand S, Franks TM, Lykke-Andersen J. (2016) Hyperphosphorylation amplifies UPF1 activity to resolve stalls in nonsense-mediated mRNA decay. Nat Commun. 7:12434. Arribas-Layton M, Dennis J, Bennett EJ, Damgaard CK, Lykke-Andersen J. (2016) The C-Terminal RGG Domain of Human Lsm4 Promotes Processing Body Formation Stimulated by Arginine Dimethylation. Mol Cell Biol. 36(17):2226-35. Fu R, Olsen MT, Webb K, Bennett EJ, Lykke-Andersen J. (2016) Recruitment of the 4EHP-GYF2 cap-binding complex to tetraproline motifs of tristetraprolin promotes repression and degradation of mRNAs with AU-rich elements. RNA. 22(3):373-82. Wang Y, Arribas-Layton M, Chen Y, Lykke-Andersen J, Sen GL. (2015) DDX6 Orchestrates Mammalian Progenitor Function through the mRNA Degradation and Translation Pathways. Mol Cell. 60(1):118-30. Lee SR, Pratt GA, Martinez FJ, Yeo GW, Lykke-Andersen J. (2015) Target Discrimination in Nonsense-Mediated mRNA Decay Requires Upf1 ATPase Activity. Mol Cell. 59(3):413-25. Erickson SL, Corpuz EO, Maloy JP, Fillman C, Webb K, Bennett EJ, Lykke-Andersen J. (2015) Competition between Decapping Complex Formation and Ubiquitin-Mediated Proteasomal Degradation Controls Human Dcp2 Decapping Activity. Mol Cell Biol. 35(12):2144-53. Hausburg MA, Doles JD, Clement SL, Cadwallader AB, Hall MN, Blackshear PJ, Lykke-Andersen J, Olwin BB. (2015) Post-transcriptional regulation of satellite cell quiescence by TTP-mediated mRNA decay. Elife. 4:e03390. Toma KG, Rebbapragada I, Durand S, Lykke-Andersen J. (2015) Identification of elements in human long 3' UTRs that inhibit nonsense-mediated decay. RNA. 21(5):887-97. Reznik B, Clement SL, Lykke-Andersen J. (2014) hnRNP F complexes with tristetraprolin and stimulates ARE-mRNA decay. PLoS One. 9(6):e100992. Lykke-Andersen J, Bennett EJ. (2014) Protecting the proteome: Eukaryotic cotranslational quality control pathways. J Cell Biol. 204(4):467-76. Damgaard CK, Lykke-Andersen J. (2013) Regulation of ARE-mRNA Stability by Cellular Signaling: Implications for Human Cancer. Cancer Treat Res. 158:153-80. Lee SR, Lykke-Andersen J. (2013) Emerging roles for ribonucleoprotein modification and remodeling in controlling RNA fate. Trends Cell Biol. 23(10):504-10. Durand S, Lykke-Andersen J. (2013) Nonsense-mediated mRNA decay occurs during eIF4F-dependent translation in human cells. Nat Struct Mol Biol. 20(6):702-9. Arribas-Layton M, Wu D, Lykke-Andersen J, Song H. (2013) Structural and functional control of the eukaryotic mRNA decapping machinery. Biochim Biophys Acta. 1829(6-7):580-9. Brannan K, Kim H, Erickson B, Glover-Cutter K, Kim S, Fong N, Kiemele L, Hansen K, Davis R, Lykke-Andersen J, Bentley DL. (2012) mRNA decapping factors and the exonuclease Xrn2 function in widespread premature termination of RNA polymerase II transcription. Mol Cell. 46(3):311-24. Damgaard CK, Lykke-Andersen J. (2011) Translational coregulation of 5'TOP mRNAs by TIA-1 and TIAR. Genes Dev. 25(19):2057-68. Durand S. and Lykke-Andersen J. (2011) SnapShot: Nonsense-Mediated mRNA Decay. Cell. 145(2): 324. (Mini-Review). http://www.ncbi.nlm.nih.gov/pubmed/21496649 Clement S.L., Scheckel C., Stoecklin G. and Lykke-Andersen J. (2011) Phosphorylation of Tristetraprolin by MK2 Impairs AU-Rich Element mRNA Decay by Preventing Deadenylase Recruitment. Mol Cell Biol. 31(2): 256-266. http://www.ncbi.nlm.nih.gov/pubmed/21078877 Erickson S. and Lykke-Andersen J. (2011) Cytoplasmic mRNP granules at a glance. J. Cell Sci. 124: 293-297 (Review). http://www.ncbi.nlm.nih.gov/pubmed/21242308 Franks T.M., Singh G. and Lykke-Andersen J. (2010) Upf1 ATPase-dependent mRNP disassembly is required for completion of nonsense-mediated mRNA decay. Cell. 143(6): 938-950. http://www.ncbi.nlm.nih.gov/pubmed/21145460 Rebbapragada, I. and Lykke-Andersen, J. (2009) Execution of Nonsense-Mediated mRNA Decay: What Defines a Substrate? Curr. Opin. Cell Biol. 21: 394-402 (Review). http://www.ncbi.nlm.nih.gov/pubmed/19359157 Singh, G., Rebbapragada, I., and Lykke-Andersen, J. (2008) A competition between stimulators and antagonists of Upf complex recruitment governs human nonsense-mediated mRNA decay. PLoS Biology 6(4): 860-871. http://www.ncbi.nlm.nih.gov/pubmed/18447585 Franks, T. and Lykke-Andersen, J. (2008) The control of mRNA decapping and processing body formation. Molecular Cell. 32: 605-615 (Review). http://www.ncbi.nlm.nih.gov/pubmed/19061636 Singh, G., Jakobs, S., Kleedehn, J., and Lykke-Andersen, J. (2007) Communication with the exon-junction complex and activation of nonsense-mediated decay by human Upf proteins occur in the cytoplasm. Molecular Cell. 27: 780-792. http://www.ncbi.nlm.nih.gov/pubmed/17803942 Franks, T. and Lykke-Andersen, J. (2007) TTP and BRF proteins nucleate processing body formation to silence mRNAs with AU-rich elements. Genes Dev. 21: 719-735. http://www.ncbi.nlm.nih.gov/pubmed/17369404 Wagner, E., Clement, S.L. and Lykke-Andersen, J. (2007) An unconventional human Ccr4-Caf1 deadenylase complex in nuclear cajal bodies. Mol. Cell. Biol. 27: 1686-1695. http://www.ncbi.nlm.nih.gov/pubmed/17178830 Fenger-Gron, M., Fillman, C., Norrild, B. and Lykke-Andersen, J. (2005) Multiple human processing body factors and the ARE-binding protein TTP activate decapping. Molecular Cell 20: 905-915. http://www.ncbi.nlm.nih.gov/pubmed/16364915 Lykke-Andersen, J. and Wagner, E. (2005) Recruitment and activation of mRNA decay enzymes by two ARE-mediated decay activation domains in the proteins TTP and BRF-1. Genes Dev. 19: 351-361. http://www.ncbi.nlm.nih.gov/pubmed/15687258 \n",
            "-------------------- Scraping faculty url 105/183 --------------------\n",
            "dmitry lyumkis \n",
            "-------------------- Scraping faculty url 106/183 --------------------\n",
            "eduardo macagno  Biography Eduardo Macagno received his Ph.D. from Columbia University in 1968. He was on the faculty of Columbia from 1973 to 2000. From 1993 to 2000 he served as Associate Vice President for Research and Graduate Education and Dean of the Graduate School of Arts and Sciences at Columbia. He joined the faculty of the Division of Biological Sciences as Founding Dean in 2001. He is the Editor of Developmental Neurobiology and the President of the Academy of Neuroscience for Architecture. He also serves on review committees for the Life Sciences Research Foundation and the W. M. Keck Foundation.  Research In the area of basic neurobiology, our laboratory is currently investigating interactions leading to establishment of neuronal arbors with specific morphologies. In particular, we have focused on the mechanisms which receptors and signaling factors use to mediate self- and homologue-avoidance in the process of defining cell arbor boundaries, a phenomenon that is known as tiling and defines the topology of sensory and motor fields Figure 1. Pair of Live Medicinal Leech Embryos Our studies are carried out in the medicinal leech (Hirudo medicinalis, Hirudo verbana), for which we have created genomic and transcriptomic databases and are in the process of annotating the genome. Its nervous system provides a very accessible experimental platform for molecular and cellular studies; we have developed or adapted all the tools required for modulating the expression of specific genes in individual embryonic or adult identified leech neurons (see Figure 1). Figure 2. Bilateral pair of electrically-coupled central neurons in the medicinal leech We are particularly interested currently in the roles of gap junctions (GJs) in the early differentiation of the central nervous system. We have cloned and characterized the expression patterns of the 21 leech GJ genes (innexins) in the Hirudo genome; 14 or 15 are expressed in central and/or peripheral neurons in the embryo and adult (see Figure 2). In one set of studies, we have shown that innexins can not only form gap junctions but can also form functional hemi-channels that can provide intracellular and intercellular signaling via small molecules, such as ATP. Individual identified central neurons express different subsets of innexin genes, which we have proposed as the determining factor in the specificity of electrically-coupled networks. In one set of studies, we also demonstrated that expression of certain pan-neuronal innexins is necessary for the formation of of gap junctions, while ectopic expression of certain other innexins is sufficient for the formation of novel coupled networks in the CNS. A major effort at present is to explore, using null and functionally-deficient mutations, the role of innexins in interactions between segmental homologs that result in the retraction of some neuronal projections and the maintenance of others (see Figure 3). Figure 3. Some pairs of homologous central neurons retract overlapping projections (AP cell; panel A, panel D), while others form GJs and are retained without major overlap (S cell; panel B) or do not form GJs and ignore each other (P cell; panel C). Knockout of a necessary GJ gene, Inx1, leads to the retention of AP interganglionic projections (cf. panels E and F), with continued growth into adjacent segment (panels G and H) In a very different area of research, we are also involved in a multidisciplinary project at the boundary of Neuroscience and Architecture. This project brings together human brain activity monitoring using portable high-definition EEG with cutting-edge Virtual Reality facilities on campus to assay human responses to the built environment, in particular, as the subjects attempt to navigate through a complex building. This project has exciting practical ramifications, for example, in providing evidence for the design of better healthcare and educational facilities, as well as testing therapies for improving the status of humans with neural impairment conditions.  Select Publications The Medicinal Leech Genome Encodes 21 Innexin Genes: Different Combinations Are Expressed by Identified Central Neurons. B. Kandarian, J. Sethi, A.Y. Wu, M.W. Baker, N. Yazdani, E. Kym, A. Sanchez, L. Edsall, T. Gaasterland and E.R. Macagno. Develop Genes Evol 222:29-44 (2012). PMID:22358128 Ectopic expression of select innexins in individual central neurons couples them to pre-existing neuronal or glial networks that express the same innexin. C.P. Firme, R.G. Natan, N. Yazdani, E.R. Macagno, M.W. Baker. J Neurosci 32:14265-70 (2012).PMID:23055495 Expression of a dominant negative mutant innexin in identified neurons and glial cells reveals selective interactions among gap junctional proteins. N. Yazdani, C.P. Firme 3rd, E.R. Macagno, MW Baker. Dev Neurobiol 1002/dneu.22082. [Epub ahead of print] (2013) PMID:23447124 Microscopy Ambient Ionization Top-Down Mass Spectrometry Reveals Developmental Patterning. C.-C. Hsu, N. White, M. Hayashi, E.C. Lin, T. Poon, I. Banerjee, J. Chen, S. Pfaff, E.R. Macagno, P.C. Dorrestein. Proc. Nat. Acad. Sci. USA 110:14855-60 (2013) PMID:23969833 Gap Junction-Dependent Homolog Avoidance in the Developing CNS. M.W. Baker, N. Yazdani, E.R. Macagno. J Neurosci 33:16673-83 (2013) PMID:24133270 Control of Neuronal Morphology and Connectivity: Emerging Developmental Roles for Gap Junctional Proteins. M.W. Baker, E.R. Macagno. FEBS Letters FEBS Lett 588:1470-9 (2014). \n",
            "-------------------- Scraping faculty url 107/183 --------------------\n",
            "cressida madigan  Biography Cressida Madigan earned her Ph.D. in microbiology and molecular genetics at Harvard Medical School, where she studied TB lipid chemistry in Branch Moodys lab at Brigham and Womens Hospital. As an NRSA Postdoctoral Fellow, she studied infection biology in zebrafish with Lalita Ramakrishnan at the University of Washington, and completed her training at UCLA in the labs of Alvaro Sagasti, Robert Modlin and Stephen Smale. She joined the Molecular Biology faculty at UCSD in 2018.  Research My lab uses genetic and imaging tools to study mechanisms of infection-mediated inflammation and neurological injury. We use zebrafish as a model host, which allows for real-time observation of infections that typically play out at difficult-to-access sites in humans. We are currently focused on understanding how the mycobacteria that cause leprosy and tuberculosis instigate neurological injury. In leprosy, this nerve damage and inflammation contributes to the stigmatizing deformities that patients endure, such as blindness and loss of fingers. In tuberculosis, central nervous system infection is the deadliest form of TB, and is most common in children under 5 or in HIV-positive adults. By observing mycobacterial infection of the zebrafish brain, we hope to identify pathways of central nervous system inflammation that can be targeted by host-directed therapies. Many of the pathogens we study have caused significant epidemics in human history or are diseases of poverty, and are therefore under-studied.  Select Publications Madigan CA, Cambier CJ, Kelly-Scumpia KM, Scumpia PO, Cheng TY, Zailaa J, Bloom BR, Moody DB, Smale ST, Sagasti A, Modlin RL, Ramakrishnan L. (2017) A macrophage response to Mycobacterium leprae phenolic glycolipid initiates nerve damage in leprosy. Cell. 170(5):973-985. e10. doi: 10.1016/j.cell.2017.07.030 Madigan CA, Cameron J, Ramakrishnan L. (2017) A zebrafish model for Mycobacterium leprae granulomatous infection. Journal of Infectious Diseases , doi: 10.1093/infdis/jix329 Inkeles MS, Teles RM, Pouldar D, Andrade PR, Madigan CA, Lopez D, Ambrose M, Noursadeghi M, Sarno EN, Rea TH, Ochoa MT, Iruela-Arispe ML, Ottenhoff TH, Geluk A, Bloom BR, Modlin RL. (2016) Cell-type deconvolution with immune response pathways identifies gene networks of host defense and immunopathology in leprosy. Journal of Clinical Investigation Insight . 1(15):e88843. Madigan CA, Martinot AJ, Wei J-R, Madduri A, Cheng T-Y, Young DC, Layre E, Murry JP, Rubin EJ, Moody DB. (2015) Mycobactin Synthase K links iron, lipid homeostasis, and virulence in M. tuberculosis . PLoS Pathogens . 11(3):e1004792. Madigan CA, Cheng TY, Young DC, Layre E, McConnell MJ, Debono CA, Murry JP, Wei JR, Barry CE 3rd, Rodriguez GM, Matsunaga I, Rubin EJ, Moody DB. (2012) Lipidomic discovery of deoxysiderophores reveals a revised pathway of mycobactin biosynthesis in M. tuberculosis . Proceedings of the National Academy of Science . 109(4):1257-62 Layre E, Sweet L, Hong S, Madigan CA, Desjardins D, Young DC, Cheng TY, Annand JW, Kim K, Shamputa IC, McConnell MJ, Debono CA, Behar SM, Minnaard AJ, Murray M, Barry CE 3rd, Matsunaga I, Moody DB. (2011) A Comparative Lipidomics Platform for Chemotaxonomic Analysis of Mycobacterium tuberculosis . Chemistry & Biology . 18(12): 1537-49 Huang S, Cheng TY, Young DC, Layre E, Madigan CA, Shires J, Cerundolo V, Altman JD, Moody DB. (2011) Discovery of deoxyceramides and diacylglycerols as CD1b scaffold lipids among diverse groove-blocking lipids of the human CD1 system. Proceedings of the National Academy of Science 108(48): 19335-40. Molofsky AB, Byrne BG, Whitfield NN, Madigan CA, Fuse ET, Tateda K, Swanson MS. (2006) Cytosolic recognition of flagellin by mouse macrophages restricts Legionella pneumophila infection. Journal of Experimental Medicine . 203(4): 1093-104. \n",
            "-------------------- Scraping faculty url 108/183 --------------------\n",
            "roberto malinow  Research Dr. Malinow's research is directed towards an understanding of how the brain forms and stores memories. His laboratory examines how neuronal activity controls the strength of communication between neurons, at sites called synapses. Such synaptic plasticity is thought to underlie the formation and storage of memories. Synapses are key sites affected by diseases of cognition. It is believed that a detailed understanding of synaptic plasticity will identify critical steps that may be the targets of diseases such as Alzheimer's disease. Such an understanding may eventually lead to treatments that prevent the disease. Dr. Malinow uses a combination of techniques including patch clamp electrophysiology, two-photon laser scanning microscopy and molecular biology to examine the function and plasticity of synapses. He applies these techniques onto different animal models of normal function and disease. His research over the past decade has led him to focus on the biology of postsynaptic receptors. He has found that a change in the number of synaptic receptors, and long-term maintenance of this enhanced number of synaptic receptors, is a critical mechanism underlying synaptic plasticity. His studies further demonstrated that such a mechanism underlies a form of associative memory in rodents. More recently his laboratory has expanded into disease related studies, including Alzheimer's disease, schizophrenia and depression. In a series of studies he examined the effects of A-beta on synapses. A-beta peptide, a preteolytic product of APP, is thought to be central to the pathogenesis of Alzheimer's disease. However, the functional relationship of APP and A-beta to neuronal electrophysiology was not known. His laboratory has shown that neuronal activity modulates the formation and secretion of A-beta peptides from neurons. In turn, A-beta depresses synaptic transmission. Synaptic depression from excessive A-beta could contribute to cognitive decline during early Alzheimer's disease. In addition, activity-dependent modulation of A-beta production may normally participate in a negative feedback that could keep neuronal hyperactivity in check. Disruption of this feedback system could contribute to disease progression in Alzheimer's disease. These studies shed new light on the role of A? in normal and diseased states, and may lead to new treatments of Alzheimer's disease. His laboratory has recently begun to examine a synaptic basis for behavioral depression. They have found that in rat models of depression excitatory synapses are abnormally strong in a part of the brain that may act as a \"disappointment center.\" Thus, negative rewards are likely heightened while positive rewards are diminished. They will use molecular, genetic, electrophysiological, optical and behavioral methods to test if these hyperactive synapses are responsible for behavioral depression.  Select Publications Park H, Rhee J, Park K, Han JS, Malinow R, Chung C. Exposure to stressors facilitates long-term synaptic potentiation in the lateral habenula. J Neurosci. 2017 May 24. Pii: 2281-16. Reinders NR, Pao Y, Renner MC, da Silva-Matos CM, Lodder TR, Malinow R, Kessels HW. Amyloid- effects on synapses and memory require AMPA receptor subunit GluA3. Proc Natl Acad Sci U S A. Oct 18;113(42):E6526-E6534.(2016). Alfonso SI, Callender JA, Hooli B, Antal CE, Mullin K, Sherman MA, Lesn SE, Leitges M, Newton AC, Tanzi RE and Malinow R. Gain-of-function mutations in the kinase PKC may promote synaptic defects in Alzheimer's Disease. Science Signaling May 10;9(427):ra47 (2016). Aow J, Dore K, Malinow R. Conformational signaling required for synaptic plasticity by the NMDA receptor complex. Proc Natl Acad Sci U S A. Nov 9. Nov 24;112(47):14711-6 (2015). Dore K, Aow J, Malinow R. Agonist binding to the NMDA receptor drives movement of its cytoplasmic domain without ion flow. Proc Natl Acad Sci U S A. Nov 24;112(47):14705-10 (2015). Shabel S, Proulx C, Piriz J and Malinow R. GABA/glutamate co-release controls habenula output and is modified by antidepressant treatment. Science 345:1494-8 (2014). Nabavi S, Fox R, Lin J, Tsien R, and Malinow R. Engineering memories with LTP and LTD. Nature 511(7509):348-52 (2014). Shabel SJ, Murphy RT and Malinow R. Negative learning bias is associated with risk aversion in a genetic animal model of depression. Frontiers in Human Neuroscience 8:1 (2014). Li K, Zhou T, Liao L, Yang Z, Wong C, Henn F, Malinow R, Yates JR 3rd, Hu H. CaMKII in lateral habenula mediates core symptoms of depression. Science 341:1016-20 (2013). Nabavi S, Kessels H, Alfonso S, Aow J, Fox R, and Malinow R. Metabotropic NMDA receptor function is required for NMDA-dependent long-term depression. Proc. Nat. Ac. Sci 110:4027-32 (2013). Kessels H, Nabavi S and Malinow R. Metabotropic NMDA receptor function is required for A-induced synaptic depression. Proc. Nat. Ac. Sci 110(10):4033-8 (2013). Hu, H, Real, E, Takamiya K, Kang M-G, LeDoux J, Huganir RL, and Malinow R. Emotion Enhances Memory via Norepinephrine Regulation of AMPA Receptor Trafficking Cell 131:160-173 (2007). Hsieh H., Boehm J., Sato C., Iwatsubo T., Tomita T., Sisodia S., and Malinow R. AMPA-R Removal Underlies A-beta-induced Synaptic Depression and Dendritic Spine Loss. Neuron 52:831-843 (2006). Rumpel, S., Ledoux, J., Zador, T. and Malinow R. Postsynaptic receptor trafficking underlying a form of associative learning. Science 308:83-88 (2005). Takahashi, T., Svoboda, K. and Malinow, R. Experience strengthens transmission by driving AMPA receptors into synapses. Science 299:1585-1588 (2003). Kamenetz, F., Tomita, T., Hsieh, H., Seabrook, G., Borchelt, D., Iwatsubo, T., Sisodia, S. and Malinow R. APP processing and synaptic function. Neuron 37:925-937 (2003). Shi, S-H., Hayashi, Y., Esteban, J. and Malinow, R. Subunit-specific rules governing AMPA receptor trafficking to synapses in hippocampal pyramidal neurons. Cell 105:331-343 (2001). Hayashi Y, Shi S, Esteban J, Piccini A, Poncer JC, and Malinow R: Driving AMPA receptors into synapses by CaMKII or LTP: Requirement of GluR1-PDZ-domain interaction. Science 287:2262-2267 (2000). \n",
            "-------------------- Scraping faculty url 109/183 --------------------\n",
            "teri markow  Research SPECIATION AND ADAPTATION TO NOVEL ENVIRONMENTS The process of speciation, despite its overwhelming role in the origins of biodiversity, remains poorly understood. For sexually reproducing organisms defined by the Biological Species Concept, we still are unclear regarding many basic questions: How often does sympatric rather than allopatric speciation occur? What are the relative roles of sexual behavior, postcopulatory-prezygotic incompatibilities, and postzygotic problems such as hybrid sterility and inviability in reproductive isolation? Is there a particular degree of genetic differentiation associated with the appearance of one or more of the categories of isolating mechanisms? What is the role of ecological adaptations in reproductive isolation? In order to address these issues in a comprehensive framework, we need to utilize a group of organisms that not only is ecologically defined, but one for which we can employ modern molecular approaches. One group of organisms, the cactus breeding Drosophila (fruit flies) of the Sonoran Desert of North America, are ideally suited to studies of adaptation and speciation. Four species of Drosophila are endemic to the Sonoran Desert of North America: D. nigrospiracula, D. mojavensis, D. mettleri, and D. pachea. Each uses the necrotic tissue of a different columnar cactus species as a feeding and breeding site, with the exception of D. mettleri, which breeds in soil beneath a necrotic cactus that has been soaked with necrotic juice. The Sea of Cortez separates the portion of the Sonoran Desert found in Baja California from the part on Mexico's mainland, acting as a barrier to gene flow for many organisms. We utilize molecular markers (Ross et al. 2003; Hurtado et al. 2004) to study the degree to which populations of different Drosophila species from the two sides of the Sea of Cortez exhibit differentiation with respect to genetics. We then measure reproductive isolation at the pre- and post zygotic levels (Reed & Markow 2004), especially assortative fertilization (Markow 1997) to test hypotheses regarding when reproductive isolation appears, and in what form, relative to the earliest stages of genetic differentiation. Cacti have a very different chemical composition compared to the host resources used by most Drosophila species. They are very low in nitrogen and phosphorus (Markow et al. 1999; Jaenike & Markow 2003) and they contain chemicals that are toxic to many species of organisms. Yet the resident Drosophila species can feed and breed in these plants without ill effects. We employ functional genomic approaches, custom microarrays for D. melanogaster and D. mojavensis, to study the genes involved in the utilization of different dietary resources by Drosophila melanogaster (Cartsen et al. 2005) and Drosophila mojavensis (Matzkin et al. 2005). We also study the influence of nutrients on growth rates in Drosophila species that specialize on nutrient-rich versus nutrient-poor food (Watts et al. 2005) MATING SYSTEM EVOLUTION Species of the genus Drosophila exhibit amazing variability in mating system characters such as age at reproductive maturity, sexual dimorphism in age at reproductive maturity, sperm length, ovariole number, female incorporation of ejaculatory substances, copulation duration, and the presence or absence of male secondary sexual characters (Markow 1996; Markow 2002; Markow and O'Grady 2005). The laboratory is interested in identifying the long-term evolutionary as well as the ecological (Markow et al 2001) factors that underlie this variation. See Powerpoint of sperm length figure for here. Mechanisms of postcopulatory but prezygotic reproductive isolating mechanisms are likely to be rooted in mating system evolution. Interactions between male and female reproductive molecules and morphology are potentially involved in incompatibilities that occur after mating but before fertilization. Reproductive isolating mechanisms that operate at this level could be due to mismatches between either the morphological (Pitnick et al 2003) or the molecular characteristics (Knowles and Markow 2001) of male and female reproductive tissues. DROSOPHILA EVOLUTIONARY GENOMICS The NSF-supported Stock Center is a living collection of more than 1600 strains of over 250 species of Drosophila. The genomes of 12 of these species are now fully sequenced. Sequenced species vary widely with respect to traits like morphology, longevity, learning ability, and resistance to stressors like chemicals, heat, desiccation, starvation that make them ideal model systems for biomedical studies.  Select Publications Bono, J. M., Matzkin, L. M., Kelleher, E. S.,Markow, T. A.2011. Postmating transcriptional changes in reproductive tracts of con- and heterospecifically-matedDrosophila mojavensisfemales.Proceedings of the National Academy of Sciences, in press. Matzkin, L.M., Johnson, S., Paight, C., Bozinovic, G., andMarkow, T.A.2011. Dietary protein and sugar differentially affect development and metabolic pools in ecologically diverse Drosophila.Journal of Nutrition, in press. Hardy, R., Lougheed, A. andMarkow, T.A.2011. Reproductive tract and spermatid abnormalities of hybrid males from reciprocal crosses betweenDrosophila mojavensisandD. arizonae.Fly, in press. Song, X., Goicoechea, J.L., Ammiraju, S.J.J., et al, (Markow, T.A., corresponding author). 2011. The 19 genomes of Drosophila: a BAC library resource for genus-wide and genome scale comparative evolutionary research.Genetics, in press. Kelleher, E.S., Clark, N. andMarkow, T.A.2011. Diversifying selection acts on a female reproductive protease family in four subspecies ofDrosophila mojavensis.Genetics, doi: 10.1534/genetics.110.124743. Markow, T.A.and Pfeiler, E. 2010. Mitochondrial DNA evidence for deep genetic divergences in allopatric populations of the rocky intertidal isopodLigia occidentalisfrom the eastern Pacific.Molecular Phylogenetics and Evolution56:468-473. Kelleher, E.S., Watts, T.D. LaFlamme, B., Haynes, P. andMarkow, T.A.2009. Proteomic Analysis of Drosophila mojavensis Male Accessory Glands Suggests Novel Classes of Seminal Fluid Proteins.Insect Biochemistry and Molecular Biology39:366-371. Kelleher, E.S. andMarkow, T.A2009. Duplication, selection, and gene conversion in aDrosophila mojavensisfemale reproductive protein family.Genetics 181:1451-1465. \n",
            "-------------------- Scraping faculty url 110/183 --------------------\n",
            "stephen mayfield  Biography Stephen Mayfield obtained bachelor of science degrees in Biochemistry and in Plant Biology from California Polytechnic University in San Luis Obispo in 1979, and a Ph.D. in Molecular Genetics from the University of California at Berkeley in 1984. From 1984 to 1987 Steve was an NIH post-doctoral fellow at the University of Geneva, Switzerland. In 1987 he joined the Scripps Research Institute and become a professor and associate dean of the graduate school before leaving to join UCSD in 2009.  Research Engineering algae for the production of therapeutic proteins and biofuels The research in the Mayfield lab focuses on molecular genetics in green algae, and on the production of therapeutic proteins and biofuel molecules using algae as a production platform. To develop algae as a biotechnology platform, we are developing the molecular tools that will be used to produce either recombinant proteins or engineered biofuel molecules. We have developed a number of expression vectors that allow for robust accumulation of complex mammalian proteins including monoclonal antibodies, mammalian growth factors, and a variety of potential industrial enzymes. We have shown that this algal-based system is capable of producing these recombinant proteins at very high levels. Recent studies have focused on producing antibody-toxin fusion proteins, in which a targeting antibody domain is linked to a cell-killing toxin. Using this technology, we have produced an antibody-toxin fusion that binds and kills human B-cell lymphomas, and cell-based assays have demonstrated the utility and specificity of this molecule. These proteins have great potential for the treatment of cancers and infectious diseases, and chloroplasts offer perhaps the only system in which these types of proteins can be produced. We have also recently succeeded in generating expression vectors that allow for expression of recombinant proteins from the nuclear genome at relatively high levels, and that also allow for secretion of these proteins from the cell. These new vectors will increase the repertoire of recombinant proteins that can be produced in algae thereby expanding the platform. Engineering algae for the production of hydrocarbon molecules that can be used as biofuels is also a major focus of the lab, and recent studies have demonstrated the tremendous potential of eukaryotic algae for the expression of recombinant enzymes that allow for greater production of biofuels. Our continued genetic, biochemical and structural studies should lead to a greater understanding of the mechanism of gene expression in algae, and allow algae to become a viable source for sustainable biofuel and recombinant protein production.  Select Publications Full listing of publications Barrera, D. J., Mayfield, S. P. (2013) High-value Recombinant Protein Production in Microalgae. Handb. Microalgal Cult. (Emeritus, A. R. P. D. & Ph.D, Q. H.) 532544 (John Wiley & Sons, Ltd.). Gimpel, J.A., Mayfield S.P. (2013) Analysis of heterologous regulatory and coding regions in algal chloroplasts. Applied microbiology and biotechnology. 97:4499-4510. Gimpel JA, Specht EA, Georgianna DR, Mayfield SP. (2013) Advances in microalgae engineering and synthetic biology applications for biofuel production. Current Opinion in Chemical Biology, 17:489495. Specht EA, Mayfield SP. (2013) Synthetic oligonucleotide libraries reveal novel regulatory elements in Chlamydomonas chloroplast mRNAs. American Chemical Society Synthetic Biology, 2(1): 34-46. Rasala BA, Barrera DJ, Ng J, Plucinak TM, Rosenberg JN, Weeks DP, Oyler GA, Peterson TC, Haerizadeh F, Mayfield SP. (2013) Expanding the spectral palette of fluorescent proteins for the green microalga Chlamydomonas reinhardtii. The Plant Journal, 74(4):545-56. Rasala BA, Gimpel J, Tran M, Hannon MJ, Miyake-Stone SJ, Specht EA, Mayfield SP. (2013) Genetic engineering to improve algal biofuels production. In Algae for Biofuels and Energy (Michael A. Borowitzka, ed), Springer, New York, pp. 99-114. Tran M, Henry RE, Siefker D, Van C, Newkirk G, Kim J, Bui J, Mayfield SP. (2013) Production of anti-cancer immunotoxins in algae: ribosome inactivating proteins as fusion partners. Biotechnology and bioengineering, 110(11):2826-2835. Tran, M., Van, C., Barrera, D.J., Pettersson, P.L., Peinado, C.D., Bui, J., Mayfield, S.P. (2013) Production of unique immunotoxin cancer therapeutics in algal chloroplasts. Proc. Natl. Acad. Sci. U. S. A. 110, E1522. Vinyard, D.J., J. Gimpel, G.M. Ananyev, M.A. Cornejo, S.S. Golden, S.P. Mayfield, & G.C. Dismukes. (2013) Natural variants of Photosystem II subunit D1 tune photochemical fitness to solar intensity. Journal of Biological Chemistry. 288:5451-5462. Blatti, J. L., Beld, J., Behnke, C. A., Mendez, M., Mayfield, S. P., & Burkart, M. D. (2012). Manipulating Fatty Acid Biosynthesis in Microalgae for Biofuel through Protein-Protein Interactions. PLOS ONE, 7(9), e42949. \n",
            "-------------------- Scraping faculty url 111/183 --------------------\n",
            "lisa mcdonnell  Biography Lisa McDonnell received her Ph.D. in the Faculty of Forestry from the University of British Columbia (UBC) where she used tools from cell biology, molecular biology, and biotechnology to understand the role of various genes in the production of cellulose. She later returned to UBC as a post-doctoral fellow with the Carl Wieman Science Education Initiative. She joined the faculty of Biological Sciences in 2015.  Research My biology education research broadly aims to understand how teaching practices influence student learning and development of scientific thinking. Scientific thinking skills, such as data analysis, evaluation, forming scientific arguments, and problem solving, are a key component of a biology education, but understanding when and how these skills develop requires more investigation. Currently I am interested in how students develop critical thinking skills, and how their understanding of biology research is influenced by engaging in research experiences. I collaborate with Dr. Stanley Lo to investigate these questions in biology lab classes here at UCSD. Another area of interest for me is how teaching practices influence student learning and development of writing and the ability to form scientific arguments. Two current projects are collaborations with colleagues here at UCSD (Sarah Stockwell, Emily Grossman, Stanley Lo, Stephanie Mel) to investigate the impacts of two interventions on students writing and scientific argumentation. The first is looking at how the incorporation of opportunities for students to explore academic integrity, in the context of a biology class writing assignment, influences their writing. The second is looking at how the incorporation of writing and peer review impacts the quality of students constructed scientific arguments and the quality of the reviews they provide to peers. Both of these projects aim to contribute to our understanding of the teaching methods we can employ to improve the development of knowledge and skills that are necessary to be successful biologists.  Select Publications Zukswert, J. M., Barker, M. K., & Mcdonnell, L. 2019. Identifying Troublesome Jargon in Biology: Discrepancies between Student Performance and Perceived Understanding. CBE Life Sciences Education, 48 (1): ar1-mr1. PMID: 30707640 Stains, M; Harshman, J; Barker, MK; Chasteen, SV; Cole, R; DeChenne-Peters, SE; Eagan, MK; Esson, JM; Knight, JK; Laski, FA; Levis-Fitzgerald, M; Lee, CJ; Lo, SM; McDonnell, LM; McKay, TA; Michelotti, N; Musgrove, A; Palmer, MS; Plank, KM; Rodela, TM; Sanders, ER; Schimpf, NG; Schulte, PM; Smith, MK; Stetzer, M; Van Valkenburgh, B; Vinson, E; Weir, LK; Wendel, PJ; Wheeler, LB; Young, AM. 2018. Anatomy of STEM teaching in North American universities. Science, 359(6383): 1468-1470. PMID: 29599232 Aikens, M.L., Corwin, L.A., Andrews, T.C., Couch, B.A., Eddy, S.L., McDonnell, L., Trujillo, G. 2016. A guide for graduate students interested in postdoctoral positions in biology education research. CBE Life Sciences Education, 15(4). PMID: 27856554 McDonnell, L., and Mullally, M. (2016). Teaching students how to check their work while solving problems in genetics. Journal of College Science Teaching. 46: 68-75. Aikens, M.L., Corwin, L.A., Andrews, T.C., Couch, B.A., Eddy, S.L., McDonnell, L., and Trujillo, G. A (2016) Guide for Graduate Students Interested in Postdoctoral Positions in Biology Education. CBE Life Sciences Education. 15: es10. McDonnell, L., Barker, M.B., and Wieman, C. (2015). Concepts first, jargon second improves student articulation of understanding. Journal of Biochemistry and Molecular Biology Education 44: 12-19. Maxwell, J.E., McDonnell, L.M., and Wieman, C. (2015). An improved design for in-class review. Journal of College Science Teaching 44: 47-52. McDonnell, L., and Klenz, J. (2015). Teaching Genetic Linkage through Mapping with Molecular Markers. CourseSource (coursesource.org) Ye, X., Busov, V. Zhao, N., Meilan, R., McDonnell, L.M., Coleman, H.D., Mansfield, S.D., Chen, F., Li, Y., and Cheng, Z-M. (2011). Transgenic Populus trees for forest products, bioenergy, and functional genomics. Critical Reviews in Plant Sciences 5: 415-434. Joshi, C.P., Thammannagowda, S., Takeshi, F., Gou, J., Avci, U., Haigler, C., McDonnell, L.M., Mansfield, S.D., Menghesa, B., Carpita, N.C., Harris, D., DeBolt, S., and Peter, G. (2010). Perturbation of wood cellulose synthesis causes pleiotropic effects in transgenic aspen. Molecular Plant 4: 331-345. Plett, J.M, McDonnell, L., and Regan, S. 2009. Plant encoded 1-aminocyclopropane-1-carboxylic acid deaminase activity implicated in different aspects of plant development. Plant Signaling and Behavior 4(12), 11861189. McDonnell, L., Plett, J. M., Andersson-Gunners, S., Kozela, C., Dugardeyn, J., Straeten, D. V. D., Glick, B.R., Sundberg, B. and Regan, S. (2009). Ethylene levels are regulated by a plant encoded 1-aminocyclopropane-1-carboxylic acid deaminase. Physiologia Plantarum 136: 94-109. \n",
            "-------------------- Scraping faculty url 112/183 --------------------\n",
            "bill mcginnis  Biography Bill McGinnis received his Ph.D from UC Berkeley in 1982 and was a Jane Coffin Childs postdoctoral fellow at the University of Basel. From 1984 to 1995, he was on the faculty of Yale University. He received a Searle Scholar Award, a Presidential Young Investigator Award, and a Dreyfuss Teacher/Scholar Award. In 2010, Dr. McGinnis was elected to the American Academy of Arts and Sciences.  Research One major research interest of our lab is the regulation of transcription patterns of Hox genes in animal embryos. Hox gene expression patterns determine the morphological features that develop on the anterior posterior axis of animal embryos, and how Hox expression patterns are regulated are critical to their developmental functions, as well as the evolutionary diversity in form that arises in different animal embryos. One current specific interest is how distant Hox enhancer-promoter interactions regulate transcriptional activation of Hox genes in Drosophila embryos. We have developed methods to detect transcriptional activation patterns of many genes simultaneously in individual nuclei, and at the same time detect mRNA molecules from individual genes, as well as the DNA regulatory sequences that activate transcription from those genes, in individual nuclei in which the genes are activated or repressed. We are also exploring the genes underlying a signaling pathway that controls regeneration of animal epidermis after wounding, using Drosophila as a model system. This pathway, which involves the activation and repression of thousands of genes that regulate epidermal regeneration, can be activated by sterile puncture wounds of fly embryos. Genetic screens are identifying new components in the wound response pathway, including the extracellular signals and receptors that instruct cells around wounds to initiate epidermal barrier repair. New methods to visualize wound activated gene regulation are also being developed.  Select Publications Juarez, M.T., Patterson, R.A., Li, W. and McGinnis, W. (2013) Microinjection wound assay and in vivo localization of epidermal wound response reporters in Drosophila embryos. Journal of Visualized Experiments, (81), e50750. Patterson, R.A., Juarez, M.T., Hermann, A., Sasik, R., Hardiman, G., and McGinnis, W. (2013) Serine proteolytic pathway activation reveals an expanded ensemble of wound response genes in Drosophila. PLoS ONE, 8: e61773. Par, A., Kim, M., Brody, S. and McGinnis, W. (2012) The functions of Grainy head family proteins in Animals and Fungi and the evolution of apical extracellular barriers. PLoS ONE, 7, e36254. Lemons, D., Par, A. and McGinnis, W. (2012) Three Drosophila Hox complex miRNAs do not have major effects on expression of evolutionarily conserved Hox gene targets during embryogenesis, PLoS ONE 7, e31365. Juarez, M. T. Patterson, R., Sandoval-Guillen, E. and McGinnis, W. (2011) Duox, Flotillin-2, and Src42A are Required to Activate or Delimit the Spread of the Transcriptional Response to Epidermal Wounds in Drosophila. PLoS Genetics 7, e1002424. McHale, P., Mizutani, C.M., Kosman, D., MacKay, D.L., Belu. M., McGinnis. W., Ethan Bier, and Hwa, T. (2011) Gene length may contribute to graded transcriptional responses in the Drosophila embryo. Developmental Biology, 360, 230-240. Kim, M. and McGinnis, W. (2011) Phosphorylation of Grainy head by ERK is essential for wound-dependent regeneration of an epidermal barrier but dispensable for embryonic barrier development. Proc. Natl. Acad. Sci. USA 108, 650-655. \n",
            "-------------------- Scraping faculty url 113/183 --------------------\n",
            "stephanie mel \n",
            "-------------------- Scraping faculty url 114/183 --------------------\n",
            "justin meyer  Biography Justin Meyer received his Ph.D. from Michigan State University and was a Systems Biology Departmental Fellow at Harvard Medical School, where he was awarded the James S. McDonnell Foundation Fellowship for Studying Complex Systems. He joined the faculty of Ecology Behavior and Evolution and the Quantitative Biology Initiative in 2014.  Research What changes to viral genomes would allow them to infect new species? And, what are the natural processes mutation, recombination, and natural selection that permit their evolution? My lab group uses experimental evolution of viruses to find answers to these questions. We are able to culture viruses in the laboratory, watch them evolve in real time, and on occasion, witness them gain access to new host species. Our studies combine methods from many fields of biology. We typically begin research by modifying classic ecological niche theory to make predictions for viral host-range evolution. Next we test the theory by culturing viruses under controlled conditions. We sequence the evolved viral genomes to identify adaptive mutations, and then we determine the effects of the mutations by introducing them into nave viral genomes using Multiplexed Automated Genome Engineering. Lastly, we characterize their newly evolved qualities by using single molecule biophysics assays. Altogether, the studies provide a complete picture for how gain-of-function mutations evolve in viruses. Most of our work to date has been preformed on the virus Bacteriophage , which infects E. coli. We have observed evolve to exploit a novel receptor, the outer-membrane protein OmpF. Typically only targets another protein, LamB, however under the right conditions it naturally evolves this innovation. Which receptors a virus exploits largely determine their host-range, and so, acquiring a new receptor constitutes a major transition for the virus. Currently our research group is continuing to test hypotheses for how this evolved, but we are also applying the tools we developed to study to other viral systems. Bacteriophage Image by Brian Wade New receptor, OmpF Image by Justin Meyer  Select Publications Petrie, K.P., Palmer, N.D., Johnson, D.T., Medina, S.J., Yan, S.J., Li, V., Burmeister, A.R., Meyer, J.R. (2018) Destabilizing mutations encode nongenetic variation that drives evolutionary innovation. Science 359: 1542-1545. Meyer, J.R., Dobias, D.T., Medina, S.J., Servilio, L., Gupta, A. and Lenski, R.E. (2016) Ecological speciation of bacteriophage lambda in allopatry and sympatry. Science, 354,1301-1304. Meyer, J. R., Gudelj, I., & Beardmore, R. (2015) Biophysical mechanisms that maintain biodiversity through trade-offs. Nature communications, 6, 6278. J. R. Meyer, D. T. Dobias, J. S. Weitz, J. E. Barrick, R. T. Quick, R. E. Lenski (2012) Repeatability and contingency in the evolution of a key innovation in Lambda Phage. Science 335, 428-432. C. Flores, J. R. Meyer, S. Valverde, L. Farr, J. S. Weitz (2011) Statistical structure of host-phage interactions. Proceedings of the National Academy of Sciences 108, E288-297. J. R. Meyer, R. Kassen (2007) The effects of competition and predation on diversification in a model adaptive radiation. Nature 446, 432-435. J. R. Meyer, S. P. Ellner, N. G. Hairston, Jr., L. E. Jones, T. Yoshida (2006) Prey evolution on the time scale of predator-prey dynamics revealed by allele-specific quantitative PCR. Proceedings of the National Academy of Sciences 103, 10690-10695. \n",
            "-------------------- Scraping faculty url 115/183 --------------------\n",
            "maurice montal  Biography Mauricio Montal received his Ph.D. from the University of Pennsylvania and his M.D. from the National University of Mexico. He is the recipient of a K.C. Cole Award of the Biophysical Society for outstanding contributions in Membrane Biophysics and a Research Scientist Award from NIMH. Dr. Montal is a recipient of a Gugenheim Fellowship and a Fellow of the Biophysical Society.  Research The key scientific question has been how membrane proteins function in terms of the underlying structure. These entities are truly smart materials. Proteins are nanomachines and marvels of protein chemistry. Lipid bilayers, the basic structural elements of biological membranes, are nanodevices which exploit self-assembly principles to provide the exquisite environment for proteins to operate. Together as a system, proteins embedded within lipid bilayers, embody the basis for biosensor and nanosensor technology. They also emerge as templates for molecular mimicry and design and follow the hierarchical organization underpinning complex systems. Our programs are of prime significance to health science and biodefense; they also comprise components of relevance to the pharmaceutical and cosmeceutical industries. Specifically, our aim is to understand the structures and the mechanisms underlying nerve cell function and dysfunction. We are interested in the molecular basis of the intrinsic excitability of neurons, the communication of excitability between neurons, neuronal cell survival and death, the processes leading to neurodegeneration, and the genetic components in the pathogenesis of some forms of infant epilepsy. Given that our subjects of inquiry are ion-conducting and protein-conducting channels, key elements of the nerve cell, they embody drug targets; therefore, our programs encompass a component directed to screen or discover agents that impact these processes and, may in due turn, become clinically important drugs. Botulinum neurotoxin (BoNT) is considered the most potent toxin; it is the causative agent of botulism, a neuroparalytic illness, and a major bioweapon without an antidote. A research program is directed towards this goal. Our objective is to understand the intricacies of intracellular trafficking of BoNT that may disclose novel pathways to escort, target, and insert it into membranes. This information may prove diagnostic in identifying unsuspected sites for intervention and plausible strategies for targeted drug delivery. BoNT is a modular nanomachine in which one of its modules -- the heavy chain channel -- operates as a specific protein translocating transmembrane chaperone for another of its component modules -- the light chain protease. We investigate the dynamics of protein-translocation in neuronal cells focusing on the interactions between the channel/chaperone and its cargo using an assay with single molecule sensitivity. This strategy led us to uncover an entirely unknown succession of intermediate states during translocation of the cargo which reveal transient protein-protein interactions occurring at distinct stages during protease translocation through the channel. Another major area of interest concerns the design of biosensors based on lipid bilayers membranes containing a number of designed channel proteins that emulate those naturally occurring in the brain. Such devices are indeed nanomachines which may display unparalleled sensitivity in analyte detection, including but not limited to biological and environmental toxins, and also in sensing physical stimuli such as light, temperature, pressure or electric fields.  Select Publications Fischer, A. and Montal, M. Characterization of Clostridial botulinum neurotoxin channels in neuroblastoma cells. Neurotoxicity Res. 9:93-100 (2006). Santos, J.S., Lundby, A., Zazueta, C. and Montal, M. Molecular template for a voltage sensor in a novel K-plus channel. I. Identification and functional characterization of KvLm, a voltage-gated K-plus channel from Listeria monocytogenes. J. Gen. Physiol. 128:283-292 (2006). Lundby, A., Santos, J.S., Zazueta, C. and Montal, M. Molecular template for a voltage sensor in a novel K-plus channel. II. Conservation of a eukaryotic sensor fold in a prokaryotic K-plus channel. J. Gen. Physiol. 128:293-300 (2006). Fischer, A. and Montal, M. Single molecule detection of intermediates during Botulinum neurotoxin translocation across membranes. Proc Natl Acad Sci U S A 104: 10447-10452 (2007). Brunger, A.T., Breidenbach, M.A., Jin, R., Fischer, A., Santos, J.S., and Montal, M. Botulinum neurotoxin heavy chain belt as an intramolecular chaperone for the light chain. PLoS Pathogens 3:1191-1194 (2007). Fischer, A., and Montal, M. Crucial role of the disulfide bridge between Botulinum neurotoxin light and heavy chains in protease translocation across membranes. J. Biol. Chem. 282: 29604-29611 (2007). Fischer, A., Garcia-Rodriguez, C., Geren, I., Lou, J., Marks, J.D., Nakagawa, T., and Montal, M. Molecular architecture of Botulinum neurotoxin E revealed by single particle electron microscopy. J. Biol. Chem. 283: 3997-4003 (2008). doi/10.1074, published online on November 20, 2007. Fischer, A., Koriazova, L., Oblatt-Montal, M. and Montal, M. Botulinum Neurotoxin -- A Modular Nanomachine. In: Botulinum Toxin: Therapeutic Clinical Practice and Science. Edited by J.J. Jankovic, A. Albanese, M. Z. Atassi, J.O. Dolly, M. Hallett and N.H. Mayer. Elsevier Publ. (in press). Fischer, A., Mushrush, D., Lacy, D. B., and Montal, M. Botulinum neurotoxin devoid of receptor binding domain translocates active protease. PLoS Pathogens (in press). Montal, M. Translocation of Botulinum neurotoxin light chain protease by the heavy chain protein-conducting channel. Special Issue of Toxicon (in press). Santos, J., Grigoryev, S. M. and Montal, M. Molecular template for a voltage sensor in a novel K-plus channel. III. Functional reconstitution of a sensor-less pore module from a prokaryotic K-plus channe. J. Gen. Physiol. (December 2008, in press). Montal, M. Vpu Matchmakers as a Therapeutic Strategy for HIV infection. PLoS Pathogens (in press). \n",
            "-------------------- Scraping faculty url 116/183 --------------------\n",
            "marc montminy \n",
            "-------------------- Scraping faculty url 117/183 --------------------\n",
            "cornelis murre  Biography Kees Murre performed his graduate work at Harvard Medical School and was a postdoctoral fellow at MIT. He is a Searle Scholar and the recipient of the National Institutes of Health Merit Award.  Research We study gene regulation in adult stem cells and developing lymphocytes. Both global and single cell strategies are being utilized with the aim to describe normal development as well as aged and diseased states in molecular terms. We have a long-standing interest in deciphering the role and regulation of helix-loop-helix proteins in lymphocyte development. This class of factors plays key roles in hematopoiesis. Currently our main interests are in the role of these factors in the control of hematopoietic stem cell homeostasis, B- and T-lineage specification and commitment, aging, inflammatory disease and their roles in the periphery in the response to invading pathogens. Recent genome-wide studies have identified that a large fraction of the genome transcribed in non-coding regions. We are now faced with the question as to how these large non-coding RNAs relate to the control of gene expression. We have identified a subset of lineage-specific and developmental-stage specific non-coding RNAs. Our interests are to determine the function of these non-coding RNAs and to study their potential roles in modulating long-range chromatin structure and genomic interactions. Our knowledge of chromatin structure and long-range genomic interactions in mammalian cells is still rudimentary. How do the genomes of lymphocytes, plasma cells and granulocytes adopt such unique and distinct structures? We aim to resolve these questions using genome-wide chromosome-conformation capture studies (HiC) in conjunction with computational approaches to describe the topologies of lymphoid and myeloid genomes in molecular terms. During developmental progression of lymphoid cells coding and regulatory elements interact to induce lineage-specific programs of gene expression. We recently found that in eukaryotic cells coding and regulatory genomic elements bounce back and forth within the chromatin network until specific genomic interactions are established, and that spatial confinement of topological domains largely controls the times for such encounters. Our future studies aim to examine how epigenetic and structural determinants affect the trajectories adopted by the chromatin fiber in living cells and how this relates to genomic encounters involving enhancers and promoters. We have recently demonstrated that genes encoding for key developmental regulators reposition during developmental progression. We now aim to address the question as to why and how genomic regions encoding for developmental regulator reposition during developmental progression, how their release from the lamina is regulated and how they associate with active transcription factories.  Select Publications Isoda, T., Moore, A., He, Chandra, V., Aida, M., Denholtz, van Hamburg, J.P., Fisch, K., Chang, A.N., Fahl, S. Wiest, D.L. and Murre, C. 2017. Non-Coding Transcription Instructs Cohesin-Dependent Chromatin Folding and Compartmentalization to Dictate Enhancer-Promoter Communication and T Cell Fate. Cell 171, 103-119. Miyazaki, M., Miyazaki, K., Chen, K., Jin, Y., Turner, J., Moore, A.J., Saito, R., Yoshida, K., Ogawa, S., Rodewald, H.R., Lin, Y.C., Kawamoto, H. and Murre C. 2017. The E-Id protein axis specifies adaptive lymphoid cell identity and suppresses thymic innate lymphoid cell development. Immunity 46, 818-834. Zhu, Y., Gong, K., Denholtz, M., Chandra, V., Kamps, M.P., Alber, F. and Murre, C. (2017). Comprehensive characterization of neutrophil genomes. Genes Dev. 31, 141-153. Miyazaki. M., Miyazaki, K., Chen, S.Y., Chandra, V., Agata, Y., Chang, A., Kawamoto, H. and Murre, C. 2015. The E-Id protein axis modulates the activities of the PI3K-AKT-mTORC1-Hif1a and c-myc/p19Arf pathways to suppress innate variant TFH cell development, thymocyte expansion and lymphomagenesis. Genes Dev. 29, 409-425. Bossen, Murre, C.S., Chang, A.N., Mansson, R., Rodewald, H.R. and Murre, C. 2015. The chromatin remodeler Brg1 activates enhancer repertoires to establish B cell identity and modulate cell growth. Nat. Immunol. 16, 775-784. Lucas, J.S., Zhang, J., Dudko, O.S. and Murre, C. 2014. The 3-D trajectories adopted by the immunoglobulin heavy chain locus: First-passage times and long-range genomic interactions. Cell 158, 339-352. The 3-D trajectories adopted by coding and regulatory DNA elements: First-passage times for genomic interactions. Lucas JS, Zhang J, Dudko OS, Murre C. Cell 158, 339-352 (2014). Id2 and Id3 maintain the regulatory T cell pool to suppress inflammatory disease. Miyazaki M, Miyazaki K, Chen SY, Itoi M, Miller M, Lu L-F, Varki N, Chang A, Broide DH, Murre C. Nat Immunol 15, 767-776 (2014) The establishment of B versus T cell identity. Miyazaki K, Miyazaki M, Murre C. Trends Immunol 35, 205-210 (2014) Boosting lymphocyte production. Murre C. Immunity 38, 1081-1083 (2013) Nuclear location and the control of developmental progression. Lin YC, Murre C. Curr Opion Genet Dev 23, 104-108 (2013) A HESitant decision for T cells. Bortnick A, Murre C. Nat Immunol 14, 1209-1210 (2013) Global changes in the nuclear positioning of genes and intra- and interdomain genomic interactions that orchestrate B cell fate. Lin YC, Benner C, Mansson R, Heinz S, Miyazaki M, Miyazaki K, Chancera V, Bossen C, Glass CK, Murre C. Nat Immunol 12,1196-1204 (2012) Chromatin topology and the regulation of antigen receptor assembly. Bossen C, Mansson R, Murre C . Annual Rev Immunol 30, 337-356 (2012) Positive intergenic feedback circuitry, involving EBF1 and FOXO1, orchestrates B-cell fate. Mansson R, Welinder E, Ashber J, Lin YC, Benner C, Glass CK, Sigvardsson M, Murre C. Proc Natl Acad. Sci USA 108, 17402-17407 (2012) Multilineage priming of enhancer repertoires precedes commitment to the B and myeloid cell lineage in hematopoietic progenitors. Mercer EM, Lin YC, Jhunjhunwala S, Dutkowski J, Flores M, Sigvardsson M, Ideker T, Glass CK, Murre C. Immunity 35, 413-425 (2011) The opposing roles of the transcription factor E2A and its antagonist Id3 that orchestrate and enforce the nave fate of T cells. Miyazaki M, Rivera RR, Miyazaki K, Lin YC, Agata Y, Murre C. Nat Immunol 12, 992-1001 (2011) E2A and HEB act in concert to induce the expression of FOXO1 in the common lymphoid progenitor . Welinder E, Mansson R, Mercer EM, Bryder D, Sigvardsson M, Murre C. Proc Natl Acad Sci USA 108, 17402-17407 (2011) CTCF-binding elements mediate control of V(D)J recombination. Guo C, Yoon HS, Franklin A, Jain S, Ebert A, Cheng HL, Hansen E, Despo O, Bossen C, Vettermann C, Bates JG, Richards N, Myers D, Patel H, Gallagher M, Schlissel MS, Murre. C, Busslinger M, Giallourakis CC, Alt FW. Nature 477, 424-430 (2011) Transcription and recombination factories: Common features? . Lucas JL, Bossen C, Murre C. Curr Opin Cell Biol 23, 318-324 (2011) Ldb1, a new guardian of hematopoietic stem cell maintenance. Welinder E, Murre C. Nat Immunol 12, 113-114 (2011) CCCTC-binding factor (CTCF) and cohesin influence the genomic architecture of the Igh locus and antisense transcription in pro-B cells. Degner SC, Verma-Gaur J, Wong TP, Bossen C, Iverson GM, Torkamani A, Vettermann C, Lin YC, Ju Z, Schulz D, Murre CS, Birshtein BK, Schork NJ, Schlissel MS, Riblet R,. Murre C, Feeney AJ Proc Natl Acad Sci USA 108, 9566-9571 (2011) Factors and networks that underpin early hematopoiesis. Mercer EM, Lin YC, Murre C. Semin Immunol 5, 317-325 (2011) A global network of transcription factors, involving E2A, EBF1 and Foxo1, that orchestrates B cell fate. Lin YC, Jhunjhunwala S, Benner C, Heinz S, Welinder E, Mansson R, Sigvardsson M, Hagman J, Espinoza A, Dutkowski J, Ideker T, Glass CK, Murre C. Nat Immunol 11, 635-643 (2010) Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Heinz S, Benner C, Spann N, Bertolino E, Lin YC, Laslo P, Cheng JX, Murre C, Singh H, Glass CK. Mol Cell 38, 576-89 (2010) Chromatin architecture and the generation of antigen receptor diversity. Jhunjhunwala S, van Zelm MC, Peak M, Murre C. Cell 138, 435-448 (2009) Distinct roles for E12 and E47 in B cell specification and the sequential rearrangement of immunoglobulin light chain loci. Beck K, Peak M, Ota T, Nemazee D. Murre C. J Exp Med 206, 2271-2284 (2009) E2A proteins maintain the hematopoietic stem cell pool and promote the maturation of myelolymphoid and myeloerythroid progenitors. Semerad CL, Mercer EM, Inlay MA, Weissman IL, Murre C. Proc Natl Acad Sci USA 106, 1930-1935 (2009) Developmental trajectories in early hematopoiesis. Murre C . Genes Devel 15, 2366-2370 (2009) The 3D-structure of the immunoglobulin heavy chain locus: Implications for long-range genomic interactions. Jhunjhunwala S, van Zelm MC, Peak M, Cutchin S, Riblet R, van Dongen JJM, Grosveld F, Knoch TA, Murre C. Cell 133, 265-279 (2008) \n",
            "-------------------- Scraping faculty url 118/183 --------------------\n",
            "sonya neal  Biography Dr. Neal received her Ph.D. from U.C. Los Angeles in 2013 after working in the laboratory of Dr. Carla Koehler. She then carried out her postdoctoral studies at U.C. San Diego in the laboratory of Dr. Randolph Hampton. She was the recipient of the Burroughs Wellcome Postdoctoral Diversity Enrichment Award and Ruth L. Kirschstein NRSA Postdoctoral Fellowship. Dr. Neal joined the faculty in 2018.  Research \"Understanding the mechanistic underpinnings and medical importance of misfolded proteins\" Life is harsh, and hard on protein structure. Continuously-produced misfolded proteins represent a threat to proper cell and organ function. To survive this threat, organisms are equipped with quality control systems to maintain properly folded proteins; a process known as protein homeostasis. For the past two decades, it has become increasingly clear that protein homeostasis is critical to the health of cells and organisms. Defects in protein homeostasis underlie many of the most pressing maladies, including aging, cancer and neurodegenerative diseases. Our research strives to understand the basic biology of protein homeostasis and how organisms use quality control pathways to mitigate this universal biological stress. There are three questions that motivate our research: How do cells maintain protein homeostasis? How do changes in protein homeostasis lead to pathology? Can modulation of protein homeostasis be used to treat disease? Our research aim is to understand the cellular and tissue-specific roles of factors involved in protein quality control pathways, with a particular emphasis on the pathways that detect and specifically destroy these dangerous molecules. An understanding of these pathways allows us to define the mechanisms that protect the organismal proteome in health and disease, and eventually devise methods to harness cellular quality control to modify the proteome in the laboratory and the clinic. As a first step towards this goal, we are focused on discovering and studying proteostasis pathways in yeast, mammalian cells and zebrafish, allowing us to synergistically capitalize on the investigative strengths that each has to offer. We take advantage of the laboratorys unique set of skills in genetics, biochemistry, functional genomics, cell and molecular biology to extend these lines of inquiry and to train the next generation of scientific leaders.  Select Publications Neal, S.E., Duttke S.H., and Hampton R.Y. (2019). Assays for protein retrotranslocation in ERAD. Methods Enzymol. 619 , 1-26. Neal, S.E., Jaeger, P., Duttke, S., Benner, C., Glass, C., Ideker, T., R. Hampton. (2018). The Dfm1 Derlin is Required for ERAD Retrotranslocation of Integral Membrane Proteins. Molecular Cell. 69 , 306-320. (Highlighted: Avci, D. and Lemberg, A. (2018) Molecular Cell ) Neal, S.E., Dabir, D.V., Boon, C., and Koehler C.M. (2017). Osm1 is an electron acceptor of Erv1 in the Mia40-dependent import pathway. MBoC. 28 , 2773-2785. Neal, S.E., Mak, R., Bennett, E.J., and Hampton, R. (2017). A Cdc48 Retrochaperone Function Is Required for the Solubility of Retrotranslocated, Integral Membrane Endoplasmic Reticulum-associated Degradation (ERAD-M) Substrates. J. Biol. Chem. 292 , 31123128. Vashistha, N., Neal, S.E., Singh, A., Carroll, S.M., and Hampton, R.Y. (2016). Direct and essential function for Hrd3 in ER-associated degradation. Proc. Natl. Acad. Sci. 113 , 59345939. Neal, S.E., Dabir, D. V., Tienson, H.L., Horn, D.M., Glaeser, K., Ogozalek Loo, R.R., Barrientos, A., and Koehler, C.M. (2015). Mia40 Protein Serves as an Electron Sink in the Mia40-Erv1 Import Pathway. J. Biol. Chem. 290 , 2080420814. Tienson, H.L., Dabir, D. V, Neal, S.E., Loo, R., Hasson, S.A., Boontheung, P., Kim, S.-K., Loo, J.A., and Koehler, C.M. (2009). Reconstitution of the mia40-erv1 oxidative folding pathway for the small tim proteins. Mol. Biol. Cell 20 , 34813490. \n",
            "-------------------- Scraping faculty url 119/183 --------------------\n",
            "james nieh  Biography Dr. Nieh received his BA from Harvard in 1991 and his Ph.D. from Cornell University in 1997. He completed a NSF-NATO postdoctoral fellowship at the University of Wrzburg, Germany and was a Harvard Junior Fellow from 1998-2000.  Research Research in the Nieh lab focuses on how natural and man-made stressors affect the biology of and cognitively sophisticated behaviors exhibited by bees. Our research focuses on two areas: (1) the selective pressures that may have shaped the evolution of communication in highly social bees and (2) honey bee health. We use the tools of Behavioral Ecology, Chemical Ecology, Animal Communication and Neuroethology to work with bumble bees, stingless bees, and honey bees. Five different topic areas are detailed below.For further information, please view the Nieh Lab Homepage . Evolution of communication Selective pressures from competitors and predators has shaped social bee communication. Our lab studies multiple bee groups: honey bees, stingless bees, and bumble bees to learn how this communication works and why it may have evolved. Honey Bee health Concern is growing over pollinator declines. Our lab examines the effects of natural stressors, such as pathogens, and man-made stressors, such as pesticides, on honey bee health, foraging, flight, and orientation. Superorganism inhibitory communication What happens if conditions change and the communicated food source becomes depleted, contested, or dangerous? The honey bee stop signal provides inhibition that counteracts the positive feedback of honey bee waggle dances. Using field studies and modeling, we are studying this signal in detail and exploring conditions under which inhibitory signals may evolve. Superorganism inhibitory communication We study olfactory eavesdropping in stingless bees and honey bees and examine the advantages of eavesdropping upon competitors and predators. Neuroethology of bee learning and memory Despite their small brain size and limited number of neurons relative to the central nervous systems of many vertebrates, social insects have evolved sophisticated learning and memory abilities and are therefore important models for animal cognition. However, these abilities can be impaired by field-realistic exposure to pesticides and other man-made stressors.  Select Publications James Nieh - Full Publication list Tong, L, Nieh, JC, Tosi, S. (2019) Combined nutritional stress and a new systemic pesticide (flupyradifurone, Sivanto) reduce bee survival, food consumption, flight success, and thermoregulation. Chemosphere . 237: 12448. Gong Z., Tan K., and Nieh J.C. (2019) Hornets possess long-lasting olfactory memories. Journal of Experimental Biology. 222, jeb200881. doi:10.1242/jeb.200881 Harmen P. Hendriksma, Collin D. Pachow, and James C. Nieh (2019) Effects of essential amino acid supplementation to promote honey bee gland and muscle development in cages and colonies. Journal of Insect Physiology 117: 103906. Tosi, S. and Nieh, J.C. (2019) Lethal and sublethal synergistic effects of a new systemic pesticide, flupyradifurone (Sivanto) on honey bees. Proceedings of the Royal Society B . 286: 20190433. Rubanov, A., Russell, K., Rothman, J., Nieh J, and McFrederick Q. (2019) Intensity of Nosema ceranae infection is associated with specific honey bee gut bacteria but not overall gut microbiome structure. Scientific Reports . 9:3820. Dong S., Tan K., Zhang Q., and Nieh J.C. (2019) Playbacks of Asian honey bee stop signals demonstrate referential inhibitory communication. Animal Behaviour. 148: 29-37. Kheradmand, B., Cassano J., Gray, S., and Nieh, J. C. (2018). Influence of visual targets and landmarks on honey bee foraging and waggle dancing. Insect Science. 00, 112, DOI 10.1111/1744-7917.12651. Gong, Z., Tan, K., and Nieh, J. C. (2018) First demonstration of olfactory learning and long term memory in honey bee queens. The Journal of Experimental Biology. 221:1-10. PMID: 29776994 Dong, S., Wen, P., Zhang, Q., Wang, Y., Cheng, Y., Tan, K., and Nieh, J. C. (2018) Olfactory eavesdropping of predator alarm pheromone by sympatric but not allopatric prey. Animal Behaviour. 141: 115-125. Zhang, J., Wang, Z., Wen, P., Qu, Y., Tan, K., and Nieh, J.C. (2018) The reluctant visitor: a terpenoid in toxic nectar can reduce olfactory learning and memory in Asian honey bees. Journal of Experimental Biology. 221, doi:10.1242/jeb.168344. Tosi, S., Nieh, J. C., Sgolastra, F., Cabbri R., and Medrzycki, P. (2017) Neonicotinoid pesticides and nutritional stress synergistically reduce survival in honey bees. Proceedings of the Royal Society B. 284. 20171711. Tan, K., Wang, C., Dong, S., Li, X., and Nieh, J.C. (2017) The pesticide flupyradifurone impairs olfactory learning in Asian honey bees (Apis cerana) exposed as larvae or as adults. Scientific Reports 7:17772: DOI:10.1038/s41598-017-18060-z. Tosi, S. and Nieh, J. C. (2017) A common neonicotinoid pesticide, thiamethoxam, alters honey bee activity, motor functions, and phototaxis. Scientific Reports. 7: 15132, DOI:10.1038/s41598-017-15308-6. BenVau, L. R., and Nieh, J. C. (2017) Larval honey bees infected with Nosema ceranae have age-accelerated vitellogenin levels as young adults. Scientific Reports. DOI:10.1038/s41598-017-14702-4 Ping Wen, Ya-Nan Cheng, Shi-Hao Dong, Zheng-Wei Wang, Ken Tan, James C. Nieh. (2017) The sex pheromone of a globally invasive honey bee predator, the Asian eusocial hornet, Vespa velutina. Scientific Reports. DOI:10.1038/s41598-017-13509-7. Ping Wen, Yanan Cheng, Yufeng Qu, Hongxia Zhang, Jianjun Li, Heather Bell, Ken Tan, and Nieh, J.C. (2017) Foragers of sympatric Asian honey bee species intercept competitor signals by avoiding benzyl acetate from Apis cerana alarm pheromone. Scientific Reports. DOI:10.1038/s41598-017-03806-6. Park, B. and Nieh, J.C. (2017) Seasonal trends in honey bee pollen foraging revealed through DNA barcoding of bee-collected pollen. Insectes Sociaux. 64(3): 425-437. DOI 10.1007/s00040-017-0565-8 Lam, C., Li., Y., Landgraf, T., and Nieh, J.C. (2017) Dancing attraction: followers of honey bee tremble and waggle dances exhibit similar behaviors. Biology Open. 6: 810-817, DOI:10.1242/bio.025445 Tosi, S., G. Burgio, and Nieh, J.C. (2017) A common neonicotinoid pesticide, thiamethoxam, impairs honey bee flight ability. Scientific Reports. DOI:10.1038/s41598-017-01361-8. PMID: 28446783 Huey, S., and Nieh, J. C. (accepted) Foraging at a safe distance: crab spider effects on pollinators. Ecological Entomology. Dong Shihao, Wen Ping, Xinyu Li, Tan Ken, and Nieh, J.C. (2017) Resisting majesty: Apis cerana, has lower antennal sensitivity and decreased attraction to queen mandibular pheromone than Apis mellifera. Scientific Reports. DOI: 10.1038/srep44640. Ya-nan Cheng, Wen Ping, Shi-hao Dong, Ken Tan, Nieh, J.C. (2017) Poison and alarm: The Asian hornet Vespa velutina uses sting venom volatiles as an alarm pheromone. Journal of Experimental Biology. 220: 645-651. Gong, Z, Wang, C., Nieh, J.C., and Tan, K. (2016) Inhibiting DNA methylation alters olfactory learning extinction but not acquisition in Apis cerana and Apis mellifera. Journal of Insect Physiology. 90:43-49. Wang, Z., Wen, P., Yufeng, Q., Dong, S., Li, J. Tan, K., and Nieh, J.C. (2016) Bees eavesdrop upon informative and persistent signal compounds in alarm pheromones. Scientific Reports. 6: 25693 EP. Tan, K., Dong, S., Liu, X., Wang, C., Li, J., and Nieh J.C. (2016) Honey bee inhibitory signaling is tuned to threat severity and can act as a colony alarm signal. PLOS Biology. 14(3): e1002423-19. Lecocq, A., Jensen, A.B., Kryger, P., and Nieh, J.C. (2016) Parasite infection accelerates age polyethism in young honey bees. Scientific Reports. 6: 11. DOI:10.1038/srep22042. Lau, P. and Nieh J.C. (2016) Salt preferences of honey bee water foragers. Journal of Experimental Biology 219: 790-796. Zhang, E. and Nieh, J.C. (2015) A neonicotinoid, imidacloprid, impairs honey bee aversive learning of simulated predation. Journal of Experimental Biology 218 3199-3205. Jack-McCollough R.T. and Nieh, J.C. (2015) Honey bees tune excitatory and inhibitory recruitment signaling to resource value and predation risk. Animal Behaviour. 110: 9-17. Eiri, D., Endler, M., Suwannapong, G., and Nieh, J.C. (2015) Nosema ceranae can infect honey bee larvae and reduce subsequent adult longevity.PLOS One. DOI:10.1371/journal.pone.0126330. Tan, K., Cheng, W., Dong, S., Liu, X., Wang, Y. and Nieh, J. C. (2015) A neonicotinoid impairs olfactory learning in Asian honey bees (Apis cerana) exposed as larvae or as adults. Scientific Reports. DOI:10.1038/srep10989. Len, A., Arias-Castro, C., Rodrguez-Mendiola, A., Meza-Gordillo, R., Gutirrez-Miceli, F. A., and Nieh, J.C. (2015) Colony foraging allocation is finely tuned to food distance and sweetness even close to a bee colony. Entomologia Experimentalis et Applicata 155: 47-53. Li, J., Wang, Z., Tan, K., Qu, Y., and Nieh JC (2014) Effects of natural and synthetic alarm pheromone and individual pheromone components on foraging behavior of the giant Asian honey bee, Apis dorsata. Journal of Experimental Biology. 217:3512-3518. Li, J., Wang, Z., Qu, Y., and Nieh J.C. (2014) Giant Asian honey bees use olfactory eavesdropping to detect and avoid ant predators. Animal Behaviour. 97:69-76. Tan, K., Chen, W., Dong, S., Liu, X., Wang, Y., and Nieh. J.C. (2014) Imidacloprid alters foraging and increases risky behavior in bees. PLOS One 9(7): e102725. Goodale, E., Kim, E., Nabors, A., Henrichon, S., and Nieh, J. C. (2014). The innate responses of bumble bees to flower patterns: separating the nectar guide from the nectary changes bee movements and search time. Naturwissenschaften. DOI 10.1007/s00114-014-1188-9. Lichtenberg, E., Graff Zivin, J., Hrncir, M. and Nieh, J. C. (2014) Eavesdropping selects for conspicuous signals. Current Biology. 24(1): R598-599. DOI: http://dx.doi.org/10.1016/j.cub.2014.05.062 Bray, A. and Nieh, J. C. (2014) Non-consumptive predator effects shape honey bee foraging and recruitment dancing. PLOS One 9(1) e87459. Tan, K., Hu, Z., Chen, W., Wang, Z., Wang, Y., and Nieh, J. C. (2013) Fearful foragers: honey bees tune colony and individual foraging to multi-predator presence and food quality. PLOS ONE. 8(9) e75841, DOI:10.1371/journal.pone.0075841. Hagbery, J. and Nieh, J. C. (2012) Individual lifetime pollen and nectar foraging preferences in bumble bees. Naturwissenschaften. DOI 10.1007/s00114-012-0964-7. 99:821-832. Goodale, E. and Nieh, J. C. (2012) Public use of olfactory information associated with predation in two species of social bees. Animal Behaviour. 84:919-924. Eckles, M. A., Roubik, D. W. and Nieh, J. C. (2012) A stingless bee can use visual odometry to estimate both height and distance. Journal of Experimental Biology. 215:3155-3160. Eiri D, Nieh JC (2012) A nicotinic acetylcholine receptor agonist affects honey bee sucrose responsiveness and decreases waggle dancing. Journal of Experimental Biology. 215:2022-2029. Nguyen H, Nieh JC (2012) Colony and individual forager responses to food quality in the New World bumble bee, Bombus occidentalis. Journal of Insect Behavior. 25:60-69. Nguyen, H., and Nieh, J. C. (2011) Colony and individual forager responses to food quality in the New World Snchez, D., Nieh, J. C., and Vandame, R. (2011) Visual and chemical cues provide redundant information in the multimodal recruitment system of the stingless bee Scaptotrigona mexicana (Apidae, Meliponini). Insectes Sociaux, in press. Lichtenberg, E.M., Hrncir, M., and Nieh, J.C. (2011) Optimal eavesdropping by foraging stingless bees: empirical and theoretical support. Behavioral Ecol. Sociobiol. 65: 763-774. Sadler, N. and Nieh, J. C. (2011) Honey bee forager thoracic temperature is tuned to broad scale differences in recruitment motivation. J. Exper. Biol.. 214: 469-475. Johnson, B.R., Nieh, J.C. (2010) Modeling the adaptive role of negative signaling in honey bee intraspecific competition. J. Insect Behavior. 23: 459-471. Ramrez, S.R., Nieh, J. C., Quental, T.B., Roubik, D.W., Imperatriz-Fonseca, V.L., and Pierce, N.E. (2010) Molecular phylogeny of the stingless bee genus Melipona (Hymenoptera: Apidae) and the evolution of recruitment communication in eusocial Apidae. Mol. Phylogenetic Evolution. 56: 519-525. Nieh, J. C. (2010) A negative feedback signal that is triggered by peril curbs honey bee recruitment. Curr. Biol. 20: 310-315. Lau, C.W. & Nieh, J.C. (2010) Honey bee stop signal production under overcrowded feeder conditions. Apidologie 41: 87-95. Lichtenberg, E.M., Imperatriz-Fonseca, V.L., and Nieh, J.C. (2010) Behavioral suites mediate group-level foraging dynamics in communities of tropical stingless bees. Insectes Sociaux 57: 105-113. Hammer, T.J., Hata, C., & Nieh, J.C. (2009) Thermal learning in the honey bee, Apis mellifera. Jou Snchez, D., Nieh, J.C., Len, A., & Vandame, R. (2009) Food recruitment information can spatially redirect employed stingless bee foragers. Ethology.115: 1175-1181. Toda, N., Song, J., & Nieh, J.C. (2009) Bumble bees exhibit the memory spacing effect. Naturwissenschaften. 96: 1185-1191. Kuhn-Neto, B., Contrera, F.A.L., Castro, M.S., & Nieh, J.C. (2009) Long distance foraging and recruitment by a stingless bee, Melipona mandacaia. Apidologie. 40: 472-480. Lichtenberg, E.M., Hrncir, M., & Nieh, J.C. (2009) A scientific note: Foragers deposit attractive scent marks in a stingless bee that does not communicate food location. Apidologie. 40: 1-2. Kitaoka, T.K. & Nieh, J.C. (2009) Bumble bee pollen foraging regulation: role of pollen quality, storage levels, and odor. Behavioral Ecology and Sociobiology. 63: 501-510. Renner, M.A. & Nieh, J.C. (2008) Bumble bee olfactory information flow and contact-based foraging activation. Insectes Sociaux 55: 417-424. Mapalad, K. S., Leu, D. & Nieh, J. C. (2008). Bumble bees heat up for high quality pollen. Journal of Experimental Biology, 211:2239-2242. Eckles M. A., Wilson E. E., Holway D. A., and Nieh, J. C. (2008) Yellowjackets (Vespula pensylvanica) thermoregulate in response to changes in protein concentration. Naturwissenschaften. 95: 787-792. Snchez, D., Nieh, J., and Vandame, R. (2008) Experience-based interpretation of visual and chemical information in Scaptotrigona mexicana (Apidae, Meliponini). Animal Behavior 76: 407-414 Contrera, F. A. L. and J. C. Nieh (2007). Effect of forager-deposited odors on the intra-patch accuracy of recruitment in the stingless bees Melipona panamica and Partamona peckolti (Apidae, Meliponini). Apidologie 38:584-594 Contrera, F. A. L. and J. C. Nieh. (2007). The effect of ambient temperature on forager sound production and thoracic temperature in the stingless bee, Melipona panamica. Behavioral Ecology and Sociobiology 61:887-897. Nieh, J. C., Len, A., Cameron, S., and Vandame, R. (2006) Hot bumblebees at good food: thoracic temperature of feeding Bombus wilmattae foragers is tuned to sucrose concentration. Journal of Experimental Biology 209: 4185-4192. Wilson ES, Holway D, Nieh JC (2006) Cold anesthesia decreases foraging recruitment in the New World bumblebee, Bombus occidentalis. Journal of Apicultural Research 45:169-172 Nieh, J. C. & Snchez, D. (2005) Effect of food quality, distance, and height on thoracic temperature in the stingless bee, Melipona panamica. Journal of Experimental Biology 208:3933-3943. Saraiva, A. M., Nieh, J. C., Cartolano Jr., E. A. (2005) EthoLog: a tool for data acquisition in behavioral studies. 2005 EFITA/WCCA Joint Congress on Information Technology in Agriculture. 1, 1-7. Contrera, F. A. L., Imperatriz-Fonseca, V. L., Nieh, J. C. (2005) Temporal and climatological influences on flight activity in Trigona hyalinata (Apidae, Meliponini). Revista Tecnologia e Ambiente. 10:35-43 Nieh, J. C., Kruizinga, K., Contrera, F. A. L., Barreto, L. S. & Imperatriz-Fonseca, V. L. (2005) Effect of group size on the aggression strategy of an extirpating stingless bee, Trigona spinipes. Insectes Sociaux. 52, 1-8. Nieh, J. C., Contrera, F. A. L., Yoon, R. R., Barreto, L. S., & Imperatriz-Fonseca, V. L. (2004) Polarized short odor-trail recruitment communication by a stingless bee, Trigona spinipes. Behavioral Ecology and Sociobiology, 56, 435-448. Nieh, J. C., Barreto, L. S., Contrera, F. A. L. & Imperatriz-Fonseca, V. L. (2004) Olfactory eavesdropping by a competitively foraging stingless bee, Trigona spinipes. Proceedings of the Royal Society of London B, 271, 1633-1640. Snchez, D., Nieh, J. C., Hnaut, Y., Cruz, L., & Vandame, R. (2004) High precision during food recruitment of experienced (reactivated) foragers in the stingless bee Scaptotrigona mexicana (Apidae, Meliponini). Naturwissenschaften. 91, 346-349. Nieh, J. C., Contrera, F. A. L. & Nogueira-Neto, P. (2003) Pulsed mass-recruitment by a stingless bee, Trigona hyalinata. Proceedings of the Royal Society of London B, 270, 2191-2196. Nieh, J. C., Contrera, F. A. L., Ramrez, S. & Imperatriz-Fonseca, V. L. (2003) Variation in the ability to communicate 3-D resource location by stingless bees from different habitats. Animal Behaviour, 66, 1129-1139. Nieh, J. C., Contrera, F. A. L., Rangel, J. & Imperatriz-Fonseca, V. L. (2003) Effect of food location and quality on recruitment sounds and success in two stingless bees, Melipona mandacaia and Melipona bicolor. Behavioral Ecology and Sociobiology, 55, 87-94. Nieh, J. C., Ramrez, S. & Nogueira-Neto, P. (2003) Multi-source odor-marking of food by a stingless bee, Melipona mandacaia. Behavioral Ecology and Sociobiology, 54, 578-586. Nieh, J. C. & Tautz, J. (2000) Behavior-locked signal analysis reveals weak 200-300 Hz comb vibrations during the honeybee waggle dance. Journal of Experimental Biology, 203, 1573-1579. Nieh, J. C., Tautz, J., Spaethe, J. & Bartareau, T. (2000) The communication of food location by a primitive stingless bee, Trigona carbonaria. Zoology, 102, 239-246. Nieh, J. C. (1999) Stingless-bee communication. American Scientist, 87, 428-435. Nieh, J. C. & Roubik, D. W. (1998) Potential mechanisms for the communication of height and distance by a stingless bee, Melipona panamica. Behavioral Ecology and Sociobiology, 43, 387-399. Nieh, J. C. (1998) The food recruitment dance of the stingless bee, Melipona panamica. Behavioral Ecology and Sociobiology, 43, 133-145. Nieh, J. C. (1998) The honey bee shaking signal: function and design of a modulatory communication signal. Behavioral Ecology and Sociobiology, 42, 23-36. Nieh, J. C. (1998) The role of a scent beacon in the communication of food location in the stingless bee, Melipona panamica. Behavioral Ecology and Sociobiology, 43, 47-58. Nieh, J. C. & Roubik, D. W. (1995) A stingless bee (Melipona panamica) indicates food location without using a scent trail. Behavioral Ecology and Sociobiology, 37, 63-70. Nieh, J. C. (1993) The stop signal of honey bees: Reconsidering its message. Behavioral Ecology and Sociobiology, 33, 51-56. Runestad, J. A., Ruff, C. B., Nieh, J. C., Thorington, R. W., Jr. & Teaford, M. F. (1993) Radiographic estimation of long bone cross-sectional geometric properties. American Journal of Physical Anthropology, 90, 207-213. Allmon, W. D., Nieh, J. C. & Norris, R. D. (1990) Drilling and peeling of turreteline gastropods since the Late Cretaceous. Palaeontology, 33, 595-611. \n",
            "-------------------- Scraping faculty url 120/183 --------------------\n",
            "axel nimmerjahn  Research The human brain consists of sets of cells that form networks of dazzling complexity. Much research has focused on understanding the circuits formed by neurons, the electrically-excitable cells that process and transmit information. However, glial cells, the second major cell type in the brain, account for up to ninety percent of human brain cells and more than fifty percent of the brain's volume. For a long time, these cells were believed to have a merely passive, supportive role. However, over the past few years, it has become clear that glial cells make crucial contributions to the formation, operation and adaptation of neural circuitry. Work in my lab is centered on innovating light microscopic tools that enable the study of these electrically largely non-excitable cells and their interaction with other cells in the intact mammalian brain. We have created tools for cell-type-specific staining and genetic manipulation, for imaging cellular dynamics in awake behaving mammals and for automated analysis of large-scale imaging data. This allows us to directly address longstanding questions regarding glial function in the intact healthy and diseased brain. Resolving these fundamental questions has broad implications for our view of glial cells, the way information is processed in the brain, the interpretation of functional brain imaging signals and the treatment of neurodegenerative brain disease.  Select Publications List of Publications Nimmerjahn, A., Mukamel, E.A., and Schnitzer, M.J. (2009). Motor behavior activates Bergmann glial networks. Neuron 62, 400-412. Mukamel, E.A., Nimmerjahn, A., and Schnitzer, M.J. (2009). Automated analysis of cellular signals from large-scale calcium-imaging data. Neuron 63, 747-760. Flusberg, B.A.*, Nimmerjahn, A.*, Cocker, E.D.*, Mukamel, E.A., Barretto, R.P.J., Ko, T.H., Burns, L.D., Jung, J.C., and Schnitzer, M.J. (2008). High-speed, miniaturized fluorescence microscopy in freely moving mice. Nature Methods 5, 935-938. *Authors contributed equally Nimmerjahn, A., Kirchhoff, F., and Helmchen, F. (2005). Resting microglial cells are highly dynamic surveillants of brain parenchyma in vivo. Science 308, 1314-1318. Nimmerjahn, A., Kirchhoff, F., Kerr, J.N.D., and Helmchen, F. (2004). Sulforhodamine 101 as a specific marker of astroglia in the neocortex in vivo. Nature Methods 1, 31-37. Dittgen, T., Nimmerjahn, A., Komai, S., Licznerski, P., Waters, J., Margrie, T.W., Helmchen, F., Denk, W., Brecht, M., and Osten, P. (2004). Lentivirus-based genetic manipulations of cortical neurons and their optical and electrophysiological monitoring in vivo. Proc. Natl. Acad. Sci. USA 101, 18206-18211. \n",
            "-------------------- Scraping faculty url 121/183 --------------------\n",
            "maho niwa rosen  Biography Maho Niwa received a MS from the Chemistry Department at Brandeis University and a Ph.D. from the Biochemistry Department at Baylor College of Medicine. She was a Jane Coffin Childs Cancer postdoctoral fellow in Dr. Peter Walter's laboratory at UCSF.  Research How does a cell tailor its components to the many different demands or insults it receives during its lifetime? From yeast to humans, the endoplasmic reticulum (ER) is a critical compartment where a large fraction of total cellular proteins, including almost all plasma membrane proteins and secreted proteins undergo folding, modification, and maturation. In addition, synthesis of almost all cellular lipids, at least in their initial steps, occurs in the ER. The ER plays further vital roles in the detoxification of drugs and poisons (liver ER), as well as in intracellular Ca2+ regulation. As complex as these roles already are, ER functions need to change constantly to respond to alterations in developmental stage, new environmental stresses, or a whole host of cellular insults. Cells have to recognize such changes and make proper adjustments for the needed ER functions. It is increasingly recognized that insufficient ER protein folding capacity can be an important contributing factor to the pathology of human diseases including a long list of cancers, diabetes, obesity, Alzheimers, Cystic Fibrosis, and Parkinsons disease. We are specifically interested in two major areas: (1) How increased demands for ER functions in response to cellular stress and insults are recognized and lead to production of the needed ER functions, and (2) How is the ER divided the cell cycle? Progress in this latter area has led us to describe a new cell checkpoint. (1) How increased demands for ER functions in response to cellular stress and insults are recognized and leads to production of the needed ER functions? The Unfolded Protein Response (UPR) operates in a variety of immune and other cells as the signaling pathway to up-regulate ER levels or functions so that cells may deal with stressful or altered circumstances. Three UPR transmembrane protein sensors, IRE1, PERK and ATF6, in the ER membrane initiate these regulatory downstream events via distinct and unusual mechanisms. IRE1 is an ER transmembrane receptor kinase that senses changes in the ER via its ER luminal domain. When activated, IRE1 acts as a sequence specific RNAase (splicase) to generate the spliced form of XBP1 mRNA, which then can generate active XBP1 transcription factor. Because activation of this IRE1 pathway is observed in many solid tumors and hematological cancers, we searched for and identified inhibitors of IRE1s XBP1 RNA splicease. We discovered Irestatin (STF-083010), which inhibits the IRE1 splicease and also has anti-myeloma activity on cancer biopsies from myeloma patients. Interestingly, we show that cancer chemotherapy drug doxobuciin, a DNA replication inhibitor, and acridine derivatives also inhibit the IRE1 splicease activity. Activated IRE1 is also known to have a second RNase activity that cleaves a subset of mRNAs for degradation, reducing translation of such mRNA. Recently, we found that the activation mechanisms of these two IRE1 RNases are distinct and can be regulated separately. We also found that the two IRE1 RNase activities contribute differentially to cell death and survival after ER stress. All these issues are ongoing projects in the lab. ATF6 is another unique UPR sensor. Interestingly, ATF6 is an ER transmembrane protein that is a cryptic transcription factor. Upon activation, ATF6 is known to be cleaved within its ER transmembrane domain by proteases, which frees the cytoplasmic domain to travel into the nucleus to become a UPR-specific transcription factor. Most recently, we have found that ATF6 can become an activated transcription factor by a previously undiscovered mechanism. Activating ATF6 in this manner preferentially causes transcription of a set of target genes involved in lipid biosynthesis and regulation. These discoveries should allow us to probe the differences between proteotoxic and lipotoxic stressors and the roles they play in different disease. (2) A new cell cycle checkpoint ensuring that progeny cells get functional ER before cell division proceeds: the ERSU pathway. We have made the pioneering discovery of a new yeast cell cycle checkpoint, the ER Stress Surveillance pathway (ERSU), which acts to ensure the inheritance of a functional ER. We show the ERSU pathway is independent of the yeast UPR pathway. This cell cycle block to stressed ER inheritance is accompanied by a cytokinesis block. While the cell cycle is halted by ERSU, the UPR pathway becomes activated and ultimately restores ER function, releasing the ERSU block and allowing re-entry into the cell cycle. We have found that one of the ERSU pathways initiating signals is the sphingolipid, phytosphingosine (PHS). Indeed, we find that ER stress leads to a transient increase in PHS levels. Moreover, we show that a family of ER membrane proteins, the Reticulons (Rtn 1, 2, and Yop1), are intimately involved in the early steps of the ERSU pathway. This means that Reticulons, previously known to be involved primarily in generating ER curvature, have a second important function the ERSU cell cycle checkpoint that governs recognition and inheritance of a functional ER. Our understanding of the UPR and ERSU pathways holds unique promise for the development of new methods to approach human diseases caused by misregulation of the ER.  Select Publications Miller, M., Tam, A. B., Mueller, J., L., Rosenthal, P., Beppu, A., Gordillo, R., McGeough, M. D., Voung, C., Doherty, T. A., Hoffman, H. M., Suzukawa, M., Niwa, M, and Broide, D. H. (2017) Tageting epithelial ORMDL3 increases, rather than reduces, airway responsiveness through increased sphingosine-1-phosphate. J of Immnol. 198:3017-3022. Jiang D, Tam AB, Alagappan M, Hay MP, Gupta A, Kozak MM, Solow-Cordero DE, Lum PY, Denko NC, Giaccia AJ, Le QT, Niwa M, Koong AC. (2016) Acridine Derivatives as Inhibitors of the IRE1-XBP1 Pathway Are Cytotoxic to Human Multiple Myeloma. Mol Cancer Ther. 15: 2055-65. PMCID:PMC5010920) Jiang D, Lynch C, Medeiros BC, Liedtke M, Bam R, Tam AB, Yang Z, Alagappan M, Abidi P, Le QT, Giaccia AJ, Denko NC, Niwa M, Koong AC. (2016) Identification of Doxorubicin as an inhibitor of the IRE1a-XBP1 Axis of the Unfolded Protein Response. Sci Rep. 6: 33353. (PMCID:PMC5025885) Pina-Nunez, F. N., Fleming, T., Pogliano, K., and Niwa, M. (2016) Reticulons are important for ER inheritance regulation and function during ER stress. Dev Cell. 37:279-288. (PMCID:PMC4911346) Jiang, D., Niwa,, M., and Koong, A. C.(2015) Targeting the IRE1a-XBP1 Branch of Unfolded Protein Response in Human Diseases. Seminars in Cancer Biology, 33: 45-56. (PMCID:PMC4523453) Pina-Nunez, F. N., Niwa, M. (2015) Initiation of the Endoplasmic Reticulum (ER) inheritance is determined by the ER functional status during the cell cycle. eLife 4:e06970 (PMCID:PMC4555637) Tam, A. B., Koong, A. C., and Niwa, M. (2014) IRE1 Differentially Cleaves HAC1/XBP1 RNA and ER-Associated RNA. Cell Reports 9:850-858 (PMCID:PMC4486022) Miller, M., Rosenthal, P., Beppu, A., Mueller, J. L., Hoffman, H. M. Tam, A. B., Doherty, T. A., McGeough, M. D., Pena, C., Suzukawa, M., Niwa, M., and Broide, D. H. (2014) ORMDL3 transgenic mice have increased airway remodeling characteristic of asthma. J. immunology. 192: 3475-87 (PMCID:PMC3981544) Miller, M., Tam, A. B., Cho, J. Y., Doherty, T. A., Pham, A., Khorram, N., Rosenthal, P., Mueller, J. L., Hoffman, H. M., Suzukawa, M., Niwa, M, and Broide, D. H. (2012) ORMDL3 is an inducible lung epithelial gene regulating metalloproteases, chemokines, OAS, and ATF6. Proc Natl Acad Sci U S A. 109:16648-53 Tam, A. B., ODea, E., Hoffmann, A., and Niwa M. (2012) ER stress activates NF-B by integrating function of basal IKK activity, IRE1 and PERK. PLoS One 7(10):e45078 Martin-Perez, R., Niwa, M., and Lopez-Rivas, A. (2012) ER stress sensitizes cells to TRAIL through down-regulation of FLIP and Mcl-1 and PERK-dependent up-regulation of TRAIL-R2. Apoptosis 17: 349-63. Chawla, A., Chakrabarti, S., Ghosh, G., and Niwa M. (2011) Attenuation of the yeast UPR response is essential for survival and is mediated by IRE1 kinase. J. Cell Biol. 193:41-50. (PMC3082189). Papandreou, I., Olson, M., Van Melckebeke, H., Lust, S., Tam, A., Solow-Cordero, D. E., Bouley, D. M., Denko, N., Offner, F., Niwa, M. & Koong, A. C. (2011) Identification of a small-molecule inhibitor of Ire1 endonuclease activity with cytotoxic activity against multiple myeloma. Blood 117: 1311-1314. (PMID:21081713) Babour A., Bicknell, A. B. Tourtellotte, J., and Niwa M. (2010) An organelle surveillance pathway monitors functional fitness of the endoplasmic reticulum to control its inheritance. Cell 142: 256-269. (PMC3359143) Bicknell A., and Niwa M. (2010) Late Phase of the Endoplasmic Reticulum Stress Response Pathway is Regulated by Hog1 MAP Kinase. J of Biol. Chem. 285: 17545-17555. (PMC2878519) Bicknell A., and Niwa M. (2009) Regulating ER function through the Unfolded Protein Response. The Handbook of Cell Signaling 2nd ed., Oxford: Academic Press. 2511-2525. (PMC2064362) DuRose, J., Kaufman, R. A., Rothblum, L., and Niwa M. (2009) PERK regulates both cellular translation repression and RRN3 mediated rRNA transcription down-regulation during the Unfolded Protein Response pathway. Mol Cell Biol. 29: 4295-4307. (PMC19470760) Brunsing, R., Oomori, S., Weber, F., Ware, A., Friend, L., Rickert, R., and Niwa, M. (2008) B- and T-cell development both involve activity of the Unfolded Protein Response pathway. J. Biol. Chem. 283: 17954-17961. (PMID:18375386) Bicknell A.A., Babour A., Federovitch C.M., and Niwa M. (2007) A Novel role for the Unfolded Protein Response (UPR) in cytokinesis reveals previously uncharacterized housekeeping function for the UPR. J. Cell Bio.l 177: 1017-1027. (PMC2064362) DuRose, J., Tam A., and Niwa M. (2006) Molecular mechanism of differential activation of three ER proximal UPR components. Mol. Biol. Cell 17, 3095-3107 (PMC1483043) \n",
            "-------------------- Scraping faculty url 122/183 --------------------\n",
            "clodagh o'shea \n",
            "-------------------- Scraping faculty url 123/183 --------------------\n",
            "karen oegema  Biography Karen Oegema received her PhD in cell biology from the University of California, San Francisco and conducted postdoctoral work at the European Molecular Biology Laboratory in Heidelberg and the Max Planck Institute for Cell Biology & Genetics in Dresden. She was a Pew fellow in the Biomedical Sciences and is a recipient of the American Society of Cell Biology WICB Early and Mid-Career Awards. She is a monitoring editor for the Journal of Cell Biology and a lifetime fellow of the American Society for Cell Biology.  Research My lab is interested in the interface between cell and cancer biology. My group has had a long-term interest in the molecular machinery underlying mitosis and cytokinesis. This research has revealed that in addition to common themes, there is also significant variation between cell types in how cell division is accomplished. A current focus in the lab is to determine whether these differences can be leveraged to induce death due to aberrant mitosis in a cell type specific fashion as a chemotherapeutic strategy. An example of this has arisen from our work on centrosomes. Through a combination of experiments in the model system C. elegans and in human cells, we have made important contributions to understanding the pathway that controls centrosome duplication, which depends on a centrally important regulatory kinase called Plk4. To explore the therapeutic potential of centrosome removal in cancer, we collaborated with the Small Molecule Discovery Group in the Ludwig Institute for Cancer Research to develop the first potent, specific, cellularly active Plk4 inhibitor. Our experiments using this inhibitor, which we call centrinone, revealed striking differences in how cells respond to centrosome removal, and have suggested that centrosome removal could have therapeutic potential for specific cancer types. We are currently working to understand the molecular basis of these differential effects and are initiating screens to expand our understanding of variation in the cell division machinery across cell types. A second research avenue in my lab has been aimed at generating a phenotypic map of the genes essential for embryonic development in a model metazoan. In this project, we are taking a high-content approach based on time-lapse imaging of tissue specification and morphogenesis during embryogenesis to generate a functional gene network for the set of ~2000 C. elegans genes required for embryonic development. To analyze the 26,000 time-lapse movies resulting from this effort, we have developed automated methods for phenotypic parameterization and comparison that allow us to identify genes with similar phenotypes, suggesting that they function in common pathways. Of the genes that we are characterizing in this effort, 75% are conserved in humans and 900 are functionally uncharacterized, suggesting that this approach will allow us to predict function for a large number of human genes.  Select Publications Wang S., Ochoa S.D., Khaliullin R.N., Gerson-Gurwitz A., Hendel J.M., Zhao Z., Biggs R., Chisholm A.D., Desai A., Oegema, K.*, and R.A. Green*. 2019. A high-content imaging approach to profile C. elegans embryonic development. Dev Cell . 2019. 48:864-872. doi: 10.1016/j.devcel.2019.02.002. PMID: 30827898 *Co-corresponding Khaliullin, R., Green, R.A., Shi, L.Z., Berns, M.W., Gomez-Cavazos, J.S., Desai, A. and K. Oegema. 2018. Positive feedback between contractile ring myosin and ring-directed cortical flow drives cytokinesis. Elife . doi: 10.7554/eLife.36073. PMID: 29963981. Lee, K-Y., Green, R.A. Gutierrez, E., Gomez-Cavazos, J.S., Kolotuev, I., Wang, S., Desai, A., Groisman, A., and K. Oegema. 2018. CYK-4 functions independently of its centralspindlin partner ZEN-4 to cellularize oocytes in germline syncytia. Elife . doi: 10.7554/eLife.36919. PMID: 29989548. Meitinger, F., Anzola, J.V., Kaulich, M., Richardson, A., Stender, J.D., Benner, C., Glass, C.K., Dowdy, S.F., Desai, A., Shiau, A.K. and K. Oegema. 2016. 53BP1 and USP28 mediate p53 activation and G1 arrest after centrosome loss or extended mitotic duration. J Cell Bio . 214:155-66. doi: 10.1083/jcb.201604081. PMID: 27432897 Wang, S., Wu, D., Quintin, S., Green, R.A., Cheerambathur, D.K., Ochoa, S.D., Desai, A., and K. Oegema. 2015. NOCA-1 Functions with -tubulin and in parallel to Patronin to assemble non-centrosomal microtubule arrays in C. elegans . Elife . doi: 10.7554/eLife.08649. PMID: 26371552. Woodruff, J.B., Wueseke, O., Viscardi, V., Mahamid, J., Ochoa, S.D., Bunkenborg, J., Widlund, P.O., Pozniakovsky, A., Zanin, E., Bahmanyar, S., Zinke, A., Hong, S.H., Decker, M., Baumeister, W., Andersen, J.S., Oegema, K.,* and A.A. Hyman*. 2015. Regulated assembly of a supramolecular centrosome scaffold in vitro. Science . 348(6236):808-12. doi: 10.1126/science.aaa3923. PMID: 25977552. *Co-corresponding Wong, Y.L., Anzola, J.V., Davis, R.L., Yoon, M., Motamedi, A., Kroll, A., Seo, C.P., Hsia, J.E., Kim, S.K., Mitchell, J.W., Mitchell, B.J., Desai, A., Gahman, T.C., Shiau, A.K., and K. Oegema. 2015. Reversible centriole depletion with an inhibitor of Polo-like kinase 4. Science >. 348:1155-60. doi: 10.1126/science.aaa5111. PMID: 25931445 Zanin, E., Desai, A., Poser, I., Toyoda, Y., Andree, C., Moebius, C., Bickle, M., Conradt, B., Piekny, A., and K. Oegema. 2013. A Conserved RhoGAP limits M-phase contractility and coordinates with microtubule asters to confine RhoA during cytokinesis. Dev Cell . 26:496-510. Lettman, M.M., Wong, Y.L., Viscardi, V., Niessen, S., Chen, S., Shiau, A.K., Zhou, H., Desai, A., and K. Oegema. 2013. Direct binding of SAS-6 to ZYG-1 recruits SAS-6 to the mother centriole for cartwheel assembly. Dev Cell . 25:284-298. Green, R., Kao, H.L., Audhya, A., Arur, S., Mayers, J.R., Fridolfsson, H., Schulman, M., Schloissnig, S., Niessen, S., Laband, K., Wang, S., Starr, D., Hyman, A., Schedl, T., Desai, A., Piano, F., Gunsalus, K.C., and K. Oegema. A high-resolution C. elegans essential gene network based on phenotypic profiling of a complex tissue. 2011. Cell . 145:470-482. PMID: 21529718. \n",
            "-------------------- Scraping faculty url 124/183 --------------------\n",
            "melinda owens  Biography Melinda Owens is passionate about effectively helping all students learn biology. After receiving her PhD in neuroscience from UCSF, she taught at a range of institutions, including Stanford University and California State University, East Bay. She then was a SEPAL Postdoctoral Fellow in Biology Education with Kimberly Tanner at San Francisco State University. She joined UCSD in 2018.  Research It is critically important to teach undergraduates about biology more effectively and to retain more of them in scientific and health careers. My lab addresses this issue by examining two broad questions: How do college instructors teach? How do college students think about biology? How Do Instructors Teach? Prior research shows that certain teaching strategies are more likely to promote engagement, retention, and learning among diverse students, but it has been difficult to examine the strategies instructors use on a large scale. Therefore, in collaboration with San Francisco State, we developed DART (Decibel Analysis for Research in Teaching). DART (dart.sfsu.edu) is a software tool that can quickly analyze classroom audio recordings to estimate with ~90% accuracy the time spent using certain teaching activities. We use DART to investigate a variety of questions, with the goals of better understanding how instructors teach and promoting evidence-based teaching practices. DART uses classroom sound levels (dark blue line) to estimate the extent of \"active learning\" by classifying class time into these categories: Single Voice (e.g. lecture), Multiple Voice (e.g. pair and group discussions), and No Voice (e.g. student thinking or writing). Owens et al. 2017 How Do Students Think? Students learn more effectively when instructors know what ideas their students already have. In particular, previous studies reveal that students often harbor misconceptions that inhibit their ability to develop biological expertise. Many misconceptions are rooted in ways of intuitive thinking that predate modern scientific thought. We are particularly interested in how undergraduate students understand and reason about health topics, such as vaccines. We ask undergraduates about their ideas about specific topics and examine their misconceptions and the types of reasoning they use. We will use our results to design education interventions to promote better learning about these topics.  Select Publications Research in Biology Education M.T. Owens , G. Trujillo, S.B. Seidel, C.D. Harrison, K.M. Farrar, H.P. Benton, J.R. Blair, K.E. Boyer, J. Breckler, L.W. Burrus, D.T. Byrd, N. Caporale, E.J. Carpenter, Y.-H.M. Chan, J. Chen, L. Chen, L.H. Chen, D.S. Chu, W.P. Cochlan, R.J. Crook, K.D. Crow, J.R. de la Torre, W.F. Denetclaw, L.M. Dowdy, D. Franklin, M. Fuse, M.A. Goldman, B. Govindan, M. Green, H.E. Harris, Z.-H. He, S. Ingalls, P.D. Ingmire, A.R.B. Johnson, J.D. Knight, G. LeBuhn, T.L. Light, C. Low, L. Lund, L.M. Mrquez-Magaa, V.C. Miller-Sims, C.A. Moffatt, H. Murdock, G.L. Nusse, V.T. Parker, S.G. Pasion, R. Patterson, P.S. Pennings, J.C. Ramirez, R.M. Ramirez, B. Riggs, R.V. Rohlfs, J.M. Romeo, B.S. Rothman, S.W. Roy, T. Russo-Tait, R.N.M Sehgal, K.A. Simonin, G.S. Spicer, J.H. Stillman, A. Swei, L.C. Timpe, V.T. Vredenburg, S.L. Weinstein, A.G. Zink, L.A. Kelley, C.R. Domingo, K.D. Tanner. 2018. Collectively Improving Our Teaching: Attempting Biology Department-wide Professional Development in Scientific Teaching. CBE: Life Sciences Education. March 1; 17(1):ar2. (PMID: 29326102) M.T. Owens *, S.B. Seidel*, M. Wong*, T.E. Bejines, S. Lietz, J.R. Perez, S. Sit, Z.-S. Subedar, G.N. Acker, S.F. Akana, B. Balukjian, H.P. Benton, J.R. Blair, S.M. Boaz, K.E. Boyer, J.B. Bram, L.W. Burrus, D.T. Byrd, N. Caporale, E.J. Carpenter, Y.-H.M. Chan, L. Chen, A. Chovnick, D.S. Chu, B.K. Clarkson, S.E. Cooper, C. Creech, K.D. Crow, J.R. de la Torre, W.F. Denetclaw, K.E. Duncan, A.S. Edwards, K.L. Erickson, M. Fuse, J.J. Gorga, B. Govindan, L.J. Green, P.Z. Hankamp, H.E. Harris, Z.-H. He, S. Ingalls, P.D. Ingmire, J.R. Jacobs, M. Kamakea, R.R. Kimpo, J.D. Knight, S.K. Krause, L.E. Krueger, T.L. Light, L. Lund, L.M. Mrquez-Magaa, B.K. McCarthy, L.J. McPheron, V.C. Miller-Sims, C.A. Moffatt, P.C. Muick, P.H. Nagami, G.L Nusse, K.M. Okimura, S.G. Pasion, R. Patterson, P.S. Pennings, B. Riggs, J. Romeo, S.W. Roy, T. Russo-Tait, L.M. Schultheis, L. Sengupta, R. Small, G.S. Spicer, J.H. Stillman, A. Swei, J.M. Wade, S.B. Waters, S.L. Weinstein, J.K. Willsie, D.W. Wright, C.D. Harrison, L.A. Kelley, G. Trujillo, C.R. Domingo, J.N. Schinske, K.D. Tanner. 2017. Classroom Sound Can Be Used To Classify Teaching Practices in College Science Courses. Proc. Natl Acad. Sci USA. Mar. 21; 114(12): 30853090. (PMID: 28265087) H. Kver*, S.E. Wirt*, M.T. Owens *, A.J. Dosmann*. 2014. Thinking like a Neuroscientist: using scaffolded grant proposals to foster scientific thinking in a freshman neuroscience course. The Journal of Undergraduate Neuroscience Education (JUNE), Fall, 13(1):A29-A40. (PMID: 25565917) Reviews in Biology Education M.T. Owens , K.D. Tanner. Approaches to Biology Teaching and Learning: Teaching as Brain Changing: Exploring Connections between Neuroscience and Innovative Teaching. 2017. CBE: Life Sciences Education, Jun 1; 16(2):fe2. (PMID: 28450442) Research in Neuroscience M.T. Owens , D.A. Feldheim, M.P. Stryker, J.W. Triplett. 2015. Stochastic interaction between neural activity and molecular cues in the formation of topographic maps. Neuron. Sept 23; 87(6):1261-73. (PMID: 26402608) J.W. Triplett, M.T. Owens , J. Yamada, G. Lemke, J. Cang, M.P. Stryker, D.A. Feldheim. 2009. Retinal input instructs alignment of visual topographic maps. Cell. Oct 2; 139(1):175-85. (PMID: 19804762) \n",
            "-------------------- Scraping faculty url 125/183 --------------------\n",
            "satchidananda panda \n",
            "-------------------- Scraping faculty url 126/183 --------------------\n",
            "amy pasquinelli  Research The recent discovery of microRNAs (miRNAs) has revolutionized our understanding of gene control. Genetic studies in the nematode Caenorhabditis elegans (Figure 1) revealed the first members of what we now recognize as an extensive family of regulatory RNAs that exist in most multicellular organisms. Already there is evidence that specific miRNAs play key roles in controlling development, stem cell fates and neuronal differentiation, and mutations in human miRNA genes have been linked to oncogenic and other disease states. The Pasquinelli lab couples C. elegans genetics with molecular and biochemical techniques to understand the basic mechanisms of miRNA expression and function and to elucidate the biological roles of specific miRNAs in cellular differentiation programs. How is the expression of miRNAs regulated? MiRNA genes typically encode long primary transcripts (pri-miRNAs) that undergo multiple processing steps to generate the mature ~22 nucleotide miRNA (Figure 2). Many miRNA genes are expressed at precise times in development and in specific tissues. To understand how these temporal and spatial expression patterns are achieved, we study the transcriptional and processing events that cooperate to produce specific miRNAs at the right time and in the right place. How do miRNAs regulate gene expression? The miRNAs regulate specific genes by partially base-pairing to complementary sequences in the messenger RNAs (mRNAs) of protein-coding genes (Figure 2). The human genome contains over 700 different miRNA genes, each of which may directly regulate hundreds of protein coding genes. To help elucidate how miRNAs find and regulate targets with limited sequence complementarity, we have focused on specific miRNA genes and have performed genome wide analyses to identify potential targets. Regulation by miRNAs can result in degradation or translational repression of the target mRNA (Figure 2), but the molecular mechanisms behind these inhibitory strategies are not entirely understood. By studying defined miRNA and target pathways in C. elegans, my lab aims to elucidate how miRNAs control gene expression in the endogenous context. What is the biological function of miRNA regulatory pathways? Some miRNA genes, like let-7, are essential for normal development (Figure 1). The let-7 miRNA and its temporally regulated expression pattern are widely conserved across animal phylogeny and misexpression of this miRNA has been linked to cancer in humans. A goal of our studies on the worm let-7 gene is to understand the broad role let-7 plays in cellular differentiation events across species.  Select Publications Pasquinelli AE. A rADAR defense against RNAi. Genes Dev, 2018 Feb 1;32(3-4):199-201. Aalto AP, Nicastro IA, Broughton JP, Chipman LB, Schreiner WP, Chen JS, Pasquinelli AE. Opposing roles of microRNA Argonautes during Caenorhabditis elegans aging. PLoS Genet. 2018 Jun 21;14(6):e1007379. doi: 10.1371/journal.pgen.1007379. eCollection 2018 Jun. Broughton JP, Pasquinelli AE. Detection of microRNA-Target Interactions by Chimera PCR (ChimP). Methods Mol Biol. 2018;1823:153-165. doi: 10.1007/978-1-4939-8624-8_12. Azoubel Lima S, Chipman LB, Nicholson AL, Chen YH, Yee BA, Yeo GW, Coller J, Pasquinelli AE. Short Poly(A) Tails are a Conserved Feature of Highly Expressed Genes, Nat Struct Mol Biol, 2017 Dec;24(12):1057-1063. doi: 10.1038/nsmb.3499. Epub 2017 Nov 6. Schreiner WP and Pasquinelli AE. Making and Maintaining MicroRNAs in Animals, Chapter for Essentials of microRNAs in neurogenesis: ontogenesis and plasticity of the mammalian neural system 1st Ed. De Pietri Tonelli, Elsevier, 2017. Pasquinelli AE. A sense-able microRNA. Genes Dev, 2016 Sep 15;30(18):2019-2020. Broughton JP, Lovci MT, Huang JL, Yeo GW, Pasquinelli AE. Pairing Beyond the Seed Supports MicroRNA Targeting Specificity. Mol Cell, 2016 Oct 20:64:1-14. Broughton JP, Pasquinelli AE. A tale of two sequences: microRNA-target chimeric reads. Genet Sel Evol. 2016 Apr 4;48:31. doi: 10.1186/s12711-016-0209-x. Review. Van Wynsberghe PM, Pasquinelli AE. Period Homolog LIN-42 regulates miRNA transcription to impact developmental timing. Worm, 2014 3(4)e974453. Mondol V, Ahn BC, Pasquinelli AE. Splicing remodels the let-7 primary microRNA to facilitate Drosha processing in Caenorhabditis elegans. RNA, 2015 21(8):1396-1403 Pasquinelli AE. MicroRNAs: heralds of the non-coding RNA revolution. RNA, 2015. 21(4):709-710. Mondol V, Ahn BC, Pasquinelli AE. Splicing remodels the let-7 primary microRNA to facilitate Drosha processing in Caenorhabditis elegans. RNA, 2015 21(8):1396-1403. Van Wynsberghe PM, Finnegan EF, Stark T, Angelus EP, Homan KE, Yeo GW, Pasquinelli AE. The Period protein homolog LIN-42 negatively regulates microRNA biogenesis in C. elegans. Dev Biol. 2014 Jun 15;390(2):126-35. Fonslow BR, Moresco JJ, Tu PG, Aalto AP, Pasquinelli AE, Dillin AG, Yates Iii JR. Mass spectrometry-based shotgun proteomic analysis of C. elegans protein complexes. WormBook. 2014 Jun 24:1-18. doi: 10.1895/wormbook.1.171.1. Lima SA, Pasquinelli AE. Identification of miRNAs and their targets in C. elegans. Adv Exp Med Biol. 2014; 825:431-50. Van Wynsberghe PM, Pasquinelli AE. Period Homolog LIN-42 regulates miRNA transcription to impact developmental timing. Worm, 2014 3(4)e974453. Finnegan EF, Pasquinelli AE. MicroRNA biogenesis: Regulating the regulators. Critical Reviews In Biochemistry & Molecular Biology, 2013 Jan-Feb;48(1):51-68. PMC3557704 Kai, ZS, Finnegan EF, Huang S, Pasquinelli AE. Multiple cis-elements and trans-acting factors regulate dynamic spatio-temporal transcription of let-7 in Caenorhabditis elegans. Dev Biology, 2013 Feb 1;374(1):223-33. PMC3548979 Massirer KB, Pasquinelli AE. MicroRNAs that interfere with RNAi. Worm, 2013 Jan/Feb/Mar;2(1):e1-6. Hunter SE, Finnegan EF, Zisoulis DG, Lovci MT, Melnik-Martinez KV, Yeo GW, Pasquinelli AE. Functional genomic analysis of the let-7 regulatory network in Caenorhabditis elegans. PLoS Genetics, 2013 Mar;9(3)e1003353. PMC3597506 Broughton JP, Pasquinelli AE. Identifying Argonaute binding sites in Caenorhabditis elegans using iCLIP. Methods, 2013 Apr 10. S1046-2023(13)00102-3. Pasquinelli AE. The primary target of let-7 miRNA. Biochem. Soc. Trans. 2013 Aug 1;41(4):821-4. Pasquinelli AE. A team effort blocks the ribosome in its tracks. Nat Struct Mol Biol. 2012 19(2):133-4. Mondol V, Pasquinelli AE. Lets make it happen: The role of let-7 microRNA in development. In Eran Hornstein, editors: MicroRNAs in Development, CTDB, UK: Academic Press, 2012, 99:1-30. Massirer KB, Perez SG, Mondol V, Pasquinelli AE. The miR-35-41 family of microRNAs regulates RNAi sensitivity in Caenorhabditis elegans. PLoS Genetics. 2012 Mar;8(3):e1002536. Pasquinelli AE. MicroRNAs and their targets: recognition, regulation and an emerging reciprocal relationship. Nat Rev Genet. 2012 Mar 13;13(4):271-82. Aalto AP, Pasquinelli AE. Small non-coding RNAs mount a silent revolution in gene expression. Current Opinion in Cell Biology, 2012 Jun;24(3):333-40. Zisoulis DG, Kai ZS, Chang RC, Pasquinelli AE. Auto-regulation of miRNA biogenesis by let-7 and Argonaute. Nature, 2012 Jun 28;486(7404):541-4. Pasquinelli AE. MicroRNAs and their targets: recognition, regulation and an emerging reciprocal relationship. Nat Rev Genet. 2012 Mar 13;13(4):271-82. Aalto AP, Pasquinelli AE. Small non-coding RNAs mount a silent revolution in gene expression. Current Opinion in Cell Biology, 2012 Jun;24(3):333-40. Zisoulis DG, Kai ZS, Chang RC, Pasquinelli AE. Auto-regulation of miRNA biogenesis by let-7 and Argonaute. Nature, 2012 Jun 28;486(7404):541-4. Pasquinelli AE. Birthing histone mRNAs by CSR-1-section. Embo J, 2012 Oct 3;31(19):3790-1. Van Wynsberghe PM, Kai ZS, Massirer KB, Burton VH, Yeo GW, Pasquinelli AE. Co-transcriptional association of LIN-28 with let-7 primary transcripts regulates miRNA maturation in C. elegans. Nat Struct Mol Biol. 2011 Mar;18(3):302-8. Zisoulis DG, Yeo GW, Pasquinelli AE. Comprehensive identification of miRNA target sites in live animals. Methods Mol Biol. 2011 732:169-85. Van Wynsberghe PM, Chan S-P, Slack FJ*, Pasquinelli AE*. Analysis of microRNA Expression and Function. Methods Cell Biol. 2011;106:219-52. PMID: 22118279 Hinton A, Afrikanova I, Wilson M, King C, Maurer B, Yeo G, *Hayek A, *Pasquinelli A. A Distinct MicroRNA Signature for Definitive Endoderm Derived From Human Embryonic Stem Cells. Stem Cells Dev. 2010 19:797-807. Zisoulis DG, Lovci MT, Wilbert ML, Hutt KR, Liang TY, *Pasquinelli AE, *Yeo GW. Comprehensive discovery of endogenous Argonaute binding sites in Caenorhabditis elegans. Nat Struct Mol Biol. 2010 17:173-9. Kai ZS, Pasquinelli AE. MicroRNA assassins: factors that regulate the disappearance of miRNAs. Nat Struct Mol Biol. 2010 Jan;17(1):5-10. Review. Kai Z, Pasquinelli AE. A genome wide view of hunchback-like-1 targets. Cell Cycle. 2010 Jan 15;9(2):230-1. Godshalk SE, Melnik-Martinez KV, Pasquinelli AE*, Slack FJ*. MicroRNAs and cancer: a meeting summary of the eponymous Keystone Conference. Epigenetics. 2010 Feb;5(2):164-8. Pasquinelli AE. Molecular biology. Paring miRNAs through pairing. Science. 2010 Jun 18;328(5985):1494-5. Bracht JR, Van Wynsberghe PM, Mondol V, Pasquinelli AE. Regulation of lin-4 miRNA Expression, Organismal Growth and Development by a Conserved RNA Binding Protein in C. elegans. Dev Biol. 2010 Dec 15;348(2):201-21. Holtz J, Pasquinelli AE. Uncoupling of lin-14 mRNA and protein repression by nutrient deprivation in Caenorhabditis elegans. RNA. 2009 15:400-5. Chendrimada TP, Finn KJ, Ji X, Baillant D, Gregory RI, Liebhaber SA, *Pasquinelli AE, *Shiekhattar R. MicroRNA silencing through RISC recruitment of eIF6. Nature 2007 447:823-828. Norden-Krichmar TM, Holtz J, Pasquinelli AE, Gaasterland T. Computational prediction and experimental validation of Ciona intestinalis microRNA genes. BMC Genomics 2007 8:445. Bagga S, Pasquinelli AE. Identification and analysis of microRNAs. Genet Eng (N Y) 2006 27:1-20. Pasquinelli AE. Demystifying Small RNA Pathways - A Meeting Review. Dev Cell 2006 10:419-424. Massirer K, Pasquinelli AE. The evolving role of microRNAs in animal gene expression. BioEssays 2006 28:449-452. Pasquinelli AE. MicroRNAs: a Small Contribution from Worms. In RNA Interference Technology: From Basic Science to Drug Development, ed. K Appasani. Cambridge, UK: Cambridge University Press. 2005. p69-83. Pasquinelli AE, Hunter SE, Bracht J. MicroRNAs: A Developing Story. Curr. Opinion in Cell and Devel. Biol. 2005 15:200-205. Bagga S, Bracht J, Hunter S, Massirer K, Holtz J, Eachus R, Pasquinelli AE. Regulation by let-7 and lin-4 miRNAs results in target mRNA degradation. Cell. 2005 122:553-563. Bracht J, Hunter S, Eachus R, Weeks P, Pasquinelli AE. Trans-splicing and polyadenylation of let-7 microRNA primary transcripts. RNA. 2004 10:1586-1594. *Co-corresponding authors \n",
            "-------------------- Scraping faculty url 127/183 --------------------\n",
            "gentry patrick  Biography Dr. Patrick received his Ph.D. from Harvard University in 1999 after working in the laboratory of Dr. Li-Huei Tsai. He was a Damon Runyon Cancer Research Foundation and a United Negro College Fund/Merck postdoctoral fellow with Dr. Erin Schuman at the California Institute of Technology. Dr. Patrick joined the faculty in 2004.  Research Our laboratory is interested in how synaptic activity modulates the molecular make-up of synaptic connections in the mammalian central nervous system (CNS), which in many cases leads to long-lasting changes in synaptic efficacy. The concerted regulation of protein synthesis and degradation is fundamental for the control of diverse cellular events. Many studies have provided evidence that new protein synthesis likely takes place at synapses and is required for plasticity. Protein degradation, on the other hand, provides another way to regulate protein levels. In fact the ability to dynamically control protein levels allows for very tight control of rapid signaling cascades. We study the ubiquitin-proteasome system (UPS), one of the major cellular pathways controlling protein turnover in mammalian cells. The UPS is a complex proteolytic pathway whereby proteins are targeted to the 26S proteasome for degradation. Ubiquitin is covalently attached to a target protein through a series of steps: first an E1 ubiquitin activating enzymes pass ubiquitin to E2 transferase and E3 ligases. At this point, many times in concert with the help of an E2 enzyme, the E3 ligase binds and modifies the target protein with the ubiquitin. Multiple ubiquitin molecules are added and the protein is recognized and degraded by the 26S proteasome. Many cellular roles have been defined for the UPS such as cell cycle control, cell fate and growth determination, antigen presentation, and many cell signaling pathways. In contrast the mechanisms of how the UPS regulates the growth and development, maintenance, and remodeling of synaptic connection in the mammalian central nervous system (CNS) is less understood. An interesting problem is how activated synapses of a single neuron become selectively modified as a result of synaptic plasticity. It is known, for example, that synaptic modifications can occur selectively at one group of synapses, but not at another group of synapses on the same neuron. This property is known as \"input-\" or \"synapse specificity\". It is plausible that the selective degradation of proteins that restrict or limit plasticity may be required for these synaptic changes to occur. Alternatively, various proteolytic activities may provide specificity for long-term synaptic changes. This could be accomplished through the degradation of some proteins at specific locations or by targeting regulatory components of a proteolytic pathway to modified or unmodified sites. We study the UPSs role in synaptic plasticity. One approach we take is to use genetically encoded fluorescent-based proteasome (or degradation) reporters and time-lapse confocal microscopy to assay how neuronal activity modulates the rate of degradation within the spines and dendrites of neurons. Using these techniques, we hope to understand how and why the UPS is activated at or recruited to synapses in response to neuronal activity. We are also interested in identifying the protein targets of the UPS at synapses. In particular, we study how the UPS regulates alpha-amino-3-hydroxy-5-methyl-4-isoxaolepropionic acid (AMPA) type ionotropic glutamate receptor trafficking in neurons. The removal and insertion of AMPA receptors at glutamatergic synapses is one mechanism to either decrease or increase synaptic strength. We and others have found that the activity of the UPS is required for agonist-mediated endocytosis of AMPA receptors. The simple model is that protein degradation via the UPS is required for normal AMPA receptor trafficking in neurons. Using molecular and biochemical approaches, we hope to identify the components of the UPS and protein targets involved AMPA receptor trafficking. We are also interested in neurodegenerative disease. In humans, many neurodegenerative diseases are characterized post-mortem by anatomical hallmarks visible by standard light microscopy. Alzheimers and Lewy body disease are the most common causes of dementia in elderly populations. These diseases are characterized by neurofibrillary tangles (NFT), plaques, and lewy bodies (large dense inclusions). Familial Parkinsons disease is also characterized by Lewy bodies. In addition, large dense inclusions are characteristic of many of the CAG repeat diseases such as Huntington and ALS. Biochemical and immunohistochemical characterization of these inclusions, tangles, and neuritic plaques has indicated that they contain high concentrations of ubiquitin and ubiquitinated proteins. Does the ubiquitin-proteasome pathway contribute to these abnormalities and to neuronal death? We hope that the understanding of UPS in normal neuronal function will help elucidate how it may be involved in neuronal dysfunction and disease.  Select Publications Kyriakakis P, Catanho M, Hoffner N, Thavarajah W, Hu VJ, Chao SS, Hsu A, Pham V, Naghavian L, Dozier LE, Patrick GN, Coleman TP. \"Biosynthesis of Orthogonal Molecules Using Ferredoxin and Ferredoxin-NADP(+) Reductase Systems Enables Genetically Encoded PhyB Optogenetics.\" ACS Synth Biol. 2018 Feb 16;7(2):706-717. Tong W, Dwyer CA, Thacker BE, Glass CA, Brown JR, Hamill K, Moremen KW, Sarrazin S, Gordts PLSM, Dozier LE, Patrick GN, Tor Y, Esko JD. \"Guanidinylated Neomycin Conjugation Enhances Intranasal Enzyme Replacement in the Brain\". Mol Ther. 2017 Dec 6;25(12):2743-2752. Gonzales FR, Howell KK, Dozier LE, Anagnostaras SG, Patrick GN. Proteasome phosphorylation regulates cocaine-induced sensitization. Mol Cell Neurosci. 2017 Dec 5;88:62-69. Goo, M.S., Sancho, L., Slepak, N.; Boassa, D.; Deerinck, T.J., Ellisman, M.H., Bloodgood, B.L., and Patrick, G.N.; \"Activity-dependent trafficking of lysosomes in dendritic shaft and spine\", J Cell Biol. 2017 Aug 7;216(8):2499-2513. (cover mention and top 10 neuroscience articles in JCB in 2017) Marquez-Lona EM, Torres-Machorro AL, Gonzales FR, Pillus L, Patrick GN. \"Phosphorylation of the 19S regulatory particle ATPase subunit, Rpt6, modifies susceptibility to proteotoxic stress and protein aggregation.\" PLoS One. 2017 Jun 29;12(6):e0179893. Dwyer, C.A, Scudder, S.L., Lin, Y., Dozier, L.A., Phan; D., Allen, N. J., Patrick, G.N. and Jeffrey D., J.D. Neurodevelopmental Changes in Excitatory Synaptic Structure and Function in the Cerebral Cortex of Sanfilippo Syndrome IIIA Mice Sci Rep. 2017 Apr 18;7:46576. Cifelli JL, Dozier L, Chung T, Patrick GN, Yang J. Benzothiazole Amphiphiles Promote the Formation of Dendritic Spines in Primary Hippocampal Neurons.J Biol Chem. (2016) Jun 3;291(23):11981-92 (cover article). Rodrigues EM, Scudder SL, Goo MS, Patrick GN. A-Induced Synaptic Alterations Require the E3 Ubiquitin Ligase Nedd4-1. J Neurosci. (2016) Feb 3;36(5):1590-5 (cover article). Scudder SL, Patrick GN. Synaptic structure and function are altered by the neddylation inhibitor MLN4924. Mol Cell Neurosci. (2015) Mar;65:52-7. Goo MS, Scudder SL, Patrick GN. Ubiquitin-dependent trafficking and turnover of ionotropic glutamate receptors. Front Mol Neurosci. (2015) Oct 16;8:60. Scudder SL, Goo MS, Cartier AE, Molteni A, Schwarz LA, Wright R, Patrick GN. Synaptic strength is bidirectionally controlled by opposing activity-dependent regulation of Nedd4-1 and USP8. J Neurosci. (2014) Dec 10;34(50) (cover article). Hamilton, A.M, Oh, W.C., Vega-Ramirez, H, Stein, I.S., Hell, J.W., Patrick, G.N., Zito, K. Activity-dependent growth of new dendritic spines is regulated by the proteasome (2012) Neuron Jun 21;74(6):1023-30 Anna E. Cartier, A.E., Ubhi, K., Spencer, B., Vazquez-Roque, R.A., Kosberg, K.A., Fourgeaud, L., Kanayson, P., Patrick, C., Rockenstein, E., Patrick, G.N., Masliah, E. Differential Effects of UCHL1 Modulation on Alpha-Synuclein in PD-Like Models of Alpha-Synucleinopathy (2012) PLoS One April 13: 7(4): e34713. Djakovic, S.N., Marquez-Lona, E.M., Jakawich, S.K., Chu, C., Sutton, M.S., Patrick, G.N. Phosphorylation of Rpt6 regulates proteasome function and synaptic strength (2012) J. Neuroscience Apr 11;32(15):5126-31. Schwarz, L.A., and Patrick, G.N. \" Ubiquitin-dependent endocytosis, trafficking and turnover of neuronal membrane protein. (2011) Mol Cell Neurosci 2012 Mar;49(3):387-93. Epub 2011 Aug 22. Schwarz, L.A., Hall, B.J., and Patrick, G.N. \"Activity-Dependent Ubiquitination of GluA1 Mediates a Distinct AMPAR Endocytosis and Sorting Pathway\". (2010) J. Neuroscience 30(49):16718-29. Keil, J, Shen, Zhouxin, Briggs, S, and Patrick, G.N. Regulation of STIM1 and SOCE by the ubiquitin-proteasome system (UPS)\" (2010) PLoS One Oct 18;5(10):e13465. Jakawich S, Nealy R, Djakovic S, Patrick G.N., Sutton M. The Ubiquitin Proteasome System mediates slow homeostatic changes in synaptic strength. (2010) Neuroscience 29;171(4):1016-31 Djakovic, S.N., Schwarz, L.A., DeMartino, G.N. and Patrick, G.N., Regulation of the Proteasome by Neuronal Activity and CaMKII (2009) J. Biological Chemistry 284(39):26655-65. Cartier, A.E., Djakovic, S.N., Wilson, S. and Patrick, G.N. Regulation of synapse structure by the ubiquitin c-terminal hydrolase UCH-L1. (2009) J. Neuroscience 29(24):7857-7868 Kerjan G, Han E, Dube C, Djakovic S, Patrick G.N., Baram T, Heinemann S, Gleeson J. Mice lacking doublecortin and doublecortin-like kinase 2 display altered hippocampal neuronal maturation and spontaneous seizures.(2009) Proc Natl Acad Sci U S A. Apr 21;106(16):6766-71. Epub 2009 Apr 2. Patrick G.N. Synapse formation and plasticity: recent insights from the perspective of the ubiquitin proteasome system. (2006) Current opinion in neurobiology Feb;16(1):90-4 ; Jan 18; [Epub ahead of print] Patrick, G.N., Bingol, B., Weld, H., Schuman, E.M., Ubiquitin-mediated proteasome activity is required for agonist-induced endocytosis of GluRs. (2003) Curr Biol. 2003 Dec 2;13(23):2073-81 Zukerberg, L.R.*, Patrick, G.N., Nikolic, M., Humbert, S., Lanier, L.M., Gertler, F.B., Vidal, M., Van Etten, R.A. & Tsai, L.-H-. Cables links Cdk5 and c-Abl, Facilitating Cdk5 Tyrosine Phosphorylation and Stimulation of Kinase Activity. (2000) Neuron Jun;26(3):633-46. Patrick, G.N., Zukerberg, L., Nikolic, M., Dikkes, P., de la Monte, S. & Tsai, L.-H. Conversion of p35 to p25 deregulates cdk5 activity and promotes neurodegeneration. (1999) Nature v.402 pp 615-622 Huang, D., Patrick, G.N., Moffat, J., Tsai, L.-H., & Andrews, B. Mammalian Cdk5 is a functional homologue of the budding yeast Pho85 cyclin-dependent protein kinase. (1999) PNAS v.96 No.25 pp. 14445-14450 Patrick, G.N., Zhou, P., Kwon, Y.T., Howley, P.M. & Tsai, L.-H. p35, the neuronal specific activator of cdk5, is degraded by the ubiquitin-proteasome pathway. (1998) J. Biol. Chem v. 273 N0. 37, pp. 24057-24064 \n",
            "-------------------- Scraping faculty url 128/183 --------------------\n",
            "gulcin pekkurnaz  Research The goal of our laboratory is to understand how mitochondrial function is coupled to neuronal metabolism. Mitochondria provide 95% of the energy utilized by the brain. In addition to energy production in the form of ATP, mitochondria carry out many other vital functions such as calcium buffering and neurotransmitter metabolism; therefore, their dysfunction underlies many neurological disease states. Although all neurons use ATP, metabolic requirements vary greatly among brain regions, classes of neurons, and even neuronal sub-compartments. We are interested in understanding the interplay between metabolism and mitochondrial function across distinct cell classes in the nervous system under both physiological and pathological conditions. Due to the nature of our interdisciplinary research, we utilize a wide array of approaches combining molecular biology, cell culture, metabolic flux analysis, mouse genetics, neurodegenerative disease models, biochemistry, proteomics, live-cell, and super-resolution microscopy to reveal novel pathways.  Select Publications Pekkurnaz G, Trinidad JC, Wang X, Kong D, Schwarz TL. Glucose regulates mitochondrial motility via Milton modification by O-GlcNAc transferase. Cell. 2014 Jul 3;158(1):54-68. Teodoro RO, Pekkurnaz G, Nasser A, Higashi-Kovtun ME, Balakireva M, McLachlan IG, Camonis J, Schwarz TL. Ral mediates activity-dependent growth of postsynaptic membranes via recruitment of the exocyst. EMBO J. 2013 Jul 17;32(14):2039-55. Pekkurnaz G, Fera A, Zimmerberg-Helms J, Degiorgis JA, Bezrukov L, Blank PS, Mazar J, Reese TS, Zimmerberg J. Isolation and ultrastructural characterization of squid synaptic vesicles. Biol Bull. 2011 Apr;220(2):89-96. \n",
            "-------------------- Scraping faculty url 129/183 --------------------\n",
            "michael perry  Biography Michael Perry got his start in biology while an undergraduate and M.S. student at the University of Florida, working at the Florida Museum of Natural History, where he collected butterflies in the field at sites that ranged from the Rocky Mountains to the cloud forests of Costa Rica. To begin to understand the molecular origins of the diversity he encountered in the field he shifted his focus to developmental biology and gene regulation during his Ph.D. research at UC Berkeley with Mike Levine and Nipam Patel, where he was supported by an NSF-GRFP fellowship. He was able to combine his diverse interests in evolution and developmental biology as a postdoctoral fellow with Claude Desplan at New York University. Michael was the recipient of NIH Ruth L. Kirschstein and Charles H. Revson Foundation fellowships as well as a K99/R00 Pathway to Independence Award from the NIH National Eye Institute. He will join the Division of Biological Sciences faculty at UC San Diego in March 2019.  Research Stochastic gene expression in developing photoreceptors One of the most interesting questions in biology is how the complexity of the nervous system is encoded in the genome: how it provides rules to build a system that can compute, remember, and produce behavioral responses. To understand which genetic changes shape neural function we leverage naturally occurring variation in nervous systems within and across species. The lab uses the insect visual system as a model, from Drosophila and other flies all the way to butterflies and wasps. Insect eyes and brains have been shaped by a rich diversity of functional requirements to exquisitely tune the nervous system and behavioral responses to allow them to thrive in an incredible range of environments. A flower fly showing stochastic patterns on the retina Image by Yousef Al Habshi Natural diversity of the visual system comes in large part from changes in how and when genes are expressed during development. The advent of powerful genomic tools such as CRISPR/Cas9 and single cell sequencing have radically changed the questions we can ask in non-model systems: we can now perform targeted comparisons and functional molecular experiments in almost any species. There has never been a better time to combine approaches in model systems with experiments directly in species that exhibit interesting differences in form, function, and behavior as a targeted means of unraveling neural complexity. How are the full diversity of neural types specified, and how do new neural types evolve and plug in to existing circuity? How do such changes modify downstream neural function? Does the interconnected nature of the nervous system influence how it evolves? The long-term goal of the lab is to understand the genetic basis of nervous system development and evolution at levels ranging from molecules to behavior. CRISPR in butterflies: mosaic yellow mutant Papilio xuthus and Vanessa cardui  Select Publications Yan, H., Opachaloemphan, C., Mancini, G., Yang, H., Gallitto, M., Mlejnek, J., Leibholz, A., Haight, K., Ghaninia, M., Huo, L., Perry, M.W., Slone, J., Zhou, X., Traficante, M., Penick, C., Dolezal, D., Gokhale, D., Stevens, D., Fetter-Pruneda, I., Bonasio, R., Zwibel, L.J., Berger, S., Liebig, J., Reinberg, D., and Desplan, C. 2017. An engineered orco mutation produces aberrant social behavior and defective neural development in ants. Cell, 170: 736-747. PMID: 28802043 Perry, M.W., Konstantinides, N., Pinto-Teixeira, F., and Desplan, C. 2017. Generation and evolution of neural cell types and circuits: insights from the Drosophila visual system. Annual Reviews of Genetics, 51: 501-527. PMID: 28961025 Zhang, L., Martin, A., Perry, M.W., Burg, K., Matsuoka, Y., Monteiro, A., and R. Reed. 2017. Genetic basis of melanin pigmentation in butterfly wings. Genetics, 205:1537-1550. PMID: 28193726 Perry, M.W., Kinoshita, M., Saldi, G., Huo, L., Arikawa, K., and C. Desplan. 2016. Molecular logic behind the three-way stochastic choices that expand butterfly color vision. Nature, 535: 280-284. PMID: 27383790 Perry, M.W., and C. Desplan. 2016. Quick Guide: Love spots. Current Biology, 26: R1-R2. PMID: 27326705 Bothma, J.P., Garcia, H.G., Ng, S., Perry, M.W., Gregor, T., and M. Levine 2015. Enhancer additivity and non-additivity are determined by enhancer strength in the Drosophila embryo. Elife, 4: e07956. PMID: 26267217 Wernet, M.F, Perry, M.W., and C. Desplan. 2015. The evolutionary diversity of insect retinal mosaics: common design principles and emerging molecular logic. Trends Genetics, 31(6): 316-28. PMID: 26025917 Perry, M.W., Bothma, J., Luu, R.L., and M. Levine. 2012. Precision of Hunchback expression in the Drosophila embryo. Current Biology, 22: 2247-2252. PMID: 23122844 Hilgers, V., Perry, M.W., Hendrix, D., Stark, A., Levine, M., and B. Haley. 2011. Neural-specific elongation of 3 UTRs during Drosophila development. PNAS, 108: 15864-9. PMID: 21896737 Perry, M.W., Boettiger, A., Levine, M. 2011. Shadow enhancers ensure precision of gap gene expression patterns in the Drosophila embryo. PNAS, 108: 13570-5. PMID: 21825127 Perry, M.W.*, Boettiger, A.*, Bothma, J. and Levine, M. 2010. Shadow enhancers foster robustness of Drosophila gastrulation. Current Biology, 20: 1562-1567. PMID: 20797865 Perry, M.W., Boettiger, A., Cande, J. and M. Levine. 2009. Evolution of Insect Dorsoventral Patterning Mechanisms. Cold Spring Harb Symp on Quant Biol, 74: 275-279. PMID: 19843594 Parchem, R.J., Perry, M.W., and N.H. Patel. 2007. Patterns on the insect wing. Current Opinion in Genetics and Development, 17: 300-308. PMID: 17627807 \n",
            "-------------------- Scraping faculty url 130/183 --------------------\n",
            "katherine petrie  Biography Katie Petrie received her Ph.D. in Biology from The Scripps Research Institute in 2014, where she studied the processes that drive evolution by combining in vitro molecular model systems with high throughput sequencing. She completed her postdoctoral work as an EON Research Fellow, splitting her time between UC San Diego and the EarthLife Science Institute (ELSI) in Tokyo, Japan. She joined UC San Diego as an Assistant Teaching Professor in 2018.  Research Education As an assistant teaching professor, I bring bioinformatics and genomicskey tools for doing biology in the 21st centuryinto the undergraduate curriculum at UC San Diego. I am interested in how enabling students to use this technology impacts their learning and careers. In my teaching efforts, I use proven active learning strategies to maximize student engagement. I am interested in developing new teaching strategies for building scientific literacy, and on expanding access to STEM fields by incorporating research with primary data into the classroom experience. As part of this effort, I am incorporating CURES (Course-Based Undergraduate Research Experiences) into the courses I teach. These gives students the opportunity to make a genuine contribution to scientific knowledge during their laboratory classes, and have the potential to increase inclusivity and retention in STEM careers. Biological As an evolutionary biologist, I am interested in understanding the driving forces for evolution, including the mechanisms underlying evolutionary innovations. In collaboration with members of Justin Meyers lab here at UCSD, I use genomic and laboratory approaches to better understand how evolution can lead to new traits and functions. Recently, weve discovered a multi-tasking protein that enables an important evolutionary transition: the ability of a virus, bacteriophage , to infect a new host . In collaboration with researchers at the EarthLife Science Institute in Tokyo, Japan, I am also studying how selective interactions between nucleic acids and minerals might have contributed to pre-Darwinian enrichment processes on the early earth prior to the origin of life.  Select Publications Rohan Maddamsetti, Daniel T. Johnson, Stephanie J. Spielman, Katherine L. Petrie, Debora S. Marks, Justin R. Meyer. Gain-of-function experiments in bacteriophage lambda uncover residues under diversifying selection in nature. Evolution in press Katherine L. Petrie, Nathan D. Palmer, Daniel T. Johnson, Sarah J. Medina, Stephanie J. Yan, Victor Li, Alita R. Burmeister, Justin R. Meyer. Destabilizing mutations encode nongenetic variation that drives evolutionary innovation. Science 359, 1542-1545 (2018) Katherine L. Petrie, Gerald F. Joyce. Limits of neutral drift: Lessons from the in vitro evolution of two ribozymes. Journal of Molecular Evolution 79, 75-90 (2014) Katherine L. Petrie, Gerald F. Joyce. Deep sequencing analysis of mutations resulting from the incorporation of dNTP analogs. Nucleic Acids Research 38, 8095-8104 (2010) \n",
            "-------------------- Scraping faculty url 131/183 --------------------\n",
            "samuel pfaff \n",
            "-------------------- Scraping faculty url 132/183 --------------------\n",
            "lorraine pillus  Biography Lorraine Pillus carried out graduate studies with Frank Solomon at MIT. Her postdoctoral research with Jasper Rine at the University of California, Berkeley was supported by a John D. and Catherine T. MacArthur Foundation Fellowship of the Life Sciences Research Foundation. Prof. Pillus has been named a Pew Scholar and a National Science Foundation New Young Investigator and has received teaching excellence awards from the Mortar Board Honor Society and the Associated Students of the Skaggs School of Pharmacy & Pharmaceutical Sciences. She served as Chair of Molecular Biology from 2012-2016 and is currently Associate Dean of the Division. She was elected as a Fellow of the American Association for the Advancement of Science in 2017 and to the American Academy of Microbiology in 2018.  Research A gene's chromosomal position and local chromatin structure can have profound consequences for its expression. We are interested in all aspects of these chromosomal and chromatin effects because they underlie epigenetic processes that are critical for normal growth and development. We are particularly focused on the form of chromatin regulation known as transcriptional silencing and study its effects on yeast and human genes. Chromatin is controlled by post-translational modification of its core subunits, the histones, and other transcriptional factors. The enzymes catalyzing these modifications include acetyltransferases, (HATs), deacetylases (HDACs), methyltransferases (MTs), kinases, and others. The enzymes work together to promote dynamic patterns of modification that help to specify epigenetic processes. We study several classes of the chromatin modifying enzymes to understand the range of their in vivo functions, including mechanisms of their activity, genomic targeting, and interaction with transcriptional and DNA repair complexes. Our experimental approaches include genetic, genomic and biochemical strategies to understand the MYST family HATs, the SIR2 family HDACs, and several classes of evolutionarily conserved MTs.  Select Publications Searle NE and Pillus L. (2018) Critical genomic regulation mediated by Enhancer of Polycomb Current Genetics 64:147-154 PMID: 28884217 Li Y, Jin M, O'Laughlin R, Bittihn P, Tsimring LS, Pillus L, Hasty J, Hao N (2017) Multi-generational silencing dynamics control cell aging. Proc Natl Acad Sci, USA 114:11253-11258 PMID: 29073021 Mason AG, Garza RM, McCormick MA, Patel B, Kennedy BK, Pillus L, La Spada AR (2017)The replicative lifespan-extending deletion of SGF73 results in altered ribosomal gene expression in yeast. Aging Cell 16(4):785-796 PMID: 28568901 Marquez-Lona EM, Torres-Machorro AL, Gonzales FR, Pillus L, Patrick GN (2017) Phosphorylation of the 19S regulatory particle ATPase subunit, Rpt6, modifies susceptibility to proteotoxic stress and protein aggregation. PLoS One 12(6):e0179893 PMID: 28662109 Searle NE, Torres-Machorro AL, Pillus L. (2017) Chromatin Regulation by the NuA4 Acetyltransferase Complex Is Mediated by Essential Interactions Between Enhancer of Polycomb (Epl1) and Esa1. Highlighted in Genetics. 205:1125-1137. PMID: 28108589 Petty EL, Lafon A, Tomlinson SL, Mendelsohn BA, Pillus L. (2016) Promotion of Cell Viability and Histone Gene Expression by the Acetyltransferase Gcn5 and the Protein Phosphatase PP2A in Saccharomyces cerevisiae. Genetics 203:1693-707. PMID: 27317677 Su, XB and Pillus, L. (2016) Functions for diverse metabolic activities in heterochromatin. Proceedings of the National Academies of Sciences, USA. doi: 10.1073/pnas.1518707113. Torres-Machorro AL, Clark LG, Chang CS, Pillus L. (2015) The Set3 Complex Antagonizes the MYST Acetyltransferase Esa1 in the DNA Damage Response. doi: 10.1073/pnas.1518707113. Torres-Machorro AL, Aris JP, Pillus L. (2015) A moonlighting metabolic protein influences repair at DNA double-stranded breaks. Nucleic Acids Res. 2015 43:1646-1658. PMID: 25628362. Basnet H, Su XB, Tan Y, Meisenhelder J, Merkurjev D, Ohgi KA, Hunter T, *Pillus L, *Rosenfeld MG (2014) Tyrosine phosphorylation of histone H2A by CK2 regulates transcriptional elongation.Nature 516:267-71. PMID: 25252977 McCormick MA, Mason AG, Guyenet SJ, Dang W, Garza RM, Ting MK, Moller RM, Berger SL, Kaeberlein M, Pillus L, La Spada AR, Kennedy BK (2014) The SAGA histone deubiquitinase module controls yeast replicative lifespan via Sir2 interaction. Cell Reports 8:477-86. PMID: 25043177. Torres-Machorro AL, Pillus L. (2014) Bypassing the Requirement for an Essential MYST Acetyltransferase. Genetics 197:851-863. PMID: 24831819 Eustice M, Pillus L. (2014) Unexpected function of the glucanosyltransferase Gas1 in the DNA damage response linked to histone H3 acetyltransferases in Saccharomyces cerevisiae. Genetics 196:1029-39. PMID: 24532730 Petty E, Pillus L. (2013) Balancing chromatin remodeling and histone modifications in transcription. Trends in Genetics 29:621-9. PMID: 23870137 Chen, M., Licon, K., Otsuka, R., Pillus, L. and Ideker, T. (2013) Decoupling chromatin and genetic effects through systematic analysis of gene position. Cell Reports 3(1):128-37. PMID:23291096 Chang, C.S., Clarke, A.S. and Pillus, L. (2012) Suppression analysis of esa1 mutants in S. cerevisiae links NAB3 to transcriptional silencing and nucleolar functions G3: Genes, Genomes, Genetics 2(10):1223-32.PMID: 23050233 Lafon, A., Petty, E. and Pillus, L. (2012) Functional antagonism between Sas3 and Gcn5 acetyltransferases and ISWI chromatin remodelers PLoS Genetics 8(10):e1002994. PMID: 23055944. Garza, R. and Pillus, L. (2012) STUbLs in chromatin and genome stability. 99(2):146-54 PMID: 23175389 Scott, E.M. and Pilllus, L. (2010) Homocitrate synthase connects amino acid metabolism to chromatin functions through Esal and DNA damage.Genes & Development 24:1903-1913 PMID: 20810648. Bulfer, S.L., Scott, E.M., Pillus, L. and Trievel, R.C. (2010) Structural basis for feedback inhibition of homocitrate synthase by L-lysine. Journal of Biological Chemistry 285:10466-10453 PMID: 20089861. Bulfer SL, Scott EM, Couture JF, Pillus L, Trievel RC. (2009) Crystal structure and functional analysis of homocitrate synthase, an essential enzyme in lysine buisynthesis. Journal of Biological Chemistry284:35769-35780 PMID: 19776021. Jacobson, S. and Pillus, L. (2009) The SAGA subunit Ada2 functions in transcriptional silencing. Molecular and Cellular Biology29:6033-6045. Chang, C.S. and Pillus, L. (2009) Collaboration Between the Essential Esa1 acetyltransferase and the Rpd3 Deacetylase is Mediated by H4K12 Histone Acetylation in Saccharomyces cerevisiae. Genetics 183:149-160 PMID: 19596907. Koch, M.R. and Pillus, L. (2009) The Gas1 Glucanosyltransferase functions in transcriptional silencing. Proceedings of the National Academy of Sciences 106:11224-11229. PMID: 19541632. Mak, H.C., Pillus, L., and Ideker, T. (2009) Dynamic reprogramming of transcription factors to and from the subtelomere. Genome Research 9:1014-25 PMID: 19372386. Koch, M.R. and Pillus, L. (2009) Silent chromatin formation and regulation in the yeast Saccharomyces cerevisiae. Handbook of Cell Signaling, 2nd Edition, Oxford: Academic Press eds., R. Bradshaw and E. Dennis, pp. 2427-2436. Kennedy, E.J., Ghosh G., and Pillus, L. (2008) Identification of functionally distinct regions that mediate biological activity of the Protein Kinase A homolog TPK2. Journal of Biological Chemistry 283:1084-1093 PMID: 17971450. Lafon A, Chang CS, Scott EM, Jacobson SJ, and Pillus L. (2007) MYST opportunities for growth control: yeast genes illuminate human cancer gene functions. Oncogene. 26:5373-5384. PMID: 17694079. Darst, R.P., Garcia, S.N., Krick, M., and Pillus, L. (2008) Slx5p promotes transcriptional silencing and is required for robust growth in the absence of Sir2p. Molec. Cell. Biol. 28:1361-1372 PMID: 18086879. Pillus L (2008) MYSTs Mark Chromatin for Chromosomal Functions. Curr. Opin. Cell Biology 20:326-333 PMID: 18511253. \n",
            "-------------------- Scraping faculty url 133/183 --------------------\n",
            "joseph pogliano  Research Evolution of the microbial cytoskeleton Prokaryotic cells have a complex subcellular organization but how this organization is established and maintained remains poorly understood due to our limited knowledge of the proteins that make up the prokaryotic cytoskeleton. The goals of my lab are to identify and characterize the major families of dynamic cytoskeletal proteins that exist in bacteria, to determine the various functions that they perform, and to understand how these polymers are regulated spatially and temporally by other factors within the cell. My lab specializes in using genetics and cell biology to study the function and in vivo assembly dynamics of cytoskeletal polymers in many different species of prokaryotes, including E. coli , Bacillus subtillis , B. megaterium , B. thuringiensis , Mycobacterium smegmatis , and Pseudomonas species. Our approach has been to focus on bacterial plasmids as tools to discover new families of cytoskeletal polymers. Plasmids offer the advantages that they are genetically diverse and each encodes its own segregation system. Our studies have led to the discovery of new types of cytoskeletal structures involved in DNA segregation in bacteria (Pogliano 2008), including a new family of tubulins (TubZ) (Larsen et al. 2007) and a new family of actins (AlfA) (Becker et al. 2006). To complete some of our long term goals, we have initiated collaborations with biochemists, structural biologists and physicists to study polymer assembly in vitro and in silico to generate an integrated and detailed mechanistic understanding of cytoskeletal dynamics in prokaryotes. Studies of the bacterial cytoskeleton will lead to an understanding of how prokaryotic cells generate and maintain their subcellular organization and will provide insight into how the eukaryotic cytoskeleton evolved. Image of sporulating Bacillus megaterium with membranes stained red with FM 4-64.  Select Publications Kraemer JA, Erb ML, Waddling CA, Montabana EA, Zehr EA, Wang H, Nguyen K, Pham DS, Agard DA, Pogliano J. A phage tubulin assembles dynamic filaments by an atypical mechanism to center viral DNA within the host cell. Cell. 2012 Jun 22;149(7):1488-99. Pogliano, J., Pogliano, N. and Silverman, J. Daptomycin mediated reorganization of membrane architecture causes mislocalization of essential cell division proteins J Bacteriol. September 2012 vol. 194 no. 17: 4494-4504. PMID: 22661688. Sakoulas G, Bayer AS, Pogliano J, Tsuji BT, Yang SJ, Mishra NN, Nizet V, Yeaman MR, Moise PA. Ampicillin Enhances Daptomycin- and Cationic Host Defense Peptide-Mediated Killing of Ampicillin- and Vancomycin-Resistant Enterococcus faecium . (2012) Antimicrob Agents Chemother. 2012 Feb;56(2):838-44. Epub 2011 Nov 28. Derman AI, Nonejuie P, Michel BC, Truong BD, Fujioka A, Erb ML, Pogliano J. Alp7R regulates expression of the actin-like protein Alp7A in Bacillus subtilis . (2012) J Bacteriol. May;194(10):2715-24. Epub 2012 Mar 16. Eswaramoorthy, P., Erb, M., Gregory, J., Silverman, J. Pogliano,K., Pogliano,J., Ramamurthi, K. Cellular architecture mediates DivIVA ultrastructure and regulates Min activity in Bacillus subtilis . (2011) mBIO 2011 Nov-Dec; 2(6): e00257-11. Purdy Drew KR, Pogliano J. Dynamic instability-driven centering/segregating mechanism in bacteria. (2011) Proc Natl Acad Sci U S A. 2011 Jul 5;108(27):11075-80. Epub 2011 Jun 17. PMCID:21685333 Abhay Dhand, Arnold S. Bayer, Joe Pogliano, Soo-Jin Yang, Michael Bolaris, Victor Nizet, Guiquing Wang, and George Sakoulas Use of Anti-Staphylococcal b-Lactams to Increase Daptomycin Activity in Eradicating Persistent Bacteremia Due to Methicillin-Resistant Staphylococcus aureus: Role of Enhanced Daptomycin Binding. (2011) Clinical and Infectious Disease, 53(2):p.158-163. Eloe EA, Malfatti F, Gutierrez J, Hardy K, Schmidt WE, Pogliano K, Pogliano J, Azam F, Bartlett DH. Isolation and characterization of a psychropiezophilic alphaproteobacterium. Appl Environ Microbiol. 2011 Nov;77(22):8145-53. Epub 2011 Sep 23. PMID: 21948832 Liu WT, Yang YL, Xu Y, Lamsa A, Haste NM, Yang JY, Ng J, Gonzalez D, Ellermeier CD, Straight PD, Pevzner PA, Pogliano J, Nizet V, Pogliano K, Dorrestein PC. (2010) Imaging mass spectrometry of intraspecies metabolic exchange revealed the cannibalistic factors of Bacillus subtilis . Proc Natl Acad Sci. Sep 14; 107(37):16286-90. Derman, A.I., Truong, B., Becker, E., Fujioka, A., Tucy, T.M. Erb, M., Patterson, P. and Pogliano, J. Phylogenetic analysis identifies many uncharacterized actin-like proteins (Alps) in bacteria: regulated polymerization, dynamic instability and treadmilling in Alp7A. (2009) Mol Microbiol. Aug;73(4):534-52. Pogliano J. (2008) The Bacterial Cytoskeleton. Curr. Opin. Cell Biol. 20:19-27. Derman, AI, Lim-Fong, G., Pogliano J. (2008) Intracellular mobility of plasmid DNA is limited by the ParA family of partitioning proteins. Mol. Microbiol. 67:935-946. Larsen RA, Cusumano C, Fujioka A, Lim-Fong G, Patterson P, and Pogliano J. (2007) Treadmilling of a prokaryotic tubulin-like protein, TubZ, required for plasmid stability in Bacillus thuringiensis . Genes & Dev. Jun 1;21(11):1340-52. Becker E, Herrera NC, Gunderson FQ, Derman AI, Dance AL, Sims J, Larsen RA, Pogliano J. (2006) DNA segregation by the bacterial actin AlfA during Bacillus subtilis growth and development. EMBO J. 13;25(24):5919-31. \n",
            "-------------------- Scraping faculty url 134/183 --------------------\n",
            "kit pogliano  Biography Kit Pogliano received her Ph.D. from the Department of Microbiology and Molecular Genetics at Harvard Medical School and was a Damon Runyon-Walter Winchell postdoctoral fellow at Harvard University. She is a recipient of the Searle Scholar and Beckman Young Investigator awards. Dr. Pogliano was named in 2009 as a Fellow in the American Academy of Microbiology.  Research Our research is organized around three key themes: Cellular organization and dynamics: What are the design principles of bacterial cells? Microbial interactions: How are the outcomes of interspecies interactions determined? Chemical Cell Biology: How do secreted metabolites and antibiotics affect bacterial cells? Our toolkit We take a multidisciplinary approach to our studies, using the tools of genetics, cell biology, proteomics, biochemistry, and we have recently added analytical chemistry to our toolkit. Our lab is fully equipped for molecular biology, genetics and for work with BSL2 pathogens. Our microscopy facility includes one dissecting stereomicroscope equipped for fluorescence and timelapse imaging as well as deconvolution microscopes equipped with growth chambers and microfluidic devices for time-lapse microscopy and lasers for photobleaching and total internal reflection microscopy. We have an HPLC apparatus for the analysis and purification of natural products, and more extensive equipment is available in the labs of our campus collaborators. We have ready access to excellent campus facilities that provide cutting edge equipment and analytical tools for protein and small molecule mass spectrometry, super resolution microscopy and sequencing, with excellent technical assistance available as needed. We also apply our imaging expertise to a variety of collaborative projects and enjoy the diversity these projects and visitors bring to our lab. The SpollQ landmark protein (green) localizes into puncta surrounding the developing spore, here visualized using structured illumination microscopy. Cellular organization and dynamics: What are the design principles of bacterial cells? The phagocytosis-like process of engulfment is a hallmark of endospore formation in Bacillus subtilis and its relatives. Engulfment provides a unique example of phagocytosis in a bacterium and it is an ideal system to study protein localization and dynamics as well as membrane movement and fission. Engulfment depends on three protein complexes, the DMP complex that degrades peptidoglycan, the Q-AH complex that provides a Velcro-like attachment between the two cells and the peptidoglycan biosynthetic complex. Our goal is to elucidate the mechanisms by which bacterial cells mediate membrane movement and membrane fission, coordinate peptidoglycan degradation and synthesis, and how they complete membrane fission after engulfment and cell division. B. subtilis (green) invading and destroying a neighboring bacterial colony (red). What role do antibiotics play in microbial communities? The vast majority of commercial antibiotics are derived from secreted bacterial natural products. However, despite the clinical importance of these molecules and the pressing need to discover new antibiotics, it remains unclear exactly what role these molecules play in nature. The secreted metabolome of B. subtilis contains a wide array of well-characterized antibacterial and antifungal compounds, providing an ideal starting point to elucidate the specific roles each play in interspecies interactions and development. Undomesticated B. subtilis strains have the ability to invade and destroy neighboring colonies or to form mixed biofilms in a more cooperative mode of growth. Our goal is to identify the genes and secreted metabolites that mediate interspecies interactions, using the tools of analytical chemistry and imaging mass spectrometry (in collaboration with Pieter Dorrestein) as well as genetics and cell biology. B. subtilis cells before and after treatment with the cannibalism toxin SDP. Chemical Cell Biology: How do secreted metabolites and antibiotics affect bacterial cells? We have developed cell biological tools to characterize the cellular targets for secreted metabolites and antibiotics, in collaboration with Joe Pogliano. This approach, which we call cytological profiling, has allowed us to rapidly identify molecules that kill bacterial cells and to determine their likely mechanism of action. It is remarkably specific, as we can discriminate between the cytological effects of molecules that dissipate either the pH or or both components of the PMF, as well as the effects of a wide array of antibiotics with well-characterized activities. These studies provide insight into the mechanisms by which essential biological processes are coordinated to maintain the appropriate size and they have several biomedical applications.  Select Publications Peters, C., A. Lamsa, R.B. Liu, D.T. Quach, J. Sugie, J. Pogliano, J. Lpez -Garrido and K. Pogliano. (2018) Rapid Inhibition Profiling identifies a keystone target in the nucleotide biosynthesis pathway. Journal of Antimicrobial Chemotherapy, Aug 22. doi: 10.1021/acschembio.8b00273. PMID: 30133247 Riley, E.P., A. Trinquier, M.L. Reilly, M. Durchon, R. Perera, K. Pogliano and J. Lpez-Garrido. (2018) Spatiotemporally regulated proteolysis to dissect the role of vegetative proteins during Bacillus subtilis sporulation: cell-specific requirement of H and A . Molecular Microbiology 108:45-62. PMID: 29363854. Lpez-Garrido, J., N. Ojkic, K. Khanna, E. Villa, R.G. Endres and K. Pogliano. (2018) Chromosome translocation inflates the Bacillus subtilis forespore and impacts cellular morphology. Cell, 172:758-770. PMID: 29425492 Ojkic, N. Lpez-Garrido, J., K. Pogliano and R.G. Endres. Cell wall remodeling drives engulfment during Bacillus subtilis sporulation, eLife, Nov. 17, e18657. PMID: 27852437 Lamsa, A., J. Lpez-Garrido, D.T. Quach, E.P. Riley, J. Pogliano and K. Pogliano. (2016) Rapid inhibition profiling in Bacillus subtilis to identify the mechanism of action of new antimicrobials, American Chemical Society Chemical Biology, 11:2222-2231. PMID: 27193499 Nguyen, K.B., A. Sreelatha, E. Durrant, J. Lpez-Garrido, A. Muszewska, M. Dudkiewicz, M. Grynberg, S. Yee, K. Pogliano, D. Tomchick, K. Pawowski, J.E. Dixon and V.S. Tagliabracci. (2016) Phosphorylation of spore coat proteins by a new family of kinases. Proc. Natl. Acad. Sci., USA 113:E3482-3491. PMID: 27185916 Nonejuie, G.L., R. Trial, G.L. Newton, A. Lamsa, V.R. Perera, J. Aguilar, W.-T. Liu, P.C. Dorrestein, J. Pogliano and K. Pogliano. (2015) Application of bacterial cytological profiling to crude natural product extracts reveals the antibacterial arsenal of Bacillus subtilis , Journal of Antibiotics. Dec 9. doi: 10.1038/ja.2015.116. PMID: 26648120. Shin, J.Y., J. Lpez-Garrido, S.-H. Lee, C. Diaz, T. Fleming, C. Bustamante and K. Pogliano. (2015) Visualization and functional dissection of coaxial paired SpoIIIE channels across the sporulation septum. eLife, May 7;4. doi: 10.7554. PMID: 25950186 Nonejuie P., M. Burkhart, K. Pogliano and J. Pogliano. (2013) Bacterial cytological profiling rapidly identifies the cellular pathways targeted by antibacterial molecules. Proc. Natl. Acad. Sci., USA 110:16169-16174. PMID:24046367 Lamsa, A., W.-T. Liu, P.C. Dorrestein and K. Pogliano. (2012) The Bacillus subtilis cannibalism toxin SDP collapses the proton motive force and induces autolysis. Molecular Microbiology 84:486-500. PMID: 22469514 Fleming, T., J.Y. Shin, S.H. Lee, E.C. Becker, K.C. Huang, C. Bustamante and K. Pogliano. (2010) Dynamic SpoIIIE assembly mediates septal membrane fission during Bacillus subtilis sporulation. Genes and Development 24:1160-1172. PMID:20516200 Chiba, S. A. Lamsa and K. Pogliano. (2009) A ribosome nascent chain sensor of membrane protein biogenesis in Bacillus subtilis, EMBO J. 28:3461-3475 PMID: 19779460 Gregory, J.A., E.C. Becker and K. Pogliano. (2008) Bacillus subtilis MinC destabilizes FtsZ-rings at new cell poles and contributes to the timing of cell division. Genes and Development 22:3475-3488. PMID: 19141479 Broder, D. and K. Pogliano. (2006) Forespore engulfment mediated by a ratchet-like mechanism. Cell 126:917-928. PMID: 16959571 \n",
            "-------------------- Scraping faculty url 135/183 --------------------\n",
            "jim posakony  Biography Jim Posakony received his Ph.D. from the California Institute of Technology in 1982. Prior to coming to UCSD in 1985, he was a postdoctoral fellow at Harvard University. In 1986 he was named a Pew Scholar in the Biomedical Sciences. He was elected as a AAAS Fellow in 2011.  Research The driving interest of our lab is in the mechanisms by which cell-cell signaling pathways and transcriptional regulatory networks act to specify cell fates during animal development. The peripheral nervous system (PNS) of the fruit fly Drosophila melanogaster has proven to be a particularly accessible setting in which to investigate these problems, and we have been exploiting its advantages for many years. Our work is currently focused in the following areas: (1) Regulatory functions of the Notch cell-cell signaling pathway. The Notch pathway is one of a handful of signaling systems that are collectively responsible for the vast majority of conditional cell fate specification events in metazoan development. We have a long history of helping to elucidate the pathway's structure and functions, including the identification of its transducing transcription factor, Suppressor of Hairless [Su(H)]. Our present interests are in (a) understanding how Notch signaling switches Su(H) from being a repressor to an activator, and (b) unraveling structure-function relationships in the Bearded family of proteins, which act as antagonists of Notch signaling and are encoded by genes that are themselves targets of the pathway. (2) Dual enhancers controlling the expression of neural precursor genes. Neural precursor cells, such as those that give rise to the sensory bristles covering the fly's body, express a distinctive set of regulatory genes. We have found that many of these genes are served by duplicate enhancer modules, each of which drives the gene's expression specifically in neural precursors. We are interested both in how the modules encode this specificity, and in the functional advantages of the dual-enhancer control strategy. (3) Role of architecture in the function and specificity of transcriptional enhancer modules. Enhancers make use of particular regulatory \"codes\" (combinations of transcription factor binding sites) to generate their characteristic expression output. But these codes do not by themselves explain enhancer specificity the module's \"architecture\" (the arrangement and spacing of the factor binding sites) typically plays a critical role as well. We are investigating the mechanistic basis for this; in particular, how the constrained spacing of pairs of sites leads to synergies in the action of the cognate transcription factors. (4) Long-term evolutionary conservation of specific cis-regulatory elements in developmental control genes. It is now well established that changes in transcriptional regulatory connections are a major engine for the generation of developmental novelty during evolution. We have been looking at the other side of the coin by identifying and studying specific elements of transcriptional regulatory architecture that are ancient and have been conserved in orthologous genes from a deep ancestor. We are particularly interested in the functional basis of this long-term conservation; i.e., what distinctive regulatory capabilities are conferred by these special elements that would account for their durability?  Select Publications Miller, S. W., Movsesyan, A., Zhang, S., Fernndez, R., and Posakony, J. W. (2019). Evolutionary emergence of Hairless as a novel component of the Notch signaling pathway. eLife 8: e48115. Miller, S. W., and Posakony, J. W. (2018). Lateral inhibition: Two modes of nonautonomous negative autoregulation by neuralized . PLoS Genet. 14: e1007528. Smith, A. F., Posakony, J. W., and Rebeiz, M. (2017). Automated tools for comparative sequence analysis of genic regions using the GenePalette application. Dev. Biol. 429: 158-164. Miller, S. W., Rebeiz, M., Atanasov, J. E., and Posakony, J. W. (2014). Neural precursor-specific expression of multiple Drosophila genes is driven by dual enhancer modules with overlapping function. Proc. Natl. Acad. Sci. USA 111: 17194-17199. Liu, F., and Posakony, J. W. (2014). An enhancer composed of interlocking submodules controls transcriptional autoregulation of Suppressor of Hairless . Dev. Cell 29: 88-101. Liu, F., and Posakony, J. W. (2012). Role of architecture in the function and specificity of two Notch-regulated transcriptional enhancer modules. PLoS Genet. 8: e1002796. Rebeiz, M., Castro, B., Liu, F., Yue, F., and Posakony, J. W. (2012). Ancestral and conserved cis-regulatory architectures in developmental control genes. Dev. Biol. 362: 282-294. Ngre, N., et al. (2011). A cis-regulatory map of the Drosophila genome. Nature 471: 527-531. Rebeiz, M., Miller, S. W., and Posakony, J. W. (2011). Notch regulates numb : Integration of conditional and autonomous cell fate specification. Development 138: 215-225. Fontana, J. R., and Posakony, J. W. (2009). Both inhibition and activation of Notch signaling rely on a conserved Neuralized-binding motif in Bearded proteins and the Notch ligand Delta. Dev. Biol. 333: 373-385. Miller, S. W., Avidor-Reiss, T., Polyanovsky, A., and Posakony, J. W. (2009). Complex interplay of three transcription factors in controlling the tormogen differentiation program of Drosophila mechanoreceptors. Dev. Biol. 329: 386-399. Castro, B., Barolo, S., Bailey, A. M., and Posakony, J. W. (2005). Lateral inhibition in proneural clusters: Cis-regulatory logic and default repression by Suppressor of Hairless. Development 132: 3333-3344. Rebeiz, M., Stone, T., and Posakony, J. W. (2005). An ancient transcriptional regulatory linkage. Dev. Biol. 281: 299-308. Reeves, N., and Posakony, J. W. (2005). Genetic programs activated by proneural proteins in the developing Drosophila PNS. Dev. Cell 8: 413-425. Barolo, S., Stone, T., Bang, A. G., and Posakony, J. W. (2002). Default repression and Notch signaling: Hairless acts as an adaptor to recruit the corepressors Groucho and dCtBP to Suppressor of Hairless. Genes Dev. 16: 1964-1976. Rebeiz, M., Reeves, N. L., and Posakony, J. W. (2002). SCORE: A computational approach to the identification of cis-regulatory modules and target genes in whole-genome sequence data. Proc. Natl. Acad. Sci. USA 99: 9888-9893. Barolo, S., and Posakony, J. W. (2002). Three habits of highly effective signaling pathways: Principles of transcriptional control by developmental cell signaling. Genes Dev. 16: 1167-1181. Barolo, S., Walker, R. G., Polyanovsky, A. D., Freschi, G., Keil, T., and Posakony, J. W. (2000). A Notch-independent activity of Suppressor of Hairless is required for normal mechanoreceptor physiology. Cell 103: 957-969. \n",
            "-------------------- Scraping faculty url 136/183 --------------------\n",
            "paul price  Research We study the mechanisms by which proteins control normal calcification of bone and prevent abnormal calcification of soft tissues. Our focus in these studies is on two vitamin K dependent proteins we have discovered, the 50 residue bone Gla protein (BGP) and the 83 residue matrix Gla protein (MGP). Studies using the vitamin K antagonist warfarin and targeted gene deletion have shown that BGP and MGP both inhibit calcification in vivo . Impaired BGP expression causes the spread of calcification from bone into adjacent cartilaginous structures and impaired MGP expression causes calcification of arteries. We are presently investigating the relationship between the impaired synthesis of MGP in aging humans and the calcification of arteries as well as the development of atherosclerotic plaque. Our hypothesis is that calcification of the artery in aging humans is caused in part by dietary vitamin K deficiency, and that artery calcification contributes to the formation of the atherosclerotic plaque. This hypothesis is supported by the observation that the extent of artery calcification is the best predictor of future myocardial infarction found to date. It is also supported by the fact that animals on a cholesterol rich diet will not form atherosclerotic plaques unless arteries have been previously induced to calcify. We are also investigating the in vitro mechanism by which MGP and BGP bind to the surface of hydroxyapatite crystals and prevent their growth and the relationship between the structure of these proteins and their ability to inhibit calcification. We are particularly interested in the role of the vitamin K-dependent calcium binding amino acid, g-carboxyglutamic acid (Gla), in the activities of these proteins and have developed a variety of methods to detect and modify Gla residues. We have shown that MGP and BGP are normally expressed in bone and that the expression of both proteins by osteoblasts is strongly induced by the active metabolite of vitamin D, 1,25(OH) 2 D 3 . 1,25(OH) 2 D 3 restores serum calcium levels to normal by reducing the rate at which serum calcium is deposited in bone. We believe that the induction of BGP and MGP by 1,25(OH) 2 D 3 is part of the mechanism by which the hormone regulates bone calcification, and we are studying the mechanisms by which the expression of these proteins mediate the action of vitamin D on bone. We have recently found that the only class of drugs currently approved for the treatment of osteoporosis, the bisphosphonates, have strong and specific effects on the levels of BGP and MGP in serum. There is a similarity in the chemical structure of bisphonates and the g-carboxyglutamyl side chain, and we believe that these effects of bisphonates on MGP and BGP are due in part to the competition of the drug with these proteins for binding to sites in bone mineral. We are presently investigating the relationship between the action of bisphonates on bone and the effect of these drugs on the metabolism of MGP and BGP.  Select Publications This is a partial listing of recent publications from the Paul Price laboratory. Full text versions of these and other manuscripts can be found at the Price lab website . Price PA, Roublick AM, Williamson MK. (2006). Artery calcification in uremic rats is increased by a low protein diet and prevented by treatment with ibandronate. Kidney International November; 70 :1577-1583. Hamlin NJ, Ong KG, Price PA. (2006). A serum factor that recalcifies demineralized bone is conserved in bony fish and sharks but is not found in invertebrates. Calcified Tissue International. 76 :326-334. Price PA, Chan WS, Jolson DM, Williamson MK. (2006). The elastic lamellae of devitalized arteries calcify when incubated in serum: evidence for a serum calcification factor. Arterioscler Thromb Vasc Biol. May; 26(5) :1079-85. Hamlin NJ and Price PA. (2004). Mineralization of decalcified bone occurs under cell culture conditions and requires bovine serum but not cells. Calcified Tissue International. 10.1007/s00223-004-0190-1 Published online: 28 May. Price PA, June HH, Hamlin NJ, Williamson MK. (2004). Evidence for a serum factor that initiates the re-calcification of demineralized bone. J Biol Chem. Apr 30; 279(18) :19169-80. Epub 2004 Feb 20. Simes DC, Williamson MK, Schaff BJ, Gavaia PJ, Ingleton PM, Price PA, Cancela ML. (2004). Characterization of osteocalcin (BGP) and matrix Gla protein (MGP) fish specific antibodies: validation for immunodetection studies in lower vertebrates. Calcif Tissue Int. Feb; 74(2) :170-80. Epub 2003 Dec 15. Price PA, Williamson MK, Nguyen TM, Than TN. (2004). Serum levels of the fetuin-mineral complex correlate with artery calcification in the rat. J Biol Chem. Jan 16; 279(3) :1594-600. Epub 2003 Oct 24. Price PA and Lim JE. (2003). The Inhibition of Calcium Phosphate Precipitation by Fetuin is Accomanied by the Formation of a Fetuin-Mineral Complex. J Biol Chem. 278(24) :22144-52. Price PA, Nguyen TMT and Williamson MK. (2003). Biochemical Characterization of the Serum Fetuin-Mineral Complex. J Biol Chem. 278(24) :22153-60. Simes DC, Williamson MK, Ortiz-Delgado JB, Viegas CS, Price PA, Cancela ML. (2003). Purification of matrix Gla protein from a marine teleost fish, Argyrosomus regius: calcified cartilage and not bone as the primary site of MGP accumulation in fish. J Bone Miner Res Feb; 18(2) :244-59. Price PA, Omid N, Than TN, Williamson MK. (2002). The Amino Bisphosphonate Ibandronate Prevents Calciphylaxis in the Rat at Doses that Inhibit Bone Resorption. Calcif Tissue. Int. Oct; 71(4) :356-63. Price PA, June HH, Buckley JR, Williamson MK. (2002). SB 242784, a selective inhibitor of the osteoclastic V-H+ATPase, inhibits arterial calcification in the rat. Circ Res Sep 20; 91(6) :547-52. Price PA, Caputo JM, Williamson MK. (2002). Bone origin of the serum complex of calcium, phosphate, fetuin, and matrix Gla protein: Biochemical evidence for the cancellous bone-remodeling compartment. J.Bone Min. Research.; 17(7) :1171-79. Price PA, Thomas GR, Pardini AW, Figueira WF, Caputo JM, Williamson MK. (2002). Discovery of a high molecular weight complex of calcium, phosphate, fetuin, and matrix gamma-carboxyglutamic acid protein in the serum of etidronate-treated rats. J Biol Chem. Feb 8; 277(6) :3926-34. \n",
            "-------------------- Scraping faculty url 137/183 --------------------\n",
            "jose pruneda-paz  Biography Dr. Pruneda-Paz received his Ph.D. in Biochemistry from the National University of Cordoba (Argentina), and carried out his postdoctoral research at The Scripps Research Institute and the University of California, San Diego.  Research Most organisms live in environmental conditions that oscillate with a 24 hour period and evolved an endogenous mechanism, known as the circadian clock, which coordinates the temporal organization of biological processes to optimal times of the day or night. Biological clocks allows an organism to anticipate the onset of periodic environmental changes providing an adaptive advantage that results in increased fitness and survival. In plants, the pervasiveness of the clock control extends to nearly every aspect of growth and development through an extensive transcriptional regulatory network that impose circadian oscillations to more than one third of the whole transcriptome. My laboratory is focused in studying the molecular mechanisms by which the circadian clock controls critical biological processes (i.e. outputs). In particular, we use plant-pathogen interactions as an experimental platform and aim to understand the transcriptional networks that control the circadian regulation of plant defense responses. Host-pathogen interactions are regulated by sophisticated mechanisms shaped by millions of years of coevolution. In fact, immune responses are regulated by the circadian clock not only in plants but also in other organisms such as flies and mice, suggesting that this is an evolutionary advantageous mechanism. Since the clock function and plant-pathogen responses are largely dependent on the regulation of gene expression, an extensive transcriptional network is anticipated at the interface of clock and defense signaling pathways. Using forward and reverse genetics, as well as systems approaches, we are primarily focused on identifying critical components of these transcriptional regulatory networks. Plant pests account for great losses in crop yield around the world, thus our findings may ultimately transform our understanding of plant microbial infections and how to treat them. Moreover, plant research made major impact in understanding critical mechanisms of great biological and biomedical relevance including RNA-silencing, DNA-methylation, proteasomal-control of protein stability and programmed cell death among others. Therefore, our research could potentially uncover highly conserved regulatory networks orchestrating immune defenses in other organisms as well.  Select Publications Nagel DH, Doherty CJ, Pruneda-Paz JL, Schmitz RJ, Ecker JR, Kay SA (2015) Genome-wide identification of CCA1 targets uncovers an expanded clock network in Arabidopsis. Proc Natl Acad Sci USA, 112(34):E4802-10. PMID: 26261339 Zheng XY, Zhou M, Yoo H, Pruneda-Paz JL, Spivey NW, Kay SA, Dong X (2015) Spatial and temporal regulation of biosynthesis of the plant immune signal salicylic acid. Proc Natl Acad Sci USA, 112(30):9166-73. PMID: 26139525. Taylor-Teeples M, Lin L, de Lucas M, Turco G, Toal TW, Gaudinier A,Young NF, Trabucco GM, Veling MT, Lamothe R, Handakumbura PP, Xiong G, Wang C, Corwin J, Tsoukalas N, Pauly M, Kliebenstein DJ, Dehesh K, Tagkopoulos I, Breton G, Pruneda-Paz JL, Ahnert SE, Kay SA, Brady SM, Hazen SP (2015) An Arabidopsis Gene Regulatory Network for Xylem Specification and Secondary Wall Biosynthesis. Nature, 517(7536):571-5. PMID: 25533953. Zhou Y, Liu X, Engstrom EM, Nimchuk ZL, Pruneda-Paz JL, Tarr PT, Yan A, Kay SA, Meyerowitz EM (2015) Control of plant stem cell function by conserved interacting transcriptional regulators. Nature, 517(7534):377-80. PMID: 25363783. Kolmos E, Chow BY, Pruneda-Paz JL, Kay SA (2014) HsfB2b-mediated repression of PRR7 directs abiotic stress responses of the circadian clock. Proc Natl Acad Sci USA, 111(45):16172-7. PMID: 25352668. Guan P, Wang R, Nacry P, Breton G, Kay SA, Pruneda-Paz JL, Davani A, Crawford NM (2014) Nitrate foraging by Arabidopsis roots is mediated by the transcription factor TCP20 through the systemic signaling pathway. Proc Natl Acad Sci USA, 111(42):15267-72. PMID: 25288754. Nagel D, Pruneda-Paz JL, Kay SA (2014) FLOWERING BHLH 1 Affects Warm Temperature Responses in the Arabidopsis Circadian Clock. Proc Natl Acad Sci USA 111(40):14595-600. PMID: 25246594. Pruneda-Paz JL*1, Breton G*, Nagel DH, Kang SE, Bonaldi K, Doherty C, Ravelo S, Galli M, Ecker JR, Kay SA1 (2014) A Genome Scale Resource for the Functional Characterization of Arabidopsis Transcription factors. Cell Reports 8: 1-11. PMID: 25043187. (*co-first authors, 1co-corresponding authors). Chow BY, Sanchez SE, Breton G, Pruneda-Paz JL, Krogan NT, Kay SA (2014) Transcriptional Regulation of LUX by CBF1 Mediates Cold Input to the Circadian Clock in Arabidopsis. Curr Biol. 4(13):1518-24. PMID: 24954045. Niwa M, Daimon Y, Kurotani KI, Higo A, Pruneda-Paz JL, Breton G, Mitsuda N, Kay SA, Ohme-Takagi M, Endo M, Araki T (2013) BRANCHED1 Interacts with FLOWERING LOCUS T to Repress the Floral Transition of the Axillary Meristems in Arabidopsis. Plant Cell 25(4):1228-42. PMID:23613197. Li L, Ljung K, Breton G, Schmitz R, Pruneda-Paz J, Cowing-Vitron C, Cole BJ, Ivans LJ, Pedmale UV Jung HS, Ecker JR, Kay SA, Chory J (2012) Linking photoreceptor excitation to changes in plant architecture. Genes Dev. 26(8):785-90. PMID:22508725. Gendron JM, Pruneda-Paz JL, Doherty CJ, Gross AM, Kang SE, Kay SA (2012) Arabidopsis circadian clock protein, TOC1, is a DNA-binding transcription factor. Proc Natl Acad Sci U S A, 109(8):3167-72. PMID:22315425. Gaudinier A, Zhang L, Reece-Hoyes JS, Taylor-Teeples M, Pu L, Liu Z, Breton G, Pruneda-Paz JL, Kim D, Kay SA, Walhout AJ, Ware D, Brady SM (2011) Enhanced Y1H assays for Arabidopsis. Nat. Methods 8(12):1053-5. PMID:22037706. Pruneda-Paz JL and Kay SA. (2010) An expanding universe of circadian networks in higher plants. Trends Plant Sci. 15(5):259-265. Baudry A, Ito S, Song YH, Strait AA, Kiba T, Lu S, Henriques R, Pruneda-Paz JL, Chua NH, Tobin EM, Kay SA, Imaizumi T. (2010) FKF1 and LKP2 act in concert with ZTL to control Arabidopsis clock progression. Plant Cell 22(3): 606-22. PMID:20354196. Pruneda-Paz JL, Breton G, Para A, Kay SA. (2009) A functional genomics approach reveals CHE as a component of the Arabidopsis circadian clock. Science 323(5920):1481-1485. Hazen SP, Schultz TF, Pruneda-Paz JL, Borevitz JO, Ecker JR, Kay SA. (2005) LUX ARRHYTHMO encodes a novel Myb transcription factor essential for circadian rhythms. Proc. Natl. Acad. Sci. U S A 102:10387-10392. Hazen SP, Borevitz JO, Harmon FG, Pruneda-Paz JL, Schultz TF, Yanovsky MJ, Liljegren SJ, Ecker JR, Kay SA. (2005) Rapid array mapping of circadian clock and developmental mutations in Arabidopsis . Plant Physiol. 138(2):990-907. Pruneda-Paz JL, Linares M, Cabrera JE, Genti-Raimondi S. (2004) TeiR , a LuxR-type transcription factor required for testosterone degradation in Comamonas testosteroni . J. Bacteriol. 186(5):1430-1437. \n",
            "-------------------- Scraping faculty url 138/183 --------------------\n",
            "samara reck-peterson  Biography Samara Reck-Peterson received her Ph.D. from the Department of Cell Biology at Yale University and did postdoctoral research with Ron Vale at the University of California, San Francisco. She was an Assistant and Associate professor of Cell Biology at Harvard Medical School and the Associate Director of the Biological and Biomedical Sciences Graduate Program at Harvard Medical School. She joined the UCSD faculty in 2015. Dr. Reck-Peterson was the recipient of an NIH New Innovator Award.  Research Eukaryotic cells use microtubule tracks to move cargo over long distances. There are two types of molecular motors that move on microtubules: dyneins and kinesins. Dyneins move cargo towards the minus ends of microtubules, typically towards the cell interior. Kinesins move cargo in the opposite direction, generally towards the cell surface. Our lab is tackling three broad questions. How does the dynein motor work and how is it regulated? What are the rules governing transport in cells? Why do defects in transport cause disease? How does the dynein motor work and how is it regulated? Cytoplasmic dynein-1 is the only motor used for long-distance minus-end-directed microtubule-based trafficking in eukaryotic cells ranging from human neurons to the hyphae of filamentous fungi. Yet, it transports dozens, if not hundreds of different cargos. One of our labs goals is to understand how this large, multi-subunit, motor is regulated. Dynein has two regulators that are conserved and required for most (perhaps all) of its functions: the dynactin complex and Lis1/ Nudel. Current experiments in the lab are focused on determining how these regulators work. We use quantitative approaches to do this, including single-molecule imaging, cryo-electron microscopy (in collaboration with the lab of Andres Leschziner), proteomics, and live-cell imaging. What are the rules governing transport? In addition to cytoplasmic dynein-1, at least 15 kinesin motors (in humans) are responsible for moving cargo in the opposite direction as dynein. This small subset of motors is responsible for nearly all long distance transport in eukaryotic cells. What is the full list of dynein and kinesin cargos? What links the motors to different cargos? How is specificity achieved? Do all cargos have a distinct mechanism for recruiting motors? We are addressing these questions using two different discovery-based approaches: We use the filamentous fungus, Aspergillus nidulans, as a powerful genetic system to dissect microtubule-based transport. We have conducted screens to identify genes required for the transport of endosomes, peroxisomes, and nuclei. Current projects in the lab are characterizing these novel transport factors. We also use proteomic approaches to identify the dynein and kinesin transportome. Current efforts in the lab are focused on using proximity-dependent labeling in living human cells to identify the dynein proteome. Why do defects in transport cause disease? Mutations in every component of the transport systemtubulins (the building blocks of microtubules), dynein, kinesins, and proteins that regulate the motors or their trackscause neurological diseases. For example, mutations in cytoplasmic dynein-1 cause a peripheral neuropathy called Charcot Marie Tooth type 2, Spinal Muscular Atrophy (SMA), malformation of cortical development, and intellectual disability. Like some mutations in dynein, mutations in the dynein activator BICD2 also cause SMA. Mutations in a subunit of dynactin result in motor neuron disease and Perry syndrome, an early onset form of Parkinsons disease. Defects in microtubule-based axonal transport are also observed in other neurodegenerative diseases including Amyotrophic Lateral Sclerosis, Alzheimers and Huntingtons, suggesting that there may be additional links between dynein and neurological disease yet to be uncovered. One hypothesis is that neurodegenerative diseases can result from the compromised trafficking of particular cargos. Current projects in the lab are aimed at identifying which cargos or points of regulation are defective in disease states.  Select Publications Salogiannis J, Egan MJ, Reck-Peterson SL. (2016) Peroxisomes move by hitchhiking on early endosomes using the novel linker proteins PxdA. J Cell Biol 212: 289-296. Commentaries on this research appeared in: J Cell Biol 212: 258 and Nat Rev Mol Cell Biol 17: 134. Cianfrocco MA, DeSantis ME, Leschziner AE, Reck-Peterson SL. (2015) Mechanism and Regulation of Cytoplasmic Dynein. Annu Rev Cell Dev Biol 31: 83-108. Toropova K*, Zou S*, Roberts AJ, Redwine WB, Goodman BS, Reck-Peterson SL#, Leschziner AE#. (2014). Lis1 regulates dynein by sterically blocking its mechanochemical cycle. eLIFE 3:e03372. Roberts AJ, Goodman BS, Reck-Peterson SL. (2014) Reconstitution of dynein transport to the microtubule plus end by kinesin. eLIFE 3:e02641. Tan K, Roberts AJ, Chonofsky M, Egan MJ, Reck-Peterson SL. (2014) A microscopy-based screen employing multiplex genome sequencing identifies cargo-specific requirements for dynein velocity. Mol Biol Cell 25: 669-678. Derr ND*, Goodman BS*, Jungmann R, Leschziner AE, Shih WM, Reck-Peterson SL. (2012) Tug of war in motor protein ensembles revealed with a programmable DNA origmai scaffold. Science 338: 662-665. Redwine WB*, Hernandez-Lopez R*, Zou S, Huang J, Reck-Peterson SL, Leschziner AE. (2012) Structural basis for microtubule binding and release by dynein. Science 337: 1532-1536. Huang J*, Roberts AJ*, Leschziner AE, Reck-Peterson SL. (2012) Lis1 acts as a \"clutch\" between the ATPase and microtubule-binding domains of the dynein motor. Cell 150: 975-986. Commentary on this research appeared in: Cell 150: 877-879 Egan M, Tan K, Reck-Peterson SL. (2012) Lis1 is an initiation factor for dynein-driven organelle transport. J Cell Biol 197: 971-982. Commentary on this research appeared in: J Cell Biol 197: 852. Laan L, Pavin N, Husson J, Romet-Lemonne G, van Duin M, Lopez MP, Vale RD, Julicher F, Reck-Peterson SL, Dogterom M. (2012) Cortical dynein controls microtubule dynamics and length, generating pulling forces that reliably position microtubule asters. Cell 148: 502-514. Qiu W*, Derr ND*, Goodman BS, Villa E, Wu D, Shih W, Reck-Peterson SL. (2012) Dynein achieves processive motion using both stochastic and coordinated stepping. Nat Struct Mol Biol 19: 193-200. Commentary on this research appeared in: Nature 482: 7383. Kardon J, Reck-Peterson SL, Vale RD. (2009) Regulation of the processivity and intracellular localization of S. cerevisiae dynein by dynactin. PNAS 106: 5669-74. Cho C, Reck-Peterson SL, Vale RD. (2008) Cytoplasmic dynein's regulatory ATPase sites affect processivity and force generation. J Biol Chem 283: 25839-45. Gennerich A, Carter AP, Reck-Peterson SL, Vale RD. (2007) Force-induced bidirectional stepping of cytoplasmic dynein. Cell 131: 952-965. Reck-Peterson SL, Yildiz Y*, Carter AP*, Gennerich A, Zhang N, and Vale RD. (2006) Single molecule analysis of dynein processivity and stepping behavior.Cell 126: 335-348. Commentaries on this research appeared in: Cell 126: 242-244, Nat Rev Mol Cell Biol 7: 625, J Cell Biol 172: 486-492. \n",
            "-------------------- Scraping faculty url 139/183 --------------------\n",
            "pamela reinagel  Biography Pam Reinagel received her Ph.D. in Biochemistry at Harvard University. She conducted postdoctoral research in theoretical neuroscience with Christof Koch at the California Institute of Technology (Caltech), and in experimental neuroscience with R. Clay Reid at Harvard Medical School. Dr. Reinagel is the recipient of a Sloan Research Fellowship. She started her lab at UCSD in 2003.  Research The Reinagel lab studies perceptually guided selection of action in the visually behaving rat. Our primary experimental tool is quantitative visual behavior. Much of our effort is focused on building computational models to explain reward-motivated choice behavior in the face of sensory uncertainty. Lesions and pharmacology are used to probe neural mechanisms. One current focus is the speed-accuracy trade-off of rats as they perform different types of visual discriminations. A second focus is blindsight: identifying which visual behaviors are dependent on primary visual cortex in the rat, and characterizing visual decision-making and learning in V1-independent visual behaviors.  Select Publications Reinagel, P (2013) Speed and Accuracy of Visual Motion Discrimination by Rats. PLoS-ONE 8(6):e68505. Petruno, S, Clark, RE, and Reinagel, P (2013) Evidence that primary visual cortex is required for image orientation and motion discrimination by rats. PLoS-ONE.0056543 Clark, RE, Reinagel P, Broadbent NJ, Flister ED, and Squire, LR (2011) Intact performance on feature ambiguous discriminations in rats with lesions of the perirhinal cortex. Neuron 70(1):132-140. Meier*, PM, Flister*, ED, and Reinagel P (2011) Collinear features impair visual detection by rats. Journal of Vision 11(3)22. \n",
            "-------------------- Scraping faculty url 140/183 --------------------\n",
            "scott rifkin  Biography Scott Rifkin received his PhD from Yale University and was a postdoctoral fellow at Harvard and an NIH postdoctoral fellow at MIT. He joined the faculty of UCSD in the summer of 2009.  Research My laboratory studies how stochastic, genetic, and environmental variation is filtered through development and physiology to produce phenotypic variation and how evolutionary forces shape that process. We work primarily at the level of gene expression, taking advantage of technologies for visualizing gene expression dynamics at single-transcript resolution, among others. Current projects in the lab include probing the dynamics and robustness of the genetic networks underlying sex-determination and intestinal specification in nematodes and investigating how natural variation affects the dynamic properties of signaling and regulatory networks in yeasts. Individual transcripts of xol-1 (yellow) and sdc-2 (red) in early C. elegans embryos. Blue spots are DAPI-stained nuclei.  Select Publications L Du, S Tracy, SA Rifkin (2016). Mutagenesis of GATA motifs controlling the endoderm regulator elt-2 reveals distinct dominant and secondary cis-regulatory elements. Developmental Biology. (in press) M Maduro, G Broitman-Maduro, H Choi, F Carranza, AC-Y Wu, SA Rifkin (2016). MED GATA factors promote robust development of the elegans endoderm. Developmental Biology. 404:66-79. SR Stockwell, CR Landry, SA Rifkin (2015). The yeast galactose network as a quantitative model for cellular memory. Molecular Biosystems. 11: 28-37. AC-Y Wu, SA Rifkin (2015). Aro: A machine learning approach to identifying single molecules and estimating classification error in fluorescence microscopy images. BMC Bioinformatics. 16: 102. MA Bakowski, CA Desjardins, MG Smelkinson, TA Dunbar, IF Lopez-Moyado, SA Rifkin, CA Cuomo, ER Troemel (2014). Ubiquitin-mediated response to microsporidia and virus infection. in C. elegans. PLoS Pathogens. 10: e1004200. CR Landry*, SA Rifkin* (2012). The genotype-phenotype maps of systems biology and quantitative genetics: distinct and complementary. in Evolutionary Systems Biology. (Soyer, ed.). Advances in Experimental Medicine and Biology Series. 751:371-398. *equal contribution SA Rifkin (ed.) (2012). Quantitative Trait Loci: methods and protocols. Methods in Molecular Biology, 871. Springer, New York. SA Rifkin (2011). Identifying fluorescently labeled single molecules in image stacks using machine learning. in Molecular Methods for Evolutionary Genetics (Orgogozo & Rockman eds.) Methods in Molecular Biology. 772: 329-348. CR Landry, SA Rifkin (2010). Chromatin regulators shape the genotypephenotype map. Molecular Systems Biology. 6:434. A Raj*, SA Rifkin*, E Andersen, A van Oudenaarden (2010). Variability in gene expression underlies partial penetrance in Caenorhabditis elegans . Nature. 463: 913-918. *equal contribution A Raj, P van den Bogaard, SA Rifkin, A van Oudenaarden, S Tyagi (2008). Imaging individual mRNA molecules using multiple singly labeled probes. Nature Methods 5:877-889. Y Gilad, SA Rifkin, JK Pritchard (2008). Revealing the architecture of gene regulation: the promise of eQTL studies. Trends in Genetics 24:408-413. CR Landry, B Lemos, SA Rifkin, WJ Dickinson, DL Hartl (2008). Genetic properties influencing the evolvability of gene expression. Science 317:118-121. Y Gilad, A Oshlack, SA Rifkin (2006). Natural selection on gene expression. Trends in Genetics 22:456-461. SA Rifkin, D Houle, J Kim, KP White (2005). A mutation accumulation assay reveals a broad capacity for rapid evolution of gene expression. Nature 438:220-223. Y Gilad, SA Rifkin, P Bertone, M Gerstein, KP White (2005). Multi-species microarrays reveal the effect of sequence divergence on gene expression profiles. Genome Research 15:674-680. SA Rifkin, J Kim, KP White (2003). Evolution of gene expression in the Drosophila melanogaster subgroup. Nature Genetics 33:138-144. SA Rifkin, J Kim (2002). Geometry of gene expression dynamics. Bioinformatics. 18:1176-1183. SA Rifkin, K Atteson, J Kim (2000). Constraint structure analysis of gene expression. Functional and Integrative Genomics. 1:174-185. KP White, SA Rifkin, P Hurban, DS Hogness (1999). Microarray analysis of Drosophila development during metamorphosis. Science. 286:2179-2184. \n",
            "-------------------- Scraping faculty url 141/183 --------------------\n",
            "cory root  Biography Dr. Root received his B.A. in Biology and Chemistry from Whitman College and his Ph.D. in Biology from UC San Diego. He did postdoctoral research with Dr. Richard Axel at Columbia University. Dr. Root joined the faculty in 2016 and has been the recipient of the Elkins Memorial Award for an outstanding doctoral thesis in the field of Drosophila Neurobiology and a K99/R00 Pathway to Independence Award from the NIDCD.  Research Sensory systems detect features of the external world, which are represented in the activity of ensembles of neurons in the brain. My lab seeks to understand how sensory information is represented in the brain and how internal representations acquire meaning that can be used to inform behavior. Animals have evolved neural circuits that drive innate behavioral responses to environmental cues that have been essential to the survival of a species over long periods of evolutionary time. However, most stimuli do not have innate significance. Rather, meaning is acquired over the lifetime of an animal through learning. We exploit the sense of smell to investigate how sensory representations afford meaning for innate and experience-dependent behavior. We use endoscopic 2-photon calcium imaging to monitor neuronal activity, and molecular genetics, viral tracing and optogenetics to trace and manipulate the neural circuits of the mouse brain. The Olfactory System Olfactory perception is initiated upon the recognition of odorants by a large repertoire of receptors in the sensory epithelium. Neurons expressing a given receptor are randomly distributed within zones of the epithelium but project with precision to two spatially invariant glomeruli in the olfactory bulb. This invariant glomerular map in the bulb is transformed in the representations in higher olfactory centers. For instance, the piriform cortex receives distributed input from seemingly random sets of glomeruli in the olfactory bulb, and is implicated in learned associations. In contrast, the cortical amygdala receives topographically organized input from the olfactory bulb, and in recent work I have demonstrated that the neurons in this pathway mediate innate behaviors. The Cortical Amygdala Mediates Innate Behavior The higher order circuits mediating innate behaviors have not been well defined, however, anatomic tracing experiments implicated the cortical amygdala in determined behaviors. With Richard Axel at Columbia University, I devised experiments to directly determine if the cortical amygdala is involved in innate odor-driven behavior. We found that the stereotyped circuits from the olfactory bulb to the cortical amygdala are required for innate aversive and appetitive responses to odor. Moreover, we identified and manipulated the activity of these neurons by exploiting the promoter of the activity-dependent gene, arc, to express channelrhodopsin in neurons of the cortical amygdala activated by odors that elicit innate behaviors. Optical activation of these neurons leads to appropriate behaviors that recapitulate the responses to odors. Together these data demonstrate that the cortical amygdala contains distinct neurons that are necessary and sufficient for the generation of innate, odor-driven behaviors (Root et al., Nature 2014). Moreover, we have revealed that neurons mediating aversive and appetitive behaviors have distinct projections to downstream targets. The identification of neural circuits that elicit innate behaviors provides a foundation for dissecting sites of plasticity to afford behavioral flexibility.  Select Publications Root CM, Denny CA, Hen R, Axel R. (2014) The participation of cortical amygdala in innate, odor-driven behavior. Nature 515:269-273. Root CM, Ko KI, Jafari A, Wang JW. (2011) Presynaptic facilitation by neuropeptide signaling mediates odor-driven food search. Cell 145:133-44. Ignell R,* Root CM,* Birse RT, Wang JW, Nassel DR, Winther AME. (2009) Presynaptic peptidergic modulation of olfactory receptor neurons in Drosophila. PNAS 106:13070- 13075. *these authors contributed equally Dacks AM, Green DS, Root CM, Nighorn AJ, Wang JW. (2009) Serotonin modulates olfactory processing in the antennal lobe of Drosophila. J. Neurogenetics 23:366-77. Root CM, Masuyama K, Green DS, Enell LE, Nassel DR, Lee C-H, Wang JW. (2008) A presynaptic gain control mechanism fine-tunes olfactory behavior. Neuron 59:311-321. Root CM, Velazquez-Ulloa NA, Monsalve GC, Minakova E, Spitzer NC. (2008) Embryonically expressed GABA and glutamate drive activity regulating neurotransmitter specification. J. Neurosci 28:4777-4784. Root CM,* Semmelhack JL,* Wong AM, Flores J, Wang JW. (2007) Propagation of olfactory information in drosophila. PNAS 14: 11826-11831. *these authors contributed equally Borodinsky LN, Root CM, Cronin J, Sann SB, Gu X, Spitzer NC. (2004) Activity-dependent homeostatic specification of transmitter expression in embryonic neurons. Nature 429: 523- 530. \n",
            "-------------------- Scraping faculty url 142/183 --------------------\n",
            "michael rosenfeld \n",
            "-------------------- Scraping faculty url 143/183 --------------------\n",
            "kaustuv roy  Biography Kaustuv Roy received his Ph.D. from the University of Chicago, where he completed postdoctoral training.  Research What are the physical and biotic processes that determine large scale spatial patterns of biodiversity? How do species and communities respond to climate change and other anthropogenic impacts and what are the ecological and evolutionary consequences of such responses? These and related questions form the focus of my research. We work primarily in marine systems using interdisciplinary approaches that combine ecological and genetic information from living species and populations with the temporal perspective afforded by the rich fossil record of marine invertebrates. Origin and maintenance of latitudinal diversity gradients The latitudinal gradient in diversity - the dramatic increase in the number of species and higher taxa from the poles to the tropics - is a global biodiversity pattern shared by both marine and terrestrial systems. Understanding the processes responsible for the origin and maintenance of such gradients is essential for informed management and conservation of biodiversity, especially in a world faced with rising temperatures, rampant overexploitation of natural resources and increasing urbanization. Yet despite over a century of work such processes remain poorly understood. We are working to better understand (i) the role of ecological processes in maintaining spatial gradients in diversity and (ii) the role of evolutionary and historical processes in generating such gradients. The former includes the role of temperature and other environmental variables in determining the number of species that can coexist in a given region as well as how species respond to changes in climate. As far as evolutionary dynamics are concerned, we are not only quantifying how origination and extinction rates change along latitude, a dynamic still poorly documented, but also how shifts in geographic distributions of taxa over time affect regional patterns of species richness. I am also involved in developing theoretical models to explore how in situ speciation and extinction rates interact with patterns of biotic interchange between regions to create large scale spatial gradients in biological diversity. Anthropogenic impacts on coastal marine ecosystems Intertidal and subtidal habitats harbor a diverse fauna of invertebrates, plants and fish yet these habitats are among those most impacted by human activities and least protected. While harvesting practices and other activities of an expanding coastal population all over the world are generally thought to be negatively affecting the species living in these habitats, little quantitative data still exist on the nature and extent of such impacts. In addition, how human impacts are affecting macroecological relationships or biogeographic patterns also remain poorly studied. We have initiated large-scale studies that will quantify the ecological consequences of harvesting (legal and illegal), urbanization and climate change along the California coast. We also plan to initiate similar studies along other coastlines in the near future. A more detailed description of our research and a complete list of our publications can be found at http://www.biology.ucsd.edu/labs/roy/ .  Select Publications Pfister, Catherine A., Kaustuv Roy, J. Timothy Wootton, Sophie J. McCoy, Robert T. Paine, Thomas H. Suchanek and Eric Sanford. 2016. Historical baselines and the future of shell calcification for a foundation species in a changing ocean. Proc. Royal Soc. B 283:20160392. Huang, Danwei, Emma E. Goldberg and Kaustuv Roy 2015. Fossils, phylogenies, and the challenge of preserving evolutionary history in the face of anthropogenic extinctions. Proc. Nat. Acad. Sci. U.S.A. 112:4909-4914. Huang, Shan, Kaustuv Roy, James W. Valentine and David Jablonski 2015. Convergence, divergence, and parallelism in marine biodiversity trends: Integrating present-day and fossil data. Proc. Nat. Acad. Sci. U.S.A 112:4903-4908. Huang, Shan, Kaustuv Roy and David Jablonski. 2015. Origins, bottlenecks, and present-day diversity: Patterns of morphospace occupation in marine bivalves. Evolution 69:735-746. Huang, Danwei and Kaustuv Roy. 2014. The future of evolutionary diversity in reef corals. Phil. Trans. Royal Society London B370:20140010. Huang, Shan., Kaustuv Roy and David Jablonski. 2014. Do past climate states influence diversity dynamics and the present-day latitudinal diversity gradient? Global Ecol. Biogeog. DOI: 10.1111/geb.12153 Jablonski, D., C.L. Belanger, S.K. Berke, S. Huang, A. Z. Krug, K. Roy, A. Tomasovych and J.W. Valentine. 2013. Out of the tropics, but how? Fossils, bridge species, and thermal ranges in the dynamics of the marine latitudinal diversity gradient. Proc. Nat. Acad. Sci. U.S.A. 110:10487-10494. Huang, Danwei and Kaustuv Roy. 2013. Anthropogenic extinction threats and future loss of evolutionary history in reef corals. Ecology and Evolution 3:1184-1193. Eagle, R.A., J.M. Eiler, A.K. Tripati, J.B. Ries, P.S. Freitas, C. Hiebenthal, A.D. Wanamaker, M. Taviani, M. Elliot, S. Marenssi, K. Nakamura, P. Ramirez, and K. Roy. 2013. The influence of temperature and seawater carbonate saturation state on 13C - 18O bond ordering in bivalve mollusks. Biogeosciences 10:1-16. Berke, Sarah K., David Jablonski, Andrew Z. Krug, Kaustuv Roy and Adam Tomasovych. 2013. Beyond Bergmann's rule: size-latitude relationships in marine Bivalvia world-wide. Global Ecol. Biogeog. 22:173-183. Belanger, Christina L., David Jablonski, Kaustuv Roy, Sarah, K. Berke, Andrew Z. Krug, and James W. Valentine. 2012. Global environmental predictors of benthic marine biogeographic structure. Proc. Nat. Acad. Sci. U.S.A. 109:14046-14051. Fenberg, Phillip B. and Kaustuv Roy. 2012. Anthropogenic harvesting pressure and changes in life history: Insights from a rocky intertidal limpet. American Naturalist 180:200-210. Chiba, Satoshi and Kaustuv Roy (2011). Selectivity of terrestrial gastropod extinctions on an oceanic archipelago and insights into the anthropogenic extinction process. Proc. Nat. Acad. Sci. U.S.A. 108:9496-9501. Fenberg, Phillip B., Michael E. Hellberg, Lynne Mullen and Kaustuv Roy (2010). Genetic diversity and population structure of the size-selectively harvested owl limpet, Lottia gigantea. Marine Ecology 31:574-583. Roy, Kaustuv, Gene Hunt and David Jablonski (2009). Phylogenetic conservatism of extinctions in marine bivalves. Science 325:733-737. Schemske, Douglas W., Gary G. Mittelbach, Howard V. Cornell, James M. Sobel and Kaustuv Roy (2009). Is there a latitudinal gradient in the importance of biotic interactions? Annu. Rev. Ecol. Evol. Syst. 40:245-269. J. D. Witman and Kaustuv Roy (eds). 2009. Marine Macroecology. University of Chicago Press. Fenberg, Phillip B. and Kaustuv Roy (2008). Ecological and evolutionary consequences of size-selective harvesting: how much do we know? Molecular Ecology 17:209-220. Roy, Kaustuv and Emma E. Goldberg (2007). Origination, extinction and dispersal: integrative models for understanding present-day diversity gradients. American Naturalist 170:S71-S85. Jablonski, David, Kaustuv Roy and James W. Valentine (2006). Out of the tropics: evolutionary dynamics of the latitudinal diversity gradient. Science 314:102-106. Hunt, Gene and Kaustuv Roy (2006). Climate change, body size evolution and Copes Rule in deep-sea ostracodes. Proc. Nat. Acad. Sci. U.S.A. 103:1347-1352. Hellberg, Michael E., Deborah P. Balch and Kaustuv Roy. (2001). Climate-driven range expansion and morphological evolution in a marine gastropod. Science 292: 1707-1710. \n",
            "-------------------- Scraping faculty url 144/183 --------------------\n",
            "alistair russell  Biography Alistair Russell received his PhD from the University of Washington in the laboratory of Joseph Mougous, where, supported by the NSF, he studied interbacterial interactions driven by the type VI secretion system, using a combination of genetic, informatic, and biochemical tools. For his postdoctoral research, he worked as a Merck Fellow of the Damon Runyon Cancer Research Foundation, in the laboratory of Jesse Bloom at the Fred Hutchinson Cancer Research Center, trying to use novel single-cell methodologies to unravel events precipitating induction of an interferon response by influenza A virus. He joined the Molecular Biology faculty at UCSD in 2019.  Research Nearly every cell in the human body has the capacity to respond to a viral infection with the production of interferonscritical components of cell intrinsic immunity that drive resolution of disease, and, unfortunately, much of the resultant immunopathology. Unsurprisingly, viruses circumvent, co-opt, or otherwise subvert the induction of an antiviral state. Regardless, no defense (or offense) is perfect, and viral populations almost invariably induce interferon in a small fraction of infected cells. Some of this rarity of response is driven by rare viral failure to suppress interferon, in part due to the extreme variation evinced by many viral populations. However, the rarity of response is also partially explained by intrinsic host mechanisms suppressing spurious activationseeking to avoid the resultant cost from an overly sensitive triggering of inflammatory processes. My lab seeks to study the rarity of the interferon response during viral infection using influenza A virus as a model, using single-cell transcriptomics, single virion genomics, flow cytometry, population-based sequencing, and fine-grained genetic analyses to delineate the molecular events that proceed engagement of innate immune signaling pathways in responding cells. With these data in hand, we will seek to better model disease processes, with the hope that our work may help development of therapeutic interventions or more robust vaccine responses.  Select Publications Russell, A.B., Elshina, E., Kowalsky, J.R., te Velthuis, A.J., Bloom, J.D., (2019) Single-cell virus sequencing of influenza infections that trigger innate immunity. Journal of Virology. 0022-538X Russell, A.B., Trapnell, C., Bloom, Bloom, J.D., (2018) Extreme heterogeneity of influenza virus infection in single cells. eLife. 7, e02020 Russell, A.B., Wexler, A.G., Brittany, H.N., Whitney, J.C., Bohn, A.J., Young Ah, G., Tran, B.Q., Natasha, B., Zheng, H., Peterson, S.B., Chou, S., Gonen, T., Goodlett, D.R., Goodman, A.L., Mougous, J.D. (2014) A type VI secretion-related pathway in Bacteroidetes mediates interbacterial antagonism. Cell Host and Microbe. 16, 227-236 Russell, A.B., Peterson, S.B., Mougous, J.D. (2014) Type VI secretion effectors: poisons with a purpose. Nature Reviews Microbiology. 12, 137-148 Russell, A.B., LeRoux, M., Hathazi, K., Agnello, D.M., Ishikawa, T., Wiggins, P.A., Wai, S.W., Mougous, J.D. (2013) Diverse type VI secretion phospholipases are functionally plastic antibacterial effectors. Nature. 496, 508-512 Russell, A.B.*, Singh, P.*, Brittnacher, M., Bui, N.K., Hood, R.D., Carl, M.A., Agnello, D.M., Schwarz, S., Goodlett, D.R., Vollmer, W., and Mougous, J.D. (2012) A widespread bacterial type VI secretion effector superfamily identified using a heuristic approach. Cell Host and Microbe. 11, 538-549 Russell, A.B., Hood, R.D., Bui, N.K., LeRoux, M., Vollmer, W. and Mougous, J.D. (2011) Type VI secretion delivers lytic effectors to the periplasm of target cells. Nature. 475, 343-347 Munkvold, K.R.*, Russell, A.B.*, Kvitko, B.H., and Collmer, A.R. (2009) Pseudomonas syringae pv. tomato DC3000 type III effector HopAA1-1 functions redundantly with chlorosis-promoting factor PSPTO4723 to produce bacterial speck lesions in host tomato. Molecular Plant Microbe Interactions. 22, 1341-1355 Wexler, A.G., Bao, Y., Whitney, J.C., Bobay, L.M., Xavier, J.B., Schofield, W.B., Barry, N.A., Russell, A.B., Tran, B.Q., Goo, Y.A., Goodleyy, D.R., Ochman, H., Mougous, J.D., Goodman, A.L. (2016) Human symbionts inject and neutralize antibacterial toxins to persist in the gut. Proceedings of the National Academy of Sciences. 113, 3639-3644 LeRoux, M., Kirkpatric, R.L., Montauti, E.I., Tran, B.Q., Peterson, S.B., Harding, B.N., Whitney, J.C., Russell, A.B., Traxler, B., Goo, Y.A., Goodlett, D.R., Wiggins, P.A., Mougous, J.D. (2015) Kin cell lysis is a danger signal that activates antibacterial pathways of Pseudomonas aeruginosa. eLife. 4, e05701 Whitney, J.C., Beck, C.M., Goo, Y.A., Russell, A.B., Harding, B., De Leon, J.A., Cunningham, D.A., Tran, B.Q., Low, D.A., Hayes, C.S., Mougous, J.D. (2014) Genetically distinct pathways guide effector export through the type VI secretion system. Molecular Microbiology. 92, 529-542 \n",
            "-------------------- Scraping faculty url 145/183 --------------------\n",
            "oliver ryder  Biography Oliver A. Ryder received his Ph.D. from UCSD. He is the geneticist and recipient of the Kleberg Genetics Chair at the Center for Reproduction of Endangered Species, Zoological Society of San Diego. He is a member of Specialist Groups of the Species Survival Commission of World Conservation Union: Equid Specialist Group, Captive Breeding Specialist Group, and Re-introduction Specialists Group. Dr. Ryder is an associate editor of the Journal of Heredity.  Research Challenges involved in conserving biological diversity include addressing the numerous scientific issues limiting our understanding of the Earth's biota. Speciation and species diversity, the role of genetic variation in population persistence, its rate of generation and loss in different taxonomic groups and under different ecological conditions, as well as the development of tools for addressing these questions are all significant topics in biology pertinent to conservation issues. The genetics group at the CRES (Center for Reproduction of Endangered Species - a division of the Zoological Society of San Diego) undertakes comparative cytogenetic and molecular genetics studies involving a diverse array of species suitable as models for investigating relative rates of evolution among different taxa, population structure and systematics. Additionally, research pertinent to planning and management for selected mammalian, reptilian and avian species of conservation concern is undertaken utilizing appropriate techniques, including the development and application of new methodologies. Current emphases include molecular evolutionary and population studies involving analysis of nuclear microsatellite allelic variation and nucleotide sequence analysis of mitochondrial genomes. Assessment, planning, monitoring, and management of small populations for conservation will benefit from application of more effective methods of collection and analysis of genetic data. Black rhinoceros male in Masia Mara Reserve in Kenya. Increased knowledge of genetics of black rhinos contributes to monitoring and recovery efforts for this endangered species. An increased understanding of many threatened species at the cytogenetic and molecular genetic level will contribute to gene pool conservation strategies and management as well as establishing a broader knowledge of the dynamics of genome evolution. Located in lab facilities on the grounds of the San Diego Zoo, our studies are facilitated by unique resources, including cell lines and other biological samples that are available as a result of the animal collections of the San Diego Zoo and Wild Animal Park as well as through associated field activities of CRES staff.  Select Publications Ivy, J.A., A.S. Putnam, A.Y. Navarro, J. Gurr, and O.A. Ryder. 2016. Applying SNP-derived molecular ancestry estimates to captive breeding programs. Journal of Heredity 107(5). 403-412. Saragusty, J., S. Diecke, M. Drukker, B.S. Durrant, I.F. Ben-Nun, C. Galli, F. Goritz, K. Hayashi, R. Hermes, S. Holtze, S. Johnson, G. Lazzari, P. Loi, J.F. Loring, K. Okita, M.B. Renfree, S. Seet, T. Voracek, J. Stejskal, O.A. Ryder, and T.B. Hildebrandt. 2016. Rewinding the Process of Mammalian Extinction. Zoo Biology 9999. 1-13. Yu, L., G-D. Wang, J. Ruan, Y-B. Chen, C-P. Yang, X. Cao, H. Wu, Y-H. Liu, Z-L. Du, X-P. Wang, J. Yang, S-C. Cheng, L. Zhong, L. Wang, X. Wang, J-Y. Hu, L. Fang, B. Bai, K-L. Wang, N. Yuan, S-F. Wu, B-G. Li, J-G. Zhang, Y-Q. Yang, C-L. Zhang, Y-C. Long, H-S. Li, J-Y. Yang, D.M. Irwin, O.A. Ryder, Y. Li, C-I. Wu, and Y-P. Zhang. 2016. Genomic analysis of snub-nosed monkeys (Rhinopithecus) identifies genes and processes related to high-altitude adaptation. Nature Genetics 48(8). 947-955. Bosse, M., H-J. Megens, O. Madsen, R.P.M.A. Crooijmans, O.A. Ryder, F. Austerlitz, M.A.M. Groenen, and M.A.R. de Cara. 2015. Using genome-wide measures of coancestry to maintain diversity and fitness in endangered and domestic pig populations. Genome Research 25. 970-981. Buchalski, M.R., A.Y. Navarro, W.M. Boyce, T. Winston Vickers, M.W. Tobler, L.A. Nordstrom, J.A. Garcia, D.A. Gille, M.C.T. Penedo, O.A. Ryder, and H.B. Ernest. 2015. Genetic population structure of Peninsular bighorn sheep (Ovis canadensis nelsoni) indicates substantial gene flow across USMexico border. Biological Conservation 184. 218-228. Fisch, K.M., C.C. Kozfkay, J.A. Ivy, and O.A. Ryder. 2015. Fish hatchery genetic management techniques: Integrating theory with implementation. North American Journal of Aquaculture 77. 343-357. Koepfli, K.-P., B. Paten, The Genome 10K Community of Scientists, C. Steiner, O. Ryder, and S.J. OBrien. 2015. The genome 10K project: A way forward. Annual Review of Animal Biosciences 3. 57-111. Nam, K., K. Munch, A. Hobolth, J.Y. Dutheil, K.R. Veeramah, A.E. Woerner, M.F. Hammer, Great Ape Genome Diversity Project, O.A. Ryder, T. Mailund, and M.H. Schierup. 2015. Extreme selective sweeps independently targeted the X chromosomes of the great apes. Proceedings of the National Academy of Sciences 112. 6413-6418. Steiner, C.C., S.J. Charter, N. Goddard, H. Davis, M. Brandt, M.L. Houck, and O.A. Ryder. 2015. Chromosomal variation and perinatal mortality in San Diego Zoo Soemmerrings gazelles. Zoo Biology 9999. 1-11. Wisely, S.M., O.A. Ryder, J.F. Santymire, Engelhardt, and B.J. Novak. 2015. A road map for 21st century genetic restoration: Gene pool enrichment of the black-footed ferret. Journal of Heredity 106. 581-592. Liu, J., X. Wang, S. Cho, B.K. Lim, D.M. Irwin, O.A. Ryder, Y. Zhang, and L. Yu. 2014. Evolutionary and functional novelty of pancreatic ribonuclease: A study of Musteloidea (order Carnivora). . Scientific Reports. DOI: 10.1038/srep05070. OBrien, S.J., D. Haussler, and O.A. Ryder. 2014. The birds of Genome10K. GigaScience 3. 32. Steiner, C.C., S.J. Charter, M.L. Houck, and O.A. Ryder. 2014. Molecular phylogeny and chromosomal evolution of Alcelaphini (Antilopinae). Journal of Heredity 105. 324-333. Evans, J.D., S.J. Brown, K.J. Hackett, G. Robinson, S. Richards, D. Lawson, C. Elsik, J. Coddington, O. Edwards, S. Emrich, T. Gabaldon, M. Goldsmith, G. Hanes, B. Misof, M. Munoz-Torres, O. Niehuis, A. Papanicolaou, M. Pfrender, M. Poelchau, M. Purcell-Miramontes, H.M. Robertson, O.A. Ryder, D. Tagu, T. Torres, E. Zdobnov, G. Zhang, and X. Zhou. 2013. The i5K Initiative: Advancing arthropod genomics for knowledge, human health, agriculture, and the environment. Journal of Heredity 104. 594-600. Luan, P.-T., O.A Ryder, H. Davis, Y.-P. Zhang, and L. Yu. 2013. Incorporating indels as phylogenetic characters: Impact for interfamilial relationships within Arctoidea (Mammalia: Carnivora). . Molecular Phylogenetics and Evolution 66. 748756. Moser, A.B., J. Hey, P.K. Dranchak, M.W. Karaman, J. Zhao, L.A. Cox, O.A. Ryder, and J.G. Hacia. 2013. Diverse captive non-human primates with phytanic acid-deficient diets rich in plant products have substantial phytanic acid levels in their red blood cells. Lipids in Health and Disease. DOI: 10.1186/1476-511X-12-10. Orlando, L., A. Ginolhac, G. Zhang, D. Froese, A. Albrechtsen, M. Stiller, M. Schubert, E. Cappellini, B. Petersen, I. Moltke, P.L. Johnson, M. Fumagalli, J.T. Vilstrup, M. Raghavan, T. Korneliussen, A.S. Malaspinas, J. Vogt, D. Szklarczyk, C.D. Kelstrup, J. Vinther, A. Dolocan, J. Stenderup, A.M. Velazquez, J. Cahill, M. Rasmussen, X. Wang, J. Min, G.D. Zazula, A. Seguin-Orlando, C. Mortensen, K. Magnussen, J.F. Thompson, J. Weinstock, K. Gregersen, K.H. Roed, V. Eisenmann, C.J. Rubin, D.C. Miller, D.F. Antcza, M.F. Bertelsen, S. Brunak, K.A. Al-Rasheid, O.A. Ryder, L. Andersson, J. Mundy, A. Krogh, M.T. Gilbert, K. Kjr, T. Sicheritz-Ponten, L.J. Jensen, J.V. Olsen, M. Hofreiter, R. Nielsen, B. Shapiro, J. Wang, and E. Willerslev. 2013. Recalibrating Equus evolution using the genome sequence of an early Middle Pleistocene horse. Nature 499. 74-78. Prado-Martinez, J., P.H. Sudmant, J.M. Kidd, H. Li, J.L. Kelley, B. Lorente-Galdos, K.R. Veeramah, A.E. Woerner, T.D. OConnor, G. Santpere, A. Cagan, C. Theunert, F. Casals, H. Laayouni, K. Munch, A. Hobolth, A.E. Halager, M. Malig, J. Hernandez-Rodriguez, I. Hernando-Herraez, K. Prufer, M. Pybus, L. Johnstone, M. Lachmann, C. Alkan, D. Twigg, N. Petit, C. Baker, F. Hormozdiari, M. Fernandez-Callejo, M. Dabad, M.L. Wilson, L. Stevison, C. Camprub, T. Carvalho, A. Ruiz-Herrera, L. Vives, M. Mele, T. Abello, I. Kondova, R.E. Bontrop, A. Bontrop, F. Lankester, J.A. Kiyang, R.A. Bergl, E. Lonsdorf, S. Myers, M. Ventura, P. Gagneux, D. Comas, H. Siegismund, J. Blanc, L. Agueda-Calpena, M. Gut, L. Fulton, S.A. Tishkoff, J.C. Mullikin, R.K. Wilson, I.G. Gut, M.K. Gonder, O.A. Ryder, B.H. Hahn, A. Navarro, J.M. Akey, J. Bertranpetit, D. Reich, T. Mailund, M.H. Schierup, C. Hvilsom, A.M. Andres, J.D. Wall, C.D. Bustamante, M.F. Hammer, E.E. Eichler, and T. Marques-Bonet. 2013. Great ape genetic diversity and population history. Nature 499. 471-475. Steiner, C.C, and O.A. Ryder. 2013. Characterization of Prdm9 in equids and sterility in mules. PLoS One 8. e61746. Steiner, C.C, A.S. Putnma, P.E.A. Hoeck, and O.A. Ryder. 2013. Conservation genomics of threatened animal species. Annual Review of Animal Biosciences 1. 261-281. Bernardi, G., E.O. Wiley, H. Mansour, M.R. Miller, G. Orti, D. Haussler, S.J. OBrien, O.A. Ryder, and B. Venkatesh. 2012. The fishes of Genome 10K. Marine Genomics 7. 3-6. Dennis, P., S. Ellis, J. Mellen, P. Lee, F. Olea-Popelka, A. Petric, and O.A. Ryder. 2012. IOD in rhinos--epidemiology group report: Report from the epidemiology working group of the international workshop on iron overload disorder in browsing rhinoceros (February 2011). . Journal of Zoo and Wildlife Medicine 43. 114-116. Frankham, R., J.D. Ballou, M.R. Dudash, M.D.B. Eldridge, C.B. Fenster, R.C. Lacy, J.R. Mendelson, I.J. Porton, K. Ralls, and O.A. Ryder. 2012. Implications of different species concepts for conserving biodiversity. . Biological Conservation 153. 25-31. McCue, M.E., D.L. Bannasch, J.L. Petersen, J. Gurr, E. Bailey, M.M Binns, O. Distl, G. Guerin, T. Hasegawa, E.W. Hill, T. Leeb, G. Lindgren, M.C.T. Penedo, K.H. Roed, O.A. Ryder, J.E. Swinburne, T. Tozaki, S.J. Valberg, M. Vaudin, K. Lindblad-Toh, C.M. Wade, and J.R. Mickelson. 2012. A high density SNP array for the domestic horse and extant Perissodactyla: Utility for association mapping, genetic diversity, and phylogeny studies. PLoS Genetics 8. e1002451. Ryder, O.A. 2012. The Frozen Zoo. GeneWatch 25. 19-20. Scally, A., J.Y. Dutheil, L.W. Hillier, G.E. Jordan, I. Goodhead, J. Herrero, A. Hobolth, T. Lappalainen, T. Mailund, T. Marques-Bonet, S. McCarthy, S.H. Montgomery, P.C. Schwalie, Y.A. Tang, M.C. Ward, Y. Xue, B. Yngvadottir, C. Alkan, L.N. Andersen, Q. Ayub, E.V. Ball, K. Beal, B.J. Bradley, Y. Chen, C.M. Clee, S. Fitzgerald, T.A. Graves, Y. Gu, P. Heath, A. Heger, E. Karakoc, A. Kolb-Kokocinski, G.K. Laird, G. Lunter, S. Meader, M. Mort, J.C. Mullikin, K. Munch, T.D. OConnor, A.D. Phillips, J. Prado-Martinez, A.S. Rogers, S. Sajjadian, D. Schmidt, K. Shaw, J.T. Simpson, P.D. Stenson, D.J. Turner, L. Vigilan, A.J. Vilella, W. Whitener, B. Zhu, D.N. Cooper, P. de Jong, E.T. Dermitzakis, E.E. Eichler, P. Flicek, N. Goldman, N.I. Mundy, Z. Ning, D.T. Odom, C.P. Ponting, M.A. Quail, O.A. Ryder, S.M. Searle, W.C. Warren, R.K. Wilson, M.H. Schierup, J. Rogers, C. Tyler-Smith, and R. Durbin. 2012. Insights into hominid evolution from the gorilla genome sequence. Nature 483. 169-175. Field, D., Chemnick, L., Robbins, M., Garner, K. and Ryder, O.A. (1998). Paternity determination in captive lowland gorillas and orangutans and wild mountain gorillas by microsatellite analysis. Primates 39:199-209. Burrows, W. and Ryder, O.A. (1997). Y-chromosome variation in great apes. Nature 385:125-126. Garner, K.J. and Ryder, O.A. (1996). Mitochondrial DNA diversity in gorillas. Molec. Phyl. Evol. (In press). Zhang, Y. and Ryder, O.A. (1995). Different rates of mitochondrial DNA sequence evolution in Kirk's dik-dik (Madoqua kirkii) populations. Mol. Phyl. Evol. 4:291-297. Ryder, O. A. and Feistner, A.T.C. (1995). Research in zoos: a growth area in conservation. Biodiversity and Conservation 4: 671-677. Zhang, Y. and Ryder, O.A. (1993). Mitochondrial DNA sequence evolution in the Arctoidea. Proc. Natl. Acad. Sci. USA 90: 9557-9561. \n",
            "-------------------- Scraping faculty url 146/183 --------------------\n",
            "alan saghatelian \n",
            "-------------------- Scraping faculty url 147/183 --------------------\n",
            "milton saier  Biography Milton Saier received his Ph.D. from UC Berkeley and was a postdoctoral fellow at Johns Hopkins University.  Research Our laboratory has three primary research interests, one concerned with transcriptional and metabolic regulation in bacteria, a second with transport protein evolution, and a third with the recently identified process of transposon-mediated directed mutation. We also maintain the IUBMB-approved Transporter Classification Database, TCDB , which classifies transport systems found in all living organisms on Earth into five categories: class, subclass, family, subfamily and transport system. Our laboratory takes a multidisciplinary approach to science, using biochemical, molecular genetic, physiological, and computational approaches. Directed mutation is a proposed process that allows mutations to occur at higher frequencies when they are beneficial than when detrimental. Until recently, the existence of such a process has been controversial. However, we have described a novel mechanism of directed mutation mediated by the transposon, IS 5 in Escherichia coli. crp deletion mutants mutate specifically to glycerol utilization (Glp+) at rates that are enhanced by glycerol or the loss of the glycerol repressor (GlpR), depressed by glucose or glpR overexpression, and RecA-independent. Of the four tandem GlpR binding sites ( O1O4 ) upstream of the glpFK operon, O4 specifically controls glpFK expression while O1 primarily controls mutation rate in a process mediated by IS 5 hopping to a specific site on the E. coli chromosome upstream of the glpFK promoter. IS 5 insertion into other gene activation sites is unaffected by the presence of glycerol or the loss of GlpR. The results establish an example of transposon-mediated directed mutation, identify the protein responsible and define the mechanism involved. Most recently, we have identified two additional operons in E. coli that appear to be subject to transposon-mediated directed mutation: The flhDC flagellar master switch operon controlling motility and the fucAO operon controlling L-fucose and propanediol utilization. Our efforts have revealed three basic mechanisms of transcriptional control concerned with catabolite repression/activation in bacteria. Two of these occur in E. coli, and one occurs in B. subtilis. In E. coli, two DNA binding proteins, the cyclic AMP receptor protein (Crp) and the catabolite repressor/activator (Cra) protein, mediate transcriptional regulation of hundreds of genes encoding key enzymes of carbon and energy metabolism. Virtually every pathway of carbon metabolism is subject to these regulatory constraints. Crp generally controls the initiation of exogenous carbon source metabolism and senses cytoplasmic cyclic AMP levels. These levels are controlled by complex mechanisms, involving phosphorylated proteins of the sugar-transporting phosphotransferase system (PTS). Cra generally controls the flux of carbon through metabolic pathways and senses cytoplasmic metabolite concentrations. Cra usually controls gene expression independently of Crp, but it sometimes acts cooperatively with or antagonistically to Crp, depending on the target gene. It thereby mediates catabolite repression of catabolic operons by an indirect mechanism. Most recently, we have demonstrated that Cra regulates growth by activating expression of the crp gene. In B. subtilis and other Gram-positive bacteria, a metabolite-activated protein kinase phosphorylates a serine residue in a protein of the PTS called HPr. Phosphorylated HPr allosterically controls the activities of many target proteins (transport proteins, enzymes and transcription factors). It thereby controls the cytoplasmic concentrations of inducers as well as the activities of transcription factors that mediate catabolite repression. We are coming to realize that the mechanisms of catabolite control are very different for phylogenetically divergent bacteria. Phylogenetic analyses of integral membrane transport protein sequences have yielded a plethora of information about the times of appearance, the routes of evolution, and the relative rates of divergence of the proteins and protein domains which comprise various families of transport systems. These studies have shown that families of transport proteins of similar topology have evolved independently of each other, at different times in evolutionary history, using different routes. They have also revealed extensive domain shuffling in some such families but not in others. The probable means by which energy coupling became superimposed on transport during the evolutionary process has also come to light.  Select Publications Bioinformatic characterization of the Anoctamin Superfamily of Ca2+-activated ion channels and lipid scramblases. Medrano-Soto A, Moreno-Hagelsieb G, McLaughlin D, Ye ZS, Hendargo KJ, Saier MH Jr.PLoS One. 2018 Mar 26;13(3):e0192851. doi: 10.1371/journal.pone.0192851. eCollection 2018. PMID:29579047 Global landscape of cell envelope protein complexes in Escherichia coli. Babu M, Bundalovic-Torma C, Calmettes C, Phanse S, Zhang Q, Jiang Y, Minic Z, Kim S, Mehla J, Gagarinova A, Rodionova I, Kumar A, Guo H, Kagan O, Pogoutse O, Aoki H, Deineko V, Caufield JH, Holtzapple E, Zhang Z, Vastermark A, Pandya Y, Lai CC, El Bakkouri M, Hooda Y, Shah M, Burnside D, Hooshyar M, Vlasblom J, Rajagopala SV, Golshani A, Wuchty S, F Greenblatt J, Saier M, Uetz P, F Moraes T, Parkinson J, Emili A. Nat Biotechnol. 2018 Jan;36(1):103-112. doi: 10.1038/nbt.4024. Epub 2017 Nov 27. PMID: 29176613 The phosphocarrier protein HPr of the bacterial phosphotransferase system globally regulates energy metabolism by directly interacting with multiple enzymes in Escherichia coli. Rodionova IA, Zhang Z, Mehla J, Goodacre N, Babu M, Emili A, Uetz P, Saier MH Jr. J Biol Chem. 2017 Aug 25;292(34):14250-14257. doi: 10.1074/jbc.M117.795294. Epub 2017 Jun 20. PMID: 28634232 Hopping into a hot seat: Role of DNA structural features on IS5-mediated gene activation and inactivation under stress. Humayun MZ, Zhang Z, Butcher AM, Moshayedi A, Saier MH Jr. PLoS One. 2017 Jun 30;12(6):e0180156. doi: 10.1371/journal.pone.0180156. eCollection 2017. PMID: 28666002 Comparative genomics of transport proteins in probiotic and pathogenic Escherichia coli and Salmonella enterica strains. Do J, Zafar H, Saier MH Jr. Microb Pathog. 2017 Jun;107:106-115. doi: 10.1016/j.micpath.2017.03.022. Epub 2017 Mar 24. PMID: 28344124 Environment-directed activation of the Escherichia coli flhDC operon by transposons. Zhang Z, Kukita C, Humayun MZ, Saier MH. Microbiology. 2017 Apr;163(4):554-569. doi: 10.1099/mic.0.000426. Epub 2017 Apr 12. PMID: 28100305 Characterization of the Tetraspan Junctional Complex (4JC) superfamily. Chou A, Lee A, Hendargo KJ, Reddy VS, Shlykov MA, Kuppusamykrishnan H, Medrano-Soto A, Saier MH Jr. Biochim Biophys Acta Biomembr. 2017 Mar;1859(3):402-414. doi: 10.1016/j.bbamem.2016.11.015. Epub 2016 Dec 2. PMID: 27916633 Transposon-mediated directed mutation in bacteria and eukaryotes. Saier MH Jr, Kukita C, Zhang Z. Front Biosci (Landmark Ed). 2017 Mar 1;22:1458-1468. Review. PMID: 28199212 The Membrane Attack Complex/Perforin Superfamily. Moreno-Hagelsieb G, Vitug B, Medrano-Soto A, Saier MH Jr. J Mol Microbiol Biotechnol. 2017;27(4):252-267. doi: 10.1159/000481286. Epub 2017 Nov 17. PMID: 29145176 \n",
            "-------------------- Scraping faculty url 148/183 --------------------\n",
            "moselio schaechter \n",
            "-------------------- Scraping faculty url 149/183 --------------------\n",
            "immo scheffler  Biography After receiving his Ph.D. from Stanford University, Immo Scheffler was a Helen Hay Whitney Fellow at Harvard University and the Pasteur Institute, Paris. He is the recipient of an Alexander von Humbolt Award for Senior U.S. Scientists. At UCSD, Professor Scheffler has been honored with a Chancellor's Associates Excellence in Teaching Award, an Alumni Association Teacher of the Year Award, and a teaching award from the Academic Senate. Professor Scheffler was the first holder of the Paul D. Saltman Chair in Science Education at UCSD (2001-2004). Editorial boards include the Journal of Biological Chemistry. Dr. Scheffler was recently elected a Fellow of the American Association for the Advancement of Science (AAAS).  Research My laboratory was closed after 35 years of operation. I continue to have an interest in all aspects of mitochondrial biology, and a second, updated edition of my book MITOCHONDRIA was published recently. Collaborations continue with laboratories at UCSD (Taylor, Perkins), at UC Irvine (Wallace) and Neuzil (Australia).  Select Publications Potluri, P., Procaccio, V., Scheffler, I. E., and Wallace, D. C. (2016). High throughput gene complementation screening permits identification of a mammalian mitochondrial protein synthesis () mutant. Biochimica et Biophysica Acta (BBA)-Bioenergetics. Scheffler, I. E. (2015). Mitochondrial disease associated with complex I (NADH-CoQ oxidoreductase) deficiency. Journal of inherited metabolic disease, 38(3), 405-415. Scheffler, I.E. (2008). MITOCHONDRIA. 2nd edition, J. Wiley and Sons, Inc., Hoboken, New Jersey. 462 pages. Yadava, N., Potluri, P., and I.E. Scheffler (2008) Investigations of the potential effects of phosphorylation of the MWFE and ESSS subunits on complex I activity and assembly. The Int. J.of Biochem. and Cell Biol. 40:447 460. Scheffler, I.E. (2008) Signaling pathways from mitochondria to cytoplasm and nucleus. In Handbook of Cell Signaling (R.A. Bradshaw and E.A. Dennis, series editors). Part IV: Events in Intracellular Compartments (M. Farquahar and S.Subramani, editors). Elsevier Science. Invited review, in press. Scheffler, I.E. (2008) Basic Molecular Biology of Mitochondrial Replication. Chapter2, in Drug-Induced Mitochondrial Dysfunction. J. Dykens and Y. Will, editors; John Wiley and Sons, Hoboken, NJ. Dong, L.-F., Low, P., Dyason, J.C., Wang, X.-F., Prochazka, L., Witting, P.K., Freeman, R., Swettenham, E., Valis, K., Turanek, J., Spitz, D.R., Domann, F.E., Scheffler, I.E., Ralph, S.J., and J. Neuzil (2008). Tocopheryl succinate induces apoptosis by targeting ubiquinone-binding sites in mitochondrial respiratory complex II. Oncogene, 28:1-12. Neuzil, J., Dyason, J.C., Freeman, R., Dong, L.-F., Prochazka, L., Wang, X.-F., Scheffler, I., and Ralph, S.J. (2007). Mitocans as anticancer agents targeting mitochondria: lessons from studies with vitamin E analogues, inhibitors of complex II. J. Bioenerg. Biomembr. 39:65-72. Broderick, K.E., Potluri, P., Zhuang, S., Scheffler, I.E., Sharma, V.S., Pilz, R.B., and Boss, G.R. (2006). Cyanide detoxification by the cobalamin precursor cobinamide. Exp. Biology and Medicine, 231:641-49. Sonoda, J.,Laganiere, J., Mehl, I.R., Barish, G.D., Chong, L.W., Li, X., Scheffler, I.E., Mock, D.C. Bataille, A.R., Robert, F., Lee, C.H. Giguere, V., Evans, R.M. (2007) Nuclear receptor ERR alpha and coactivator PGC-1 beta are effectors of IFN-gamma-induced host defense. Genes Dev. 21:1909-1920. Potluri,P., Davila,A., Ruiz-Pesini,E., Mishmar,D., O'Hearn,S., Hancock,S., Simon,M., Scheffler,I.E., Wallace,D.C., and Procaccio,V. (2009). A novel NDUFA1 mutation leads to a progressive mitochondrial complex I-specific neurodegenerative disease. Molecular Genetics and Metabolism, in press. \n",
            "-------------------- Scraping faculty url 150/183 --------------------\n",
            "eric schmelz  Biography Eric Schmelz completed his PhD within the Center for Insect Science/Plant-Insect Interactions Group at the University of Arizona, Tucson. He then carried out his postdoctoral and research scientist studies at the USDA-Agricultural Research Service located at the University of Florida, Gainesville. He joined the Division of Biological Sciences faculty at U.C. San Diego in July 2014.  Research Ive have always been fascinated by the existence of specific biochemicals that that significantly govern interactions between organisms. Plants, arthropods and microbes are the most species rich life forms on earth. As primary producers, photosynthetic organisms are the ultimate source of most fixed carbon and under significant pressure to relinquish these resources. For reproductive success, plants must balance both growth and defense against a myriad of herbivorous arthropods and pathogens. These real-time interactions are of relevance to our global food supply and are underpinned by genetics, biochemistry, physiology, and behavior. We utilize maize ( Zea mays ) and bean ( Phaseolus & Vigna spp.) to examine dynamic inducible innate immune responses that limit losses against attacking organisms. In response to herbivory by generalist lepidopteran pests, plants rapidly initiate the biosynthesis and release of low molecular weight Volatile Organic Compounds (VOCs). Prominent in this VOC signature are monoterpenes, homoterpenes and sesquiterpenes that serve as indirect plant defenses by enabling the attraction of predators, parasitioids, and other natural enemies. How is this process enabled? We examine the mechanistic role of exogenous elicitors, termed Herbivore Associated Molecular Patterns (HAMPs), and endogenous signal transduction cascades that regulate insect-induced plant responses. Plant recognition and elicitation of inducible volatiles following herbivory in beans ( Phaseolus and Vigna spp.) is mediated by femtomolar amounts of the peptide elicitor termed inceptin, derived from chloroplastic ATP synthase -subunit proteolysis. Specialist herbivores evade detection by converting inceptin into a potent antagonist of elicitation. Molecular level details of herbivore perception by plants are essential for the understanding defense and susceptibility in crops. In contrast to foliar herbivores, maize fungal pathogens elicit the biosynthesis of non-volatile acidic terpenoid phytoalexins with direct antibiotic activity. The most highly inducible transcripts following pathogen attack encode the -macrocarpene synthases (ZmTps) 6/11 and the ent -copalyl diphosphate synthase termed Anther Ear 2 (ZmAn2). Down stream predicted enzymatic products of ZmTps6/11 and ZmAn2, termed zealexins and kauralexins respectively, are the predominant phytoalexins of mature maize plants. These recent discoveries raise many questions including: what exogenous and endogenous signals strongly promote maize phytoalexin biosynthesis? What are the roles of these terpenoids in plant protection? plant signaling? rhizosphere interactions? allelopathy? autotoxicity? and human health? Given that approximately 150,000 square of maize is annually planted in the U.S. and that fungal attack is aggressive during senescence it is of interest to know, where do these compounds go? what is their fate in the environment? (A) Maize plants must defend against significant biotic threats that include fungal pathogens [(B) southern leaf blight, Cochliobolus heterostrophus ; (C) stalk rot, Fusarium graminearum ; (D) ear rot, Aspergillus flavus ) and stem boring lepidoptera [(E) European corn borer: Ostrinia nubilalis ]. All of these stimuli result in localized zealexin and kauralexin production to limit microbial spread. Mass spectrometry-based metabolomic profiling of phytohormones and defenses is a powerful tool to aid in the discovery of novel metabolites that additionally serve as signals and plant protectants. We utilize biochemical discovery approaches based on the coupling of separations with rapid activity based bioassays, NMR structure elucidation, mass spectrometry for the plant metabolomic analyses of novel induced defenses and signals, genetic resources to test candidate pathways, global expression analyses for co-regulated processes, expression quantitative trait loci (eQTL) for gene discovery, transient and heterologous expression of enzymes for product analyses, and other tools to uncover previously hidden pathways and mechanisms underlying plant-mediated interactions with their biotic and abiotic environment.  Select Publications Christensen SA, Huffaker A, Kaplan F, Sims J, Ziemann S, Doehlemann G, Ji L, Schmitz RJ, Kolomiets MV, Alborn HT, Mori N, Jander G Ni X, Sartor RC, Byers S, Abdo Z, Schmelz EA (2015) Maize death acids, 9-lipoxygenase-derived cyclopente(a)nones, display activity as cytotoxic phytoalexins and transcriptional mediators. Proc. Natl. Acad. Sci. USA 112:11407-11412 Christensen SA, Huffaker A, Hunter CT, Alborn HT, Schmelz EA (2015) A maize death acid, 10-oxo-11-phytoenoic acid, is the predominant cyclopentenone signal present during multiple stress an developmental conditions. Plant Signal Behav. 15:0 [Epub ahead of print] Yan J, Lipka AE, Schmelz EA, Buckler ES, Jander G (2015) Accumulation of 5-hydroxynorvaline in maize (Zea mays) leaves is induced by insect feeding and abiotic stress. J. Exp. Bot. 66(2): 593602 Vaughan MM, Christensen S, Schmelz EA, Huffaker A, McAuslane HJ, Alborn HT Romero M, Allen L H, and Teal PEA (2015) Accumulation of terpenoid phytoalexins in maize roots is associated with drought tolerance. Plant Cell Environ. 38(11): 21952207 Schmelz EA (2015) Impacts of insect oral secretions on defoliation-induced plant defense. Current Opinion in Insect Science. 9:715 Tzin V, Fernandez-Pozo N, Richter A, Schmelz EA, Schoettner M, Schfer M, Ahern KR, Meihls LN, Kaur H, Huffaker A, Mori N, Degenhardt J, Mueller LA, Jander G (2015) Dynamic Maize Responses to Aphid Feeding Are Revealed by a Time Series of Transcriptomic and Metabolomic Assays. Plant Physiol. 169:1727-1743 Martins V, Vaughan MM, Huffaker A, Schmelz EA, Christensen S, Sims J, Benda ND, Swerbilow J, Alborn H, Teal PE (2014) Seed treatment with live or dead Fusarium verticillioides equivalently reduces the severity of subsequent stalk rot. Journal of Phytopathology 162(3): 201204 Christensen S, Nemchenko A, Park Y-S, Schmelz E, Kunze S, Feussner I, Yalpani N, Meeley R, Kolomiets M (2014) The novel maize 9-lipoxygenase, ZmLOX12, is required to mount an effective defense against Fusarium verticillioides. Mol. Plant. Microbe Interact. 27(11): 1263-76. Vu HS, Roth MR, Tamura P, Samarakoon T, Shiva S, Honey S, Lowe K, Schmelz EA, Williams TD, Welti R (2014). Head-group acylation of monogalactosyldiacylglycerol is a common stress response, and the acyl-galactose acyl composition varies with the plant species and applied stress. Physiol. Plant. 150(4): 517528. Schmelz EA, Huffaker A, Sims J, Christensen S, Okada K, Peters RJ (2014) Biosynthesis, elicitation and roles of monocot terpenoid phytoalexins. Plant J. 79(4): 659-678 Shen J, Tieman D, Jones JB, Taylor MG, Schmelz E, Huffaker A, Bies D, Chen K, Klee HJ (2014) A 13-lipoxygenase, TomloxC, is essential for synthesis of C5 flavour volatiles in tomato. J Exp. Bot. 65(2): 419-428 Ni X, Toews MD, Buntin GD, Carpenter JE, Huffaker A, Schmelz EA, Cottrell TE, Abdo Z (2014) Influence of brown stink bug feeding, planting date and sampling time on common smut infection of maize. Insect Sci. 21(5): 564-571 Vaughan MM, Huffaker A, Schmelz EA, Dafoe NJ, Christensen S, Sims J, Martins VF, Swerbilow J, Romero M, Alborn HT, Allen LH, Teal PE (2014) Effects of elevated [CO2] on maize defence against mycotoxigenic Fusarium verticillioides. Plant Cell Environ. 37(12): 26912706 Murakami S, Nakata R, Aboshi T, Yoshinaga N, Teraishi M, Okumoto Y, Ishihara A, Morisaka H, Huffaker A, Schmelz EA, Mori N (2014) Insect-induced daidzein, formononetin and their conjugates in soybean leaves. Metabolites, 4(3): 532-546 Dafoe NJ, Thomas JD, Shirk PD, Legaspi ME, Vaughan MM, Huffaker A, Teal PE, and Schmelz EA (2013) European corn borer (Ostrinia nubilalis) induced responses enhance susceptibility in maize. PLoS ONE 8(9): e73394 Huffaker A, Pearce G, Veyrat N, Erb M, Turlings TCJ, Sartor R, Shen Z, Briggs SP, Vaughan MM, Alborn HT, Teal PEA, Schmelz, EA (2013) Plant Elicitor Peptides are Conserved Signals Regulating Direct and Indirect Anti-Herbivore Defense. Proc. Natl. Acad. Sci. USA 110:5707-12 Anderson RG, Casady MS, Fee RA, Vaughan MM, Deb D, Fedkenheuer K, Huffaker A, Schmelz, EA, Tyler, BM and McDowell, JM (2012) Homologous RXLR effectors from Hyaloperonospora arabidopsidis and Phytophthora sojae suppress immunity in distantly related plants. Plant Journal 72, 882893 Schmelz EA, Huffaker A, Carroll MJ, Alborn HT, Ali JG, Teal PE (2012) An amino acid substitution inhibits specialist herbivore production of an antagonist effector and recovers insect-induced plant defenses. Plant Physiology 160: 1468-1478 Dafoe NJ, Huffaker A, Vaughan MM, Duehl AJ, Teal PE, Schmelz EA (2011) Rapidly induced chemical defenses in maize stems and their effects on short-term growth of Ostrinia nubilalis. J. Chem. Ecol. 37:984-991 Schmelz EA, Kaplan F, Huffaker A, Dafoe NJ, Vaughan MM, Ni X, Rocca JR, Alborn HT, Teal PEA (2011) Identity, regulation, and activity of inducible diterpenoid phytoalexins in maize. Proc. Natl. Acad. Sci. USA 108: 5455-5460 Huffaker A, Kaplan F, Vaughan MV, Dafoe NJ, Ni X, Rocca JR, Alborn HT, Teal PEA, Schmelz EA (2011) Novel acidic sesquiterpenoids constitute a dominant class of pathogen-induced phytoalexins in maize. Plant Physiol. 156: 2082-2097 Acosta IF, Laparra H, Romero SP, Schmelz E, Hamberg M, Mottinger JP, Moreno MA, Dellaporta SL (2009) \"tasselseed1 is a lipoxygenase affecting jasmonic acid signaling in sex determination of maize\" Science 323(5911): 262-265 Schmelz EA, Engelberth J, Alborn HT, Tumlinson JH, Teal PEA (2009) Phytohormone-based activity mapping of insect herbivore-produced elicitors. Proc. Natl. Acad. Sci. USA 106: 653-657 Schmelz EA, LeClere S, Carroll MJ, Alborn HT, Teal PEA (2007) Cowpea chloroplastic ATP synthase is the source of multiple plant defense elicitors during insect herbivory. Plant Physiol.144: 793-805 Alborn HT, Hansen TV, Jones TH, Bennett DC, Tumlinson JH, Schmelz EA, Teal PEA (2007) Disulfooxy fatty acids from the American bird grasshopper Schistocerca americana, elicitors of plant volatiles. Proc. Natl. Acad. Sci. USA 104: 12976-12981 Schmelz EA, Carroll MJ, LeClere S, Phipps SM, Meredith J, Chourey PS, Alborn HT, Teal PEA (2006) Fragments of ATP synthase mediate plant perception of insect attack. Proc. Natl. Acad. Sci. USA 103: 8894-8899 \n",
            "-------------------- Scraping faculty url 151/183 --------------------\n",
            "julian schroeder  Biography Julian Schroeder did his PhD research at the Max Planck Institute for Biophysical Chemistry with Erwin Neher and was a von Humboldt postdoctoral fellow at UCLA School of Medicine. He received awards, including the Presidential Young Investigator Award (NSF), the ASPB Charles Albert Shull Award (1997), a DFG Heinz-Maier-Leibnitz Prize, the Blasker Award in Environmental Science, is Churchill Overseas Fellow at Cambridge University and with collaborators shared the Cozzarelli Prize from PNAS (2010) and a top 10 breakthrough of the year selected by Science (2009). He has served on several advisory boards, including Co-Director of the Food and Fuel for the 21st Century Center. He was von Humboldt Fellow at the MPI for Biochemistry, visiting Professor at the ETH Zurich and is a member of the U.S. National Academy of Sciences, a Fellow of AAAS and the German National Academy of Sciences Leopoldina.  Research Identifying the basic molecular mechanisms by which plants respond to and mount resistance to environmental stresses is fundamental to understanding stress resistance mechanisms to these abiotic in plants and is an important goal for developing future strategies for engineering stress resistance in plants. Several abiotic stress mechanisms that we are characterizing are directly linked to water, including drought stress-induced signal transduction mechanisms, salinity resistance mechanisms and how plants respond to the continuing rise in the atmospheric CO 2 concentration. Our laboratorys research is directed at the signal transduction mechanisms and pathways that mediate resistance to environmental (abiotic) stresses in plants, in particular responses to elevated CO 2 , drought, salinity stress, and heavy metal stress. These abiotic stresses have substantial negative impacts and reduce global plant growth and biomass production. These environmental stresses are also relevant in reference to climate change and to expanding available arable land to meet the food and energy needs of the growing human population. Our research is elucidating the molecular and cell biological stress-induced signal transduction cascades in higher plant cells, examining the chain of events by which plant cells respond to elevated CO 2 , the drought stress hormone abscisic acid and salinity stress to mount specific resistance and adaptation responses. We have developed and adapted interdisciplinary and systems biological approaches to guard cells, which control water loss and CO 2 intake in plants and which have become a key model system for understanding dynamic cellular signal transduction and ion channel functions in plants. Stomatal pores in the epidermis of leaves allow CO 2 influx into leaves from the atmosphere and also mediate transpirational water loss of plants (see figure). Two guard cells surround each pore and control the opening and closing of stomata. In guard cells, cell biological, molecular, patch clamp and time-resolved calcium imaging studies on genetic signaling mutants in Arabidopsis are allowing us to identify and characterize stress-induced signal transduction mechanisms and cascades. We are combining these analyses with new genomic, systems, bioinformatic and proteomic approaches towards discovering new signaling mechanisms and principles. We have identified CO 2 binding proteins and early CO 2 signal transduction mechanisms, including ion channels in guard cells through which elevated CO 2 closes stomatal pores. We have recently identified new early signal transduction mechanisms and contributed to the characterization and co-identified receptors for the plant stress hormone abscisic acid and have obtained molecular genetic, cell biological, genomic, biophysical whole plant physiological evidence for new genes and mechanisms in guard cells that reduce water loss of Arabidopsis during drought. A second effort in the lab focuses on identifying genes that mediate salt (sodium) stress resistance and heavy metal uptake and detoxification in plants. In this research we identified the plant HKT transporter family and showed its central role in mediating salinity resistance in the reference plant, Arabidopsis thaliana . Research on the staple crops rice and wheat is showing that this same HKT transporter mechanism plays a major role in determining salinity resistance. HKT gene-focused molecular breeding efforts are indicating major improvements in yield, illustrating how basic Arabidopsis research is leading to innovation in agriculture. Our research into heavy metal stress led to the parallel discovery of the genes encoding the central heavy metal detoxification enzymes in plants, phytochelatin synthases. Furthermore recent research has identified the long sought family of transporters that mediate heavy metal accumulation in plant vacuoles. These basic research advances can provide key tools for avoiding toxic heavy metal and arsenic accumulation in edible plant tissues, a problem facing millions of people today leading to cancer and other diseases. Furthermore, these basic research advances can contribute key tools for engineering plants for environmental remediation (bioremediation) by removal of heavy metals from soils. Members in our lab are being trained in interdisciplinary and systems biological techniques while pursuing individual research projects.  Select Publications Zhang J, Wang N, Miao Y, Hauser F, McCammon JA, Rappel WJ, Schroeder JI. Identification of SLAC1 anion channel residues required for CO2/bicarbonate sensing and regulation of stomatal movements. Proc Natl Acad Sci U S A. 115(44):11129-11137 doi: 10.1073/pnas.1807624115. (2018). PMC6217375 Li Z, Takahashi Y, Scavo A, Brandt B, Nguyen D, Rieu P, Schroeder JI. Abscisic acid-induced degradation of Arabidopsis guanine nucleotide exchange factor requires calcium-dependent protein kinases. Proc Natl Acad Sci USA. 115(19):E4522-E4531. doi: 10.1073/pnas.1719659115 (2018). PMC5948973. Hauser F, Li Z, Waadt R, Schroeder JI. SnapShot: Abscisic Acid Signaling. Cell 171(7): 1708-1708 doi: 10.1016/j.cell.2017.11.045. PMID: 29245015 (2017). Li Z, Waadt R, Schroeder JI. Release of GTP Exchange Factor Mediated Down-Regulation of Abscisic Acid Signal Transduction through ABA-Induced Rapid Degradation of RopGEFs. PLoS Biol. May 18;14(5):e1002461. doi: 10.1371/journal.pbio.1002461. eCollection (2016). Stephan A. B., Kunz H. H., Yang E. & Schroeder J.I. Rapid hyperosmotic-induced Ca2+ responses in Arabidopsis thaliana exhibit sensory potentiation and involvement of plastidial KEA transporters. Proc Natl Acad Sci USA pii: 201519555 (2016). Wang C, Hu H, Qin X, Zeise B, Xu D, Rappel WJ, Boron WF, Schroeder JI. Reconstitution of CO2 regulation of SLAC1 anion channel and function of CO2-permeable PIP2;1 aquaporin as carbonic anhydrase 4 interactor. Plant Cell. (2016). Brandt B, Munemasa S, Wang C, Nguyen D, Yong T, Yang PG, Poretsky E, Belknap TF, Waadt R, Alemn F, Schroeder JI. Calcium specificity signaling mechanisms in abscisic acid signal transduction in Arabidopsis guard cells. Elife. Jul 29;4. doi: 10.7554/eLife.10328. (2015). Engineer, C. B., Ghassemian, M., Anderson, J., Hu, H., Peck, S. & Schroeder, J. I. Carbonic anhydrases, EPF2 and a novel protease mediate CO2 control of stomatal development. Nature 513: 246-250 (2014). Waadt, R., Hitomi, K., Nishimura, N., Hitomi, C., Adams, S. R., Getzoff, E. D. & Schroeder, J. I. FRET-based reporters for the direct visualization of abscisic acid concentration changes and distribution in Arabidopsis. eLife 3:e01739 doi: 10.7554/eLife.01739 (2014). Kunz, H. H., Gierth, M., Herdean, A., Cruz, M., Kramer, D.M., Spetea, C. & Schroeder, J. I. Plastidial Transporters KEA1, 2 and 3 are essential for chloroplast osmoregulation, integrity and pH regulation in Arabidopsis. Proc Natl Acad Sci USA 111: 7480-7485 (2014). Jones, A.M., Xuan, Y., Xu, M., Wang, R.-S., Ho, C.-H., Lalonde, S., You, C.H., Sardi, M.I., Parsa, S.A., Smith-Valle, E., Su, T., Frazer, K.A., Pilot, G., Pratelli, R., Grossmann, G., Acharya, B.R., Hu, H.-C., Engineer, C., Villiers, F., Ju, C., Takeda, K., Su, Z., Dong, Q., Assmann, S.M., Chen, J., Kwak, J.M., Schroeder, J.I., Albert, P., Rhee, S.Y., Frommer, W.B. Border control a membrane-linked interactome of Arabidopsis. Science 344: 711-716. (2014). Hauser F, Chen W, Deinlein U, Chang K, Ossowski S, Fitz J, Hannon GJ, Schroeder JI. A Genomic-Scale Artificial MicroRNA Library as a Tool to Investigate the Functionally Redundant Gene Space in Arabidopsis. Plant Cell 25: 2848-2863 (2013). Schroeder, J.I., Delhaize, E., Frommer, W.B., Guerinot, M.L., Harrison, M.J., Herrera-Estrella,L., Horie, T., Kochian, L.V., Munns, R., Nishizawa, N.K., Tsay, Y.F. & Sanders, D. Using Membrane Transporters to Improve Crops for Sustainable Food Production. Nature 497: 60-66 (2013). Brandt, B., Brodsky, D.E., Xue, S., Negi, J., Iba, K., Kangasjrvi, J., Ghassemian, M., Stephan, A.B., Hu, H., Schroeder, J.I. Reconstitution of abscisic acid activation of SLAC1 anion channel by CPK6 and OST1 kinases and branched ABI1 PP2C phosphatase action. Proc Natl Acad Sci USA 109:10593-10598 (2012). Kim, T.H., Kunz, H.H., Bhattacharjee, S., Hauser, F., Park, J., Engineer, C., Liu, A., Ha, T., Parker, J.E., Gassmann, W., Schroeder, J.I. Natural Variation in Small Molecule-Induced TIR-NB-LRR Signaling Induces Root Growth Arrest via EDS1- and PAD4-Complexed R Protein VICTR in Arabidopsis. Plant Cell 24: 5177-5192 (2012). Kim, T.H., Hauser, F., Ha T., Xue S., Bhmer M., Nishimura N., Munemasa S., Hubbard K., Peine N., Lee B.H., Lee S., Robert N., Parker J.E., Schroeder J.I. Chemical genetics reveals negative regulation of abscisic Acid signaling by a plant immune response pathway. Current Biology 21: 990-997 (2011). Xue, S., Hu, H., Ries, A., Merilo, E., Kollist, H. & Schroeder, J.I. Central functions of bicarbonate in S-type anion channel activation and OST1 protein kinase in CO2 signal transduction in guard cell. EMBO J. 30: 1645-1658 (2011). Hauser, F., Waadt, R & Schroeder, J.I. Evolution of abscisic acid synthesis and signaling mechanisms. Current Biology 21: R346-355 (2011). Song, W.Y., Park, J., Mendoza-Czatl, D.G., Suter-Grotemeyer, M., Shim, D., Hrtensteiner, S., Geisler, M., Weder, B., Rea, P.A., Rentsch, D., Schroeder, J.I., Lee, Y., Martinoia, E. Arsenic tolerance in Arabidopsis is mediated by two ABCC-type phytochelatin transporters. Proc Natl Acad Sci USA 107:21187-21192 (2010). Hubbard, K., Nishimura, N., Hitomi, K., Getzoff, E.D. and Schroeder, J.I. Early Abscisic Acid Signal Transduction Mechanisms. Genes & Development 24: 1695-1708 (2010). Hu., H., Boisson-Dernier, A., Israelsson -Nordstrm, M., Bhmer, M., Xue, S., Ries, A., Godoski, J., Kuhn, J.M. & Schroeder, J.I. Carbonic Anhydrases are Upstream Regulators of CO2-controlled Stomatal Movements in Guard Cells. Nature Cell Biol. 12: 87-93 (2010). Mendoza-Cozatl, D.G., Zhai, Z., Jobe, T.O., Akmakjian, G.Z., Song, W.Y., Limbo, O., Russell, M.R., Kozlovskyy, V.I., Martinoia, E., Vatamaniuk, O.K., Russell, P., Schroeder, J.I. Tonoplast-localized Abc2 Transporter Mediates Phytochelatin Accumulation in Vacuoles and Confers Cadmium Tolerance. J. Biol. Chem. (2010). Kim, T.H., Bhmer, M., Hu, H., Nishimura, N. & Schroeder, J.I. Guard cell signal transduction network; advances in understanding abscisic acid, CO2, and Ca2+ signaling Annual Reviews of Plant Biol. 61: 561-91 (2010). Nishimura, N., Hitomi K, Arvai A.S., Rambo R.P., Hitomi C., Cutler S.R., Schroeder J.I., Getzoff E.D. Structural mechanism of abscisic acid binding and signaling by dimeric PYR1. Science 326:1356-7 (2009). Nishimura N., Sarkeshik A., Nito K., Park S.Y., Wang A., Carvalho P.C., Lee S., Caddell D.F., Cutler S.R., Chory J., Yates J.R., Schroeder J.I. PYR/PYL/RCAR family members identified as major in-vivo ABI1 protein phosphatase 2C-interacting proteins in Arabidopsis. Plant Journal 61: 290-299 (2010). Park, S.Y., Fung, P., Nishimura, N., Jensen, D.R., Fujii, H., Zhao, Y., Lumba, S., Santiago, J., Rodrigues, A., Chow, T.F., Alfred, S.E., Bonetta, D., Finkelstein, R., Provart, N.J., Desveaux, D., Rodriguez, P.L., McCourt, P., Zhu, J.K., Schroeder, J.I., Volkman, B.F., Cutler, S.R. Abscisic acid inhibits type 2C protein phosphatases via the PYR/PYL family of START proteins. Science 324: 1068-1071 (2009). Siegel, R.S., Xue, S., Murata, Y., Yang, Y., Nishimura, N., Wang, A. & Schroeder, J.I. Calcium elevation-dependent and attenuated resting calcium-dependent abscisic acid induction of stomatal closure and abscisic acid-induced enhancement of calcium sensitivities of S-type anion and inward-rectifying K channels in Arabidopsis guard cells. Plant Journal 59(2): 207-20 (2009). Ward, J.M., Mser, P., Schroeder, J.I. Plant Ion Channels: Gene Families, Physiology, and Functional Genomics Analyses. Annual Reviews of Physiology 71 (2009). Vahisalu, T., Kollist, H., Wang, Y.-F., Nishimura, N., Chan, W.-Y., Valerio, G., Lamminmaki, A., Brosche, M., Moldau, H., Desikan, R., Schroeder, J.I., Kangasjarvi, J. SLAC1 is required for plant guard cell S-type anion channel function in stomatal signalling Nature 452:487-491 (2008). Boisson-Dernier, A., Frietsch, S., Kim, T.H., Dizon, M.B. and Schroeder, J.I. The peroxin ABSTINENCE BY MUTUAL CONSENT disrupts male-female gametophyte recognition. Current Biology 18:63-68 (2008). Imaizumi, T., Kay, S., Schroeder, J.I. Daily watch on Metabolism. Science 318:1730-1731 (2007). Horie, T., Costa, A., Kim, TH, Han, MH, Horie, R., Leung, HY, Miyao, A., Hirochika, H., An, G. and Schroeder, J.I. Rice Os HKT2;1 transporter mediates large Na+ influx component into K+-starved roots for growth. EMBO J. 26:3003-3014 (2007). Suh, S.J., Wang, Y.F., Frelet, A., Leonhardt, N., Klein, M., Forestier, C., Mueller-Roeber, B., Cho, M.H., Martinoia, E., Schroeder, J.I. The ATP Binding Cassette Transporter AtMRP5 Modulates Anion and Calcium Channel Activities in Arabidopsis Guard Cells. J Biol Chem 282:1916-1924 (2007). Mori, I.C., Murata, Y., Yang, Y., Munemasa, S., Wang, Y.F., Andreoli, S., Tiriac, H., Alonso, J.M., Harper, J.F., Ecker, J.R., Kwak, J.M., Schroeder, J.I. (2006). CDPKs CPK6 and CPK3 Function in ABA Regulation of Guard Cell S-Type Anion- and Ca(2+)- Permeable Channels and Stomatal Closure. PloS Biol. 4. Schroeder, J.I., Nambara, E. (2006). A quick release mechanism for abscisic acid. Cell 126:1023-1025. Young, J.J., Mehta, S., Israelsson, M., Godoski, J., Grill, E., Schroeder, J.I. (2006). CO2 signaling in guard cells: Calcium sensitivity response modulation, a Ca 2+ --independent phase, and CO2 insensitivity of the gca2 mutant. Proc. Natl. Acad. Sci. USA 103:7506-7511. Schroeder, J.I. (2006). Physiology: nitrate at the ion exchange. Nature 442:877-878. Hashimoto, M., Negi, J., Young, J., Israelsson, M., Schroeder, J.I., Iba, K. (2006). Arabidopsis HT1 kinase controls stomatal movements in response to CO 2 . Nature Cell Biol. 8:391-397. Colcombet, J., Boisson-Dernier, A., Ros-Palau, R., Vera, C.E., Schroeder, J.I. (2005). Arabidopsis somatic embryogenesis receptor kinases 1 and 2 are essential for tapetum development and microspore maturation. Plant Cell 17:3350-3361. Leonhardt, N., Kwak, J. M., Robert, N., Waner, D., Leonhardt, G. & Schroeder, J. I. (2004). Microarray expression analyses of Arabidopsis guard cells and isolation of a recessive ABA hypersensitive Protein Phosphatase 2C mutant. Plant Cell 16:596-615. Gong, J.I., Waner, D.A., Horie, T., Li, S.L., Horie, R., Abid, K.B. and J.I. Schroeder (2004). Microarray-based rapid cloning of an ion accumulation deletion mutant in Arabidopsis thaliana . Proc. Natl. Acad. Sci. USA 101:15404-15409. Kwak, J.M., I.C. Mori, Z-M. Pei, N. Leonhardt, M.A. Torres, J.L. Dangl, R.E. Bloom, S. Bodde, J.D.G. Jones and J.I. Schroeder. (2003). NADPH oxidase AtrbohD and AtrbohF genes function in ROS-dependent ABA signaling in Arabidopsis . EMBO J. 22:2623-2633. Gong, J-M., D.A. Lee and J.I. Schroeder. (2003). Long-distance root-to-shoot transport of phytochelatins and cadmium in Arabidopsis . Proc. Natl. Acad. Sci. USA 100:10118-10123. Lahner, B, J. Gong, M. Mahmoudian, E.L. Smith, K.B. Abid, E.E. Rogers, M.L. Guerinot, J.F. Harper, J.M. Ward, L. McIntyre, J.I. Schroeder and D.E. Salt. (2003). Genomic scale profiling of nutrient and trace elements in Arabidopsis thaliana . Nature Biotechnology 21:1215-1221. Kwak, J.M., J-H. Moon, Y. Murata, K. Kuchitsu, N. Leonhardt, A. DeLong and J.I. Schroeder. (2002). Disruption of a guard cell-expressed protein phosphatase 2A regulatory subunit, RCN1, confers abscisic acid insensitivity in Arabidopsis . Plant Cell 14:2849-2861. Mser, P., Y. Hosoo, S. Goshima, T. Horie, B. Eckelman, K. Yamada, K. Yoshida, E.P. Bakker, A. Shinmyo, Soiki, J.I. Schroeder, and N. Uozumi (2002). Glycine residues in potassium channel-like selectivity filters determine potassium selectivity in four-loop-per-subunit HKT transporters from plants. Proc. Nat. Acad. Sci. USA 99:6428-6433. Hugouvieux, V., J.M. Kwak and J.I. Schroeder. (2001). A mRNA cap binding protein, ABH1, modulates early abscisic acid signal transduction in Arabidopsis . Cell 106:477-487. Allen, G.J., S.P. Chu, C.L. Harrington, K. Schumacher, T. Hoffmann, Y.Y. Tang, E. Grill and J.I. Schroeder. (2001). A defined range of guard cell calcium oscillation parameters encodes stomatal movements. Nature 411:1053-1057. Schroeder, J.I., J.M. Kwak. and G.J. Allen. (2001). Guard cell abscisic acid signalling and engineering of drought hardiness in plants. Nature 410:327-330. Allen, G.J., Chu, S.P., Schumacher, K., Shimazaki, C.T., Vafeados, D., Kemper, A., Hawke, S.D., Tallman, G., Tsien, R.Y., Harper, J.F., Chory, J. and Schroeder, J.I. (2000). Alteration of Stimulus Specific Guard Cell Calcium Oscillations and Stomatal Closing in Arabidopsis det3 Mutant. Science 289:2338-2342. Pei, Z-M., Murata, Y., Benning, G., Thomine, S., Klsener, B., Allen, G., Grill, E., and Schroeder, J.I. (2000). Calcium channels activated by hydrogen peroxide mediate abscisic acid signalling in guard cells. Nature 406:731-734. Clemens, S., E.J. Kim, D. Neumann and J.I. Schroeder. (1999). Tolerance to toxic metals by a gene family of phytochelatin synthases from plants and yeast. EMBO J. 18:3325-3333. Clemens, S., Antosiewicz, D.M., Ward, J.M., Schachtman, D.P. and Schroeder, J.I. (1998) The plant cDNA LCT1 mediates the uptake of calcium and cadmium in yeast. Proc. Natl. Acad. Sci. USA 95:12043-12048. Pei, Z-M., Ghassemian, M., Kwak, C.M., McCourt, P. and Schroeder, J.I. (1998) Role of farnesyltransferase in ABA regulation of guard cell anion channels and plant water loss. Science 282:287-290. Hildebrand, M., Volcani, B.E., Gassmann, W. and Schroeder, J.I. (1997) A gene family of silicon transporters. Nature 385:688-689. Pei, Z.M., Kuchitsu, K., Ward, J.M., Schwarz, M. and Schroeder, J.I. (1997) Differential abscisic acid regulation of guard cell slow anion channels in Arabidopsis wild-type and abi1 and abi2 mutants. Plant Cell 9:409-423. Pei, Z.M., Ward, J.M., Harper, J.F. and Schroeder, J.I. (1996) A novel chloride channel in Vicia faba guard cell vacuoles activated by the serine threonine kinase CDPK. EMBO J. 15:6564-6574. Rubio, F., Gassmann, W. and Schroeder, J.I. (1995). Sodium-driven potassium uptake by the plant potassium transporter HKT1 and mutations conferring salt tolerance. Science 270:1660-1663. Schachtman, D.P. and Schroeder, J.I. (1994). Structure and transport mechanism of a high-affinity K+ uptake transporter from higher plants. Nature 370:655-658. \n",
            "-------------------- Scraping faculty url 152/183 --------------------\n",
            "terrence sejnowski  Biography Dr. Sejnowski received his Ph.D. in physics from Princeton University. He was a postdoctoral fellow at Princeton University and the Harvard Medical School. He served on the faculty of Johns Hopkins University and was a Wiersma Visiting Professor of Neurobiology and a Sherman Fairchild Distinguished Scholar at Caltech.  Research The goal of my laboratory is to discover the principles linking brain mechanisms and behavior. The laboratory uses both experimental and modeling techniques to study the biophysical properties of synapses and neurons and the population dynamics of large networks of neurons. New computational models and new analytical tools have been developed to understand how the brain represents the world and how new representations are formed through learning algorithms for changing the synaptic strengths of connections between neurons. Some of the questions that my laboratory is exploring include: How do neurons communicate at synapses? Monte Carlo models of molecular diffusion and interactions at synapses are being used to understand the mechanisms underlying the release of neurotransmitter from presynaptic terminals, the activation of receptors on the postsynaptic neuron and subsequent biochemical pathways that lead to short-term and long-term changes in the synapse. Single molecules at synapses are probed with 3-D random access 2-photon microscopy to locate important molecules and follow them as they diffuse within a spine. What regulates the flow of information into the cerebral cortex and between cortical areas? My laboratory has shown that cortical neurons are capable of initiating spikes precisely and reliably. We have used detailed models of neurons to explore the role of spike time synchrony in reliably transmitting information between populations of neurons and in attentional modulation of firing rates. Fast-spiking inhibitory neurons are studied in vitro to examine their role in synchronizing other neurons. EEG and fMRI recordings have been analyzed with Independent Component Analysis (ICA), an algorithm for blind source separation developed in my lab, to explore the global flow of information in the cortex. Why do we sleep? As we fall asleep, the patterns of cortical activity become more spatially and temporally coherent, especially at low frequencies, and during the night the brain cycles between REM sleep and slow wave sleep. The mechanisms underlying these sleep rhythms are being explored with large-scale network models of neural populations. New algorithms have been found to automatically classify sleep states from single EEG recordings.  Select Publications Qian, N. and Sejnowski, T. J., (1988). Predicting the secondary structure of globular proteins using neural network models. Journal of Molecular Biology 202:865-884. Steriade, M., D.A. McCormick and T.J. Sejnowski (1993). Thalamocortical oscillations in the sleeping and aroused brain. Science 262:679-685. Mainen, Z.F. and T.J. Sejnowski (1995). Reliability of spike timing in neocortical neurons. Science 268:1503-1506. Bell, A.J. and T.J. Sejnowski (1995). An information-maximization approach to blind separation and blind deconvolution. Neural Computation 7:1129-1159. Montague, P.R., P. Dayan and T.J. Sejnowski (1996). A framework for mesencephalic dopamine systems based on predictive Hebbian learning. Journal of Neuroscience 16(5):1936-1947. McKeown, M.J., S. Makeig, G.G. Brown, T.-P. Jung, S. Kindermann,A.J. Bell and T.J. Sejnowski (1998). Analysis of fMRI by blind separation into independent spatial components. Human Brain Mapping 6:1-31. Makeig. S., M. Westerfield,T.-P. Jung, S. Enghoff, J. Townsend,E. Courchesne and T.J. Sejnowski (2002). Dynamic brain sources of visual evoked responses. Science 295:690-694. Coggan, J. S., T.M. Bartol, E.I. Esquenazi, J.R. Stiles, S. Lamont, M.E. Martone, D.K. Berg, M. Ellisman and T.J. Sejnowski (2005). Evidence for ectopic neurotransmission at a neuronal synapse. Science 39:446-451. Behrens, M.M. and T.J. Sejnowsk (2009). Does schizophrenia arise from oxidative dysregulation of parvalbumin-interneurons in the developing cortex? Neuropharmacology 57:193-200. Wang, H.P., D. Spencer, J.-M. Fellous and T.J. Sejnowski(2010). Synchrony of thalamocortical inputs maximizes cortical reliability. Science 328:106-109. \n",
            "-------------------- Scraping faculty url 153/183 --------------------\n",
            "gerald shadel \n",
            "-------------------- Scraping faculty url 154/183 --------------------\n",
            "reuben shaw \n",
            "-------------------- Scraping faculty url 155/183 --------------------\n",
            "jonathan shurin  Biography Dr. Shurin studies the causes and consequences of variation in species diversity, the flow of energy between producers and consumers, and the impacts of predators on the functioning of ecosystems.  Research My lab studies the connections between population, community and ecosystem ecology in aquatic environments. We are particularly interested in the flow of energy between trophic levels, the impact of consumers on their resources, and the control of species diversity. We use broad geographic comparisons, mesocosm experiments and theory to address these questions. My students have worked in communities of freshwater plankton, marine fishes and intertidal algae and invertebrates.  Select Publications Symons, C.C. and Shurin, J.B., 2016, June. Climate constrains lake community and ecosystem responses to introduced predators. In Proc. R. Soc. B (Vol. 283, No. 1832, p. 20160825). The Royal Society. Markel, R.W.* and J.B. Shurin. 2015. Indirect effects of sea otters on rockfish (Sebastes spp.) in giant kelp forests. Ecology 96: 2877-2890. Hong, B.C.* and J.B. Shurin. 2015. Latitudinal variation in the response of tidepool copepods to mean and daily range in temperature. Ecology 96: 23482359. Clasen, J.L. and J.B. Shurin. 2015. Kelp forest size alters microbial community structure and function on Vancouver Island, Canada. Ecology96: 862-872. Shurin, J.B., R.L. Abbott, M.S. Deal, G.T. Kwan*, E. Litchman, R. McBride, S. Mandal, V.H. Smith. 2013. Industrial-strength ecology: Tradeoffs and opportunities in algal biofuel production. Ecology Letters 16: 1393-1404. Atwood, T.B.*, E. Hammill, H.S. Greig, P. Kratina, J.B. Shurin, D.S. Srivastava and J.S. Richardson. 2013. Predators reduce CO2 emissions from freshwater ecosystems. Nature Geoscience 6: 191-194. Featured in Science News, New Scientist and Faculty of 1000. \n",
            "-------------------- Scraping faculty url 156/183 --------------------\n",
            "doug smith  Biography Doug Smith received his Ph.D. from Stanford University and was a postdoctoral fellow at the Max-Planck-Institut fr Virusforshung in Tbingen. He was named Outstanding Educator of America in 1973.  Research Our studies of DNA replication center on understanding initiation of rounds of replication in procaryotic cells: the structural requirements for the origin of replication (oriC), the role of GATC methylation state in initiation, and the mechanisms of control of initiation. We have recently shown that Dam methylation is required for precise timing of initiation of rounds of DNA replication; current emphasis is on the mechanism involved. Other related projects include delineation of control circuits regulated by the DnaA protein and state of GATC methylation and studies of gram-negative origins from bacteria which have no GATC methylation. We are also engaged in projects concerned with genome (large and small) mapping and sequencing, and concomitant computer database handling and sequence analysis.  Select Publications Zyskind, J.W. and Smith, D.W. (1992). DNA Replication, the bacterial cell cycle, and cell growth. Cell 69 :5-8. Smith, D.W., Yee, T.W., Baird, C. and Krishnapillai, V. (1991). Pseudomonad replication origins: a paradigm for bacterial origins? Mol. Microbiol. 5: 2581-2587. Seely, O., Jr., Feng, D.-F., Smith, D.W., Sulzbach, D. and Doolittle, R.F. (1990). Construction of a facsimile-data set for large genome sequence analysis. Genomics 8: 71-82. Loomis, W.F. and Smith, D.W. (1990). Molecular phylogeny of dictyostelium-discoideum by protein sequence comparison. Proc. Natl. Acad. Sci. USA 87 :9093-9097. \n",
            "-------------------- Scraping faculty url 157/183 --------------------\n",
            "laurie smith  Biography After receiving her Ph.D. from University College, London, Laurie Smith conducted postdoctoral study at Stanford University and the UC Berkeley/USDA Plant Gene Expression Center.  Research The shapes and positions of plant cells are fixed by cell walls. Consequently, plant cells must be formed during development in the positions where they are needed, requiring exquisite spatial regulation of the cell cycle and cell division planes. We are studying mechanisms responsible for orienting division planes in plant cells. Much of our past work on division plane determination has focused on TANGLED, a protein that is localized at the division plane throughout prophase, mitosis and cytokinesis and functions to promote proper division plane orientation (Figure 1; Walker et al., 2007). Recent studies on TAN have elucidated requirements for its localization throughout the cell cycle, revealing that a series of distinct mechanisms involving different parts of TAN protein function to maintain TAN at the division site as cells progress through division (Figure 1; Rasmussen et al., 2011). These studies have shown that division site maintenance is plant cells is a more dynamic process than previously envisioned. Figure 1. Schematic of cytokinesis in a somatic plant cell. TANGLED protein (blue) co-localizes with the preprophase band (PPB), predicting the future division plane during prophase. Unlike the PPB, TAN remains at the cortical division site throughout mitosis and cytokinesis, where it functions to promote proper placement of the new cell wall. Maintenance of TAN at the division site requires different parts of TAN protein and different interacting factors at different stages of the cell cycle. Our current work focuses on orientation of asymmetric cell divisions. In plants, as in other eukaryotes, asymmetric divisions are associated with pattern formation during embryogenesis, establishment of new cell lineages, and the formation of specialized cell types. In all of these processes, developmental asymmetry is closely tied to division polarity, but little is known about polarizing cues that orient asymmetric divisions in plants, or how cells respond to them to establish asymmetric division planes. Our studies in this area have focused on the formation of subsidiary cells of maize stomata, which are created by asymmetric divisions of subsidiary mother cells (SMCs, Figure 2). We have discovered a cooperatively acting pair of receptor-like proteins, PAN1 and PAN2, that are implicated in transmission or amplification of a cue from adjacent guard mother cells that polarizes the asymmetric divisions of SMCs (Cartwright et al., 2009 and unpublished work on PAN2). ROP GTPases act downstream of PAN1 to polarize SMC divisions (Humphries et al., 2011). Ongoing work utilizes proteomic and molecular genetic approaches to identify additional components of the SMC division polarity pathway. Figure 2. Stomatal development in wild type and pan mutant maize. All stages are illustrated schematically; Toluidine Blue O-stained mature epidermis is also shown for wild type (top) and pan mutant (bottom). Subsidiary cells form via asymmetric divisions of subsidiary mother cells (SMCs), which are polarized by a cue from the adjacent guard mother cell (GMC). PANs function together with ROP GTPases to polarize premitotic SMCs.  Select Publications Facette, M.R., Park, Y., Sutimantanapi, D., Luo, A., Cartwright, H.N., Yang, B., Bennett, E., Sylvester, A.W., and Smith, L.G. (2015). The SCAR/WAVE complex polarizes PAN receptors and promotes division asymmetry in maize. Nature Plants 1:article no 14024. doi:10.1038/nplants.2014.24. Sutimantanapi, D., Pater, D., and Smith, L.G. (2014). Divergent roles for maize PAN1 and PAN2 receptor-like proteins in cytokinesis and cell morphogenesis. Plant Physiology, 164:1905-1917. Facette, M.R., Shen, X., Bjornsdottir, F.R., Briggs, S.P., and Smith, L.G. (2013). Parallel proteomic and phosphoproteomic analyses of successive stages of maize leaf development. Plant Cell, 25:2798-2812. Facette, M., and Smith, L.G. (2012). Division polarity in developing stomata. Curr. Op. Plant Biol., 15:585-592. Zhang, X., Facette, M., Humphries, J.A., Shen, X., Park, Y., Sutimantanapi, D., Sylvester, A.W., Briggs, S.P. and Smith, L.G. (2012). Identification of PAN2 by quantitative proteomics as a LRR-RLK acting upstream of PAN1 to polarize cell division in maize. Plant Cell, 24:4577-4589. Rasmussen, C.G., Sun, B., and Smith, L.G. (2011). TANGLED localization at the cortical division site of plant cells occurs by several mechanisms. J. Cell Sci. 124:270-279. Rasmussen, C.G., Humphries, J.A., and Smith, L.G. (2011). Determination of symmetric and asymmetric division planes in plant cells. Annu. Rev. Plant Biol. 62:387-409. Humphries, J.A, Vejlupkova, Z., Luo, A., Meeley, R.B., Sylvester, A.W., Fowler, J.E., and Smith, L.G. (2011). ROP GTPases act with the receptor-like protein PAN1 to polarize asymmetric cell division in maize. Plant Cell, 23:2273-2284. \n",
            "-------------------- Scraping faculty url 158/183 --------------------\n",
            "nicholas spitzer  Biography Nick Spitzer received his Ph.D. from Harvard University and was a postdoctoral fellow at Harvard and University College, London. He joined the faculty in 1972 and has been the recipient of a Sloan Fellowship, a Javits Neuroscience Investigator Award and a Guggenheim Fellowship. He was founding editor-in-chief of BrainFacts.org , and is a fellow of the American Association for the Advancement of Science, a member of the American Academy of Arts and Sciences and the National Academy of Sciences and co-director of the Kavli Institute for Brain and Mind.  Research The brain is capable of a rich variety of forms of plasticity, changing its structure and function in response to changes in the environment. We study neurotransmitter switching, a newly appreciated form of neuroplasticity in which neurons change the transmitters that they make and release in response to sustained sensory or motor activity. Transmitter identity shifts from excitatory to inhibitory or vice versa, with matching changes in postsynaptic receptors, both in the developing amphibian nervous system and in the adult rodent brain. Perhaps unsurprisingly the reversal of the sign of synaptic transmission is accompanied by changes in the animals behavior. Our work is focused on understanding the cues that stimulate neurotransmitter switching in the developing and adult mouse brain, the prevalence and forms of this plasticity, and its functions. We want to learn the molecular mechanisms that regulate neurotransmitter switching and the mechanisms by which the appropriate, matching transmitter receptors are expressed on postsynaptic cells. We are investigating neurotransmitter switching in response to normal physiological stimuli and to aversive stimuli such as stress that lead to neurological disorders like depression and schizophrenia. Our recent studies have demonstrated the roles of sensory stimuli and direct manipulations of activity in neurotransmitter switching in the developing Xenopus hypothalamus, brainstem, and accessory olfactory bulb (Dulcis & Spitzer, 2008; Demarque & Spitzer, 2010; Dulcis et al., in review) and in the adult rat hypothalamus (Dulcis et al. 2013) that change specific behaviors. Ongoing projects address the role of motor activity in neurotransmitter switching in the adult mouse midbrain and hippocampus that may regulate procedural learning and neurogenesis. We are also investigating neurotransmitter switching generated by environmental mouse models of autism spectrum disorders.  Select Publications Spitzer, N.C. (2017) Neurotransmitter switching in the developing and adult brain. Ann. Rev. Neurosci. 40: 1-19. Dulcis, D., Lippi, G., Stark, C.J., Do, L.H., Berg, D.K., Spitzer, N.C. (2017) Neurotransmitter switching regulated by miRNAs controls changes in social preference. Neuron. 95: 1319-1333. Aumann, T.D., Raabus, M., Tomas, D., Prijanto, A., Churilov, L., Spitzer, N.C., Horne, M.K. (2016) Differences in number of midbrain dopamine neurons associated with summer and winter photoperiods in humans. PLoS One 11: e0158847. Spitzer, N.C. (2015) Neurotransmitter switching? No surprise. Neuron 86: 1131-1144. Gemez-Gamboa, A. Xu, L., Meng, D., Spitzer, N.C. (2014) Non-cell-autonomous mechanism of activity-dependent neurotransmitter switching. Neuron 82: 1004-1016. Dulcis, D., Jamshidi, P., Leutgeb, S. and Spitzer, N.C. (2013) Neurotransmitter switching in the adult brain regulates behavior. Science 340: 449-453. Plazas, P.V., Nicol, X. and Spitzer, N.C. (2013) Activity-dependent competition regulates motor neuron axon pathfinding via PlexinA3. Proc. Nat. Acad. Sci. USA. 110: 1524-1529. Spitzer, N.C. (2012) Activity-dependent neurotransmitter respecification. Nature Reviews Neurosci. 13: 94-106. Dulcis, D. and Spitzer, N.C. (2012) Reserve pool neuron transmitter respecification: novel neuroplasticity. Dev. Neurobiol. 72: 465-474. Demarque, M. and Spitzer, N.C. (2012) Neurotransmitter phenotype plasticity: an unexpected mechanism in the toolbox of network activity homeostasis. Dev. Neurobiol. 72: 22-32. Nicol, X., Hong, K.P. and Spitzer, N.C. (2011) Spatial and temporal second messenger codes for growth cone turning. Proc. Nat. Acad. Sci. USA. 108: 13776-13781. Velzquez-Ulloa, N.A., Spitzer, N.C. and Dulcis, D. (2011) Context-dependent dopamine specification by calcium activity across the central nervous system. J. Neurosci. 31: 78-88. Demarque, M. and Spitzer, N.C. (2010) Activity-dependent expression of Lmx1b regulates specification of serotonergic neurons modulating swimming behavior. Neuron 67: 321-334. Marek, K.W., Kurtz, L.M. and Spitzer, N.C. (2010) cJun integrates calcium spike activity and tlx3 expression to regulate neurotransmitter specification. Nature Neurosci. 13: 944-950. Xiao, Q., Xu, L. and Spitzer, N.C. (2010) Muscle-dependent regulation of neurotransmitter specification and embryonic neuronal calcium spike activity. J. Neurosci. 30: 5792-5801. Chang, L.W. and Spitzer, N.C. (2009) Spontaneous calcium spike activity in embryonic spinal neurons is regulated by developmental expression of the Na+, K+-ATPase 3 subunit. J. Neurosci. 29: 7877-7885. Dulcis, D. and Spitzer, N.C. (2008) Illumination controls dopaminergic differentiation regulating behavior. Nature 456: 195-201. Root, C.M., Velazquez-Ulloa, N.A., Monsalve, G.C., Minakova, E. and Spitzer, N.C. (2008). Embryonically expressed GABA and glutamate drive electrical activity regulating neurotransmitter specification. J. Neurosci. 28: 4777-4784. Sann, S.B., Xu, L., Nishimune, H., Sanes, J.R. and Spitzer, N.C. (2008). Neurite outgrowth and in vivo sensory innervation mediated by a CaV2.2-laminin beta-2 stop signal. J. Neurosci. 28: 2366-2374. Borodinsky, L.N. and Spitzer, N.C. (2007). Activity-dependent neurotransmitter-receptor matching at the neuromuscular junction. Proc. Natl. Acad. Sci. 104: 335-340. Spitzer, N.C. (2006). Electrical activity in early neuronal development. Nature 444: 707-712. Conklin, M.W., Lin, M.S. and Spitzer, N.C. (2005). Local calcium transients contribute to disappearance of pFAK, focal complex removal and deadhesion of neuronal growth cones and fibroblasts. Dev. Biol. 287: 201-212. Borodinsky, L.N., Root, C.M., Cronin, J.A., Sann, S.B., Gu, X. and Spitzer, N.C. (2004). Actiity-dependent homeostatic specification of transmitter expression in embryonic neurons. Nature 429: 523-530. Gorbunova, Y.V. and Spitzer, N.C. (2002). Dynamic interactions of cAMP transients and spontaneous calcium spikes. Nature 418: 93-96. Gomez, T.M., Robles, E., Poo, M. and Spitzer, N.C. (2001). Filipodial calcium transients promote substrate-dependent growth cone turning. Science 291: 1983-1987. \n",
            "-------------------- Scraping faculty url 159/183 --------------------\n",
            "sarah stockwell  Research My background is in evolutionary systems biology, investigating biological questions that span the disciplines of genetics, molecular biology, computational modeling, applied mathematics, and evolutionary biology. It is an inherently interdisciplinary field, which requires communication and collaboration with scientists having different backgrounds, training, research culture, and professional language. I particularly enjoy this kind of science because it offers the opportunity to explore diverse ideas from disparate lines of investigation, with all of them connected by a rigorous reliance on data. I approach teaching in the same spirit. One of the most rewarding aspects of teaching undergraduates is the opportunity to take them on a tour through what we have learned about biology and to help them catch my enthusiasm about the beauty of those ideas. I enjoy the challenges of communicating with students across the potential divides of different life experiences, assumptions, and languages to help their learning. And, just as with my biological research, I base my teaching on data: on what my own experiences and assessments and the research of others has shown to be effective. I am particularly interested in designing and assessing curricular elements that address common student misconceptions about evolutionary processes. Ladder-like thinking and a teleological, deterministic view of evolutionary change emerge naturally from our human tendency to organize the world into stories. In order to encourage students to reconstruct their understanding of the world to incorporate tree thinking and random processes, we need inviting curriculum that engages students of all backgrounds and experiences.  Select Publications SR Stockwell, CR Landry, and SA Rifkin (2014). The yeast galactose network as a quantitative model for cellular memory. Molecular BioSystems. DOI: 10.1039/C4MB00448E. SR Stockwell, RN Gutenkunst, CR Myers, and AG Clark. Predicting the mutational robustness of gene regulatory networks. (In revision.) SR Stockwell.Sound Trees, a curriculum unit for the Cornell Lab of Ornithologys Online Research in Biology program. AJ Greenberg, SR Stockwell, and AG Clark (2008). Evolutionary constraint and adaptation in the metabolic network of Drosophila. Molecular Biology and Evolution 25(12):2537-2546. \n",
            "-------------------- Scraping faculty url 160/183 --------------------\n",
            "chih-ying su  Biography Chih-Ying Su received her Ph.D. in Neuroscience from Johns Hopkins University School of Medicine and carried out her postdoctoral research with Dr. John Carlson at Yale University. Chih-Ying has been the recipient of a Mette Strand Young Investigator Award.  Research The sense of smell is one of the most intriguing senses: animals detect and discriminate an almost limitless number of odors in order to find food and mates while avoiding predators. Our research is focused on addressing how olfactory receptor neurons (ORNs) process odor information because these neurons are the primary source of sensory input. We are especially interested in how compartmentalization allows insect ORNs to process information through ephaptic interaction. We also investigate how neuromodulation can act on ORNs to regulate responses and influence odor-guided behaviors in Drosophila . To understand the functional significance of ORN compartmentalization, we recently developed a new recording paradigm to demonstrate that ephaptic interaction is sufficient to mediate lateral inhibition between ORNs housed in the same sensillum ( Zhang et al., 2019 ). This unconventional lateral inhibition has a profound impact on neuronal spike activity and can thus regulate information processing at the earliest stages of olfactory coding. Furthermore, combining electrophysiology with a novel method that permits high-quality ultrastructural preservation for 3D electron microscopy of genetically labeled tissues ( Tsang et al, 2018 ), we found that lateral inhibition is predominantly asymmetric between compartmentalized ORNs, and that this feature arises from the size difference between the neurons ( Zhang et al., 2019 ). Our study indicates that not all ORNs are equal in their circuit function, and that genetically predetermined morphometric differences between compartmentalized ORNs underlie their functional disparity. Our recent studies also identified the courtship-promoting Or47b ORNs as a neural substrate for a fly reproductive hormone, named juvenile hormone. We showed that the sensitivity of Or47b ORNs to an aphrodisiac pheromone is low in immature young males but high in older males at peak fertility ( Lin et al., 2016 ). This age-dependent sensitization permits coordination of courtship activity with reproductive maturity, thereby providing a means to increase male reproductive fitness. Building on these findings, we further identified a signal amplification mechanism by which olfactory responses can be amplified in insect ORNs in response to juvenile hormone signaling ( Ng et al., 2019 ). Surprisingly, this amplification mechanism only applies to courtship-promoting ORNsO47b and Ir84a neuronsamong the 50 ORN types in the antenna. Together, our finding reveals the surprising complexity of insect olfactory transduction: not all ORNs expressing receptors of the same class employ identical transduction mechanisms. Rather, a common transduction channel can function downstream of two distinct classes of receptors.  Select Publications Ng, R, Salem, SS, Wu, S-T, Lin, H-H, Wu, M, Shepherd, AK, Joiner, WJ, Wang, JW, Su, C-Y (2019). Amplification of Drosophila olfactory responses by a calcium-gated DEG/ENaC channel. Neuron , in press. Zhang, Y # , Tsang TK # , Bushong, EA, Chu, L-A, Chiang, A-S, Ellisman, MH, Reingruber, J, Su, C-Y (2019). Asymmetric ephaptic inhibition between compartmentalized olfactory receptor neurons. Nature Communications >, (10):1560 ( # equal contribution). PMID: 30952860 Tsang, TK # , Bushong, EA # , Boassa, D, Hu, J, Romoli, B, Phan, S, Dulcis, D, Su, C-Y * , Ellisman, MH* (2018). High-quality ultrastructural preservation using cryofixation for 3D electron microscopy of genetically labeled tissues. eLife e35524.( # equal contribution; *corresponding authors). PMID: 29749931 Ng, R, Lin, H-H, Wang, JW, Su C-Y (2017). Electrophysiological recording from Drosophila trichoid sensilla in response to odorants of low volatility. J. Vis. Exp. (125):e56147. PMID: 28784950 Kim, SM, Su, C-Y * , Wang, JW * (2017). Neuromodulation of innate behaviors in Drosophila, Annu. Rev. Neurosci 30, 327-348. ( * corresponding authors). PMID: 28441115 Lin, H-H # , Cao, D-S # , Sethi, S, Zeng, Z, Chin, JS, Chakraborty, TS, Shepherd, AK, Nguyen, CA, Yew, JY, Su, C-Y * , Wang, JW * (2016). Hormonal modulation of pheromone detection enhances male courtship success, Neuron 90, 1272-1285 ( # equal contribution; * corresponding authors). PMID: 27263969 Su, C-Y, Menuz, K, Reisert, J and Carlson, JR (2012). Non-synaptic inhibition between grouped neurons in an olfactory circuit. Nature 492, 66-71. PMID: 23172146 Su, C-Y, Martelli, C, Emonet, T. and Carlson, JR (2011). Temporal coding of odor mixtures in a receptor neuron. Proc. Natl. Acad. Sci. USA 108, 5075-5080. PMID: 21383179 Su, C-Y * , Menuz, K * and Carlson, JR (2009). Olfactory perception: receptors, cells, and circuits. Cell 139, 45-59. ( * equal contribution) PMID: 19804753 \n",
            "-------------------- Scraping faculty url 161/183 --------------------\n",
            "suresh subramani  Biography Suresh Subramani received his Ph.D. from the University California (UC), Berkeley. He joined the faculty at UC San Diego in 1982, after completion of post-doctoral work in the laboratory of Nobel Laureate, Paul Berg, at Stanford University (1979-81). He is the recipient of many honors and awards and has been involved in many leadership roles in academia and in the biotechnology industry. His service to UC includes his role as the Executive Vice Chancellor (2010-16), Assoc. Vice Chancellor (2009-10), Interim Dean (2006-07) and Chair (1999-2000). He is currently a Distinguished Professor (Emeritus), the Global Director of the Tata Institute for Genetics and Society (TIGS, at UC San Diego, USA and at inStem, Bangalore, India) and holds the Tata Chancellors Endowed Professorship in Molecular Biology at UC San Diego.  Research PEROXISOME HOMEOSTASIS IN YEAST AND HUMANS We are interested in the problem of organelle homeostasis, using peroxisomes as the model organelle. Like all other subcellular organelles, peroxisomes and peroxisomal functions are indispensable for human survival. Our studies on peroxisome homeostasis examine how peroxisomes are assembled (biogenesis) and destroyed (turnover or pexophagy), as well as the coordination of these two processes within cells. We are also interested in the role of peroxisome biogenesis and turnover in disease states. Peroxisome biogenesis: Work done in this and other labs has uncovered about 36 peroxins, encoded by PEX genes, involved in peroxisome biogenesis. Many proteins involved in this process are conserved in evolution from yeast to man, and inactivating mutations in at least half of these peroxins cause fatal human disorders (Zellweger syndrome, rhizomelic chondrodysplasia punctata and infantile Refsum disease). Work on peroxisome biogenesis is summarized in a review (Farr et al., EMBO Reports, 2018, PMID: 30530632). Current work focuses on the trafficking of peroxisomal membrane proteins via the endoplasmic reticulum, their budding into pre-peroxisomal vesicles (ppV) and the proteins and mechanisms involved in this ppV budding. Peroxisome degradation: Autophagy is a process in which cells eat themselves, especially under starvation conditions, and recycle cellular building blocks such as amino acids, lipids and sugars. Autophagy plays a key role in development, aging, neurodegeneration, cell death, cell survival, and innate immunity. While autophagy generally degrades cargoes non-selectively, it can also be adapted to degrade and recycle selective cargoes, including organelles and protein aggregates. Pexophagy is a turnover pathway in which redundant, damaged or unnecessary peroxisomes are selectively degraded by the autophagy machinery in response to specific environmental cues. Other selective autophagy pathways include mitophagy, ribophagy, ER-phagy, micronucleophagy and the Cvt pathway in yeasts. Progress in the field of selective autophagy may be found in a review (Farr et al., Nature Rev. Mol. Cell Biol., 2016, PMC5549613). Current work focuses on the selective autophagy pathways and the manner in which selective autophagy receptors function.  Select Publications Agrawal, G., Fassas, S. N., Xia, Z., Subramani, S. Distinct requirements for intra-ER sorting and budding of peroxisomal membrane proteins from the endoplasmic reticulum. J. Cell Biol., 212: 335-48 (2016). PMCID: PMC4788575. Zientara-Rytter, K., Subramani, S. Autophagic degradation of peroxisomes in mammals. Biochem. Soc. Trans. 44, 431440.(2016). PMCID: PMC4958620. Farr, J.C., Subramani, S. Mechanistic insights regarding selective autophagy pathways: lessons from yeast. Nature Reviews Mol. Cell Biol., 17: 537552. (2016). PMC: PMC5549613. Zientara-Rytter, K., Subramani, S. Role of actin in shaping autophagosomes. Autophagy, 12: 2512-2515 (2016). PMCID: PMC5173263. Kramer, M.H., Farr, J.C., Mitra, K., Yu, M.K., Ono, K., Demchak, B., Licon,, K., Flagg, M., Balakrishnan, R., Cherry, J.M., Subramani, S., Ideker, T. Active Interaction Mapping reveals the hierarchical organization of autophagy. Mol Cell,Evolving 65: 761774. (2017). PMCID: PMC5439305. Wang, W., Subramani, S. Assays to monitor pexophagy in yeast. In Meth. Enzymology. 588: 413-427. (2017). PMCID: PMC5546006. Agrawal, G., Shang, H.H., Xia, Z.-J., Subramani, S. Functional regions of the peroxin Pex19 necessary for peroxisome biogenesis. J. Biol. Chem. 292: 1154711560. (2017). PMCID: PMC5500816. Farre, J.-C., Kramer, M., Ideker, T., Subramani, S. Active Interaction Mapping as a tool to elucidate hierarchical functions of biological processes. Autophagy. 13: 12481249. (2017). PMCID: PMC5529073. Wang, W., Xia, Z., Farr, J.C., Subramani, S. TRIM37, a novel E3 ligase for PEX5-mediated peroxisomal matrix protein import. J. Cell Biol. 216: 2843-2858. (2017). PMCID: PMC5584156. Wang. W., Subramani, S. Role of PEX5 ubiquitination in maintaining peroxisome dynamics and homeostasis. Cell Cycle, 6: 2037-2045. (2017). PMCID: PMC5731411. Zientara-Rytter, K., Ozeki, K., Nazarko, T.Y., Subramani, S. Pex3 and Atg37 compete to regulate the interaction between the pexophagy receptor, Atg30, and the Hrr25 kinase. Autophagy, 14: 368-384. (2017). PMID: 29260977, PMC journal in process. Farre, J.-C., Carolino, K., Stasyk, O.V., Stasyk, O. G., Hodzic, Z., Agrawal, G., Till, A., Proietto, M., Cregg, J., Sibirny, A. A., Subramani, S. A new yeast peroxin, Pex36, a functional homologue of mammalian PEX16, functions in the ERto-peroxisome traffic of peroxisomal membrane proteins. J. Mol. Biol. 429: 37433762. (2017). PMCID: PMC5693695. Wang, W., Xia, Z., Farr, J.-C., Subramani, S. TRIM37 deficiency induces autophagy through de-regulating the MTORC1-TFEB axis. Autophagy, 14: 1574-1585. (2018). PMID: 29940807, PMC journal in process. Zientara-Rytter, K., Subramani, S. AIM/LIR-based fluorescent sensors new tools to monitor mAtg8 functions. Autophagy, 4: 1074-1078. (2018). PMID: 27068951, PMC journal in process. \n",
            "-------------------- Scraping faculty url 162/183 --------------------\n",
            "gurol suel  Biography Dr. Sel received his PhD in Molecular Biophysics from the University of Texas, Southwestern Medical Center in the laboratory of Dr. Rama Ranganathan and conducted his postdoctoral studies in the laboratory of Dr Michael Elowitz in the Department of Biology and Applied Physics at the California Institute of Technology.  Research Our laboratory discovered ion channel-mediated electrical signaling among bacteria that reside in biofilm communities. We are investigating how these action potentials in biofilms give rise to emergent collective behaviors in biofilms and beyond. Accordingly, we integrate quantitative biology approaches with mathematical modeling and theory to identify principles of bacterial organization and coordination.  Select Publications Species-independent attraction to biofilms through electrical signaling Humphries J, Xiong L, Liu L, Prindle L, Yuan F, Tsimring L and Sel GM Cell, (2017) vol 168, num 1 Ion channels enable electrical communication in bacterial communities Prindle A., Liu J., Asally M., Ly S., Garcia-Ojalvo J., Sel G.M. Nature, (2015), vol 527, num 7576 Metabolic codependence gives rise to collective oscillations within biofilms. Liu J., Prindle A,. Humphries J., Gabalda-Sagarra M., Asally M., Lee D.D., Ly S., Garcia-Ojalvo J., Sel, G.M. Nature (2015) vol 523, num 7562 Chromosomal locations of phosphorelay genes controls the Bacillus subtilis response to starvation. Narula J.*, Kuchina A.*, Lee D., Fujita M., Sel G.M. and Igoshin O.A. Cell (2015) vol 163, num 2 Localized cell death focuses mechanical forces during 3D patterning in a biofilm Asally M., Kittisopikul M., Ru P., Du Y., Hu Z., aatay T., Robinson A.B., Lu H., Garcia-Ojalvo J., and Sel G.M. Proc Natl Acad Sci U S A., (2012), vol 109, num 46 Identification of F-actin as a dynamic hub in a microbial-induced GTPase polarity circuit Orchard RC, Kittisopikul M, Altschuler SJ, Wu LF, Sel GM, Alto NL. Cell (2012), vol 148, num 4 Architecture-dependent noise discriminates functionally analogous differentiation circuits aatay T., Turcotte M., Elowitz M.B., Garcia-Ojalvo J. and Sel G.M. Cell (2009), vol 139, num 3 \n",
            "-------------------- Scraping faculty url 163/183 --------------------\n",
            "xin sun  Biography Xin Sun received her Ph.D. from the Department of Biology at Yale University and postdoctoral training with Dr. Gail Martin at UCSF. She was on the faculty in the Genetics Department at University of Wisconsin-Madison from 2002 to 2016, when she joined the UCSD faculty. Dr. Sun was the recipient of a Burroughs-Wellcome career award, a March of Dimes Basil OConner award, and Romnes Faculty Fellowship by Wisconsin Alumni Research Foundation.  Research The building of an organ starts from an amorphous cluster of cells, and proceeds following an elaborate blueprint of coordinated proliferation, morphogenesis and differentiation. Organ function relies on the harmonization with other tissues, the infusion by blood and immune cells, and the wiring by the nervous system. We employ cutting-edge technologies, including single-cell transcriptomics and epigenomics, genome editing, whole-organ clearing and imaging, to delineate the robustness that dictates normal organ function, and uncover the imperfections that initiate pathogenesis. At the center of our investigation is the lung, an organ that is vital at first breath. It is the setting for many fetal, pediatric and adult diseases with unknown cause and no cure. We use disease modeling as a starting point for fundamental discoveries in developmental biology, stem cell biology and the neural-immune-physiology interface. Consider the lung as a sensory organ A mouse airway outlined by an epithelial marker for club cells (green), annotated by clusters of pulmonary neuroendocrine cells (magenta) which act as airway sensors. At resting, an average person inhales/exhales 5-8 litters of air. This air may vary in oxygen content, carry allergen, pollutants or pathogens. Each of these signals would trigger a specific set of responses. For example, allergen causes asthma while intermittent hypoxia causes pulmonary hypertension. How this exquisite specificity is established is poorly understood. We recently found that pulmonary neuroendocrine cells, which represents less than 1% of epithelial cells in the lung, function as sensors on the airway wall (Branchfield et al., Science , 2016) . These cells are neuronally innervated, and act through secreting potent neuropeptides, neurotransmitters and amines. We found that overactivation of these cells leads to heightened baseline immune responses. Furthermore, they form neural-immune niches with residential immune cells such as innate lymphoid group 2 cells (Sui et al., Science , 2018) . Together, these niches are essential for mediating allergen-induced asthmatic responses. Study of the pulmonary neuroendocrine cells have also led us to investigate the molecular nature of the neural circuit that controls lung function. We are teaming up with neurobiologists, genomicists and physiologists to map the neural circuit that originates from and returns to the lung. Our goals are to understand which aspects of lung function is controlled by the neural circuit, what underlie the specificity between inputs and outputs, and how to manipulate the circuit to restore lung function in disease settings. Mechanisms and long-term outcomes of prematurity In a normal adult, the gas-exchange surface area is estimated to be ~1,000ft2, equivalent to half of a tennis court. This surface is packed into our chest in the form of ~480 millions of gas-exchange units called alveoli. Approximately 95% of the gas-exchange surface area is built in the early periods after birth in a process called alveologenesis. Disruption of the final steps of lung development results in alveoli simplification, as is seen in a large proportion of premature infants diagnosed with bronchopulmonary dysplasia (BPD). BPD is often associated with lifelong breathing deficiencies, pulmonary hypertension and accelerated decline in respiratory capacity later in life. To date, a majority of studies of alveologenesis rely on two-dimensional (2D) analysis of tissue sections. Given that an overarching theme of alveologenesis is thinning and extension of the epithelium and mesenchyme to facilitate gas exchange, often only a small portion of a cell or a cellular structure is represented in a single 2D plane. We have used a three-dimensional (3D) approach to examine the structural architecture and cellular composition of myofibroblasts, lipofibroblasts, alveolar type 2 cells and elastin extracellular matrix in normal as well as BPD-like mouse lungs (Branchfield et al., Dev Biol , 2015) . The insights revealed by 3D reconstruction of the alveoli set the foundation for future investigations of the mechanisms driving alveologenesis, as well as causes of alveolar simplification in BPD. We have also used a genetic approach to investigate the role of signaling pathways in this process (Li et al, Development, 2017; Li et al., Elife , 2018) . Ongoing work in this research direction interrogates how different elements of prematurity impacts long-term lung function. We are interested in why some BPD patients develop pulmonary hypertension while others do not, and why BPD patients are more susceptible to viral infections later in life. Genetic modeling of lung diseases As whole-genome sequencing of patients becomes routine, we have established fruitful collaborations with human geneticists, and are using advanced technologies such as CRISPR/Cas9 genome editing to interrogate causal relationships between patient-specific genomic variants and phenotypes and disease phenotypes. Towards this goal, we started with rare congenital disorders, and are also extending our effort to study common and complex diseases. In the past few years, among lung-related congenital disorders, we have studied tracheo-esophageal fistula (Domyan et al., Development 2011) , tracheobronchomalacia (Hines et al., PNAS 2013 ) , and congenital diaphragmatic hernia (CDH) (Domyan et al., Dev Cell , 2013; Branchfield et al., Science , 2016, McCulley et al., JCI, 2018 ) . Using CDH as an example, we have established a pipeline to first determine candidate gene expression in the developing lung (Herriges et al., Dev Dynamics , 2012) , and then use CRISPR/Cas9genome editing and other genetic approaches to recapitulate the mutations in mice, and determine which variants may be causal to phenotypes As a demonstration of the effectiveness of this approach, we have uncovered distinct mechanisms from different genetic models of CDH (Domyan et al., Dev Cell , 2013; Branchfield et al., Science , 2016, McCulley et al., JCI, 2018 ) . These findings provide an explanation for why standard treatment works in some, but not other patients. More importantly, each of the mechanism makes prediction of personalized treatments based on patient genotype. Ongoing work in this research direction aims to expand to additional genes and diseases.  Select Publications Pulmonary neuroendocrine cells amplify allergic asthma responses. Sui P, Wiesner DL, Xu J, Zhang Y, Lee J, Van Dyken S, Lashua A, Yu C, Klein BS, Locksley RM, Deutsch G, Sun X. Science . 2018 8;360(6393). pii: eaan8546. Lats inactivation reveals hippo function in alveolar type I cell differentiation during lung transition to air breathing. Nantie LB, Young RE, Paltzer WG, Zhang Y, Johnson RL, Verheyden JM, Sun X. Development . 2018 Oct 10. pii:dev.163105. doi: 10.1242/dev.163105. Pdgfra marks a cellular lineage with distinct contributions to myofibroblasts in lung maturation and injury response. Li R, Bernau K, Sandbo N, Gu J, Preissl S, Sun X. Elife . 2018 Sep 4;7. pii: e36865. doi: 10.7554/eLife.36865. PBX transcription factor drive pulmonary vascular adaptation to birth. McCulley DJ, Wienhold MD, Hines EA, Hacker TA, Rogers A, Pewowaruk RJ, Zewdu R, Chesler NC, Selleri L, Sun X. Journal of Clinical Investigation , 2018 128(2): 655-667. FGF receptors control alveolar elastogenesis. Li R, Herriges JC, Chen L, Mecham RP, Sun X. Development . 2017, 144(24):4563-4572. TET-mediated DNA demethylation controls gastrulation by regulating Lefty-Nodal signalling.Dai HQ, Wang BA,Chapman DL, Fuchou Tang FC, Sun X*, Xu GL*. Nature , 2016 Oct 538(7626):528-532. *Co-corresponding authors. E3 ubiquitin ligase RFWD2 controls lung branching through protein-level regulation of ETV transcription factors.Zhang Y, Yokoyama S, Herriges JC, Zhang Z, Verheyden JM, Young RE, Sun X. PNAS . 2016 Jul 113(27):7557-62. Pulmonary neuroendocrine cells function as airway sensors to control lung immune response.Branchfield K, Nantie L, Verheyden JM, Sui P, Wienhold MD, Sun X. Science . 2016 Feb, 351:707-10. Perspective of this study was published in Science here. A three-dimensional study of alveologenesis in mouse lung Branchfield K, Li R, Lungova V, Verheyden JM, McCulley D, Sun X. Dev Biol . 2016 Jan 409:429-41. Developmental Biology top outstanding paper award for 2016. FGF-Regulated ETV Transcription Factors Control FGF-SHH Feedback Loop in Lung Branching Herriges JC, Verheyden JM, Zhang Z, Sui P, Zhang Y, Anderson MJ, Swing DA, Zhang Y, Lewandoski M, Sun X. Dev Cell . 2015 Nov 9;35(3):322-32. Establishment of smooth muscle and cartilage juxtaposition in the developing mouse upper airways. Hines, E., Jones, M., Verheyden, J., Harvey, J. and Sun, X. PNAS . 2013 Nov 110(48):19444-9. Roundabout receptors are critical for foregut separation from the body wall Domyan, E.T., Branchfield, K., Gibson, G.A., Naiche, L.A., Lewandoski, M., Tessier-Lavigne, M., Ma, L. and Sun, X. Dev Cell , 2013 Jan 24(1):52-63. Selected as featured article of the issue. Genome-scale study of transcription factor expression in the branching mouse lung Herriges J.C., Yi L., Hines E.A., Harvey J.F., Xu G., Gray P.A., Ma Q., Sun X. Dev Dyn . 2012 Sep 241(9):1432-1453. The microRNA-processing enzyme Dicer is dispensable for somite segmentation but essential for limb bud positioning Zhang Z, O'Rourke JR, McManus MT, Lewandoski M, Harfe BD and Sun X. Dev Dyn . 2011 Mar 351(2), 254-265. Signaling through BMP receptors promotes respiratory identity in the foregut via repression of Sox2 Domyan, E.T., Ferretti, E.,ockmorton, K., Mishina, Y., Nicolis, S.K. and Sun, X. Development . 2011 Mar 138(5): 971-981. Preaxial polydactyly: interactions among ETV, TWIST1 and HAND2 control anterior-posterior patterning of the limb Zhang, Z., Sui, P., Dong, A., Hassell, J., Cserjesi, P., Chen, Y.T., Behringer, R.R. and Sun, X. Development . 2010 Oct 137(20), 3417-3426. Beta-Catenin promotes respiratory progenitor identity in mouse foregut Harris-Johnson, K.S., Domyan, E.T., Vezina, C.M. and Sun, X. PNAS . 2009 Sep 106(38), 16287-92. FGF-regulated Etv genes are essential for repressing Shh expression in mouse limb buds Zhang, Z., Verheyden, J.M. and Sun, X. Dev Cell . 2009 Apr 16(4): 607-613. An Fgf/Gremlin inhibitory feedback loop triggers termination of limb bud outgrowthVerheyden, J.M. and Sun, X. Nature . 2008 Jul 454(7204): 638-641. Dicer function is essential for lung epithelium morphogenesis Harris, K.S., Zhang, Z., McManus, M.T., Harfe, B.D. and Sun, X. PNAS . 2006 Feb 103 (7), 2208-2213. \n",
            "-------------------- Scraping faculty url 164/183 --------------------\n",
            "ella tour  Biography Ella Tour received her Ph.D. in Genetics at the Hebrew University of Jerusalem, where she studied the molecular mechanisms of brain development in the frog Xenopus laevis. She conducted her post-doctoral training at UCSD, investigating the early development of the fruit fly embryos.  Research Education Critical thinking skills, such as the ability to analyze data, solve problems, and develop new ideas, are crucial for success in the modern workforce. Although the majority of biology educators consider critical and creative scientific thinking some of the most important skills to be acquired as a result of science education, few biology faculty report spending enough time practicing these skills in their classrooms (Coil et al., 2010). One of my primary education research goals is to increase our understanding of how to effectively teach critical thinking skills in biology and how objectively test whether students are learning these skills. In my classes, I strive to provide my students with ample opportunities to practice data analysis and experimental design. Currently, my collaborators and I are investigating the effectiveness of in-depth analysis of primary research papers and designing experiments that follow up on these papers, on students' critical thinking skills. An additional area of education research is developing methods to improve the quality of scientific writing and prevent plagiarism in Biology labs. This research is performed in collaboration with Tricia Bertram Gallant (UCSD Office of Academic Integrity), Julian Parris (Psychology), Goran Bozinovic (Biology), and Madeline Picciotto (UCSD Writing Center). Biological In collaboration with the members of Bill McGinnis's lab here at UCSD, I study the role of chromatin structure and long non-coding RNA's in the regulation of gene expression in developing embryos of the fruit fly Drosophila melanogaster . Early embryonic development is a precisely orchestrated process in which genes that control the formation of the various body parts have to be expressed at the right time and the right place. One such master control gene, Ultrabithorax ( Ubx ) is responsible for the correct development of parts of fly's thorax and abdomen. The expression of Ubx is regulated by multiple enhancers that are located tens of thousands base pairs away from the Ubx promoter and its transcription start site. Using high-resolution fluorescent in situ hybridization (FISH), we study the conformational changes involving Ubx promoter and its enhancers that occur in silent and transcriptionally active states . We are also investigating a potential role of an antisense transcript that originates in one of the introns of Ubx in the regulation of this gene. This image shows one actively transcribed Ubx gene in 3 hours-old Drosophila embryo. P (red) Ubx promoter, mRNA (blue) nascent Ubx mRNA originating from the 1st intron of this gene, bx (green) bx enhancer, M (purple) a region located midway between the promoter and bx enhancer. bx enhancer, located 34 kbp downstream of Ubx promoter, appears to be in contact with this promoter.  Select Publications Bertram Gallant T , Picciotto M, Bozinovic G, and Tour E. (2019) Plagiarism or Not? Investigation of TurnitinDetected Similarity Hits in Biology Laboratory Reports. Biochem Mol Biol Educ. doi:10.1002/bmb.21236 Lie R, Abdullah C, He W, Tour E. (2016) Perceived Challenges in Primary Literature in a Masters Class: Effects of Experience and Instruction. CBE Life Science Education Volume 15: ar77. pmid:27909027 Abdullah C, Parris J, Lie, R, Guzdar A, and Tour E (2015) Critical analysis of primary literature in a Masters-level class: effects on self-efficacy and science process skills. CBE-Life Sciences Education , Volume 14, 1-13. Brydges S, Chilukuri L, Cook G, Feeley M, Herbst M, Tour E, and Van Den Einde L* (2012). Developing a Faculty Learning Community at a Large Research University: Interdisciplinary Connections and Competencies in the Scholarship of Teaching and Learning. Currents in Teaching and Learning Spring 2013. Volume 6. Tour E and McGinnis W. (2006) Gap peptides: A new way to control embryonic patterning? Cell , 126(3): 448-9. Tour E, Hittinger CT, and McGinnis W. (2005) Evolutionarily conserved domains required for activation and repression functions of the Drosophila Ubx protein. Development , 132: 5271-5281. Tour E and McGinnis W. (2003) Hox Genes and Evolution of Body Plan. In: Encyclopedia of the Human Genome . Macmillan Publishers Ltd, Nature Publishing Group. Tour E, Pillemer G, Gruenbaum Y, and Fainsod A. (2002) Gbx2 interacts with Otx2 and patterns the anterior-posterior axis during gastrulation in Xenopus . Mech Dev . 112(1-2): 141-51. Tour E, Pillemer G, Gruenbaum Y, and Fainsod A. (2002) Otx2 can activate the isthmic organizer genetic network in the Xenopus embryo. Mech Dev . 110(1-2): 3-13. Tour E, Pillemer G, Gruenbaum Y, and Fainsod A. (2001) The two Xenopus Gbx2 genes exhibit similar, but not identical expression patterns and can affect head formation. FEBS Lett. 507(2):205-9. * Authors are listed in alphabetical order \n",
            "-------------------- Scraping faculty url 165/183 --------------------\n",
            "david traver  Biography David Traver received his Ph.D. in immunology from Stanford University where he received the McDevitt Prize for best thesis. He then completed postdoctoral studies at Harvard University as a fellow of the Irvington Institute. Since joining UCSD in 2004, he has received a Young Investigator Award from the Beckman Foundation, a Scholar Award from the American Society of Hematology and a New Faculty Award from the California Institute for Regenerative Medicine.  Research Utilizing zebrafish for new insights into the biology of hematopoietic stem cells Hematopoietic stem cells (HSCs) exist as rare populations within blood-forming tissues that both self-renew and generate all blood cell lineages for life. HSCs have been isolated to relative purity in humans and mice, leading to a general understanding of their cellular and functional properties. Relatively little is known, however, regarding the genetic program underlying the embryonic formation and subsequent maintenance of HSC fate. The emergence of zebrafish as a model system has introduced unbiased genetic approaches, such as forward genetic screens and high-throughput expression screens, to the study of vertebrate blood cell formation. Fluorescent transgenesis technology combined with the external fertilization and transparency of zebrafish embryos allows early events in HSC specification, maintenance, and differentiation to be easily observed. Collectively, we are utilizing these advantages to image, fate map, and interrogate genetically the earliest embryonic blood-forming cells. This has helped address many long-standing questions in the development of the immune system, including the direct demonstration of HSC emergence from the transdifferentiation of aortic endothelium, and the demonstration of the importance of neighboring tissues in the instruction of HSC fate. We are performing forward genetic and high-throughput expression screens to identify new gene functions in vertebrate stem cells. Novel genes that regulate HSC development, self-renewal, or lineage commitment will subsequently be examined for similar roles in mammalian systems. Despite decades of efforts, it is not yet possible to instruct HSC fate in vitro from pluripotent precursors. We hope that discovery of factors required to specify HSCs in the embryo will ultimately be useful in advancing this important goal of regenerative medicine.  Select Publications Sertori R, Liongue C, Basheer F, Lewis KL, Rasighaemi P, de Coninck D, Traver D, Ward AC. Conserved IL-2Ryc Signaling Mediates Lymphopoiesis in Zebrafish. J Immunol. (2016) 196:135-43. Butko E, Pouget C, Traver D. Complex regulation of HSC emergence by the Notch signaling pathway.Dev Biol. (2016) 409:129-38. Campbell C, Su T, Lau RP, Shah A, Laurie PC, Avalos B, Aggio J, Harris E, Traver D, Stachura DL. Zebrafish embryonic stromal trunk (ZEST) cells support hematopoietic stem and progenitor cell (HSPC) proliferation, survival, and differentiation. Exp Hematol. (2015) 43:1047-61. Kim BJ, Hancock BM, Bermudez A, Del Cid N, Reyes E, van Sorge NM, Lauth X, Smurthwaite CA, Hilton BJ, Stotland A, Banerjee A, Buchanan J, Wolkowicz R, Traver D, Doran KS. Bacterial induction of Snail1 contributes to blood-brain barrier disruption. J Clin Invest. (2015) 125:2473-83. Butko E, Distel M, Pouget C, Weijts B, Kobayashi I, Ng K, Mosimann C, Poulain FE, McPherson A, Ni CW, Stachura DL, Del Cid N, Espn-Palazn R, Lawson ND, Dorsky R, Clements WK, Traver D. Gata2b is a restricted early regulator of hemogenic endothelium in the zebrafish embryo. Development. (2015) 142:1050-61. Sahai-Hernandez P, Traver D. Intimacy of the niche: perivascular remodeling cuddles incoming HSCs.Cell Stem Cell. (2015) 16:109-10. Kim BJ, Hancock BM, Del Cid N, Bermudez A, Traver D, Doran KS. Streptococcus agalactiae infection in zebrafish larvae. Microb Pathog. (2015) 79:57-60. Kim AD, Stachura DL, Traver D. Cell signaling pathways involved in hematopoietic stem cell specification. Exp Cell Res. (2014) 329:227-33. Review. Pouget C, Peterkin T, Simes FC, Lee Y, Traver D, Patient R. FGF signalling restricts haematopoietic stem cell specification via modulation of the BMP pathway. Nature Commun. (2014) 5:5588. Lee Y, Manegold JE, Kim AD, Pouget C, Stachura DL, Clements WK, Traver D. FGF signalling specifies haematopoietic stem cells through its regulation of somitic Notch signalling. Nature Commun. (2014) 5:5583. Espn-Palazn R, Stachura DL, Campbell CA, Garca-Moreno D, Del Cid N, Kim AD, Candel S, Meseguer J, Mulero V, Traver D. Proinflammatory signaling regulates hematopoietic stem cell emergence. Cell. (2014) 159:1070-85. Kim AD, Melick CH, Clements WK, Stachura DL, Distel M, Pankov D, MacRae C, Mork LA, Crump JG, Traver D. Discrete Notch signaling requirements in the specification of hematopoietic stem cells .EMBO J. (2014) 33:2363-73. Ouyang K, Leandro Gomez-Amaro R, Stachura DL, Tang H, Peng X, Fang X, Traver D, Evans SM, Chen J.Loss of IP3R-dependent Ca2+ signalling in thymocytes leads to aberrant development and acute lymphoblastic leukemia. Nature Commun. (2014) 5:4814. Stachura DL, Traver D. Hematopoietic ontogeny in the axolotl. Blood. (2014) 124:1204-6. Kobayashi I, Kobayashi-Sun J, Kim AD, Pouget C, Fujita N, Suda T, Traver D. Jam1a-Jam2a interactions regulate haematopoietic stem cell fate through Notch signalling. Nature. (2014) 512:319-23. Wittamer, V., J.Y. Bertrand, P.W. Gutschow, and D. Traver. 2011. Characterization of the mononuclear phagocyte system in zebrafish. Blood, Epub March 15. Clements, W.C., A.D. Kim, K.G. Ong, J.C. Moore, N.D. Lawson, and D. Traver. 2011. A somitic Wnt16/Notch pathway specifies haematopoietic stem cells. Nature, 474: 220-224. Clements, W.C., and D. Traver. 2013. Signalling pathways that control vertebrate haematopoietic stem cell specification. Nature Reviews Immunology, 13: 336-348. Bertrand, J.Y., N.C. Chi, B. Santoso, S. Teng, D.Y.R. Stainier, and D. Traver. 2010. Hematopoietic stem cells derive directly from aortic endothelium during development. Nature, 464: 108-111. \n",
            "-------------------- Scraping faculty url 166/183 --------------------\n",
            "emily troemel  Biography Emily Troemel received her B.S. from the University of Wisconsin-Madison and then spent a year in Japan before receiving her Ph.D. in Cell Biology at UC-San Francisco. She worked for a start-up biotech company in the Bay Area before returning to academia to do a postdoc at Massachusetts General Hospital and then joined the UCSD faculty in 2008. Since joining UCSD, she has received a Searle Scholars Award, David & Lucile Packard Foundation Award and a Burroughs Wellcome Fund Investigators in the Pathogenesis of Infectious Disease Award.  Research How do pathogens attack their hosts, and how do hosts defend themselves against these attacks? We are using the nematode C. elegans to study these questions of host/pathogen interactions in the intestine. The nematode C. elegans provides a wonderful system in which to study these questions. The worm has intestinal cells that are similar in structure to human intestinal cells, but C. elegans is transparent and easy to study in lab. C. elegans lacks a professional immune system, and instead relies solely on epithelial defense to fight off pathogen attack. It is being increasingly appreciated that epithelial cells in the intestine are involved in detecting and responding to pathogens in mammals, and are implicated in inflammatory disorders. It is critical to understand more about how these cells detect and respond to pathogens. What we find in C. elegans intestinal immunity may be directly applicable to host/pathogen interactions in the human intestine. Microsporidia infecting C. elegans intestine microsporidia are labeled in red, C. elegans intestine labeled in green, and DNA labeled in blue. Our research focuses on two different pathogens. The first pathogen is a natural intracellular pathogen isolated from a wild-caught C. elegans strain, which we have found defines a genus and species of microsporidia (intestinal infection of C. elegans intestine shown below). Microsporidia are poorly understood eukaryotic pathogens that are emerging pathogens of humans and can cause problems for immunocompromised patients. The second pathogen is an extracellular bacterial pathogen called Pseudomonas aeruginosa , which is a clinically relevant pathogen that causes opportunistic infections in humans.  Select Publications Troemel Lab selected publications: Research Articles Intracellular pathogen response pathway promotes proteostasis in C. elegans . Reddy, KC Dror T, Panek J, Chen K, Lim, ES, Wang, D, Troemel, ER bioRxiv 2017. In vivo mapping of tissue- and subcellular-specific proteomes in Caenorhabditis elegans. Reinke AW, Mak R, Troemel ER, Bennett EJ. Sci Adv. 2017 May 10;3(5):e1602426. doi: 10.1126/sciadv.1602426. eCollection 2017 May. Identification of microsporidia host-exposed proteins reveals a repertoire of rapidly evolving proteins. Reinke AW*, Balla, KM, Bennett EJ, Troemel ER. Nature Communications, 2017 Jan 9;8:14023. doi: 10.1038/ncomms14023. *corresponding author. A Large Collection of Novel Nematode-Infecting Microsporidia and Their Diverse Interactions with Caenorhabditis elegans and Other Related Nematodes. Zhang G, Sachse M, Prevost MC, Luallen RJ, Troemel ER, Flix MA. PLoS Pathog. 2016 Dec 12;12(12):e1006093. doi: 10.1371/journal.ppat.1006093. Cell-to-cell spread of microsporidia causes Caenorhabditis elegans organs to form syncytia. Balla KM, Luallen RJ, Bakowski MA, Troemel ER. Nature Microbiology 1, (2016) doi:10.1038/nmicrobiol.2016.144 Microsporidia Intracellular Development Relies on Myc Interaction Network Transcription Factors in the Host. Botts MR, Cohen LB, Probert CS, Wu F, Troemel ER. G3 (Bethesda). 2016 Jul 8. pii: g3.116.029983. doi: 10.1534/g3.116.029983. [Epub ahead of print] PMID: 27402359 Discovery of a Natural Microsporidian Pathogen with a Broad Tissue Tropism in Caenorhabditis elegans. Luallen RJ, Reinke AW, Tong L, Botts MR, Flix MA, Troemel ER. PLoS Pathog. 2016 Jun 30;12(6):e1005724. doi: 10.1371/journal.ppat.1005724. eCollection 2016 Jun. PMID: 27362540 The C. elegans CCAAT-Enhancer-Binding Protein Gamma Is Required for Surveillance Immunity. Reddy KC, Dunbar TL, Nargund AM, Haynes CM, Troemel ER. Cell Rep. 2016 Feb 10. pii: S2211-1247(16)30033-X. doi: 10.1016/j.celrep.2016.01.055. [Epub ahead of print] PMID:26876169 Small GTPases promote actin coat formation on microsporidian pathogens traversing the apical membrane of C. elegans intestinal cells. Szumowski SC, Estes KA, Popovich JJ, Botts MR, Sek G, Troemel ER. Cell Microbiol. 2015 Jul 6. doi: 10.1111/cmi.12481. [Epub ahead of print] PMID:26147591 Balla KM, Andersen EC, Kruglyak L, Troemel ER. A wild C. elegans strain has enhanced epithelial immunity to a natural microsporidian parasite. PLoS Pathogens. 2015 Feb 13;11(2):e1004583. doi: 10.1371/journal.ppat.1004583. PMID:25680197 Szumowski SC, Botts MR, Popovich JJ, Smelkinson MG, Troemel ER. The small GTPase RAB-11 directs polarized exocytosis of the intracellular pathogen N. parisii for fecal-oral transmission from C. elegans. Proc Natl Acad Sci U S A. 2014 May 19. PMCID: PMC4050618 Bakowski MA, Desjardins CA, Smelkinson MG, Dunbar TA, Lopez-Moyado IF, Rifkin SA, Cuomo CA, Troemel ER. Ubiquitin-Mediated Response to Microsporidia and Virus Infection in C. elegans. PLoS Pathog. 2014 Jun 19;10(6):e1004200. PMCID: PMC4063957 Cuomo CA, Desjardins CA, Bakowski MA, Goldberg J, Ma AT, Becnel JJ, Didier ES, Fan L, Heiman DI, Levin JZ, Young S, Zeng Q, Troemel ER. Microsporidian genome analysis reveals evolutionary strategies for obligate intracellular growth. Genome Research 2012 Dec;22(12):2478-88. PMCID: PMC3514677 Dunbar TL, Yan Z, Balla KM, Smelkinson MG, Troemel ER. C. elegans Detects Pathogen-Induced Translational Inhibition to Activate Immune Signaling. Cell Host Microbe. 2012 Apr 19;11(4):375-86. PMCID: PMC3334869 Reviews The Development of Genetic Modification Techniques in Intracellular Parasites and Potential Applications to Microsporidia. Reinke AW, Troemel ER. PLoS Pathog. 2015 Dec 31;11(12):e1005283. doi: 10.1371/journal.ppat.1005283. eCollection 2015 Dec. No abstract available. PMID:26720003 Szumowski SC, Troemel ER. Microsporidia-host interactions. Curr Opin Microbiol. 2015 Apr 3;26:10-16. doi: 10.1016/j.mib.2015.03.006. [Epub ahead of print] Review. PMCID: PMC4577307 Cohen LB, Troemel ER. Microbial pathogenesis and host defense in the nematode C. elegans. Curr Opin Microbiol. Curr Opin Microbiol. 2015 Feb;23:94-101. PMCID: PMC4324121 \n",
            "-------------------- Scraping faculty url 167/183 --------------------\n",
            "kay tye \n",
            "-------------------- Scraping faculty url 168/183 --------------------\n",
            "elizabeth villa  Biography Elizabeth Villa completed her PhD in Biophysics at the University of Illinois at Urbana-Champaign as a Fulbright Fellow. She was a Marie Curie Postdoctoral Fellow in the Max Planck Institute of Biochemistry in Munich. She joined the Chemistry and Biochemistry Department faculty at UC San Diego in July 2014. In 2016, she was awarded an NIH Directors New Innovator Award. She joined the Division of Biological Sciences faculty at UC San Diego in March 2017.  Research Macromolecules rarely operate in isolation inside cells. At any given time, the average protein is part of a complex of over 10 macromolecules, and these supramolecular complexes are in turn embedded in intricate networks inside cells. We are broadly interested in revealing the structure and function of macromolecular complexes in their natural environment at the highest possible resolution in order to reveal their structural dynamics and interactions. We call it bringing structure to cellular biology. We have a strong focus on building tools for quantitative cell biology, using cryo-electron microscopy and tomography, cell biophysics, computational analysis, and integrative modeling. This potent combination allows us to look at macromolecular complexes in their native environment and derive their structure, context, and interaction partners. Our biological focus is on the study of the nuclear periphery, as nuclear biology remains one of the most exciting challenges in the cell, and it is uncharted territory structurally. Our thrust in this area includes projects such as: the structural dynamics of the yeast nuclear pore complex, the mechanical communication between the cytoskeleton and the nucleus, and the molecular architecture of the genome and its association to the nuclear envelope. We also collaborate with different laboratories to open windows into various cellular events. These projects tend to have a translational component, and include studying the inner life of bacteria and studying the effects of LRRK2 in Parkinsons disease, among others.  Select Publications Assembly of a nucleus-like structure during viral replication in bacteria. V. Chaikeeratisak, K. Nguyen, K. Khanna, A.F. Brilot, M.L. Erb, J.K. C. Coker, A. Vavilina, G.L. Newton, R.Buschauer, K.Pogliano, E.Villa, D.A. Agard, J. Pogliano. Science. 355:194-197, 2017. Visualizing the molecular sociology at the HeLa cell nuclear periphery. J. Mahamid, S. Pfeffer, M. Schaffer, E. Villa, R. Danev, L. Kuhn-Cuellar, F. Foerster, A. A. Hyman, J. M. Plitzko and W. Baumeister. Science. 351:969-972, 2016. A photoactivable multi- inhibitor nanoliposome for tumour control and simultaneous inhibition of treatment escape pathways. B. Q. Spring, R. B. Sears, L.Z. Zheng, Z. Mai, R. Watanabe, M. E. Sherwood, D. A. Schoenfeld, B. W. Pogue, S. P. Pereira, E. Villa, and T. Hasan. Nature Nanotechnology, 11:378, 2016. Architecture of the RNA polymerase II-Mediator core transcription initiation complex. C. Plaschka, L. Lariviere, L. Wenzeck, M. Hemann, D. Tegunov, E.V. Petrotchenko, C.H. Borchers, W. Baumeister, F. Herzog, E. Villa, and P. Cramer. Nature, 518, 2014. Native architecture of the Chlamydomonas chloroplast revealed by in situ cryo-electron tomography. B. D. Engel, M. Schaffer, L. K. Cuellar, E. Villa, J.M. Plitzko, W. Baumeister. eLife, 2014. Finding the right fit: chiseling structures out of cryo-electron microscopy maps. E. Villa, K. Lasker. Curr. Opin. Struct. Biol., 25:11825, 2014. Opening windows into the cell: focused-ion-beam milling for cryo-electron tomography. E. Villa, M. Schaffer, J.M. Plitzko and W. Baumeister. Curr. Opin. in Struct. Biol., 23, 771777, 2013. \n",
            "-------------------- Scraping faculty url 169/183 --------------------\n",
            "geoffrey wahl  Research Stem cells in normal and cancerous growth; gene control pathways relevant to normal development and cancer; development of new cancer therapies Dr. Wahl obtained a B.A. in Bacteriology from the University of California, Los Angeles, his PhD in Biological Chemistry from Harvard University, and was a postdoctoral fellow at Stanford University. He joined The Salk Institute and became Professor in the Gene Expression Laboratory in 1989. His research focuses on three important problems related to cancer biology. The first project involves determining relationships between the state of stemness and cancer. This project identified, isolated, and characterized the stem cells generated in the embryonic mammary rudiment and showed that the transcriptomes of these cells share properties with stem-like cells in breast cancer. Recent work has deciphered the epigenomes of normal and cancerous breast cells, showed how they are related, and demonstrated a new mechanism of metastasis involving cell state reprogramming. The second project involves studying protein-protein interactions and their impact on signal transduction. These studies have used a powerful platform for identifying protein interactions in living cells in real time, and have impacted our understanding of the p53, Myc, and Ras pathways. The third project deals with pancreatic cancer. Here, we are studying how changes in the cancerous epithelium affect the non-cancerous cells in the stroma to affect growth of the pancreatic cancer as an aberrant organ. These studies are revealing previously unexpected patterns of cell state reprogramming, and the impact of tuft cells on the immune microenvironment and cancer progression. All of this research has significant therapeutic potential. Dr. Wahl has published more than 150 articles and reviews related to genetic instability, p53 functions, the control of DNA replication, and the stem cell state and cancer. His lab has developed many technologies in wide use in molecular and cellular biology. He is a Fellow of the AAAS, was appointed as a Susan G. Komen Scholar, and served as President of the American Association for Cancer Research.  Select Publications Saison-Ridinger, M., DelGiorno, K.E., Zhang, T., Kraus, A., French, R., Jaquish, D., Tsui, C., Erikson, G., Spike, B.T., Shokhirev, M.N., Liddle, C., Yu, R.T., Downes, M., Evans, R.M., Saghatelian, A., Lowy, A.M., Wahl, G.M. Reprogramming pancreatic stellate cells via p53 activation: A putative target for pancreatic cancer therapy. PLos One 12(12):e0189051. doi: 10.1371/journal.pone.0189051. PMCID:PMC5718507 Wahl, G.M. and Spike, B.T. (2017) Cell state plasticity, stem cells, EMT, and the generation of intra-tumoral heterogeneity. NPJ Breast Cancer, 3:14. PMCID: PMC5460241. doi:10.1038/s41523-017-0012-z Turner, K.M., Deshpande, V., Beyter, D., Koga, T., Rusert, J., Lee, C., Li, B., Arden, K., Ren, B., Nathanson. D.A., Kornblum, H.I., Taylor, M.D., Kaushal, S., Cavenee. W.K., Wechsler-Reya, R., Furnari, F.B., Vandenberg, S.R., Rao, P.N., Wahl, G.M., Bafna, V., Mischel, P.S. (2017) Extrachromosomal oncogene amplification drives tumour evolution and genetic heterogeneity. Nature. 543(7643):122-125. PMCID: PMC5334176 Sancho-Martinez, I., Nivet, E., Xia, Y., Hishida, T., Aguirre, A., Ocampo, A., Ma, L., Morey, R., Krause, M.N., Zembrzycki, A., Ansorge, O., Vazquez-Ferrer, E., Dubova, I., Reddy, P., Lam, D., Hishida, Y., Wu, M.-Z., Rodriguez Esteban, C., OLeary, D., Wahl, G.M., Verma, I. M., Laurent, L. C. and Izpisua Belmonte, J.C. (2016) Establishment of human iPSC-based models for the study and targeting of glioma initiating cells. Nature Communications. 7:10743. PMCID: PMC4764898 Dravis, C., Spike, B.T., Harrell, C.J., Johns, C., Trejo, C., Southard-Smith, M.E., Perou, C.M. and Wahl, G.M. (2015) Sox10 regulates stem cell and mesenchymal states in mammary epithelial cells. Cell reports, pii: S2211-1247(15)00925-0. PMCID: PMC4591253 Pfefferle, A.D., Spike, B.T., Wahl, G.M. and Perou, C.M. (2015) Luminal progenitor and fetal mammary stem cell expression features predict breast tumor response to neoadjuvant chemotherapy. Breast Cancer Res Treat., 149: 425437. PMCID: PMC4308649 Li, Y-C., Rodewald, L.W., Hoppmann, C., Wong, E.T., Lebreton, S., Safar, P., Patek, M., Wang, L., Wertman, K.F., and Wahl, G.M. (2014) A versatile platform to analyze low-affinity and transient protein-protein interactions in living cells in real time. Cell Reports, 9(5):1946-58. doi: 10.1016/j.celrep.2014.10.058. Epub 2014 Nov 20. PMCID: PMC4269221 Makarem, M., Spike, B.T., Dravis, C., Kannan, N., Wahl, G.M., and Eaves, C.J. (2013) Stem cells and the developing mammary gland. J Mammary Gland Biol Neoplasia, 18(2):209-19. doi: 10.1007/s10911-013-9284-6. Epub 2013 Apr 27. PMCID:PMC4161372 Green, J.L., La, J., Yum, K.W., Desai, P., Rodewald, L.W., Zhang, X., Leblanc, M., Nusse, R., Lewis, M.T., and Wahl, G.M. (2013). Paracrine Wnt signaling both promotes and inhibits human breast tumor growth. Proc. Natl. Acad. Sci. U.S.A. 110(17):6991-6 PMCID:PMC3637696 Spike, B.T., Engle, D., Lin, J., Cheung, S., and Wahl, G.M. (2012) A mammary stem cell population identified and characterized in late embryogenesis reveals similarities to human breast cancer. Cell-Stem-Cell. Volume 10, Issue 2, 183-197. PMCID: PMC3277444 \n",
            "-------------------- Scraping faculty url 170/183 --------------------\n",
            "jing wang  Biography Dr. Wang received his Ph.D. in Neurobiology from the University of Iowa. He did postdoctoral research in the laboratory of Alan Gelperin at Bell Labs, then in the laboratory of Richard Axel at Columbia University. Since joining the faculty in 2004, he has been named a Chris and Warren Hellman Faculty Scholar, a Searle Scholar, and a Beckman Young Investigator.  Research The olfactory sense is essential for basic animal behaviors such as finding food, avoiding predators, and locating mating partners. Understanding the mechanism of olfaction has been accelerated by the discovery of an olfactory map at the first synapse. In mammals, these synapses are organized into 1000 glomeruli, each of which receives thousands of sensory neurons that express the same odorant receptor gene. The driving force of my lab is to find answers to such questions as: Is the pattern of glomerular activity evoked by a given odor related to the detection of this odor? How is the olfactory information represented and processed in the brain? How is olfactory valence represented in the brain? How is this representation modulated to meet the demands of different internal physiological states? The long-term goal of my laboratory is to elucidate the mechanism of olfaction. We choose Drosophila as a model system for two main reasons. First, Drosophila has fewer than 50 glomeruli in the antennal lobe. A sophisticated genetic toolkit makes it possible to establish causality between neural circuit function and behavior. Second, despite the numerical simplicity, the anatomical organization of the Drosophila olfactory system is fundamentally similar to that of mammals, suggesting a shared mechanism for smell. Our current research activities aim to identify specific neural circuits underlying olfactory behaviors, investigate how different physiological states of an organism modulate the function of olfactory circuits, and establish causality between neural plasticity in specific neuron populations and olfactory behaviors. 1) Synaptic modulations in neural circuits Olfaction is an ancient sense that makes important contributions to the perception of food quality and dietary selection. Learning how olfactory neural circuits impact dietary choices is relevant towards better understanding factors that contribute to obesity as well as anorexia in the infirm and elderly. Like many other animals, fruit flies exhibit different food searching and food-intake behaviors upon starvation. The sense of smell plays an important role in appetitive behavior in Drosophila . The DM1 glomerulus mediates attraction behavior and the DM5 glomerulus mediates avoidance behavior (Semmelhack and Wang, 2009). In a recent study, we showed that starvation sensitizes DM1's olfactory response (Root et al., 2011). A local signal by the neuropeptide sNPF/NPY and a global metabolic cue by insulin are integrated at specific sensory neurons to change olfactory sensitivity. This suggests the odor map is fine-tuned by the internal state of the animal. Furthermore, our studies in the last few years have provided insights into establishing the notion that neuromodulation mechanisms are shared between Drosophila and mammals (Root et al., 2008; Dacks et al., 2009). In both groups, GABAB receptor-mediated presynaptic inhibition provides a gain control mechanism to modulate olfactory sensitivity. Furthermore, serotonin fine-tunes this feedback inhibition by controlling the excitability of the local interneurons in Drosophila and periglomerular cells in mice. 2) Neural plasticity in olfactory circuit During a sensitive period in many animal species, usually early in life, sensory cues signaling safety and food are quickly learned and encoded into the memories of young animals through modifications of neurons in sensory neural circuits. As an important survival mechanism and a rudimentary form of learning, imprinting of sensory cues can guide and shape an animal's preferences and behaviors. We have obtained results to demonstrate that Drosophila exhibits a robust form of olfactory imprinting. Exposure to a given odorant in early adulthood dramatically increases a fly's preference for that odorant at a later stage. We aim to establish a causal link between neural plasticity in specific neuron populations and the imprinting behavior. By integrating several neural techniques, including single-neuron electrophysiology, optical imaging with genetically encoded activity indicators and genetic tools to silence or activate specific neurons in the stereotypic olfactory circuit, we hope to understand the neuronal bases of olfactory behaviors and test different hypotheses of olfactory codes with unprecedented resolution.  Select Publications Kim SM, Su CY, Wang JW. (2017) Neuromodulation of Innate Behaviors in Drosophila. Annu Rev Neurosci. 40:327-348. Sethi S and Wang JW. (2017) A versatile genetic tool for post-translational control of gene expression in Drosophila melanogaster. eLife. 6:e30327. Tao X, Lin HH, Lam T, Rodriguez R, Wang JW, Kubby J. (2017) Transcutical imaging with cellular and subcellular resolution. Biomedical Optics Express. 8(3):1277. Lin HH, Cao DS, Sethi S, Zeng Z, Chin JS, Chakraborty TS, Shepherd AK, Nguyen CA, Yew JY, Su CY, Wang JW. (2016) Hormonal Modulation of Pheromone Detection Enhances Male Courtship Success. Neuron. 90(6):1272-85. Ko, K.I., Root, C.M., Lindsay, S.A., Zaninovich, O.A., Shepherd, A.W., Wasserman, S.A., Kim, S.M., Wang, J.W. (2015) Starvation Promotes Concerted Modulation of Appetitive Olfactory Behavior via Parallel Neuromodulatory Circuits. eLife. 4:e08298. Masuyama K, Zhang Y, Rao Y, Wang JW. (2012) Mapping neural circuits with activity-dependent nuclear import of a transcription factor. J. Neurogenetics. 26: 89-102. Root C.M., Ko K.I., Jafari A., and Wang J.W. (2011) Presynaptic facilitation by neuropeptide signaling mediates odor-driven food search. Cell 145:133-144. Dacks A.M., Green D.S., Root C.M., Nighorn A.J., Wang J.W. (2009) Serotonin modulates olfactory processing in the antennal lobe of Drosophila . J. Neurogenetics 23:366-377. Semmelhack J.L. and Wang J.W. (2009) Select Drosophila glomeruli mediate innate olfactory attraction and aversion. Nature 459:218-223. Root, C.M., Masuyama, K., Green, D.S., Enell, L.E., Nssel, D.R., Lee, C.H., Wang, J.W. (2008) A presynaptic gain control mechanism fine-tunes olfactory behavior. Neuron 59:311-321. Root, C.M., Semmelhack, J.L., Wong, A.M., Flores, J., Wang, J.W. (2007) Propagation of olfactory information in Drosophila . Proc. Natl. Acad. Sci. U S A. 104:11826-11831. Wang, J.W., Wong, A.M., Flores, J., Vosshall, L.B., and Axel, R. (2003). Two-photon calcium imaging reveals an odor-evoked map of activity in the fly brain. Cell 112:271-282. Wong, A.M., Wang, J.W., and Axel, R. (2002). Spatial representation of the glomerular map in the Drosophila protocerebrum. Cell 109: 229-241. \n",
            "-------------------- Scraping faculty url 171/183 --------------------\n",
            "carl ware \n",
            "-------------------- Scraping faculty url 172/183 --------------------\n",
            "steven wasserman  Biography Steven Wasserman received his Ph.D. from MIT and was a postdoctoral fellow at UC Berkeley. He has been the recipient of a Lucille P. Markey Scholarship in Biomedical Sciences, a David and Lucile Packard Fellowship in Science and Engineering, and a Distinguished Teaching Award from the UCSD Academic Senate.  Research We are interested in how biological information regulating gene expression is encoded, transmitted, and interpreted. In addressing this question, we focus on Toll signaling, a pathway for transducing information within cells that is conserved from insects to humans and that has diverse roles in development and innate immunity. Our studies focus on the fruit fly Drosophila melanogaster , with which we can readily generate mutations that disrupt pathway activity; monitor and manipulate gene activity; and map out signaling modules and regulatory circuitry using molecular, biochemical, and bioinformatic techniques. In this way, we have defined the structure and organization of a trimeric signaling complex, elucidated function-specific evolutionary adaptations, and identified novel pathway functions and effectors. Signal Transduction in Disease and Development The signal transduction pathway defined by the Toll transmembrane receptor is a critical component of the Drosophila innate immune response to microbial infection. Upon exposure to a fungal pathogen, wild-type fruit flies express an array of genes encoding antimicrobial peptides, including Drosomycin, a potent anti-fungal agent. The transcription factor activated by Toll in this setting is the Drosophila Immunity Factor (DIF), which belongs to the NF-kB protein family. In mammals, Toll-like receptors (TLRs) activate NF-kB as a critical step in innate immune response to infection. Furthermore, aberrant Toll signaling underlies a number of debilitating inflammatory disorders. The function of Toll in innate immunity is thus broadly conserved and of interest from both a fundamental and clinical perspective. In Drosophila , Toll signaling also establishes the dorsoventral axis of the embryo. Localized activation of the transmembrane receptor Toll leads to the graded nuclear translocation of the transcription factor Dorsal, another NF-kB family member. By activation of ventral-specific loci and repression of dorsal-specific loci, the Dorsal gradient establishes subdivides the dorsoventral axis. The same signaling pathway is used in immunity and development. Prior to signaling, the target NF-kB protein, either DIF or Dorsal, is bound in the cytoplasm by an inhibitor, Cactus, that blocks nuclear translocation. Following generation of the active Toll ligand Sptzle by upstream signals, Toll transduces signals leading to degradation of Cactus, freeing the NF-kB protein to direct gene expression. Currently, our studies of Toll signaling are centered on three research areas: Dissecting the Mechanism of Signal Transduction. Toll-mediated inactivation of the inhibitor Cactus requires signal transduction by MyD88, Tube, and Pelle. We have demonstrated that these three proteins function in signaling by forming a trimeric protein complex. We have used molecular genetic, biochemical, and biophysical approaches to define and mutate the precise sites of interaction in the trimer, generating powerful reagents for our ongoing dissection of the signaling mechanism. We have extended our studies to explore how formation of the death domain trimer leads to phosphorylation and degradation of the inhibitor protein, Cactus. In particular, we have recently identified Pelle as the protein kinase required for Cactus inactivation. An Informatics-Based Approach to Decoding Global Transcriptional Control. The Toll pathway operates in parallel with a second signaling system the Imd pathwayto govern innate immune responses. Both systems rely on NF-kB related transcription factors to regulate gene expression. Applying a molecular genetic approach, we have demonstrated that cis -acting control elements act as a specificity code for the response to either or both pathways. Using our understanding of these elements to inform bioinformatic studies, we have mapped out the global system of control for the major innate immune responses in flies. Exploring Evolutionary Pathway Adaptations to Embryonic Patterning. Although the Toll pathway acts in innate immunity in a wide range of animals, it has been specifically adapted to pattern formation in the insects. There it must function in a syncytial environment and be regulated in time and space in a manner distinct from that required for defensive responses. Through identification of pathway components from a variety of insects and functional assays of chimeric proteins, we are defining the domains and activities that have undergone modification to fulfill these specialized functions in the context of conservation of the basic signaling mechanism.  Select Publications Lindsay, S.A., and Wasserman, S.A. (2013) Conventional and non-conventional Drosophila Toll signaling. Dev. Comp. Immunol. , 42, 16-24. Daigneault, J., Klemetsaune, L., and Wasserman S.A. (2013) The IRAK homolog Pelle is the functional counterpart of IB Kinase in the Drosophila Toll pathway. PLoS One , 8:e75150. Clemmons A.W., and Wasserman, S.A. (2013) Combinatorial effects of transposable elements on gene expression and phenotypic robustness in Drosophila melanogaster . G3: Genes, Genomes, Genetics , 3, 1531-1538. Higgins, R., Gendron, J.M., Rising, L., Mak, R., Webb, K., Kaiser, S.E., Zuzow, N., Riviere, P., Yang, B., Fenech, E., Tang, X., Lindsay, S.A., Christianson, J.C., Hampton, R.Y., Wasserman, S.A., and Bennett, E.J. (2015). The unfolded protein response triggers site-specific regulatory ubiquitylation of 40S ribosomal proteins. Mol. Cell. 59,35-49. Clemmons, A.W., Lindsay, S.A., and Wasserman S.A (2015). An effector peptide family required for Drosophila Toll-mediated immunity. PLoS Pathogens . 11:e1004876. Zhou, B., Lindsay, S.A., and Wasserman S.A. (2015). Alternative NF-B isoforms in the Drosophila neuromuscular junction and brain. PLoS One . 10:e0132793. Ko, K., Root, C.M., Lindsay, S.A., Zaninovich, O.A., Shepherd, A.K., Wasserman, S.A., Ki, S.M., and Wang J.W. (2015) Starvation promotes concerted modulation of appetitive olfactory behavior via parallel neuromodulatory circuits. Elife . 4:e08298. \n",
            "-------------------- Scraping faculty url 173/183 --------------------\n",
            "gabriele wienhausen  Biography Gabriele Wienhausen received her Ph.D. from the Westfalische Wilhelms-Universitat, Munster, Germany. She is currently the co-Principal Investigator of the Howard Hughes Undergraduate Science Enrichment Program and co-Director of the doctoral program in Mathematics and Science Education offered jointly by UCSD and SDSU.  Research Our research is focused on (1) how to promote \"meaningful learning\" in college science teaching, i.e., how to assure that students store more information in a memorable, readily usable fashion, and (2) how to facilitate the development of effective problem solving skills. Specifically, we investigate what role interactive, computer-based multimedia packages can play and how effective they are as a tool to facilitate the understanding of complex biological/scientific processes and concepts. We study the impact of a systematic, computer-based representation of a scientific topic on a student's learning and recall of that topic. To accomplish our goals, we develop multimedia modules which are being used as tools to improve science teaching in the laboratory setting and to integrate the process of scientific research and education. These modules have been designed on the premise that a deep understanding of biological and biomedical systems and processes depends on a meaningful inclusion of related knowledge from all the sciences. Our innovative methods of instruction address the dramatic changes in the pace of scientific learning exemplified in areas such as molecular biology or structural biochemistry. Scientific results are superseded quickly, and it is mandatory that students are familiarized with the know-how of information acquisition and its effective use. Students are taught that concepts and knowledge have to be applied across scientific disciplines. A key thesis of our research is that interactive, computer-based multimedia packages provide a potent vehicle for addressing many of the previously listed problems. We believe that the flexibility of multimedia, computer-based instructional systems allows for the provision of information in a way that more closely resembles the way research scientists think and acquire new knowledge; typically, both intellectual processes occur in a non-linear, interdisciplinary fashion where new pieces of information are fitted into an existing framework of knowledge. For instance, in the software module titled \"Spectrophoto-metry\", students can explore the interaction of light and matter, follow the mathematical derivation of Beer's Law, and practice the operating of a UV-visible virtual spectrophotometer using an on-screen working model. At all times, students using the software are in control of the pace and the path. They determine the extent of interactivity, they determine the direction, and depth of their learning path.  Select Publications Wienhausen, G. and Sawrey B. (1994). Understanding Science: Visualizing the Molecular World and Simulating Equipment, Proceedings of the 1994 International Symposium on Mathematics/Science Education and Technology. Wienhausen, G. and Sawrey, B. (1992). Interactive Multimedia Courseware for BIO 103, Syllabus (1992), 20, 20-21. \n",
            "-------------------- Scraping faculty url 174/183 --------------------\n",
            "jim wilhelm  Biography Jim Wilhelm received his Ph.D. in Cell Biology from University of California, San Francisco. He then took a Staff Associate (independent postdoctoral position) in the Department of Embryology, Carnegie Institution of Washington where he received funding from the Life Sciences Research Foundation as a Howard Hughes Medical Institute Fellow. He is also a Sloan Research Fellow in Neuroscience, an Ellison Medical Foundation New Scholar in Aging, and March of Dimes Basil O'Connor Scholar.  Research Cytoplasmic Organization in Development, Metabolism, and Disease How is a symmetrical egg turned into an embryo with a head and a tail? How are neuronal connections strengthened or weakened in response to experience? Why dont incompatible chemical reactions that are necessary for life interfere with each other? The solution to these seemingly unconnected questions lies in the ability of cells to target proteins and nucleic acids to specific subcellular domains. My laboratory is focused on identifying and characterizing novel mechanisms for generating subcellular organization and studying how those mechanisms are used in development, metabolism, and the nervous system. As part of this effort, we use a multidisciplinary approach that applies techniques from biochemistry, genetics, microscopy, and systems biology. A systems biology approach to identifying novel principles of subcellular organization In order to identify novel types of intracellular organization, we are conducting a visual screen of the yeast GFP strain collection to identify proteins that assemble into previously unidentified intracellular structures. A pilot version of this screen has identified 38 novel intracellular structures the majority of which are comprised of proteins involved in metabolic pathways suggesting that self-assembly is a major mechanism for regulating metabolic pathways in vivo. Consistent with this, one of the proteins we identified, CTP synthase, forms a novel intracellular filament that is conserved from bacteria to mammals. We have also shown that its ability to self-assemble is connected to the regulation of CTP synthase activity. We propose that this form of enzyme regulation functions analogously to the way in which microtubules and actin operate the core of the polymer is locked into a particular conformation by end limited disassembly. Since we have identified a number of enzymes that exhibit polymerization in vivo, it is likely that this mode of regulation is not unique to CTP synthase. Interestingly, we have also found that CTP synthase filaments form in axons, but not in dendrites arguing that CTP synthase filaments may also have a cytoskeletal role in neurons apart from their role in regulating enzyme activity. We are currently characterizing CTP synthase filaments in neurons. The role of information sorting in Drosophila development One mechanism for localizing cytoplasmic proteins is to transport the mRNA encoding the protein to the desired location, so that the synthesis of the protein is spatially restricted to particular cytoplasmic regions. Establishment of these domains is further enhanced by a translational control mechanism that ensures that only the properly targeted messages are translated. This sorting mechanism has been implicated in processes as diverse as stem cell differentiation, regulating synaptic strength in neurons and embryonic pattern underscoring the importance of understanding mRNA localization for medicine, neuroscience, and developmental biology. While the importance of this method of sorting cytoplasmic proteins is clear, it is unclear how localized messages are targeted to different domains or how many cellular or developmental processes utilize mRNA localization. We have used a biochemical approach to purify mRNA transport/translational control complexes from Drosophila melanogaster. By exploiting the powerful genetic tools available in Drosophila, we have begun dissecting how these transport complexes are assembled and disassembled in order to establish the primary body axes of the Drosophila embryo.  Select Publications Noree, C., Sato, B.K., Broyer, R.M., Wilhelm, J.E. (2010). Identification of novel filament forming proteins in Saccharomyces cerevisiae and Drosophila melanogaster. Journal of Cell Biology. 190(4):541-551. Wang, Y., Stary, J.M., Wilhelm, J.E., and Newmark, P.A. (2010) A functional genomic screen in planarians identifies novel regulators of germ cell development. Genes and Development. 24(18):2081-2092. Buszczak, M., Paterno, S., Lighthouse, D., Bachman, J., Planck, J., Owen, S., Skora, A.D., Nystul, T.G., Ohlstein, B., Allen, A., Wilhelm, J.E., Murphy, T.D., Levis, R.W., Matunis, E., Srivali, N., Hoskins, R.A., Spradling, A.C. (2007) The Carnegie protein trap library: a versatile tool for Drosophila developmental studies. Genetics 155(3):1505-1531. Barbee, S.A., Estes P.S., Cziko, A.M., Hillebrand, J., Luedeman R., Coller, J.M., Johnson, N., Howlett, I., Geng, C., Brand, A., Newbury, S.N., Levine, R.B., Wilhelm, J.E., Nakamura, A., Parker, R., and Ramaswami, M. (2006) FMRP and Staufen containing neuronal RNA granules are structurally and functionally related to somatic P-bodies. Neuron, 52(6):997-1009. Wilhelm, J. E.*, Buszczak, M., and Sayles, S. (2005) Efficient protein trafficking requires Trailer Hitch, a component of a ribonucleoprotein complex localized to the ER in Drosophila. Dev. Cell 9(5):675-685. Wilhelm,J.E.*, Hilton,M., Amos,Q. and Henzel,W.J. (2003) Cup is an eIF4E binding protein required for both the translational repression of oskar and the recruitment of Barentsz. J. Cell Biol. 163:1197-1204. Mansfield, J.H., Wilhelm, J.E. and Hazelrigg, T. (2002) Ypsilon Schachtel, a Drosophila Y-box Protein, Acts Antagonistically to Orb in the oskar mRNA Localization and Translation Pathway, Development. 129:197-209. Wilhelm, J.E., Vale, R.D., Hegde, R.S. (2000) Coordinate Control of Translation and Localization of Vg1 mRNA in Xenopus Oocytes, P.N.A.S. 97: 13132-13137. Takizawa, P.A., DeRisi, J.L., Wilhelm, J.E., Vale, R.D. (2000) mRNA Localization and a Septin-Based Diffusion Barrier Create an Asymmetric Distribution of a Yeast Plasma Membrane Protein, Science. 290:341-344. Wilhelm, J.E., Mansfield, J., Hom-Booher, N., Wang, S., Turck, C.W., Hazelrigg, T., Vale, R.D. (2000) Isolation of a Ribonucleoprotein Complex Involved in mRNA Localization in Drosophila Oocytes, J. Cell Biol. 148:427-439. Reviews Wilhelm, J.E. and Vale, R.D. (1993) RNA on the Move: The mRNA Localization Pathway. J. Cell Biol. 123:269-274. Wilhelm, J.E.*, Smibert, C.A.(2005) Mechanisms of translational regulation in Drosophila. Biol Cell. 97(4):235-252. * Indicates papers in which Dr. Wilhelm is the corresponding author \n",
            "-------------------- Scraping faculty url 175/183 --------------------\n",
            "christopher wills  Biography Christopher Wills received his Ph.D. from UC Berkeley. As a Guggenheim Fellow, he worked at the Karolinska Institute, Stockholm, on protein chemistry and evolution. He is the recipient of the 1999 Award for the Public Understanding of Science and Technology, given by the American Association for the Advancement of Science. His book Children of Prometheus (Perseus Books 1998) was a finalist for the 2000 Aventis Prize for science books.  Research Our laboratory is examining a broad range of evolutionary questions, including the evolution of sex and recombination, the evolution of microsatellites in lower organisms, microevolutionary events that take place in the course of infection in the genomes of pathogens, the evolution of HIV and other viruses, the timing of genetic events that have taken place during the history of our species, and the evolutionary divergence of distantly related proteins. We are also examining the maintenance of diversity in complex genetic systems and complex ecosystems. We have found that the major gene directing sexual recombination, the MAT locus, has pleiotropic effects that range far beyond meiotic and mitotic recombination and DNA repair, to encompass cellular processes as different as the uptake of exogenous DNA and the regulation of mitochondrial activity. Our results strongly support the concept that the evolution of sex had its beginnings in the uptake and incorporation by cells of exogenous genetic material. We are investigating large numbers of polymorphic microsatellites in yeast, some of which may be correlated with fitness. Such microsatellites are surprisingly common in lower organisms, and we have shown that microsatellites of the pathogenic fungus Candida albicans undergo alterations in the course of infection in AIDS patients. We have also found abundant polymorphic microsatellites in the important viral pathogen cytomegalovirus. We are also developing a selective PCR system in Saccharomyces cerevisiae which will allow us to follow the accumulation of microsatellite length mutations over time in stressed and unstressed populations. We have developed a number of methods for increasing the amount of evolutionary information in phylogenetic trees. This has enabled us to obtain a more accurate estimate of the date, and the probable geographic location, of the mitochondrial Eve. This methodology has also enabled us to determine more accurate human retrovirus evolutionary trees. Using this approach, we were able to show that the evolution of n-glycosylation sites (\"sequons\") in the envelope gene has proceeded at three times the rate in HIV-1 that it has in the less pathogenic HIV-2. This is the largest difference in evolutionary rate that has yet been detected in these viruses, and is likely to be connected to their interaction with the host immune system. Another research direction in the laboratory has been the mechanisms maintaining genetic diversity. We recently proposed a model of \"genetic herd-immunity\" to explain the maintenance of such highly polymorphic genetic systems as MHC. This frequency-dependent model permits a genetically heterogeneous host population to keep many pathogens at bay simultaneously. Stephen Hubbell, Richard Condit and I have recently extended this model to the maintenance of diversity in the rainforest, and have evidence from well-studied rainforests in Panama and Malaysia that herd-immunity at the level of species is in fact operating. Most recently, we have shown that for seven different tropical rainforests, in both the Old and New Worlds, cohorts of trees increase in diversity over time because the locally commoner species have a higher mortality than the locally rarer ones. These results, which show that there are strong non-random processes increasing diversity in these forests, are consistent with several frequency-dependent models for the maintenance of diversity. I have published a number of books about evolution for general readers, most recently\"Green Equilibrium: Balancing the Futures of Our Species and Our Planet,\" (Oxford University Press, 2013). The book deals with how ecosystems throughout the world have been shaped by the process of evolution. I continue to work with tropical ecologists around the planet, and we have recently shown that, in two different tropical forests, ecological pressures continue to shape the evolution of even distantly related species of trees that share the same ecosystem.  Select Publications Wills, C., and J. Bada. (2000). The Spark of Life: Darwin and the Primeval Soup. Perseus Books, Cambridge, MA. Metzgar, D., L. Liu, C. Hansen, K. Dybvig, and C. Wills. (2002). Domain-level differences in microsatellite distribution and content result from different relative rates of insertion and deletion mutations. Genome Research 12:408-413. Birdsell, J., and C. Wills. (2003). The evolutionary origin and maintenance of sexual recombination: a review of contemporary models. Evolutionary Biology 33:27-138. Wills, C., R. Condit, S. P. Hubbell, R. B. Foster, and N. Manokaran. (2004). Comparable nonrandom forces act to maintain diversity in both a New World and an Old World rainforest plot. Pp. 384-407 in E. C. Losos and E. G. J. Leigh, eds. Tropical Forest Diversity and Dynamism: Findings from a Large-Scale Network. University of Chicago Press, Chicago. Wills, C. (2005). Revision of the evolution section of Campbells Biology, 7th edition. This text, the most widely used introductory biology text, provides an introduction to evolution for 150,000 beginning biology majors each year. Wills, C., K. E. Harms, R. Condit, D. King, J. Thompson, F. He, H. C. Muller-Landau, P. Ashton, E. Losos, L. Comita, S. Hubbell, J. LaFrankie, S. Bunyavejchewin, H. S. Dattaraja, S. Davies, S. Esufali, R. Foster, N. Gunatilleke, S. Gunatilleke, P. Hall, A. Itoh, R. John, S. Kiratiprayoon, S. L. de Lao, M. Massa, C. Nath, Md. N. S. Noor, A. R. Kassim, R. Sukumar, H. S. Suresh, I-F. Sun, S. Tan, T. Yamakura and J. Zimmerman, 2006. Nonrandom processes maintain diversity in tropical forests. Science 311: 527-531. Wills, C. (2010). The Darwinian Tourist, Oxford University Press. This book, illustrated with 150 of the authors photographs from above and under the water around the world, shows how the worlds living systems have been shaped by the process of evolution. It was called probably the years most important travel book by Conde Nast Traveler. Wills, C. (2011a). Genetic and phenotypic consequences of introgression between humans and Neanderthals. Advances in Genetics 76:27-54. Wills, C. (2011b). Rapid recent human evolution and the accumulation of balanced genetic polymorphisms. High Altitude Medicine & Biology 12:149-155. Wills, C., 2013. Green Equilibrium: Balancing the Futures of Our Species and Our Planet. Oxford University Press. Wills, C., Harms, K.E., Wiegand, T., Punchi-Manage, R., Gilbert, G.S., Erickson, D., Kress, W.J., Hubbell, S.P., Gunatilleke, C.S. and Gunatilleke, I.N., 2016. Persistence of Neighborhood Demographic Influences over Long Phylogenetic Distances May Help Drive Post-Speciation Adaptation in Tropical Forests. PloS one, 11(6), p.e0156913. \n",
            "-------------------- Scraping faculty url 176/183 --------------------\n",
            "yang xu  Biography Yang Xu received his Ph.D. from Harvard University. He conducted postdoctoral research at the Massachusetts Institute of Technology as a fellow of the Damon Runyon-Walter Winchell Cancer Research Fund.  Research Our laboratory is interested in understanding the signaling pathways involved in maintaining genetic stability in mammalian cells, particularly embryonic stem cells (ESCs). With their capability of unlimited self-renewal and pluripotency to differentiate into any cell types in the body, ESCs hold great promise for human cell replacement therapy. Therefore, it is critical to elucidate the mechanism to safeguard the genetic integrity of ESCs so that DNA damage will not be passed on to progeny. We are investigating the pathways that coordinate the DNA damage responses and self-renewal of ESCs and adult stem cells. We are testing the hypothesis that disruption of this coordination will induce genetic instability in adult stem cell population, leading to the generation of cancer stem cells. We are focusing on the roles of tumor suppressors such as ATM and p53, which are critical to maintain genetic stability in somatic cells. To facilitate the studies of gene function in human ESCs, we recently developed technologies that allow efficient knock-out and knock-in through homologous recombination in hESCs. Induced pluripotent stem cells (iPSCs), reprogrammed from somatic cells with defined factors, could provide unlimited autologous cells for human therapy. Since most of the reprogramming factors have oncogenic potential, we are investigating the genetic stability in iPSCs. In addition, we are developing strategies to address the bottlenecks that hinder the clinic development of human ESCs and iPSCs. For example, we are identifying the pathways that can be used to eliminate the teratomas risk associated with ESCs and iPSCs-based therapy. In the context of hESCs, we are investigating the options to induce immune tolerance to allogenic grafts derived from hESCs.  Select Publications Zhao, T., Zhang, Z.N., Westenskow, P.D., Todorova, D., Hu, Z., Lin, T., Rong, Z., Kim, J., He, J., Wang, M., Clegg, D.O., Yang, Y.G., Zhang, K., Friedlander, M., & Xu, Y.* (2015). Humanized Mice Reveal Differential Immunogenicity of Cells Derived from Autologous Induced Pluripotent Stem Cells. Cell Stem Cell 17(3):353-359. Xiong, J., Todorova, D., Su, NY., Kim, J., Lee, PJ., Shen, Z., Briggs, SP., & Xu, Y.* (2015). Stemness factor Sall4 is required for DNA damage response in embryonic stem cells. J Cell Biol. 208(5):513-20. Rong, Z., Wang, M., Hu, Z., Stradner, M., Zhu, S., Kong, H., Yi, H., Goldrath, A., Yang, YG., Xu, Y.,* and Fu, X.* (2014). An Effective Approach to Prevent Immune Rejection of Human ESC-Derived Allografts. Cell Stem Cell 14:121-30. (Co-corresponding author) Zhao, T., Zhang, Z.N, Rong, Z. and Xu. Y. *(2011). The immunogenicity of the induced pluripotent stem cells. Nature 474: 212-215. Liu, D., Ou, L., Clemenson Jr, G.D., Chao, C., Lutske, M.E., Zambetti, G.P., Gage, F.H. and Xu. Y.* (2010). Puma is required for p53-induced depletion of adult stem cells. Nat. Cell Biol. 12:993-8. Moretto-Zita, M, Jin, H, Shen, Z., Zhao, T., Briggs, S.P. and Xu, Y. (2010). Phosphorylation stabilizes Nanog by promoting its interaction with Pin1. Proc. Natl. Acad. Sci. U S A 107:13312-13317. Song, H., Chung, S.K. and Xu, Y.* (2010). Modeling disease in human ESCs using an efficient BACbased homologous recombination system. Cell Stem Cell 6, 180-89. Song, H., Hollstein, M. and Xu, Y.* (2007). p53 gain-of-function cancer mutants induce genetic instability by inactivating ATM. Nat. Cell. Biol. 9:573-80. Xu, Y. (2006) DNA damage: a trigger of innate immunity but essential for adaptive immune homeostasis. Nat Reviews Immunol. 6:261-70. Zhao, T. and Xu, Y. (2010). p53 and Stem cells: new developments and new concerns. Trends Cell Biol. 20:170-175. Deng W. and Xu, Y. (2009). Genome integrity: linking pluripotency and tumorgenicity. Trends Genet. 25:425-427. Song, H., Hollstein, M. and Xu, Y. (2007). p53 gain-of-function cancer mutants induce genetic instability by inactivating ATM. Nat. Cell. Biol. 9:573-580. Xu, Y. (2006) DNA damage: a trigger of innate immunity but essential for adaptive immune homeostasis. Nat Reviews Immunol. 6:261-70. Lin, T., Chao, C., Saito, S., Muzar, S., Murphy, M., appella, E. and Xu, Y.* (2005). p53 induces differentiation of mouse embryonic stem cells by suppressing Nanog expression. Nat. Cell Biol. 7, 165- 171. \n",
            "-------------------- Scraping faculty url 177/183 --------------------\n",
            "martin yanofsky  Biography Marty Yanofsky received his Ph.D. from the University of Washington and was an NSF Postdoctoral Fellow in Plant Biology at Caltech. He was the recipient of a Packard Fellowship for Science and Engineering and a Beckman Young Investigator Award. Marty Yanofsky is a member of the National Academy of Sciences.  Research Research in the Yanofsky lab focuses the genes involved in fruit development in Arabidopsis thaliana . The fruit, which is arguably the most complex plant organ, is derived from the fertilized gynoecium (carpels), and consists of a number of distinct tissue types, including the valves, replum and valve margin. The valve margin, which forms at the boundary between the valve and replum regions, consists of a narrow stripe of cells that extends along the entire length of the fruit. Upon fruit maturation, cell separation within the valve margin region allows the valves to detach from the replum and for the seeds to be dispersed. We have shown that the SHATTERPROOF1 ( SHP1 ) and SHP2 MADS-box genes act redundantly to specify valve margin cell fate. The SHP genes act to promote the expression of the INDEHISCENT ( IND ) and ALCATRAZ ( ALC ) bHLH genes, which are also required for differentiation of valve margin cells. Because the valve margin fails to differentiate in these mutants, the fruits fail to open and the seeds cannot be dispersed through their normal mechanism. We have also shown that the FRUITFULL ( FUL ) MADS-box gene is required for the post-fertilization elongation of the fruit and is required for valve cell differentiation. Interestingly, the FRUITFULL gene product is expressed in valve cells where it negatively regulates expression of SHP , IND , and ALC genes such that the SHP , IND , and ALC genes are ectopically expressed in fruitfull mutant valves. Similarly, the REPLUMLESS ( RPL ) gene encodes a homeodomain transcription factor that is required for formation of the replum. RPL is expressed in the replum region where it acts to negatively regulate SHP , IND , and ALC . The combined action of FUL in the valves and RPL in the replum allows for the valve margin identity genes to be expressed in the narrow stripe of cells that differentiate into the valve margin. More recently, we have begun to understand the role of additional fruit development genes leading to a framework model for the genetic interactions that pattern the Arabidopsis fruit. Our lab has also initiated a project aimed at identifying the major genes that specify the female reproductive tract in Arabidopsis . This region plays a key role in ensuring efficient pollen tube migration as they seek out unfertilized ovules, and yet little is known about the regulatory genes that control the formation of the reproductive tract. We have recently shown that the NO TRANSMITTING TRACT ( NTT ) gene encodes a zinc finger transcription factor that is necessary for formation of transmitting tract cells and for preventing the cell death that normally occurs within the transmitting tract region.  Select Publications Crawford BC, Sewell J, Golembeski G, Roshan C, Long JA, Yanofsky M.F. (2015) Plant development. Genetic control of distal stem cell fate within root and embryonic meristems. Science. 347:655-9. Ripoll, J.J., Bailey, L.J., Mai, Q.A., Wu, S.L., Hon, C.T., Chapman, E.J., Ditta, G.S., Estelle, M., and Yanofsky, M.F. (2015) microRNA regulation of fruit growth. Nature Plants. 1:15036 doi:10.1038/nplants.2015.36. Crawford, B.C.W. and Yanofsky, M.F. (2011). HALF FILLED promotes reproductive tract development and fertilization efficiency in Arabidopsis thaliana. Development 138:2999-3009. Crawford, B. C. W., Ditta, G., and Yanofsky, M.F. (2007) The NTT gene is required for transmitting-tract development in carpels of Arabidopsis thaliana . Current Biology 17:1101-1108. Dinneny, J. R., Weigel, D., and Yanofsky, M.F. (2005) A genetic framework for fruit patterning in Arabidopsis thaliana . Development 132:4687-4696. Liljegren, S.J., Roeder, A.H.K., Kempin, S.A., Gremski, K, stergaard, L., Guimil, S., Khammungkhune, D. and Yanofsky, M.F. (2004) Control of fruit patterning in Arabidopsis by INDEHISCENT. Cell 116:843-853. \n",
            "-------------------- Scraping faculty url 178/183 --------------------\n",
            "deborah yelon  Biography Deborah Yelon received her Ph.D. in 1996 from Harvard University, where she studied T cell development with Leslie Berg. She then began to investigate heart development in zebrafish as a Life Sciences Research Foundation postdoctoral fellow with Didier Stainier at the University of California, San Francisco. She was a member of the faculty of the Skirball Institute of Biomolecular Medicine at New York University School of Medicine from 2000 to 2009. She has received a Burroughs Wellcome Fund Career Award and an American Heart Association Established Investigator Award.  Research Organogenesis requires the carefully orchestrated arrangement of multiple cell types into a specific three-dimensional form that is essential for effective organ function. The embryonic zebrafish heart provides a clear example of the relationship between organ form and function: it is composed of two major chambers, a ventricle and an atrium, each with discrete morphological and physiological characteristics that define its functional capacity. The goal of our laboratory's research is to understand the molecular mechanisms that specify the size, shape, and attributes of each cardiac chamber. By taking advantage of the unique arsenal of genetic and embryologic experimental approaches available in zebrafish, we can identify crucial regulators of chamber formation and determine their precise impacts on cell fate and cell behavior. In the long term, this work will improve our comprehension of the etiology of congenital heart disease and inform strategies for regenerative medicine and tissue engineering. Regulation of heart size Specification of the proper number of cardiac progenitor cells is a critical determinant of heart size, but we do not fully understand the network of signals that regulate the dimensions of the cardiac progenitor pool. Several types of secreted factors, including BMP, FGF, and Hedgehog, are implicated in promoting the assignment of cardiac identity. However, much less is known about the opposing factors that set limits for cardiac progenitor specification. We are therefore particularly interested in zebrafish mutants with large hearts composed of too many cardiomyocytes. Notably, our recent analyses of these mutants have revealed two potent mechanisms for restricting the formation of cardiac progenitors. These studies indicate that generation of the proper number of progenitors involves interplay between inductive and repressive pathways. Cardiac cells (red) develop in proximity to the vascular plexi (green) prior to heart formation. Allocation of cardiac and vascular progenitors requires interplay between cardiovascular pathways: components of vessel and blood specification pathways repress formation of cardiac progenitors and thereby limit heart size (Schoenebeck et al., 2007). Regulation of heart shape An elaborate series of cell movements and cell shape changes creates the characteristic contours of the heart, and all of these cellular activities can be influenced by the surrounding environment. However, it is not clear which extrinsic and intrinsic cues regulate cell behaviors during chamber morphogenesis. By combining high-resolution live imaging with genetic analysis, we can elucidate pathways with a crucial influence on the actions of individual cardiomyocytes. To date, we have focused our attention on two particular steps of cardiac morphogenesis: the midline merger of the bilateral cardiomyocyte precursor populations and the emergence of chamber curvatures. During both of these processes, regionally restricted patterns of cell behavior underlie key features of cardiac morphology, and interactions of the cardiomyocytes with their environment have a significant impact on morphogenetic control. Regionally confined cell shape changes underlie the emergence of chamber curvatures. A portion of the linear heart tube (LHT) transforms into an expanded ventricle with a bulging outer curvature (OC) and a recessed inner curvature (IC). Individual cells (red, arrows) are small and round in the LHT. Cells in the OC become flattened and elongated, whereas cells in the IC remain relatively cuboidal, creating the characteristic shape of the ventricle (Auman et al., 2007).  Select Publications Keegan, B.R., J. Feldman, G. Begemann, P. Ingham, and D. Yelon. (2005) Retinoic acid signaling restricts the cardiac progenitor pool. Science 307: 247-249. Auman, H.J., H. Coleman, H.E. Riley, F. Olale, H.-J. Tsai, and D. Yelon. (2007) Functional modulation of cardiac form through regionally confined cell shape changes. PLoS Biology 5: e53. Schoenebeck, J.J., B.R. Keegan, and D. Yelon. (2007) Vessel and blood specification override cardiac potential in anterior mesoderm. Developmental Cell 13: 254-267. Waxman, J.S., B.R. Keegan, R.W. Roberts, K.D. Poss, and D. Yelon. (2008) Hoxb5b acts downstream of retinoic acid signaling in the forelimb field to restrict heart field potential in zebrafish. Developmental Cell 15: 923-934. Garavito-Aguilar, Z.V., H.E. Riley, and D. Yelon. (2010) Hand2 ensures an appropriate environment for cardiac fusion by limiting fibronectin function. Development 137: 3215-3220. Totong, R., T. Schell, F. Lescroart, L. Ryckebsch, Y.-F. Lin, T. Zygmunt, L. Herwig, A. Krudewig, D. Gershoony, H.-G. Belting, M. Affolter, J. Torres-Vzquez, and D. Yelon. (2011) The novel transmembrane protein Tmem2 is essential for coordination of myocardial and endocardial morphogenesis. Development 138: 4199-4205 Lin, Y.-F., I. Swinburne, and D. Yelon. (2012) Multiple influences of blood flow on cardiomyocyte hypertrophy in the embryonic zebrafish heart. Developmental Biology 362:242-253. Targoff, K.L., S. Colombo, V. George, T. Schell, S.-H. Kim, L. Solnica-Krezel, and D. Yelon. (2013) Nkx genes are essential for maintenance of ventricular identity. Development 140:4203-4213. Schumacher, J.A., J. Bloomekatz, Z.V. Garavito-Aguilar, and D. Yelon. (2013) tal1 regulates the formation of intercellular junctions and the maintenance of identity in the endocardium. Developmental Biology 383:214-226. Zeng, X.-X.I. and D. Yelon. (2014) Cadm4 restricts the production of cardiac outflow tract progenitor cells. Cell Reports 7:951-960. Schindler, Y.L., K.M. Garske, J. Wang, B.A. Firulli, A.B. Firulli, K.D. Poss, and D. Yelon. (2014) Hand2 elevates cardiomyocyte production during zebrafish heart development and regeneration. Development 141:3112-3122. \n",
            "-------------------- Scraping faculty url 179/183 --------------------\n",
            "dong-er zhang  Biography Dr. Dong-Er Zhang received her BS in Biochemistry at Peking (Beijing) University, China. She was selected to enter a graduate school in the United States through the CUSBEA (China-United States Biochemistry and Molecular Biology Examination and Administration) program and received her PhD in Biochemistry at the University of Houston. She was a Postdoctoral Scholar/Instructor at the University of Texas and then Instructor/Assistant Professor at the Harvard Medical School. In 1999, Dr. Zhang was recruited by The Scripps Research Institute as Associate Professor and was subsequently promoted to the Full Professor level before joining UCSD. Dr. Zhang was a Leukemia and Lymphoma Society Scholar (1998-2003) and received the prestigious Stolhman Scholar Award from the Leukemia and Lymphoma Society. She is involved in multiple international grant review committees and also contributes to the organization of scientific conferences.  Research Dr. Zhangs laboratory studies the molecular basis of cancer development, progression, and treatment. Her laboratory uses molecular biology, protein biochemistry, cell biology, and animal models to address questions related to blood cell differentiation, transformation, and innate immune responses. A major project underway in the Zhang laboratory focuses on the transcription factor AML1 (RUNX1) and its fusion protein AML1-ETO. As a second major project, Dr. Zhang studies the role of a ubiquitin-like modifier, ISG15, and the ISG15specific protease, USP18 (UBP43), in interferon-initiated cell-signaling and cancer.  Select Publications Chen L, Chen JY, Huang YJ, Gu Y, Qiu J, Qian H, Shao C, Zhang X, Hu J, Li H, He S, Zhou Y, Abdel-Wahab Omar, Zhang DE, Fu XD. Augmented R-loop is a Unifying Mechanism for Myelodysplastic Syndromes Induced by High Risk Splicing Factor Mutations (* co-corresponding). Mol Cell. 2018 69:421-425. Arimoto KI*, Lchte S*, Stoner SA, Burkart C, Zhang Y, Miyauchi S, Wilmes S, Fan JB, Heinisch JJ, Li Z, Yan M, Pellegrini S, Colland F, Piehler J*, Zhang DE*. STAT2 is an essential adaptor in USP18-mediated suppression of type I interferon signaling (* co-first or co-corresponding authors). Nat Struct Mol Biol. 2017. 24:279-89. Weng S, Matsuura S, Mowery C, Stoner SA, Lam K, Ran D, Davis AG, Lo MC, Zhang DE. Restoration of MYC-repressed targets mediates the negative effects of GM-CSF on RUNX1-ETO leukemogenicity. Leukemia 2017 31:159-169. Qiu J, Zhou B, Thol F, Zhou Y, chen L, shao C, DeBoever C, Hou J, Li H, Chaturvedi A, Ganser A, Bejar R, Zhang DE*, Fu SD*, Heuser M*. Distinct splicing signatures affect converged pathways in myelodysplastic syndrome patients carrying mutations in different splicing regulators (co-corresponding). RNA 2016 22:1535-49. Lam K, Muselman A, Du R, Yan M, Matsuura S, Zhang DE. Loss of RUNX1 function results in enhanced granulocyte-colony-stimulating factor-mediated mobilization. Blood Cancer J. 2016 6:e407. Fan JB, Arimoto KI, Yan M, Liu C, Gyorffy B, Zhang DE. Type I IFN induces protein ISGylation to enhance cytokine expression and augments colonic inflammation. PNAS, 2015, 112:14313-8. Fan JB, Arimoto KI, Motamedchaboki K, Yan M Wolf DA, Zhang DE. Identification and characterization of a novel ISG15-ubiquitin mixed chain and its role in regulating protein homeostasis. Scientific Report 2015, 5:12704. Komeno Y, Yan M, Matsuura S, Lam K, Lo MC, Huang YJ, Tenen DG, Downing JR, Zhang DE. Runx1 exon 6 related alternative splicing isoforms differentially regulate hematopoiesis in mice. Blood. 2014 123:3760-9. Lam L, Muselman A, Du R, Harada Y, Scholl AG, Yan M, Matsuura S, Weng S, Harada H, Zhang DE. Hmga2 is a direct target gene of RUNX1 and regulates expansion of myeloid progenitors in mice. Blood. 2014. 124:2203-12. Burkart C, Arimoto K-I, Tang T, Liu YC, Kotenko SV, Ellies LG, Zhang DE. Usp18 deficient mammary epithelial cells create an antitumour environment driven by hypersensitivity to IFN- and elevated secretion of Cxcl10. EMBO Mol Med 2013 5:967-82. Ran D, Shia WJ, Lo MC, Fan JB, Knorr DA, Ferrell PI, Ye Z, Yan M, Cheng L, Kaufman DS, Zhang DE. RUNX1a enhances hematopoietic lineage commitment from human embryonic stem cells and inducible pluripotent stem cells. Blood 2013 121:2882-90. DeKelver RC, Lewin B, Lam K, Komeno Y, Yan M, Rundle C, Lo MC, Zhang DE. Cooperation between RUNX1-ETO9a and novel transcriptional partner KLF6 in upregulation of Alox5 in acute myeloid leukemia. PLOS Genetics, 2013 9:e1003765. Lo MC, Peterson LF, Yan M, Cong X, Jin F, Shia W-J, Matsuura S, Ahn E-Y, Komeno Y, Ly M, Ommen HB, Chen I-M, Hokland P, Willman CL, Ren B, Zhang DE. Combined gene expression and DNA occupancy profiling identifies potential therapeutic targets of t(8;21) AML. Blood. 2012, 120:1473-84. Matsuura S, Komeno Y, Stevenson KE, Biggs JR, Lam K, Tang T, Lo M-C, Cong X, Yan M, Neuberg DS, Zhang DE. Expression of the runt homology domain of RUNX1 disrupts homeostasis of hematopoietic stem cells and induces progression to myelodysplastic syndrome, Blood, 2012, 120:4028-37. Ahn E-Y, DeKelver RC, Lo MC, Nguyen TA, Matsuura S, Boyapati A, Pandit S, Fu X-D*, Zhang DE*, SON Controls Cell Cycle Progression by Coordinated Regulation of RNA Splicing (*co-corresponding). 2011, Molecular Cell, 42:185-98. References From PubMed (NCBI) \n",
            "-------------------- Scraping faculty url 180/183 --------------------\n",
            "yunde zhao  Biography Yunde Zhao received his Ph.D. in biochemistry from the University of Michigan and his postdoctoral training in plant genetics at the Salk Institute, where he was a Howard Hughes Medical Institute Fellow of the Life Sciences Research Foundation.  Research Our research focuses on the plant hormone auxin, which regulates almost every aspect of plant growth and development. My laboratory mainly works on two areas of auxin biology: 1) auxin biosynthesis and its regulation, 2) the roles of auxin in axillary meristem development and plant organogenesis. We have elucidated the main auxin biosynthesis pathway in Arabidopsis and identified YUC flavin monooxygenases as key enzymes for auxin biosynthesis. We currently investigate the allosteric regulation of YUC enzymes and characterize the transcription factors that control the expression of YUC genes. We conducted a genetic screen for enhancers / suppressors of yuc mutants and uncovered a novel signaling pathway that is essential for the formation of lateral organs in Arabidopsis. We are continuing to clone and characterize additional yuc enhancers / suppressors to establish a genetic framework for plant organogenesis. We are also developing tools to spatially and temporally control auxin production.  Select Publications Dai X., Zhang Y., Zhang D., Chen J., Gao X., Estelle M., and Zhao, Y. (2015) Embryonic lethality of Arabidopsis abp1-1 is caused by deletion of the adjacent BSM gene. Nature Plants, 9 NOVEMBER 2015 DOI:10.1038/NPLANTS.2015.183 Gao Y., Zhang Y., Zhang D., Dai X., Estelle M., and Zhao Y. (2015) Auxin binding protein 1 (ABP1) is not required for either auxin signaling or Arabidopsis development. PNAS 112, 22752280. PMID: 25646447 Zhao Y (2014) Auxin biosynthesis. The Arabidopsis Book, First published on June 13, 2014: e0173. doi: 10.1199/tab.0173. PMID: 24955076 Chen Q, Dai X, De-Paoli H, Cheng Y, Takebayashi Y, Kasahara H, Kamiya Y, and Zhao Y (2014) Auxin overproduction in shoots cannot rescue auxin deficiencies in Arabidopsis roots. Plant Cell Physiology 55(6):1072-9. PMID: 24562917 Gao Y and Zhao Y (2014) Selfprocessing of ribozymeflanked RNAs into guide RNAs in vitro and in vivo for CRISPRmediated genome editing. Journal of Integrative Plant Biology. 56(4):343-9 PMID: 24373158 \n",
            "-------------------- Scraping faculty url 181/183 --------------------\n",
            "ye zheng  Biography Dr. Ye Zheng earned his B.S. degree in Biochemistry and Molecular Biology from Peking University in Beijing, China, and his Ph.D. degree in Biological Sciences from Columbia University in New York. He was awarded a postdoctoral fellowship sponsored by Cancer Research Institute to conduct research in Dr. Alexander Rudensky's lab at University of Washington in Seattle. After a brief sojourn as a research scholar at Memorial Sloan-Kettering Cancer Center, he joined the Salk Institute for Biological Studies as an assistant professor in 2009. He was named a Rita Allen Scholar in 2010.  Research The immune system is a powerful double-edged sword. On one hand, it is armed to fight a wide range of invading foreign pathogens. On the other hand, if left unchecked, it can also attack self-tissues and cause autoimmune diseases, such as type-1 diabetes, multiple sclerosis, and rheumatoid arthritis. There are multiple safeguard mechanisms built in our immune system to prevent autoimmune reaction. A subset of T cells, named regulatory T cells (Treg), play a key role in the maintenance of immune homeostasis. Abnormal Treg function has been linked to a number of autoimmune diseases. Recent study showed that transcription factor Foxp3 is a pivotal regulator for Treg differentiation and function. Mutations of Foxp3 in human and mice lead to deficiency of regulatory T cells and fatal autoimmune disease. Zhengs lab is interested in elucidating both the upstream pathways that turn on Foxp3 expression and the downstream genes that are regulated by Foxp3. Zheng and colleagues identified several DNA elements in Foxp3 locus that are conserved in several mammalian species. They are involved in the induction and maintenance of Foxp3 expression, and regulate the development and stability of regulatory T cell lineage. Using genomics approaches, Zheng and colleagues were able to map all Foxp3 downstream target genes. They showed that, among all Foxp3 targets, there is a small group of transcription factors that are implicated in Treg mediated suppression of different subtypes of autoimmune responses. Zheng and his team are now further dissecting the Foxp3 transcriptional network in regulatory T cells and searching for key molecules involved in Treg suppression function. Since manipulations of Tregs can either attenuate or strengthen immune response, their findings can potentially open new avenues in treatment of autoimmune diseases, improvement of organ transplant survival, and enhancement of anti-tumor immunity.  Select Publications Zheng Y*, Josefowicz SZ*, Chadhry A, Peng XP, Forbush K, Rudensky AY (2010). Role of conserved non-coding DNA elements in the Foxp3 gene in regulatory T-cell fate. Nature 463:808-812. (*Equal Contribution) Zheng Y, Chaudhry A, Kas A, deRoos P, Kim JM, Chu TT, Corcoran L, Treuting P, Klein U, Rudensky AY (2009). Regulatory T cell suppressor program co-opts transcription factor IRF4 to control Th2 responses. Nature 458:351-356. Zheng Y, Rudensky AY (2007). Foxp3 in control of the regulatory T cell lineage. Nat. Immunol. 8:457-462. Zheng Y, Josefowicz SZ, Kas A, Chu TT, Gavin MA, Rudensky AY (2007). Genome-wide analysis of Foxp3 target genes in developing and mature regulatory T cells. Nature 445:936-940. Hsieh CS, Zheng Y, Liang Y, Fontenot JD, Rudensky AY (2006). An intersection between the self-reactive regulatory and nonregulatory T cell receptor repertoires. Nat. Immunol. 7: 401-410. \n",
            "-------------------- Scraping faculty url 182/183 --------------------\n",
            "yimin zou  Biography Yimin Zou graduated from Shanghai Fudan University and was a CUSBEA (China and United States Biochemistry Examination and Application Program) student and received his Ph. D. from University of California, Davis and San Diego, in 1995. He did his postdoctoral fellowship at University of California, San Francisco from 1996 to 2000 and was an Assistant and Associate Professor at the University of Chicago from 2000 to 2006. He joined the faculty in July, 2006.  Research Axon Guidance We are interested in the guidance cues that provide directional information for axon wiring and identified the Wnt family proteins as key guidance molecules along the major axes of the central nervous system for axon pathfinding and topographic mapping. We are currently studying how these guidance cues are laid out in concentration gradients and how axonal growth cones recognize and respond to these gradients. We found that the conserved apical-basal and planar cell polarity pathways mediate growth cone turning and an apical-basal polarity signaling component, aPKC, promotes the endocytosis of a Wnt/planar cell polarity component, Frizzled3. We are testing whether this is part of an amplification mechanism for asymmetric signaling to polarize growth cones. Circuit repair after traumatic injury Traumatic injuries of the adult brain and the spinal cord lead to loss of many important functions. We found that the Wnt signaling pathways are reactivated after spinal cord injury. A reinduced Wnt inhibitory system limits motor and sensory axon growth and functional recovery. Our recent studies showed that combining molecular manipulation to enhance axon plasticity with behaviorally-guided training offers the maximal functional recovery. We are currently studying the network basis of neural circuit remodeling after spinal cord injury and traumatic brain injury. Synapse formation and neurodevelopmental disorders The signaling mechanisms that assemble neuronal synapses have been elusive. Our recent finding suggests that planar cell polarity signaling components play essential roles in glutamatergic synapse formation. This opens up new opportunities to study synaptogenesis and plasticity, fundamental to understanding neural circuit function. Mutations in planar cell polarity genes, such as the Prickles, are associated with neurodevelopmental disorders, such as autism and epilepsy. We are studying the molecular and circuit mechanisms of these diseases and exploring strategies for therapeutic interventions.  Select Publications Keisuke Onishi and Yimin Zou. Sonic Hedgehog switches on Wnt/planar cell polarity signaling in commissural axon growth cones by reducing levels of Shisa2. Elife. 2017 Sep 8;6. pii: e25269. doi: 10.7554/eLife.25269. Sonal Thakar, Liqing Wang, Ting Yu, Mao Ye, Keisuke Onishi, John Scott, Jiaxuan Qi, Catarina Fernandes, Darwin Berg and Yimin Zou*. Evidence for opposing roles of Celsr3 and Vangl2 in glutamatergic synapse formation. Proc Natl Acad Sci U S A. 2017 Jan 5. pii: 201612062. doi: 10.1073/pnas.1612062114. [Epub ahead of print] PMID: 28057866 Edmund Hollis 2nd, Nao Ishiko, Ting Yu , Chin-Chun Lu, Ariela Haimovich, Kristine Tolentino, Alisha Richman, Anna Tury, Shih-Hsiu Wang, Maysam Pessian, Euna Jo, Alex Kolodkin and Yimin Zou. Ryk controls remapping of motor cortex during functional recovery after spinal cord injury. Nature Neurosci. 2016 May;19(5):697-705 Edmund R Hollis II, Nao Ishiko, Maysam Pessian, Kristine Tolentino, Corinne A. Lee-Kubli, Nigel A. Calcutt and Yimin Zou*. Remodeling of spared proprioceptive circuit of a small number of neurons supports major functional recovery. Nature Communications. 2015 Jan 19;6:6079. doi: 10.1038/ncomms7079. Edmund R Hollis II, Nao Ishiko, Kristine Tolentino, Ernest Doherty, Maria J. Rodriguez, Nigel Calcutt and Yimin Zou*. A novel and robust pre-conditioning lesion induced by ethidium bromide. Experimental Neurology. 2014 Dec 23. pii: S0014-4886(14)00389-6. Keisuke Onishi, Edmund Hollis and Yimin Zou. Axon guidance and injury-lessons from Wnts and Wnt signaling. Curr Opin Neurobiol. 2014 Aug;27C:232-240. Keisuke Onishi, Beth Shafer, Charles Lo, Fadel Tissir, Andre M. Goffinet, and Yimin Zou. Antagonistic Functions of Dishevelleds Regulate Frizzled3 Endocytosis via Filopodia Tips in Wnt-Mediated Growth Cone Guidance. The Journal of Neuroscience, December 4, 2013 *33(49):19071-1908. Anna Tury, Kristine Tolentino and Yimin Zou. Altered expression of atypical PKC and Ryk in the spinal cord of a mouse model of amyotrophic lateral sclerosis. Dev Neurobiology 2013 Oct 7. doi: 10.1002/dneu.22137 Yimin Zou (2012). Does planar cell polarity signaling steer growth cones? Curr. Top. Dev. Biol. 101:141-160. Edmund Hollis II and Yimin Zou (2012). Reinduced Wnt signaling limits regenerative potential of sensory axons in the spinal cord following conditioning lesion. Proc. Natl. Acad. Sci. USA 109(36):14663-14668. Edmund Hollis II and Yimin Zou (2012). Expression of the Wnt signaling system in central nervous system axon guidance and regeneration. Front. Mol. Neurosci. 5:5. Yimin Zou (2011). When it is hard to get to with genetics--planar cell polarity under a chemical scalpel. Chem. Biol. 18(11):1350-1351. Beth Shafer, Keisuki Onishi, Charles Lo, Gulsen Colakoglu, Zou Y. (2011). Vangl2 promotes Wnt/planar cell polarity-like signaling by antagonizing Dvl1-mediated feedback inhibition in growth cone guidance. Devel. Cell 20(2):177-191. Ali G. Fenstermaker, Asheeta A. Prasad, Ahmad Bechara, Youri Adolfs, Fadel Tissir, Andre Goffinet, Yimin Zou, and R. Jeroen Pasterkamp (2010). Wnt/Planar cell polarity signaling controls the anterior-posterior organization of monoaminergic axons in the brainstem. J. Neurosci. 30(47):16053-16064. Barry J. Dickson and Yimin Zou (2010). Navigating Intermediate Targets: The Nervous System Midline. Cold Spring Harbor Perspectives in Biology. Published in Advance June 9, 2010 (doi: 10.1101/cshperspect.a002055). Liseth M. Parra and Yimin Zou (2010). Sonic hedgehog induces response of commissural axons to Semaphorin repulsion during midline crossing. Nature Neurosci. 13:29-35 (doi:10.1038/nn.2457). Esther Stoeckli and Yimin Zou (2009). How are neurons wired to form functional and plastic circuits? Meeting on Axon Guidance, Synaptogenesis and Neural Plasticity (Cold Spring Harbor Laboratories). EMBO J. 10(4):326-330. Yimin Zou (2009). Axons find their way in the snow. Development 136:2135-2139. Elizabeth K. Davis, Yimin Zou, and Anirvan Ghosh (2008). Wnts acting through canonical and noncanonical signaling pathways exert opposite effects on hippocampal synapse formation. Neural Development 3:32. Yaobo Liu, Xiaofei Wang X, Chin-Chun Lu, Rachel Kerman, Oswald Steward, Xiao-Ming Xu, and Yimin Zou. (2008). Repulsive Wnt signaling inhibits axon regeneration after CNS injury. J. Neurosci. 28:8376-8382. Patricia C. Salinas and Yimin Zou (2008). Wnt Signaling in neural circuit assembly. Annu. Rev. Neurosci. 31:339-358. Alex M. Wolf, Anna I. Lyuksyutova, Ali G. Fenstermaker, Beth Shafer, Charles G. Lo, and Yimin Zou. (2008). Phosphatidylinositol-3-kinase-atypical protein kinase C signaling is required for Wnt attraction and anterior-posterior axon guidance. J. Neurosci. 28:3456-3467. Yimin Zou and Anya I. Lyuksyutova (2007). Morphogens as conserved axon guidance cues. Curr. Opin. Neurobiol. 17:22-28. Yimin Zou. (2006). Navigating the anterior-posterior axis with Wnts. Neuron. 49(6):787-789. Adam Schmitt, Jun Shi, Alex Wolf, Chin-Chun Lu, Leslie A. King and Yimin Zou. (2006). Wnt-Ryk signaling mediates medial-lateral retinotectal topographic mapping. Nature 439(7072):31-37. Lee Fradkin, Gian Garriga , Patricia C. Salinas, John Thomas, Xiang Yu and Yimin Zou (2005). Wnt signaling in neural circuit development (minireview). J. Neurosci. 25(45):10376-10378. Yaobo Liu, Jun Shi, Chin-Chun Lu, Zheng Bei Wang, Anna I. Lyuksyutova, Xuejun Song and Yimin Zou (2005). Ryk-mediated Wnt repulsion regulates posterior-directed growth of corticospinal tract. Nature Neurosci. 8(9):1151-1159. Yimin Zou (2004). Wnt signaling in axon guidance (review). Trends Neurosci. 27(9):528-532. Yimin Zou, Florian Engert and Huizhong W. Tao (2004). The assembly of neural circuits (meeting report). Neuron 43(2):159-163. Anna I. Lyuksyutova, Chin-Chun Lu, Nancy Milanesio, Leslie A. King, Nini Guo, Yanshu Wang, Jeremy Nathans, Marc Tessier-Lavigne and Yimin Zou (2003). Anterior-posterior guidance of commissural axons by Wnt-frizzled signaling. Science 302:1984-1988. Elke Stein, Yimin Zou, Mu-ming Poo, and Marc Tessier-Lavigne (2001). Netrin-1 binds DCC to stimulate axon outgrowth and turning independent of adenosine A2B receptor activation. Science 291(5510):1976-1982. Yimin Zou, Esther Stoeckli, Hang Chen, and Marc Tessier-Lavigne (2000). Squeezing axons out of the gray matter: A role for slit and semaphorin proteins from midline and ventral spinal cord. Cell 102:363-375. Hang Chen, Anil Bagri, Joel A. Zupicich, Yimin Zou, Esther Stoeckli, Samuel J. Pleasure, Daniel H. Lowenstein, William C. Skarnes, Alain Chedotal, and Marc Tessier-Lavigne (2000). Neuropilin-2 regulates the development of select cranial and sensory nerves and hippocampal mossy fiber projections. Neuron 25(1):43-56. \n",
            "-------------------- Scraping faculty url 183/183 --------------------\n",
            "elina zuniga  Biography Elina Zuniga received her Ph.D. in Biochemistry from the National University of Cordoba, Argentina. She conducted postdoctoral research at The Scripps Research Institute where she held two post-doctoral fellowships from Antorchas Foundation and PEW Charitable Trust, respectively. After joining UCSD in 2007 she has received a Hellman Foundation Scholar Award, The Vilcek Finalist Prize for Creative Promise, the Leukemia and Lymphoma Society Scholar Award and the American Cancer Society Scholar Award (a lifetime honor).  Research The immune system is responsible for the tremendous task of fighting a wide range of pathogens to which we are constantly exposed. This system can be broadly subdivided into innate and adaptive arms, which act in conjunction to protect the host. The innate immune system exists in both vertebrate and invertebrate organisms and represents the first barrier against microbial invasion. This arm of the immune system rapidly eliminates the vast majority of microorganisms that we daily encounter and is responsible for limiting early pathogen replication and imprinting the profile of the subsequent adaptive response. The adaptive immune response is a more sophisticated feature, found only in vertebrate animals, involving a broad repertoire of genetically rearranged receptors that specifically recognize microbial antigens. The hallmark of the adaptive response is the generation of a potent and long-lasting protection specifically directed against the invading pathogen. Despite highly evolved immune responses, pathogens have co-evolved ways to evade or subvert the immune system to favor their replication and transmission. Therefore, despite major scientific and medical efforts, infectious diseases remain among the leading causes of mortality and disability worldwide. Dramatic examples of microbial immune-evasion are chronic viral infections such as Human Immunodeficiency virus, Hepatitis B and C viruses, which currently infect about 500 million people around the world. Our laboratory studies cellular and molecular aspects of immune responses during acute and chronic viral infections to determine general principles of antiviral immunity, immune-evasion, persistence and pathogenesis. We use a combination of both standard and cutting-edge molecular approaches as well as functional assays with recombinant viruses and genetically modified hosts. Both hypothesis driven and unbiased strategies allow us to unravel novel aspects of host-virus interactions in mouse models of natural viral infections that we aim at translating to humans. The ultimate goal is to generate fundamental knowledge on immune-regulation that could help modulating immune responses to prevent or treat infectious diseases and may have implications for other immune-related illnesses.  Select Publications Full list on PubMed Loureiro ME, Zorzetto-Fernandes AL, Radoshitzky S, Chi X, Dallari S, Marooki N, Lger P, Foscaldi S, Harjono V, Sharma S, Zid BM, Lpez N, de la Torre JC, Bavari S, Zuniga El. DDX3 suppresses type I interferons and favors viral replication during Arenavirus infection. PLoS Pathog. 2018 Jul 12;14(7):e1007125. doi: 10.1371/journal.ppat.1007125 Macal M, Jo Y, Dallari S, Chang AY, Dai J, Swaminathan S, Wehrens EJ, Fitzgerald P, Zuniga EI. Self-Renewal and Toll-like Receptor Signaling Sustain Exhausted Plasmacytoid Dendritic Cells during Chronic Viral Infection. Immunity. 2018 Apr 17: 48(4):730-744.e5. doi:10.1016/j.immuni Dallari S, Macal M, Loureiro ME, Jo Y, Swanson L, Hesser C, Ghosh P, Zuniga EI. Src Family Kinases Fyn and Lyn are Constitutively Activated and Mediate Plasmacytoid Dendritic Cell Responses. Nat Commun. 2017 Apr 3;8:14830. doi: 10.1038/ncomms14830. Lewis GM, Wehrens EJ, Labarta-Bajo L, Streeck H, Zuniga EI. TGF- Receptor Maintains CD4 T Helper Cell Identity during Chronic Viral Infections. J Cling Invest. 2016 Oct 3;126(10):3799-3813. doi: 10.1172/JCI87041. Macal M, Tam MA, Hesser C, Di Domizio J, Leger P, Gilliet M, Zuniga EI. CD28 Deficiency Enhances Type I IFN Production by Murine Plasmacytoid Dendritic Cells. J Immunol. 2016 Feb 15;196(4):1900-9. doi: 10.4049/jimmunol.1501658. Zuniga EI, Macal M, Lewis GM, Harker JA. Innate and Adaptive Immune Regulation During Chronic Viral Infections. Annu Rev Virol. 2015 Nov;2(1):573-97. doi: 10.1146/annurev-virology-100114-055226. Harker JA, Wong KA, Dolgoter A, Zuniga EI. Cell-Intrinsic gp130 Signaling on CD4+ T Cells Shapes Long-Lasting Antiviral Immunity. J Immunol. 2015 Aug 1;195(3):1071-81. doi: 10.4049/jimmunol.1402402. Harker JA, Dolgoter A, Zuniga EI. Cell-Intrinsic IL-27 and gp130 Cytokine Receptor Signaling Regulates Virus-Specific CD4+ T Cell Responses and Viral Control during Chronic Infection. Immunity. 2013 Sep 19;39(3):548-59. doi: 10.1016/j.immuni.2013.08.010. Macal M, Lewis GM, Kunz S, Flavell R, Harker JA, Zuniga EI. Plasmacytoid Dendritic Cells are Productively Infected and Activated Through TLR-7 Early after Arenavirus Infection. Cell Host Microbe. 2012 Jun 14;11(6):617-30. doi: 10.1016/j.chom.2012.04.017. Harker JA, Lewis GM, Mack L, Zuniga EI. Late interleukin-6 Escalates T Follicular Helper Cell Responses and Controls a Chronic Viral Infection. Science. 2011 Nov 11;334(6057):825-9. doi: 10.1126/science.1208421. Tinoco R, Alcalde V, Yang Y, Sauer K, Zuniga EI. Cell-Intrinsic Transforming Growth Factor- Signaling Mediates Virus-Specific CD8+ T Cell Deletion and Viral Persistence In Vivo. Immunity. 2009 Jul 17;31(1):145-57. doi: 10.1016/j.immuni.2009.06.015. Zuniga EI, Liou LY, Mack L, Mendoza M, Oldstone MB. Persistent Virus Infection Inhibits Type I interferon Production by Plasmacytoid Dendritic Cells to Facilitate Opportunistic Infections. Cell Host Microbe. 2008 Oct 16;4(4):374-86. doi: 10.1016/j.chom.2008.08.016. Lauterbach H, Zuniga EI, Truong P, Oldstone MB, McGavern DB. Adoptive Immunotherapy Induces CNS Dendritic Cell Recruitment and Antigen Presentation during Clearance of a Persistent Viral Infection. J Exp Med. 2006 Aug 7;203(8):1963-75. Epub 2006 Jul 17. Zuniga EI, McGavern DB, Pruneda-Paz JL, Teng C, Oldstone MB. Bone Marrow Plasmacytoid Dendritic Cells can Differentiate into Myeloid Dendritic Cells upon Virus Infection. Nat Immunol. 2004 Dec;5(12):1227-34. Epub 2004 Nov 7. \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eF0dmFHsRIqi",
        "colab_type": "text"
      },
      "source": [
        "Finally, write urls and extracted bio to txt files"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "eL0ZjnaPRIqj",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def write_lst(lst,file_):\n",
        "  with open(file_,'w') as f:\n",
        "    for l in lst:\n",
        "        f.write(l)\n",
        "        f.write('\\n')"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "iCex1alZRIqm",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "bio_urls_file = 'bio_urls.txt'\n",
        "bios_file = 'bios.txt'\n",
        "write_lst(bio_urls,bio_urls_file)\n",
        "write_lst(bios,bios_file)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "z7a1cxjYVwGo",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        ""
      ],
      "execution_count": 0,
      "outputs": []
    }
  ]
}