{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.6.5"
    },
    "colab": {
      "name": "scraper.ipynb",
      "provenance": [],
      "include_colab_link": true
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/ashwinamrutphale/aml-homeworks/blob/master/scraper.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "GAv8JFIaRIp-",
        "colab_type": "text"
      },
      "source": [
        "# Scraping UIUC CS faculty homepages"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jVXCTmoWRIqA",
        "colab_type": "text"
      },
      "source": [
        "## System setup \n",
        "\n",
        "Before we start, make sure to install the required libraries\n",
        "    \n",
        "    pip install bs4\n",
        "    pip install selenium\n",
        "\n",
        "Since UIUC's website has some javascript rendered HTML content, we'll be using Selenium for scraping the content loaded dynamically by javascript. For this,you would also need to download a selenium supported browser webdriver.\n",
        "\n",
        "e.g. For Chrome, download the appropriate webdriver from here: http://chromedriver.chromium.org/downloads, unzip it and save in current directory."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "x3dwT4f0RPGI",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "cfc2b3ba-15f0-4099-9d52-692f7cd41859"
      },
      "source": [
        "!pip install bs4\n",
        "!pip install selenium\n",
        "!apt-get update # to update ubuntu to correctly run apt install\n",
        "!apt install chromium-chromedriver\n",
        "!cp /usr/lib/chromium-browser/chromedriver /usr/bin"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: bs4 in /usr/local/lib/python3.6/dist-packages (0.0.1)\n",
            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.6/dist-packages (from bs4) (4.6.3)\n",
            "Requirement already satisfied: selenium in /usr/local/lib/python3.6/dist-packages (3.141.0)\n",
            "Requirement already satisfied: urllib3 in /usr/local/lib/python3.6/dist-packages (from selenium) (1.24.3)\n",
            "Ign:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease\n",
            "Get:2 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]\n",
            "Ign:3 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease\n",
            "Get:4 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release [564 B]\n",
            "Get:5 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release [564 B]\n",
            "Get:6 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release.gpg [819 B]\n",
            "Get:7 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release.gpg [833 B]\n",
            "Hit:8 http://archive.ubuntu.com/ubuntu bionic InRelease\n",
            "Get:9 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ InRelease [3,626 B]\n",
            "Get:10 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease [21.3 kB]\n",
            "Get:11 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Packages [113 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]\n",
            "Get:13 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Packages [20.6 kB]\n",
            "Get:14 http://ppa.launchpad.net/marutter/c2d4u3.5/ubuntu bionic InRelease [15.4 kB]\n",
            "Get:15 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ Packages [70.9 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]\n",
            "Get:17 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [5,230 B]\n",
            "Get:18 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [672 kB]\n",
            "Get:19 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic/main amd64 Packages [31.7 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [21.0 kB]\n",
            "Get:21 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [10.4 kB]\n",
            "Get:22 http://ppa.launchpad.net/marutter/c2d4u3.5/ubuntu bionic/main Sources [1,709 kB]\n",
            "Get:23 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [771 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [1,288 kB]\n",
            "Get:25 http://ppa.launchpad.net/marutter/c2d4u3.5/ubuntu bionic/main amd64 Packages [822 kB]\n",
            "Get:26 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [967 kB]\n",
            "Get:27 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [8,310 B]\n",
            "Get:28 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [4,227 B]\n",
            "Fetched 6,811 kB in 5s (1,353 kB/s)\n",
            "Reading package lists... Done\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  chromium-browser chromium-browser-l10n chromium-codecs-ffmpeg-extra\n",
            "Suggested packages:\n",
            "  webaccounts-chromium-extension unity-chromium-extension adobe-flashplugin\n",
            "The following NEW packages will be installed:\n",
            "  chromium-browser chromium-browser-l10n chromium-chromedriver\n",
            "  chromium-codecs-ffmpeg-extra\n",
            "0 upgraded, 4 newly installed, 0 to remove and 134 not upgraded.\n",
            "Need to get 70.8 MB of archives.\n",
            "After this operation, 254 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 chromium-codecs-ffmpeg-extra amd64 77.0.3865.90-0ubuntu0.18.04.1 [1,079 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 chromium-browser amd64 77.0.3865.90-0ubuntu0.18.04.1 [62.3 MB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 chromium-browser-l10n all 77.0.3865.90-0ubuntu0.18.04.1 [3,011 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 chromium-chromedriver amd64 77.0.3865.90-0ubuntu0.18.04.1 [4,388 kB]\n",
            "Fetched 70.8 MB in 8s (8,689 kB/s)\n",
            "Selecting previously unselected package chromium-codecs-ffmpeg-extra.\n",
            "(Reading database ... 131183 files and directories currently installed.)\n",
            "Preparing to unpack .../chromium-codecs-ffmpeg-extra_77.0.3865.90-0ubuntu0.18.04.1_amd64.deb ...\n",
            "Unpacking chromium-codecs-ffmpeg-extra (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Selecting previously unselected package chromium-browser.\n",
            "Preparing to unpack .../chromium-browser_77.0.3865.90-0ubuntu0.18.04.1_amd64.deb ...\n",
            "Unpacking chromium-browser (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Selecting previously unselected package chromium-browser-l10n.\n",
            "Preparing to unpack .../chromium-browser-l10n_77.0.3865.90-0ubuntu0.18.04.1_all.deb ...\n",
            "Unpacking chromium-browser-l10n (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Selecting previously unselected package chromium-chromedriver.\n",
            "Preparing to unpack .../chromium-chromedriver_77.0.3865.90-0ubuntu0.18.04.1_amd64.deb ...\n",
            "Unpacking chromium-chromedriver (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Processing triggers for mime-support (3.60ubuntu1) ...\n",
            "Setting up chromium-codecs-ffmpeg-extra (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n",
            "Processing triggers for hicolor-icon-theme (0.17-2) ...\n",
            "Setting up chromium-browser (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "update-alternatives: using /usr/bin/chromium-browser to provide /usr/bin/x-www-browser (x-www-browser) in auto mode\n",
            "update-alternatives: using /usr/bin/chromium-browser to provide /usr/bin/gnome-www-browser (gnome-www-browser) in auto mode\n",
            "Setting up chromium-chromedriver (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "Setting up chromium-browser-l10n (77.0.3865.90-0ubuntu0.18.04.1) ...\n",
            "cp: '/usr/lib/chromium-browser/chromedriver' and '/usr/bin/chromedriver' are the same file\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZX3XpFqQRIqB",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "from bs4 import BeautifulSoup\n",
        "from selenium import webdriver \n",
        "from selenium.webdriver.chrome.options import Options\n",
        "import re \n",
        "import urllib"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "I03pA0HMRIqF",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 71
        },
        "outputId": "7c8e543a-ca3f-4936-bee5-bbbf294a4d6e"
      },
      "source": [
        "#create a webdriver object and set options for headless browsing\n",
        "import sys\n",
        "sys.path.insert(0,'/usr/lib/chromium-browser/chromedriver')\n",
        "from selenium import webdriver\n",
        "chrome_options = webdriver.ChromeOptions()\n",
        "chrome_options.add_argument('--headless')\n",
        "chrome_options.add_argument('--no-sandbox')\n",
        "chrome_options.add_argument('--disable-dev-shm-usage')\n",
        "browser = webdriver.Chrome('chromedriver',chrome_options=chrome_options)\n"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.6/dist-packages/ipykernel_launcher.py:8: DeprecationWarning: use options instead of chrome_options\n",
            "  \n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FDuWNcnGRIqJ",
        "colab_type": "text"
      },
      "source": [
        "If you visit UIUC's CS Faculty Directory Listing: https://cs.illinois.edu/people/faculty/department-faculty , you'll notice that it has all the faculty listed there. \n",
        "Clicking on a faculty's image takes you to the Faculty Profile page. \n",
        "The profile page has more info about the faculty and sometimes contains the faculty hompage url under the Overview Tab > For More Information which is ultimately what we want to scrape.\n",
        "\n",
        "Before we start scraping, we'll define some helper functions"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Bc5RW7r5RIqK",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#uses webdriver object to execute javascript code and get dynamically loaded webcontent\n",
        "def get_js_soup(url,browser):\n",
        "    browser.get(url)\n",
        "    res_html = browser.execute_script('return document.body.innerHTML')\n",
        "    soup = BeautifulSoup(res_html,'html.parser') #beautiful soup object to be used for parsing html content\n",
        "    return soup\n",
        "\n",
        "#tidies extracted text \n",
        "def process_bio(bio):\n",
        "    bio = bio.encode('ascii',errors='ignore').decode('utf-8')       #removes non-ascii characters\n",
        "    bio = re.sub('\\s+',' ',bio)       #repalces repeated whitespace characters with single space\n",
        "    return bio\n",
        "\n",
        "''' More tidying\n",
        "Sometimes the text extracted HTML webpage may contain javascript code and some style elements. \n",
        "This function removes script and style tags from HTML so that extracted text does not contain them.\n",
        "'''\n",
        "def remove_script(soup):\n",
        "    for script in soup([\"script\", \"style\"]):\n",
        "        script.decompose()\n",
        "    return soup\n",
        "\n",
        "\n",
        "#Checks if bio_url is a valid faculty homepage\n",
        "def is_valid_homepage(bio_url,dir_url):\n",
        "    try:\n",
        "        #sometimes the homepage url points to the faculty profile page\n",
        "        #which should be treated differently from an actual homepage\n",
        "        ret_url = urllib.request.urlopen(bio_url).geturl() \n",
        "    except:\n",
        "        return False       #unable to access bio_url\n",
        "    urls = [re.sub('((https?://)|(www.))','',url) for url in [ret_url,dir_url]] #removes url scheme (https,http) or www \n",
        "    return not(urls[0]== urls[1])"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "d2dat_ShRIqP",
        "colab_type": "text"
      },
      "source": [
        "We will now start scraping.\n",
        "\n",
        "First, let's get links to all Faculty Profile pages by scraping the Directory Listing. You can use your browser's developer tools to find the required links within the HTML content. In Chrome, this can be done by right cliking on the webpage and choosing Inspect. Some basic knowledge of HTML and CSS would be required. After a bit of digging, you should notice that the link can be found under the <a\\> tag under <div\\> with class \"photo nocaption\" as shown:\n",
        "\n",
        "\n",
        "![Screen%20Shot%202019-02-11%20at%2012.39.36%20PM.png](attachment:Screen%20Shot%202019-02-11%20at%2012.39.36%20PM.png)\n",
        "\n",
        "Now we can specify exactly what needs to be extracted from the directory listing page"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6yi8Ql1JRIqQ",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#extracts all Faculty Profile page urls from the Directory Listing Page\n",
        "def scrape_dir_page(dir_url,browser):\n",
        "    print ('-'*20,'Scraping directory page','-'*20)\n",
        "    faculty_links = []\n",
        "    faculty_base_url = 'https://biology.ucsd.edu'\n",
        "    #execute js on webpage to load faculty listings on webpage and get ready to parse the loaded HTML \n",
        "    soup = get_js_soup(dir_url,browser)     \n",
        "    for link_holder in soup.find_all('td',class_='name'): #get list of all <div> of class 'photo nocaption'\n",
        "        #print(link_holder)\n",
        "        try:\n",
        "          rel_link = link_holder.find('a')['href'] #get url\n",
        "          print(rel_link)\n",
        "          #url returned is relative, so we need to add base url\n",
        "          faculty_links.append(faculty_base_url+rel_link) \n",
        "        except Exception:\n",
        "          pass  # o\n",
        "        \n",
        "    print ('-'*20,'Found {} faculty profile urls'.format(len(faculty_links)),'-'*20)\n",
        "    return faculty_links"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "sYOOc14ERIqU",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "5054f8c0-5708-4394-bd02-739498db7e0e"
      },
      "source": [
        "dir_url = 'https://biology.ucsd.edu/publicinfo/dwc?action=faculty_research_list' #url of directory listings of CS faculty\n",
        "faculty_links = scrape_dir_page(dir_url,browser)\n",
        "print(faculty_links)"
      ],
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "-------------------- Scraping directory page --------------------\n",
            "/research/faculty/sackerman\n",
            "/research/faculty/oakbari\n",
            "/research/faculty/eallen\n",
            "/research/faculty/njallen\n",
            "/research/faculty/carmour\n",
            "/research/faculty/kasahina\n",
            "/research/faculty/jsayres\n",
            "/research/faculty/eazim\n",
            "/research/faculty/tsb\n",
            "/research/faculty/mbanghart\n",
            "/research/faculty/adbarton\n",
            "/research/faculty/e1bennett\n",
            "/research/faculty/dberg\n",
            "/research/faculty/ebier\n",
            "/research/faculty/blbloodgood\n",
            "/research/faculty/sbriggs\n",
            "/research/faculty/sbrody\n",
            "/research/faculty/wbusch\n",
            "/research/faculty/mabutler\n",
            "/research/faculty/hcang\n",
            "/research/faculty/schalasani\n",
            "/research/faculty/lchao\n",
            "/research/faculty/lchilukuri\n",
            "/research/faculty/adchisholm\n",
            "/research/faculty/schoe\n",
            "/research/faculty/jchory\n",
            "/research/faculty/mchrispeels\n",
            "/research/faculty/ecleland\n",
            "/research/faculty/abcoleman\n",
            "/research/faculty/hcookandersen\n",
            "/research/faculty/j2cooke\n",
            "/research/faculty/kcooper\n",
            "/research/faculty/ncrawford\n",
            "/research/faculty/mddaugherty\n",
            "/research/faculty/midavid\n",
            "/research/faculty/abdesai\n",
            "/research/faculty/rdoolittle\n",
            "/research/faculty/rjdutton\n",
            "/research/faculty/jecker\n",
            "/research/faculty/dengle\n",
            "/research/faculty/mestelle\n",
            "/research/faculty/rmevans\n",
            "/research/faculty/efarley\n",
            "/research/faculty/gfeng\n",
            "/research/faculty/rafirtel\n",
            "/research/faculty/dforbes\n",
            "/research/faculty/pfortes\n",
            "/research/faculty/kfrench\n",
            "/research/faculty/fgage\n",
            "/research/faculty/pgeiduschek\n",
            "/research/faculty/tgentner\n",
            "/research/faculty/sglasgow\n",
            "/research/faculty/jwgolden\n",
            "/research/faculty/sgolden\n",
            "/research/faculty/agoldrath\n",
            "/research/faculty/mgoulding\n",
            "/research/faculty/bjgrant\n",
            "/research/faculty/mgreen\n",
            "/research/faculty/rgreenspan\n",
            "/research/faculty/shalpain\n",
            "/research/faculty/rhampton\n",
            "/research/faculty/shan\n",
            "/research/faculty/nhao\n",
            "/research/faculty/dhargreaves\n",
            "/research/faculty/jhasty\n",
            "/research/faculty/shedrick\n",
            "/research/faculty/dhelinski\n",
            "/research/faculty/mhetzer\n",
            "/research/faculty/dholway\n",
            "/research/faculty/ahuffaker\n",
            "/research/faculty/enfuhui\n",
            "/research/faculty/arhunter\n",
            "/research/faculty/thwa\n",
            "/research/faculty/jbelmonte\n",
            "/research/faculty/xij014\n",
            "/research/faculty/yijin\n",
            "/research/faculty/k5jones\n",
            "/research/faculty/ajuavinett\n",
            "/research/faculty/s2jun\n",
            "/research/faculty/jkadonaga\n",
            "/research/faculty/smkaech\n",
            "/research/faculty/jkarlseder\n",
            "/research/faculty/akiger\n",
            "/research/faculty/ckintner\n",
            "/research/faculty/dkleinfeld\n",
            "/research/faculty/jkohn\n",
            "/research/faculty/tkomiyama\n",
            "/research/faculty/wkristan\n",
            "/research/faculty/mkronenberg\n",
            "/research/faculty/skryazhimskiy\n",
            "/research/faculty/ckurle\n",
            "/research/faculty/rlande\n",
            "/research/faculty/slauberth\n",
            "/research/faculty/j2law\n",
            "/research/faculty/kflee\n",
            "/research/faculty/aleschziner\n",
            "/research/faculty/jleutgeb\n",
            "/research/faculty/sleutgeb\n",
            "/research/faculty/blillemeier\n",
            "/research/faculty/byl015\n",
            "/research/faculty/smlo\n",
            "/research/faculty/lil034\n",
            "/research/faculty/vlundblad\n",
            "/research/faculty/jlykkeandersen\n",
            "/research/faculty/dmlyumkis\n",
            "/research/faculty/emacagno\n",
            "/research/faculty/cmadigan\n",
            "/research/faculty/rmalinow\n",
            "/research/faculty/tmarkow\n",
            "/research/faculty/smayfield\n",
            "/research/faculty/lmcdonnell\n",
            "/research/faculty/wmcginnis\n",
            "/research/faculty/smel\n",
            "/research/faculty/jrmeyer\n",
            "/research/faculty/mmontal\n",
            "/research/faculty/montminy\n",
            "/research/faculty/cmurre\n",
            "/research/faculty/seneal\n",
            "/research/faculty/jnieh\n",
            "/research/faculty/animmerjahn\n",
            "/research/faculty/mniwarosen\n",
            "/research/faculty/coshea\n",
            "/research/faculty/koegema\n",
            "/research/faculty/mtowens\n",
            "/research/faculty/spanda\n",
            "/research/faculty/apasquinelli\n",
            "/research/faculty/gpatrick\n",
            "/research/faculty/gpekkurnaz\n",
            "/research/faculty/mperry\n",
            "/research/faculty/kpetrie\n",
            "/research/faculty/sapfaff\n",
            "/research/faculty/lpillus\n",
            "/research/faculty/jpogliano\n",
            "/research/faculty/kpogliano\n",
            "/research/faculty/jposakony\n",
            "/research/faculty/pprice\n",
            "/research/faculty/jprunedapaz\n",
            "/research/faculty/sreckpeterson\n",
            "/research/faculty/preinagel\n",
            "/research/faculty/sarifkin\n",
            "/research/faculty/cmroot\n",
            "/research/faculty/mrosenfeld\n",
            "/research/faculty/kroy\n",
            "/research/faculty/arussell\n",
            "/research/faculty/oryder\n",
            "/research/faculty/asaghatelian\n",
            "/research/faculty/msaier\n",
            "/research/faculty/mschaechter\n",
            "/research/faculty/ischeffler\n",
            "/research/faculty/eschmelz\n",
            "/research/faculty/jischroeder\n",
            "/research/faculty/tsejnowski\n",
            "/research/faculty/gsshadel\n",
            "/research/faculty/rshaw\n",
            "/research/faculty/jshurin\n",
            "/research/faculty/dsmith\n",
            "/research/faculty/lgsmith\n",
            "/research/faculty/nspitzer\n",
            "/research/faculty/sstockwell\n",
            "/research/faculty/c8su\n",
            "/research/faculty/ssubramani\n",
            "/research/faculty/gsuel\n",
            "/research/faculty/xis101\n",
            "/research/faculty/etour\n",
            "/research/faculty/dtraver\n",
            "/research/faculty/etroemel\n",
            "/research/faculty/kmtye\n",
            "/research/faculty/evilla\n",
            "/research/faculty/gwahl\n",
            "/research/faculty/jw800\n",
            "/research/faculty/cware\n",
            "/research/faculty/sawasserman\n",
            "/research/faculty/gwienhausen\n",
            "/research/faculty/jwilhelm\n",
            "/research/faculty/cwills\n",
            "/research/faculty/yangxu\n",
            "/research/faculty/myanofsky\n",
            "/research/faculty/dyelon\n",
            "/research/faculty/d7zhang\n",
            "/research/faculty/y3zhao\n",
            "/research/faculty/yezheng\n",
            "/research/faculty/yzou\n",
            "/research/faculty/eizuniga\n",
            "-------------------- Found 183 faculty profile urls --------------------\n",
            "['https://biology.ucsd.edu/research/faculty/sackerman', 'https://biology.ucsd.edu/research/faculty/oakbari', 'https://biology.ucsd.edu/research/faculty/eallen', 'https://biology.ucsd.edu/research/faculty/njallen', 'https://biology.ucsd.edu/research/faculty/carmour', 'https://biology.ucsd.edu/research/faculty/kasahina', 'https://biology.ucsd.edu/research/faculty/jsayres', 'https://biology.ucsd.edu/research/faculty/eazim', 'https://biology.ucsd.edu/research/faculty/tsb', 'https://biology.ucsd.edu/research/faculty/mbanghart', 'https://biology.ucsd.edu/research/faculty/adbarton', 'https://biology.ucsd.edu/research/faculty/e1bennett', 'https://biology.ucsd.edu/research/faculty/dberg', 'https://biology.ucsd.edu/research/faculty/ebier', 'https://biology.ucsd.edu/research/faculty/blbloodgood', 'https://biology.ucsd.edu/research/faculty/sbriggs', 'https://biology.ucsd.edu/research/faculty/sbrody', 'https://biology.ucsd.edu/research/faculty/wbusch', 'https://biology.ucsd.edu/research/faculty/mabutler', 'https://biology.ucsd.edu/research/faculty/hcang', 'https://biology.ucsd.edu/research/faculty/schalasani', 'https://biology.ucsd.edu/research/faculty/lchao', 'https://biology.ucsd.edu/research/faculty/lchilukuri', 'https://biology.ucsd.edu/research/faculty/adchisholm', 'https://biology.ucsd.edu/research/faculty/schoe', 'https://biology.ucsd.edu/research/faculty/jchory', 'https://biology.ucsd.edu/research/faculty/mchrispeels', 'https://biology.ucsd.edu/research/faculty/ecleland', 'https://biology.ucsd.edu/research/faculty/abcoleman', 'https://biology.ucsd.edu/research/faculty/hcookandersen', 'https://biology.ucsd.edu/research/faculty/j2cooke', 'https://biology.ucsd.edu/research/faculty/kcooper', 'https://biology.ucsd.edu/research/faculty/ncrawford', 'https://biology.ucsd.edu/research/faculty/mddaugherty', 'https://biology.ucsd.edu/research/faculty/midavid', 'https://biology.ucsd.edu/research/faculty/abdesai', 'https://biology.ucsd.edu/research/faculty/rdoolittle', 'https://biology.ucsd.edu/research/faculty/rjdutton', 'https://biology.ucsd.edu/research/faculty/jecker', 'https://biology.ucsd.edu/research/faculty/dengle', 'https://biology.ucsd.edu/research/faculty/mestelle', 'https://biology.ucsd.edu/research/faculty/rmevans', 'https://biology.ucsd.edu/research/faculty/efarley', 'https://biology.ucsd.edu/research/faculty/gfeng', 'https://biology.ucsd.edu/research/faculty/rafirtel', 'https://biology.ucsd.edu/research/faculty/dforbes', 'https://biology.ucsd.edu/research/faculty/pfortes', 'https://biology.ucsd.edu/research/faculty/kfrench', 'https://biology.ucsd.edu/research/faculty/fgage', 'https://biology.ucsd.edu/research/faculty/pgeiduschek', 'https://biology.ucsd.edu/research/faculty/tgentner', 'https://biology.ucsd.edu/research/faculty/sglasgow', 'https://biology.ucsd.edu/research/faculty/jwgolden', 'https://biology.ucsd.edu/research/faculty/sgolden', 'https://biology.ucsd.edu/research/faculty/agoldrath', 'https://biology.ucsd.edu/research/faculty/mgoulding', 'https://biology.ucsd.edu/research/faculty/bjgrant', 'https://biology.ucsd.edu/research/faculty/mgreen', 'https://biology.ucsd.edu/research/faculty/rgreenspan', 'https://biology.ucsd.edu/research/faculty/shalpain', 'https://biology.ucsd.edu/research/faculty/rhampton', 'https://biology.ucsd.edu/research/faculty/shan', 'https://biology.ucsd.edu/research/faculty/nhao', 'https://biology.ucsd.edu/research/faculty/dhargreaves', 'https://biology.ucsd.edu/research/faculty/jhasty', 'https://biology.ucsd.edu/research/faculty/shedrick', 'https://biology.ucsd.edu/research/faculty/dhelinski', 'https://biology.ucsd.edu/research/faculty/mhetzer', 'https://biology.ucsd.edu/research/faculty/dholway', 'https://biology.ucsd.edu/research/faculty/ahuffaker', 'https://biology.ucsd.edu/research/faculty/enfuhui', 'https://biology.ucsd.edu/research/faculty/arhunter', 'https://biology.ucsd.edu/research/faculty/thwa', 'https://biology.ucsd.edu/research/faculty/jbelmonte', 'https://biology.ucsd.edu/research/faculty/xij014', 'https://biology.ucsd.edu/research/faculty/yijin', 'https://biology.ucsd.edu/research/faculty/k5jones', 'https://biology.ucsd.edu/research/faculty/ajuavinett', 'https://biology.ucsd.edu/research/faculty/s2jun', 'https://biology.ucsd.edu/research/faculty/jkadonaga', 'https://biology.ucsd.edu/research/faculty/smkaech', 'https://biology.ucsd.edu/research/faculty/jkarlseder', 'https://biology.ucsd.edu/research/faculty/akiger', 'https://biology.ucsd.edu/research/faculty/ckintner', 'https://biology.ucsd.edu/research/faculty/dkleinfeld', 'https://biology.ucsd.edu/research/faculty/jkohn', 'https://biology.ucsd.edu/research/faculty/tkomiyama', 'https://biology.ucsd.edu/research/faculty/wkristan', 'https://biology.ucsd.edu/research/faculty/mkronenberg', 'https://biology.ucsd.edu/research/faculty/skryazhimskiy', 'https://biology.ucsd.edu/research/faculty/ckurle', 'https://biology.ucsd.edu/research/faculty/rlande', 'https://biology.ucsd.edu/research/faculty/slauberth', 'https://biology.ucsd.edu/research/faculty/j2law', 'https://biology.ucsd.edu/research/faculty/kflee', 'https://biology.ucsd.edu/research/faculty/aleschziner', 'https://biology.ucsd.edu/research/faculty/jleutgeb', 'https://biology.ucsd.edu/research/faculty/sleutgeb', 'https://biology.ucsd.edu/research/faculty/blillemeier', 'https://biology.ucsd.edu/research/faculty/byl015', 'https://biology.ucsd.edu/research/faculty/smlo', 'https://biology.ucsd.edu/research/faculty/lil034', 'https://biology.ucsd.edu/research/faculty/vlundblad', 'https://biology.ucsd.edu/research/faculty/jlykkeandersen', 'https://biology.ucsd.edu/research/faculty/dmlyumkis', 'https://biology.ucsd.edu/research/faculty/emacagno', 'https://biology.ucsd.edu/research/faculty/cmadigan', 'https://biology.ucsd.edu/research/faculty/rmalinow', 'https://biology.ucsd.edu/research/faculty/tmarkow', 'https://biology.ucsd.edu/research/faculty/smayfield', 'https://biology.ucsd.edu/research/faculty/lmcdonnell', 'https://biology.ucsd.edu/research/faculty/wmcginnis', 'https://biology.ucsd.edu/research/faculty/smel', 'https://biology.ucsd.edu/research/faculty/jrmeyer', 'https://biology.ucsd.edu/research/faculty/mmontal', 'https://biology.ucsd.edu/research/faculty/montminy', 'https://biology.ucsd.edu/research/faculty/cmurre', 'https://biology.ucsd.edu/research/faculty/seneal', 'https://biology.ucsd.edu/research/faculty/jnieh', 'https://biology.ucsd.edu/research/faculty/animmerjahn', 'https://biology.ucsd.edu/research/faculty/mniwarosen', 'https://biology.ucsd.edu/research/faculty/coshea', 'https://biology.ucsd.edu/research/faculty/koegema', 'https://biology.ucsd.edu/research/faculty/mtowens', 'https://biology.ucsd.edu/research/faculty/spanda', 'https://biology.ucsd.edu/research/faculty/apasquinelli', 'https://biology.ucsd.edu/research/faculty/gpatrick', 'https://biology.ucsd.edu/research/faculty/gpekkurnaz', 'https://biology.ucsd.edu/research/faculty/mperry', 'https://biology.ucsd.edu/research/faculty/kpetrie', 'https://biology.ucsd.edu/research/faculty/sapfaff', 'https://biology.ucsd.edu/research/faculty/lpillus', 'https://biology.ucsd.edu/research/faculty/jpogliano', 'https://biology.ucsd.edu/research/faculty/kpogliano', 'https://biology.ucsd.edu/research/faculty/jposakony', 'https://biology.ucsd.edu/research/faculty/pprice', 'https://biology.ucsd.edu/research/faculty/jprunedapaz', 'https://biology.ucsd.edu/research/faculty/sreckpeterson', 'https://biology.ucsd.edu/research/faculty/preinagel', 'https://biology.ucsd.edu/research/faculty/sarifkin', 'https://biology.ucsd.edu/research/faculty/cmroot', 'https://biology.ucsd.edu/research/faculty/mrosenfeld', 'https://biology.ucsd.edu/research/faculty/kroy', 'https://biology.ucsd.edu/research/faculty/arussell', 'https://biology.ucsd.edu/research/faculty/oryder', 'https://biology.ucsd.edu/research/faculty/asaghatelian', 'https://biology.ucsd.edu/research/faculty/msaier', 'https://biology.ucsd.edu/research/faculty/mschaechter', 'https://biology.ucsd.edu/research/faculty/ischeffler', 'https://biology.ucsd.edu/research/faculty/eschmelz', 'https://biology.ucsd.edu/research/faculty/jischroeder', 'https://biology.ucsd.edu/research/faculty/tsejnowski', 'https://biology.ucsd.edu/research/faculty/gsshadel', 'https://biology.ucsd.edu/research/faculty/rshaw', 'https://biology.ucsd.edu/research/faculty/jshurin', 'https://biology.ucsd.edu/research/faculty/dsmith', 'https://biology.ucsd.edu/research/faculty/lgsmith', 'https://biology.ucsd.edu/research/faculty/nspitzer', 'https://biology.ucsd.edu/research/faculty/sstockwell', 'https://biology.ucsd.edu/research/faculty/c8su', 'https://biology.ucsd.edu/research/faculty/ssubramani', 'https://biology.ucsd.edu/research/faculty/gsuel', 'https://biology.ucsd.edu/research/faculty/xis101', 'https://biology.ucsd.edu/research/faculty/etour', 'https://biology.ucsd.edu/research/faculty/dtraver', 'https://biology.ucsd.edu/research/faculty/etroemel', 'https://biology.ucsd.edu/research/faculty/kmtye', 'https://biology.ucsd.edu/research/faculty/evilla', 'https://biology.ucsd.edu/research/faculty/gwahl', 'https://biology.ucsd.edu/research/faculty/jw800', 'https://biology.ucsd.edu/research/faculty/cware', 'https://biology.ucsd.edu/research/faculty/sawasserman', 'https://biology.ucsd.edu/research/faculty/gwienhausen', 'https://biology.ucsd.edu/research/faculty/jwilhelm', 'https://biology.ucsd.edu/research/faculty/cwills', 'https://biology.ucsd.edu/research/faculty/yangxu', 'https://biology.ucsd.edu/research/faculty/myanofsky', 'https://biology.ucsd.edu/research/faculty/dyelon', 'https://biology.ucsd.edu/research/faculty/d7zhang', 'https://biology.ucsd.edu/research/faculty/y3zhao', 'https://biology.ucsd.edu/research/faculty/yezheng', 'https://biology.ucsd.edu/research/faculty/yzou', 'https://biology.ucsd.edu/research/faculty/eizuniga']\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ydrh0Tl5RIqZ",
        "colab_type": "text"
      },
      "source": [
        "Now let's scrape the Faculty Profile pages to find faculty homepage urls. \n",
        "\n",
        "The faculty homepage urls are listed under the Overview tab > For more information on the Faculty profile page. So, again let's try to find where these urls are placed in the HTML content. \n",
        "\n",
        "You'll notice the Overview tab is the <div\\> with id <overview\\>. \n",
        "\n",
        "There could multiple headers (<h\\>) under this tab e.g. Research etc., we're looking for the one with text \"For more information\".\n",
        "\n",
        "There could be multiple listings (<li\\>) under the header e.g. Reasearch Group Page, etc. There is no specific way to identify the one corresponding to the homepage. But, if you look at a few profile pages, you'll notice a few patterns which are generally found in the anchor text accompanying the homepage url. e.g. presence of substrings like 'page' or 'site' are good indicators as long as other words like 'group' or 'course' are not present. \n",
        "\n",
        "Ofcourse, there might still be a few cases where the patterns do not work. For such cases and for cases where the homepage url is actually not listed on the profile page, we will treat the faculty profile page as the homepage.\n",
        "\n",
        "\n",
        "![Screen%20Shot%202019-02-11%20at%201.36.02%20PM.png](attachment:Screen%20Shot%202019-02-11%20at%201.36.02%20PM.png)\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "HuiseIfXRIqa",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def scrape_faculty_page(fac_url,browser):\n",
        "\n",
        "    soup = get_js_soup(fac_url,browser)\n",
        "    homepage_found = False\n",
        "    bio_url = ''\n",
        "    bio = ''\n",
        "    #find overview tab\n",
        "    try:\n",
        "        overview_tab = soup.find('div',id='bio')\n",
        "        research_tab = soup.find('div',id='research')\n",
        "    except:\n",
        "        print (soup)\n",
        "    #define patterns for the anchor text of homepage url\n",
        "    faculty_name = soup.find('h1',id='facName').get_text().lower() #find faculty name\n",
        "    bio = bio + faculty_name + ' '\n",
        "    try:\n",
        "      bio = bio + process_bio(overview_tab.get_text(separator=' ') + ' ')\n",
        "      bio = bio + process_bio(research_tab.get_text(separator=' ') + ' ')\n",
        "    except:\n",
        "      pass\n",
        "    print(bio)\n",
        "    \n",
        "#     homepage_txts = ['site','page',' '+faculty_last_name]\n",
        "#     exceptions = ['course ','research','group','cs','mirror','google scholar']\n",
        "#     #find the homepage url and extract all text from it\n",
        "#     for hdr in overview_tab.find_all('h2'):  #first find the required header\n",
        "#         if hdr.text.lower() == 'for more information':\n",
        "#             next_tag = hdr.find_next('li')\n",
        "#             #find <li> which has homepage url\n",
        "#             while next_tag is not None: \n",
        "#                 cand = next_tag.find('a')\n",
        "#                 next_tag = next_tag.next_sibling    #sibling means element present at the same level\n",
        "#                 cand_text = cand.get_text().lower()\n",
        "#                 if (any(hp_txt in cand_text for hp_txt in homepage_txts) and \n",
        "#                     not any(e in cand_text for e in exceptions)): #compare text to predefined patterns\n",
        "#                     bio_url = cand['href'] \n",
        "#                     homepage_found = True\n",
        "#                     #check if homepage url is valid\n",
        "#                     if not(is_valid_homepage(bio_url,fac_url)): \n",
        "#                         homepage_found = False\n",
        "#                     else:\n",
        "#                         try:`\n",
        "#                             bio_soup = remove_script(get_js_soup(bio_url,browser)) \n",
        "#                         except:\n",
        "#                             print ('Could not access {}'.format(bio_url))\n",
        "#                             homepage_found = False\n",
        "#                     break \n",
        "#             if homepage_found:\n",
        "#                 #get all the text from homepage(bio) since there's no easy to filter noise like navigation bar etc\n",
        "#                 bio = process_bio(bio_soup.get_text(separator=' ')) \n",
        "\n",
        "#     if not homepage_found:\n",
        "#         bio_url = fac_url #treat faculty profile page as homepage\n",
        "#         #we're only interested in some parts of the profile page namely the address\n",
        "#         #and information listed under the Overview, Research, Publication and Awards tab\n",
        "#         bio = soup.find('div',class_='dirprofile fullpage').get_text(separator=' ')+': '\n",
        "#         for tab in soup.find_all('div',class_='tab-pane'):\n",
        "#             bio += tab.get_text(separator=' ')+'. '\n",
        "#         bio = process_bio(bio)\n",
        "\n",
        "    return fac_url,bio"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WbR5rlBbRIqe",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "6188b27d-5955-4360-efa7-ff04fc0eaef4"
      },
      "source": [
        "#Scrape all faculty homepages using profile page urls\n",
        "bio_urls, bios = [],[]\n",
        "tot_urls = len(faculty_links)\n",
        "for i,link in enumerate(faculty_links):\n",
        "    print ('-'*20,'Scraping faculty url {}/{}'.format(i+1,tot_urls),'-'*20)\n",
        "    bio_url,bio = scrape_faculty_page(link,browser)\n",
        "    bio_urls.append(bio_url)\n",
        "    bios.append(bio)"
      ],
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "-------------------- Scraping faculty url 1/183 --------------------\n",
            "susan ackerman  Biography Susan Ackerman received her Ph.D. from UCLA and was a postdoctoral fellow at University of Illinois Medical School and the Wistar Institute. Prior to her move to UCSD in 2016, Dr. Ackerman was a Professor at The Jackson Laboratory in Bar Harbor, Maine where she was a faculty member for nineteen years. She has been an Investigator of the Howard Hughes Medical Institute since 2005.  Research The goal of our laboratory is to define the molecular pathways necessary to maintain homeostasis in both developing and aging mammalian neurons. To do this we utilize forward genetics to identify mutations that are associated with loss of neurons in the aging mouse brain. To further dissect pathways underlying homeostatic disruption and disease, we also use forward genetics to identify genetic variants that enhance or suppress neural phenotypes. Our approach allows the identification, without a priori assumptions, of molecules critical for neuron homeostasis and survival, and indeed we have discovered disruptions in several novel pathways that were not previously associated with loss of neuronal function or survival. We are particularly interested in the role of alterations in translation elongation, translational fidelity, proteostasis, and RNA metabolism in neuronal function. \n",
            "-------------------- Scraping faculty url 2/183 --------------------\n",
            "omar akbari  Biography In May of 2005, Omar S. Akbari received a B.S./M.S. in Biotechnology from the University of Nevada, Reno . In December of 2008, he received a Ph.D. in Cell and Molecular Biology from the University of Nevada, Reno where he studied the role cis-regulatory modules play in cellular identity along the anterio-posterior axis in developing Drosophila melanogaster embryos. In May of 2009, he joined the laboratory of professor Bruce A. Hay at the California Institute of Technology as a Senior Postdoctoral Scholar to innovate synthetic biology of disease vectors. In summer of 2015, he became an Assistant Professor of Entomology in the Center for Infectious Disease Vector Research (CIDVR) at the University of California, Riverside . In fall of 2017, he joined the faculty as an Assistant Professor in the Cell and Developmental Biology Section , within the Division of Biological Sciences , at the University of California, San Diego .  Research Mosquitoes are perhaps the most dangerous animals in the world. They are the primary vectors for major human diseases such as yellow fever, zika, dengue fever, and malaria, which together infect hundreds of millions of humans worldwide, killing millions each year, with over 50% of the worlds population presently at risk (WHO). There are currently no effective vaccines for either dengue fever, ZIKA or malaria, and mosquitoes, including the disease-causing agents they transmit, are rapidly evolving resistance to commonly used pesticides and anti-malarial drugs, respectively. Therefore, given the number of infections and deaths, current approaches for prevention of mosquito-borne diseases are immeasurably inefficient. What remains critical for vector control is the development of catalytic approaches requiring only small efforts that can generate long lasting solutions. With rapid advances in insect genetic engineering, mathematical modeling of wild populations, synthetic biology, and the comprehensive understanding of dengue and plasmodium lifecycles in mosquitoes, unique opportunities have arisen to eliminate infectious diseases through genetic manipulation of wild insect vector populations. Therefore, our research focuses on studying the basic genetics and physiology of mosquitoes with the overall goal of developing innovative, novel, creative, synthetic biology inspired genetic control technologies for reducing the burden of mosquito-borne diseases on humans. The underlying hypothesis inspiring this work is that the introduction and spread of genes that prevent mosquitoes from transmitting pathogens should, in theory, lead to reduced transmission of these pathogens resulting in reductions of human infections and/or death. To test this hypothesis, we first need a broad understanding of the biology of the mosquito that can be used to develop gene-based strategies for engineering mosquitoes that are resistant to pathogens; secondly, we need to engineer mosquitoes that are resistant to all types of infections; and lastly, we need to develop tools to rapidly drive these laboratory developed genes into wild mosquito populations. Together, these aims can conceivably provide a foundation that has the potential to revolutionize vector control of mosquitoes. \n",
            "-------------------- Scraping faculty url 3/183 --------------------\n",
            "eric allen  Biography Eric Allen received his B.S. from the University of Oregon and his Ph.D. from the Scripps Institution of Oceanography at UC San Diego. He performed postdoctoral research at the University of California, Berkeley as an NSF Postdoctoral Fellow in Microbial Biology and joined the faculty of Biological Sciences and the Scripps Institution of Oceangraphy at UCSD in 2006.  Research Research in our laboratory centers upon the use of environmentally-derived genome sequence information (metagenomics) to explore the genetic potential, ecology, and evolution of environmental microbial populations. The nature of this work relies equally upon field-based collections, bioinformatics (tool development, genome assembly, annotation, and comparative analyses) and the tools of molecular ecology and genetics. Together, these approaches enable us to utilize environmental genome sequence data to understand natural microbial phenomena including metabolic activities, biosynthetic processes, environmental adaptation, lateral gene transfer events, biogeographical patterning, biogeochemical cycling, and microbe-microbe/microbe-host interactions. Current projects include: Marine Metazoan Microbiomes Marine microorganisms are major contributors to biogeochemical cycling and the maintenance of global homeostasis, yet relatively little is known about the diversity and function of microbes found in intimate association with marine animals. Our current understanding of these complex microbial communities has largely been built upon cultivation-based studies and limited contemporary studies using cultivation-independent methods that survey 16S rRNA gene marker diversity. We use both vertebrate (marine fish) and invertebrate (sponge, mollusks) to investigate the host-associated microbial biology of our oceans, employing metabolic functional gene markers and deep metagenomic sequencing and assembly to complement additional phylogenetic techniques such as 16S rRNA gene amplicon sequencing. We are also developing laboratory-based bioreactor cultivation systems to propagate host-associated microbial communities under controlled conditions to allow experimental manipulation of host microbiota in vitro . These studies are revealing novel microbial diversity previously unidentified in the marine environment and provide new insight into symbiotic relationships between marine animals and their microbial partners. Genomics of Marine Microbial HOC Synthesis Halogenated organic compounds (HOCs) are widely distributed among organisms occupying higher trophic levels of the marine food web. Chemical analysis of environmental and biological samples has shown that these HOCs originate from both natural and anthropogenic sources. In this work we seek a molecular-level understanding of the microbial genes and enzymatic processes responsible for the biosynthesis of HOCs by marine microorganisms, with the ultimate goal of enabling genetic source tracking through DNA sequence-based biosynthetic signatures, across all trophic levels of the marine food web. In addition to the biosynthesis of these compounds, we are also investigating microbial activities that degrade or transform these pollutants within marine animals, e.g. the fish gut microbiome. Omega-3 Polyunsaturated Fatty Acids Long-chain omega-3 polyunsaturated fatty acids (PUFAs) such as eicosapentaenoic acid (EPA; 20:5n-3) and docosahexaenoic acid (DHA; 22:6n-3) are well known for their beneficial roles in human health and development, and their ameliorative activities in diverse disease conditions, including cardiovascular, inflammatory, autoimmune, and neurodegenerative disorders. Select lineages of cosmopolitan marine prokaryotic and eukaryotic microorganisms synthesize these compounds via a unique fatty acid synthase/polyketide synthase mechanism that is distinct from the canonical desaturase/elongase-mediated pathway employed by the majority of eukaryotic single-cell microorganisms and metazoans. We are investigating the genetics and regulation of bacterial omega-3 PUFA synthesis in several lineages of marine bacteria via bacterial genetic methods to uncover genetic determinants involved in PUFA synthesis and explore the physiological role of these unique marine lipids. In this process we have engineered native and recombinant bacterial strains that synthesize high quantities of these molecules, providing a novel route for the biotechnological and/or nutraceutical exploitation of these marine bacterial resources. Bioinformatics We develop and apply bioinformatics tools and pipelines for the annotation and analysis of microbial genomes and metagenomes, collaborating with colleagues in many diverse areas of experimental biology and microbial ecology, as well as marine natural product chemistry and marine geochemistry. Our special interests include metagenomic sequence assembly of environmental and host-associated DNA samples, determining the role of specific environmental habitats in cross-species transmission of microbial genomes via horizontal gene transfer, and developing in silico tools for the discovery and novelty assignment of secondary metabolite biosynthesis genes within microbial genomes and metagenomes. \n",
            "-------------------- Scraping faculty url 4/183 --------------------\n",
            "nicola allen \n",
            "-------------------- Scraping faculty url 5/183 --------------------\n",
            "jon armour \n",
            "-------------------- Scraping faculty url 6/183 --------------------\n",
            "kenta asahina \n",
            "-------------------- Scraping faculty url 7/183 --------------------\n",
            "janelle ayres \n",
            "-------------------- Scraping faculty url 8/183 --------------------\n",
            "eiman azim \n",
            "-------------------- Scraping faculty url 9/183 --------------------\n",
            "tim baker  Biography Tim Baker received his Ph.D. from UCLA and was a Jane Coffin Childs postdoctoral fellow at Cambridge University, a NIH Postdoctoral Fellow and the Charles A. King Trust Postdoctoral Fellow at Brandeis University. From 1983-2004, he was on the Faculty of Purdue University where he earned the Herbert Newby McCoy Award for Scientific Achievement and a NIH NIGMS MERIT award. He received a joint faculty appointment in Biological Sciences and Chemistry & Biochemistry at UCSD in 2004.  Research Viruses are one of natures smallest (nano), most efficient, machines, whose primary function is to infect and replicate in host cells. The predominant thrust of our research over the past 25 years has been to image and determine the three-dimensional (3D) structures of viruses to gain insights about the mechanisms they use to interact with their hosts so they can replicate and assemble a new set of mature, infectious particles. To accomplish this, we use cryo-electron microscopy (CryoEM) and 3D image-reconstruction methods to study viruses and various kinds of virus-complexes (e.g. virus-antibody or virus-receptor) and virus-like particles at sub-nanometer (< 10-) resolutions. At this level of resolution, we can discern and distinguish viral components (protein, nucleic acid, lipid, carbohydrate) and secondary structural details such as a-helices and b-sheets in proteins. CryoEM is a powerful tool because it permits the structures of biological samples to be preserved in a near native state. Samples are flash-frozen (vitrified) in liquid ethane in a sub-millisecond time frame, which prevents ice crystal formation that would damage the specimen. This technique uses no chemical fixatives or stains and thereby helps preserve the hydrated structure of the virus (or any macromolecular complex). Specimens are then loaded into one of two, computer-controlled, transmission electron microscopes housed in Bonner Hall, both of which are able to maintain specimens at liquid nitrogen or lower temperature. These microscopes are used to collect upwards of tens of thousands or more virus images that are needed to obtain high-resolution 3D reconstructions. Such large numbers are required because each individual virus image is extremely noisy owing to radiation-induced damage to the sample caused by the high voltage electron beam and because the contrast in unstained samples is extremely low. In house, computer reconstruction techniques are continuously being developed and tested to more reliably and efficiently extract usable information from the noisy image data. We have developed a processing pipeline, called Auto3DEM, which automates nearly all of the tedious steps in the 3D reconstruction process. In favorable cases, we are able to obtain reconstructions at sub-nanometer resolution in a day or less after the images are recorded. In addition, we are developing tools designed to provide real time, 3D reconstruction feedback (i.e. in a few minutes or less) to the microscopist to further improve success of data collection. Viral pathogens from many different families are currently under investigation. These infect a wide range of hosts and include: human parvoviruses (bocavirus and several serotypes of adeno-associated viruses both as native virions and complexed with neutralizing antibodies and receptor molecules); human papilloma viruses that cause cancer (HPV-16) or genital warts (HPV -6 and HPV-11) or are listed as high risk (HPV-52); four distinct totiviruses (the invertebrate IMNV, the fungal HvV190S, and the protozoan TVV1 and GLV); the alphavirus group of small, enveloped, ssRNA viruses (e.g. Sindbis virus) that are human and insect pathogens, some of which are listed as possible agents of bio-terrorism; two large, dsDNA viruses (the insect CIV and frog FV3); three fungal partitiviruses (PsV-S, PsV-F, FpV-01); four bacteriophages (P22, Sf6, CUS-3, and 29); and an insect tetravirus (NV). \n",
            "-------------------- Scraping faculty url 10/183 --------------------\n",
            "matthew banghart  Biography Matt Banghart obtained his Ph.D. in Chemistry from UC Berkeley and pursued postdoctoral training in the Department of Neurobiology at Harvard Medical School with Bernardo Sabatini. Matt was a Helen Hay Whitney Postdoctoral Fellow and the recipient of a K99/R00 Pathway to Independence Award from NIDA. He is currently establishing his independent research laboratory at UCSD.  Research Research in the Banghart lab involves two intertwined themes: neuromodulatory mechanisms and molecular tool development. Several questions of interest include: How do neuromodulators transform neural circuit function? What principles determine the spatial and temporal domains of neuromodulatory signaling? When and where are neuromodulators released in the brain? How are neuromodulatory pathways compromised in disease and how can they be restored? We address these questions in the mammalian basal ganglia, a collection of neural circuits that rely heavily on neuromodulators to mediate decision-making, goal-oriented movements and behavioral reinforcement. Our experimental approach utilizes contemporary genetically-targeted neuroanatomical methods, whole-cell electrophysiology, two-photon imaging and rodent behavioral analysis, often in conjunction with optogenetic and chemical-genetic perturbations. To facilitate these efforts, we develop novel molecular tools for observing and manipulating neuromodulatory activity with an emphasis on photoactivatable (or caged) neurotransmitters, optical sensors and genetically-targeted pharmacological probes. Fluorescently labeled neurons demarcate neuromodulatory hot-spots known as patches (or striosomes) in the dorsal striatum. \n",
            "-------------------- Scraping faculty url 11/183 --------------------\n",
            "andrew barton  Biography Andrew Barton received his Ph.D in Climate Physics and Chemistry from Massachusetts Institute of Technology in 2011, and was an NSF International Research Postdoctoral Fellow hosted jointly between Duke University and the University of Liverpool in the United Kingdom. Dr. Barton joined the faculty of Biological Sciences and the Scripps Institution of Oceanography in 2016.  Research Marine phytoplankton are microscopic photosynthetic organisms living in the ocean surface. They form the base of the marine food chain, and can thus impact larger marine organisms, from zooplankton to commercially important fish and even whales. Phytoplankton also play instrumental roles in the global cycles of carbon, oxygen, and other elements. Yet despite their tremendous ecological, societal, and global importance, we are only beginning to explore the great biodiversity of phytoplankton, and understand their complex ecology and interactions with the environment. Our research seeks to map the distribution of phytoplankton species in the ocean, and illuminate the fundamental biological and ecological processes that underpin these patterns. We investigate how changes in Earths climate, including natural variability and long-term changes driven by human activities, have the potential to alter phytoplankton species distributions and community composition. To illustrate and study these complex natural processes, we develop cutting-edge computer models that simulate the marine environment and the many types of plankton living in the ocean. We integrate these models with real observations from the sea, using large compilations of environmental, ecological, and biological data to glean novel inferences about marine plankton life. Our research is building understanding of a critical, if individually tiny, component of life on Earth. \n",
            "-------------------- Scraping faculty url 12/183 --------------------\n",
            "eric bennett  Biography Dr. Bennett received his PhD from Stanford University. He then carried out postdoctoral work at the Harvard Medical School.  Research My lab is interested in understanding the fundamental principles of how proteins are destroyed within cells. The cellular environment must constantly adapt to changing conditions and one mechanistic strategy commonly employed is the targeted modification of proteins by ubiquitin chains and subsequent degradation by the proteasome. The ubiquitin-proteasome system (UPS) can be viewed as the cellular waste management system. The UPS is utilized as the cellular quality control system as it is tasked with the degradation of potentially toxic misfolded and misassembled proteins. The inability of the UPS to properly destroy these proteins has been linked to various human pathologies including amyotrophic lateral sclerosis (ALS, commonly referred to as Lou Gehrig's disease), Parkinson's disease, and many human cancers. Thus, understanding the mechanisms of how the UPS recognizes proteins to be destroyed and how the pathway becomes dysfunctional will be instrumental in understanding the etiology of these diseases. The lab uses integrated approaches to understand UPS activity coupling biochemistry, cell biology, and systems-level approaches. Deubiquitylating enzyme regulation The ubiquitylation of substrates occurs via a hierarchy of enzyme activity where ubiquitin activating enzymes (E1s) activate free ubiquitin and transfers the activated ubiquitin to ubiquitin conjugating enzymes, (E2s), who partner with ubiquitin ligases (E3s) to finally conjugate ubiquitin to substrates. Deubiquitylating enzymes (Dubs) remove ubiquitin chains from substrates thereby antagonizing ubiquitin ligase activity. While the basic enzymatic function of Dubs is understood, how their activity is regulated, what cellular pathways are regulated by Dub activity, and what cellular substrates are the targets for Dub regulationare not fully understood. We previously characterized 75 of the 100 human Dubs based on their cellular interaction partners using proteomic technologies. During this study we established an integrative proteomic platform, termed CompPASS, to rapidly analyze interaction proteomics data. We found that Dubs not only frequently associate with ubiquitin ligases but are themselves often modified by ubiquitin. This suggests that Dub activity may be regulated by E3 ligases and vice versa. The lab is interested in understanding the mechanisms of Dub activity regulation via association with ubiquitin ligases using both biochemical and cellular techniques. Cullin-RING ligase regulation One family of ubiquitin ligases are the multi-subunit ligases consisting of Cullin-RING ligases (CRLs) and the anaphase promoting complex (APC). Cullin-RING ligases are modular assemblies that can target many cellular proteins for regulated degradation by dynamically remodeling its subunit architecture. CRLs are known to degrade many critical oncoproteins and tumor suppressors, and as such there has been recent interest in the development of therapeutics targeting this pathway as an anti-cancer strategy. CRLs are also regulated through a switch like mechanism in which the ubiquitin like protein Nedd8 modifies and subsequently activates the cullin subunit. Thus, by simply adding and removing the Nedd8 modification, CRL activity can be switched on and off. We are utilizing quantitative proteomics to study the cellular mechanisms used to control CRL activity. Additionally, this allows us to probe the dynamic nature of CRL assemblies and to understand how their architecture is remodeled to adapt to changes in the cellular environment. Global quantitative proteomic analysis of UPS function The cellular proteome is continually monitored by the protein folding and degradation machinery to prevent the accumulation of potentially toxic misfolded or misassembled proteins. Proteins that contain errors in their primary sequence that arise during the translation process need to be recognized and selectively degraded by the UPS. When this quality control function breaks down it can have dire consequences on cellular and organismal fitness. Thus, the ability to globally monitor the activity of the UPS would allow for the identification of conditions that lead to UPS dysfunction. We have developed technologies that allow for the interrogation of the UPS activity on a systematic level. From these studies it is clear that mistranslated proteins are constantly ubiquitylated and cleared from cells. We are interested in trying to answer several questions related to quality control mechanisms. For instance, what are the ubiquitin ligases responsible for ubiquitylation mistranslated substrates? How do changes in free amino acid levels within the cell effect protein degradation and UPS function? Can we monitor alterations in UPS function in human pathologies associated with a decrease in quality control function? How does changing the redox potential within cells effect the quality control machinery? By understanding how proteins are recognized as quality control substrates and how the cell responds to challenges in protein homeostasis we hope to develop therapeutics that modulate quality control function. \n",
            "-------------------- Scraping faculty url 13/183 --------------------\n",
            "darwin berg \n",
            "-------------------- Scraping faculty url 14/183 --------------------\n",
            "ethan bier  Biography Ethan Bier received his Ph.D. from Harvard University. He has received the Sloan Foundation Award, the McKnight Award for Neuroscience, the Basil O'Connor Award, and the American Cancer Society Junior Faculty Award.  Research There are two primary areas of research in my lab: 1) developmental patterning, and 2) using Drosophila as model to understand human disease processes. Developmental Studies With respect to developmental patterning, we have had a long standing interest in understanding how graded BMP signaling establishes cell fates along the dorsal-ventral (DV) axis of Drosophila embryos and how wing vein form along borders between discrete domains in the wing imaginal disc. Our studies on BMPs in DV patterning began with our cloning of the short gastrulation ( sog ) gene [1], which encodes a secreted BMP antagonist homologous to vertebrate Chordin [2]. In collaboration with David Kimelmans group (Univ. Washington), we showed that sog activity was conserved in Xenopus embryos where, like chordin , it could induce a secondary neural axis. Our studies also revealed that Sog acts to prevent BMP signaling from spreading ventrally into the Drosophila neuroectoderm via an autoactivation loop, a process that is also conserved in vertebrate embryos [3], thus helping to reverse the prior paradigm for neural induction as a positive-acting process as originally formulated by Hans Spemann in the early 20th century. We also have studied formation of the Sog protein gradient in the dorsal epidermal region of the embryo which we visualized [4] and analyzed quantitatively in collaboration with Arthur Landers group (UCI) [5]. This BMP activity gradient arises as a consequence of an adjacent source/sink configuration wherein ventrally produced Sog diffuses dorsally, where it is degraded by the metalloprotease Tolloid and removed by endocytosis [4]. Most recently, in collaboration with Shannon Fishers group (U Penn) and Giorgos Pyrowolakis (Univ. Freiburg) we have been examining the role that BMPs play in patterning the dorsal and lateral regions of the CNS in both flies and zebrafish. In flies, BMPs function as they do during neural induction by repressing expression of the neural identity genes msh and ind (whichdefine dorsal and lateral CNS cell fates, respectively). A twist , however, is that BMPs act in a dose-dependent fashion to repress ind more efficiently than msh , thereby providing a patterning cue [6]. These experiments should shed light on the evolutionary origins of the condensed CNS, which we believe arose once in the ancestors of bilateria [7, 8]. Our studies on wing vein patterning are focused on how primary positional information provided by BMP and Hedgehog morphogen gradients is refined into sharp borders abutting broad gene expression domains to define the positions along which wing vein primordia will develop [9, 10]. Wing veins are rigid hollow fluid conducting channels that function as structural supports for flight and as conduits bringing nutrients to sensory organs in the wing. Although formation of each vein is initiated separately, these linear structures are all induced by the same general mechanism (termed for-export-only signaling) in which cells in one domain produce a diffusible factor to which they are unable to respond [10]. The vein inducing signal diffuses into the adjacent cell territory where cells nearest the source of the signal can respond by activating expression of vein specific identity genes such as knirps (for the L2 vein) [11, 12] or abrupt (for the L5 vein) [13]. Most recently we have become interested in the evolution of the gene regulatory networks underlying wing development in dipteral species with different vein patterns than in D. melanogaster . Analyzing Mechanisms of Human Disease in Flies With regard to using Drosophila as tool to study human disease [14], we have used this system to identify the Rho-GEF pebble as a candidate gene involved in Angelman syndrome [15], to examine an interaction between g-secretase and peroxiredoxins that may be relevant to Alzheimers disease [16], to identify a cooperative interaction between the cell adhesion molecule DSCAM and a collagen subunit (COL6A2) that when over-expressed leads to congenital heart defects in flies and mice and may contribute to heart defects in Down syndrome patients (in collaboration with Geoff Rosenfeld, UCSD, Medicine) [17], and, in collaboration with Michael Karin (UCSD, Pharmacology) to examine mechanisms of metabolic homeostasis relevant to aging [18]. Our current focus is on identifying genetic pathways that are disrupted by pathogenic bacterial and viral factors. For example, in collaboration with Michael Karin and Victor Nizet (UCSD, Pediatrics) we have examined the cell biological effects of anthrax toxins [19-21]. These studies identified the small GTPase Rab11 and its exocyst binding partner Sec15 as targets of the barrier disruptive anthrax toxins Edema Factor (EF) and Lethal Factor (LF) [19]. EF functions by down-regulating expression and activity of Rab11 on late recycling endosomal membranes, and thereby indirectly leads to down-regulation of Sec15, which tethers the endosome to docking sites in the plasma membrane. LF does not affect Rab11, but does inhibit Sec15 expression, and functions cooperatively with EF to disrupt exocyst mediated trafficking and endothelial barrier integrity. We are also conducting a similar analysis on the effect of cholera toxin (Ctx, an ADP ribosylase that activates Gsa proteins thereby constitutively activating endogenous adenylate cyclase to produce elevated levels of cAMP) in intestinal epithelial cells [22]. These studies have revealed that in addition to its classic well characterized role in inducing transcellular Cl- ion secretion via the CFTR channel, Ctx also weakens cell junctions, which may facilitate the accompanying efflux of Na+ counter ions and water via the paracellular route, thus resulting in the profuse salty diarrhea characteristic of cholera. Importantly, the junction disrupting effects of both EF and Ctx can be reversed by over-expressing Rab11, suggesting a new avenue for therapeutic intervention in these diseases. In collaboration with David Broide (UCSD, Medicine) we have also begun to examine whether endocytic recycling plays a role maintaining barrier integrity in airway epithelial cells, and whether defects in this trafficking contribute to asthma pathogenesis. Insights derived from these synergistic studies will have relevance to a broad range of diseases that increase epithelial or endothelial permeability and may lead to therapeutic strategies that selectively counteract such effects by restoring endocytic trafficking and improving barrier function. Quantitative Imaging and Genome Analysis In collaboration with the McGinnis group (UCSD, CDB) and Terry Hwa's lab (UCSD, Physics) we have also developed new methods and quantitative tools to detect multiple gene expression patterns in single cells using multiplex in situ hybridization [23] as well as computational techniques to quantitatively measure gene expression levels [24]. We have applied these to generate testable models of pattern formation in both the developing embryo and wing. For example, most recently, we have proposed that differences in gene length alone can create a transient nested pattern of gene expression to help seed morphogen-dependent patterning across a developing field of cells [24]. We have also developed bioinformatic tools for mining and comparing genome data [25, 26]. \n",
            "-------------------- Scraping faculty url 15/183 --------------------\n",
            "brenda bloodgood \n",
            "-------------------- Scraping faculty url 16/183 --------------------\n",
            "steve briggs  Biography Briggs earned his Ph.D. from Michigan State University and spent the first half of his career in industry as Director at Pioneer/DuPont; President of Torrey Mesa Research Institute; and Senior Vice President for Corporate Research at Diversa. He served as Senior Staff Investigator at Cold Spring Harbor Laboratory. Briggs joined the faculty at UCSD as Professor where he served as Chair of the Section of Cell & Developmental Biology, Associate Dean of the Division of Biology, and Interim Chair of the Department of Chemistry and Biochemistry. He is a member of the National Academy of Sciences, a Fellow of the American Association for the Advancement of Science, and a Fellow of the National Academy of Inventors.  Research We study molecular signaling networks with a focus on the innate immune system of plants. To support our studies, we developed a proteomics capability that is sensitive enough to discover new genes and gene models (1); discover genes based solely on comparison between the proteomes of mutant and wild type plants (2), and discover mechanisms of signaling mediated by protein phosphorylation (3). We used mass spectrometry to generate a proteomics atlas, elements of which have been published (4, 5, 6). When combined with RNA-seq data, our integrated gene expression atlas was used to derive co-expression networks and gene regulatory networks from the proteome and transcriptome alone and from integration of the data. We found that the transcriptome does not predict the proteome either quantitatively or qualitatively (4) and networks derived from protein levels have little overlap with networks of the same genes measured according to their mRNA levels (7). Recent reports on host-microbe interactions include studies of host peptide signaling (8), pathogen effector secretion (9, 10), self-acetylation of host proteins caused by a pathogen effector (11), and biosynthesis of defense metabolites in response to infection (12). We collaborate with other labs to understand problems in signaling that are less central to our interests but that can be uniquely addressed using our expertise. These include understanding endoplasmic reticulum stress (13, 14) and circadian signaling in plants (15); chemotactic signaling in Dictyostelium (16, 17); and the molecular determinants of chronic lymphocytic leukemia (18, 19). Castellana NE, Shen Z, He Y, Walley JW, Cassidy CJ, Briggs SP, Bafna V. An Automated Proteogenomic Method Uses Mass Spectrometry to Reveal Novel Genes in Zea mays. Mol Cell Proteomics. 2014 Jan;13(1):157-67. doi: 10.1074/mcp.M113.031260. Zhang X, Facette M, Humphries JA, Shen Z, Park Y, Sutimantanapi D, Sylvester AW, Briggs SP, Smith LG. Identification of PAN2 by quantitative proteomics as a leucine-rich repeat-receptor-like kinase acting upstream of PAN1 to polarize cell division in maize. Plant Cell. 2012 Nov;24(11):4577-89. doi: 10.1105/tpc.112.104125. Epub 2012 Nov 21. PubMed PMID: 23175742; PubMed Central PMCID: PMC3531853. Qiao H, Shen Z, Huang SS, Schmitz RJ, Urich MA, Briggs SP, Ecker JR. Processing and subcellular trafficking of ER-tethered EIN2 control response to ethylene gas. Science. 2012 Oct 19;338(6105):390-3. doi: 10.1126/science.1225974. Epub 2012 Aug 30. PubMed PMID: 22936567; PubMed Central PMCID: PMC3523706. Walley JW, Shen Z, Sartor R, Wu KJ, Osborn J, Smith LG, Briggs SP. Reconstruction of protein networks from an atlas of maize seed proteotypes. Proc Natl Acad Sci U S A. 2013 Dec 3;110(49):E4808-17. doi: 10.1073/pnas.1319113110. Epub 2013 Nov 18. PubMed PMID: 24248366; PubMed Central PMCID: PMC3856832. Facette MR, Shen Z, Bjrnsdttir FR, Briggs SP, Smith LG. Parallel proteomic and phosphoproteomic analyses of successive stages of maize leaf development. Plant Cell. 2013 Aug;25(8):2798-812. doi: 10.1105/tpc.113.112227. Epub 2013 Aug 9. PubMed PMID: 23933881; PubMed Central PMCID: PMC3784581. Marcon C, Malik WA, Walley JW, Shen Z, Paschold A, Smith LG, Piepho HP, Briggs SP, Hochholdinger F. A high resolution tissue-specific proteome and phosphoproteome atlas of maize primary roots reveals functional gradients along the root axis. Plant Physiol. 2015 Mar 16. pii: pp.00138.2015 Walley JW, Sartor RC, Shen Z, Schmitz RJ, Wu KJ, Urich MA, Nery JR, Smith LG, Schnable JC, Ecker JR, Briggs SP. Integration of omic networks in a developmental atlas of maize. 2016 Science. Aug 19;353(6301):814-8. doi: 10.1126/science.aag1125. PMID: 27540173 Huffaker A, Pearce G, Veyrat N, Erb M, Turlings TC, Sartor R, Shen Z, Briggs SP, Vaughan MM, Alborn HT, Teal PE, Schmelz EA. Plant elicitor peptides are conserved signals regulating direct and indirect antiherbivore defense. Proc Natl Acad Sci U S A. Apr 2;110(14):5707-12. doi: 10.1073/pnas.1214668110. 2013 Chaudhary R, Atamian HS, Shen Z, Briggs SP, Kaloshian I. Potato aphid salivary proteome: enhanced salivation using resorcinol and identification of aphid phosphoproteins. J Proteome Res. 2015 Apr 3;14(4):1762-78. doi: 10.1021/pr501128k. Chaudhary R, Atamian HS, Shen Z, Briggs SP, Kaloshian I. GroEL from the endosymbiont Buchnera aphidicola betrays the aphid by triggering plant defense. Proc Natl Acad Sci U S A. 2014 Jun 17;111(24):8919-24. doi: 10.1073/pnas.1407687111. Walley JW, Shen Z, McReynolds MR, Schmelz EA, Briggs SP. Fungal-induced protein hyperacetylation in maize identified by acetylome profiling. 2018 Proc Natl Acad Sci U S A. Jan 2;115(1):210-215. doi:10.1073/pnas.1717519115. Epub 2017 Dec 19. PMID: 29259121 Mafu S, Ding Y, Murphy KM, Yaacoobi O, Addison JB, Wang Q, Shen Z, Briggs SP, Bohlmann J, Castro-Falcon G, Hughes CC, Betsiashvili M, Huffaker A, Schmelz EA, Zerbe P. 2018 Discovery, Biosynthesis and Stress-Related Accumulation of Dolabradiene-Derived Defenses in Maize. Plant Physiol. Apr;176(4):2677-2690. doi: 10.1104/pp.17.01351. Epub 2018 Feb 23. PMID: 29475898 Wang JZ, Li B, Xiao Y, Ni Y, Ke H, Yang P, de Souza A, Bjornson M, He X, Shen Z, Balcke GU, Briggs SP, Tissier A, Kliebenstein DJ, Dehesh K. 2017 Initiation of ER Body Formation and Indole Glucosinolate Metabolism by the Plastidial Retrograde Signaling Metabolite, MEcPP. Mol Plant. Nov 6;10(11):1400-1416. doi:10.1016/j.molp.2017.09.012. Epub 2017 Sep 28. PMID: 28965830 Walley J, Xiao Y, Wang JZ, Baidoo EE, Keasling JD, Shen Z, Briggs SP, Dehesh K. Plastid-produced interorgannellar stress signal MEcPP potentiates induction of the unfolded protein response in endoplasmic reticulum. Proc Natl Acad Sci U S A. 2015 May 12;112(19):6212-7. doi: 10.1073/pnas.1504828112. Epub 2015 Apr 28. PubMed PMID: 25922532; PubMed Central PMCID: PMC4434691. Huang H, Alvarez S, Bindbeutel RK, Shen Z, Naldrett MJ, Evans BS, Briggs SP, Hicks LM, Kay SA, Nusinow DA. Identification of evening complex associated proteins in Arabidopsis by affinity purification and mass spectrometry. Mol Cell Proteomics. 2015 Nov 6. pii: mcp.M115.054064 Lacal Romero J, Shen Z, Baumgardner K, Wei J, Briggs SP, Firtel RA. The Dictyostelium GSK3 kinase GlkA coordinates signal relay and chemotaxis in response to growth conditions. 2018 Dev Biol. Mar 1;435(1):56-72.doi: 10.1016/j.ydbio.2018.01.007. Epub 2018 Jan 20. PMID: 29355521 Khanna A, Lotfi P, Chavan AJ, Montao NM, Bolourani P, Weeks G, Shen Z, Briggs SP, Pots H, Van Haastert PJ, Kortholt A, Charest PG. 2016 The small GTPases Ras and Rap1 bind to and control TORC2 activity. Sci Rep. May 13;6:25823. doi: 10.1038/srep25823. PMID: 27172998 Hasan MK, Yu J, Widhopf GF 2nd, Rassenti LZ, Chen L, Shen Z, Briggs SP, Neuberg DS, Kipps TJ. Wnt5a induces ROR1 to recruit DOCK2 to activate Rac1/2 in chronic lymphocytic leukemia. 2018 Blood. Jul 12;132(2):170-178. doi: 10.1182/blood-2017-12-819383. Epub 2018 Apr 20. PMID: 29678828May 3. PMID: 284655282016 Yu J, Chen L, Cui B, Widhopf GF 2nd, Shen Z, Wu R, Zhang L, Zhang S, Briggs SP, Kipps TJ. Wnt5a induces ROR1/ROR2 hetero-oligomerization to enhance leukemia chemotaxis and proliferation. J Clin Invest. 2015 Dec 21. pii: 83535. doi: 10.1172/JCI83535. \n",
            "-------------------- Scraping faculty url 17/183 --------------------\n",
            "stuart brody  Biography Stuart Brody received his Ph.D. from Stanford University and was a postdoctoral fellow and assistant professor at Rockefeller University. Professor Brody was a recipient of an NIH Career Development Award from 1974 to 1979.  Research Studies on the biochemical genetics of Neurospora focuses on its circadian rhythms This rhythm is visibly expressed in Neurospora as a spore-forming rhythm in a petri dish (see photo below). Areas of the culture showing the developmental phenomena of spore formation can be seen as \"bands\" on the surface of the agar. Mutations affecting the rhythm lead to different numbers of bands on a plate as seen in the cla-1 strain below. We are investigating the effects of several classes of mutations on the clock: analyzing the effects of mutants such as vivid (vvd) that affect the light response of the clock and lead to very short ( as low as 6 hrs.) periods in constant light; describing and analyzing the components of a second oscillator, the frq-less oscillator (FLO) in Neurospora , employing mutants such as csp-1 and rco-1 which affect downstream processes; using fluorescent tags to describe the synthesis and localization of key clock proteins , such as FRQ in vivo; understanding how the clock-controlled genes ( ccg) are regulated by the core oscillator. \n",
            "-------------------- Scraping faculty url 18/183 --------------------\n",
            "wolfgang busch \n",
            "-------------------- Scraping faculty url 19/183 --------------------\n",
            "madeline butler \n",
            "-------------------- Scraping faculty url 20/183 --------------------\n",
            "hu cang \n",
            "-------------------- Scraping faculty url 21/183 --------------------\n",
            "sreekanth chalasani \n",
            "-------------------- Scraping faculty url 22/183 --------------------\n",
            "lin chao  Biography Lin Chao received his Ph.D. from the University of Massachusetts, Amherst. He was an NIH postdoctoral fellow at Princeton Universit/y./ Lin joined UCSD in 1999 and he served as the Chair of the Section of Ecology, Behavior and Evolution from 2004 to 2008. In 2014, he received the UCSD Chancellor's Associates Award for Excellence in undergraduate teaching.  Research We use a combination of computer modeling and laboratory experimentation to study ecological and evolutionary processes. Ecological and evolutionary studies are often difficult to investigate because they occur slowly over long periods of time. Computer models can speed up time, but we also take advantage of the fact that microbes, bacteria and viruses provide that same advantage because of their short generation times. Furthermore, they can be easily cultured in the laboratory under controlled conditions. However, we also realize that too much control can be dangerous because an experiment that is too controlled becomes artificial and not much more than a simulation. And a simulation is always more easily obtained from a computer model. Thus, we are careful to design experiments that teach us more than a computer model. We make sure that the experiments allow us to measure an unknown that could not be determined a priori from a computer model. We have used microbial systems to study topics ranging from the ecology of host-parasite co-existence and the evolution of sex, mutation rates, transposable elements, group and individual adaptations, game theory strategies, Fishers Geometric Model of Adaptive Evolution, deleterious and compensatory mutations. As new information or interests develop, we often revisit older topics or explore new ones. Time lapse photograph E. coli cells descending from a single cell. By recording division of all cells via time lapse, the doubling time or life history fitness of every individual in a population can be determined. Our current research focuses on evolution of microbial aging or senescence, which grew out of our interest in genetic damage or mutations. Because most mutations are deleterious, they lead to fitness loss unless removed by natural selection. The transmission of mutations from mother to daughter is governed by well the well known rules of genetic inheritance and assortment. However, non-genetic or phenotypic damage can also decrease fitness and, unlike mutations, the transmission rules for phenotypic damage are not well understood. If a mother cell has ten units of phenotypic damage, for example ten proteins damaged by oxidation, should she distribute the damage evenly between her two daughters or give all ten units to one daughter and none to the other? Our theoretical models have shown that a lineage maximizes its fitness by allocating all the damage to a single daughter. Thus, asymmetry should evolve and the daughter receiving all or more damage should senesce over time. Time lapse photography of dividing cells has already shown that bacteria divide asymmetrically. We are now making our own time lapse series to test more directly the prediction of our models. \n",
            "-------------------- Scraping faculty url 23/183 --------------------\n",
            "lakshmi chilukuri \n",
            "-------------------- Scraping faculty url 24/183 --------------------\n",
            "andrew chisholm \n",
            "-------------------- Scraping faculty url 25/183 --------------------\n",
            "senyon choe \n",
            "-------------------- Scraping faculty url 26/183 --------------------\n",
            "joanne chory  Biography Joanne Chory received an A.B. degree in biology with honors from Oberlin College, OH, a Ph.D. in microbiology from the University of Illinois at Urbana-Champaign, and conducted postdoctoral research at Harvard Medical School. In 1988, she joined the faculty of the Salk Institute, where she has remained. Dr. Chory has served on numerous advisory committees and editorial boards, and has received multiple awards. In 2003, Dr. Chory was named Scientific American's Research Leader in Agriculture. She is a member of the U.S. National Academy of Sciences, the German National Academy of Sciences (Leopoldina), the American Academy of Arts and Sciences, is a fellow of the American Association for the Advancement of Science. Joanne Chory is a foreign affiliate of the French Academy of Science, a foreign member of the Royal Society of London, and is an associate member of EMBO.  Research Our lab wants to understand the mechanisms by which plants respond to changes in their light environment. We use the reference plant, Arabidopsis thaliana , to identify components of the phototransduction pathways that link changes in the light environment with differential growth and global alteration of the transcriptome. Our genetic analyses indicate that light responses are not simply endpoints of linear signal transduction pathways, but are the result of the integration of information from a variety of photoreceptors acting through a complex network of interacting signaling components. Our studies have also revealed that plant steroid hormones and auxin are involved in light-regulated development of plants. Current studies are focused on defining the mechanistic links between the photoreceptor signaling pathways and endogenous developmental programs that involve the action of plant hormones. Our studies are relevant to plant adaptation to global climate change. \n",
            "-------------------- Scraping faculty url 27/183 --------------------\n",
            "maarten chrispeels  Biography Maarten Chrispeels received his Ph.D. from the University of Illinois and was a postdoctoral fellow at the DOE Plant Research Laboratory at Michigan State University. Professor Chrispeels was awarded a Guggenheim Fellowship and is a member of the National Academy of Sciences. He is the editor in chief of Plant Physiology.  Research Maarten Chrispeels came to the US in 1960 for graduate study at University of Illinois. After receiving his PhD in Agronomy, he became a postdoctoral fellow at the DOE Plant Research Laboratory at Michigan State University and studied plant biochemistry. His active research career at UCSD spanned 42 years. Professor Chrispeels was elected to membership in the US National Academy of Sciences in 1996. For 10 years he served as the Director of the San Diego Center for Molecular Agriculture (SDCMA) on the UCSD campus. Research Research in the Chrispeels laboratory concerned the biosynthesis and transport of proteins and glycoproteins in developing legume seeds and focused on storage proteins and lectins. They elucidated the role of the Golgi apparatus in the modification of asparagine-linked glycans and the transport of glycoproteins to protein storage vacuoles. They used transgenic technologies to identify the domains of proteins that allow them to be targeted to vacuoles. This work led to the study of a unique protein in the common bean ( Phaseolus vulgaris ) called amylase inhibitor. They showed that this plant defense protein is evolutionarily related to the lectin phytohemagglutinin and used the gene that encodes amylase inhibitor to produce the first insect resistant seeds using genetic engineering. This project was carried out in collaboration with scientists at Purdue University and at the CSIRO in Canberra, Australia. A year after aquaporins were described in animal cells (by Nobel Laureate Peter Agres lab) the Chrispeels lab found homologous proteins in the tonoplasts of plants and aquaporins became the main focus of the lab until Chrispeels retirement from research in 2007. The discovery of aquaporins in plants can be said to have revolutionized the study of plant-water relations. Together with his collaborators Chrispeels published some 180 research papers and 50 reviews and book chapters. Teaching For many years Professor Chrispeels taught courses in plant biology at different levels (introductory courses for non-science majors or for science majors, or advanced courses in plant development and plant biotechnology. To aid in teaching he and his former student David Sadava co-wrote or edited several books, including Plants, Food and People (1977), Plants, Genes and Agriculture (1994) and Plants, Genes and Crop Biotechnology (2003). After his retirement from research he continued to teach at the university and became involved in outreach to San Diego area high schools. Public service and honors For more than 25 years Chrispeels played an active role in the journal Plant Physiology, first as a member of the editorial board, then as an associate member, and from 1990 to 1997 as its Editor in Chief. For 15 years he was involved with the Millennium Science Initiative in Chile and became the convener of the Program Committee that evaluates research proposals from Chilean Scientists. As director of the SDCMA he organized ten major symposia and other events for San Diego area plant biologists. He also produced a colorful 16-page brochure entitled Foods from Genetically Engineered Crop s that was widely distributed throughout the world. His work was recognized by his peers through his election to the US National Academy of Sciences (1996), and receipt of the Stephen Hales Prize of the American Society of Plant Biologists (ASPB) in 1996. In 2000 he received a Doctor Honoris Causa from the University of Guelph (Canada) and was elected to the rank of Fellow of the ASPB in 2006. In 2007 he became a Fellow of the American Association for the Advancement of Science, and in 2008 he was elected to Foreign Corresponding membership in the Academia Chilena de Ciencias. Industrial activities Professor Chrispeels holds several patents, has consulted for a number of biotechnology companies and is the co-founder of two companies: Phylogics (with Dr. Jeffrey Moore) and Arterra Biosciences in Naples, Italy (with Dr. Gabriella Colucci). \n",
            "-------------------- Scraping faculty url 28/183 --------------------\n",
            "elsa cleland \n",
            "-------------------- Scraping faculty url 29/183 --------------------\n",
            "aaron coleman  Biography Aaron Coleman obtained his Ph.D. in Biomedical Sciences from the University of California, Riverside. He conducted post-doctoral research at the Beckman Research Institute at City of Hope National Medical Center. He joined the Division of Biological Sciences in 2007.  Research Biology Education Research All fields of biology are based on methodical experimentation and careful interpretation of data. Laboratory classes where students perform well-designed experiments using relevant techniques are an essential part of an undergraduate education in the biological sciences. An effective laboratory curriculum needs to immerse students in the experimental process. This means not only teaching the concepts underlying the laboratory work but also fostering the development of scientific reasoning. Our research investigates approaches to improve laboratory education on two fronts. First, we are trying to understand how students learn the scientific reasoning skills that are necessary for them to understand research and become scientifically-literate citizens. Second, we seek to design biochemistry laboratory modules that allow students to have a research-like experience in terms of the experimental systems and techniques used, but that are feasible to implement in high-enrollment courses. Teaching Scientific Reasoning The Vision and Change in Biology Undergraduate Education report (2011) from AAAS lists six core competencies that should be expected from graduates with a degree in the biological sciences. Competencies such as the ability to apply the process of science are best, if not exclusively, learned in the context of a laboratory class. Despite its importance, we are only beginning to understand how students learn the reasoning skills required to apply the process of science. In particular, what are the common misconceptions that interfere with this learning process, and what are the most effective strategies for teaching these skills in the laboratory? These questions are the focus of our research. One area of investigation has been to determine how students use the concepts of correlation, necessity, and sufficiency in interpreting experimental data. These concepts are derived from the basic idea of correlation vs. causation, and using them correctly is essential for reading and evaluating the scientific literature. We found that students frequently make two specific types of reasoning errors that interfere with their ability to correctly interpret data (Coleman et al., 2015). When presented with an experiment that only establishes a correlation between a potential intermediate and a known effect, students frequently interpret the intermediate as being sufficient (causative) for the effect. Also, when presented with an experiment that only determines the necessity of an intermediate for an effect, students frequently interpret it as also providing information on the sufficiency of the intermediate, and vice versa. By identifying these common errors, lab course curricula can be modified to correct for them and more successfully teach these scientific reasoning skills. Another ongoing area of investigation is to understand how the degree of inquiry in a laboratory course affects the learning of scientific reasoning skills. Inquiry is defined as the students determining one or more elements of the scientific process carried out in a lab experiment or project, rather than these being provided by the instructor. We have created a rubric of inquiry elements, such as interpreting preliminary data to form a hypothesis or designing an experiment by selecting different experimental conditions to test. Allowing the students to determine these elements is thought to improve learning outcomes but increases the logistical difficulty of implementing the experiment, particularly in a high-enrollment course. By understanding which inquiry-elements are most important for the learning of specific scientific reasoning skills, these competing factors can be balanced to maximize learning outcomes within the constraints of a given lab course. We are currently studying how specific inquiry elements affect students learning of hypothesis formation, data interpretation, and experimental design. Research on Fibroblast Growth Factor Signal Transduction There are 18 members of the fibroblast growth factor (FGF) family that act in a paracrine or endocrine fashion to accomplish a diverse array of biological effects, from pattern formation during development to wound healing. All of this is mediated through only four transmembrane FGF receptors. I am interested in deciphering the signal transduction by which individual FGFs produce their unique biological responses. \n",
            "-------------------- Scraping faculty url 30/183 --------------------\n",
            "heidi cook-andersen \n",
            "-------------------- Scraping faculty url 31/183 --------------------\n",
            "james cooke  Biography James Cooke received his Ph.D. from the University of British Columbia and was a postdoctoral fellow at the Hospital for Sick Children in Toronto. Dr. Cooke joined the faculty in 2015. He is currently the Head Faculty Advisor for the Saltman Quarterly.  Research As a teaching faculty member, my research is centred on undergraduate student learning. Our group is broadly interested in retention of course concepts by undergraduate students. One ongoing project uses multi-stage collaborative exams to try and improve long-term retention of course content. Our early results suggested that benefits conferred by writing answers in groups varied by individual. As a follow-up, we are measuring student behaviours during collaborative exams to identify whether certain behaviours are predictive of improved retention at a later time point. If we are able to identify specific actions that are beneficial to student learning, we can then guide future students to engage in these behaviours. Another ongoing project is evaluating whether we can improve content retention and exam performance by reducing the cognitive load due to jargon. We are curious to know if mastering key jargon terms prior to class removes barriers imposed by jargon, freeing up students ability to focus on the deeper, conceptual nature of the material. This project is done in collaboration with Dr. Timothy Rickard in the Department of Psychology at UCSD, and with Dr. Mark McDaniel at the University of Washington, St. Louis. \n",
            "-------------------- Scraping faculty url 32/183 --------------------\n",
            "kimberly cooper  Biography Kimberly Cooper completed her PhD at the Fred Hutchinson Cancer Research Center in Seattle, WA. She then carried out her postdoctoral studies in the Department of Genetics at Harvard Medical School. She joined the Division of Biological Sciences faculty at U.C. San Diego in November 2013.  Research I can get lost in the exhibits and drawers of a natural history museum, and to me the skeletons are the best part. Its fascinating that different species are so easily recognized by the isolated bones that are left behind when the natural processes of decomposition and fossilization are complete. One cant help wonder how the dolphin turned an arm into a flipper, how the aye-aye got its creepy long finger, and how the horse evolved to run gracefully on a single toe. Decades of research in the mouse and chick model systems as well as analyses of human birth defects have unlocked some of the mysteries of how the limb skeleton is patterned along three coordinate axes from the shoulder to finger tips, thumb to pinky finger, and back of the hand to palm. Yet neither human syndromes, nor the sledgehammer approach to knocking out genes in the mouse replicate the extraordinary range of adaptive morphologies observed in diverse species. This suggests a vast and untapped source of information about limb development and plasticity exists in the 400 million year genetic experiment thats been continuing since the first fishapod flopped onto land. To gain a window into what evolution tells us about the developmental potential of the limb, we primarily focus on the three-toed jerboa, a small bipedal rodent that is closely related to the laboratory mouse and amenable to rearing in captivity. Compared to its quadrupedal relative, the jerboa has greatly elongated hindlimbs with three toes and metatarsals that fuse into a single bone. The unusual morphology of the jerboa skeleton allows us to ask what keeps one bone from growing into its adjacent neighbor? What mechanisms determine skeletal size and the relative proportions of individual elements? How are the correct numbers and positions of the digits established, and are the same mechanisms of digit loss redeployed in species that converge on similar morphologies (i.e. jerboas and horses, pigs and camels)? How can the forelimb and hindlimb evolve independently when both utilize the same developmental pathways? We apply classical embryology, cell biology, quantitative microscopy, high throughput sequence analysis, and mouse genetics in an integrative approach to understand these developmental mechanisms at the tissue and genome level. \n",
            "-------------------- Scraping faculty url 33/183 --------------------\n",
            "nigel crawford  Biography Nigel Crawford received his Ph.D. from the Massachusetts Institute of Technology. He was an NSF Postdoctoral Fellow in the Department of Biochemistry at Stanford University.  Research Plants have evolved elaborate regulatory networks to respond rapidly to changes in the environment. Environmental and internal signals such as nitrogen and carbon metabolites, phytohormones, light, and circadian rhythms dramatically affect metabolic and developmental programs in plants. We are studying the metabolism of nitrate and are determining how it regulates plant metabolism and growth using microarray and mutant analyses in Arabidopsis. We have identified genes involved in nitrate uptake, reduction and regulation and have generated comprehensive datasets of nitrate-responsive genes. We have also studied nitric oxide metabolism and signaling and identified several genes that regulate nitric oxide accumulation and affect plant development and stress responses. Our work on nitrate uptake has focused on the functions of the AtNRT1.1 (CHL1) gene. AtNRT1.1 encodes a nitrate transporter whose expression is induced by nitrate, acid pH and carbon. For over twenty years, AtNRT1.1 was thought to function only in low affinity nitrate uptake in roots. Our work has shown that AtNRT1.1s function is much broader. First, AtNRT1.1 contributes to high affinity nitrate uptake in young plants and thus spans both high and low affinity systems. AtNRT1.1's contribution to each system depends on the environmental conditions in which the plants are grown and the age of the plants. Second, AtNRT1.1 is induced by auxin and is preferentially expressed in actively growing regions of the plant (root tips, young leaves and flowers) and contributes to their growth. AtNRT1.1 is also expressed and functions in guard cells. Compared to wildtype plants, Atnrt1.1 mutants have reduced stomatal opening and reduced transpiration rates in the light or when deprived of CO2 in the dark. At the cellular level, Atnrt1.1 mutants show reduced nitrate accumulation in guard cells during stomatal opening and fail to show a nitrate-induced depolarization of guard cells. In wildtype guard cells, nitrate induces depolarization, and nitrate concentrations increase three-fold during stomatal opening. These results identify the first anion transporter that functions in stomatal opening and suggest that nitrate plays an important role in stomatal movements and plant transpiration. In addition to serving as a nutrient and as an osmolyte, nitrate also acts as a signal. Nitrate reprograms nitrogen and carbon metabolism and influences root and shoot growth. To understand how plants perceive and respond to nitrate, we performed transcriptome analyses in Arabidopsis. Using ATH1 microarrays containing over 22,000 probe sets, we have uncovered over a thousand genes that are rapidly induced or repressed by nitrate treatment. In addition to known nitrate-responsive genes (e.g. those encoding nitrate transporters, nitrate reductase, nitrite reductase, ferredoxin reductase and enzymes in the pentose phosphate pathway), genes involved in glycolysis, trehalose-6-P biosynthesis, iron transport/metabolism, and sulfate uptake/reduction are also induced by nitrate. Because nitrate can be metabolized to nitrite, ammonium and amino acids, it is possible that downstream metabolites of nitrate could be the signals that induce some of these responses. To identify genes that respond specifically to nitrate, we constructed a nitrate reductase-null mutant in Arabidopsis. Microarray analysis revealed that almost 600 genes respond to nitrate (5 mM nitrate for 2 h) in both WT and mutant plants. This group of genes is over-represented most significantly in the functional categories of energy, metabolism, and glycolysis and gluconeogenesis. Because the nitrate response of these genes was nitrate reductase-independent, nitrate and not a downstream metabolite served as the signal for these genes. A different suite of genes was identified that require nitrate reduction for their response, implicating a downstream metabolite as the signal. We have shown that nitrite, the product of nitrate reduction, can act as a signal that is more potent than nitrate itself. To elucidate the genes and mechanisms that mediate these responses, we have developed several tools to facilitate our studies of nitrate signaling. We have developed a transient expression system to assay nitrate induction of gene expression and used it to identify a powerful cis-regulatory module that contains three nitrate enhancer elements (Plant Phys 154: 423-432). We used this regulatory module to drive GFP/YFP and GUS expression in transgenic plants to identify mutants defective in nitrate signaling. The first two genes identified encoded a transcription factor (AtNLP7) and the nitrate transporter AtNRT1.1 (Plant Phys. 151: 472-478). Atnrt1.1 mutants are defective in nitrate signaling under conditions where nitrate uptake is replete; thus, AtNRT1.1 serves as a nitrate regulator as well as transporter. Independently, the lab of Dr. Yi-Fang Tsay has shown that AtNRT1.1 transport function can be separated from its signaling function indicating that AtNRT1.1 acts as a nitrate transceptor (Cell 138: 1184-1194). In collaboration with Dr. Yong Wangs lab at Shandong Agricultural University, we identified a third gene encoding a novel, nuclear-localized protein (NRG2) that functions upstream of NRT1.1 to regulate nitrate-responsive genes (Plant Cell 28: 485-504). Lastly, we used the nitrate enhancer DNA in a one-hybrid screen of transcription factors. We identified a member of a large, plant-specific gene family (TCP20) that plays a role in nitrate foraging in plant roots, specifically in the systemic response system. These tools are making it possible to identify and study numerous genes that function in nitrogen signaling. \n",
            "-------------------- Scraping faculty url 34/183 --------------------\n",
            "matthew daugherty  Biography Matt Daugherty received his undergraduate degree from Wesleyan University and then spent three years working for a start-up biotech company in Chicago. He received his Ph.D. from UC San Francisco and did his postdoctoral training at the Fred Hutchinson Cancer Research Center in Seattle WA. He joined the Division of Biological Sciences at UCSD in 2016.  Research \"It takes all the running you can do, to keep in the same place.\" - The Red Queen to Alice, in Lewis Carrolls Through the Looking Glass Pathogens and their hosts are locked in a constant struggle for evolutionary dominance. Any host adaptation that leads to more effective defense against a pathogen will almost inevitably drive adaptation in the pathogen to escape such defenses. Thus, the only way either side can survive is to continually adapt, as proposed in Leigh Van Valens Red Queen hypothesis. These escalating arms races play out at the level of rapid evolution of host-pathogen interactions, constantly selecting for molecular innovations that either protect the host or evade host defenses. However, the molecular details of these rapidly evolving host-pathogen interactions, and the consequences of these conflicts on human susceptibility to current and emerging infectious diseases, are mostly unexplored. Our research aims to use these evolutionary principles, combined with virology and mechanistic biochemistry, to understand how our immune system has evolved to defend against pathogens and how pathogens counter-evolve to defeat host immunity. One of the approaches we take is to use comparative genomics to identify characteristic signatures of pathogen-driven innovation (e.g. rapid sequence evolution, horizontal gene transfer, gene birth and death, and gene recombination). Interestingly, we expect these rapidly evolving genes to be at the center of host-pathogen conflicts, yet many of them are almost entirely uncharacterized. This provides an opportunity to use pathogens to discover novel biological functions that are involved in not only pathogen immunity but also more global cellular regulation. Current projects in the lab focus on functional characterization of two such poorly understood systems: post-transcriptional modification of RNA to molecularly distinguish between different RNAs (e.g. host RNA versus viral RNA), and post-translational modifications of proteins to regulate their function. The ultimate goal of our research is to understand the rapidly evolving molecular means by which hosts and pathogens battle each other, while at the same time providing insight into how genetic variation resulting from these conflicts contributes to differences in human susceptibility to infectious diseases. \n",
            "-------------------- Scraping faculty url 35/183 --------------------\n",
            "michael david  Biography Michael David received his Ph.D in Pharmacology from the University of Vienna, Austria and did his postdoctoral research at the Center of Biologics Evaluation and Research in Bethesda, Maryland. He was an Erwin-Schroedinger and a Fogarty Fellow, and received Scholar Awards from the Sidney Kimmel Cancer Foundation and the National Multiple Sclerosis Society.  Research Interferons (IFNs) represent a group of cytokines that are best characterized for their ability to interfere with viral infection, but are also appreciated for their antiproliferative (antitumor) effects and their immunemodulatory properties. Consequently, interferons are used in clinical settings for the treatment of viral infections (e.g. Hepatitis C), malignancies (e.g. melanoma) and autoimmune disorders (e.g. Multiple Sclerosis). Our goal is to understand the mechanisms by which interferonsmediate these pleiotropic effects on a cellular and molecular level by investigating the following areas: 1) Characterization of the signal transduction mechanisms that permit IFNs to activate the transcription of defined cellular genes: Formation of DNA-binding complexes comprised of tyrosine phosphorylated STAT transcription factors involves the activation of members of the JAK family of tyrosine kinases. However, additional post-translational events ( e.g. modification through ubiquitin-like molecules, protein methylation, etc.) exert control over activation of STAT proteins, in addition, not all biological effects of IFNs are exclusively mediated by the JAK/STAT pathway. 2) Define the cellular and molecular events that govern the immunemodulatory properties of interferons: We have identified an important novel function of type I interferons and STAT1 in the positive and negative selection of T cells during thymic development, as well as T cell activation. Our findings are likely to be of great significance as a link between viral infections and the development of autoimmune diseases, and furthermore have direct implications on the efficacy of interferons in the treatment of immune disorders. 3) Elucidate the pathways by which the immune system recognizes bacterial or viral infection and initiates an innate immune response: The induction of type I interferons is one of the earliest innate immune responses to viral as well as bacterial pathogens. We had previously discovered that Toll-like receptors (TLRs) activate the transcription factor IRF3, which is essential for the production of interferons as well as other pro-inflammatory cytokines after pathogen recognition. We are interested in the signaling mechanisms that lead to IRF3 activation, and how invading pathogens (e.g. Hepatitis C virus, B. anthracis) can interfere with these pathways and consequently paralyze the hosts innate immune response? \n",
            "-------------------- Scraping faculty url 36/183 --------------------\n",
            "arshad desai  Biography Arshad Desai received his Ph.D. in cell biology from the University of California, San Francisco and conducted postdoctoral work at the European Molecular Biology Laboratory in Heidelberg and the Max Planck Institute for Cell Biology & Genetics in Dresden. He joined the UC San Diego faculty in late 2002 and has been the recipient of a Damon Runyon Scholar Award, the American Society for Cell Biology Early Career Award, and a Keith Porter Fellowship. He is a Senior Editor of the Journal of Cell Biology, has chaired review panels for the National Institutes of Health and the European Research Council, and is a lifetime fellow of the American Society for Cell Biology.  Research We are focused on understanding the pathways and molecular mechanisms that ensure accurate genome inheritance during cell division. Errors in genome inheritance are the cause of birth defects and are implicated in both cancer initiation and progression. In addition, the machinery involved in genome inheritance is a prominent target in cancer chemotherapy. Our interest in cell division stems from a long-standing fascination with the microtubule cytoskeleton, comprised of highly dynamic protein polymers that employ energy from assembly-coupled nucleotide hydrolysis to perform useful work in the cell. This interest led us to focus on the kinetochore, the macromolecular machine assembled on the centromere regions of chromosomes during cell division in order to harness the intrinsic dynamics of microtubule polymers for chromosome segregation. To study the pathways and mechanisms operating during chromosome segregation, we have developed the early C. elegans embryo as a powerful model for discovery and characterization of the underlying molecular machinery. Our early efforts led to the definition of the core microtubule binding protein network at the kinetochore and the identification of a conserved protein family involved in epigenetic propagation of centromeric chromatin. In more recent work, we have elucidated multiple mechanisms acting in concert with the core microtubule-binding machinery to ensure accurate chromosome segregation and have begun to untangle the complex coupling between mechanical and signaling events at the kinetochore that is critical for accurate segregation. In addition to our focus on the dynamic kinetochore-microtubule interface and the regulatory mechanisms operating to ensure its correct formation, we are also pursuing projects on the epigenetic basis of centromere identity, and on the checkpoint pathways coordinating chromosome segregation with cell cycle progression. Our findings are relevant to the genesis of cancer genome instability and are leading us to collaboratively explore new mechanism-targeted approaches focused on cell division for cancer therapy. Our use of C. elegans is also leading us to discover new conserved cell division-independent functions for ancient chromosome segregation machinery, e.g. in neurodevelopment. \n",
            "-------------------- Scraping faculty url 37/183 --------------------\n",
            "russell doolittle  Biography Russell Doolittle received his Ph.D. from Harvard University. He was named a Guggenheim Fellow and was elected to the National Academy of Sciences and the American Academy of Arts and Sciences. Professor Doolittle is a co-recipient of the Paul Ehrlich Prize.  Research Our group is mainly concerned with the structure and evolution of proteins. In this regard, we have two quite distinct projects under way. One is very general and has to do with reconstructing the evolutionary histories of a wide variety of proteins. This is a computer-based study that draws upon published sequence data for its raw material. We employ a number of sequence searching and alignment programs, many of which we have written ourselves. Among the questions we are trying to answer are: When did prokaryotic organisms diverge from eukaryotes? How many rudimentary families of proteins are there? How are \"new proteins\" invented? All of these questions can be answered, given enough sequences to compare Our second major research interest is laboratory-based and deals with the invention and evolution of vertebrate blood plasma proteins, and especially the clotting proteins. In the past, we have cloned and sequenced a number of these proteins from the most primitive of vertebrates, the lamprey. Comparison with the corresponding mammalian proteins has afforded us some important clues as to how these proteins function. We also succeeded in identifying equivalent gene products in even more distantly related creatures, including protochordates and invertebrates. Of all of these, we have focused most on the fibrinogen molecule. Our goal here is to understand how fibrin formation (clotting) occurs and was invented. In this regard, after many years of trying we managed to crystallize a native fibrinogen molecule (chicken fibrinogen = 320 KDa) and solved its X-ray structure. We have also solved the structure of various fibrinogen fragments with bound ligands involved in fibrin formation, and also factor XIII-generated crosslinked versions. The crystallography project should shed light not only on how clotting works but also on where some of the components came from. Fibrinogen is a multidomained mosaic protein, a key part of which is found in numerous other animal proteins, including various cytotactins (e.g., tenascin and T-cell factors). Crystal structure of fragment D from human fibrinogen \n",
            "-------------------- Scraping faculty url 38/183 --------------------\n",
            "rachel dutton \n",
            "-------------------- Scraping faculty url 39/183 --------------------\n",
            "joseph ecker \n",
            "-------------------- Scraping faculty url 40/183 --------------------\n",
            "dannielle engle  Biography Dannielle Engle finished her Ph.D. in the laboratory of Dr. Geoffrey M. Wahl at the Salk Institute for Biological Studies in the Biological Sciences program of the University of California, San Diego. Engle completed her postdoctoral fellowship in the laboratory of Dr. David. A. Tuveson beginning at the Cambridge Research Institute and finishing at Cold Spring Harbor Laboratory as a Senior Fellow. Engle also organized scientific outreach programs as the Lustgarten Foundation Pancreatic Cancer Research Laboratory Ambassador and is a recipient of the Theodore T. Puck and NCI K99/R00 Pathway to Independence Awards. Engle returned to the Salk Institute in 2019 as an Assistant Professor.  Research Having lost close family members to pancreatic cancer, Dannielle Engle has a personal connection to the disease. Engle uses her personal and scientific passion as well as her expertise in modeling to facilitate progress in pancreatic cancer research by creating better representations of what actually happens in patients. The identification of biomarkers and therapeutic targets has been hindered by the limited access to pure sample populations and accurate models. When looking at a patient's blood samples, there can be hidden signals from the incipient tumor, but these are masked and diluted by the multitude of other signals from all over the body. In addition, when looking at the pancreatic tumor tissue itself, most of our time is spent looking at signals from non-cancer cell types in the scar-like tissue that takes up the majority of the tumor in human patients. Traditional approaches have used cell lines that do not accurately represent the early stages of pancreatic cancer. Engle tackles these challenges by creating organoid and genetically engineered mouse models to better recapitulate pancreatic disease. Using these systems, the Engle lab identifies new therapeutic targets in organoids and tests their efficacy in autochthonous, immune competent mouse models. In addition, the need for more effective treatment options goes hand-in-hand with the ability to diagnose pancreatic cancer before it spreads. Accordingly, the Engle lab also focuses on the identification of biomarkers that unambiguously differentiate between pancreatic cancer and other inflammatory conditions. \n",
            "-------------------- Scraping faculty url 41/183 --------------------\n",
            "mark estelle \n",
            "-------------------- Scraping faculty url 42/183 --------------------\n",
            "ronald evans \n",
            "-------------------- Scraping faculty url 43/183 --------------------\n",
            "emma farley \n",
            "-------------------- Scraping faculty url 44/183 --------------------\n",
            "gen-sheng feng  Biography Dr. Feng received his PhD degree at Indiana University Bloomington and did a postdoctoral fellowship at the University of Toronto.  Research Molecular Signaling in Cancer, Metabolism and Stem Cells At the heart of cancer research is the elucidation of oncogenes and tumor suppressor genes (or anti-oncogenes), which has been a driving force in the cancer field for several decades. Most recent studies in this and other labs have revealed dual roles of genes in promoting and suppressing hepatocellular carcinoma (HCC). Deciphering the paradoxical effects of these genes may provide a fresh view on the fundamental mechanisms of carcinogenesis, which can lead to development of mechanism-based therapeutics for HCC. Our current focuses are on delineating genetic and epigenetic interactions and cell-cell communications that drive hepatocarcinogenesis. We are very interested in isolating and characterizing cancer stem cells, CSCs (or tumor-initiating cells, TICs) in the liver, and in dissecting the dynamic interplay between tumors and the hepatic microenvironment. Efforts are also being devoted to elucidation of metabolic changes in tumor cells and to search of biomarkers for early diagnosis of liver cancers. We are also interested in deciphering the common and distinct biochemical pathways in self-renewal of hematopoietic stem cells (HSCs) and leukemia stem cells (LSCs), with the objective of understanding how an HSC pool is maintained throughout life, and with the ultimate goal of identifying novel targets for elimination of LSCs. Another focus is on dissecting the signaling events involved in metabolic disorders such as obesity and diabetes. Ongoing experiments are aimed at understanding the intertwining of leptin and estrogen signaling cascades, to illustrate why estrogen has a leptin-like effect in control of metabolism, why postmenopausal women have the tendency to develop obesity, and why obesity promotes breast cancer. Whereas obesity has become a serious health problem in developed countries worldwide, one ongoing project in the lab is to investigate an indispensable physiological role of adipogenesis and adipose tissue in mammals, as revealed by the most recent data obtained in this lab. \n",
            "-------------------- Scraping faculty url 45/183 --------------------\n",
            "rick firtel  Biography Rick Firtel was an undergraduate in Biology and Chemistry and Rufus Choate Scholar at Dartmouth College (1966) and received his PhD at Caltech in James Bonners laboratory as an NSF fellow (1971). Rick did his postdoctoral work with Harvey Lodish at MIT as a Helen Hay Whitney Foundation Fellow and joined the faculty at UCSD in 1973. He has been the recipient of an American Cancer Society Faculty Research Award, an International Union Against Cancer Fellowship, a Japanese Society for the Promotion of Sciences Fellowship, and a John Simon Guggenheim Fellowship. He has served as Director of the Center for Molecular Genetics (1995-2004), Chair of the Section of Cell and Developmental Biology (1999-2006), and Associate Dean of Biological Sciences (2008-2017).  Research Chemotaxis of wild-type cells to cAMP emitted from a pipette. Our laboratory focuses on the signal transduction pathways and the mechanics that control directional cell movement, or chemotaxis, and morphogenesis of eukaryotic cells. Chemotaxis is a basic property of many eukaryotic cells and plays critical roles in diverse biological pathways including wound healing, migration of leukocytes to sites of inflammation and bacterial infection, morphogenesis, and metastasis of a variety of cancer cell types. Chemotaxis is mediated by small molecule ligands (chemoattractants and chemokines) that interact with cell surface receptors to control downstream effector pathways. Cells are able to sense and respond to even weak chemoattractant gradients and to amplify a weak chemical gradient into a steep intracellular gradient of signaling molecules. Cells then translate this into changes in the cytoskeleton that control the mechanical forces that drive cell movement. Co-activation of Ras and PI3K at the leading edge of chemotaxing cells using real-time reporters Over the past few years, great progress has been made in elucidating the molecular mechanisms controlling the ability of cells to sense and respond to chemical gradients. My laboratory and others have discovered that Ras lies at the top of the signal transduction cascade leading to chemotaxis. We demonstrated that Ras controls chemotaxis through the activation of phosphatidylinositol 3-kinase (PI3K) and Tor Complex 2, which are also essential for cell growth and survival in metazoan cells. Our goal is to reveal the molecular mechanisms regulating these and other signaling pathways that enable cells to chemotax. Because most of these components are highly conserved evolutionarily in cells ranging from Dictyostelium amoebae to human leukocytes, we use Dictyostelium as our primary experimental system, which allows us to employ a wide range of genetic as well as biochemical approaches to dissect these signaling pathways. These methods are combined with in vivo, real-time analysis of GFP fusions of signaling proteins in living cells to understand the spatio-temporal changes of these proteins in response to directional signals. These diverse approaches have allowed us elucidate the basic principles by which signal transduction pathways control directional sensing and chemotaxis that apply to human leukocytes as well as Dictyostelium cells. A chemotaxing Dictyostelium cell (gray) forms stationary traction adhesions with the substratum along its anterior-posterior axis that transmit force (indicated by heat map) and allow the cell to move. The colored cylinder shows the path of the migrating cell over time. A major unanswered question in cell motility is how the signaling pathways and the reorganizations of the cytoskeleton function together to control cell movement. In collaboration with the groups of Juan Lasheras and Juan Carlos del Alamo in the Department of Mechanical and Aerospace Engineering, we have developed quantitative approaches to examine the time evolution of the traction forces that mediate each stage of the cell motility cycle. We have made the novel finding that amoeboid cell movement is controlled by a complex interplay between lateral and normal (vertical) forces mediated by cortical tension controlled through the lateral cell cortex as well as the more classic F-actin/myosin-mediated axial forces. We have achieved this by integrating the biochemical and mechanical measurements of cell motility and obtained the quantitative information necessary to connect specific biochemical and signaling processes to the physical events regulating the mechanical forces controlling cell motility and, in do so, understand how cells move. In collaboration with Wouter-Jan Rappel, Alex Groisman, and Bill Loomis, we are attempting to mathematically model the signaling events controlling cell polarization. \n",
            "-------------------- Scraping faculty url 46/183 --------------------\n",
            "douglass forbes  Biography Douglass Forbes received her Ph.D. from the University of Oregon and was an American Cancer Society postdoctoral fellow at UCSF in the Dept. of Biochemistry and Biophysics. She was a PEW Fellow in the Biomedical Sciences, and an elected member of the Governing Council of the American Society of Cell Biology. Dr. Forbes served as the Vice Chair of Cell and Developmental Biology from 2000-2005 and 2007-2009. She has served on multiple editorial boards, both past and present.  Research Our lab has two primary areas of interest: (1) the structure, function and assembly of the vertebrate nucleus, and (2) the mechanism by which the cell regulates where and how to assemble major mitotic structures, such as the mitotic spindle, the nuclear membranes and nuclear pore complexes. The Nucleus: Structure, Function and Assembly A focus of the research in my laboratory is the eukaryotic nucleus: its structure and function. We work both in vivo and in vitro. For in vivo studies, we use mammalian cultured cell lines to examine nuclear transport or assembly using RNAi technology to knock down expression of genes of interest to determine their effect on function. For in vitro studies, we use a nuclear reconstitution system which efficiently assembles nuclei in the test tube from soluble and membrane components. This system makes use of an extract of Xenopus eggs, which naturally store abundant amounts of disassembled nuclear components in preparation for the rapid cleavages of early division. Upon addition of DNA or chromatin to the in vitro interphase extract, nuclei containing nuclear membranes, a nuclear lamina, and nuclear pores quickly assemble. The in vitro reconstituted nuclei are capable of nuclear import, DNA replication, pol III transcription, and even mitotic disassembly. In parallel, we use a mitotic egg extract to study spindle and kinetochore assembly. A long-standing interest of the laboratory has been the nuclear pore, a large macromolecular complex of 120 million daltons that spans the nuclear membranes. All communication between the nucleus and cytoplasm occurs through the nuclear pore. In vivo, the pore complex imports nuclear proteins and exports mRNA, tRNA, snRNAs, and ribosomal precursors. We study: (1) assembly of the nucleus itself, (2) assembly of nuclear pores, (3) the regulation of these processes, and (4) the mechanism of nuclear transport, including mRNA export, protein export, and protein import. Using antibodies we can immunodeplete a nuclear assembly extract of individual nuclear pore proteins, then reconstitute nuclei which lack that protein. Such \"designer\" nuclei can be tested for alterations in nuclear pore structure, pore function, or pore assembly. In this way we discovered the protein responsible for initiating pore assembly at the nucleus (Rasala et al, 2006, 2008). This protein, ELYS, acts by binding to chromatin at AT-rich sequences and recruiting key structural subunits of the nuclear pore to these sites to initiate nuclear pore assembly. With experiments of this type, we hope to molecularly dissect and elucidate the gates that \"guard the fortress\" of the genome. Regulation of the Major Mitotic Assembly Events: Cellular GPSs The second major interest of the laboratory involves understanding the regulation of assembly of the major mitotic structures. Specifically, we are focused on how assembly is regulated such that the mitotic spindle, and later the nuclear membranes and nuclear pores, form only around chromatin, i.e., in a spatially precise manner. We have shown that an abundant cellular protein, Importin beta, acts in mitosis to negatively regulate both nuclear membrane fusion and nuclear pore assembly everywhere except in the vicinity of the chromosomes (Harel et al, M.B.C, 2003). The small GTPase Ran, produced in its GTP-bound form only near chromatin, acts as the counteracting positive regulator, a GPS to tell where assembly should occur. We have extended this work to Transportin, a second karyopherin. We believe that Importin beta and Transportin act as global regulators of cellular events involving the genome, including those controlling nuclear import, spindle formation, correct nuclear membrane assembly, and nuclear pore assembly. \n",
            "-------------------- Scraping faculty url 47/183 --------------------\n",
            "p. a. george fortes \n",
            "-------------------- Scraping faculty url 48/183 --------------------\n",
            "kathleen french \n",
            "-------------------- Scraping faculty url 49/183 --------------------\n",
            "fred gage \n",
            "-------------------- Scraping faculty url 50/183 --------------------\n",
            "e. peter geiduschek  Biography E. Peter Geiduschek received his Ph.D. from Harvard University. He was awarded a Guggenheim Fellowship and has served on the editorial boards of several scientific journals. Dr. Geiduschek is a member of the National Academy of Sciences and the American Academy of Arts and Sciences and is also a fellow of the American Association for the Advancement of Science and the American Academy of Microbiology.  Research The central theme of my research has been the enzymology of gene regulation, specifically of transcriptional regulation and its principal focus has been on fundamental mechanisms, approached from a molecular genetic and biochemical perspective. It is now widely appreciated that the transcriptional apparatus of all cells (archaeal, bacterial and eukaryotic) has common evolutionary roots. It follows that transcription-regulatory mechanisms should also share certain broadly similar mechanistic features. In that spirit, my laboratory worked in parallel with bacteria and their viruses, with eukaryotes, and with archaea. \n",
            "-------------------- Scraping faculty url 51/183 --------------------\n",
            "tim gentner \n",
            "-------------------- Scraping faculty url 52/183 --------------------\n",
            "stacey glasgow  Biography Stacey Glasgow received her Ph.D. in Integrative Biology from UT Southwestern Medical Center with Dr. Jane E. Johnson. She carried out her postdoctoral training with Dr. Benjamin Deneen in the Center for Cell and Gene Therapy at Baylor College of Medicine. Stacey is the recipient of a K01 Mentored Research Scientist Development Award issued by the NIH/NCI.  Research Glia are the most abundant cell type in the adult central nervous system (CNS) and support a diverse array of physiological functions including myelination, synaptogenesis, neurotransmission, and blood-brain barrier maintenance. Consistent with this functional heterogeneity, disruptions in normal gliogenesis and glial maintenance have been reported as defining events in the onset and progression of several neurological disorders and malignancies. Despite their essential and emerging roles in CNS functions and pathologies, the molecular mechanisms that govern glial cell specification from neural progenitors and how these mechanisms are correlated with associated diseases are poorly defined. Specification and lineage commitment involves coordinated gene expression regulated by chromatin remodeling, transcriptional cascades, and binding site recognition. Understanding how these regulatory layers converge to influence gene expression remains a key question with implications in neural development, function, and associated disease. In my laboratory, we use interdisciplinary approaches to study both glial development and ultimately, neurodegenerative diseases and glioma. Specifically, we are interested in dissecting the mechanisms underlying transcriptional regulation and chromatin architecture of glial cell development and applying these developmental concepts to glioma tumorigenesis. Methodologically, we utilize manipulation of both chick and mouse model systems in combination with biochemical and transcriptional assays to help elucidate these molecular underpinnings. \n",
            "-------------------- Scraping faculty url 53/183 --------------------\n",
            "james w. golden  Biography Dr. James W. Golden received a B.S. (1977) in Microbiology from the University of Maryland-College Park and a Ph.D. (1983) in Biology from the University of Missouri-Columbia. After postdoctoral work as an NIH Fellow at The University of Chicago, he joined the Department of Biology at Texas A&M University in 1986. He was promoted to Associate Professor in 1990 and then to Professor in 1996. Dr. Golden moved to the University of California, San Diego in 2008.  Research Genetics, Molecular Biology, and Biotechnology of Cyanobacteria Current areas of research include the development of improved genetic tools for cyanobacterial genetic engineering and the use of these tools to answer basic-science questions and for biotechnology applications. Our biotechnology-related research involves genetic engineering of cyanobacteria to synthesize bioactive natural products and biofuels. Other projects include identifying cyanobacterial genes related to resistance to grazing by amoebal and ciliate predators, the study of genes required for biosynthesis of cyanobacterial toxins, and the identification of cyanobacterial genes that affect fitness for growth during spaceflight or on Mars. These research projects include work with different strains of cyanobacteria, but focus on the laboratory model strain Synechococcus elongatus, strain PCC 7942, which has excellent genetics and is widely used for synthetic biology studies. Past research has focused on the developmental biology of cyanobacterial heterocyst formation, with an emphasis on the genetic regulation of cellular differentiation and the cell-to-cell signaling mechanisms that control multicellular pattern formation. This research uses methods of genetics and molecular biology to understand basic principles of regulation and signaling pathways that control development in a simple prokaryotic multicellular organism, the filamentous cyanobacterium Anabaena (Nostoc), strain PCC 7120. Like all cyanobacteria, Anabaena uses light energy for photosynthesis. Anabaena is also capable of nitrogen fixation, a process that is incompatible with photosynthesis because the nitrogenase enzyme is destroyed by oxygen, a byproduct of photosynthesis. Anabaena solves this problem by spatially separating the two processes into different cell types: photosynthetic vegetative cells and nitrogen-fixing heterocysts. Anabaena grows as a very simple multicellular organism composed of filaments of vegetative cells containing about 10 percent heterocysts. Heterocysts differentiate from vegetative cells at semiregular intervals along the filament and supply fixed nitrogen to neighboring vegetative cells to support their growth. We identified a gene, patS, which encodes a small peptide that functions as a diffusible cell-to-cell signal that acts to control heterocyst pattern formation. \n",
            "-------------------- Scraping faculty url 54/183 --------------------\n",
            "susan golden  Biography Susan Golden received a B.A. (1978) in Biology from Mississippi University for Women and a Ph.D. (1983) in Genetics from the University of Missouri. After postdoctoral research at the University of Chicago, she joined the faculty of Biology at Texas A&M University (1986), where she was promoted to Distinguished Professor in 2003. She joined the Division of Biological Sciences at UCSD in 2008. During her graduate work she developed genetic tools for the cyanobacterium Synechococcus elongatus (PCC 7942), the first cyanobacterium shown to be subject to genetic transformation. This led to work on regulation of light-responsive photosynthesis gene expression in this organism during her postdoctoral research and at Texas A&M. In the early 1990s she began a collaborative project with C.H. Johnson (Vanderbilt University) and T. Kondo (Nagoya University) that demonstrated circadian rhythms of gene expression in S. elongatus, which is currently the only model organism for a prokaryotic circadian clock. The molecular basis of timekeeping in S. elongatus is now a major focus of her lab. Susan is a member of the Faculty of 1000 Biology, a Fellow of the American Academy of Microbiology, and a Member of the National Academy of Sciences.  Research Circadian Rhythms of Gene Expression in Cyanobacteria Diverse eukaryotes, and at least one group of prokaryotescyanobacteriause a circadian (24 h) clock to control physiological events and gene expression. S. elongatus shows a circadian rhythm of bioluminescence when it is transformed with a reporter gene that encodes the luciferase of a bioluminescent marine bacterium or firefly. We are using this reporter system to identify components of circadian system. A group of interacting proteins, KaiA, KaiB, and KaiC are at the heart of the cyanobacterial clock, and kinases help to set the clock and transmit temporal information from it to the genes it regulates. Our goal is to understand the basic mechanism of timekeeping and how the clock becomes synchronized with the environment and controls cellular processes. Functional Genomics in S. elongatus We are identifying all of the genes in the S. elongatus genome that contribute to circadian rhythms by creating a mutation in each gene and assaying each mutant for circadian phenotypes. This project uses transposon insertions into cosmids and plasmids that contain pieces of S. elongatus DNA; each clone that carries a transposon insertion can be used for recombination into the S. elongatus genome to produce a gene \"knockout.\" Metabolic Engineering of Cyanobacteria for the production of Biofuels and other Molecules of Interest Because cyanobacteria grow photosynthetically using water and CO 2 and are easy to manipulate genetically, they are attractive organisms for the production of molecules that have industrial applications. One such application is the production of biofuels as a supplementation of or eventual replacement of petroleum fuels. We are using the powerful genetic tools that have been developed for S. elongatus to explore the production of biofuels in cyanobacteria. \n",
            "-------------------- Scraping faculty url 55/183 --------------------\n",
            "ananda goldrath \n",
            "-------------------- Scraping faculty url 56/183 --------------------\n",
            "martyn goulding \n",
            "-------------------- Scraping faculty url 57/183 --------------------\n",
            "barry grant  Biography Barry Grant received his undergraduate degree from Queens University Belfast and masters in Bioinformatics from the University of York (UK). He then worked at DeCode Genetics, Iceland before obtaining a Ph.D. from the University of York and Marie Curie Research Institute. Barry a was postdoctoral fellow at UC San Diego and the Howard Hughes Medical Institute. Before joining our faculty in 2017 Barry was an Assistant Professor of Computational Medicine and Bioinformatics at the University of Michigan, Ann Arbor.  Research My research focuses on the development and application of state-of-the-art computational and theoretical techniques to investigate the structure, dynamics and interactions of biomedically important proteins. At the interface of molecular biology, biophysics and bioinformatics the unique insight gained from our research is tightly coupled to a wide range of biochemical and biophysical experiments. Using this approach, we have recently developed a new class of allosteric Ras & Rho molecular switch inhibitors, reported the first rationally designed faster velocity kinesin motors, demonstrated for the first time that kinesin processivity can be modulated by rational mutation of the motor domain, discovered key atomistic detriments of allosteric activation in G proteins, and developed the Bio3D software package used by thousands of researchers around the world. Our specific areas of expertise include: Bioinformatics Development and application of bioinformatics methods for the analysis of protein sequence-structure-function relationships. Biomolecular Simulation Application of advanced molecular simulation techniques to probe the structure, dynamics and interactions of biomolecules. Drug Discovery & Design Virtual screening, structure-based drug design, cheminformatics and molecular modeling supporting drug discovery and design. \n",
            "-------------------- Scraping faculty url 58/183 --------------------\n",
            "melvin green \n",
            "-------------------- Scraping faculty url 59/183 --------------------\n",
            "ralph greenspan  Biography Dr. Ralph J. Greenspan has worked on the genetic basis of behavior and brain function in fruit flies (Drosophila melanogaster) almost since the inception of the field, studying with one of its founders, Jeffery Hall, at Brandeis University in Massachusetts, where he received his Ph.D. in biology in 1979. He is currently Director of the Center for Brain Activity Mapping of the Kavli Institute for Brain and Mind at the University of California San Diego, and Professor in the Division of BIologys Neurobiology Section and the Dept. of Cognitive Science.  Research Dr. Greenspans research includes studies of the consequences of mutations and localized genetic alterations in the nervous system, molecular identification of genes underlying naturally occurring variation in behavior, and the genetic analysis of sleep and attention in the fruit fly. His current research addresses large-scale network interactions pertaining to the action of genes and neurons. In 2011, he was one of the small team of scientists that produced the white paper for the White House Office of Science and Technology Policy that eventuated in the BRAIN Initiative, and in 2014 spearheaded the effort to establish the state-funded Cal-BRAIN program and serves as its co-Director. In addition to research papers, he has authored: Fly Pushing: The Theory and Practice of Drosophila Genetics, An Introduction to Nervous Systems, and How Genes Influence Behaviour (with Jonathan Flint and Ken Kendler). \n",
            "-------------------- Scraping faculty url 60/183 --------------------\n",
            "shelley halpain  Biography Shelley Halpain received her B.S. in Biological Sciences at The University of California Irvine, and her Ph.D. in Neuroscience from the Rockefeller University. Her postdoctoral training was under Professor Paul Greengard, a 2001 Nobel laureate in Physiology or Medicine. Dr. Halpain was on the faculty at the University of Virginia Health Sciences Center 1992-1996, then at The Scripps Research Institute 1996-2007. She became Professor in the Division of Biological Sciences at UCSD in 2007. Dr. Halpain served as Council Delegate for the AAAS Section on Neuroscience from 1998-2000, is a member of national and international scientific review panels, and is a frequent contributor of reviews and commentary for international science journals. Professor Halpain's research has been published in top scientific journals and has been featured in National Geographic magazine.  Research Dr. Halpains research concerns the molecular basis for neural development. Her laboratory uses advanced light microscopy and quantitative cellular imaging methods to investigate neurite outgrowth and synapse formation. The lab also develops automated, high-content screening assays of neuronal morphology for use in drug discovery and identification of novel signaling pathways. Proteomics analyses are used to investigate key players in neuritogenesis. A particular focus is the role that cytoskeletal proteins, especially microtubules and actin filaments, play in establishing and maintaining neuronal networks. The laboraotorys work is fundamentally relevant to many brain disorders, ranging from Alzheimers disease to mood disorders and autism. \n",
            "-------------------- Scraping faculty url 61/183 --------------------\n",
            "randy hampton  Biography Randy Hampton received his Ph. D. from the University of Wisconsin-Madison and did his postdoctoral work at UC Berkeley.  Research Our group studies a variety of questions that surround how the cell selectively destroys proteins. This work started with understanding how these sorts of mechanisms are employed to allow regulation of cellular cholesterol synthesis. These studies are ongoing and have led us in a number of interesting directions including cellular management of misfolded proteins, lipid signaling, and the dynamics of the ER and other organelles. For a complete picture of what weve done, what we do, and what we want to do, see the Hampton lab web site listed above. Thanks for stopping by! \n",
            "-------------------- Scraping faculty url 62/183 --------------------\n",
            "sung han \n",
            "-------------------- Scraping faculty url 63/183 --------------------\n",
            "nan hao  Biography Nan Hao received his Ph.D in Biochemistry and Biophysics from the University of North Carolina at Chapel Hill. He was a postdoctoral fellow with Timothy Elston at the University of North Carolina and a postdoctoral fellow with Erin OShea at Harvard University/Howard Hughes Medical Institute.  Research In recent years, genomic sequencing and systematic analysis have revealed many molecular components that control gene expression at multiple levels, and detailed the myriad of interactions among and between these regulatory modules. However, it remains unclear how these molecular networks operate in time and space to carry out diverse cellular functions. The goal of our lab is to understand how network architecture governs the dynamics and function of regulatory responses in the context of stresses , aging , or diseases . Our lab uses multidisciplinary approaches, integrating biology, engineering and computer science, in our studies. We develop and use novel microfluidics and imaging technologies to quantitatively track the dynamics of molecular processes in single living cells and construct computational models to describe and predict cellular behaviors. In particular, our laboratory focuses on the following three major research themes: Decoding the dynamics of stress responses Cells could transmit environmental information and initiate cellular responses through regulating the temporal dynamics of signaling activities. Our current work focuses on understanding: (1) the network structures that encode the environmental information into signaling dynamics; (2) the gene circuits that decode signaling dynamics into functions; (3) the mechanisms that underlie cellular memory of environmental changes. Probing the causes of cellular aging Cellular aging plays an important role in many diseases, such as cancers and neurodegenerative disorders, but the mechanisms that drive the aging process remain largely unclear. We aim to combine transformative microfluidics and single-cell imaging technologies with computational modeling to obtain a quantitative and predictive understanding of the aging process and to develop new strategies to promote longevity. Quantifying the heterogeneity in cancer cells We have expanded our quantitative biology research into mammalian cell systems. We have developed new microfluidic and imaging technologies to track individual mammalian cells over a very long period of time. Our current work focuses on studying the heterogeneous signaling dynamics in cancer cells and how these dynamics influence cancer cell growth. \n",
            "-------------------- Scraping faculty url 64/183 --------------------\n",
            "diana hargreaves \n",
            "-------------------- Scraping faculty url 65/183 --------------------\n",
            "jeff hasty  Biography Jeff Hasty received his Ph.D. in Physics from the Georgia Institute of Technology in 1997, where he worked with Kurt Wiesenfeld. He was a postdoctoral fellow with Jorge Vinals at the Supercomputing Research Institute ('97-'98), and apostdoctoral fellow with Jim Collins in the Applied BioDynamics Lab at Boston University ('98-'01). Somewhereduring his postdoctoral stay at Boston University, he mutated into a hybrid computational/molecular biologist. He is currently at the University of California, San Diego, where he is a Professor in the Departmentsof Molecular Biology and Bioengineering, and the Director of the BioCircuits Institute. His main interest is thedesign and construction of synthetic gene-regulatory and signaling networks.  Research Dr. Hasty's research focuses on the construction and utilization of synthetic gene circuits for dissecting, analyzing, and controlling the dynamical interactions involved in gene regulation. \n",
            "-------------------- Scraping faculty url 66/183 --------------------\n",
            "stephen hedrick  Biography Dr. Hedrick received his Ph.D. from the University of California, Irvine and conducted postdoctoral studies at the National Institutes of Health.  Research Immune homeostasis: the sum of lymphocyte dynamics Although homeostatic regulation is universal in biological systems, it is prominently in evidence within lymphocyte populations, and like almost every aspect of the immune system, the dynamics of lymphoid populations are wildly exaggerated. An infectious agent can provoke quiescent, antigen-specific T cells to suddenly grow in size, proliferate more than 10,000-fold, and then at a prescribed point in the process collectively undergo an equally dramatic and selective contraction in numbers. Although such events are extravagant compared with the constant turnover exhibited by other organismal systems, they are not without control. The height of the expansion and the plateau of survival are both regulated and in some manner self-limited. Moreover, the body appears to have a set-point for total lymphocytes diffuse throughout the body, about which the populations oscillate. In addition to interesting biology, this regulation of lymphocytes is important for almost all of the diseases affecting an aging population: immune hypersensitivity, including asthma; autoimmunity, such as rheumatoid arthritis and psoriasis; cardiovascular disease; cancer; and metabolic diseases such as Type II diabetes. We are interested in understanding the conceptual basis for these immunopathologies. Lymphocyte survival and quiescence. We have found that a family of Forkhead transcription factors (named Foxo) are essential for the maintenance of many aspects of T cell homeostasis. Two of the family members, Foxo1 and Foxo3, have unique functions within the immune system, where Foxo1 regulates survival and homing properties of T cells (Kerdiles et al., 2009) and Foxo3 regulates the ability of dendritic cells to promote T cell survival (Dejean et al., 2009). In addition, the absence of Foxo1 in T cells results in chronic immune inflammation and autoreactive antibodies. The factors also have overlapping functions in that the T cell-specific deletion of Foxo1,3 results in florid autoimmunity. We are interested in understanding the program of gene expression controlled by Foxo transcription factors, and how the disregulation of this control results in immunopathology. Lymphocyte expansion and survival. A central signaling pathway in T cell activation is centered around the mitogen-activated protein kinases, Erk1 and Erk2. These factors are collectively essential for progression through the cell cycle, and individually they regulate proliferation, survival and the fate of antigen-activated T cells. We have found that antigen-mediated T cell activation is governed in part by the strength of signal transmitted by Erk2, and this regulates the magnitude of T cell expansion (DSouza et al., 2008). We have also found that the absence of Erk2 promotes an alternative differentiation program in CD4 T cells resulting in an abnormally high number of Foxp3+ T regulatory cells. We are exploring a connection between Erk2, TGFb, and Foxo1 in this process. Alternate forms of lymphocyte programmed death. The contraction of a lymphocyte population following the clearance of an infectious agent occurs via apoptotic death defined as the induction of DNA fragmentation, nuclear blebbing, and cell shrinkage. In many cases, infectious agents have evolved mechanisms to prevent apoptotic death, and in particular, many viruses prevent apoptosis, and this appears to promote virulence. A second pathway of cell occurs when apoptosis is inhibited, and we are studying the basis of this mode of programmed cell death (Chen et al., 2008). \n",
            "-------------------- Scraping faculty url 67/183 --------------------\n",
            "donald helinski  Biography Don Helinski earned his Ph.D at Case Western University and held a postdoctoral fellowship at Stanford University. Before joining the faculty at UCSD, he was on the faculty at Princeton University. He is a member of the National Academy of Sciences and the American Academy of Arts and Sciences. He is also a Fellow of the American Association for the Advancement of Science and an elected member of the European Molecular Biology Organization and the American Academy of Microbiology. \n",
            "-------------------- Scraping faculty url 68/183 --------------------\n",
            "martin hetzer \n",
            "-------------------- Scraping faculty url 69/183 --------------------\n",
            "david holway  Biography David Holway received a B.A. in zoology from UC Berkeley and a Ph.D in biology from the University of Utah. He conducted postdoctoral research at UC San Diego.  Research Research in our lab is currently divided into two areas: Biological invasions: The introduction of species into new environments has increasingly become an economically costly and environmentally disruptive phenomenon. Our research on invasions focuses on social insects (ants, bees and wasps) and primarily encompasses the following questions. (i) What factors control susceptibility to invasion? (ii) What accounts for how invasion impacts change in magnitude over time? (iii) Why do native species assemblages differ in their ability to recover following experimental invader removal? Pollination services: Declining pollinator populations threaten to compromise the integrity of pollination services in both agricultural and non-managed systems. Our research addresses this general issue from the following perspectives. (i) To what degree does land-use intensification affect pollinator assemblages? (ii) To what extent do pollinator declines compromise pollination services? (iii) How do non-native species interact with native pollinators, and when do these interactions negatively impact pollination services? \n",
            "-------------------- Scraping faculty url 70/183 --------------------\n",
            "alisa huffaker  Biography Dr. Huffaker received her Ph.D. from the Institute of Biological Chemistry at Washington State University. She continued at the Institute of Biological Chemistry for her postdoctoral training in plant defense signaling. Dr. Huffaker joined the section of Cell and Developmental Biology in 2014.  Research Like animals, plants continually interact with a plethora of other organisms in their environment, including potentially pathogenic microbes and herbivorous insects that can cause disease or damage. To minimize the detrimental impact of invading organisms, plants must mount a rapid and robust immune response. My research interests center on plant peptide signals, Peps, which regulate broad spectrum plant defense responses against pathogens and herbivores, and their application to manipulate plant resistance to biotic attack. Multigene families encode Plant Elicitor Peptides (Peps) and their receptors (PEPRs), which act together to coordinate plant immunity. Plant recognition of invading organisms occurs through perception of foreign molecules associated with attacking microbes and herbivores, known as elicitors, MAMPs (microbe-associated molecular patterns) or HAMPs (herbivore-associated molecular patterns). Perception of these molecules by pattern recognition receptors (PRRs) leads to subsequent launching of complex and multilayered defenses. These defenses, also known as basal or innate immune responses, have damaging effects on a wide array of pathogens and pests and involve accumulation of antimicrobial proteins and toxic metabolites. In addition to perceiving molecules from foreign organisms to detect biotic attack, plants also produce and recognize endogenous elicitors to assist in recognition and amplification of the immune response. Arabidopsis Plant Elicitor Peptide1 (AtPep1) was the first endogenous peptide signal found to regulate plant anti-microbial defenses. AtPep1 is derived from a larger precursor encoded by the Arabidopsis gene designated AtPROPEP1 , which is expressed in response to treatment with microbe-derived elicitors. AtPROPEP1 is a member of an eight gene family in Arabidopsis thaliana , each of which encode peptides that interact with the Plant Elicitor Peptide Receptors (PEPRs) to regulate plant immune responses. Treatment of Arabidopsis plants with AtPep1 induces production of defensive phytohormones and expression of pathogen defense genes. Transgenic over-expression of AtPROPEP1 causes constitutive expression of defense genes and enhances resistance to root degradation by the oomycete Pythium irregulare . Similarly, pretreatment of Arabidopsis plants with AtPeps prior to infection with Pseudomonas syringae Pv. tomato DC3000 restricts bacterial growth and decreases symptoms of disease. Peps coordinate and augment pathogen defense responses both spatially and temporally through amplification of MAMP-induced signaling. AtPeps are active only in Arabidopsis and closely related Brassicaceous species, but gene orthologs have been identified in more than 50 plant species. These Pep orthologs also regulate innate immunity. Treatment of maize plants with the ortholog ZmPep1 prior to infection with fungal pathogens caused accumulation of defense transcripts and metabolites and reduced symptoms and cell death caused by both Cochliobolus heterostrophus (Southern leaf blight) and Colletotrichum heterostrophus (Anthracnose stalk rot). Peps also mediate anti-herbivore defenses. Upon attack by herbivores, plants emit a complex blend of chemicals composed predominately of terpenes and six carbon (C6) green leafy volatiles (GLVs). These volatiles deployed by the plant recruit other organisms from the community to defend it from damage. Beneficial insects such as wasp species are attracted to the volatile blend and either eat or lay eggs in the attacking herbivorous pests. A maize Plant Elicitor Peptide, ZmPep3, is a potent signal promoting emission of herbivore associated volatiles and treatment of maize leaves with ZmPep3 results in accumulation of transcripts encoding biosynthetic enzymes for volatiles and other defense metabolites. These ZmPep3-induced responses are effective as direct and indirect defenses against herbivores, limiting growth of Spodoptera exigua larvae and attracting nave Cotesia marginiventris , a parasitoid of Lepidopteran larvae. Similar regulation of anti-herbivore defense by Peps occurs in numerous and diverse plant species. Some Peps promote plant defense against herbivores. Because PROPEP genes and signaling by Peps are ubiquitous throughout angiosperm plants, knowledge gleaned from Pep research in maize and Arabidopsis can be extrapolated and applied to other plants. Peps act as regulatory nodes for defenses against pathogens and herbivores, and are valuable candidate tools for genetic improvement of crop resistance. Study of Peps have generated a number of compelling questions for future research. How is signal specificity achieved? What signaling components are involved? How can this endogenous regulation of the plant immune system be improved to protect plants? As individual Peps appear to have differing functions, were studying the overlap of signaling and defense strategies mediated by individual Peps and by exogenous elicitors from other organisms. We hope to decode Pep-induced response specificity and to identify molecular components that can be successfully used to enhance resistance against a variety of attackers, including biotrophic and nectrotrophic pathogens as well as specialist and generalist herbivores. Current lab efforts are focused on 1. Study of Pep-PEPR ligand-receptor interactions and determination of structural requirements for signaling activity. 2. Characterization of downstream Pep signaling components identified through genetic screens and profiling of rapid changes in transcriptome and phosphoproteome of maize and Arabidopsis. 3. Development of Pep-based strategies for enhancement of plant immunity against pathogens and pests. \n",
            "-------------------- Scraping faculty url 71/183 --------------------\n",
            "enfu hui  Biography Enfu Hui obtained his Ph.D. in 2009 from UW Madison. He then completed his postdoctoral training at UCSF as a fellow of the Leukemia & Lymphoma Society. He joined the Division of Biological Sciences at UCSD in July 2016 and was named as a Searle Scholar and Pew Scholar.  Research Ideally, the immune system identifies tumors as threatening elements and deploys immune cells (T cells) to find and kill them. However, many tumor cells have evolved to employ a protein called PD-L1 to blind T cells from carrying out their functions and evade immune defenses. PD-L1 protects tumor cells by activating a \"molecular brake\" or checkpoints known as PD-1 to stop T cells. Remarkably, drugs that block PD-1 or its ligand PD-L1 have proven to release the PD-1 brake from T cells, and demonstrated unprecedented clinical activities in a variety of human cancers. Still, durable response is limited to a small subset of cancer patients. Identification of reliable predictive biomarkers, and targeting other immune checkpoints, either alone or in combination with PD-1 inhibitors, hold the promise of extending the therapy to more tumor types and a larger populations of patients. However, these efforts are slowed down by the incomplete molecular understanding of immune checkpoints. We are a group of biochemists and cell biologists aiming to dissect the mechanisms of immune checkpoints, using cell-free membrane reconstitution, time-resolved live cell microscopy and cutting-edge cell biology approaches. We recently uncovered two novel aspects of PD-L1/PD-1 signaling. Intracellularly, we showed that the T cell costimulatory receptor CD28 is a primary target of PD-1 associated phosphatases. Extracellularly, we discovered that PD-L1, a key weapon of tumor cells, can be neutralized in cis by PD-1 expressed on the same cells. Now, we will continue to elucidate the PD-1 signaling pathway by identification of novel regulators and targets of PD-1. In addition, we are investigating whether and how additional immune brakes operate in conjunction with PD-1 to turn off T cell activity. We will identify the triggering molecules that turn these brakes on, and signal transducers these brakes recruit to suppress the immune response. Our findings could lead to the development of novel biomarkers and drug targets of cancer immunotherapy. \n",
            "-------------------- Scraping faculty url 72/183 --------------------\n",
            "tony hunter \n",
            "-------------------- Scraping faculty url 73/183 --------------------\n",
            "terry hwa  Biography Dr. Terence Hwa is the Presidential Chair Professor with a joint appointment in the Department of Physics at U.C. San Diego. He is the inaugural Director of the Ph.D. Specialization in Quantitative Biology at U.C. San Diego.  Research Current activities in our lab are in the area of quantitative and systemic biology. This is an emerging area of research at the interface of biology, biochemistry, engineering, and physics. In this post-genome era, it is clear that the complexity of a biological organism resides not merely in the intricacies of its components (e.g., proteins), but more importantly in the array of interactions these components can have with each other. This shift of paradigm is initiating a revolution in biology: Instead of focusing on genes, proteins, and pathways, bioloigsts are beginning to think in terms of modules and networks. However, a significant challenge lies in the fact that much of the interactions of the subsystems are yet unknown. Our research is focused on microbiology, particularly on the bacterium E. coli which is the best characterized living organism both molecularly and physiologically. In the long run, we wish to develop a comprehensive understanding of the organism, i.e., understanding its physiological response under various growth conditions in molecular terms. However even for E. coli, a global approach seems premature in our opinion due to the many missing molecular players and interactions. Thus, we work on smaller sub-systems, with the insistence on developing quantitative links from molecules all the way to cell physiology. Our emphasis can be regarded as the \"vertical approach\", which complements the \"horizontal approach\" favored by the 'omics technology. An introduction to our vertical, quantitative approach is described in a recent course on quantitative molecular biology developed by our laboratory. Vertical integration requires a command of the vast knowledge accumulated over many decades of research in molecular and microbiology, as well as a combination of experimental and theoretical efforts. Some of the experiments are being carried out in our own lab; others are done together with biology collaborators. Our experiments are often theory-motivated, yet discoveries made in the lab has inspired fresh theoretical view points. In similar ways, our theoretical studies play the dual role of analyzing experimetal results and guiding new generation of experiments. The close interaction between theory and experiment allows the theorists to be in contact with biological reality, and allows the experimentalists to have a good sense of the power (and limitations) of quantitative analysis. In this way, we hope to train a new generation of scientists who can freely exploit opportunities at the interface between biology and physics. Below are some topics our lab has been working in recent years; all projects involve experimental and theoretical components. gene regulation: quantitative characterization of combinatorial transcriptional control; de novo synthesis of novel transcriptional control; quantitative characterization of post-transriptional control by regulatory small RNA; protein-protein interaction: specificity and cross-talk in two-component signaling; assembly of macromolecular complexes; molecular evolution: directed evolution of promoters, cis-regulatory sequences, and signaling molecules; analysis of evolved seqeunces to deduce rules of molecular interaction; genetic circuits: quantitative characterization of various endogenous genetic circuits; synthesis and characterization of artificial circuits; metabolic control: interaction and coordination of metabolic pathways; systems being studied include carbon/nitrogen metabolism, aerobic/anarobic control, Fe/oxidative stress; physiology & cell growth:coordination of growth and metabolic activities; control of ribosome synthesis and cell growth; control of cell division. \n",
            "-------------------- Scraping faculty url 74/183 --------------------\n",
            "juan carlos izpisua belmonte \n",
            "-------------------- Scraping faculty url 75/183 --------------------\n",
            "xin jin \n",
            "-------------------- Scraping faculty url 76/183 --------------------\n",
            "yishi jin  Biography Dr. Jin received her B.S. degree from Peking University, China, and her Ph.D. from the University of California, Berkeley. She completed her postdoctoral training at MIT.  Research The Jin lab research focuses on the molecular genetic mechanisms underlying the development and function of the nervous system using the nematode Caenorhabditis elegans . The transparency, defined anatomy, and rapid life cycle of this organism greatly facilitate our studies at the subcellular resolution. Moreover, the entre cell lineage and connectome are known, enabling functional understanding at deep levels. Through forward genetic screening in combination with multi-layered molecular and cellular manipulations, we are discovering key molecules that play conserved roles in synapse formation, maintenance, and function, as well as those underlying adult axon regeneration. Our ultimate goal is to connect the studies of basic mechanisms to the understanding of human neurological disorders and neuronal repair. How do neurons form synapses with their partners? Synapses are unique structures that neurons use to communicate with others. At the presynaptic terminal, neurons develop elaborate subcellular organizations to facilitate the accumulation of synaptic vesicles and their release (Kittelmann et al., 2013). We label different components of presynaptic terminals in vivo using fluorescent markers. Through forward genetic screening, we have identified multiple signaling pathways that involve conserved scaffolding proteins, MAP kinases, GTPases, and ubiquitin-mediated protein degradation to specify spatial domains of the presynaptic terminals (Nakata et al., Cell 2005; Grill et al., Neuron, 2007; Yan et al., Cell 2009; Kittelmann et al ., 2013). Synapses also undergo remodeling as animals grow. Our recent studies have elucidated roles of dynamic microtubule cytoskeleton in synaptic remodeling (Kurup et al., Current Biology, 2015). Furthermore, combining genomics with genetics, we have discovered unexpected pathways linking synapse development to nuclear pre-mRNA regulation (Van Epps et al., Development, 2010; Chen et al., Genes and Development, 2015). How do neural circuits operate to control movement? C. elegans movement, namely locomotion, depends on balanced excitation and inhibition to body muscles. The locomotion circuit consists excitatory cholinergic and inhibitory GABAergic motor neurons. We have examined the roles of neuronal ACh receptors in modulating the excitation state of these two types of neurons (Jospin et al., PLoSBiol, 2009). Our analysis of genetic mutations also led us to establish a novel model for epilepsy (Stawicki et al., Current Biology, 2011; Stawicki et al., PLoSGenetics, 2013; Zhou et al., eLife, 2013). In our recent studies, we have discovered novel inter-tissue interactions in the regulation of neuronal circuits, and identified an immunoglobulin protein that plays instructive roles to control synapse numbers through phagocytosis (Cherra et al., Neuron, 2016). How do adult axons regenerate? Understanding how neural circuit forms during normal development has direct implications to the process of how injured neurons repair and regain function in adult animals. Using an ultrafast laser-based microsurgery procedure to perform axotomy in C. elegans neurons (Yanik et al., Nature 2004), we have identified numerous pathways regulating the rate and accuracy of adult axon regeneration (Chen et al., Neuron, 2011). Our current studies focus on the roles of the conserved MAPKKK DLK-1 and the microtubule regulator EFA-6 in axon regeneration (Yan et al., Cell 2009; Yan and Jin, Neuron, 2012; Chen et al., Neuron, 2011; Chen et al.,, eLife, 2015). By advancing our knowledge of intrinsic signaling pathways, we hope to develop new strategies to enhance regrowth ability of injured axons. Novel applications of optogenetic methodology: Optogenetic manipulation of cells and molecules using genetically encoded tags is revolutionizing neurobiology. Mini Singlet Oxygen Generator (miniSOG) is a genetically encoded photosensitizer, created by R. Tsien's lab (Shu et al., PLoS Biology, 2011). We have employed the ability of inducible ROS generation by miniSOG to develop innovative optogenetic tools to perturb neural circuit (Qi et al., PNAS 2012) and to perform genome wide random mutagenesis (Noma and Jin, Nat. Comm. 2015). \n",
            "-------------------- Scraping faculty url 77/183 --------------------\n",
            "katherine jones \n",
            "-------------------- Scraping faculty url 78/183 --------------------\n",
            "ashley juavinett  Biography Ashley Juavinett received her Ph.D. from UC San Diego and was a Postdoctoral Fellow at Cold Spring Harbor Laboratory. She returned to UC San Diego as an Assistant Teaching Professor in 2018.  Research My research and writing focuses broadly on education and careers in the field of neuroscience. Im interested in capitalizing on open access datasets as a course-based undergraduate research experience. We are developing scaffolds for such datasets that will enable students to ask their own questions of cutting edge data. I am interested in whether these approaches will create realistic, engaging laboratory experiences and ultimately invite diverse students into the field of neuroscience. \n",
            "-------------------- Scraping faculty url 79/183 --------------------\n",
            "suckjoon jun  Biography Suckjoon Jun received his Ph.D. in theoretical physics from Simon Fraser University in Vancouver, Canada. He continued to work as a theorist for his post-doctoral training at the FOM-Institute AMOLF in Amsterdam and briefly at Necker Hospital in Paris with NSERC (Canada) and Marie Curie (Europe) post-doctoral fellowships. He was a Bauer Fellow at Harvard University between 2007-2012 before moving to UCSD, during which he completely switched to experimental physical and systems biology.  Research The driving force of our lab is the curiosity and desire to understand why the biological systems are the way they are. We are often attracted to long-standing questions in biology, for which quantitative, inter- and multi-disciplinary approaches are essential. Some of these questions include: What is the driving force underlying chromosome segregation in bacteria? What is the relationship between growth and cell cycle? What is the origin of the plasticity of cell shape? While our research is naturally question-driven, we also develop tools when necessary. In doing so, we try to meet the highest standard in the relevant discipline (be it physics, biology, or engineering). We value both logic and intuition, but more of \"night science\" than \"day science.\" \n",
            "-------------------- Scraping faculty url 80/183 --------------------\n",
            "james kadonaga  Biography Jim Kadonaga was an undergraduate in Chemistry at MIT, where he received the Alpha Chi Sigma Prize as well as the American Institute of Chemists Certificate. He carried out his graduate studies with Jeremy R. Knowles in the Department of Chemistry at Harvard University, where he was a DuPont Fellow. Jim was a postdoctoral associate with Robert Tjian at UC Berkeley as a Fellow of the Miller Institute for Basic Research in Science, American Cancer Society California Division, and Lucille P. Markey Charitable Trust. He joined the UCSD faculty in 1988 and was one of 15 scientists to be named as a Presidential Faculty Fellow by President George H.W. Bush in 1992. In 1994, he was elected to Fellow of the American Association for the Advancement of Science, and in 1995, he was elected to Fellow of the American Academy of Microbiology. He served as the Chair of the Section of Molecular Biology from 2003 to 2007. In 2012, he received the UCSD Chancellor's Associates Award for Excellence in Research in Science and Engineering. In 2017, he was elected to the American Academy of Arts & Sciences. Jim is presently the Amylin Endowed Chair and a Distinguished Professor in the Section of Molecular Biology. Profile in J. Cell Biol.  Research Transcriptional Regulation and Chromatin Dynamics A diverse range of biological phenomena, including many human diseases, can be traced to the proper or improper expression of a gene. New insights into the mechanisms of gene expression will have a broad impact upon science and society. The objective of our work is to understand the regulation of RNA polymerase II transcription. We study the transcription process as well as the chromatin template to decipher the mechanisms by which genes are regulated in animals. Development of Sequence-specific DNA Affinity Chromatography & the Purification and Cloning of Transcription Factor Sp1 Prior to my arrival at UCSD, I was involved in early studies of eukaryotic sequence-specific DNA-binding transcriptional factors as a postdoctoral researcher in the laboratory of Robert Tjian at UC Berkeley. In this work, I developed sequence-specific DNA affinity chromatography for the purification of transcription factor Sp1 (Fig. 1). The purification of Sp1 enabled me to isolate its corresponding cDNA and to characterize its binding to DNA via three zinc finger motifs (Kadonaga et al., 1987). Sequence-specific DNA affinity chromatography has since been used by many investigators to purify numerous factors that function in a wide range of biological processes. The remainder of the work below was carried out in my lab at UCSD. The RNA Polymerase II Core Promoter In the area of transcription, we have focused on the RNA polymerase II core promoter. The core promoter is the stretch of DNA that directs transcription initiation, and it typically comprises sequences from about -40 to +40 nt relative to the +1 transcription start site. The core promoter is the gateway to transcription, as it is the site at which the signals that activate transcription ultimately converge. Core promoters are diverse in terms of their structure and function. The transcriptional activity of core promoters is driven by DNA sequence motifs such as the TATA box, initiator (Inr), TCT motif, downstream core promoter element (DPE), motif ten element (MTE), and others (Fig. 2). Any particular core promoter will have one or more core promoter motifs. There are, however, no universal core promoter elements. In addition, there are core promoter motifs that have yet to be discovered. We study core promoters at three levels: DNA sequence motifs, transcription factors, and biological networks. (1). DNA sequence motifs. We have identified and characterized new core promoter elements, such as the DPE and MTE, and articulated the nature and properties of the TCT motif. We are continuing to search for new core promoter elements. We also seek to gain a better understanding of the known elements. (2). Transcription factors. In Drosophila , we have found that TATA-less transcription with TCT-dependent core promoters does not require TBP (TATA box-binding protein), but is instead dependent upon TRF2 (TBP-related factor 2). We also discovered that Caudal protein (a DNA-binding transcription factor that is a key regulator of the Hox genes) is a DPE-specific activator. This finding is a specific example of the concept that some transcriptional enhancers are specific for core promoters with DPE or TATA motifs (Fig. 3). More generally, we seek to understand the different sets of transcription factors that function at different types of core promoters. (3). Biological networks. We found that some core promoter motifs have specific functions in biological networks the DPE with the Hox gene network and the TCT motif with the ribosomal protein gene network (Fig. 4). Notably, in Drosophila, both the DPE and TCT motifs function with TRF2 rather than TBP. Thus, these findings have also led us to investigate the evolution of TRF2 (Fig. 5). It is important to understand the function of the core promoter in a broader biological context. Our current studies include the identification and analysis of new core promoter motifs and structures, particularly in humans. We are also investigating the distinct mechanisms of transcription from different types of core promoters as well as the basis of enhancer-core promoter specificity. The core promoter continues to be a rich source of new discoveries. Early Studies of the Role of Chromatin in the Regulation of Transcription We carried out early experiments on the role of chromatin structure in the regulation of transcription by RNA polymerase II. These studies were initiated in the late 1980s this was a time when many people in the transcription field felt that chromatin was unimportant. We were primarily motivated to perform this work because chromatin is the natural state of DNA in the eukaryotic nucleus. We found that sequence-specific DNA-binding transcription factors act primarily to counteract chromatin-mediated repression we termed this phenomenon \"antirepression\" (Laybourn and Kadonaga, 1991, 1992; Paranjape et al., 1994). This paradigm was in contrast to the prevailing belief that sequence-specific factors increase the rate of the transcription reaction in a manner that is independent of chromatin structure. These studies of chromatin and transcription led to the model for transcriptional activation that is shown in Fig. 6 (Pazin et al., 1994). In this project, we were also able to achieve ligand-regulated transcription in vitro by the estrogen receptor by using chromatin templates (Kraus and Kadonaga, 1998). Today, it is generally accepted that sequence-specific activators counteract chromatin-mediated repression of basal (unactivated) transcription, and chromatin is extensively studied as Chromatin Assembly and Dynamics Chromatin assembly is important for processes, such as DNA replication, transcription, and repair, that involve the disruption and reformation of nucleosomes. Our studies of chromatin assembly began with the development of a crude extract from Drosophila embryos, termed the S-190, that is competent for ATP-dependent chromatin assembly. We then undertook the biochemical fractionation and purification of the essential components in the S-190 extract. This work included the discovery, purification, and cloning of ACF (ATP-utilizing chromatin assembly and remodeling factor; Ito et al., 1997, 1999) and the discovery that Asf1 is a chromatin assembly factor that serves as a chaperone for newly synthesized histones H3 and H4 (Tyler et al., 1999). The analysis of the mechanism of the ATP-dependent assembly of periodic nucleosome arrays led to the model shown in Fig. 7. With purified and defined factors, we have been studying the mechanism of chromatin assembly. This work led to the discovery of the prenucleosome, which is a stable conformational isomer of the nucleosome that associates with approximately 80 bp DNA (Fig. 8). Prenucleosomes are formed rapidly upon the deposition of histones onto DNA, and can be converted into canonical nucleosomes by an ATP-driven chromatin assembly factor such as ACF. Moreover, p300 acetylates histone H3K56 in prenucleosomes but not in nucleosomes. Multiple independent lines of evidence suggest that prenucleosomes or prenucleosome-like particles are at active chromatin such as promoters (Fig. 9). First, there is a strong correlation between psoralen-crosslinked species observed with prenucleosomes assembled in vitro and the active yeast PHO5 promoter in vivo. Second, chromatin particles with prenucleosome-sized DNA and core histones are observed at the upstream region of active promoters in mouse cells. Third, prenucleosomes, but not nucleosomes, can be acetylated at H3K56 by p300, and H3K56 acetylation occurs at active promoters and enhancers in yeast, Drosophila , and humans. It is possible that the transcriptional machinery is optimized to function in conjunction with a prenucleosome. More generally, prenucleosomes or prenucleosome-like particles are observed at active chromatin. In other words, prenucleosomes are generated in dynamic chromatin at which there is active disassembly and reassembly of nucleosomes. Annealing Helicases Our studies of chromatin dynamics took an unexpected turn in the course our studies of the HARP protein (also known as SMARCAL1). We were interested in HARP because it is in the same family of ATPases (the SNF2-like family) as the chromatin assembly proteins ISWI (the ATPase subunit of ACF) and Chd1. In addition, mutations in HARP were found to be responsible for a rare human disorder known as Schimke immuno-osseous dysplasia (SIOD). We thus embarked on the analysis of HARP with the notion that it might alter chromatin structure; however, HARP did not exhibit chromatin remodeling activity. Instead, we discovered that HARP possesses a novel annealing (reverse) helicase activity. In other words, HARP is a motorized molecular zipper that rewinds DNA, such as single DNA strands that are bound by RPA, the major single-stranded DNA-binding protein in the nucleus (Fig. 10). In addition, we found a second annealing helicase termed annealing helicase 2 (AH2, also known as ZRanB3). Unlike HARP, which forms a stable complex with RPA, AH2 does not bind to RPA. More generally, in these studies, we have identified a DNA structure-modifying activity that acts in opposition to the DNA-unwinding activities of helicases and polymerases. NDF, a Nucleosome Destabilizing Factor that Facilitates Transcription through Nucleosomes Our studies of prenucleosomes led us to examine factors that mediate the destabilization or disassembly of conventional nucleosomes. To this end, we devised a nucleosome destabilization assay that was based on the ability of p300 to acetylate histone H3K56 in a partially unwrapped nucleosome. By using this assay, we identified a nucleosome destabilizing activity in Drosophila embryo extracts, and then purified this activity by conventional chromatography. This work led to the identification of a factor that we term NDF, for nucleosome destabilizing factor. NDF is present in most animals, and perhaps in plants, but it does not appear to have a homolog in Saccharomyces cerevisiae . NDF has a conserved PWWP motif, interacts with nucleosomes near the dyad, destabilizes nucleosomes in an ATP-independent manner, and facilitates transcription by RNA polymerase II through nucleosomes in a purified and defined transcription system as well as in cell nuclei (Fig. 11). Upon transcriptional induction, NDF is recruited to the transcribed regions of thousands of genes and co-localizes with a subset of H3K36me3-enriched regions (Fig. 12). Notably, the recruitment of NDF to gene bodies is accompanied by an increase in the transcript levels of many of the NDF-enriched genes. In addition, the global loss of NDF results in a decrease in the RNA levels of many genes. In humans, NDF is present at high levels in all tested tissue types, is essential in stem cells, and is frequently overexpressed in breast cancer. These findings indicate that NDF is a nucleosome destabilizing factor that is recruited to gene bodies during transcriptional activation and facilitates RNA polymerase II transcription through nucleosomes. The properties of NDF indicate that it has a key role in the transcription of chromatin by RNA polymerase II in animals. The further analysis of NDF should reveal additional factors and mechanisms that orchestrate the procession of RNA polymerase II through active genes. \n",
            "-------------------- Scraping faculty url 81/183 --------------------\n",
            "susan kaech \n",
            "-------------------- Scraping faculty url 82/183 --------------------\n",
            "jan karlseder \n",
            "-------------------- Scraping faculty url 83/183 --------------------\n",
            "amy kiger  Biography Amy Kiger received her Ph.D. as a Howard Hughes Medical Institute Predoctoral Fellow from the Department of Developmental Biology at Stanford School of Medicine. She then completed postdoctoral studies in the Department of Genetics at Harvard Medical School as a Fellow of the Jane Coffin Childs Memorial Fund for Medical Research.As Principal Investigator in her own lab, Amy Kiger has been awarded fellowships from The David & Lucille Packard Foundation, The Sidney Kimmel Foundation for Cancer Research, The March of Dimes and The Hellman Family Foundation, as well as grants from the agencies Human Frontier Science Program and National Institutes of Health.  Research Cellular remodeling Investigating membrane regulation of dynamic cell organization & shape Cells must continuously maintain integrity and compartmentalization with changes in cell size and shape that occur in development, immunity, aging and disease. How cellular spatiotemporal organization is achieved with remodeling is therefore a fundamental problem of broad significance. Our laboratory is studying central roles for novel membrane trafficking processes, under control of spatiotemporal lipid regulation, that coordinate dynamic cell structure. Using a foundation in genetic and cell biology experiments in the fruit fly, Drosophila, we are linking specific lipid kinase and phosphatase activities to cell remodeling. Our work aims to identify conserved mechanisms of membrane regulation in immune and muscle cell functions, and provide insights into related human diseases. Our research is currently focused in three major areas: roles for phosphoinositide lipid regulation in cellular remodeling roles for autophagy in cellular remodeling, and systems-biology approaches to understanding cellular remodeling To address these problems, we are investigating questions specific to the dynamic organization of two distinct cell types, immune cells and muscle cells. The different elaborated morphologies and functions of immune cells and muscle cells are helping us to illuminate both broadly conserved and tissue-specific requirements in cellular remodeling. Regulation of cellular remodeling by phosphoinositide-directed endocytic recycling Phosphoinositide lipids, the seven different forms of phosphorylated phosphatidylinositol, provide a membrane code that controls a wide array of cellular functions. However, gaps remain in understanding the complex functional relationships between the relevant kinases, phosphatases and the specific roles mediated through phosphoinositide regulation. The large family of Myotubularin (MTM) phosphoinositide phosphatases (15 human, 7 fly, 1 yeast) are selective for PI(3)P and PI(3,5)P 2 turnover, pointing to possible roles in complex regulation of endocytosis and/or autophagy. Further confounding the understanding of MTM family functions are that, curiously, half of the members encode for catalytically inactive, or pseudo-phosphatases. Importantly, mutations in MTM catalytic and pseudophosphatase members are both associated with human disease, including centronuclear myopathy (XLMTM) and Charcot-Marie-Tooth neuropathy (CMT4B). How potential MTM roles in endocytosis or autophagy relate to disease are not understood. We identified mtm , a Drosophila MTM1/MTMR2 phosphoinositide phosphatase, as necessary and sufficient for formation of dynamic immune cell protrusions and immune cell recruitment to wounds. We determined that Mtm and Class II PI3-kinase, called Pi3K68D in flies, co-regulates a PI(3)P subpool on endosomes for endocytic recycling important for immune cell dynamic cortical functions. We have also demonstrated a central role for a Pi3K68D/Mtm pathway for phosphoinositide regulation of integrin endocytic trafficking crucial for maintenance of muscle attachments during muscle cell remodeling. We showed that mtm mutant myofibers exhibit hallmarks of the human XLMTM myopathy, and that the fly defects predicted disruption of integrin localization in human XLMTM. The identification of functionally distinct PI(3)P subpools under control of Class II PI3-kinase has broad implications, including therapeutic strategies for MTM-related human disease. Additional implications include possible roles for an Mtm pathway to regulate integrin trafficking in cell migration and metastasis. In ongoing studies , we are addressing how phosphoinositide-directed endocytic recycling provides key input into the dynamic balance between distinct, cortical F-actin structures in immune cells. In muscle, we are exploring the regulation of integrin trafficking during cellular remodeling, as well as the relationships between mtm and orthologs of other genes also mutated in different forms of human centronucelar myopathy. We are pursuing how PI3KC2 and Mtm are spatiotemporally regulated. Important insights are coming from investigation of the MTM pseudo-phosphatase, Sbf, as a key coordinator of both PI3KC2/Mtm phosphoinositide regulation and membrane trafficking. Regulation of cellular remodeling by autophagy Autophagy is the membrane-mediated process of cellular self-eating through encapsulation of proteins or organelles for lysosomal delivery and degradation, with well-known roles in stress responses. Autophagy has been the focus of growing interest due to its involvement in many processes, including immunity, neurodegeneration and cancer. We showed that autophagy is continuously required for macrophage cell spreading, with functions in proper cytoskeletal reorganization, and has a conserved role in the cortical remodeling of both Drosophila and mouse macrophages. Consequently, macrophages disrupted for autophagy are impaired in their recruitment to epidermal wounds. We showed a requirement for the p62 multiadaptor involved in selective targeting of ubiquitinated proteins for autophagic degradation, pointing to a role for selective autophagy (versus bulk autophagy) as a regulatory mechanism for cortical remodeling. This is the first demonstration of a true cell remodeling function at the level of cell shape for autophagy, which has been previously proposed but never shown. In ongoing studies , we are continuing to address how autophagy specifically regulates immune cell remodeling, in part through the use of systems biology approaches to identify targets of selective autophagy. We are also investigating phosphoinositide regulation of autophagy. Functional genomics of cellular remodeling The regulation of cell morphology can be viewed as a complex system that must integrate the temporal-spatial control of multiple cellular processes, including cytoskeletal and membrane dynamics, cell adhesion and growth. To understand morphogenetic programs, we are applying functional genomics to comprehensively probe the cellular-genetic networks that control cellular remodeling. We pioneered RNA-interference (RNAi) screens in Drosophila cells that permit unprecedented genome-wide loss-of-function analysis of metazoan cell morphology. Our RNAi microscopy screens have identified gene functions required to maintain distinct round or flat immune cell morphologies. Now, we are using RNAi screens to study how the addition of a hormonal cue culminates in the change of cells from a round to elongated cell shape, relevant to numerous cellular behaviors in vivo. Furthermore, co-RNAi screens targeting combinations of multiple genes are an ideal method for identifying new components of a pathway through suppressor or enhancer effects. In this way, we have isolated and continue to seek modifiers of specific phosphoinositide and autophagy pathways important in cellular remodeling. \n",
            "-------------------- Scraping faculty url 84/183 --------------------\n",
            "christopher kintner \n",
            "-------------------- Scraping faculty url 85/183 --------------------\n",
            "david kleinfeld  Biography David Kleinfeld received his Ph.D. from UCSD and was a Member of Technical Staff at the former AT&T Bell Laboratories prior to joining the faculty at UCSD. He is a recipient of a David and Lucile Packard Foundation Interdisciplinary Science Award, a NIH Directors Pioneer Award, a NINDS Research Program Award, and currently holds the Experimental Biophysics Endowed Chair through Physics. David is a fellow of the American Association for the Advancement of Science and a member of the American Academy of Arts and Sciences. He currently leads the Specialization on Computational Neuroscience through the Neurosciences Graduate Program.  Research Our laboratory is focused on two problems in neurobiology: (1) Active sensing, where we are delineating the brainstem circuitry that coordinates orofacial motor actions, e.g., sniffing, licking, head bobbing, and whisking, into behaviors. This work involves anatomy, behavior, and electro- and opto-physiology with rodents ad strives to make connections with control theory. (2) Blood flow in the brain, for which we connect measurements of the topology of the vasculature with neuronal control of flow dynamics from the level of large-scale vascular networks down to single microvessels. This work involves anatomy, physiology, and deep-brain optical imaging with rodents and strives to make connections with graph theory and fluid dynamics. Our efforts in both areas involve a broad range of approaches together with the opportunity to develop new tools. Please see our web site for further information concerning our work and recent publications. \n",
            "-------------------- Scraping faculty url 86/183 --------------------\n",
            "joshua kohn  Biography Joshua R. Kohn received his Ph.D. from the University of Pennsylvania and was a postdoctoral fellow at both the University of Toronto and Cornell University.  Research We use both molecular and experimental tools to study plant mating systems and floral evolution. The mating system can vary from obligate outcrossing to complete selfing. The mating system is a prime factor in explaining differences in levels of genetic variation among populations and species. Limited variation, as is often found in selfing populations, may strongly reduce the evolutionary potential of selfers. Current work focuses on three issues: 1) The molecular evolution of the self-incompatibility locus, 2) The evolution of mating system diversity in higher plants. 3) Floral variation, its molecular causes and selective consequences. Self-Incompatibility Many plants reject their own pollen to avoid the detrimental effects of inbreeding. Self-pollen recognition and rejection is often controlled by a single locus (S) where a match between genotype of pollen and style results in arrest of pollen tube germination or growth. Rare alleles at this locus have a selective advantage, being compatible with more potential mates. This frequency-dependent selection leads to some of the highest levels of polymorphism known for any locus with 30-50 alleles often segregating within single populations. Alleles at the S-locus are also very old because, if they drift towards low frequency, they become selectively favored to increase. At the molecular level, allelic lineages are often tens of millions of years old, older than the species and genera in which they currently reside. This is reflected in the fact that an allele found in one species is often more closely related to an allele found in another species than it is to other alleles from the same species. Because of this property, the S-locus provides a tool for historical inference that extends much deeper in time than neutral variation. Using RT-PCR to amplify S-alleles from stylar tissue, we can survey S-allele diversity within and between natural populations and simultaneously gain sequence information that can be used to study evolutionary processes above the species level. We have worked primarily on systems in which RNases are the stylar product of the S-locus, as is true in the families Solanaceae, Scrophulariaceae, and Rosaceae. Current work also includes assaying S-locus variation in the Papaveraceae, a family with an entirely different molecular mechanism of self-incompatibility. Fig. 1. Floral variation in the Papaveraceae, a family with gametophytic self-incompatibility. The evolution of plant mating systems Flowering plants exhibit far more breeding system diversity than do most animals. We use both experimental and phylogenetic approaches to test evolutionary hypotheses concerning mating system diversity. We are particularly interested in the evolution of separate sexes vs. hermaphroditism and self-fertilization vs. outcrossing. Dove-tailing with our work on incompatibility, we have used inferences from the 40 million year history of shared S-locus polymorphism to trace the history of mating system changes across entire plant families. These data have provided some of the strongest evidence yet that selfing is often an evolutionary dead end. Ongoing work is aimed at assessing whether incompatibility, a trait that is frequently lost but very rarely gained, provides a macroevolutionary advantage in terms of either increased speciation or reduced extinction. Floral variation The California flora is the richest in the continental US providing abundant systems for ecological and evolutionary studies. In San Diego County, students in the lab have done two Ph.D. theses on Mimulus aurantiacus, bush monkeyflower, a species with a sensitive stigma that closes rapidly after being touched by its hummingbird pollinator. Stigma closure was found to increase the hermaphroditic flowers male reproductive success as flowers with closed stigmas exported 2-3 times more pollen to the stigmas of subsequently-visited flowers than did flowers with open stigmas. This plant also has two color morphs locally. The red morph occurs close to the coast while the yellow morph occurs inland with a narrow hybrid zone in between. Experimental studies have shown that hummingbirds, though they visit yellow populations, strongly prefer the red-flowered form when given a choice and that hawkmoths only visit the yellow form. Flower color variation is caused by the presence of red, anthocyanin pigments in red flowers and their absence in yellow flowers, though both forms make red pigments in their stems. Progress is being made in understanding the molecular basis of color variation so that selection from the molecular to the phenotypic level can be better understood. The Kohn lab accepts students interested in any aspect of plant reproductive ecology. Fig. 2. Color morphs of Mimulus aurantiacus . \n",
            "-------------------- Scraping faculty url 87/183 --------------------\n",
            "takaki komiyama  Biography Dr. Komiyama was a Postdoc at Janelia Farm, Howard Hughes Medical Institute, received his Phd in Neurosciences at Stanford University (2006) and his BA in Biochemistry at University of Tokyo (2001). His Honors include: Helen Hay Whitney Foundation Postdoctoral Fellowship, Harold M. Weintraub Graduate Student Award, Japan-Stanford Association Graduate Fellowship, Stanford Graduate Fellowship, Pew Scholars Program, Packard Fellowship and Sloan Research Fellowship. Dr. Komiyama is a NYSCF-Robertson Investigator.  Research Animals constantly modify their behavior through experience. Flexible behavior is key to our ability to adapt to the ever-changing environment. My laboratory is interested in studying the activity of neuronal ensembles in behaving animals, and how it changes with learning. We have recently set up a paradigm where mice learn to associate sensory information (two different odors) to motor outputs (lick vs no-lick) under head-fixation. We combined this with two-photon calcium imaging, which can monitor the activity of a microcircuit of many tens of neurons simultaneously from a small area of the brain. Imaging the motor cortex during the learning of this task revealed neurons with diverse task-related response types. Intriguingly, different response types were spatially intermingled; even immediately adjacent neurons often had very different response types. As the mouse learned the task under the microscope, the activity coupling of neurons with similar response types specifically increased, even though they are intermingled with neurons with dissimilar response types. This suggests that intermingled subnetworks of functionally-related neurons form in a learning-related way, an observation that became possible with our cutting-edge technique combining imaging and behavior. We are working to extend this study. How plastic are neuronal microcircuits during other forms of learning? How plastic are they in other parts of the brain? What are the cellular and molecular mechanisms of the microcircuit plasticity? Are the observed activity and plasticity required for learning? How does the activity of identified individual neurons change over days to weeks? We are asking these questions, combining a variety of techniques including in vivo two-photon imaging, optogenetics, electrophysiology, genetics and behavior. Figure 1: In vivo two-photon calcium imaging from head-fixed mice during learning Figure 2: A snapshot of calcium imaging \n",
            "-------------------- Scraping faculty url 88/183 --------------------\n",
            "william kristan  Biography Bill Kristan received his Ph.D. from the University of Pennsylvania. He was a postdoctoral fellow at Stanford University and an assistant research biologist at UC Berkeley. He held a Guggenheim Fellowship and was a Fellow of the University of Bielefeld, Germany.  Research My lab studies how networks of nerve cells produce different behaviors and how these neuronal networks are established during embryogenesis. We use electrophysiological, imaging, anatomical, computational, and embryological techniques to characterize these circuits in the relatively tractable nervous system of the medicinal leech, and to construct computerized models of them. \n",
            "-------------------- Scraping faculty url 89/183 --------------------\n",
            "mitchell kronenberg \n",
            "-------------------- Scraping faculty url 90/183 --------------------\n",
            "sergey kryazhimskiy  Biography Sergey Kryazhimskiy received his Ph.D. from Princeton University in 2008. He did his postdoctoral work at the University of Pennsylvania and Harvard University where he was awarded the Burroughs Wellcome Fund Career Award at Scientific Interface. He will join the EBE faculty in January 2016.  Research We are interested in uncovering general principles of how mutations affect phenotypes at multiple levels of biological organization (gene expression, metabolic fluxes, etc.), and how these effects combine to determine organisms fitness across environments. We approach this problem using experiments in model microorganisms S. cerevisiae and E.coli and mathematical modeling. Our lab has three complementary themes. First, we empirically characterize how the effects of hundreds of mutations on fitness depend on the external environment and on the genetic background of the organism. In technical terms, we characterize the statistics of epistasis and pleiotropy. Second, we attempt to predict from first principles how mutations are expected to perturb cellular physiology and fitness. Third, we measure some of these mutational perturbations of cell physiology (e.g., changes in gene expression profiles) and test our theoretical predictions. \n",
            "-------------------- Scraping faculty url 91/183 --------------------\n",
            "carolyn kurle \n",
            "-------------------- Scraping faculty url 92/183 --------------------\n",
            "russell lande \n",
            "-------------------- Scraping faculty url 93/183 --------------------\n",
            "shannon lauberth  Biography Dr. Shannon Lauberth received her Ph.D. in Biochemistry and Molecular Biology from Saint Louis University. She performed her postdoctoral training at the Rockefeller University where she was a recipient of a Ruth L. Kirschstein National Research Service Award (NRSA) and an active member of the Anderson Cancer Center.  Research Mutations resulting in the inactivation of the tumor suppressor p53 are the most common lesions present in human malignancies, and thereby contribute significantly to the pathogenesis of human cancer. Importantly, the reactivation of wild-type p53 leads to efficient tumor regression indicating the clinical implications for understanding p53 function. In normal cells, the primary function of p53 is to respond to stress by activating multiple target genes that then execute specific pathways that include cell cycle arrest and programmed cell death. The activation kinetics of the various p53 target genes varies significantly, ranging from a rapid induction of cell cycle control genes to a delayed induction of pro-apoptotic genes. Establishing the proper activation kinetics and balance between the p53-response pathways is essential for preventing the propagation of cells that harbor DNA mutations and can lead to tumor development. A fundamental question that remains is how p53 determines which of the diverse transcription programs to activate in order to facilitate the appropriate stress response. Changes in gene expression that arise in the absence of alterations in the DNA sequence are referred to as epigenetic. A primary epigenetic mechanism that contributes to gene regulation includes the posttranslational modifications of histone proteins. Histone modifications function by facilitating direct effects on chromatin structure and through the recruitment of chromatin-associated factors, or effector proteins that translate histone modifications into specific biological outcomes. A wide array of histone modifications and effector proteins have been identified, and as a result, we are faced with the challenge of understanding the mechanisms by which histone modifications, both independently and in conjunction with each other, function through effector interactions to regulate gene expression. A role for histone modifications in the regulation of p53 target gene expression is demonstrated by several studies showing an accumulation of histone modifications at p53 target gene promoters in response to stress. Yet, the precise functions of histone modifications in regulating the p53 gene network remain to be elucidated. The primary goal of our research is to understand the molecular basis for the epigenetic regulation of the gene-selective functions of p53. Toward this goal, we are employing a powerful combination of biochemical, molecular, and genomic approaches. By implementing this multi-faceted strategy, we recently revealed a mechanism by which the prominent histone modification, histone H3 trimethylated at lysine 4 (H3K4me3), through direct interactions with the general initiation factor TAF3/TFIID, facilitates selective p53-dependent activation of cell cycle control genes (Figure 1). By exploiting cell free transcription systems reconstituted with purified factors and recombinant chromatin templates, we identified direct (causal) effects of this histone modification and a functional interplay between this histone mark and core promoter DNA sequences. Thus, our in vitro assays provide a powerful and unique means to identify regulatory mechanisms that function both independently and cooperatively at both the genetic and epigenetic level to mediate coordinated transcriptional control. We are also developing single cell methods to quantitatively measure the temporal dynamics of the recruitment/stabilization of epigenetic readers during a p53 dependent response and to measure whether changes in effector binding correlate with the kinetics and distinct patterns of p53 target gene expression. Ongoing projects in the lab are focused on: The biochemical identification and characterization of histone modifications and effector proteins that regulate differential p53 target gene expression during stress responses. By employing in vitro transcription systems with recombinant chromatin templates, the concerted function of epigenetic marks and effector proteins will be determined at the chromatin level. Single cell methods will be employed to quantitatively measure temporal changes in the accumulation of histone modifications and effector protein recruitment/stabilization and to determine whether changes in effector interactions correlate with the kinetics of p53 target gene expression. The biological significance of histone effector proteins in regulating the tumor suppressor functions of p53 will be examined using cell-based assays and mouse knockout models. Our long-term research goal is to develop target-based screening methods using our defined in vitro systems to examine the effects of small molecule compounds on the functions of histone effector proteins. Together our ongoing studies will aid in the understanding of how histone modifications precisely contribute to the dynamics of p53 target gene regulation, and will ultimately advance the potential for target based screening approaches that will allow for the development of therapeutic targeting strategies, an issue of great significance in light of the recent successes in targeting histone modification effector proteins/readers for developing cancer therapies. \n",
            "-------------------- Scraping faculty url 94/183 --------------------\n",
            "julie law \n",
            "-------------------- Scraping faculty url 95/183 --------------------\n",
            "kuo lee \n",
            "-------------------- Scraping faculty url 96/183 --------------------\n",
            "andres leschziner \n",
            "-------------------- Scraping faculty url 97/183 --------------------\n",
            "jill leutgeb  Biography Jill Leutgeb received her Ph.D. in Neurophysiology from the Leibniz Institute for Neurobiology, Germany, and was a postdoctoral fellow at the Kavli Institute for Systems Neuroscience, Norway, in the lab of Nobel laureates May-Britt and Edvard Moser. She received the prestigious Young Researcher Prize from the Royal Norwegian Society of Sciences and Letters in recognition of her postdoctoral research. Dr. Leutgeb joined UC San Diego in 2008 and is now Associate Professor in the Neurobiology Section of the Division of Biological Sciences. She is also a member of the Center for Neural Circuits and Behavior, and the Traumatic Brain Injury Team of the Kavli Institute of Brain and Mind, UC San Diego. Dr. Leutgeb has received the Hellman Fellowship, the Ray Thomas Edwards Foundation Early Career Award, and the Walter F. Heiligenberg Professorship in Neuroethology.  Research How do neural networks in the brain support the formation of distinct and long-lasting memories? Dr. Jill Leutgebs research is focused on understanding these questions by identifying neural circuit computations which underlie learning and memory. In addition, she also investigates how the function of memory circuits is altered in the development and expression of acquired epilepsies. Her laboratory combines high-density electrophysiology with behavioral testing, theoretical modeling, and pharmacological and molecular manipulations as a multidisciplinary approach to understanding the neural basis of cognition. By understanding the biological basis of memory in health and its dysfunction in disease, the long-term research goals of Dr. Leutgebs laboratory are to develop new therapeutic approaches for restoring memory loss and for seizure prediction, intervention, or prevention. \n",
            "-------------------- Scraping faculty url 98/183 --------------------\n",
            "stefan leutgeb \n",
            "-------------------- Scraping faculty url 99/183 --------------------\n",
            "bjorn lillemeier \n",
            "-------------------- Scraping faculty url 100/183 --------------------\n",
            "byungkook lim  Biography Byungkook Lim received his Ph.D. in Molecular & Cell Biology from UC Berkeley with Dr. Mu-ming Poo and carried out his postdoctoral training with Dr. Robert Malenka in the Department of Psychiatry and Behavioral Science at the Stanford University School of Medicine. Byungkook is a recipient of Kingenstein-Simmons Fellowship, Searle Scholar Fellowship, NIMH Biobehavioral Research Awards for Innovative New Scientists (BRAINS) and BRAIN Initiative Cell Census Network (BICCN) awards.  Research Lab of Functional Anatomy of the Neural Circuitry The basal ganglia and interconnected cortical, amygdalar and thalamic circuits subserve several crucial brain functions including reward-based learning, motor learning, motivated behavior and stress-related behaviors. Dysfunction in these circuits due to alterations of synaptic properties has been proposed to play a critical role in a variety of neuropsychiatric disorders including anxiety, addiction, autism spectrum disorder and major depressive disorder. However, the brain circuits relaying information from or to basal ganglia and their detailed synaptic organization have not been clearly delineated. The main objective of research in our laboratory is to understand the role of the circuit-specific synaptic organization in basal ganglia in animal models of neuropsychiatric disease including drug addiction, feeding disorder, depressive disorder, and movement disorder. Given the circuit and synaptic complexity of the brain, characterizing specific neural circuitry responsible for behavioral changes has been challenging. Dendtiric spines of amygdala neurons projecting to dopamine receptor 1-expressing medium spiny neurons (D1-MSNs) in nucleus accumbens To address the anatomical and functional properties of basal ganglia circuitry, our laboratory applies and develops a variety of molecular strategies (ex. virus-mediated tracing and gene expression, transgenic mice expressing marker proteins in specific population of neurons) together with electrophysiological approaches. Furthermore, we applies cell-type specific or input-specific optogenetic manipulation in vitro and in vivo in pharmacological animal models of human disease as well as a battery of behavioral testing. Future research in my own lab will continue with this strategy and set out to anatomically and functionally define neuronal identity and connectivity of brain circuits beyond basal ganglia and their interplay in mediating reward- and motivation-related behavior in the pathogenesis of neuropsychiatric and neuropathological disease. Double rabies virus-mediated retrograde tracing showing cortical neurons projecting to dorsal striatum (red) and neurons projecting to nucleus accumbens (green). \n",
            "-------------------- Scraping faculty url 101/183 --------------------\n",
            "stanley lo  Biography Stanley Lo received his Ph.D. in biochemistry from Harvard University and was a senior research associate at the Searle Center for Advancing Learning and Teaching at Northwestern University. He joined the faculty of Biological Sciences in 2014. Stanley is the biochemistry editor for CourseSource, an open-access journal of peer-reviewed teaching resources for college biological science courses. He was a National Academies Education Fellow in the Life Sciences in 2011-2012 and has been a National Academies Education Mentor in the Life Sciences since 2013.  Research Our research centers upon two key aspects of undergraduate biology education: faculty teaching and student learning. The broad mission is to understand how faculty teaching promotes student learning, what factors support sustainable classroom innovations, and how students learn core concepts and develop core competencies in biological sciences. Specifically, we investigate how faculty conceptions (beliefs and understanding) and approaches (intentions and strategies) to teaching inform instructional practices in the classroom. Current projects include identifying specific faculty actions in different instructional practices (e.g. the use of personal response devices or clickers) that correlate with positive student outcomes, examining the underlying conceptions and approaches to teaching that are aligned with effective instructional practices, and exploring factors that promote sustainable faculty implementation of these practices. We use a combination of quantitative and qualitative methods (e.g. surveys, observations, interviews, and statistics) to examine how faculty teach in a longitudinal timeframe (i.e. over multiple years and many courses). For student learning, we are interested in defining learning progressions for core concepts and competencies in biological sciences. Learning progressions are descriptions of increasingly sophisticated ways of thinking about a concept or performing a task. Essentially, learning progressions are conceptual pathways that students navigate as they develop understanding of a concept or skills in a competency. Defining learning progressions will provide the theoretical foundation to design instructional material that specifically targets transitions along these conceptual pathways. We are interested in understanding how students and scientists formulate arguments based on evidence and to develop instructional material that promote learning in scientific argumentation. \n",
            "-------------------- Scraping faculty url 102/183 --------------------\n",
            "li-fan lu  Biography Li-Fan Lu received his Ph.D. in Immunology from Dartmouth College. He did his postdoctoral training first at the University of Washington and then at the Memorial Sloan-Kettering Cancer Center where he was the Leukemia and Lymphoma Society Fellow. In 2011, he received the K99/R00 Career Development Award through NIAID.  Research During the last decade a unique mechanism of negative regulation of immune responses and inflammation by a dedicated population of so-called regulatory T (Treg) cells has become the focus of intensive investigation. Expression of the transcription factor Foxp3, a hallmark of Treg cells, is necessary for the establishment and maintenance of a distinct transcriptional program that determines metabolic, signaling, and effector features (e.g., suppressive function), distinguishing these cells from other T cell lineages. The long-term goals of our research are to identify the molecular determinants of differentiation and function of Treg cells and to ultimately gain insight into the complex biology of these cells, including the maintenance of immunological tolerance to self and the regulation of immune responses to pathogens, commensals, and tumors. 1. Identification of miRNAs crucial for mediating Treg cell biology in healthy and diseased mice Several thousands genes are differentially expressed in Foxp3+ Treg cells, including a subset of non-coding small regulatory microRNA (miRNA), some of which are directly regulated by Foxp3 suggesting a role for miRNA-mediated regulation of gene expression in Treg cell differentiation, maintenance or function. Catastrophic consequences and the development of complex disease phenotypes in mice harboring Treg cells devoid of miRNA suggested miRNAs control multiple facets of Treg cell biology (Fig.1). Furthermore, our subsequent studies in miR-155 and miR-146a have demonstrated that individual miRNA could affect distinct aspect of Treg cell biology. By employing genetic, biochemical, immunological approaches and whole animal experimentation, we aim to identify miRNA expression profiles in Treg cells isolated from different tissues in unmanipulated mice as well as in mice suffering different pathologies. Moreover, we are interested in examining how miRNAs themselves are regulated in response to different environmental cues. Ultimately, we hope to elucidate how such regulation of miRNA expression impacts their respective targets and how the dynamic miRNA-dependent regulation enforces Treg cell functionality in diverse disease settings. 2. Elucidating molecular and cellular mechanisms by which Treg cells control type I inflammation Recent studies revealed that the expression of Th lineage-specific transcription factors in Treg cells is crucial for Treg cell-mediated control of corresponding immune responses in particular tissue and inflammatory settings. Our analysis of the role of miR-146a and SOCS1 in Treg cell-dependent Th1 regulation have further extended this concept and demonstrated that both lack of Stat1 and unrestrained Stat1 activation in Treg cells would result in a similar breakdown of immunologic tolerance (Fig.2). These results suggest that Treg cell-mediated suppression of a given Th-specific immunopathology is depending upon an optimal range of activation in corresponding Th-lineage specific transcription factors. One fundamental question that remained is how Treg cells control Th1 responses. At the molecular level, we are interested in identifying important effector molecules and signalling pathways underlying Treg cell-mediated Th1 regulation. At the cellular level, we hope to understand the complex interplay between Treg cells, effector T cells and other immune cells in the surrounding environment specifically in a type I inflammatory setting. Moreover, IFN-mediated Th1 responses and Treg cells have been associated with inflammation in many clinical settings such as infection (e.g., T. gondii), autoimmune diseases (e.g., multiple sclerosis and rheumatoid arthritis), tumors, and organ transplantation. Through our study, we hope to establish the foundation for the future development of means to manipulating Treg and T effector cell function for better treatment in human patients. \n",
            "-------------------- Scraping faculty url 103/183 --------------------\n",
            "victoria lundblad \n",
            "-------------------- Scraping faculty url 104/183 --------------------\n",
            "jens lykke-andersen  Biography Jens Lykke-Andersen received his Ph.D. from University of Copenhagen, Denmark in 1997. He was a postdoctoral fellow at Yale University Medical School before joining the faculty of MCD Biology at University of Colorado Boulder in 2001. He was named a Pew Scholar in 2003. He joined the Division of Biological Sciences at UCSD in 2009.  Research Our laboratory studies the mechanisms of regulation of translation and mRNA turnover in human gene expression. Research in recent years has revealed the importance of regulated mRNA translation and stability in the correct control of gene expression, and how its deregulation can lead to disease. Many of the general factors that direct mRNA translation and the enzymes that degrade mRNAs have been described in recent years. Our laboratory is interested in how these factors are differentially regulated on individual mRNAs to control their rates of translation and mRNA turnover, and how this is regulated by cell signaling. Current research questions in the lab include: How are specific mRNAs targeted for rapid decay? The mechanisms by which specific mRNAs are targeted for rapid degradation remains poorly understood. We study the basic mechanisms that underlie the recognition and degradation of mRNAs targeted for various mRNA decay pathways, including the nonsense-mediated mRNA decay (NMD) pathway, an mRNA quality control pathway targeting aberrant mRNAs that fail to produce full-length protein, and the mRNA decay pathway that targets mRNAs containing AU-rich elements (AREs) in their 3UTRs for rapid degradation. How is translation and mRNA decay regulated by signaling? The response of cells to changes in their environment often requires co-regulation of gene networks, but little is known about how this occurs at the post-transcriptional level. In two recent studies from the lab, we identified the stress granule-associated proteins TIA-1 and TIAR as key translational regulators in response to cell growth conditions of the network of 5TOP mRNAs encoding protein biosynthesis factors (Damgaard and Lykke-Andersen, Genes&Dev, 2011), and we identified a mechanism by which an mRNA decay factor, TTP, which controls the degradation of mRNAs encoding cytokines, fails to recruit deadenylases to target mRNAs when TTP becomes phosphorylated in response to infection, thereby allowing cytokine production and an immune response (Clement et al., MCB, 2011). A current major focus of the lab is to understand the mechanisms by which signaling events modulate the activities of factors that control translation and degradation of regulated mRNA networks. How do mRNA turnover pathways deal with the mRNP? Just like our genomic DNA is coated in chromatin, mRNAs are coated with proteins in mRNPs. We recently observed that in the nonsense-mediated mRNA decay (NMD) pathway, failure of a central RNA helicase, called Upf1, to hydrolyze ATP causes the accumulation of partially degraded mRNP intermediates resistant to exonucleolytic decay (Franks et al., Cell, 2010). These observations suggest that active disassembly of the mRNP constitutes a critical step in mRNA turnover. Consistent with this idea, earlier observations primarily in yeast have shown that stalled ribosomes and strong RNA structure can act as obstacles to mRNA degradation, and that cap binding factors and poly-A binding protein can inhibit decapping and deadenylation, respectively. A current major focus of the laboratory is to understand the importance of ATPases/RNA helicases and of post-translational mRNP modifications in mRNA turnover, to learn whether mRNP remodeling is as critical to mRNA turnover as chromatin remodeling is to transcription. \n",
            "-------------------- Scraping faculty url 105/183 --------------------\n",
            "dmitry lyumkis \n",
            "-------------------- Scraping faculty url 106/183 --------------------\n",
            "eduardo macagno  Biography Eduardo Macagno received his Ph.D. from Columbia University in 1968. He was on the faculty of Columbia from 1973 to 2000. From 1993 to 2000 he served as Associate Vice President for Research and Graduate Education and Dean of the Graduate School of Arts and Sciences at Columbia. He joined the faculty of the Division of Biological Sciences as Founding Dean in 2001. He is the Editor of Developmental Neurobiology and the President of the Academy of Neuroscience for Architecture. He also serves on review committees for the Life Sciences Research Foundation and the W. M. Keck Foundation.  Research In the area of basic neurobiology, our laboratory is currently investigating interactions leading to establishment of neuronal arbors with specific morphologies. In particular, we have focused on the mechanisms which receptors and signaling factors use to mediate self- and homologue-avoidance in the process of defining cell arbor boundaries, a phenomenon that is known as tiling and defines the topology of sensory and motor fields Figure 1. Pair of Live Medicinal Leech Embryos Our studies are carried out in the medicinal leech (Hirudo medicinalis, Hirudo verbana), for which we have created genomic and transcriptomic databases and are in the process of annotating the genome. Its nervous system provides a very accessible experimental platform for molecular and cellular studies; we have developed or adapted all the tools required for modulating the expression of specific genes in individual embryonic or adult identified leech neurons (see Figure 1). Figure 2. Bilateral pair of electrically-coupled central neurons in the medicinal leech We are particularly interested currently in the roles of gap junctions (GJs) in the early differentiation of the central nervous system. We have cloned and characterized the expression patterns of the 21 leech GJ genes (innexins) in the Hirudo genome; 14 or 15 are expressed in central and/or peripheral neurons in the embryo and adult (see Figure 2). In one set of studies, we have shown that innexins can not only form gap junctions but can also form functional hemi-channels that can provide intracellular and intercellular signaling via small molecules, such as ATP. Individual identified central neurons express different subsets of innexin genes, which we have proposed as the determining factor in the specificity of electrically-coupled networks. In one set of studies, we also demonstrated that expression of certain pan-neuronal innexins is necessary for the formation of of gap junctions, while ectopic expression of certain other innexins is sufficient for the formation of novel coupled networks in the CNS. A major effort at present is to explore, using null and functionally-deficient mutations, the role of innexins in interactions between segmental homologs that result in the retraction of some neuronal projections and the maintenance of others (see Figure 3). Figure 3. Some pairs of homologous central neurons retract overlapping projections (AP cell; panel A, panel D), while others form GJs and are retained without major overlap (S cell; panel B) or do not form GJs and ignore each other (P cell; panel C). Knockout of a necessary GJ gene, Inx1, leads to the retention of AP interganglionic projections (cf. panels E and F), with continued growth into adjacent segment (panels G and H) In a very different area of research, we are also involved in a multidisciplinary project at the boundary of Neuroscience and Architecture. This project brings together human brain activity monitoring using portable high-definition EEG with cutting-edge Virtual Reality facilities on campus to assay human responses to the built environment, in particular, as the subjects attempt to navigate through a complex building. This project has exciting practical ramifications, for example, in providing evidence for the design of better healthcare and educational facilities, as well as testing therapies for improving the status of humans with neural impairment conditions. \n",
            "-------------------- Scraping faculty url 107/183 --------------------\n",
            "cressida madigan  Biography Cressida Madigan earned her Ph.D. in microbiology and molecular genetics at Harvard Medical School, where she studied TB lipid chemistry in Branch Moodys lab at Brigham and Womens Hospital. As an NRSA Postdoctoral Fellow, she studied infection biology in zebrafish with Lalita Ramakrishnan at the University of Washington, and completed her training at UCLA in the labs of Alvaro Sagasti, Robert Modlin and Stephen Smale. She joined the Molecular Biology faculty at UCSD in 2018.  Research My lab uses genetic and imaging tools to study mechanisms of infection-mediated inflammation and neurological injury. We use zebrafish as a model host, which allows for real-time observation of infections that typically play out at difficult-to-access sites in humans. We are currently focused on understanding how the mycobacteria that cause leprosy and tuberculosis instigate neurological injury. In leprosy, this nerve damage and inflammation contributes to the stigmatizing deformities that patients endure, such as blindness and loss of fingers. In tuberculosis, central nervous system infection is the deadliest form of TB, and is most common in children under 5 or in HIV-positive adults. By observing mycobacterial infection of the zebrafish brain, we hope to identify pathways of central nervous system inflammation that can be targeted by host-directed therapies. Many of the pathogens we study have caused significant epidemics in human history or are diseases of poverty, and are therefore under-studied. \n",
            "-------------------- Scraping faculty url 108/183 --------------------\n",
            "roberto malinow \n",
            "-------------------- Scraping faculty url 109/183 --------------------\n",
            "teri markow \n",
            "-------------------- Scraping faculty url 110/183 --------------------\n",
            "stephen mayfield  Biography Stephen Mayfield obtained bachelor of science degrees in Biochemistry and in Plant Biology from California Polytechnic University in San Luis Obispo in 1979, and a Ph.D. in Molecular Genetics from the University of California at Berkeley in 1984. From 1984 to 1987 Steve was an NIH post-doctoral fellow at the University of Geneva, Switzerland. In 1987 he joined the Scripps Research Institute and become a professor and associate dean of the graduate school before leaving to join UCSD in 2009.  Research Engineering algae for the production of therapeutic proteins and biofuels The research in the Mayfield lab focuses on molecular genetics in green algae, and on the production of therapeutic proteins and biofuel molecules using algae as a production platform. To develop algae as a biotechnology platform, we are developing the molecular tools that will be used to produce either recombinant proteins or engineered biofuel molecules. We have developed a number of expression vectors that allow for robust accumulation of complex mammalian proteins including monoclonal antibodies, mammalian growth factors, and a variety of potential industrial enzymes. We have shown that this algal-based system is capable of producing these recombinant proteins at very high levels. Recent studies have focused on producing antibody-toxin fusion proteins, in which a targeting antibody domain is linked to a cell-killing toxin. Using this technology, we have produced an antibody-toxin fusion that binds and kills human B-cell lymphomas, and cell-based assays have demonstrated the utility and specificity of this molecule. These proteins have great potential for the treatment of cancers and infectious diseases, and chloroplasts offer perhaps the only system in which these types of proteins can be produced. We have also recently succeeded in generating expression vectors that allow for expression of recombinant proteins from the nuclear genome at relatively high levels, and that also allow for secretion of these proteins from the cell. These new vectors will increase the repertoire of recombinant proteins that can be produced in algae thereby expanding the platform. Engineering algae for the production of hydrocarbon molecules that can be used as biofuels is also a major focus of the lab, and recent studies have demonstrated the tremendous potential of eukaryotic algae for the expression of recombinant enzymes that allow for greater production of biofuels. Our continued genetic, biochemical and structural studies should lead to a greater understanding of the mechanism of gene expression in algae, and allow algae to become a viable source for sustainable biofuel and recombinant protein production. \n",
            "-------------------- Scraping faculty url 111/183 --------------------\n",
            "lisa mcdonnell  Biography Lisa McDonnell received her Ph.D. in the Faculty of Forestry from the University of British Columbia (UBC) where she used tools from cell biology, molecular biology, and biotechnology to understand the role of various genes in the production of cellulose. She later returned to UBC as a post-doctoral fellow with the Carl Wieman Science Education Initiative. She joined the faculty of Biological Sciences in 2015.  Research My biology education research broadly aims to understand how teaching practices influence student learning and development of scientific thinking. Scientific thinking skills, such as data analysis, evaluation, forming scientific arguments, and problem solving, are a key component of a biology education, but understanding when and how these skills develop requires more investigation. Currently I am interested in how students develop critical thinking skills, and how their understanding of biology research is influenced by engaging in research experiences. I collaborate with Dr. Stanley Lo to investigate these questions in biology lab classes here at UCSD. Another area of interest for me is how teaching practices influence student learning and development of writing and the ability to form scientific arguments. Two current projects are collaborations with colleagues here at UCSD (Sarah Stockwell, Emily Grossman, Stanley Lo, Stephanie Mel) to investigate the impacts of two interventions on students writing and scientific argumentation. The first is looking at how the incorporation of opportunities for students to explore academic integrity, in the context of a biology class writing assignment, influences their writing. The second is looking at how the incorporation of writing and peer review impacts the quality of students constructed scientific arguments and the quality of the reviews they provide to peers. Both of these projects aim to contribute to our understanding of the teaching methods we can employ to improve the development of knowledge and skills that are necessary to be successful biologists. \n",
            "-------------------- Scraping faculty url 112/183 --------------------\n",
            "bill mcginnis  Biography Bill McGinnis received his Ph.D from UC Berkeley in 1982 and was a Jane Coffin Childs postdoctoral fellow at the University of Basel. From 1984 to 1995, he was on the faculty of Yale University. He received a Searle Scholar Award, a Presidential Young Investigator Award, and a Dreyfuss Teacher/Scholar Award. In 2010, Dr. McGinnis was elected to the American Academy of Arts and Sciences.  Research One major research interest of our lab is the regulation of transcription patterns of Hox genes in animal embryos. Hox gene expression patterns determine the morphological features that develop on the anterior posterior axis of animal embryos, and how Hox expression patterns are regulated are critical to their developmental functions, as well as the evolutionary diversity in form that arises in different animal embryos. One current specific interest is how distant Hox enhancer-promoter interactions regulate transcriptional activation of Hox genes in Drosophila embryos. We have developed methods to detect transcriptional activation patterns of many genes simultaneously in individual nuclei, and at the same time detect mRNA molecules from individual genes, as well as the DNA regulatory sequences that activate transcription from those genes, in individual nuclei in which the genes are activated or repressed. We are also exploring the genes underlying a signaling pathway that controls regeneration of animal epidermis after wounding, using Drosophila as a model system. This pathway, which involves the activation and repression of thousands of genes that regulate epidermal regeneration, can be activated by sterile puncture wounds of fly embryos. Genetic screens are identifying new components in the wound response pathway, including the extracellular signals and receptors that instruct cells around wounds to initiate epidermal barrier repair. New methods to visualize wound activated gene regulation are also being developed. \n",
            "-------------------- Scraping faculty url 113/183 --------------------\n",
            "stephanie mel \n",
            "-------------------- Scraping faculty url 114/183 --------------------\n",
            "justin meyer  Biography Justin Meyer received his Ph.D. from Michigan State University and was a Systems Biology Departmental Fellow at Harvard Medical School, where he was awarded the James S. McDonnell Foundation Fellowship for Studying Complex Systems. He joined the faculty of Ecology Behavior and Evolution and the Quantitative Biology Initiative in 2014.  Research What changes to viral genomes would allow them to infect new species? And, what are the natural processes mutation, recombination, and natural selection that permit their evolution? My lab group uses experimental evolution of viruses to find answers to these questions. We are able to culture viruses in the laboratory, watch them evolve in real time, and on occasion, witness them gain access to new host species. Our studies combine methods from many fields of biology. We typically begin research by modifying classic ecological niche theory to make predictions for viral host-range evolution. Next we test the theory by culturing viruses under controlled conditions. We sequence the evolved viral genomes to identify adaptive mutations, and then we determine the effects of the mutations by introducing them into nave viral genomes using Multiplexed Automated Genome Engineering. Lastly, we characterize their newly evolved qualities by using single molecule biophysics assays. Altogether, the studies provide a complete picture for how gain-of-function mutations evolve in viruses. Most of our work to date has been preformed on the virus Bacteriophage , which infects E. coli. We have observed evolve to exploit a novel receptor, the outer-membrane protein OmpF. Typically only targets another protein, LamB, however under the right conditions it naturally evolves this innovation. Which receptors a virus exploits largely determine their host-range, and so, acquiring a new receptor constitutes a major transition for the virus. Currently our research group is continuing to test hypotheses for how this evolved, but we are also applying the tools we developed to study to other viral systems. Bacteriophage Image by Brian Wade New receptor, OmpF Image by Justin Meyer \n",
            "-------------------- Scraping faculty url 115/183 --------------------\n",
            "maurice montal  Biography Mauricio Montal received his Ph.D. from the University of Pennsylvania and his M.D. from the National University of Mexico. He is the recipient of a K.C. Cole Award of the Biophysical Society for outstanding contributions in Membrane Biophysics and a Research Scientist Award from NIMH. Dr. Montal is a recipient of a Gugenheim Fellowship and a Fellow of the Biophysical Society.  Research The key scientific question has been how membrane proteins function in terms of the underlying structure. These entities are truly smart materials. Proteins are nanomachines and marvels of protein chemistry. Lipid bilayers, the basic structural elements of biological membranes, are nanodevices which exploit self-assembly principles to provide the exquisite environment for proteins to operate. Together as a system, proteins embedded within lipid bilayers, embody the basis for biosensor and nanosensor technology. They also emerge as templates for molecular mimicry and design and follow the hierarchical organization underpinning complex systems. Our programs are of prime significance to health science and biodefense; they also comprise components of relevance to the pharmaceutical and cosmeceutical industries. Specifically, our aim is to understand the structures and the mechanisms underlying nerve cell function and dysfunction. We are interested in the molecular basis of the intrinsic excitability of neurons, the communication of excitability between neurons, neuronal cell survival and death, the processes leading to neurodegeneration, and the genetic components in the pathogenesis of some forms of infant epilepsy. Given that our subjects of inquiry are ion-conducting and protein-conducting channels, key elements of the nerve cell, they embody drug targets; therefore, our programs encompass a component directed to screen or discover agents that impact these processes and, may in due turn, become clinically important drugs. Botulinum neurotoxin (BoNT) is considered the most potent toxin; it is the causative agent of botulism, a neuroparalytic illness, and a major bioweapon without an antidote. A research program is directed towards this goal. Our objective is to understand the intricacies of intracellular trafficking of BoNT that may disclose novel pathways to escort, target, and insert it into membranes. This information may prove diagnostic in identifying unsuspected sites for intervention and plausible strategies for targeted drug delivery. BoNT is a modular nanomachine in which one of its modules -- the heavy chain channel -- operates as a specific protein translocating transmembrane chaperone for another of its component modules -- the light chain protease. We investigate the dynamics of protein-translocation in neuronal cells focusing on the interactions between the channel/chaperone and its cargo using an assay with single molecule sensitivity. This strategy led us to uncover an entirely unknown succession of intermediate states during translocation of the cargo which reveal transient protein-protein interactions occurring at distinct stages during protease translocation through the channel. Another major area of interest concerns the design of biosensors based on lipid bilayers membranes containing a number of designed channel proteins that emulate those naturally occurring in the brain. Such devices are indeed nanomachines which may display unparalleled sensitivity in analyte detection, including but not limited to biological and environmental toxins, and also in sensing physical stimuli such as light, temperature, pressure or electric fields. \n",
            "-------------------- Scraping faculty url 116/183 --------------------\n",
            "marc montminy \n",
            "-------------------- Scraping faculty url 117/183 --------------------\n",
            "cornelis murre  Biography Kees Murre performed his graduate work at Harvard Medical School and was a postdoctoral fellow at MIT. He is a Searle Scholar and the recipient of the National Institutes of Health Merit Award.  Research We study gene regulation in adult stem cells and developing lymphocytes. Both global and single cell strategies are being utilized with the aim to describe normal development as well as aged and diseased states in molecular terms. We have a long-standing interest in deciphering the role and regulation of helix-loop-helix proteins in lymphocyte development. This class of factors plays key roles in hematopoiesis. Currently our main interests are in the role of these factors in the control of hematopoietic stem cell homeostasis, B- and T-lineage specification and commitment, aging, inflammatory disease and their roles in the periphery in the response to invading pathogens. Recent genome-wide studies have identified that a large fraction of the genome transcribed in non-coding regions. We are now faced with the question as to how these large non-coding RNAs relate to the control of gene expression. We have identified a subset of lineage-specific and developmental-stage specific non-coding RNAs. Our interests are to determine the function of these non-coding RNAs and to study their potential roles in modulating long-range chromatin structure and genomic interactions. Our knowledge of chromatin structure and long-range genomic interactions in mammalian cells is still rudimentary. How do the genomes of lymphocytes, plasma cells and granulocytes adopt such unique and distinct structures? We aim to resolve these questions using genome-wide chromosome-conformation capture studies (HiC) in conjunction with computational approaches to describe the topologies of lymphoid and myeloid genomes in molecular terms. During developmental progression of lymphoid cells coding and regulatory elements interact to induce lineage-specific programs of gene expression. We recently found that in eukaryotic cells coding and regulatory genomic elements bounce back and forth within the chromatin network until specific genomic interactions are established, and that spatial confinement of topological domains largely controls the times for such encounters. Our future studies aim to examine how epigenetic and structural determinants affect the trajectories adopted by the chromatin fiber in living cells and how this relates to genomic encounters involving enhancers and promoters. We have recently demonstrated that genes encoding for key developmental regulators reposition during developmental progression. We now aim to address the question as to why and how genomic regions encoding for developmental regulator reposition during developmental progression, how their release from the lamina is regulated and how they associate with active transcription factories. \n",
            "-------------------- Scraping faculty url 118/183 --------------------\n",
            "sonya neal  Biography Dr. Neal received her Ph.D. from U.C. Los Angeles in 2013 after working in the laboratory of Dr. Carla Koehler. She then carried out her postdoctoral studies at U.C. San Diego in the laboratory of Dr. Randolph Hampton. She was the recipient of the Burroughs Wellcome Postdoctoral Diversity Enrichment Award and Ruth L. Kirschstein NRSA Postdoctoral Fellowship. Dr. Neal joined the faculty in 2018.  Research \"Understanding the mechanistic underpinnings and medical importance of misfolded proteins\" Life is harsh, and hard on protein structure. Continuously-produced misfolded proteins represent a threat to proper cell and organ function. To survive this threat, organisms are equipped with quality control systems to maintain properly folded proteins; a process known as protein homeostasis. For the past two decades, it has become increasingly clear that protein homeostasis is critical to the health of cells and organisms. Defects in protein homeostasis underlie many of the most pressing maladies, including aging, cancer and neurodegenerative diseases. Our research strives to understand the basic biology of protein homeostasis and how organisms use quality control pathways to mitigate this universal biological stress. There are three questions that motivate our research: How do cells maintain protein homeostasis? How do changes in protein homeostasis lead to pathology? Can modulation of protein homeostasis be used to treat disease? Our research aim is to understand the cellular and tissue-specific roles of factors involved in protein quality control pathways, with a particular emphasis on the pathways that detect and specifically destroy these dangerous molecules. An understanding of these pathways allows us to define the mechanisms that protect the organismal proteome in health and disease, and eventually devise methods to harness cellular quality control to modify the proteome in the laboratory and the clinic. As a first step towards this goal, we are focused on discovering and studying proteostasis pathways in yeast, mammalian cells and zebrafish, allowing us to synergistically capitalize on the investigative strengths that each has to offer. We take advantage of the laboratorys unique set of skills in genetics, biochemistry, functional genomics, cell and molecular biology to extend these lines of inquiry and to train the next generation of scientific leaders. \n",
            "-------------------- Scraping faculty url 119/183 --------------------\n",
            "james nieh  Biography Dr. Nieh received his BA from Harvard in 1991 and his Ph.D. from Cornell University in 1997. He completed a NSF-NATO postdoctoral fellowship at the University of Wrzburg, Germany and was a Harvard Junior Fellow from 1998-2000.  Research Research in the Nieh lab focuses on how natural and man-made stressors affect the biology of and cognitively sophisticated behaviors exhibited by bees. Our research focuses on two areas: (1) the selective pressures that may have shaped the evolution of communication in highly social bees and (2) honey bee health. We use the tools of Behavioral Ecology, Chemical Ecology, Animal Communication and Neuroethology to work with bumble bees, stingless bees, and honey bees. Five different topic areas are detailed below.For further information, please view the Nieh Lab Homepage . Evolution of communication Selective pressures from competitors and predators has shaped social bee communication. Our lab studies multiple bee groups: honey bees, stingless bees, and bumble bees to learn how this communication works and why it may have evolved. Honey Bee health Concern is growing over pollinator declines. Our lab examines the effects of natural stressors, such as pathogens, and man-made stressors, such as pesticides, on honey bee health, foraging, flight, and orientation. Superorganism inhibitory communication What happens if conditions change and the communicated food source becomes depleted, contested, or dangerous? The honey bee stop signal provides inhibition that counteracts the positive feedback of honey bee waggle dances. Using field studies and modeling, we are studying this signal in detail and exploring conditions under which inhibitory signals may evolve. Superorganism inhibitory communication We study olfactory eavesdropping in stingless bees and honey bees and examine the advantages of eavesdropping upon competitors and predators. Neuroethology of bee learning and memory Despite their small brain size and limited number of neurons relative to the central nervous systems of many vertebrates, social insects have evolved sophisticated learning and memory abilities and are therefore important models for animal cognition. However, these abilities can be impaired by field-realistic exposure to pesticides and other man-made stressors. \n",
            "-------------------- Scraping faculty url 120/183 --------------------\n",
            "axel nimmerjahn \n",
            "-------------------- Scraping faculty url 121/183 --------------------\n",
            "maho niwa rosen  Biography Maho Niwa received a MS from the Chemistry Department at Brandeis University and a Ph.D. from the Biochemistry Department at Baylor College of Medicine. She was a Jane Coffin Childs Cancer postdoctoral fellow in Dr. Peter Walter's laboratory at UCSF.  Research How does a cell tailor its components to the many different demands or insults it receives during its lifetime? From yeast to humans, the endoplasmic reticulum (ER) is a critical compartment where a large fraction of total cellular proteins, including almost all plasma membrane proteins and secreted proteins undergo folding, modification, and maturation. In addition, synthesis of almost all cellular lipids, at least in their initial steps, occurs in the ER. The ER plays further vital roles in the detoxification of drugs and poisons (liver ER), as well as in intracellular Ca2+ regulation. As complex as these roles already are, ER functions need to change constantly to respond to alterations in developmental stage, new environmental stresses, or a whole host of cellular insults. Cells have to recognize such changes and make proper adjustments for the needed ER functions. It is increasingly recognized that insufficient ER protein folding capacity can be an important contributing factor to the pathology of human diseases including a long list of cancers, diabetes, obesity, Alzheimers, Cystic Fibrosis, and Parkinsons disease. We are specifically interested in two major areas: (1) How increased demands for ER functions in response to cellular stress and insults are recognized and lead to production of the needed ER functions, and (2) How is the ER divided the cell cycle? Progress in this latter area has led us to describe a new cell checkpoint. (1) How increased demands for ER functions in response to cellular stress and insults are recognized and leads to production of the needed ER functions? The Unfolded Protein Response (UPR) operates in a variety of immune and other cells as the signaling pathway to up-regulate ER levels or functions so that cells may deal with stressful or altered circumstances. Three UPR transmembrane protein sensors, IRE1, PERK and ATF6, in the ER membrane initiate these regulatory downstream events via distinct and unusual mechanisms. IRE1 is an ER transmembrane receptor kinase that senses changes in the ER via its ER luminal domain. When activated, IRE1 acts as a sequence specific RNAase (splicase) to generate the spliced form of XBP1 mRNA, which then can generate active XBP1 transcription factor. Because activation of this IRE1 pathway is observed in many solid tumors and hematological cancers, we searched for and identified inhibitors of IRE1s XBP1 RNA splicease. We discovered Irestatin (STF-083010), which inhibits the IRE1 splicease and also has anti-myeloma activity on cancer biopsies from myeloma patients. Interestingly, we show that cancer chemotherapy drug doxobuciin, a DNA replication inhibitor, and acridine derivatives also inhibit the IRE1 splicease activity. Activated IRE1 is also known to have a second RNase activity that cleaves a subset of mRNAs for degradation, reducing translation of such mRNA. Recently, we found that the activation mechanisms of these two IRE1 RNases are distinct and can be regulated separately. We also found that the two IRE1 RNase activities contribute differentially to cell death and survival after ER stress. All these issues are ongoing projects in the lab. ATF6 is another unique UPR sensor. Interestingly, ATF6 is an ER transmembrane protein that is a cryptic transcription factor. Upon activation, ATF6 is known to be cleaved within its ER transmembrane domain by proteases, which frees the cytoplasmic domain to travel into the nucleus to become a UPR-specific transcription factor. Most recently, we have found that ATF6 can become an activated transcription factor by a previously undiscovered mechanism. Activating ATF6 in this manner preferentially causes transcription of a set of target genes involved in lipid biosynthesis and regulation. These discoveries should allow us to probe the differences between proteotoxic and lipotoxic stressors and the roles they play in different disease. (2) A new cell cycle checkpoint ensuring that progeny cells get functional ER before cell division proceeds: the ERSU pathway. We have made the pioneering discovery of a new yeast cell cycle checkpoint, the ER Stress Surveillance pathway (ERSU), which acts to ensure the inheritance of a functional ER. We show the ERSU pathway is independent of the yeast UPR pathway. This cell cycle block to stressed ER inheritance is accompanied by a cytokinesis block. While the cell cycle is halted by ERSU, the UPR pathway becomes activated and ultimately restores ER function, releasing the ERSU block and allowing re-entry into the cell cycle. We have found that one of the ERSU pathways initiating signals is the sphingolipid, phytosphingosine (PHS). Indeed, we find that ER stress leads to a transient increase in PHS levels. Moreover, we show that a family of ER membrane proteins, the Reticulons (Rtn 1, 2, and Yop1), are intimately involved in the early steps of the ERSU pathway. This means that Reticulons, previously known to be involved primarily in generating ER curvature, have a second important function the ERSU cell cycle checkpoint that governs recognition and inheritance of a functional ER. Our understanding of the UPR and ERSU pathways holds unique promise for the development of new methods to approach human diseases caused by misregulation of the ER. \n",
            "-------------------- Scraping faculty url 122/183 --------------------\n",
            "clodagh o'shea \n",
            "-------------------- Scraping faculty url 123/183 --------------------\n",
            "karen oegema  Biography Karen Oegema received her PhD in cell biology from the University of California, San Francisco and conducted postdoctoral work at the European Molecular Biology Laboratory in Heidelberg and the Max Planck Institute for Cell Biology & Genetics in Dresden. She was a Pew fellow in the Biomedical Sciences and is a recipient of the American Society of Cell Biology WICB Early and Mid-Career Awards. She is a monitoring editor for the Journal of Cell Biology and a lifetime fellow of the American Society for Cell Biology.  Research My lab is interested in the interface between cell and cancer biology. My group has had a long-term interest in the molecular machinery underlying mitosis and cytokinesis. This research has revealed that in addition to common themes, there is also significant variation between cell types in how cell division is accomplished. A current focus in the lab is to determine whether these differences can be leveraged to induce death due to aberrant mitosis in a cell type specific fashion as a chemotherapeutic strategy. An example of this has arisen from our work on centrosomes. Through a combination of experiments in the model system C. elegans and in human cells, we have made important contributions to understanding the pathway that controls centrosome duplication, which depends on a centrally important regulatory kinase called Plk4. To explore the therapeutic potential of centrosome removal in cancer, we collaborated with the Small Molecule Discovery Group in the Ludwig Institute for Cancer Research to develop the first potent, specific, cellularly active Plk4 inhibitor. Our experiments using this inhibitor, which we call centrinone, revealed striking differences in how cells respond to centrosome removal, and have suggested that centrosome removal could have therapeutic potential for specific cancer types. We are currently working to understand the molecular basis of these differential effects and are initiating screens to expand our understanding of variation in the cell division machinery across cell types. A second research avenue in my lab has been aimed at generating a phenotypic map of the genes essential for embryonic development in a model metazoan. In this project, we are taking a high-content approach based on time-lapse imaging of tissue specification and morphogenesis during embryogenesis to generate a functional gene network for the set of ~2000 C. elegans genes required for embryonic development. To analyze the 26,000 time-lapse movies resulting from this effort, we have developed automated methods for phenotypic parameterization and comparison that allow us to identify genes with similar phenotypes, suggesting that they function in common pathways. Of the genes that we are characterizing in this effort, 75% are conserved in humans and 900 are functionally uncharacterized, suggesting that this approach will allow us to predict function for a large number of human genes. \n",
            "-------------------- Scraping faculty url 124/183 --------------------\n",
            "melinda owens  Biography Melinda Owens is passionate about effectively helping all students learn biology. After receiving her PhD in neuroscience from UCSF, she taught at a range of institutions, including Stanford University and California State University, East Bay. She then was a SEPAL Postdoctoral Fellow in Biology Education with Kimberly Tanner at San Francisco State University. She joined UCSD in 2018.  Research It is critically important to teach undergraduates about biology more effectively and to retain more of them in scientific and health careers. My lab addresses this issue by examining two broad questions: How do college instructors teach? How do college students think about biology? How Do Instructors Teach? Prior research shows that certain teaching strategies are more likely to promote engagement, retention, and learning among diverse students, but it has been difficult to examine the strategies instructors use on a large scale. Therefore, in collaboration with San Francisco State, we developed DART (Decibel Analysis for Research in Teaching). DART (dart.sfsu.edu) is a software tool that can quickly analyze classroom audio recordings to estimate with ~90% accuracy the time spent using certain teaching activities. We use DART to investigate a variety of questions, with the goals of better understanding how instructors teach and promoting evidence-based teaching practices. DART uses classroom sound levels (dark blue line) to estimate the extent of \"active learning\" by classifying class time into these categories: Single Voice (e.g. lecture), Multiple Voice (e.g. pair and group discussions), and No Voice (e.g. student thinking or writing). Owens et al. 2017 How Do Students Think? Students learn more effectively when instructors know what ideas their students already have. In particular, previous studies reveal that students often harbor misconceptions that inhibit their ability to develop biological expertise. Many misconceptions are rooted in ways of intuitive thinking that predate modern scientific thought. We are particularly interested in how undergraduate students understand and reason about health topics, such as vaccines. We ask undergraduates about their ideas about specific topics and examine their misconceptions and the types of reasoning they use. We will use our results to design education interventions to promote better learning about these topics. \n",
            "-------------------- Scraping faculty url 125/183 --------------------\n",
            "satchidananda panda \n",
            "-------------------- Scraping faculty url 126/183 --------------------\n",
            "amy pasquinelli \n",
            "-------------------- Scraping faculty url 127/183 --------------------\n",
            "gentry patrick  Biography Dr. Patrick received his Ph.D. from Harvard University in 1999 after working in the laboratory of Dr. Li-Huei Tsai. He was a Damon Runyon Cancer Research Foundation and a United Negro College Fund/Merck postdoctoral fellow with Dr. Erin Schuman at the California Institute of Technology. Dr. Patrick joined the faculty in 2004.  Research Our laboratory is interested in how synaptic activity modulates the molecular make-up of synaptic connections in the mammalian central nervous system (CNS), which in many cases leads to long-lasting changes in synaptic efficacy. The concerted regulation of protein synthesis and degradation is fundamental for the control of diverse cellular events. Many studies have provided evidence that new protein synthesis likely takes place at synapses and is required for plasticity. Protein degradation, on the other hand, provides another way to regulate protein levels. In fact the ability to dynamically control protein levels allows for very tight control of rapid signaling cascades. We study the ubiquitin-proteasome system (UPS), one of the major cellular pathways controlling protein turnover in mammalian cells. The UPS is a complex proteolytic pathway whereby proteins are targeted to the 26S proteasome for degradation. Ubiquitin is covalently attached to a target protein through a series of steps: first an E1 ubiquitin activating enzymes pass ubiquitin to E2 transferase and E3 ligases. At this point, many times in concert with the help of an E2 enzyme, the E3 ligase binds and modifies the target protein with the ubiquitin. Multiple ubiquitin molecules are added and the protein is recognized and degraded by the 26S proteasome. Many cellular roles have been defined for the UPS such as cell cycle control, cell fate and growth determination, antigen presentation, and many cell signaling pathways. In contrast the mechanisms of how the UPS regulates the growth and development, maintenance, and remodeling of synaptic connection in the mammalian central nervous system (CNS) is less understood. An interesting problem is how activated synapses of a single neuron become selectively modified as a result of synaptic plasticity. It is known, for example, that synaptic modifications can occur selectively at one group of synapses, but not at another group of synapses on the same neuron. This property is known as \"input-\" or \"synapse specificity\". It is plausible that the selective degradation of proteins that restrict or limit plasticity may be required for these synaptic changes to occur. Alternatively, various proteolytic activities may provide specificity for long-term synaptic changes. This could be accomplished through the degradation of some proteins at specific locations or by targeting regulatory components of a proteolytic pathway to modified or unmodified sites. We study the UPSs role in synaptic plasticity. One approach we take is to use genetically encoded fluorescent-based proteasome (or degradation) reporters and time-lapse confocal microscopy to assay how neuronal activity modulates the rate of degradation within the spines and dendrites of neurons. Using these techniques, we hope to understand how and why the UPS is activated at or recruited to synapses in response to neuronal activity. We are also interested in identifying the protein targets of the UPS at synapses. In particular, we study how the UPS regulates alpha-amino-3-hydroxy-5-methyl-4-isoxaolepropionic acid (AMPA) type ionotropic glutamate receptor trafficking in neurons. The removal and insertion of AMPA receptors at glutamatergic synapses is one mechanism to either decrease or increase synaptic strength. We and others have found that the activity of the UPS is required for agonist-mediated endocytosis of AMPA receptors. The simple model is that protein degradation via the UPS is required for normal AMPA receptor trafficking in neurons. Using molecular and biochemical approaches, we hope to identify the components of the UPS and protein targets involved AMPA receptor trafficking. We are also interested in neurodegenerative disease. In humans, many neurodegenerative diseases are characterized post-mortem by anatomical hallmarks visible by standard light microscopy. Alzheimers and Lewy body disease are the most common causes of dementia in elderly populations. These diseases are characterized by neurofibrillary tangles (NFT), plaques, and lewy bodies (large dense inclusions). Familial Parkinsons disease is also characterized by Lewy bodies. In addition, large dense inclusions are characteristic of many of the CAG repeat diseases such as Huntington and ALS. Biochemical and immunohistochemical characterization of these inclusions, tangles, and neuritic plaques has indicated that they contain high concentrations of ubiquitin and ubiquitinated proteins. Does the ubiquitin-proteasome pathway contribute to these abnormalities and to neuronal death? We hope that the understanding of UPS in normal neuronal function will help elucidate how it may be involved in neuronal dysfunction and disease. \n",
            "-------------------- Scraping faculty url 128/183 --------------------\n",
            "gulcin pekkurnaz \n",
            "-------------------- Scraping faculty url 129/183 --------------------\n",
            "michael perry  Biography Michael Perry got his start in biology while an undergraduate and M.S. student at the University of Florida, working at the Florida Museum of Natural History, where he collected butterflies in the field at sites that ranged from the Rocky Mountains to the cloud forests of Costa Rica. To begin to understand the molecular origins of the diversity he encountered in the field he shifted his focus to developmental biology and gene regulation during his Ph.D. research at UC Berkeley with Mike Levine and Nipam Patel, where he was supported by an NSF-GRFP fellowship. He was able to combine his diverse interests in evolution and developmental biology as a postdoctoral fellow with Claude Desplan at New York University. Michael was the recipient of NIH Ruth L. Kirschstein and Charles H. Revson Foundation fellowships as well as a K99/R00 Pathway to Independence Award from the NIH National Eye Institute. He will join the Division of Biological Sciences faculty at UC San Diego in March 2019.  Research Stochastic gene expression in developing photoreceptors One of the most interesting questions in biology is how the complexity of the nervous system is encoded in the genome: how it provides rules to build a system that can compute, remember, and produce behavioral responses. To understand which genetic changes shape neural function we leverage naturally occurring variation in nervous systems within and across species. The lab uses the insect visual system as a model, from Drosophila and other flies all the way to butterflies and wasps. Insect eyes and brains have been shaped by a rich diversity of functional requirements to exquisitely tune the nervous system and behavioral responses to allow them to thrive in an incredible range of environments. A flower fly showing stochastic patterns on the retina Image by Yousef Al Habshi Natural diversity of the visual system comes in large part from changes in how and when genes are expressed during development. The advent of powerful genomic tools such as CRISPR/Cas9 and single cell sequencing have radically changed the questions we can ask in non-model systems: we can now perform targeted comparisons and functional molecular experiments in almost any species. There has never been a better time to combine approaches in model systems with experiments directly in species that exhibit interesting differences in form, function, and behavior as a targeted means of unraveling neural complexity. How are the full diversity of neural types specified, and how do new neural types evolve and plug in to existing circuity? How do such changes modify downstream neural function? Does the interconnected nature of the nervous system influence how it evolves? The long-term goal of the lab is to understand the genetic basis of nervous system development and evolution at levels ranging from molecules to behavior. CRISPR in butterflies: mosaic yellow mutant Papilio xuthus and Vanessa cardui \n",
            "-------------------- Scraping faculty url 130/183 --------------------\n",
            "katherine petrie  Biography Katie Petrie received her Ph.D. in Biology from The Scripps Research Institute in 2014, where she studied the processes that drive evolution by combining in vitro molecular model systems with high throughput sequencing. She completed her postdoctoral work as an EON Research Fellow, splitting her time between UC San Diego and the EarthLife Science Institute (ELSI) in Tokyo, Japan. She joined UC San Diego as an Assistant Teaching Professor in 2018.  Research Education As an assistant teaching professor, I bring bioinformatics and genomicskey tools for doing biology in the 21st centuryinto the undergraduate curriculum at UC San Diego. I am interested in how enabling students to use this technology impacts their learning and careers. In my teaching efforts, I use proven active learning strategies to maximize student engagement. I am interested in developing new teaching strategies for building scientific literacy, and on expanding access to STEM fields by incorporating research with primary data into the classroom experience. As part of this effort, I am incorporating CURES (Course-Based Undergraduate Research Experiences) into the courses I teach. These gives students the opportunity to make a genuine contribution to scientific knowledge during their laboratory classes, and have the potential to increase inclusivity and retention in STEM careers. Biological As an evolutionary biologist, I am interested in understanding the driving forces for evolution, including the mechanisms underlying evolutionary innovations. In collaboration with members of Justin Meyers lab here at UCSD, I use genomic and laboratory approaches to better understand how evolution can lead to new traits and functions. Recently, weve discovered a multi-tasking protein that enables an important evolutionary transition: the ability of a virus, bacteriophage , to infect a new host . In collaboration with researchers at the EarthLife Science Institute in Tokyo, Japan, I am also studying how selective interactions between nucleic acids and minerals might have contributed to pre-Darwinian enrichment processes on the early earth prior to the origin of life. \n",
            "-------------------- Scraping faculty url 131/183 --------------------\n",
            "samuel pfaff \n",
            "-------------------- Scraping faculty url 132/183 --------------------\n",
            "lorraine pillus  Biography Lorraine Pillus carried out graduate studies with Frank Solomon at MIT. Her postdoctoral research with Jasper Rine at the University of California, Berkeley was supported by a John D. and Catherine T. MacArthur Foundation Fellowship of the Life Sciences Research Foundation. Prof. Pillus has been named a Pew Scholar and a National Science Foundation New Young Investigator and has received teaching excellence awards from the Mortar Board Honor Society and the Associated Students of the Skaggs School of Pharmacy & Pharmaceutical Sciences. She served as Chair of Molecular Biology from 2012-2016 and is currently Associate Dean of the Division. She was elected as a Fellow of the American Association for the Advancement of Science in 2017 and to the American Academy of Microbiology in 2018.  Research A gene's chromosomal position and local chromatin structure can have profound consequences for its expression. We are interested in all aspects of these chromosomal and chromatin effects because they underlie epigenetic processes that are critical for normal growth and development. We are particularly focused on the form of chromatin regulation known as transcriptional silencing and study its effects on yeast and human genes. Chromatin is controlled by post-translational modification of its core subunits, the histones, and other transcriptional factors. The enzymes catalyzing these modifications include acetyltransferases, (HATs), deacetylases (HDACs), methyltransferases (MTs), kinases, and others. The enzymes work together to promote dynamic patterns of modification that help to specify epigenetic processes. We study several classes of the chromatin modifying enzymes to understand the range of their in vivo functions, including mechanisms of their activity, genomic targeting, and interaction with transcriptional and DNA repair complexes. Our experimental approaches include genetic, genomic and biochemical strategies to understand the MYST family HATs, the SIR2 family HDACs, and several classes of evolutionarily conserved MTs. \n",
            "-------------------- Scraping faculty url 133/183 --------------------\n",
            "joseph pogliano \n",
            "-------------------- Scraping faculty url 134/183 --------------------\n",
            "kit pogliano  Biography Kit Pogliano received her Ph.D. from the Department of Microbiology and Molecular Genetics at Harvard Medical School and was a Damon Runyon-Walter Winchell postdoctoral fellow at Harvard University. She is a recipient of the Searle Scholar and Beckman Young Investigator awards. Dr. Pogliano was named in 2009 as a Fellow in the American Academy of Microbiology.  Research Our research is organized around three key themes: Cellular organization and dynamics: What are the design principles of bacterial cells? Microbial interactions: How are the outcomes of interspecies interactions determined? Chemical Cell Biology: How do secreted metabolites and antibiotics affect bacterial cells? Our toolkit We take a multidisciplinary approach to our studies, using the tools of genetics, cell biology, proteomics, biochemistry, and we have recently added analytical chemistry to our toolkit. Our lab is fully equipped for molecular biology, genetics and for work with BSL2 pathogens. Our microscopy facility includes one dissecting stereomicroscope equipped for fluorescence and timelapse imaging as well as deconvolution microscopes equipped with growth chambers and microfluidic devices for time-lapse microscopy and lasers for photobleaching and total internal reflection microscopy. We have an HPLC apparatus for the analysis and purification of natural products, and more extensive equipment is available in the labs of our campus collaborators. We have ready access to excellent campus facilities that provide cutting edge equipment and analytical tools for protein and small molecule mass spectrometry, super resolution microscopy and sequencing, with excellent technical assistance available as needed. We also apply our imaging expertise to a variety of collaborative projects and enjoy the diversity these projects and visitors bring to our lab. The SpollQ landmark protein (green) localizes into puncta surrounding the developing spore, here visualized using structured illumination microscopy. Cellular organization and dynamics: What are the design principles of bacterial cells? The phagocytosis-like process of engulfment is a hallmark of endospore formation in Bacillus subtilis and its relatives. Engulfment provides a unique example of phagocytosis in a bacterium and it is an ideal system to study protein localization and dynamics as well as membrane movement and fission. Engulfment depends on three protein complexes, the DMP complex that degrades peptidoglycan, the Q-AH complex that provides a Velcro-like attachment between the two cells and the peptidoglycan biosynthetic complex. Our goal is to elucidate the mechanisms by which bacterial cells mediate membrane movement and membrane fission, coordinate peptidoglycan degradation and synthesis, and how they complete membrane fission after engulfment and cell division. B. subtilis (green) invading and destroying a neighboring bacterial colony (red). What role do antibiotics play in microbial communities? The vast majority of commercial antibiotics are derived from secreted bacterial natural products. However, despite the clinical importance of these molecules and the pressing need to discover new antibiotics, it remains unclear exactly what role these molecules play in nature. The secreted metabolome of B. subtilis contains a wide array of well-characterized antibacterial and antifungal compounds, providing an ideal starting point to elucidate the specific roles each play in interspecies interactions and development. Undomesticated B. subtilis strains have the ability to invade and destroy neighboring colonies or to form mixed biofilms in a more cooperative mode of growth. Our goal is to identify the genes and secreted metabolites that mediate interspecies interactions, using the tools of analytical chemistry and imaging mass spectrometry (in collaboration with Pieter Dorrestein) as well as genetics and cell biology. B. subtilis cells before and after treatment with the cannibalism toxin SDP. Chemical Cell Biology: How do secreted metabolites and antibiotics affect bacterial cells? We have developed cell biological tools to characterize the cellular targets for secreted metabolites and antibiotics, in collaboration with Joe Pogliano. This approach, which we call cytological profiling, has allowed us to rapidly identify molecules that kill bacterial cells and to determine their likely mechanism of action. It is remarkably specific, as we can discriminate between the cytological effects of molecules that dissipate either the pH or or both components of the PMF, as well as the effects of a wide array of antibiotics with well-characterized activities. These studies provide insight into the mechanisms by which essential biological processes are coordinated to maintain the appropriate size and they have several biomedical applications. \n",
            "-------------------- Scraping faculty url 135/183 --------------------\n",
            "jim posakony  Biography Jim Posakony received his Ph.D. from the California Institute of Technology in 1982. Prior to coming to UCSD in 1985, he was a postdoctoral fellow at Harvard University. In 1986 he was named a Pew Scholar in the Biomedical Sciences. He was elected as a AAAS Fellow in 2011.  Research The driving interest of our lab is in the mechanisms by which cell-cell signaling pathways and transcriptional regulatory networks act to specify cell fates during animal development. The peripheral nervous system (PNS) of the fruit fly Drosophila melanogaster has proven to be a particularly accessible setting in which to investigate these problems, and we have been exploiting its advantages for many years. Our work is currently focused in the following areas: (1) Regulatory functions of the Notch cell-cell signaling pathway. The Notch pathway is one of a handful of signaling systems that are collectively responsible for the vast majority of conditional cell fate specification events in metazoan development. We have a long history of helping to elucidate the pathway's structure and functions, including the identification of its transducing transcription factor, Suppressor of Hairless [Su(H)]. Our present interests are in (a) understanding how Notch signaling switches Su(H) from being a repressor to an activator, and (b) unraveling structure-function relationships in the Bearded family of proteins, which act as antagonists of Notch signaling and are encoded by genes that are themselves targets of the pathway. (2) Dual enhancers controlling the expression of neural precursor genes. Neural precursor cells, such as those that give rise to the sensory bristles covering the fly's body, express a distinctive set of regulatory genes. We have found that many of these genes are served by duplicate enhancer modules, each of which drives the gene's expression specifically in neural precursors. We are interested both in how the modules encode this specificity, and in the functional advantages of the dual-enhancer control strategy. (3) Role of architecture in the function and specificity of transcriptional enhancer modules. Enhancers make use of particular regulatory \"codes\" (combinations of transcription factor binding sites) to generate their characteristic expression output. But these codes do not by themselves explain enhancer specificity the module's \"architecture\" (the arrangement and spacing of the factor binding sites) typically plays a critical role as well. We are investigating the mechanistic basis for this; in particular, how the constrained spacing of pairs of sites leads to synergies in the action of the cognate transcription factors. (4) Long-term evolutionary conservation of specific cis-regulatory elements in developmental control genes. It is now well established that changes in transcriptional regulatory connections are a major engine for the generation of developmental novelty during evolution. We have been looking at the other side of the coin by identifying and studying specific elements of transcriptional regulatory architecture that are ancient and have been conserved in orthologous genes from a deep ancestor. We are particularly interested in the functional basis of this long-term conservation; i.e., what distinctive regulatory capabilities are conferred by these special elements that would account for their durability? \n",
            "-------------------- Scraping faculty url 136/183 --------------------\n",
            "paul price \n",
            "-------------------- Scraping faculty url 137/183 --------------------\n",
            "jose pruneda-paz  Biography Dr. Pruneda-Paz received his Ph.D. in Biochemistry from the National University of Cordoba (Argentina), and carried out his postdoctoral research at The Scripps Research Institute and the University of California, San Diego.  Research Most organisms live in environmental conditions that oscillate with a 24 hour period and evolved an endogenous mechanism, known as the circadian clock, which coordinates the temporal organization of biological processes to optimal times of the day or night. Biological clocks allows an organism to anticipate the onset of periodic environmental changes providing an adaptive advantage that results in increased fitness and survival. In plants, the pervasiveness of the clock control extends to nearly every aspect of growth and development through an extensive transcriptional regulatory network that impose circadian oscillations to more than one third of the whole transcriptome. My laboratory is focused in studying the molecular mechanisms by which the circadian clock controls critical biological processes (i.e. outputs). In particular, we use plant-pathogen interactions as an experimental platform and aim to understand the transcriptional networks that control the circadian regulation of plant defense responses. Host-pathogen interactions are regulated by sophisticated mechanisms shaped by millions of years of coevolution. In fact, immune responses are regulated by the circadian clock not only in plants but also in other organisms such as flies and mice, suggesting that this is an evolutionary advantageous mechanism. Since the clock function and plant-pathogen responses are largely dependent on the regulation of gene expression, an extensive transcriptional network is anticipated at the interface of clock and defense signaling pathways. Using forward and reverse genetics, as well as systems approaches, we are primarily focused on identifying critical components of these transcriptional regulatory networks. Plant pests account for great losses in crop yield around the world, thus our findings may ultimately transform our understanding of plant microbial infections and how to treat them. Moreover, plant research made major impact in understanding critical mechanisms of great biological and biomedical relevance including RNA-silencing, DNA-methylation, proteasomal-control of protein stability and programmed cell death among others. Therefore, our research could potentially uncover highly conserved regulatory networks orchestrating immune defenses in other organisms as well. \n",
            "-------------------- Scraping faculty url 138/183 --------------------\n",
            "samara reck-peterson  Biography Samara Reck-Peterson received her Ph.D. from the Department of Cell Biology at Yale University and did postdoctoral research with Ron Vale at the University of California, San Francisco. She was an Assistant and Associate professor of Cell Biology at Harvard Medical School and the Associate Director of the Biological and Biomedical Sciences Graduate Program at Harvard Medical School. She joined the UCSD faculty in 2015. Dr. Reck-Peterson was the recipient of an NIH New Innovator Award.  Research Eukaryotic cells use microtubule tracks to move cargo over long distances. There are two types of molecular motors that move on microtubules: dyneins and kinesins. Dyneins move cargo towards the minus ends of microtubules, typically towards the cell interior. Kinesins move cargo in the opposite direction, generally towards the cell surface. Our lab is tackling three broad questions. How does the dynein motor work and how is it regulated? What are the rules governing transport in cells? Why do defects in transport cause disease? How does the dynein motor work and how is it regulated? Cytoplasmic dynein-1 is the only motor used for long-distance minus-end-directed microtubule-based trafficking in eukaryotic cells ranging from human neurons to the hyphae of filamentous fungi. Yet, it transports dozens, if not hundreds of different cargos. One of our labs goals is to understand how this large, multi-subunit, motor is regulated. Dynein has two regulators that are conserved and required for most (perhaps all) of its functions: the dynactin complex and Lis1/ Nudel. Current experiments in the lab are focused on determining how these regulators work. We use quantitative approaches to do this, including single-molecule imaging, cryo-electron microscopy (in collaboration with the lab of Andres Leschziner), proteomics, and live-cell imaging. What are the rules governing transport? In addition to cytoplasmic dynein-1, at least 15 kinesin motors (in humans) are responsible for moving cargo in the opposite direction as dynein. This small subset of motors is responsible for nearly all long distance transport in eukaryotic cells. What is the full list of dynein and kinesin cargos? What links the motors to different cargos? How is specificity achieved? Do all cargos have a distinct mechanism for recruiting motors? We are addressing these questions using two different discovery-based approaches: We use the filamentous fungus, Aspergillus nidulans, as a powerful genetic system to dissect microtubule-based transport. We have conducted screens to identify genes required for the transport of endosomes, peroxisomes, and nuclei. Current projects in the lab are characterizing these novel transport factors. We also use proteomic approaches to identify the dynein and kinesin transportome. Current efforts in the lab are focused on using proximity-dependent labeling in living human cells to identify the dynein proteome. Why do defects in transport cause disease? Mutations in every component of the transport systemtubulins (the building blocks of microtubules), dynein, kinesins, and proteins that regulate the motors or their trackscause neurological diseases. For example, mutations in cytoplasmic dynein-1 cause a peripheral neuropathy called Charcot Marie Tooth type 2, Spinal Muscular Atrophy (SMA), malformation of cortical development, and intellectual disability. Like some mutations in dynein, mutations in the dynein activator BICD2 also cause SMA. Mutations in a subunit of dynactin result in motor neuron disease and Perry syndrome, an early onset form of Parkinsons disease. Defects in microtubule-based axonal transport are also observed in other neurodegenerative diseases including Amyotrophic Lateral Sclerosis, Alzheimers and Huntingtons, suggesting that there may be additional links between dynein and neurological disease yet to be uncovered. One hypothesis is that neurodegenerative diseases can result from the compromised trafficking of particular cargos. Current projects in the lab are aimed at identifying which cargos or points of regulation are defective in disease states. \n",
            "-------------------- Scraping faculty url 139/183 --------------------\n",
            "pamela reinagel  Biography Pam Reinagel received her Ph.D. in Biochemistry at Harvard University. She conducted postdoctoral research in theoretical neuroscience with Christof Koch at the California Institute of Technology (Caltech), and in experimental neuroscience with R. Clay Reid at Harvard Medical School. Dr. Reinagel is the recipient of a Sloan Research Fellowship. She started her lab at UCSD in 2003.  Research The Reinagel lab studies perceptually guided selection of action in the visually behaving rat. Our primary experimental tool is quantitative visual behavior. Much of our effort is focused on building computational models to explain reward-motivated choice behavior in the face of sensory uncertainty. Lesions and pharmacology are used to probe neural mechanisms. One current focus is the speed-accuracy trade-off of rats as they perform different types of visual discriminations. A second focus is blindsight: identifying which visual behaviors are dependent on primary visual cortex in the rat, and characterizing visual decision-making and learning in V1-independent visual behaviors. \n",
            "-------------------- Scraping faculty url 140/183 --------------------\n",
            "scott rifkin  Biography Scott Rifkin received his PhD from Yale University and was a postdoctoral fellow at Harvard and an NIH postdoctoral fellow at MIT. He joined the faculty of UCSD in the summer of 2009.  Research My laboratory studies how stochastic, genetic, and environmental variation is filtered through development and physiology to produce phenotypic variation and how evolutionary forces shape that process. We work primarily at the level of gene expression, taking advantage of technologies for visualizing gene expression dynamics at single-transcript resolution, among others. Current projects in the lab include probing the dynamics and robustness of the genetic networks underlying sex-determination and intestinal specification in nematodes and investigating how natural variation affects the dynamic properties of signaling and regulatory networks in yeasts. Individual transcripts of xol-1 (yellow) and sdc-2 (red) in early C. elegans embryos. Blue spots are DAPI-stained nuclei. \n",
            "-------------------- Scraping faculty url 141/183 --------------------\n",
            "cory root  Biography Dr. Root received his B.A. in Biology and Chemistry from Whitman College and his Ph.D. in Biology from UC San Diego. He did postdoctoral research with Dr. Richard Axel at Columbia University. Dr. Root joined the faculty in 2016 and has been the recipient of the Elkins Memorial Award for an outstanding doctoral thesis in the field of Drosophila Neurobiology and a K99/R00 Pathway to Independence Award from the NIDCD.  Research Sensory systems detect features of the external world, which are represented in the activity of ensembles of neurons in the brain. My lab seeks to understand how sensory information is represented in the brain and how internal representations acquire meaning that can be used to inform behavior. Animals have evolved neural circuits that drive innate behavioral responses to environmental cues that have been essential to the survival of a species over long periods of evolutionary time. However, most stimuli do not have innate significance. Rather, meaning is acquired over the lifetime of an animal through learning. We exploit the sense of smell to investigate how sensory representations afford meaning for innate and experience-dependent behavior. We use endoscopic 2-photon calcium imaging to monitor neuronal activity, and molecular genetics, viral tracing and optogenetics to trace and manipulate the neural circuits of the mouse brain. The Olfactory System Olfactory perception is initiated upon the recognition of odorants by a large repertoire of receptors in the sensory epithelium. Neurons expressing a given receptor are randomly distributed within zones of the epithelium but project with precision to two spatially invariant glomeruli in the olfactory bulb. This invariant glomerular map in the bulb is transformed in the representations in higher olfactory centers. For instance, the piriform cortex receives distributed input from seemingly random sets of glomeruli in the olfactory bulb, and is implicated in learned associations. In contrast, the cortical amygdala receives topographically organized input from the olfactory bulb, and in recent work I have demonstrated that the neurons in this pathway mediate innate behaviors. The Cortical Amygdala Mediates Innate Behavior The higher order circuits mediating innate behaviors have not been well defined, however, anatomic tracing experiments implicated the cortical amygdala in determined behaviors. With Richard Axel at Columbia University, I devised experiments to directly determine if the cortical amygdala is involved in innate odor-driven behavior. We found that the stereotyped circuits from the olfactory bulb to the cortical amygdala are required for innate aversive and appetitive responses to odor. Moreover, we identified and manipulated the activity of these neurons by exploiting the promoter of the activity-dependent gene, arc, to express channelrhodopsin in neurons of the cortical amygdala activated by odors that elicit innate behaviors. Optical activation of these neurons leads to appropriate behaviors that recapitulate the responses to odors. Together these data demonstrate that the cortical amygdala contains distinct neurons that are necessary and sufficient for the generation of innate, odor-driven behaviors (Root et al., Nature 2014). Moreover, we have revealed that neurons mediating aversive and appetitive behaviors have distinct projections to downstream targets. The identification of neural circuits that elicit innate behaviors provides a foundation for dissecting sites of plasticity to afford behavioral flexibility. \n",
            "-------------------- Scraping faculty url 142/183 --------------------\n",
            "michael rosenfeld \n",
            "-------------------- Scraping faculty url 143/183 --------------------\n",
            "kaustuv roy  Biography Kaustuv Roy received his Ph.D. from the University of Chicago, where he completed postdoctoral training.  Research What are the physical and biotic processes that determine large scale spatial patterns of biodiversity? How do species and communities respond to climate change and other anthropogenic impacts and what are the ecological and evolutionary consequences of such responses? These and related questions form the focus of my research. We work primarily in marine systems using interdisciplinary approaches that combine ecological and genetic information from living species and populations with the temporal perspective afforded by the rich fossil record of marine invertebrates. Origin and maintenance of latitudinal diversity gradients The latitudinal gradient in diversity - the dramatic increase in the number of species and higher taxa from the poles to the tropics - is a global biodiversity pattern shared by both marine and terrestrial systems. Understanding the processes responsible for the origin and maintenance of such gradients is essential for informed management and conservation of biodiversity, especially in a world faced with rising temperatures, rampant overexploitation of natural resources and increasing urbanization. Yet despite over a century of work such processes remain poorly understood. We are working to better understand (i) the role of ecological processes in maintaining spatial gradients in diversity and (ii) the role of evolutionary and historical processes in generating such gradients. The former includes the role of temperature and other environmental variables in determining the number of species that can coexist in a given region as well as how species respond to changes in climate. As far as evolutionary dynamics are concerned, we are not only quantifying how origination and extinction rates change along latitude, a dynamic still poorly documented, but also how shifts in geographic distributions of taxa over time affect regional patterns of species richness. I am also involved in developing theoretical models to explore how in situ speciation and extinction rates interact with patterns of biotic interchange between regions to create large scale spatial gradients in biological diversity. Anthropogenic impacts on coastal marine ecosystems Intertidal and subtidal habitats harbor a diverse fauna of invertebrates, plants and fish yet these habitats are among those most impacted by human activities and least protected. While harvesting practices and other activities of an expanding coastal population all over the world are generally thought to be negatively affecting the species living in these habitats, little quantitative data still exist on the nature and extent of such impacts. In addition, how human impacts are affecting macroecological relationships or biogeographic patterns also remain poorly studied. We have initiated large-scale studies that will quantify the ecological consequences of harvesting (legal and illegal), urbanization and climate change along the California coast. We also plan to initiate similar studies along other coastlines in the near future. A more detailed description of our research and a complete list of our publications can be found at http://www.biology.ucsd.edu/labs/roy/ . \n",
            "-------------------- Scraping faculty url 144/183 --------------------\n",
            "alistair russell  Biography Alistair Russell received his PhD from the University of Washington in the laboratory of Joseph Mougous, where, supported by the NSF, he studied interbacterial interactions driven by the type VI secretion system, using a combination of genetic, informatic, and biochemical tools. For his postdoctoral research, he worked as a Merck Fellow of the Damon Runyon Cancer Research Foundation, in the laboratory of Jesse Bloom at the Fred Hutchinson Cancer Research Center, trying to use novel single-cell methodologies to unravel events precipitating induction of an interferon response by influenza A virus. He joined the Molecular Biology faculty at UCSD in 2019.  Research Nearly every cell in the human body has the capacity to respond to a viral infection with the production of interferonscritical components of cell intrinsic immunity that drive resolution of disease, and, unfortunately, much of the resultant immunopathology. Unsurprisingly, viruses circumvent, co-opt, or otherwise subvert the induction of an antiviral state. Regardless, no defense (or offense) is perfect, and viral populations almost invariably induce interferon in a small fraction of infected cells. Some of this rarity of response is driven by rare viral failure to suppress interferon, in part due to the extreme variation evinced by many viral populations. However, the rarity of response is also partially explained by intrinsic host mechanisms suppressing spurious activationseeking to avoid the resultant cost from an overly sensitive triggering of inflammatory processes. My lab seeks to study the rarity of the interferon response during viral infection using influenza A virus as a model, using single-cell transcriptomics, single virion genomics, flow cytometry, population-based sequencing, and fine-grained genetic analyses to delineate the molecular events that proceed engagement of innate immune signaling pathways in responding cells. With these data in hand, we will seek to better model disease processes, with the hope that our work may help development of therapeutic interventions or more robust vaccine responses. \n",
            "-------------------- Scraping faculty url 145/183 --------------------\n",
            "oliver ryder  Biography Oliver A. Ryder received his Ph.D. from UCSD. He is the geneticist and recipient of the Kleberg Genetics Chair at the Center for Reproduction of Endangered Species, Zoological Society of San Diego. He is a member of Specialist Groups of the Species Survival Commission of World Conservation Union: Equid Specialist Group, Captive Breeding Specialist Group, and Re-introduction Specialists Group. Dr. Ryder is an associate editor of the Journal of Heredity.  Research Challenges involved in conserving biological diversity include addressing the numerous scientific issues limiting our understanding of the Earth's biota. Speciation and species diversity, the role of genetic variation in population persistence, its rate of generation and loss in different taxonomic groups and under different ecological conditions, as well as the development of tools for addressing these questions are all significant topics in biology pertinent to conservation issues. The genetics group at the CRES (Center for Reproduction of Endangered Species - a division of the Zoological Society of San Diego) undertakes comparative cytogenetic and molecular genetics studies involving a diverse array of species suitable as models for investigating relative rates of evolution among different taxa, population structure and systematics. Additionally, research pertinent to planning and management for selected mammalian, reptilian and avian species of conservation concern is undertaken utilizing appropriate techniques, including the development and application of new methodologies. Current emphases include molecular evolutionary and population studies involving analysis of nuclear microsatellite allelic variation and nucleotide sequence analysis of mitochondrial genomes. Assessment, planning, monitoring, and management of small populations for conservation will benefit from application of more effective methods of collection and analysis of genetic data. Black rhinoceros male in Masia Mara Reserve in Kenya. Increased knowledge of genetics of black rhinos contributes to monitoring and recovery efforts for this endangered species. An increased understanding of many threatened species at the cytogenetic and molecular genetic level will contribute to gene pool conservation strategies and management as well as establishing a broader knowledge of the dynamics of genome evolution. Located in lab facilities on the grounds of the San Diego Zoo, our studies are facilitated by unique resources, including cell lines and other biological samples that are available as a result of the animal collections of the San Diego Zoo and Wild Animal Park as well as through associated field activities of CRES staff. \n",
            "-------------------- Scraping faculty url 146/183 --------------------\n",
            "alan saghatelian \n",
            "-------------------- Scraping faculty url 147/183 --------------------\n",
            "milton saier  Biography Milton Saier received his Ph.D. from UC Berkeley and was a postdoctoral fellow at Johns Hopkins University.  Research Our laboratory has three primary research interests, one concerned with transcriptional and metabolic regulation in bacteria, a second with transport protein evolution, and a third with the recently identified process of transposon-mediated directed mutation. We also maintain the IUBMB-approved Transporter Classification Database, TCDB , which classifies transport systems found in all living organisms on Earth into five categories: class, subclass, family, subfamily and transport system. Our laboratory takes a multidisciplinary approach to science, using biochemical, molecular genetic, physiological, and computational approaches. Directed mutation is a proposed process that allows mutations to occur at higher frequencies when they are beneficial than when detrimental. Until recently, the existence of such a process has been controversial. However, we have described a novel mechanism of directed mutation mediated by the transposon, IS 5 in Escherichia coli. crp deletion mutants mutate specifically to glycerol utilization (Glp+) at rates that are enhanced by glycerol or the loss of the glycerol repressor (GlpR), depressed by glucose or glpR overexpression, and RecA-independent. Of the four tandem GlpR binding sites ( O1O4 ) upstream of the glpFK operon, O4 specifically controls glpFK expression while O1 primarily controls mutation rate in a process mediated by IS 5 hopping to a specific site on the E. coli chromosome upstream of the glpFK promoter. IS 5 insertion into other gene activation sites is unaffected by the presence of glycerol or the loss of GlpR. The results establish an example of transposon-mediated directed mutation, identify the protein responsible and define the mechanism involved. Most recently, we have identified two additional operons in E. coli that appear to be subject to transposon-mediated directed mutation: The flhDC flagellar master switch operon controlling motility and the fucAO operon controlling L-fucose and propanediol utilization. Our efforts have revealed three basic mechanisms of transcriptional control concerned with catabolite repression/activation in bacteria. Two of these occur in E. coli, and one occurs in B. subtilis. In E. coli, two DNA binding proteins, the cyclic AMP receptor protein (Crp) and the catabolite repressor/activator (Cra) protein, mediate transcriptional regulation of hundreds of genes encoding key enzymes of carbon and energy metabolism. Virtually every pathway of carbon metabolism is subject to these regulatory constraints. Crp generally controls the initiation of exogenous carbon source metabolism and senses cytoplasmic cyclic AMP levels. These levels are controlled by complex mechanisms, involving phosphorylated proteins of the sugar-transporting phosphotransferase system (PTS). Cra generally controls the flux of carbon through metabolic pathways and senses cytoplasmic metabolite concentrations. Cra usually controls gene expression independently of Crp, but it sometimes acts cooperatively with or antagonistically to Crp, depending on the target gene. It thereby mediates catabolite repression of catabolic operons by an indirect mechanism. Most recently, we have demonstrated that Cra regulates growth by activating expression of the crp gene. In B. subtilis and other Gram-positive bacteria, a metabolite-activated protein kinase phosphorylates a serine residue in a protein of the PTS called HPr. Phosphorylated HPr allosterically controls the activities of many target proteins (transport proteins, enzymes and transcription factors). It thereby controls the cytoplasmic concentrations of inducers as well as the activities of transcription factors that mediate catabolite repression. We are coming to realize that the mechanisms of catabolite control are very different for phylogenetically divergent bacteria. Phylogenetic analyses of integral membrane transport protein sequences have yielded a plethora of information about the times of appearance, the routes of evolution, and the relative rates of divergence of the proteins and protein domains which comprise various families of transport systems. These studies have shown that families of transport proteins of similar topology have evolved independently of each other, at different times in evolutionary history, using different routes. They have also revealed extensive domain shuffling in some such families but not in others. The probable means by which energy coupling became superimposed on transport during the evolutionary process has also come to light. \n",
            "-------------------- Scraping faculty url 148/183 --------------------\n",
            "moselio schaechter \n",
            "-------------------- Scraping faculty url 149/183 --------------------\n",
            "immo scheffler  Biography After receiving his Ph.D. from Stanford University, Immo Scheffler was a Helen Hay Whitney Fellow at Harvard University and the Pasteur Institute, Paris. He is the recipient of an Alexander von Humbolt Award for Senior U.S. Scientists. At UCSD, Professor Scheffler has been honored with a Chancellor's Associates Excellence in Teaching Award, an Alumni Association Teacher of the Year Award, and a teaching award from the Academic Senate. Professor Scheffler was the first holder of the Paul D. Saltman Chair in Science Education at UCSD (2001-2004). Editorial boards include the Journal of Biological Chemistry. Dr. Scheffler was recently elected a Fellow of the American Association for the Advancement of Science (AAAS).  Research My laboratory was closed after 35 years of operation. I continue to have an interest in all aspects of mitochondrial biology, and a second, updated edition of my book MITOCHONDRIA was published recently. Collaborations continue with laboratories at UCSD (Taylor, Perkins), at UC Irvine (Wallace) and Neuzil (Australia). \n",
            "-------------------- Scraping faculty url 150/183 --------------------\n",
            "eric schmelz  Biography Eric Schmelz completed his PhD within the Center for Insect Science/Plant-Insect Interactions Group at the University of Arizona, Tucson. He then carried out his postdoctoral and research scientist studies at the USDA-Agricultural Research Service located at the University of Florida, Gainesville. He joined the Division of Biological Sciences faculty at U.C. San Diego in July 2014.  Research Ive have always been fascinated by the existence of specific biochemicals that that significantly govern interactions between organisms. Plants, arthropods and microbes are the most species rich life forms on earth. As primary producers, photosynthetic organisms are the ultimate source of most fixed carbon and under significant pressure to relinquish these resources. For reproductive success, plants must balance both growth and defense against a myriad of herbivorous arthropods and pathogens. These real-time interactions are of relevance to our global food supply and are underpinned by genetics, biochemistry, physiology, and behavior. We utilize maize ( Zea mays ) and bean ( Phaseolus & Vigna spp.) to examine dynamic inducible innate immune responses that limit losses against attacking organisms. In response to herbivory by generalist lepidopteran pests, plants rapidly initiate the biosynthesis and release of low molecular weight Volatile Organic Compounds (VOCs). Prominent in this VOC signature are monoterpenes, homoterpenes and sesquiterpenes that serve as indirect plant defenses by enabling the attraction of predators, parasitioids, and other natural enemies. How is this process enabled? We examine the mechanistic role of exogenous elicitors, termed Herbivore Associated Molecular Patterns (HAMPs), and endogenous signal transduction cascades that regulate insect-induced plant responses. Plant recognition and elicitation of inducible volatiles following herbivory in beans ( Phaseolus and Vigna spp.) is mediated by femtomolar amounts of the peptide elicitor termed inceptin, derived from chloroplastic ATP synthase -subunit proteolysis. Specialist herbivores evade detection by converting inceptin into a potent antagonist of elicitation. Molecular level details of herbivore perception by plants are essential for the understanding defense and susceptibility in crops. In contrast to foliar herbivores, maize fungal pathogens elicit the biosynthesis of non-volatile acidic terpenoid phytoalexins with direct antibiotic activity. The most highly inducible transcripts following pathogen attack encode the -macrocarpene synthases (ZmTps) 6/11 and the ent -copalyl diphosphate synthase termed Anther Ear 2 (ZmAn2). Down stream predicted enzymatic products of ZmTps6/11 and ZmAn2, termed zealexins and kauralexins respectively, are the predominant phytoalexins of mature maize plants. These recent discoveries raise many questions including: what exogenous and endogenous signals strongly promote maize phytoalexin biosynthesis? What are the roles of these terpenoids in plant protection? plant signaling? rhizosphere interactions? allelopathy? autotoxicity? and human health? Given that approximately 150,000 square of maize is annually planted in the U.S. and that fungal attack is aggressive during senescence it is of interest to know, where do these compounds go? what is their fate in the environment? (A) Maize plants must defend against significant biotic threats that include fungal pathogens [(B) southern leaf blight, Cochliobolus heterostrophus ; (C) stalk rot, Fusarium graminearum ; (D) ear rot, Aspergillus flavus ) and stem boring lepidoptera [(E) European corn borer: Ostrinia nubilalis ]. All of these stimuli result in localized zealexin and kauralexin production to limit microbial spread. Mass spectrometry-based metabolomic profiling of phytohormones and defenses is a powerful tool to aid in the discovery of novel metabolites that additionally serve as signals and plant protectants. We utilize biochemical discovery approaches based on the coupling of separations with rapid activity based bioassays, NMR structure elucidation, mass spectrometry for the plant metabolomic analyses of novel induced defenses and signals, genetic resources to test candidate pathways, global expression analyses for co-regulated processes, expression quantitative trait loci (eQTL) for gene discovery, transient and heterologous expression of enzymes for product analyses, and other tools to uncover previously hidden pathways and mechanisms underlying plant-mediated interactions with their biotic and abiotic environment. \n",
            "-------------------- Scraping faculty url 151/183 --------------------\n",
            "julian schroeder  Biography Julian Schroeder did his PhD research at the Max Planck Institute for Biophysical Chemistry with Erwin Neher and was a von Humboldt postdoctoral fellow at UCLA School of Medicine. He received awards, including the Presidential Young Investigator Award (NSF), the ASPB Charles Albert Shull Award (1997), a DFG Heinz-Maier-Leibnitz Prize, the Blasker Award in Environmental Science, is Churchill Overseas Fellow at Cambridge University and with collaborators shared the Cozzarelli Prize from PNAS (2010) and a top 10 breakthrough of the year selected by Science (2009). He has served on several advisory boards, including Co-Director of the Food and Fuel for the 21st Century Center. He was von Humboldt Fellow at the MPI for Biochemistry, visiting Professor at the ETH Zurich and is a member of the U.S. National Academy of Sciences, a Fellow of AAAS and the German National Academy of Sciences Leopoldina.  Research Identifying the basic molecular mechanisms by which plants respond to and mount resistance to environmental stresses is fundamental to understanding stress resistance mechanisms to these abiotic in plants and is an important goal for developing future strategies for engineering stress resistance in plants. Several abiotic stress mechanisms that we are characterizing are directly linked to water, including drought stress-induced signal transduction mechanisms, salinity resistance mechanisms and how plants respond to the continuing rise in the atmospheric CO 2 concentration. Our laboratorys research is directed at the signal transduction mechanisms and pathways that mediate resistance to environmental (abiotic) stresses in plants, in particular responses to elevated CO 2 , drought, salinity stress, and heavy metal stress. These abiotic stresses have substantial negative impacts and reduce global plant growth and biomass production. These environmental stresses are also relevant in reference to climate change and to expanding available arable land to meet the food and energy needs of the growing human population. Our research is elucidating the molecular and cell biological stress-induced signal transduction cascades in higher plant cells, examining the chain of events by which plant cells respond to elevated CO 2 , the drought stress hormone abscisic acid and salinity stress to mount specific resistance and adaptation responses. We have developed and adapted interdisciplinary and systems biological approaches to guard cells, which control water loss and CO 2 intake in plants and which have become a key model system for understanding dynamic cellular signal transduction and ion channel functions in plants. Stomatal pores in the epidermis of leaves allow CO 2 influx into leaves from the atmosphere and also mediate transpirational water loss of plants (see figure). Two guard cells surround each pore and control the opening and closing of stomata. In guard cells, cell biological, molecular, patch clamp and time-resolved calcium imaging studies on genetic signaling mutants in Arabidopsis are allowing us to identify and characterize stress-induced signal transduction mechanisms and cascades. We are combining these analyses with new genomic, systems, bioinformatic and proteomic approaches towards discovering new signaling mechanisms and principles. We have identified CO 2 binding proteins and early CO 2 signal transduction mechanisms, including ion channels in guard cells through which elevated CO 2 closes stomatal pores. We have recently identified new early signal transduction mechanisms and contributed to the characterization and co-identified receptors for the plant stress hormone abscisic acid and have obtained molecular genetic, cell biological, genomic, biophysical whole plant physiological evidence for new genes and mechanisms in guard cells that reduce water loss of Arabidopsis during drought. A second effort in the lab focuses on identifying genes that mediate salt (sodium) stress resistance and heavy metal uptake and detoxification in plants. In this research we identified the plant HKT transporter family and showed its central role in mediating salinity resistance in the reference plant, Arabidopsis thaliana . Research on the staple crops rice and wheat is showing that this same HKT transporter mechanism plays a major role in determining salinity resistance. HKT gene-focused molecular breeding efforts are indicating major improvements in yield, illustrating how basic Arabidopsis research is leading to innovation in agriculture. Our research into heavy metal stress led to the parallel discovery of the genes encoding the central heavy metal detoxification enzymes in plants, phytochelatin synthases. Furthermore recent research has identified the long sought family of transporters that mediate heavy metal accumulation in plant vacuoles. These basic research advances can provide key tools for avoiding toxic heavy metal and arsenic accumulation in edible plant tissues, a problem facing millions of people today leading to cancer and other diseases. Furthermore, these basic research advances can contribute key tools for engineering plants for environmental remediation (bioremediation) by removal of heavy metals from soils. Members in our lab are being trained in interdisciplinary and systems biological techniques while pursuing individual research projects. \n",
            "-------------------- Scraping faculty url 152/183 --------------------\n",
            "terrence sejnowski  Biography Dr. Sejnowski received his Ph.D. in physics from Princeton University. He was a postdoctoral fellow at Princeton University and the Harvard Medical School. He served on the faculty of Johns Hopkins University and was a Wiersma Visiting Professor of Neurobiology and a Sherman Fairchild Distinguished Scholar at Caltech.  Research The goal of my laboratory is to discover the principles linking brain mechanisms and behavior. The laboratory uses both experimental and modeling techniques to study the biophysical properties of synapses and neurons and the population dynamics of large networks of neurons. New computational models and new analytical tools have been developed to understand how the brain represents the world and how new representations are formed through learning algorithms for changing the synaptic strengths of connections between neurons. Some of the questions that my laboratory is exploring include: How do neurons communicate at synapses? Monte Carlo models of molecular diffusion and interactions at synapses are being used to understand the mechanisms underlying the release of neurotransmitter from presynaptic terminals, the activation of receptors on the postsynaptic neuron and subsequent biochemical pathways that lead to short-term and long-term changes in the synapse. Single molecules at synapses are probed with 3-D random access 2-photon microscopy to locate important molecules and follow them as they diffuse within a spine. What regulates the flow of information into the cerebral cortex and between cortical areas? My laboratory has shown that cortical neurons are capable of initiating spikes precisely and reliably. We have used detailed models of neurons to explore the role of spike time synchrony in reliably transmitting information between populations of neurons and in attentional modulation of firing rates. Fast-spiking inhibitory neurons are studied in vitro to examine their role in synchronizing other neurons. EEG and fMRI recordings have been analyzed with Independent Component Analysis (ICA), an algorithm for blind source separation developed in my lab, to explore the global flow of information in the cortex. Why do we sleep? As we fall asleep, the patterns of cortical activity become more spatially and temporally coherent, especially at low frequencies, and during the night the brain cycles between REM sleep and slow wave sleep. The mechanisms underlying these sleep rhythms are being explored with large-scale network models of neural populations. New algorithms have been found to automatically classify sleep states from single EEG recordings. \n",
            "-------------------- Scraping faculty url 153/183 --------------------\n",
            "gerald shadel \n",
            "-------------------- Scraping faculty url 154/183 --------------------\n",
            "reuben shaw \n",
            "-------------------- Scraping faculty url 155/183 --------------------\n",
            "jonathan shurin  Biography Dr. Shurin studies the causes and consequences of variation in species diversity, the flow of energy between producers and consumers, and the impacts of predators on the functioning of ecosystems.  Research My lab studies the connections between population, community and ecosystem ecology in aquatic environments. We are particularly interested in the flow of energy between trophic levels, the impact of consumers on their resources, and the control of species diversity. We use broad geographic comparisons, mesocosm experiments and theory to address these questions. My students have worked in communities of freshwater plankton, marine fishes and intertidal algae and invertebrates. \n",
            "-------------------- Scraping faculty url 156/183 --------------------\n",
            "doug smith  Biography Doug Smith received his Ph.D. from Stanford University and was a postdoctoral fellow at the Max-Planck-Institut fr Virusforshung in Tbingen. He was named Outstanding Educator of America in 1973.  Research Our studies of DNA replication center on understanding initiation of rounds of replication in procaryotic cells: the structural requirements for the origin of replication (oriC), the role of GATC methylation state in initiation, and the mechanisms of control of initiation. We have recently shown that Dam methylation is required for precise timing of initiation of rounds of DNA replication; current emphasis is on the mechanism involved. Other related projects include delineation of control circuits regulated by the DnaA protein and state of GATC methylation and studies of gram-negative origins from bacteria which have no GATC methylation. We are also engaged in projects concerned with genome (large and small) mapping and sequencing, and concomitant computer database handling and sequence analysis. \n",
            "-------------------- Scraping faculty url 157/183 --------------------\n",
            "laurie smith  Biography After receiving her Ph.D. from University College, London, Laurie Smith conducted postdoctoral study at Stanford University and the UC Berkeley/USDA Plant Gene Expression Center.  Research The shapes and positions of plant cells are fixed by cell walls. Consequently, plant cells must be formed during development in the positions where they are needed, requiring exquisite spatial regulation of the cell cycle and cell division planes. We are studying mechanisms responsible for orienting division planes in plant cells. Much of our past work on division plane determination has focused on TANGLED, a protein that is localized at the division plane throughout prophase, mitosis and cytokinesis and functions to promote proper division plane orientation (Figure 1; Walker et al., 2007). Recent studies on TAN have elucidated requirements for its localization throughout the cell cycle, revealing that a series of distinct mechanisms involving different parts of TAN protein function to maintain TAN at the division site as cells progress through division (Figure 1; Rasmussen et al., 2011). These studies have shown that division site maintenance is plant cells is a more dynamic process than previously envisioned. Figure 1. Schematic of cytokinesis in a somatic plant cell. TANGLED protein (blue) co-localizes with the preprophase band (PPB), predicting the future division plane during prophase. Unlike the PPB, TAN remains at the cortical division site throughout mitosis and cytokinesis, where it functions to promote proper placement of the new cell wall. Maintenance of TAN at the division site requires different parts of TAN protein and different interacting factors at different stages of the cell cycle. Our current work focuses on orientation of asymmetric cell divisions. In plants, as in other eukaryotes, asymmetric divisions are associated with pattern formation during embryogenesis, establishment of new cell lineages, and the formation of specialized cell types. In all of these processes, developmental asymmetry is closely tied to division polarity, but little is known about polarizing cues that orient asymmetric divisions in plants, or how cells respond to them to establish asymmetric division planes. Our studies in this area have focused on the formation of subsidiary cells of maize stomata, which are created by asymmetric divisions of subsidiary mother cells (SMCs, Figure 2). We have discovered a cooperatively acting pair of receptor-like proteins, PAN1 and PAN2, that are implicated in transmission or amplification of a cue from adjacent guard mother cells that polarizes the asymmetric divisions of SMCs (Cartwright et al., 2009 and unpublished work on PAN2). ROP GTPases act downstream of PAN1 to polarize SMC divisions (Humphries et al., 2011). Ongoing work utilizes proteomic and molecular genetic approaches to identify additional components of the SMC division polarity pathway. Figure 2. Stomatal development in wild type and pan mutant maize. All stages are illustrated schematically; Toluidine Blue O-stained mature epidermis is also shown for wild type (top) and pan mutant (bottom). Subsidiary cells form via asymmetric divisions of subsidiary mother cells (SMCs), which are polarized by a cue from the adjacent guard mother cell (GMC). PANs function together with ROP GTPases to polarize premitotic SMCs. \n",
            "-------------------- Scraping faculty url 158/183 --------------------\n",
            "nicholas spitzer  Biography Nick Spitzer received his Ph.D. from Harvard University and was a postdoctoral fellow at Harvard and University College, London. He joined the faculty in 1972 and has been the recipient of a Sloan Fellowship, a Javits Neuroscience Investigator Award and a Guggenheim Fellowship. He was founding editor-in-chief of BrainFacts.org , and is a fellow of the American Association for the Advancement of Science, a member of the American Academy of Arts and Sciences and the National Academy of Sciences and co-director of the Kavli Institute for Brain and Mind.  Research The brain is capable of a rich variety of forms of plasticity, changing its structure and function in response to changes in the environment. We study neurotransmitter switching, a newly appreciated form of neuroplasticity in which neurons change the transmitters that they make and release in response to sustained sensory or motor activity. Transmitter identity shifts from excitatory to inhibitory or vice versa, with matching changes in postsynaptic receptors, both in the developing amphibian nervous system and in the adult rodent brain. Perhaps unsurprisingly the reversal of the sign of synaptic transmission is accompanied by changes in the animals behavior. Our work is focused on understanding the cues that stimulate neurotransmitter switching in the developing and adult mouse brain, the prevalence and forms of this plasticity, and its functions. We want to learn the molecular mechanisms that regulate neurotransmitter switching and the mechanisms by which the appropriate, matching transmitter receptors are expressed on postsynaptic cells. We are investigating neurotransmitter switching in response to normal physiological stimuli and to aversive stimuli such as stress that lead to neurological disorders like depression and schizophrenia. Our recent studies have demonstrated the roles of sensory stimuli and direct manipulations of activity in neurotransmitter switching in the developing Xenopus hypothalamus, brainstem, and accessory olfactory bulb (Dulcis & Spitzer, 2008; Demarque & Spitzer, 2010; Dulcis et al., in review) and in the adult rat hypothalamus (Dulcis et al. 2013) that change specific behaviors. Ongoing projects address the role of motor activity in neurotransmitter switching in the adult mouse midbrain and hippocampus that may regulate procedural learning and neurogenesis. We are also investigating neurotransmitter switching generated by environmental mouse models of autism spectrum disorders. \n",
            "-------------------- Scraping faculty url 159/183 --------------------\n",
            "sarah stockwell \n",
            "-------------------- Scraping faculty url 160/183 --------------------\n",
            "chih-ying su  Biography Chih-Ying Su received her Ph.D. in Neuroscience from Johns Hopkins University School of Medicine and carried out her postdoctoral research with Dr. John Carlson at Yale University. Chih-Ying has been the recipient of a Mette Strand Young Investigator Award.  Research The sense of smell is one of the most intriguing senses: animals detect and discriminate an almost limitless number of odors in order to find food and mates while avoiding predators. Our research is focused on addressing how olfactory receptor neurons (ORNs) process odor information because these neurons are the primary source of sensory input. We are especially interested in how compartmentalization allows insect ORNs to process information through ephaptic interaction. We also investigate how neuromodulation can act on ORNs to regulate responses and influence odor-guided behaviors in Drosophila . To understand the functional significance of ORN compartmentalization, we recently developed a new recording paradigm to demonstrate that ephaptic interaction is sufficient to mediate lateral inhibition between ORNs housed in the same sensillum ( Zhang et al., 2019 ). This unconventional lateral inhibition has a profound impact on neuronal spike activity and can thus regulate information processing at the earliest stages of olfactory coding. Furthermore, combining electrophysiology with a novel method that permits high-quality ultrastructural preservation for 3D electron microscopy of genetically labeled tissues ( Tsang et al, 2018 ), we found that lateral inhibition is predominantly asymmetric between compartmentalized ORNs, and that this feature arises from the size difference between the neurons ( Zhang et al., 2019 ). Our study indicates that not all ORNs are equal in their circuit function, and that genetically predetermined morphometric differences between compartmentalized ORNs underlie their functional disparity. Our recent studies also identified the courtship-promoting Or47b ORNs as a neural substrate for a fly reproductive hormone, named juvenile hormone. We showed that the sensitivity of Or47b ORNs to an aphrodisiac pheromone is low in immature young males but high in older males at peak fertility ( Lin et al., 2016 ). This age-dependent sensitization permits coordination of courtship activity with reproductive maturity, thereby providing a means to increase male reproductive fitness. Building on these findings, we further identified a signal amplification mechanism by which olfactory responses can be amplified in insect ORNs in response to juvenile hormone signaling ( Ng et al., 2019 ). Surprisingly, this amplification mechanism only applies to courtship-promoting ORNsO47b and Ir84a neuronsamong the 50 ORN types in the antenna. Together, our finding reveals the surprising complexity of insect olfactory transduction: not all ORNs expressing receptors of the same class employ identical transduction mechanisms. Rather, a common transduction channel can function downstream of two distinct classes of receptors. \n",
            "-------------------- Scraping faculty url 161/183 --------------------\n",
            "suresh subramani  Biography Suresh Subramani received his Ph.D. from the University California (UC), Berkeley. He joined the faculty at UC San Diego in 1982, after completion of post-doctoral work in the laboratory of Nobel Laureate, Paul Berg, at Stanford University (1979-81). He is the recipient of many honors and awards and has been involved in many leadership roles in academia and in the biotechnology industry. His service to UC includes his role as the Executive Vice Chancellor (2010-16), Assoc. Vice Chancellor (2009-10), Interim Dean (2006-07) and Chair (1999-2000). He is currently a Distinguished Professor (Emeritus), the Global Director of the Tata Institute for Genetics and Society (TIGS, at UC San Diego, USA and at inStem, Bangalore, India) and holds the Tata Chancellors Endowed Professorship in Molecular Biology at UC San Diego.  Research PEROXISOME HOMEOSTASIS IN YEAST AND HUMANS We are interested in the problem of organelle homeostasis, using peroxisomes as the model organelle. Like all other subcellular organelles, peroxisomes and peroxisomal functions are indispensable for human survival. Our studies on peroxisome homeostasis examine how peroxisomes are assembled (biogenesis) and destroyed (turnover or pexophagy), as well as the coordination of these two processes within cells. We are also interested in the role of peroxisome biogenesis and turnover in disease states. Peroxisome biogenesis: Work done in this and other labs has uncovered about 36 peroxins, encoded by PEX genes, involved in peroxisome biogenesis. Many proteins involved in this process are conserved in evolution from yeast to man, and inactivating mutations in at least half of these peroxins cause fatal human disorders (Zellweger syndrome, rhizomelic chondrodysplasia punctata and infantile Refsum disease). Work on peroxisome biogenesis is summarized in a review (Farr et al., EMBO Reports, 2018, PMID: 30530632). Current work focuses on the trafficking of peroxisomal membrane proteins via the endoplasmic reticulum, their budding into pre-peroxisomal vesicles (ppV) and the proteins and mechanisms involved in this ppV budding. Peroxisome degradation: Autophagy is a process in which cells eat themselves, especially under starvation conditions, and recycle cellular building blocks such as amino acids, lipids and sugars. Autophagy plays a key role in development, aging, neurodegeneration, cell death, cell survival, and innate immunity. While autophagy generally degrades cargoes non-selectively, it can also be adapted to degrade and recycle selective cargoes, including organelles and protein aggregates. Pexophagy is a turnover pathway in which redundant, damaged or unnecessary peroxisomes are selectively degraded by the autophagy machinery in response to specific environmental cues. Other selective autophagy pathways include mitophagy, ribophagy, ER-phagy, micronucleophagy and the Cvt pathway in yeasts. Progress in the field of selective autophagy may be found in a review (Farr et al., Nature Rev. Mol. Cell Biol., 2016, PMC5549613). Current work focuses on the selective autophagy pathways and the manner in which selective autophagy receptors function. \n",
            "-------------------- Scraping faculty url 162/183 --------------------\n",
            "gurol suel  Biography Dr. Sel received his PhD in Molecular Biophysics from the University of Texas, Southwestern Medical Center in the laboratory of Dr. Rama Ranganathan and conducted his postdoctoral studies in the laboratory of Dr Michael Elowitz in the Department of Biology and Applied Physics at the California Institute of Technology.  Research Our laboratory discovered ion channel-mediated electrical signaling among bacteria that reside in biofilm communities. We are investigating how these action potentials in biofilms give rise to emergent collective behaviors in biofilms and beyond. Accordingly, we integrate quantitative biology approaches with mathematical modeling and theory to identify principles of bacterial organization and coordination. \n",
            "-------------------- Scraping faculty url 163/183 --------------------\n",
            "xin sun  Biography Xin Sun received her Ph.D. from the Department of Biology at Yale University and postdoctoral training with Dr. Gail Martin at UCSF. She was on the faculty in the Genetics Department at University of Wisconsin-Madison from 2002 to 2016, when she joined the UCSD faculty. Dr. Sun was the recipient of a Burroughs-Wellcome career award, a March of Dimes Basil OConner award, and Romnes Faculty Fellowship by Wisconsin Alumni Research Foundation.  Research The building of an organ starts from an amorphous cluster of cells, and proceeds following an elaborate blueprint of coordinated proliferation, morphogenesis and differentiation. Organ function relies on the harmonization with other tissues, the infusion by blood and immune cells, and the wiring by the nervous system. We employ cutting-edge technologies, including single-cell transcriptomics and epigenomics, genome editing, whole-organ clearing and imaging, to delineate the robustness that dictates normal organ function, and uncover the imperfections that initiate pathogenesis. At the center of our investigation is the lung, an organ that is vital at first breath. It is the setting for many fetal, pediatric and adult diseases with unknown cause and no cure. We use disease modeling as a starting point for fundamental discoveries in developmental biology, stem cell biology and the neural-immune-physiology interface. Consider the lung as a sensory organ A mouse airway outlined by an epithelial marker for club cells (green), annotated by clusters of pulmonary neuroendocrine cells (magenta) which act as airway sensors. At resting, an average person inhales/exhales 5-8 litters of air. This air may vary in oxygen content, carry allergen, pollutants or pathogens. Each of these signals would trigger a specific set of responses. For example, allergen causes asthma while intermittent hypoxia causes pulmonary hypertension. How this exquisite specificity is established is poorly understood. We recently found that pulmonary neuroendocrine cells, which represents less than 1% of epithelial cells in the lung, function as sensors on the airway wall (Branchfield et al., Science , 2016) . These cells are neuronally innervated, and act through secreting potent neuropeptides, neurotransmitters and amines. We found that overactivation of these cells leads to heightened baseline immune responses. Furthermore, they form neural-immune niches with residential immune cells such as innate lymphoid group 2 cells (Sui et al., Science , 2018) . Together, these niches are essential for mediating allergen-induced asthmatic responses. Study of the pulmonary neuroendocrine cells have also led us to investigate the molecular nature of the neural circuit that controls lung function. We are teaming up with neurobiologists, genomicists and physiologists to map the neural circuit that originates from and returns to the lung. Our goals are to understand which aspects of lung function is controlled by the neural circuit, what underlie the specificity between inputs and outputs, and how to manipulate the circuit to restore lung function in disease settings. Mechanisms and long-term outcomes of prematurity In a normal adult, the gas-exchange surface area is estimated to be ~1,000ft2, equivalent to half of a tennis court. This surface is packed into our chest in the form of ~480 millions of gas-exchange units called alveoli. Approximately 95% of the gas-exchange surface area is built in the early periods after birth in a process called alveologenesis. Disruption of the final steps of lung development results in alveoli simplification, as is seen in a large proportion of premature infants diagnosed with bronchopulmonary dysplasia (BPD). BPD is often associated with lifelong breathing deficiencies, pulmonary hypertension and accelerated decline in respiratory capacity later in life. To date, a majority of studies of alveologenesis rely on two-dimensional (2D) analysis of tissue sections. Given that an overarching theme of alveologenesis is thinning and extension of the epithelium and mesenchyme to facilitate gas exchange, often only a small portion of a cell or a cellular structure is represented in a single 2D plane. We have used a three-dimensional (3D) approach to examine the structural architecture and cellular composition of myofibroblasts, lipofibroblasts, alveolar type 2 cells and elastin extracellular matrix in normal as well as BPD-like mouse lungs (Branchfield et al., Dev Biol , 2015) . The insights revealed by 3D reconstruction of the alveoli set the foundation for future investigations of the mechanisms driving alveologenesis, as well as causes of alveolar simplification in BPD. We have also used a genetic approach to investigate the role of signaling pathways in this process (Li et al, Development, 2017; Li et al., Elife , 2018) . Ongoing work in this research direction interrogates how different elements of prematurity impacts long-term lung function. We are interested in why some BPD patients develop pulmonary hypertension while others do not, and why BPD patients are more susceptible to viral infections later in life. Genetic modeling of lung diseases As whole-genome sequencing of patients becomes routine, we have established fruitful collaborations with human geneticists, and are using advanced technologies such as CRISPR/Cas9 genome editing to interrogate causal relationships between patient-specific genomic variants and phenotypes and disease phenotypes. Towards this goal, we started with rare congenital disorders, and are also extending our effort to study common and complex diseases. In the past few years, among lung-related congenital disorders, we have studied tracheo-esophageal fistula (Domyan et al., Development 2011) , tracheobronchomalacia (Hines et al., PNAS 2013 ) , and congenital diaphragmatic hernia (CDH) (Domyan et al., Dev Cell , 2013; Branchfield et al., Science , 2016, McCulley et al., JCI, 2018 ) . Using CDH as an example, we have established a pipeline to first determine candidate gene expression in the developing lung (Herriges et al., Dev Dynamics , 2012) , and then use CRISPR/Cas9genome editing and other genetic approaches to recapitulate the mutations in mice, and determine which variants may be causal to phenotypes As a demonstration of the effectiveness of this approach, we have uncovered distinct mechanisms from different genetic models of CDH (Domyan et al., Dev Cell , 2013; Branchfield et al., Science , 2016, McCulley et al., JCI, 2018 ) . These findings provide an explanation for why standard treatment works in some, but not other patients. More importantly, each of the mechanism makes prediction of personalized treatments based on patient genotype. Ongoing work in this research direction aims to expand to additional genes and diseases. \n",
            "-------------------- Scraping faculty url 164/183 --------------------\n",
            "ella tour  Biography Ella Tour received her Ph.D. in Genetics at the Hebrew University of Jerusalem, where she studied the molecular mechanisms of brain development in the frog Xenopus laevis. She conducted her post-doctoral training at UCSD, investigating the early development of the fruit fly embryos.  Research Education Critical thinking skills, such as the ability to analyze data, solve problems, and develop new ideas, are crucial for success in the modern workforce. Although the majority of biology educators consider critical and creative scientific thinking some of the most important skills to be acquired as a result of science education, few biology faculty report spending enough time practicing these skills in their classrooms (Coil et al., 2010). One of my primary education research goals is to increase our understanding of how to effectively teach critical thinking skills in biology and how objectively test whether students are learning these skills. In my classes, I strive to provide my students with ample opportunities to practice data analysis and experimental design. Currently, my collaborators and I are investigating the effectiveness of in-depth analysis of primary research papers and designing experiments that follow up on these papers, on students' critical thinking skills. An additional area of education research is developing methods to improve the quality of scientific writing and prevent plagiarism in Biology labs. This research is performed in collaboration with Tricia Bertram Gallant (UCSD Office of Academic Integrity), Julian Parris (Psychology), Goran Bozinovic (Biology), and Madeline Picciotto (UCSD Writing Center). Biological In collaboration with the members of Bill McGinnis's lab here at UCSD, I study the role of chromatin structure and long non-coding RNA's in the regulation of gene expression in developing embryos of the fruit fly Drosophila melanogaster . Early embryonic development is a precisely orchestrated process in which genes that control the formation of the various body parts have to be expressed at the right time and the right place. One such master control gene, Ultrabithorax ( Ubx ) is responsible for the correct development of parts of fly's thorax and abdomen. The expression of Ubx is regulated by multiple enhancers that are located tens of thousands base pairs away from the Ubx promoter and its transcription start site. Using high-resolution fluorescent in situ hybridization (FISH), we study the conformational changes involving Ubx promoter and its enhancers that occur in silent and transcriptionally active states . We are also investigating a potential role of an antisense transcript that originates in one of the introns of Ubx in the regulation of this gene. This image shows one actively transcribed Ubx gene in 3 hours-old Drosophila embryo. P (red) Ubx promoter, mRNA (blue) nascent Ubx mRNA originating from the 1st intron of this gene, bx (green) bx enhancer, M (purple) a region located midway between the promoter and bx enhancer. bx enhancer, located 34 kbp downstream of Ubx promoter, appears to be in contact with this promoter. \n",
            "-------------------- Scraping faculty url 165/183 --------------------\n",
            "david traver  Biography David Traver received his Ph.D. in immunology from Stanford University where he received the McDevitt Prize for best thesis. He then completed postdoctoral studies at Harvard University as a fellow of the Irvington Institute. Since joining UCSD in 2004, he has received a Young Investigator Award from the Beckman Foundation, a Scholar Award from the American Society of Hematology and a New Faculty Award from the California Institute for Regenerative Medicine.  Research Utilizing zebrafish for new insights into the biology of hematopoietic stem cells Hematopoietic stem cells (HSCs) exist as rare populations within blood-forming tissues that both self-renew and generate all blood cell lineages for life. HSCs have been isolated to relative purity in humans and mice, leading to a general understanding of their cellular and functional properties. Relatively little is known, however, regarding the genetic program underlying the embryonic formation and subsequent maintenance of HSC fate. The emergence of zebrafish as a model system has introduced unbiased genetic approaches, such as forward genetic screens and high-throughput expression screens, to the study of vertebrate blood cell formation. Fluorescent transgenesis technology combined with the external fertilization and transparency of zebrafish embryos allows early events in HSC specification, maintenance, and differentiation to be easily observed. Collectively, we are utilizing these advantages to image, fate map, and interrogate genetically the earliest embryonic blood-forming cells. This has helped address many long-standing questions in the development of the immune system, including the direct demonstration of HSC emergence from the transdifferentiation of aortic endothelium, and the demonstration of the importance of neighboring tissues in the instruction of HSC fate. We are performing forward genetic and high-throughput expression screens to identify new gene functions in vertebrate stem cells. Novel genes that regulate HSC development, self-renewal, or lineage commitment will subsequently be examined for similar roles in mammalian systems. Despite decades of efforts, it is not yet possible to instruct HSC fate in vitro from pluripotent precursors. We hope that discovery of factors required to specify HSCs in the embryo will ultimately be useful in advancing this important goal of regenerative medicine. \n",
            "-------------------- Scraping faculty url 166/183 --------------------\n",
            "emily troemel  Biography Emily Troemel received her B.S. from the University of Wisconsin-Madison and then spent a year in Japan before receiving her Ph.D. in Cell Biology at UC-San Francisco. She worked for a start-up biotech company in the Bay Area before returning to academia to do a postdoc at Massachusetts General Hospital and then joined the UCSD faculty in 2008. Since joining UCSD, she has received a Searle Scholars Award, David & Lucile Packard Foundation Award and a Burroughs Wellcome Fund Investigators in the Pathogenesis of Infectious Disease Award.  Research How do pathogens attack their hosts, and how do hosts defend themselves against these attacks? We are using the nematode C. elegans to study these questions of host/pathogen interactions in the intestine. The nematode C. elegans provides a wonderful system in which to study these questions. The worm has intestinal cells that are similar in structure to human intestinal cells, but C. elegans is transparent and easy to study in lab. C. elegans lacks a professional immune system, and instead relies solely on epithelial defense to fight off pathogen attack. It is being increasingly appreciated that epithelial cells in the intestine are involved in detecting and responding to pathogens in mammals, and are implicated in inflammatory disorders. It is critical to understand more about how these cells detect and respond to pathogens. What we find in C. elegans intestinal immunity may be directly applicable to host/pathogen interactions in the human intestine. Microsporidia infecting C. elegans intestine microsporidia are labeled in red, C. elegans intestine labeled in green, and DNA labeled in blue. Our research focuses on two different pathogens. The first pathogen is a natural intracellular pathogen isolated from a wild-caught C. elegans strain, which we have found defines a genus and species of microsporidia (intestinal infection of C. elegans intestine shown below). Microsporidia are poorly understood eukaryotic pathogens that are emerging pathogens of humans and can cause problems for immunocompromised patients. The second pathogen is an extracellular bacterial pathogen called Pseudomonas aeruginosa , which is a clinically relevant pathogen that causes opportunistic infections in humans. \n",
            "-------------------- Scraping faculty url 167/183 --------------------\n",
            "kay tye \n",
            "-------------------- Scraping faculty url 168/183 --------------------\n",
            "elizabeth villa  Biography Elizabeth Villa completed her PhD in Biophysics at the University of Illinois at Urbana-Champaign as a Fulbright Fellow. She was a Marie Curie Postdoctoral Fellow in the Max Planck Institute of Biochemistry in Munich. She joined the Chemistry and Biochemistry Department faculty at UC San Diego in July 2014. In 2016, she was awarded an NIH Directors New Innovator Award. She joined the Division of Biological Sciences faculty at UC San Diego in March 2017.  Research Macromolecules rarely operate in isolation inside cells. At any given time, the average protein is part of a complex of over 10 macromolecules, and these supramolecular complexes are in turn embedded in intricate networks inside cells. We are broadly interested in revealing the structure and function of macromolecular complexes in their natural environment at the highest possible resolution in order to reveal their structural dynamics and interactions. We call it bringing structure to cellular biology. We have a strong focus on building tools for quantitative cell biology, using cryo-electron microscopy and tomography, cell biophysics, computational analysis, and integrative modeling. This potent combination allows us to look at macromolecular complexes in their native environment and derive their structure, context, and interaction partners. Our biological focus is on the study of the nuclear periphery, as nuclear biology remains one of the most exciting challenges in the cell, and it is uncharted territory structurally. Our thrust in this area includes projects such as: the structural dynamics of the yeast nuclear pore complex, the mechanical communication between the cytoskeleton and the nucleus, and the molecular architecture of the genome and its association to the nuclear envelope. We also collaborate with different laboratories to open windows into various cellular events. These projects tend to have a translational component, and include studying the inner life of bacteria and studying the effects of LRRK2 in Parkinsons disease, among others. \n",
            "-------------------- Scraping faculty url 169/183 --------------------\n",
            "geoffrey wahl \n",
            "-------------------- Scraping faculty url 170/183 --------------------\n",
            "jing wang  Biography Dr. Wang received his Ph.D. in Neurobiology from the University of Iowa. He did postdoctoral research in the laboratory of Alan Gelperin at Bell Labs, then in the laboratory of Richard Axel at Columbia University. Since joining the faculty in 2004, he has been named a Chris and Warren Hellman Faculty Scholar, a Searle Scholar, and a Beckman Young Investigator.  Research The olfactory sense is essential for basic animal behaviors such as finding food, avoiding predators, and locating mating partners. Understanding the mechanism of olfaction has been accelerated by the discovery of an olfactory map at the first synapse. In mammals, these synapses are organized into 1000 glomeruli, each of which receives thousands of sensory neurons that express the same odorant receptor gene. The driving force of my lab is to find answers to such questions as: Is the pattern of glomerular activity evoked by a given odor related to the detection of this odor? How is the olfactory information represented and processed in the brain? How is olfactory valence represented in the brain? How is this representation modulated to meet the demands of different internal physiological states? The long-term goal of my laboratory is to elucidate the mechanism of olfaction. We choose Drosophila as a model system for two main reasons. First, Drosophila has fewer than 50 glomeruli in the antennal lobe. A sophisticated genetic toolkit makes it possible to establish causality between neural circuit function and behavior. Second, despite the numerical simplicity, the anatomical organization of the Drosophila olfactory system is fundamentally similar to that of mammals, suggesting a shared mechanism for smell. Our current research activities aim to identify specific neural circuits underlying olfactory behaviors, investigate how different physiological states of an organism modulate the function of olfactory circuits, and establish causality between neural plasticity in specific neuron populations and olfactory behaviors. 1) Synaptic modulations in neural circuits Olfaction is an ancient sense that makes important contributions to the perception of food quality and dietary selection. Learning how olfactory neural circuits impact dietary choices is relevant towards better understanding factors that contribute to obesity as well as anorexia in the infirm and elderly. Like many other animals, fruit flies exhibit different food searching and food-intake behaviors upon starvation. The sense of smell plays an important role in appetitive behavior in Drosophila . The DM1 glomerulus mediates attraction behavior and the DM5 glomerulus mediates avoidance behavior (Semmelhack and Wang, 2009). In a recent study, we showed that starvation sensitizes DM1's olfactory response (Root et al., 2011). A local signal by the neuropeptide sNPF/NPY and a global metabolic cue by insulin are integrated at specific sensory neurons to change olfactory sensitivity. This suggests the odor map is fine-tuned by the internal state of the animal. Furthermore, our studies in the last few years have provided insights into establishing the notion that neuromodulation mechanisms are shared between Drosophila and mammals (Root et al., 2008; Dacks et al., 2009). In both groups, GABAB receptor-mediated presynaptic inhibition provides a gain control mechanism to modulate olfactory sensitivity. Furthermore, serotonin fine-tunes this feedback inhibition by controlling the excitability of the local interneurons in Drosophila and periglomerular cells in mice. 2) Neural plasticity in olfactory circuit During a sensitive period in many animal species, usually early in life, sensory cues signaling safety and food are quickly learned and encoded into the memories of young animals through modifications of neurons in sensory neural circuits. As an important survival mechanism and a rudimentary form of learning, imprinting of sensory cues can guide and shape an animal's preferences and behaviors. We have obtained results to demonstrate that Drosophila exhibits a robust form of olfactory imprinting. Exposure to a given odorant in early adulthood dramatically increases a fly's preference for that odorant at a later stage. We aim to establish a causal link between neural plasticity in specific neuron populations and the imprinting behavior. By integrating several neural techniques, including single-neuron electrophysiology, optical imaging with genetically encoded activity indicators and genetic tools to silence or activate specific neurons in the stereotypic olfactory circuit, we hope to understand the neuronal bases of olfactory behaviors and test different hypotheses of olfactory codes with unprecedented resolution. \n",
            "-------------------- Scraping faculty url 171/183 --------------------\n",
            "carl ware \n",
            "-------------------- Scraping faculty url 172/183 --------------------\n",
            "steven wasserman  Biography Steven Wasserman received his Ph.D. from MIT and was a postdoctoral fellow at UC Berkeley. He has been the recipient of a Lucille P. Markey Scholarship in Biomedical Sciences, a David and Lucile Packard Fellowship in Science and Engineering, and a Distinguished Teaching Award from the UCSD Academic Senate.  Research We are interested in how biological information regulating gene expression is encoded, transmitted, and interpreted. In addressing this question, we focus on Toll signaling, a pathway for transducing information within cells that is conserved from insects to humans and that has diverse roles in development and innate immunity. Our studies focus on the fruit fly Drosophila melanogaster , with which we can readily generate mutations that disrupt pathway activity; monitor and manipulate gene activity; and map out signaling modules and regulatory circuitry using molecular, biochemical, and bioinformatic techniques. In this way, we have defined the structure and organization of a trimeric signaling complex, elucidated function-specific evolutionary adaptations, and identified novel pathway functions and effectors. Signal Transduction in Disease and Development The signal transduction pathway defined by the Toll transmembrane receptor is a critical component of the Drosophila innate immune response to microbial infection. Upon exposure to a fungal pathogen, wild-type fruit flies express an array of genes encoding antimicrobial peptides, including Drosomycin, a potent anti-fungal agent. The transcription factor activated by Toll in this setting is the Drosophila Immunity Factor (DIF), which belongs to the NF-kB protein family. In mammals, Toll-like receptors (TLRs) activate NF-kB as a critical step in innate immune response to infection. Furthermore, aberrant Toll signaling underlies a number of debilitating inflammatory disorders. The function of Toll in innate immunity is thus broadly conserved and of interest from both a fundamental and clinical perspective. In Drosophila , Toll signaling also establishes the dorsoventral axis of the embryo. Localized activation of the transmembrane receptor Toll leads to the graded nuclear translocation of the transcription factor Dorsal, another NF-kB family member. By activation of ventral-specific loci and repression of dorsal-specific loci, the Dorsal gradient establishes subdivides the dorsoventral axis. The same signaling pathway is used in immunity and development. Prior to signaling, the target NF-kB protein, either DIF or Dorsal, is bound in the cytoplasm by an inhibitor, Cactus, that blocks nuclear translocation. Following generation of the active Toll ligand Sptzle by upstream signals, Toll transduces signals leading to degradation of Cactus, freeing the NF-kB protein to direct gene expression. Currently, our studies of Toll signaling are centered on three research areas: Dissecting the Mechanism of Signal Transduction. Toll-mediated inactivation of the inhibitor Cactus requires signal transduction by MyD88, Tube, and Pelle. We have demonstrated that these three proteins function in signaling by forming a trimeric protein complex. We have used molecular genetic, biochemical, and biophysical approaches to define and mutate the precise sites of interaction in the trimer, generating powerful reagents for our ongoing dissection of the signaling mechanism. We have extended our studies to explore how formation of the death domain trimer leads to phosphorylation and degradation of the inhibitor protein, Cactus. In particular, we have recently identified Pelle as the protein kinase required for Cactus inactivation. An Informatics-Based Approach to Decoding Global Transcriptional Control. The Toll pathway operates in parallel with a second signaling system the Imd pathwayto govern innate immune responses. Both systems rely on NF-kB related transcription factors to regulate gene expression. Applying a molecular genetic approach, we have demonstrated that cis -acting control elements act as a specificity code for the response to either or both pathways. Using our understanding of these elements to inform bioinformatic studies, we have mapped out the global system of control for the major innate immune responses in flies. Exploring Evolutionary Pathway Adaptations to Embryonic Patterning. Although the Toll pathway acts in innate immunity in a wide range of animals, it has been specifically adapted to pattern formation in the insects. There it must function in a syncytial environment and be regulated in time and space in a manner distinct from that required for defensive responses. Through identification of pathway components from a variety of insects and functional assays of chimeric proteins, we are defining the domains and activities that have undergone modification to fulfill these specialized functions in the context of conservation of the basic signaling mechanism. \n",
            "-------------------- Scraping faculty url 173/183 --------------------\n",
            "gabriele wienhausen  Biography Gabriele Wienhausen received her Ph.D. from the Westfalische Wilhelms-Universitat, Munster, Germany. She is currently the co-Principal Investigator of the Howard Hughes Undergraduate Science Enrichment Program and co-Director of the doctoral program in Mathematics and Science Education offered jointly by UCSD and SDSU.  Research Our research is focused on (1) how to promote \"meaningful learning\" in college science teaching, i.e., how to assure that students store more information in a memorable, readily usable fashion, and (2) how to facilitate the development of effective problem solving skills. Specifically, we investigate what role interactive, computer-based multimedia packages can play and how effective they are as a tool to facilitate the understanding of complex biological/scientific processes and concepts. We study the impact of a systematic, computer-based representation of a scientific topic on a student's learning and recall of that topic. To accomplish our goals, we develop multimedia modules which are being used as tools to improve science teaching in the laboratory setting and to integrate the process of scientific research and education. These modules have been designed on the premise that a deep understanding of biological and biomedical systems and processes depends on a meaningful inclusion of related knowledge from all the sciences. Our innovative methods of instruction address the dramatic changes in the pace of scientific learning exemplified in areas such as molecular biology or structural biochemistry. Scientific results are superseded quickly, and it is mandatory that students are familiarized with the know-how of information acquisition and its effective use. Students are taught that concepts and knowledge have to be applied across scientific disciplines. A key thesis of our research is that interactive, computer-based multimedia packages provide a potent vehicle for addressing many of the previously listed problems. We believe that the flexibility of multimedia, computer-based instructional systems allows for the provision of information in a way that more closely resembles the way research scientists think and acquire new knowledge; typically, both intellectual processes occur in a non-linear, interdisciplinary fashion where new pieces of information are fitted into an existing framework of knowledge. For instance, in the software module titled \"Spectrophoto-metry\", students can explore the interaction of light and matter, follow the mathematical derivation of Beer's Law, and practice the operating of a UV-visible virtual spectrophotometer using an on-screen working model. At all times, students using the software are in control of the pace and the path. They determine the extent of interactivity, they determine the direction, and depth of their learning path. \n",
            "-------------------- Scraping faculty url 174/183 --------------------\n",
            "jim wilhelm  Biography Jim Wilhelm received his Ph.D. in Cell Biology from University of California, San Francisco. He then took a Staff Associate (independent postdoctoral position) in the Department of Embryology, Carnegie Institution of Washington where he received funding from the Life Sciences Research Foundation as a Howard Hughes Medical Institute Fellow. He is also a Sloan Research Fellow in Neuroscience, an Ellison Medical Foundation New Scholar in Aging, and March of Dimes Basil O'Connor Scholar.  Research Cytoplasmic Organization in Development, Metabolism, and Disease How is a symmetrical egg turned into an embryo with a head and a tail? How are neuronal connections strengthened or weakened in response to experience? Why dont incompatible chemical reactions that are necessary for life interfere with each other? The solution to these seemingly unconnected questions lies in the ability of cells to target proteins and nucleic acids to specific subcellular domains. My laboratory is focused on identifying and characterizing novel mechanisms for generating subcellular organization and studying how those mechanisms are used in development, metabolism, and the nervous system. As part of this effort, we use a multidisciplinary approach that applies techniques from biochemistry, genetics, microscopy, and systems biology. A systems biology approach to identifying novel principles of subcellular organization In order to identify novel types of intracellular organization, we are conducting a visual screen of the yeast GFP strain collection to identify proteins that assemble into previously unidentified intracellular structures. A pilot version of this screen has identified 38 novel intracellular structures the majority of which are comprised of proteins involved in metabolic pathways suggesting that self-assembly is a major mechanism for regulating metabolic pathways in vivo. Consistent with this, one of the proteins we identified, CTP synthase, forms a novel intracellular filament that is conserved from bacteria to mammals. We have also shown that its ability to self-assemble is connected to the regulation of CTP synthase activity. We propose that this form of enzyme regulation functions analogously to the way in which microtubules and actin operate the core of the polymer is locked into a particular conformation by end limited disassembly. Since we have identified a number of enzymes that exhibit polymerization in vivo, it is likely that this mode of regulation is not unique to CTP synthase. Interestingly, we have also found that CTP synthase filaments form in axons, but not in dendrites arguing that CTP synthase filaments may also have a cytoskeletal role in neurons apart from their role in regulating enzyme activity. We are currently characterizing CTP synthase filaments in neurons. The role of information sorting in Drosophila development One mechanism for localizing cytoplasmic proteins is to transport the mRNA encoding the protein to the desired location, so that the synthesis of the protein is spatially restricted to particular cytoplasmic regions. Establishment of these domains is further enhanced by a translational control mechanism that ensures that only the properly targeted messages are translated. This sorting mechanism has been implicated in processes as diverse as stem cell differentiation, regulating synaptic strength in neurons and embryonic pattern underscoring the importance of understanding mRNA localization for medicine, neuroscience, and developmental biology. While the importance of this method of sorting cytoplasmic proteins is clear, it is unclear how localized messages are targeted to different domains or how many cellular or developmental processes utilize mRNA localization. We have used a biochemical approach to purify mRNA transport/translational control complexes from Drosophila melanogaster. By exploiting the powerful genetic tools available in Drosophila, we have begun dissecting how these transport complexes are assembled and disassembled in order to establish the primary body axes of the Drosophila embryo. \n",
            "-------------------- Scraping faculty url 175/183 --------------------\n",
            "christopher wills  Biography Christopher Wills received his Ph.D. from UC Berkeley. As a Guggenheim Fellow, he worked at the Karolinska Institute, Stockholm, on protein chemistry and evolution. He is the recipient of the 1999 Award for the Public Understanding of Science and Technology, given by the American Association for the Advancement of Science. His book Children of Prometheus (Perseus Books 1998) was a finalist for the 2000 Aventis Prize for science books.  Research Our laboratory is examining a broad range of evolutionary questions, including the evolution of sex and recombination, the evolution of microsatellites in lower organisms, microevolutionary events that take place in the course of infection in the genomes of pathogens, the evolution of HIV and other viruses, the timing of genetic events that have taken place during the history of our species, and the evolutionary divergence of distantly related proteins. We are also examining the maintenance of diversity in complex genetic systems and complex ecosystems. We have found that the major gene directing sexual recombination, the MAT locus, has pleiotropic effects that range far beyond meiotic and mitotic recombination and DNA repair, to encompass cellular processes as different as the uptake of exogenous DNA and the regulation of mitochondrial activity. Our results strongly support the concept that the evolution of sex had its beginnings in the uptake and incorporation by cells of exogenous genetic material. We are investigating large numbers of polymorphic microsatellites in yeast, some of which may be correlated with fitness. Such microsatellites are surprisingly common in lower organisms, and we have shown that microsatellites of the pathogenic fungus Candida albicans undergo alterations in the course of infection in AIDS patients. We have also found abundant polymorphic microsatellites in the important viral pathogen cytomegalovirus. We are also developing a selective PCR system in Saccharomyces cerevisiae which will allow us to follow the accumulation of microsatellite length mutations over time in stressed and unstressed populations. We have developed a number of methods for increasing the amount of evolutionary information in phylogenetic trees. This has enabled us to obtain a more accurate estimate of the date, and the probable geographic location, of the mitochondrial Eve. This methodology has also enabled us to determine more accurate human retrovirus evolutionary trees. Using this approach, we were able to show that the evolution of n-glycosylation sites (\"sequons\") in the envelope gene has proceeded at three times the rate in HIV-1 that it has in the less pathogenic HIV-2. This is the largest difference in evolutionary rate that has yet been detected in these viruses, and is likely to be connected to their interaction with the host immune system. Another research direction in the laboratory has been the mechanisms maintaining genetic diversity. We recently proposed a model of \"genetic herd-immunity\" to explain the maintenance of such highly polymorphic genetic systems as MHC. This frequency-dependent model permits a genetically heterogeneous host population to keep many pathogens at bay simultaneously. Stephen Hubbell, Richard Condit and I have recently extended this model to the maintenance of diversity in the rainforest, and have evidence from well-studied rainforests in Panama and Malaysia that herd-immunity at the level of species is in fact operating. Most recently, we have shown that for seven different tropical rainforests, in both the Old and New Worlds, cohorts of trees increase in diversity over time because the locally commoner species have a higher mortality than the locally rarer ones. These results, which show that there are strong non-random processes increasing diversity in these forests, are consistent with several frequency-dependent models for the maintenance of diversity. I have published a number of books about evolution for general readers, most recently\"Green Equilibrium: Balancing the Futures of Our Species and Our Planet,\" (Oxford University Press, 2013). The book deals with how ecosystems throughout the world have been shaped by the process of evolution. I continue to work with tropical ecologists around the planet, and we have recently shown that, in two different tropical forests, ecological pressures continue to shape the evolution of even distantly related species of trees that share the same ecosystem. \n",
            "-------------------- Scraping faculty url 176/183 --------------------\n",
            "yang xu  Biography Yang Xu received his Ph.D. from Harvard University. He conducted postdoctoral research at the Massachusetts Institute of Technology as a fellow of the Damon Runyon-Walter Winchell Cancer Research Fund.  Research Our laboratory is interested in understanding the signaling pathways involved in maintaining genetic stability in mammalian cells, particularly embryonic stem cells (ESCs). With their capability of unlimited self-renewal and pluripotency to differentiate into any cell types in the body, ESCs hold great promise for human cell replacement therapy. Therefore, it is critical to elucidate the mechanism to safeguard the genetic integrity of ESCs so that DNA damage will not be passed on to progeny. We are investigating the pathways that coordinate the DNA damage responses and self-renewal of ESCs and adult stem cells. We are testing the hypothesis that disruption of this coordination will induce genetic instability in adult stem cell population, leading to the generation of cancer stem cells. We are focusing on the roles of tumor suppressors such as ATM and p53, which are critical to maintain genetic stability in somatic cells. To facilitate the studies of gene function in human ESCs, we recently developed technologies that allow efficient knock-out and knock-in through homologous recombination in hESCs. Induced pluripotent stem cells (iPSCs), reprogrammed from somatic cells with defined factors, could provide unlimited autologous cells for human therapy. Since most of the reprogramming factors have oncogenic potential, we are investigating the genetic stability in iPSCs. In addition, we are developing strategies to address the bottlenecks that hinder the clinic development of human ESCs and iPSCs. For example, we are identifying the pathways that can be used to eliminate the teratomas risk associated with ESCs and iPSCs-based therapy. In the context of hESCs, we are investigating the options to induce immune tolerance to allogenic grafts derived from hESCs. \n",
            "-------------------- Scraping faculty url 177/183 --------------------\n",
            "martin yanofsky  Biography Marty Yanofsky received his Ph.D. from the University of Washington and was an NSF Postdoctoral Fellow in Plant Biology at Caltech. He was the recipient of a Packard Fellowship for Science and Engineering and a Beckman Young Investigator Award. Marty Yanofsky is a member of the National Academy of Sciences.  Research Research in the Yanofsky lab focuses the genes involved in fruit development in Arabidopsis thaliana . The fruit, which is arguably the most complex plant organ, is derived from the fertilized gynoecium (carpels), and consists of a number of distinct tissue types, including the valves, replum and valve margin. The valve margin, which forms at the boundary between the valve and replum regions, consists of a narrow stripe of cells that extends along the entire length of the fruit. Upon fruit maturation, cell separation within the valve margin region allows the valves to detach from the replum and for the seeds to be dispersed. We have shown that the SHATTERPROOF1 ( SHP1 ) and SHP2 MADS-box genes act redundantly to specify valve margin cell fate. The SHP genes act to promote the expression of the INDEHISCENT ( IND ) and ALCATRAZ ( ALC ) bHLH genes, which are also required for differentiation of valve margin cells. Because the valve margin fails to differentiate in these mutants, the fruits fail to open and the seeds cannot be dispersed through their normal mechanism. We have also shown that the FRUITFULL ( FUL ) MADS-box gene is required for the post-fertilization elongation of the fruit and is required for valve cell differentiation. Interestingly, the FRUITFULL gene product is expressed in valve cells where it negatively regulates expression of SHP , IND , and ALC genes such that the SHP , IND , and ALC genes are ectopically expressed in fruitfull mutant valves. Similarly, the REPLUMLESS ( RPL ) gene encodes a homeodomain transcription factor that is required for formation of the replum. RPL is expressed in the replum region where it acts to negatively regulate SHP , IND , and ALC . The combined action of FUL in the valves and RPL in the replum allows for the valve margin identity genes to be expressed in the narrow stripe of cells that differentiate into the valve margin. More recently, we have begun to understand the role of additional fruit development genes leading to a framework model for the genetic interactions that pattern the Arabidopsis fruit. Our lab has also initiated a project aimed at identifying the major genes that specify the female reproductive tract in Arabidopsis . This region plays a key role in ensuring efficient pollen tube migration as they seek out unfertilized ovules, and yet little is known about the regulatory genes that control the formation of the reproductive tract. We have recently shown that the NO TRANSMITTING TRACT ( NTT ) gene encodes a zinc finger transcription factor that is necessary for formation of transmitting tract cells and for preventing the cell death that normally occurs within the transmitting tract region. \n",
            "-------------------- Scraping faculty url 178/183 --------------------\n",
            "deborah yelon  Biography Deborah Yelon received her Ph.D. in 1996 from Harvard University, where she studied T cell development with Leslie Berg. She then began to investigate heart development in zebrafish as a Life Sciences Research Foundation postdoctoral fellow with Didier Stainier at the University of California, San Francisco. She was a member of the faculty of the Skirball Institute of Biomolecular Medicine at New York University School of Medicine from 2000 to 2009. She has received a Burroughs Wellcome Fund Career Award and an American Heart Association Established Investigator Award.  Research Organogenesis requires the carefully orchestrated arrangement of multiple cell types into a specific three-dimensional form that is essential for effective organ function. The embryonic zebrafish heart provides a clear example of the relationship between organ form and function: it is composed of two major chambers, a ventricle and an atrium, each with discrete morphological and physiological characteristics that define its functional capacity. The goal of our laboratory's research is to understand the molecular mechanisms that specify the size, shape, and attributes of each cardiac chamber. By taking advantage of the unique arsenal of genetic and embryologic experimental approaches available in zebrafish, we can identify crucial regulators of chamber formation and determine their precise impacts on cell fate and cell behavior. In the long term, this work will improve our comprehension of the etiology of congenital heart disease and inform strategies for regenerative medicine and tissue engineering. Regulation of heart size Specification of the proper number of cardiac progenitor cells is a critical determinant of heart size, but we do not fully understand the network of signals that regulate the dimensions of the cardiac progenitor pool. Several types of secreted factors, including BMP, FGF, and Hedgehog, are implicated in promoting the assignment of cardiac identity. However, much less is known about the opposing factors that set limits for cardiac progenitor specification. We are therefore particularly interested in zebrafish mutants with large hearts composed of too many cardiomyocytes. Notably, our recent analyses of these mutants have revealed two potent mechanisms for restricting the formation of cardiac progenitors. These studies indicate that generation of the proper number of progenitors involves interplay between inductive and repressive pathways. Cardiac cells (red) develop in proximity to the vascular plexi (green) prior to heart formation. Allocation of cardiac and vascular progenitors requires interplay between cardiovascular pathways: components of vessel and blood specification pathways repress formation of cardiac progenitors and thereby limit heart size (Schoenebeck et al., 2007). Regulation of heart shape An elaborate series of cell movements and cell shape changes creates the characteristic contours of the heart, and all of these cellular activities can be influenced by the surrounding environment. However, it is not clear which extrinsic and intrinsic cues regulate cell behaviors during chamber morphogenesis. By combining high-resolution live imaging with genetic analysis, we can elucidate pathways with a crucial influence on the actions of individual cardiomyocytes. To date, we have focused our attention on two particular steps of cardiac morphogenesis: the midline merger of the bilateral cardiomyocyte precursor populations and the emergence of chamber curvatures. During both of these processes, regionally restricted patterns of cell behavior underlie key features of cardiac morphology, and interactions of the cardiomyocytes with their environment have a significant impact on morphogenetic control. Regionally confined cell shape changes underlie the emergence of chamber curvatures. A portion of the linear heart tube (LHT) transforms into an expanded ventricle with a bulging outer curvature (OC) and a recessed inner curvature (IC). Individual cells (red, arrows) are small and round in the LHT. Cells in the OC become flattened and elongated, whereas cells in the IC remain relatively cuboidal, creating the characteristic shape of the ventricle (Auman et al., 2007). \n",
            "-------------------- Scraping faculty url 179/183 --------------------\n",
            "dong-er zhang  Biography Dr. Dong-Er Zhang received her BS in Biochemistry at Peking (Beijing) University, China. She was selected to enter a graduate school in the United States through the CUSBEA (China-United States Biochemistry and Molecular Biology Examination and Administration) program and received her PhD in Biochemistry at the University of Houston. She was a Postdoctoral Scholar/Instructor at the University of Texas and then Instructor/Assistant Professor at the Harvard Medical School. In 1999, Dr. Zhang was recruited by The Scripps Research Institute as Associate Professor and was subsequently promoted to the Full Professor level before joining UCSD. Dr. Zhang was a Leukemia and Lymphoma Society Scholar (1998-2003) and received the prestigious Stolhman Scholar Award from the Leukemia and Lymphoma Society. She is involved in multiple international grant review committees and also contributes to the organization of scientific conferences.  Research Dr. Zhangs laboratory studies the molecular basis of cancer development, progression, and treatment. Her laboratory uses molecular biology, protein biochemistry, cell biology, and animal models to address questions related to blood cell differentiation, transformation, and innate immune responses. A major project underway in the Zhang laboratory focuses on the transcription factor AML1 (RUNX1) and its fusion protein AML1-ETO. As a second major project, Dr. Zhang studies the role of a ubiquitin-like modifier, ISG15, and the ISG15specific protease, USP18 (UBP43), in interferon-initiated cell-signaling and cancer. \n",
            "-------------------- Scraping faculty url 180/183 --------------------\n",
            "yunde zhao  Biography Yunde Zhao received his Ph.D. in biochemistry from the University of Michigan and his postdoctoral training in plant genetics at the Salk Institute, where he was a Howard Hughes Medical Institute Fellow of the Life Sciences Research Foundation.  Research Our research focuses on the plant hormone auxin, which regulates almost every aspect of plant growth and development. My laboratory mainly works on two areas of auxin biology: 1) auxin biosynthesis and its regulation, 2) the roles of auxin in axillary meristem development and plant organogenesis. We have elucidated the main auxin biosynthesis pathway in Arabidopsis and identified YUC flavin monooxygenases as key enzymes for auxin biosynthesis. We currently investigate the allosteric regulation of YUC enzymes and characterize the transcription factors that control the expression of YUC genes. We conducted a genetic screen for enhancers / suppressors of yuc mutants and uncovered a novel signaling pathway that is essential for the formation of lateral organs in Arabidopsis. We are continuing to clone and characterize additional yuc enhancers / suppressors to establish a genetic framework for plant organogenesis. We are also developing tools to spatially and temporally control auxin production. \n",
            "-------------------- Scraping faculty url 181/183 --------------------\n",
            "ye zheng  Biography Dr. Ye Zheng earned his B.S. degree in Biochemistry and Molecular Biology from Peking University in Beijing, China, and his Ph.D. degree in Biological Sciences from Columbia University in New York. He was awarded a postdoctoral fellowship sponsored by Cancer Research Institute to conduct research in Dr. Alexander Rudensky's lab at University of Washington in Seattle. After a brief sojourn as a research scholar at Memorial Sloan-Kettering Cancer Center, he joined the Salk Institute for Biological Studies as an assistant professor in 2009. He was named a Rita Allen Scholar in 2010.  Research The immune system is a powerful double-edged sword. On one hand, it is armed to fight a wide range of invading foreign pathogens. On the other hand, if left unchecked, it can also attack self-tissues and cause autoimmune diseases, such as type-1 diabetes, multiple sclerosis, and rheumatoid arthritis. There are multiple safeguard mechanisms built in our immune system to prevent autoimmune reaction. A subset of T cells, named regulatory T cells (Treg), play a key role in the maintenance of immune homeostasis. Abnormal Treg function has been linked to a number of autoimmune diseases. Recent study showed that transcription factor Foxp3 is a pivotal regulator for Treg differentiation and function. Mutations of Foxp3 in human and mice lead to deficiency of regulatory T cells and fatal autoimmune disease. Zhengs lab is interested in elucidating both the upstream pathways that turn on Foxp3 expression and the downstream genes that are regulated by Foxp3. Zheng and colleagues identified several DNA elements in Foxp3 locus that are conserved in several mammalian species. They are involved in the induction and maintenance of Foxp3 expression, and regulate the development and stability of regulatory T cell lineage. Using genomics approaches, Zheng and colleagues were able to map all Foxp3 downstream target genes. They showed that, among all Foxp3 targets, there is a small group of transcription factors that are implicated in Treg mediated suppression of different subtypes of autoimmune responses. Zheng and his team are now further dissecting the Foxp3 transcriptional network in regulatory T cells and searching for key molecules involved in Treg suppression function. Since manipulations of Tregs can either attenuate or strengthen immune response, their findings can potentially open new avenues in treatment of autoimmune diseases, improvement of organ transplant survival, and enhancement of anti-tumor immunity. \n",
            "-------------------- Scraping faculty url 182/183 --------------------\n",
            "yimin zou  Biography Yimin Zou graduated from Shanghai Fudan University and was a CUSBEA (China and United States Biochemistry Examination and Application Program) student and received his Ph. D. from University of California, Davis and San Diego, in 1995. He did his postdoctoral fellowship at University of California, San Francisco from 1996 to 2000 and was an Assistant and Associate Professor at the University of Chicago from 2000 to 2006. He joined the faculty in July, 2006.  Research Axon Guidance We are interested in the guidance cues that provide directional information for axon wiring and identified the Wnt family proteins as key guidance molecules along the major axes of the central nervous system for axon pathfinding and topographic mapping. We are currently studying how these guidance cues are laid out in concentration gradients and how axonal growth cones recognize and respond to these gradients. We found that the conserved apical-basal and planar cell polarity pathways mediate growth cone turning and an apical-basal polarity signaling component, aPKC, promotes the endocytosis of a Wnt/planar cell polarity component, Frizzled3. We are testing whether this is part of an amplification mechanism for asymmetric signaling to polarize growth cones. Circuit repair after traumatic injury Traumatic injuries of the adult brain and the spinal cord lead to loss of many important functions. We found that the Wnt signaling pathways are reactivated after spinal cord injury. A reinduced Wnt inhibitory system limits motor and sensory axon growth and functional recovery. Our recent studies showed that combining molecular manipulation to enhance axon plasticity with behaviorally-guided training offers the maximal functional recovery. We are currently studying the network basis of neural circuit remodeling after spinal cord injury and traumatic brain injury. Synapse formation and neurodevelopmental disorders The signaling mechanisms that assemble neuronal synapses have been elusive. Our recent finding suggests that planar cell polarity signaling components play essential roles in glutamatergic synapse formation. This opens up new opportunities to study synaptogenesis and plasticity, fundamental to understanding neural circuit function. Mutations in planar cell polarity genes, such as the Prickles, are associated with neurodevelopmental disorders, such as autism and epilepsy. We are studying the molecular and circuit mechanisms of these diseases and exploring strategies for therapeutic interventions. \n",
            "-------------------- Scraping faculty url 183/183 --------------------\n",
            "elina zuniga  Biography Elina Zuniga received her Ph.D. in Biochemistry from the National University of Cordoba, Argentina. She conducted postdoctoral research at The Scripps Research Institute where she held two post-doctoral fellowships from Antorchas Foundation and PEW Charitable Trust, respectively. After joining UCSD in 2007 she has received a Hellman Foundation Scholar Award, The Vilcek Finalist Prize for Creative Promise, the Leukemia and Lymphoma Society Scholar Award and the American Cancer Society Scholar Award (a lifetime honor).  Research The immune system is responsible for the tremendous task of fighting a wide range of pathogens to which we are constantly exposed. This system can be broadly subdivided into innate and adaptive arms, which act in conjunction to protect the host. The innate immune system exists in both vertebrate and invertebrate organisms and represents the first barrier against microbial invasion. This arm of the immune system rapidly eliminates the vast majority of microorganisms that we daily encounter and is responsible for limiting early pathogen replication and imprinting the profile of the subsequent adaptive response. The adaptive immune response is a more sophisticated feature, found only in vertebrate animals, involving a broad repertoire of genetically rearranged receptors that specifically recognize microbial antigens. The hallmark of the adaptive response is the generation of a potent and long-lasting protection specifically directed against the invading pathogen. Despite highly evolved immune responses, pathogens have co-evolved ways to evade or subvert the immune system to favor their replication and transmission. Therefore, despite major scientific and medical efforts, infectious diseases remain among the leading causes of mortality and disability worldwide. Dramatic examples of microbial immune-evasion are chronic viral infections such as Human Immunodeficiency virus, Hepatitis B and C viruses, which currently infect about 500 million people around the world. Our laboratory studies cellular and molecular aspects of immune responses during acute and chronic viral infections to determine general principles of antiviral immunity, immune-evasion, persistence and pathogenesis. We use a combination of both standard and cutting-edge molecular approaches as well as functional assays with recombinant viruses and genetically modified hosts. Both hypothesis driven and unbiased strategies allow us to unravel novel aspects of host-virus interactions in mouse models of natural viral infections that we aim at translating to humans. The ultimate goal is to generate fundamental knowledge on immune-regulation that could help modulating immune responses to prevent or treat infectious diseases and may have implications for other immune-related illnesses. \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eF0dmFHsRIqi",
        "colab_type": "text"
      },
      "source": [
        "Finally, write urls and extracted bio to txt files"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "eL0ZjnaPRIqj",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def write_lst(lst,file_):\n",
        "  with open(file_,'w') as f:\n",
        "    for l in lst:\n",
        "        f.write(l)\n",
        "        f.write('\\n')"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "iCex1alZRIqm",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "bio_urls_file = 'bio_urls.txt'\n",
        "bios_file = 'bios.txt'\n",
        "write_lst(bio_urls,bio_urls_file)\n",
        "write_lst(bios,bios_file)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "z7a1cxjYVwGo",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        ""
      ],
      "execution_count": 0,
      "outputs": []
    }
  ]
}